{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Building a RAG application from scratch"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's start by loading the environment variables we need to use."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: langchain in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (0.2.1)\n",
      "Requirement already satisfied: langchain_core in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (0.2.1)\n",
      "Requirement already satisfied: langchain_community in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (0.2.1)\n",
      "Requirement already satisfied: jq in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (1.7.0)\n",
      "Requirement already satisfied: pypdf in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (4.2.0)\n",
      "Requirement already satisfied: bs4 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (0.0.2)\n",
      "Requirement already satisfied: pandas in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (2.2.2)\n",
      "Requirement already satisfied: numpy in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (1.26.4)\n",
      "Requirement already satisfied: matplotlib in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (3.8.4)\n",
      "Requirement already satisfied: seaborn in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (0.13.2)\n",
      "Requirement already satisfied: chromadb in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (0.4.24)\n",
      "Requirement already satisfied: pinecone-client in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (3.2.2)\n",
      "Requirement already satisfied: ragas in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (0.1.8)\n",
      "Requirement already satisfied: PyYAML>=5.3 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (6.0.1)\n",
      "Requirement already satisfied: SQLAlchemy<3,>=1.4 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (2.0.29)\n",
      "Requirement already satisfied: aiohttp<4.0.0,>=3.8.3 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (3.9.5)\n",
      "Requirement already satisfied: async-timeout<5.0.0,>=4.0.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (4.0.3)\n",
      "Requirement already satisfied: langchain-text-splitters<0.3.0,>=0.2.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (0.2.0)\n",
      "Requirement already satisfied: langsmith<0.2.0,>=0.1.17 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (0.1.52)\n",
      "Requirement already satisfied: pydantic<3,>=1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (1.10.8)\n",
      "Requirement already satisfied: requests<3,>=2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (2.31.0)\n",
      "Requirement already satisfied: tenacity<9.0.0,>=8.1.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain) (8.2.3)\n",
      "Requirement already satisfied: jsonpatch<2.0,>=1.33 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain_core) (1.33)\n",
      "Requirement already satisfied: packaging<24.0,>=23.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain_core) (23.2)\n",
      "Requirement already satisfied: dataclasses-json<0.7,>=0.5.7 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from langchain_community) (0.6.6)\n",
      "Requirement already satisfied: typing_extensions>=4.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from pypdf) (4.11.0)\n",
      "Requirement already satisfied: beautifulsoup4 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from bs4) (4.12.3)\n",
      "Requirement already satisfied: python-dateutil>=2.8.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from pandas) (2.9.0.post0)\n",
      "Requirement already satisfied: pytz>=2020.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from pandas) (2024.1)\n",
      "Requirement already satisfied: tzdata>=2022.7 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from pandas) (2024.1)\n",
      "Requirement already satisfied: contourpy>=1.0.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from matplotlib) (1.2.1)\n",
      "Requirement already satisfied: cycler>=0.10 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from matplotlib) (0.12.1)\n",
      "Requirement already satisfied: fonttools>=4.22.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from matplotlib) (4.51.0)\n",
      "Requirement already satisfied: kiwisolver>=1.3.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from matplotlib) (1.4.5)\n",
      "Requirement already satisfied: pillow>=8 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from matplotlib) (10.3.0)\n",
      "Requirement already satisfied: pyparsing>=2.3.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from matplotlib) (3.1.2)\n",
      "Requirement already satisfied: build>=1.0.3 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (1.2.1)\n",
      "Requirement already satisfied: chroma-hnswlib==0.7.3 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (0.7.3)\n",
      "Requirement already satisfied: fastapi>=0.95.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (0.110.3)\n",
      "Requirement already satisfied: uvicorn>=0.18.3 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from uvicorn[standard]>=0.18.3->chromadb) (0.25.0)\n",
      "Requirement already satisfied: posthog>=2.4.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (3.5.0)\n",
      "Requirement already satisfied: pulsar-client>=3.1.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (3.5.0)\n",
      "Requirement already satisfied: onnxruntime>=1.14.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (1.17.3)\n",
      "Requirement already satisfied: opentelemetry-api>=1.2.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (1.24.0)\n",
      "Requirement already satisfied: opentelemetry-exporter-otlp-proto-grpc>=1.2.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (1.24.0)\n",
      "Requirement already satisfied: opentelemetry-instrumentation-fastapi>=0.41b0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (0.45b0)\n",
      "Requirement already satisfied: opentelemetry-sdk>=1.2.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (1.24.0)\n",
      "Requirement already satisfied: tokenizers>=0.13.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (0.19.1)\n",
      "Requirement already satisfied: pypika>=0.48.9 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (0.48.9)\n",
      "Requirement already satisfied: tqdm>=4.65.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (4.66.2)\n",
      "Requirement already satisfied: overrides>=7.3.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (7.7.0)\n",
      "Requirement already satisfied: importlib-resources in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (6.4.0)\n",
      "Requirement already satisfied: grpcio>=1.58.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (1.64.0)\n",
      "Requirement already satisfied: bcrypt>=4.0.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (4.1.2)\n",
      "Requirement already satisfied: typer>=0.9.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (0.9.4)\n",
      "Requirement already satisfied: kubernetes>=28.1.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (29.0.0)\n",
      "Requirement already satisfied: mmh3>=4.0.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (4.1.0)\n",
      "Requirement already satisfied: orjson>=3.9.12 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from chromadb) (3.10.2)\n",
      "Requirement already satisfied: certifi>=2019.11.17 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from pinecone-client) (2024.2.2)\n",
      "Requirement already satisfied: urllib3>=1.26.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from pinecone-client) (2.2.1)\n",
      "Requirement already satisfied: datasets in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from ragas) (2.19.1)\n",
      "Requirement already satisfied: tiktoken in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from ragas) (0.7.0)\n",
      "Requirement already satisfied: langchain-openai in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from ragas) (0.1.7)\n",
      "Requirement already satisfied: openai>1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from ragas) (1.25.0)\n",
      "Requirement already satisfied: pysbd>=0.3.4 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from ragas) (0.3.4)\n",
      "Requirement already satisfied: nest-asyncio in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from ragas) (1.6.0)\n",
      "Requirement already satisfied: appdirs in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from ragas) (1.4.4)\n",
      "Requirement already satisfied: aiosignal>=1.1.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.3.1)\n",
      "Requirement already satisfied: attrs>=17.3.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (23.2.0)\n",
      "Requirement already satisfied: frozenlist>=1.1.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.4.1)\n",
      "Requirement already satisfied: multidict<7.0,>=4.5 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (6.0.5)\n",
      "Requirement already satisfied: yarl<2.0,>=1.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.9.4)\n",
      "Requirement already satisfied: pyproject_hooks in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from build>=1.0.3->chromadb) (1.1.0)\n",
      "Requirement already satisfied: tomli>=1.1.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from build>=1.0.3->chromadb) (2.0.1)\n",
      "Requirement already satisfied: marshmallow<4.0.0,>=3.18.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from dataclasses-json<0.7,>=0.5.7->langchain_community) (3.21.2)\n",
      "Requirement already satisfied: typing-inspect<1,>=0.4.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from dataclasses-json<0.7,>=0.5.7->langchain_community) (0.9.0)\n",
      "Requirement already satisfied: starlette<0.38.0,>=0.37.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from fastapi>=0.95.2->chromadb) (0.37.2)\n",
      "Requirement already satisfied: jsonpointer>=1.9 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from jsonpatch<2.0,>=1.33->langchain_core) (2.4)\n",
      "Requirement already satisfied: six>=1.9.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from kubernetes>=28.1.0->chromadb) (1.16.0)\n",
      "Requirement already satisfied: google-auth>=1.0.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from kubernetes>=28.1.0->chromadb) (2.29.0)\n",
      "Requirement already satisfied: websocket-client!=0.40.0,!=0.41.*,!=0.42.*,>=0.32.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from kubernetes>=28.1.0->chromadb) (1.8.0)\n",
      "Requirement already satisfied: requests-oauthlib in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from kubernetes>=28.1.0->chromadb) (2.0.0)\n",
      "Requirement already satisfied: oauthlib>=3.2.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from kubernetes>=28.1.0->chromadb) (3.2.2)\n",
      "Requirement already satisfied: coloredlogs in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from onnxruntime>=1.14.1->chromadb) (15.0.1)\n",
      "Requirement already satisfied: flatbuffers in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from onnxruntime>=1.14.1->chromadb) (24.3.25)\n",
      "Requirement already satisfied: protobuf in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from onnxruntime>=1.14.1->chromadb) (5.27.0)\n",
      "Requirement already satisfied: sympy in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from onnxruntime>=1.14.1->chromadb) (1.12)\n",
      "Requirement already satisfied: anyio<5,>=3.5.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from openai>1->ragas) (3.7.1)\n",
      "Requirement already satisfied: distro<2,>=1.7.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from openai>1->ragas) (1.9.0)\n",
      "Requirement already satisfied: httpx<1,>=0.23.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from openai>1->ragas) (0.27.0)\n",
      "Requirement already satisfied: sniffio in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from openai>1->ragas) (1.3.1)\n",
      "Requirement already satisfied: deprecated>=1.2.6 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-api>=1.2.0->chromadb) (1.2.14)\n",
      "Requirement already satisfied: importlib-metadata<=7.0,>=6.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-api>=1.2.0->chromadb) (7.0.0)\n",
      "Requirement already satisfied: googleapis-common-protos~=1.52 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-exporter-otlp-proto-grpc>=1.2.0->chromadb) (1.63.0)\n",
      "Requirement already satisfied: opentelemetry-exporter-otlp-proto-common==1.24.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-exporter-otlp-proto-grpc>=1.2.0->chromadb) (1.24.0)\n",
      "Requirement already satisfied: opentelemetry-proto==1.24.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-exporter-otlp-proto-grpc>=1.2.0->chromadb) (1.24.0)\n",
      "Collecting protobuf (from onnxruntime>=1.14.1->chromadb)\n",
      "  Using cached protobuf-4.25.3-cp37-abi3-manylinux2014_x86_64.whl.metadata (541 bytes)\n",
      "Requirement already satisfied: opentelemetry-instrumentation-asgi==0.45b0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-instrumentation-fastapi>=0.41b0->chromadb) (0.45b0)\n",
      "Requirement already satisfied: opentelemetry-instrumentation==0.45b0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-instrumentation-fastapi>=0.41b0->chromadb) (0.45b0)\n",
      "Requirement already satisfied: opentelemetry-semantic-conventions==0.45b0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-instrumentation-fastapi>=0.41b0->chromadb) (0.45b0)\n",
      "Requirement already satisfied: opentelemetry-util-http==0.45b0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-instrumentation-fastapi>=0.41b0->chromadb) (0.45b0)\n",
      "Requirement already satisfied: setuptools>=16.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-instrumentation==0.45b0->opentelemetry-instrumentation-fastapi>=0.41b0->chromadb) (69.5.1)\n",
      "Requirement already satisfied: wrapt<2.0.0,>=1.0.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-instrumentation==0.45b0->opentelemetry-instrumentation-fastapi>=0.41b0->chromadb) (1.16.0)\n",
      "Requirement already satisfied: asgiref~=3.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from opentelemetry-instrumentation-asgi==0.45b0->opentelemetry-instrumentation-fastapi>=0.41b0->chromadb) (3.8.1)\n",
      "Requirement already satisfied: monotonic>=1.5 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from posthog>=2.4.0->chromadb) (1.6)\n",
      "Requirement already satisfied: backoff>=1.10.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from posthog>=2.4.0->chromadb) (2.2.1)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from requests<3,>=2->langchain) (3.3.2)\n",
      "Requirement already satisfied: idna<4,>=2.5 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from requests<3,>=2->langchain) (3.7)\n",
      "Requirement already satisfied: greenlet!=0.4.17 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from SQLAlchemy<3,>=1.4->langchain) (3.0.3)\n",
      "Requirement already satisfied: huggingface-hub<1.0,>=0.16.4 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from tokenizers>=0.13.2->chromadb) (0.23.1)\n",
      "Requirement already satisfied: click<9.0.0,>=7.1.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from typer>=0.9.0->chromadb) (8.1.7)\n",
      "Requirement already satisfied: h11>=0.8 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from uvicorn>=0.18.3->uvicorn[standard]>=0.18.3->chromadb) (0.14.0)\n",
      "Requirement already satisfied: httptools>=0.5.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from uvicorn[standard]>=0.18.3->chromadb) (0.6.1)\n",
      "Requirement already satisfied: python-dotenv>=0.13 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from uvicorn[standard]>=0.18.3->chromadb) (1.0.0)\n",
      "Requirement already satisfied: uvloop!=0.15.0,!=0.15.1,>=0.14.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from uvicorn[standard]>=0.18.3->chromadb) (0.19.0)\n",
      "Requirement already satisfied: watchfiles>=0.13 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from uvicorn[standard]>=0.18.3->chromadb) (0.20.0)\n",
      "Requirement already satisfied: websockets>=10.4 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from uvicorn[standard]>=0.18.3->chromadb) (12.0)\n",
      "Requirement already satisfied: soupsieve>1.2 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from beautifulsoup4->bs4) (2.5)\n",
      "Requirement already satisfied: filelock in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from datasets->ragas) (3.14.0)\n",
      "Requirement already satisfied: pyarrow>=12.0.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from datasets->ragas) (16.0.0)\n",
      "Requirement already satisfied: pyarrow-hotfix in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from datasets->ragas) (0.6)\n",
      "Requirement already satisfied: dill<0.3.9,>=0.3.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from datasets->ragas) (0.3.8)\n",
      "Requirement already satisfied: xxhash in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from datasets->ragas) (3.4.1)\n",
      "Requirement already satisfied: multiprocess in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from datasets->ragas) (0.70.16)\n",
      "Requirement already satisfied: fsspec<=2024.3.1,>=2023.1.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from fsspec[http]<=2024.3.1,>=2023.1.0->datasets->ragas) (2024.3.1)\n",
      "Requirement already satisfied: regex>=2022.1.18 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from tiktoken->ragas) (2023.12.25)\n",
      "Requirement already satisfied: exceptiongroup in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from anyio<5,>=3.5.0->openai>1->ragas) (1.2.1)\n",
      "Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from google-auth>=1.0.1->kubernetes>=28.1.0->chromadb) (5.3.3)\n",
      "Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from google-auth>=1.0.1->kubernetes>=28.1.0->chromadb) (0.4.0)\n",
      "Requirement already satisfied: rsa<5,>=3.1.4 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from google-auth>=1.0.1->kubernetes>=28.1.0->chromadb) (4.9)\n",
      "Requirement already satisfied: httpcore==1.* in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from httpx<1,>=0.23.0->openai>1->ragas) (1.0.5)\n",
      "Requirement already satisfied: zipp>=0.5 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from importlib-metadata<=7.0,>=6.0->opentelemetry-api>=1.2.0->chromadb) (3.18.1)\n",
      "Requirement already satisfied: mypy-extensions>=0.3.0 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from typing-inspect<1,>=0.4.0->dataclasses-json<0.7,>=0.5.7->langchain_community) (1.0.0)\n",
      "Requirement already satisfied: humanfriendly>=9.1 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from coloredlogs->onnxruntime>=1.14.1->chromadb) (10.0)\n",
      "Requirement already satisfied: mpmath>=0.19 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from sympy->onnxruntime>=1.14.1->chromadb) (1.3.0)\n",
      "Requirement already satisfied: pyasn1<0.7.0,>=0.4.6 in /home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages (from pyasn1-modules>=0.2.1->google-auth>=1.0.1->kubernetes>=28.1.0->chromadb) (0.6.0)\n",
      "Using cached protobuf-4.25.3-cp37-abi3-manylinux2014_x86_64.whl (294 kB)\n",
      "Installing collected packages: protobuf\n",
      "  Attempting uninstall: protobuf\n",
      "    Found existing installation: protobuf 5.27.0\n",
      "    Uninstalling protobuf-5.27.0:\n",
      "      Successfully uninstalled protobuf-5.27.0\n",
      "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
      "grpcio-tools 1.64.0 requires protobuf<6.0dev,>=5.26.1, but you have protobuf 4.25.3 which is incompatible.\u001b[0m\u001b[31m\n",
      "\u001b[0mSuccessfully installed protobuf-4.25.3\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "%pip install langchain langchain_core langchain_community jq pypdf bs4 pandas numpy matplotlib seaborn chromadb pinecone-client ragas"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Necessary run"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For this example, we'll use a simple `StrOutputParser` to extract the answer as a string."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_core.output_parsers import StrOutputParser\n",
    "\n",
    "parser = StrOutputParser()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.embeddings import OllamaEmbeddings\n",
    "\n",
    "embeddings = OllamaEmbeddings(model=\"nomic-embed-text\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We want to provide the model with some context and the question. [Prompt templates](https://python.langchain.com/docs/modules/model_io/prompts/quick_start) are a simple way to define and reuse prompts."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.prompts import ChatPromptTemplate\n",
    "\n",
    "template = \"\"\"\n",
    "You are an AI assistant, trained to provide understandable and accurate information about pharmacogenomics and drugs.\n",
    "You will base your responses on the context and information provided. Output both your answer and a score of how confident you are,\n",
    " and also cite the references. Also provide the source of the chunks of the documents used for response.\n",
    "If the information related to the question is not in the context and or in the information provided in the prompt, \n",
    "you will say 'I don't know'.\n",
    "You are not a healthcare provider and you will not provide medical care or make assumptions about treatment.\n",
    "\n",
    "\n",
    "Context: {context}\n",
    "\n",
    "Question: {question}\n",
    "\"\"\"\n",
    "\n",
    "prompt = ChatPromptTemplate.from_template(template)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Vectorstore"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's start by loading the transcription in memory:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.document_loaders import PyPDFLoader\n",
    "from langchain_community.document_loaders import JSONLoader\n",
    "from langchain_community.document_loaders.csv_loader import CSVLoader\n",
    "\n",
    "\n",
    "folder_path = \"/home/dhanushb/Wellytics/RAG_data/all_files\"\n",
    "jsondata = []\n",
    "csvdata = []\n",
    "pdfdocs = []\n",
    "for filename in os.listdir(folder_path):\n",
    "    if filename.endswith(\".pdf\"):\n",
    "        file_path = os.path.join(folder_path, filename)\n",
    "        loader = PyPDFLoader(file_path)\n",
    "        doc = loader.load()\n",
    "        pdfdocs.extend(doc)\n",
    "    elif filename.endswith(\".csv\"):\n",
    "        file_path = os.path.join(folder_path, filename)\n",
    "        loader = CSVLoader(file_path)\n",
    "        data = loader.load()\n",
    "        csvdata.extend(data)\n",
    "    elif filename.endswith(\".json\"):\n",
    "        file_path = os.path.join(folder_path, filename)\n",
    "        loader = JSONLoader(file_path, jq_schema=\".\",json_lines=False,text_content=False)\n",
    "        data = loader.load()\n",
    "        jsondata.extend(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(page_content='HLA-B*5701: abacavir\\n2356\\n48% of the HLA-B*5701-positive patients develop a severe and potentially life-threatening hypersensitivity reaction to abacavir.\\nAbacavir is contra-indicated for HLA-B*5701-positive patients.\\navoid abacavir\\nLiterature:\\n1. \\nSousa-Pinto B et al. Pharmacogenetics of abacavir hypersensitivity: a systematic review and meta-analysis of the association with HLA-B*57:01. J Allergy Clin Immunol\\n2015;136:1092-4.e3.\\n2. \\nTangamornsuksan W et al. Association of HLA-B*5701 genotypes and abacavir-induced hypersensitivity reaction: a systematic review and meta-analysis. J Pharm Pharm\\nSci 2015;18:68-76.\\n3. \\nCargnin S et al. Diagnostic accuracy of HLA-B*57:01 screening for the prediction of abacavir hypersensitivity and clinical utility of the test: a meta-analytic review.\\nPharmacogenomics 2014;15:963-76.\\n4. \\nSaag M et al. High sensitivity of human leukocyte antigen-b*5701 as a marker for immunologically confirmed abacavir hypersensitivity in white and black patients. Clin\\nInfect Dis 2008;46:1111-8.\\n5. \\nMallal S et al. HLA-B*5701 screening for hypersensitivity to abacavir. N Engl J Med 2008;358:568-79.\\n6. \\nWaters LJ et al. Prospective HLA-B*5701 screening and abacavir hypersensitivity: a single centre experience. AIDS 2007;21:2533-4.\\n7. \\nRodríguez-Nóvoa S et al. Value of the HLA-B*5701 allele to predict abacavir hypersensitivity in Spaniards. AIDS Res Hum Retroviruses 2007;23:1374-6.\\n8. \\nZucman D et al. Prospective screening for human leukocyte antigen-B*5701 avoids abacavir hypersensitivity reaction in the ethnically mixed French HIV population. J\\nAcquir Immune Defic Syndr 2007;45:1-3.\\n9. \\nRauch A et al. Prospective genetic screening decreases the incidence of abacavir hypersensitivity reactions in the Western Australian HIV cohort study. Clin Infect Dis\\n2006;43:99-102.\\n10. \\nStekler J et al. Abacavir hypersensitivity reaction in primary HIV infection. AIDS 2006;20:1269-74.\\n11. \\nPhillips EJ et al. Clinical and immunogenetic correlates of abacavir hypersensitivity. AIDS 2005;19:979-81.\\n12. \\nHughes DA et al. Cost-effectiveness analysis of HLA B*5701 genotyping in preventing abacavir hypersensitivity. Pharmacogenetics 2004;14:335-42.\\n13. \\nMartin AM et al. Predisposition to abacavir hypersensitivity conferred by HLA-B*5701 and a haplotypic Hsp70-Hom variant. Proc Natl Acad Sci U S A 2004;101:4180-5.\\n14. \\nHughes AR et al. Association of genetic variations in HLA-B region with hypersensitivity to abacavir in some, but not all, populations. Pharmacogenomics 2004;5:203-11.\\n15. \\nHetherington S et al. Genetic variations in HLA-B region and hypersensitivity reactions to abacavir. Lancet 2002;359:1121-2.\\n16. \\nMallal S et al. Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet\\n2002;359:727-32.\\n17. \\nSmPC Ziagen (NL en VS).\\nDate 13-05-2019\\nCYP2C9 IM ANDERS: acenocoumarol\\n1868\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes problems when therapy is initiated as\\nusual (i.e. with frequent INR monitoring).\\nLiterature:\\n1. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical practice in Hungary. Environ Toxicol\\nPharmacol 2017;56:282-289.\\n2. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian patients with mechanical heart valves. Clin\\nAppl Thromb Hemost 2017;23: 876-882.\\n3. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n4. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective randomised study. Thromb Haemost\\n2016;115:117-25.\\n5. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian) population. Eur J Clin Pharmacol\\n2015;71:173-81.\\n6. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014;15:987-\\n96.\\n8. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n9. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n10. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during acenocoumarol treatment. Pharmacogenomics\\n2013:14;745-53.\\n11. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n12. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n13. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n14. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n16. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n17. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.\\n18. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n19. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb\\nHaemost 2007;5:1701-6.\\n20. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip replacement surgery. Mol Diagn Ther\\n2007;11:123-8.\\n21. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n23. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n24. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n25. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;1427-33.\\n28. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther 2003;7461-8.\\n29. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and stability of anticoagulation.\\nHaematologica 2002;87:1185-91.\\n31. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol. Blood 2002;99:4237-9.\\n32. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol. Pharmacogenetics 2000;10:757-60.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 0}),\n",
       " Document(page_content='Date 14-05-2018\\nCYP2C9 PM ANDERS: acenocoumarol\\n1869\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes problems when therapy is initiated as\\nusual (i.e. with frequent INR monitoring).\\nLiterature:\\n1. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical practice in Hungary. Environ Toxicol\\nPharmacol 2017;56:282-289.\\n2. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian patients with mechanical heart valves. Clin\\nAppl Thromb Hemost 2017;23: 876-882.\\n3. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n4. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective randomised study. Thromb Haemost\\n2016;115:117-25.\\n5. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian) population. Eur J Clin Pharmacol\\n2015;71:173-81.\\n6. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014;15:987-\\n96.\\n8. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n9. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.Gschwind L et al. Impact of\\nCYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during acenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n10. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n11. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n12. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n13. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n14. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n15. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n16. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.\\n17. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n18. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb\\nHaemost 2007;5:1701-6.\\n19. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip replacement surgery. Mol Diagn Ther\\n2007;11:123-8.\\n20. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n21. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n22. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n23. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n24. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n25. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n26. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;1427-33.\\n27. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther 2003;7461-8.\\n28. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n29. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and stability of anticoagulation.\\nHaematologica 2002;87:1185-91.\\n30. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol. Blood 2002;99:4237-9.\\n31. \\nVerstuyft C et al. Early acenocoumarol overanticoagulation among cytochrome P450 2C9 poor metabolizers. Pharmacogenetics 2001;11:735-7.\\n32. \\nThijssen HH et al. Altered pharmacokinetics of R- and S-acenocoumarol in a subject heterozygous for CYP2C9*3. Clin Pharmacol Ther 2001;70:292-8.\\n33. \\nRettie AE et al. A case study of acenocoumarol sensitivity and genotype-phenotype discordancy explained by combinations of polymorphisms in VKORC1 and CYP2C9. Br\\nJ Clin Pharmacol 2006;62:617-20.\\n34. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol. Pharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*1/*2: acenocoumarol\\n1863\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes problems when therapy is initiated as\\nusual (i.e. with frequent INR monitoring).\\nLiterature:\\n1. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical practice in Hungary. Environ Toxicol\\nPharmacol 2017;56:282-289.\\n2. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian patients with mechanical heart valves. Clin\\nAppl Thromb Hemost 2017;23: 876-882.\\n3. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n4. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective randomised study. Thromb Haemost\\n2016;115:117-25.\\n5. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian) population. Eur J Clin Pharmacol\\n2015;71:173-81.\\n6. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014;15:987-\\n96.\\n8. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n9. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n10. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during acenocoumarol treatment. Pharmacogenomics\\n2013:14;745-53.\\n11. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n12. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n13. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n14. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n16. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n17. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 1}),\n",
       " Document(page_content='18. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n19. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb\\nHaemost 2007;5:1701-6.\\n20. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip replacement surgery. Mol Diagn Ther\\n2007;11:123-8.\\n21. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n23. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n24. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n25. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;1427-33.\\n28. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther 2003;7461-8.\\n29. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and stability of anticoagulation.\\nHaematologica 2002;87:1185-91.\\n31. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol. Blood 2002;99:4237-9.\\n32. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol. Pharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*1/*3: acenocoumarol\\n1864\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes problems when therapy is initiated as\\nusual (i.e. with frequent INR monitoring).\\nLiterature:\\n1. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical practice in Hungary. Environ Toxicol\\nPharmacol 2017;56:282-289.\\n2. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian patients with mechanical heart valves. Clin\\nAppl Thromb Hemost 2017;23: 876-882.\\n3. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n4. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective randomised study. Thromb Haemost\\n2016;115:117-25.\\n5. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian) population. Eur J Clin Pharmacol\\n2015;71:173-81.\\n6. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014;15:987-\\n96.\\n8. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n9. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n10. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during acenocoumarol treatment. Pharmacogenomics\\n2013:14;745-53.\\n11. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n12. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n13. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n14. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n16. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n17. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.\\n18. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n19. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb\\nHaemost 2007;5:1701-6.\\n20. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip replacement surgery. Mol Diagn Ther\\n2007;11:123-8.\\n21. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n23. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n24. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n25. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;1427-33.\\n28. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther 2003;7461-8.\\n29. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and stability of anticoagulation.\\nHaematologica 2002;87:1185-91.\\n31. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol. Blood 2002;99:4237-9.\\n32. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol. Pharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*2/*2: acenocoumarol\\n1865\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes problems when therapy is initiated as\\nusual (i.e. with frequent INR monitoring).\\nLiterature:\\n1. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical practice in Hungary. Environ Toxicol', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 2}),\n",
       " Document(page_content='Pharmacol 2017;56:282-289.\\n2. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian patients with mechanical heart valves. Clin\\nAppl Thromb Hemost 2017;23: 876-882.\\n3. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n4. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective randomised study. Thromb Haemost\\n2016;115:117-25.\\n5. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n6. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014;15:987-\\n96.\\n7. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n8. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n9. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during acenocoumarol treatment. Pharmacogenomics\\n2013:14;745-53.\\n10. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n11. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n12. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n13. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n14. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n15. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n16. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.\\n17. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n18. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb\\nHaemost 2007;5:1701-6.\\n19. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip replacement surgery. Mol Diagn Ther\\n2007;11:123-8.\\n20. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n21. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n22. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n23. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n24. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n25. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n26. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;1427-33.\\n27. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther 2003;7461-8.\\n28. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n29. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and stability of anticoagulation.\\nHaematologica 2002;87:1185-91.\\n30. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol. Blood 2002;99:4237-9.\\nDate 14-05-2018\\nCYP2C9*2/*3: acenocoumarol\\n1866\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes problems when therapy is initiated as\\nusual (i.e. with frequent INR monitoring).\\nLiterature:\\n1. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical practice in Hungary. Environ Toxicol\\nPharmacol 2017;56:282-289.\\n2. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian patients with mechanical heart valves. Clin\\nAppl Thromb Hemost 2017;23: 876-882.\\n3. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n4. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective randomised study. Thromb Haemost\\n2016;115:117-25.\\n5. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian) population. Eur J Clin Pharmacol\\n2015;71:173-81.\\n6. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014;15:987-\\n96.\\n8. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n9. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n10. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during acenocoumarol treatment. Pharmacogenomics\\n2013:14;745-53.\\n11. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n12. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n13. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n14. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n16. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n17. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.\\n18. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n19. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb\\nHaemost 2007;5:1701-6.\\n20. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip replacement surgery. Mol Diagn Ther\\n2007;11:123-8.\\n21. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n23. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n24. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n25. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;1427-33.\\n28. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther 2003;7461-8.\\n29. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and stability of anticoagulation.\\nHaematologica 2002;87:1185-91.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 3}),\n",
       " Document(page_content='31. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol. Pharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*3/*3: acenocoumarol\\n1867\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose and possibly an increase in the time needed to reach a stable INR. However, there is insufficient\\nevidence that this causes problems when therapy is initiated as usual (i.e. with frequent INR monitoring).\\nLiterature:\\n1. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical practice in Hungary. Environ Toxicol\\nPharmacol 2017;56:282-289.\\n2. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n3. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective randomised study. Thromb Haemost\\n2016;115:117-25.\\n4. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n5. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n6. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n7. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during acenocoumarol treatment. Pharmacogenomics\\n2013:14;745-53.\\n8. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n9. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n10. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n11. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n12. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n13. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n14. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n15. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb\\nHaemost 2007;5:1701-6.\\n16. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip replacement surgery. Mol Diagn Ther\\n2007;11:123-8.\\n17. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n18. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n19. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n20. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n21. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n22. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n23. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n24. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol. Blood 2002;99:4237-9.\\n25. \\nVerstuyft C et al. Early acenocoumarol overanticoagulation among cytochrome P450 2C9 poor metabolizers. Pharmacogenetics 2001;11:735-7.\\nDate 14-05-2018\\nVKORC1 -1639 AA: acenocoumarol\\n1910\\nAn INR ≥ 6, resulting in an increased risk of bleeding, occurs in 8-12% of these patients during the first weeks of treatment with standard regulation by the Anticoagulation\\nClinic. The genetic variation increases the sensitivity to acenocoumarol.\\nMonitoring by the ANTICOAGULATION CLINIC (National INR Monitoring Service):\\nrecommend to use 50% of the standard initial dose\\nOTHERWISE:\\nrecommend to use 50% of the standard initial dose\\nrecommend more frequent monitoring of the INR\\nThe initial dose and the maintenance dose can be calculated using an algorithm.\\nHowever, for patients with two or more VKORC1 and/or CYP2C9 variations, the algorithm used in EU-PACT (see\\nhttps://www.knmp.nl/patientenzorg/medicatiebewaking/farmacogenetica for a calculation tool in the form of an Excel file) did not result in a significant reduction in the\\nincidence of INRs above the target range when compared to an algorithm without genetic information. We are therefore unable to recommend the use of this algorithm at this\\ntime.\\nA (non-validated) algorithm has been prescribed for children that should result in a better prediction of the maintenance dose for AA than the current guideline used by the\\nAnticoagulation Clinic (Maagdenberg H et al. The pediatric acenocoumarol dosing algorithm: The Children Anticoagulation and Pharmacogenetics Study. J Thromb Haemost\\n2018 Jun 23 [Epub ahead of print]. PubMed PMID: 29935043).\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014; 15:\\n987-96.\\n3. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. NEJM 2013;369: 2304-12.\\n4. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n5. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n6. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n7. \\nKovac MK et al. The c.-1639G>A polymorphism of the VKORC1 gene in Serbian population: retrospective study of the variability in response to oral anticoagulant\\ntherapy. Blood Coagul Fibrinolysis 2010;21:558-63.\\n8. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n9. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n10. \\nStepien E et al. A vitamin K epoxide reductase-oxidase complex gene polymorphism (-1639G>A) and interindividual variability in the dose-effect of vitamin K antagonists.\\nJ Appl Genet 2009;50:399-403.\\n11. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n12. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n13. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.\\n14. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n15. \\nGonzalez-Conejero R et al. The genetic interaction of VKORC1 c1173t/calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb Haemost\\n2007;5:1701-6.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 4}),\n",
       " Document(page_content='16. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n17. \\nMontes R et al. The c.-1639G > A polymorphism of the VKORC1 gene is a major determinant of the response to acenocoumarol in anticoagulated patients. Br J Haematol\\n2006;133:183-7.\\n18. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med 2005;2:e312.\\n19. \\nBodin L et al. Cytochrome P450 2C9 (CYP2C9) and vitamin K epoxide reductase (VKORC1) genotypes as determinants of acenocoumarol sensitivity. Blood 2005;106:135-\\n140.\\nDate 10-09-2018\\nVKORC1 -1639 GA: acenocoumarol\\n1909\\nNO action is needed for this gene-drug interaction.\\nThe genetic variation results in a reduction of the required dose, but with the current practice of initiating or reviewing treatment this results in little or no increased risk of\\nbleeding or excessive anticoagulation.\\nLiterature:\\n1. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol treatment. Pharmacogenomics 2014; 15:\\n987-96.\\n2. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. NEJM 2013;369: 2304-12.\\n3. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb Haemost 2012;10:606-14.\\n4. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese people. J Clin Pharmacol 2011;51:1418-28.\\n5. \\nKovac MK et al. The c.-1639G>A polymorphism of the VKORC1 gene in Serbian population: retrospective study of the variability in response to oral anticoagulant\\ntherapy. Blood Coagul Fibrinolysis 2010;21:558-63.\\n6. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n7. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral anticoagulants. Mol Diagn Ther 2010;14:23-30.\\n8. \\nStepien E et al. A vitamin K epoxide reductase-oxidase complex gene polymorphism (-1639G>A) and interindividual variability in the dose-effect of vitamin K antagonists.\\nJ Appl Genet 2009;50:399-403.\\n9. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol anticoagulation during the initial treatment\\nperiod. Clin Pharmacol Ther 2009;85:379-86.\\n10. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in anticoagulated patients. Br J Haematol 2008;143:727-\\n33.\\n11. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients. Pharmacogenomics 2008;9:1631-8.\\n12. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant therapy. Pharmacogenomics\\n2008;9:1237-50.\\n13. \\nGonzalez-Conejero R et al. The genetic interaction of VKORC1 c1173t/calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb Haemost\\n2007;5:1701-6.\\n14. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both genotypes affects overanticoagulation. Clin\\nPharmacol Ther 2006;80:13-22.\\n15. \\nMontes R et al. The c.-1639G > A polymorphism of the VKORC1 gene is a major determinant of the response to acenocoumarol in anticoagulated patients. Br J Haematol\\n2006;133:183-7.\\n16. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med 2005;2:e312.\\n17. \\nBodin L et al. Cytochrome P450 2C9 (CYP2C9) and vitamin K epoxide reductase (VKORC1) genotypes as determinants of acenocoumarol sensitivity. Blood 2005;106:135-\\n140.\\nDate 10-09-2018\\nABCG2 141KK: allopurinol\\n7255\\nThe effectiveness of allopurinol is reduced, meaning that a higher dose is required. The gene variation reduces the excretion of uric acid by kidneys and intestines, meaning\\nthat a stronger inhibition of the uric acid production by allopurinol is required to achieve the desired uric acid concentration.\\nuse 1.4 times the standard dose\\nThis equates to a dose titration schedule of 100, 300, 400, 600 and 700 mg/day instead of the usual schedule of 100, 200, 300, 400 and 500 mg/day.\\nLiterature:\\n1. \\nStamp LK et al. Relationships between allopurinol dose, oxypurinol concentration and urate-lowering response -in search of a minimum effective oxypurinol\\nconcentration. Clin Transl Sci 2020;13:110-5.\\n2. \\nBrackman DJ et al. Genome-wide association and functional studies reveal novel pharmacological mechanisms for allopurinol. Clin Pharmacol Ther 2019;106:623-31.\\n3. \\nZhang K et al. ABCG2 gene polymorphism rs2231142 is associated with gout comorbidities but not allopurinol response in primary gout patients of a Chinese Han male\\npopulation. Hereditas 2019;156:26.\\n4. \\nWright DFB et al. The impact of diuretic use and ABCG2 genotype on the predictive performance of a published allopurinol dosing tool. Br J Clin Pharmacol 2018;84:937-\\n43.\\n5. \\nWallace MC et al. Association between ABCG2 rs2231142 and poor response to allopurinol: replication and meta-analysis. Rheumatology (Oxford) 2018;57:656-60.\\n6. \\nRoberts RL et al. ABCG2 loss-of-function polymorphism predicts poor response to allopurinol in patients with gout. Pharmacogenomics J 2017;17:201-3.\\n7. \\nWen CC et al. Genome-wide association study identifies ABCG2 (BCRP) as an allopurinol transporter and a determinant of drug response. Clin Pharmacol Ther\\n2015;97:518-25.\\nDate 07-06-2021\\nABCG2 141QK: allopurinol\\n7256\\nThe effectiveness of allopurinol is reduced, meaning that a higher dose is required. The gene variation reduces the excretion of uric acid by kidneys and intestines, meaning\\nthat a stronger inhibition of the uric acid production by allopurinol is required to achieve the desired uric acid concentration.\\nuse 1.25 times the standard dose\\nThis equates to a dose titration schedule of 100, 200, 400 and 500 mg/day instead of the usual schedule of 100, 200, 300 and 400 mg/day.\\nLiterature:\\n1. \\nStamp LK et al. Relationships between allopurinol dose, oxypurinol concentration and urate-lowering response -in search of a minimum effective oxypurinol\\nconcentration. Clin Transl Sci 2020;13:110-5.\\n2. \\nBrackman DJ et al. Genome-wide association and functional studies reveal novel pharmacological mechanisms for allopurinol. Clin Pharmacol Ther 2019;106:623-31.\\n3. \\nZhang K et al. ABCG2 gene polymorphism rs2231142 is associated with gout comorbidities but not allopurinol response in primary gout patients of a Chinese Han male\\npopulation. Hereditas 2019;156:26.\\n4. \\nTorres RJ et al. GLUT9 influences uric acid concentration in patients with Lesch-Nyhan disease. Int J Rheum Dis 2018;21:1270-6.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 5}),\n",
       " Document(page_content='5. \\nWright DFB et al. The impact of diuretic use and ABCG2 genotype on the predictive performance of a published allopurinol dosing tool. Br J Clin Pharmacol 2018;84:937-\\n43.\\n6. \\nWallace MC et al. Association between ABCG2 rs2231142 and poor response to allopurinol: replication and meta-analysis. Rheumatology (Oxford) 2018;57:656-60.\\n7. \\nRoberts RL et al. ABCG2 loss-of-function polymorphism predicts poor response to allopurinol in patients with gout. Pharmacogenomics J 2017;17:201-3.\\n8. \\nPetru L et al. Genetic background of uric acid metabolism in a patient with severe chronic tophaceous gout. Clin Chim Acta 2016;460:46-9.\\n9. \\nWen CC et al. Genome-wide association study identifies ABCG2 (BCRP) as an allopurinol transporter and a determinant of drug response. Clin Pharmacol Ther\\n2015;97:518-25.\\nDate 07-06-2021\\nHLA-B*5801: allopurinol\\n6391\\nA strongly increased risk of developing the life-threatening cutaneous side effects Stevens-Johnson syndrome/toxic epidermal necrolysis (SJS/TEN) and DRESS is present in\\npatients with this genetic variation. The risk of an allopurinol-induced life-threatening cutaneous side effect (mortality 11%) in these patients is 1.6-2.0% for the entire group\\nand 8-18% for the group with chronic renal insufficiency.\\nChoose an alternative, such as febuxostat\\nAnother option is to induce allopurinol tolerance first:\\nTo induce allopurinol tolerance, the allopurinol dose is increased every 3 days until a dose of 100 mg/day has been achieved on Day 28. The consecutive daily doses in the\\ninduction protocol are 50 µg, 100 µg, 200 µg, 500 µg, 1 mg, 5 mg, 10 mg, 25 mg, 50 mg and 100 mg.\\nLiterature:\\n1. \\nPark HW et al. Efficacy of the HLA-B*58:01 screening test in preventing allopurinol-induced severe cutaneous adverse reactions in patients with chronic renal\\ninsufficiency-a prospective study. J Allergy Clin Immunol Pract 2019;7:1271-6.\\n2. \\nYu KH et al. Diagnostic utility of HLA-B*5801 screening in severe allopurinol hypersensitivity syndrome: an updated systematic review and meta-analysis. Int J Rheum Dis\\n2017;20:1057-71.\\n3. \\nSaksit N et al. Risk factors of allopurinol-induced severe cutaneous adverse reactions in a Thai population. Pharmacogenet Genomics 2017;27:255-63.\\n4. \\nWu R et al. Impact of HLA-B*58:01 allele and allopurinol-induced cutaneous adverse drug reactions: evidence from 21 pharmacogenetic studies. Oncotarget\\n2016;7:81870-9.\\n5. \\nNg CY et al. Impact of the HLA-B(*)58:01 allele and renal impairment on allopurinol-induced cutaneous adverse reactions. J Invest Dermatol 2016;136:1373-81.\\n6. \\nJung JW et al. An effective strategy to prevent allopurinol-induced hypersensitivity by HLA typing. Genet Med 2015;17:807-14.\\n7. \\nKo TM et al. Use of HLA-B*58:01 genotyping to prevent allopurinol induced severe cutaneous adverse reactions in Taiwan: national prospective cohort study. BMJ\\n2015;351:h4848.\\n8. \\nCheng L et al. HLA-B*58:01 is strongly associated with allopurinol-induced severe cutaneous adverse reactions in Han Chinese patients: a multicentre retrospective case-\\ncontrol clinical study. Br J Dermatol 2015;173:555-8.\\n9. \\nJung JW et al. HLA-DR9 and DR14 are associated with the allopurinol-induced hypersensitivity in hematologic malignancy. Tohoku J Exp Med 2014;233:95-102.\\n10. \\nZineh I et al. Allopurinol pharmacogenetics: assessment of potential clinical usefulness. Pharmacogenomics 2011;12:1741-9.\\n11. \\nSomkrua R et al. Association of HLA-B*5801 allele and allopurinol-induced Stevens Johnson syndrome and toxic epidermal necrolysis: a systematic review and meta-\\nanalysis. BMC Med Genet 2011;12:118.\\n12. \\nLonjou C et al. A European study of HLA-B in Stevens-Johnson syndrome and toxic epidermal necrolysis related to five high-risk drugs. Pharmacogenet Genomics\\n2008;18:99-107.\\n13. \\nSPC Zyloric.\\nDate 07-06-2021\\nCYP2D6 IM: amiodaron\\n2543\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2D6 PM: amiodaron\\n2542\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2D6 UM: amiodaron\\n2544\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2C19 IM: amitriptyline\\n7024\\nNO action is required for this gene-drug interaction.\\nThe gene variation has an effect on the exposure to amitriptyline, but not on the exposure to amitriptyline + the active metabolite nortriptyline, which determines the effect\\nand side effects.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 6}),\n",
       " Document(page_content='Literature:\\n1. \\nRyu S et al. A study on CYP2C19 and CYP2D6 polymorphic effects on pharmacokinetics and pharmacodynamics of amitriptyline in healthy Koreans. Clin Transl Sci\\n2017;10:93-101.\\n2. \\nAtasayar G et al. Association of MDR1, CYP2D6, and CYP2C19 gene polymorphisms with prophylactic migraine treatment response. J Neurol Sci 2016;366:149-154.\\n3. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n4. \\nvan der Weide J et al. Metabolic ratios of psychotropics as indication of cytochrome P450 2D6/2C19 genotype. Ther Drug Monit 2005;27:478-83.\\n5. \\nSteimer W et al. Amitriptyline or not, that is the question: pharmacogenetic testing of CYP2D6 and CYP2C19 identifies patients with low or high risk for side effects in\\namitriptyline therapy. Clin Chem 2005;51:376-85.\\n6. \\nSteimer W et al. Allele-specific change of concentration and functional gene dose for the prediction of steady-state serum concentrations of amitriptyline and nortriptyline\\nin CYP2C19 and CYP2D6 extensive and intermediate metabolizers. Clin Chem 2004;50:1623-33.\\n7. \\nShimoda K et al. The impact of CYP2C19 and CYP2D6 genotypes on metabolism of amitriptyline in Japanese psychiatric patients. J Clin Psychopharmacol 2002;22:371-8.\\n8. \\nJiang ZP et al. The role of CYP2C19 in amitriptyline N-demethylation in Chinese subjects. Eur J Clin Pharmacol 2002;58:109-13.\\nDate 04-03-2019\\nCYP2C19 PM: amitriptyline\\n7025\\nNO action is required for this gene-drug interaction.\\nThe gene variation has an effect on the exposure to amitriptyline, but not on the exposure to amitriptyline + the active metabolite nortriptyline, which determines the effect\\nand side effects.\\nLiterature:\\n1. \\nRyu S et al. A study on CYP2C19 and CYP2D6 polymorphic effects on pharmacokinetics and pharmacodynamics of amitriptyline in healthy Koreans. Clin Transl Sci\\n2017;10:93-101.\\n2. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n3. \\nvan der Weide J et al. Metabolic ratios of psychotropics as indication of cytochrome P450 2D6/2C19 genotype. Ther Drug Monit 2005;27:478-83.\\n4. \\nSteimer W et al. Amitriptyline or not, that is the question: pharmacogenetic testing of CYP2D6 and CYP2C19 identifies patients with low or high risk for side effects in\\namitriptyline therapy. Clin Chem 2005;51:376-85.\\n5. \\nSteimer W et al. Allele-specific change of concentration and functional gene dose for the prediction of steady-state serum concentrations of amitriptyline and nortriptyline\\nin CYP2C19 and CYP2D6 extensive and intermediate metabolizers. Clin Chem 2004;50:1623-33.\\n6. \\nShimoda K et al. The impact of CYP2C19 and CYP2D6 genotypes on metabolism of amitriptyline in Japanese psychiatric patients. J Clin Psychopharmacol 2002;22:371-8.\\n7. \\nJiang ZP et al. The role of CYP2C19 in amitriptyline N-demethylation in Chinese subjects. Eur J Clin Pharmacol 2002;58:109-13.\\n8. \\nSmPC Amitriptyline HCl Apotex.\\nDate 04-03-2019\\nCYP2C19 UM: amitriptyline\\n7026\\nNO action is required for this gene-drug interaction.\\nThe gene variation decreases the exposure to amitriptyline and increases the exposure to the active metabolite nortriptyline, but there is no evidence to indicate that this\\nresults in an increase in side effects or a decrease in efficacy. A higher dose is required to achieve the therapeutic range of amitriptyline+nortriptyline, but the therapeutic\\nrange of nortriptyline is achieved at the lower dose.\\nLiterature:\\n1. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\nDate 04-03-2019\\nCYP2D6 IM: amitriptyline\\n1920\\nThe risk of side effects is increased, because the gene variation leads to higher plasma concentrations of the active metabolite nortriptyline and to a lesser extent of\\namitriptyline.\\nRecommendation:\\nuse 75% of the standard dose and monitor the efficacy and side effects or the plasma concentrations of amitriptyline and nortriptyline to adjust the maintenance dose\\nLiterature:\\n1. \\nChaudhry M et al. Impact of CYP2D6 genotype on amitriptyline efficacy for the treatment of diabetic peripheral neuropathy: a pilot study. Pharmacogenomics\\n2017;18:433-443.\\n2. \\nRyu S et al. A study on CYP2C19 and CYP2D6 polymorphic effects on pharmacokinetics and pharmacodynamics of amitriptyline in healthy Koreans. Clin Transl Sci\\n2017;10:93-101.\\n3. \\nAtasayar G et al. Association of MDR1, CYP2D6, and CYP2C19 gene polymorphisms with prophylactic migraine treatment response. J Neurol Sci 2016;366:149-154.\\n4. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n5. \\nKoski A et al. CYP2D6 and CYP2C19 genotypes and amitriptyline metabolite ratios in a series of medicolegal autopsies. Forensic Sci Int 2006;158:177-83.\\n6. \\nSteimer W et al. Amitriptyline or not, that is the question: pharmacogenetic testing of CYP2D6 and CYP2C19 identifies patients with low or high risk for side effects in\\namitriptyline therapy. Clin Chem 2005;51:376-85.\\n7. \\nSteimer W et al. Allele-specific change of concentration and functional gene dose for the prediction of steady-state serum concentrations of amitriptyline and nortriptyline\\nin CYP2C19 and CYP2D6 extensive and intermediate metabolizers. Clin Chem 2004;50:1623-33.\\n8. \\nShimoda K et al. The impact of CYP2C19 and CYP2D6 genotypes on metabolism of amitriptyline in Japanese psychiatric patients. J Clin Psychopharmacol 2002;22:371-8.\\n9. \\nBreyer-Pfaff U et al. Enantioselective amitriptyline metabolism in patients phenotyped for two cytochrome P450 isozymes. Clin Pharmacol Ther 1992;52:350-8.\\n10. \\nMellstrom B et al. Amitriptyline metabolism: association with debrisoquin hydroxylation in nonsmokers. Clin Pharmacol Ther 1986;39:369-71.\\n11. \\nBaumann P et al. Amitriptyline pharmacokinetics and clinical response: II. Metabolic polymorphism assessed by hydroxylation of debrisoquine and mephenytoin. Int Clin\\nPsychopharmacol 1986;1:102-12.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 7}),\n",
       " Document(page_content='Date 10-09-2018\\nCYP2D6 PM: amitriptyline\\n1921\\nIn theory, risk of side effects is increased, because the genetic variation results in higher plasma concentrations of the active metabolite nortriptyline and to a lesser extent of\\namitriptyline.\\nuse 70% of the standard dose and monitor the effect and side effects or the plasma concentrations of amitriptyline and nortriptyline to adjust the maintenance dose\\nLiterature:\\n1. \\nAtasayar G et al. Association of MDR1, CYP2D6, and CYP2C19 gene polymorphisms with prophylactic migraine treatment response. J Neurol Sci 2016;366:149-154.\\n2. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n3. \\nHalling J et al. The CYP2D6 polymorphism in relation to the metabolism of amitriptyline and nortriptyline in the Faroese population. Br J Clin Pharmacol 2008;65:134-8.\\n4. \\nKoski A et al. CYP2D6 and CYP2C19 genotypes and amitriptyline metabolite ratios in a series of medicolegal autopsies. Forensic Sci Int 2006;158:177-83.\\n5. \\nSteimer W et al. Amitriptyline or not, that is the question: pharmacogenetic testing of CYP2D6 and CYP2C19 identifies patients with low or high risk for side effects in\\namitriptyline therapy. Clin Chem 2005;51:376-85.\\n6. \\nBreyer-Pfaff U et al. Enantioselective amitriptyline metabolism in patients phenotyped for two cytochrome P450 isozymes. Clin Pharmacol Ther 1992;52:350-8.\\n7. \\nMellstrom B et al. Amitriptyline metabolism: association with debrisoquin hydroxylation in nonsmokers. Clin Pharmacol Ther 1986;39:369-71.\\n8. \\nBaumann P et al. Amitriptyline pharmacokinetics and clinical response: II. Metabolic polymorphism assessed by hydroxylation of debrisoquine and mephenytoin. Int Clin\\nPsychopharmacol 1986;1:102-12.\\n9. \\nSmPC’s Amitriptyline HCl Apotex (NL) en Amitriptyline Hydrochloride Sandoz (VS).\\nDate 10-09-2018\\nCYP2D6 UM: amitriptyline\\n1922\\nThe risk of ineffectiveness is increased and the risk of cardiotoxic side effects may be increased. The gene variation leads to increased conversion of amitriptyline and the\\nactive metabolite nortriptyline to less active and inactive metabolites.\\nincrease the dose to 1.4 times the standard dose, monitor the effect and side effects or the plasma concentrations and be alert to increased plasma concentrations of the\\ncardiotoxic Z-10-hydroxy metabolites.\\nPlasma concentrations of Z-hydroxy nortriptyline or Z-hydroxy amitriptyline higher than 40 ng/mL are considered toxic.\\nif a dose increase is not desirable due to the cardiotoxic hydroxy metabolite: avoid amitriptyline Anti-depressants that are not metabolised by CYP2D6, or to a lesser\\nextent, include citalopram and sertraline.\\nLiterature:\\n1. \\nChaudhry M et al. Impact of CYP2D6 genotype on amitriptyline efficacy for the treatment of diabetic peripheral neuropathy: a pilot study. Pharmacogenomics\\n2017;18:433-443.\\n2. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n3. \\nSteimer W et al. Allele-specific change of concentration and functional gene dose for the prediction of steady-state serum concentrations of amitriptyline and nortriptyline\\nin CYP2C19 and CYP2D6 extensive and intermediate metabolizers. Clin Chem 2004;50:1623-33.\\n4. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and clinical effects of antidepressants in a\\nnaturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n5. \\nBertilsson L et al. Extremely rapid hydroxylation of debrisoquine: a case report with implication for treatment with nortriptyline and other tricyclic antidepressants. Ther\\nDrug Monit 1985;7:478-80.\\n6. \\nBaumann P et al. Amitriptyline pharmacokinetics and clinical response: II. Metabolic polymorphism assessed by hydroxylation of debrisoquine and mephenytoin. Int Clin\\nPsychopharmacol 1986;1:102-12.\\nDate 10-09-2018\\nCYP2D6 IM: aripiprazol\\n1541\\nNO action is needed for this gene-drug interaction.\\nThe genetic variation increases the plasma concentration of the sum of aripiprazole and the active metabolite dehydroaripiprazole to a limited degree. There is insufficient\\nevidence that this increases the risk of side effects.\\nLiterature:\\n1. \\nJallaq SA et al. CYP2D6 phenotype influences aripiprazole tolerability in pediatric patients with mood disorders. J Child Adolesc Psychopharmacol 2021;31:56-62.\\n2. \\nKoller D et al. The effects of aripiprazole and olanzapine on pupillary light reflex and its relationship with pharmacogenetics in a randomized multiple-dose trial. Br J Clin\\nPharmacol 2020;86:2051-62. en persoonlijke communicatie (supplementaire files)\\n3. \\nTveito M et al. Impact of age and CYP2D6 genetics on exposure of aripiprazole and dehydroaripiprazole in patients using long-acting injectable versus oral formulation:\\nrelevance of poor and intermediate metabolizer status. Eur J Clin Pharmacol 2020;76:41-9.\\n4. \\nJukic MM et al. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 2019;6(5):418-26. en\\npersoonlijke communicatie (correcte aantal patiënten per fenotype en per genotypegroep in de kineticasubgroup)\\n5. \\nBelmonte C et al. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 polymorphisms on pharmacokinetics and safety of aripiprazole in healthy volunteers. Basic Clin\\nPharmacol Toxicol 2018;122:596-605.\\n6. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n7. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\n8. \\nSuzuki T et al. Effects of genetic polymorphisms of CYP2D6, CYP3A5, and ABCB1 on the steady-state plasma concentrations of aripiprazole and its active metabolite,\\ndehydroaripiprazole, in Japanese patients with schizophrenia. Ther Drug Monit 2014;36:651-5.\\n9. \\nSuzuki T et al. Effects of the CYP-2D6*10 allele on the steady-state plasma concentrations of aripiprazole and its active metabolite, dehydroaripiprazole, in Japanese\\npatients with schizophrenia. Ther Drug Monit 2011;33:21-4.\\n10. \\nHendset M et al. Impact of the CYP2D6 genotype on steady-state serum concentrations of aripiprazole and dehydroaripiprazole. Eur J Clin Pharmacol 2007;63:1147-51.\\n11. \\nKubo M et al. Pharmacokinetics of aripiprazole, a new antipsychotic, following oral dosing in healthy adult Japanese volunteers: influence of CYP2D6 polymorphism. Drug\\nMetab Pharmacokinet 2007;22:358-66.\\n12. \\nKim E et al. Effects of DRD2 and CYP2D6 genotypes on delta EEG power response to aripiprazole in healthy male volunteers: a preliminary study. Hum Psychopharmacol\\n2006;21:519-28.\\n13. \\nKubo M et al. Influence of itraconazole co-administration and CYP2D6 genotype on the pharmacokinetics of the new antipsychotic aripiprazole. Drug Metab\\nPharmacokinet 2005;20:55-64.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 8}),\n",
       " Document(page_content='Date 13-09-2021\\nCYP2D6 PM: aripiprazol\\n1542\\nThe risk of side effects is increased. The genetic variation leads to an increase in the sum of the plasma concentrations of aripiprazole and the active metabolite.\\nadminister no more than 10 mg/day or 300 mg/month (68-75% of the standard maximum dose of aripiprazole).\\nLiterature:\\n1. \\nJallaq SA et al. CYP2D6 phenotype influences aripiprazole tolerability in pediatric patients with mood disorders. J Child Adolesc Psychopharmacol 2021;31:56-62.\\n2. \\nTveito M et al. Impact of age and CYP2D6 genetics on exposure of aripiprazole and dehydroaripiprazole in patients using long-acting injectable versus oral formulation:\\nrelevance of poor and intermediate metabolizer status. Eur J Clin Pharmacol 2020;76:41-9.\\n3. \\nJukic MM et al. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 2019;6(5):418-26. en\\npersoonlijke communicatie (correcte aantal patiënten per fenotype en per genotypegroep in de kineticasubgroup)\\n4. \\nBelmonte C et al. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 polymorphisms on pharmacokinetics and safety of aripiprazole in healthy volunteers. Basic Clin\\nPharmacol Toxicol 2018;122:596-605.\\n5. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n6. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\n7. \\nHendset M et al. Impact of the CYP2D6 genotype on steady-state serum concentrations of aripiprazole and dehydroaripiprazole. Eur J Clin Pharmacol 2007;63:1147-51.\\n8. \\nOosterhuis M et al. Safety of aripiprazole: high serum levels in a CYP2D6 mutated patient. Am J Psychiatry 2007;164:175.\\n9. \\nSmPC’s Abilify (NL), Abilify Maintena (NL), Abilify (USA) en Aristad (USA).\\nDate 13-09-2021\\nCYP2D6 UM: aripiprazol\\n1543\\nNO action is needed for this gene-drug interaction.\\nThe genetic variation decreases the plasma concentration of the sum of aripiprazole and the active metabolite dehydroaripiprazole to a limited degree. There is no evidence\\nthat this increases the risk of reduced effectiveness.\\nLiterature:\\n1. \\nJallaq SA et al. CYP2D6 phenotype influences aripiprazole tolerability in pediatric patients with mood disorders. J Child Adolesc Psychopharmacol 2021;31:56-62.\\n2. \\nKoller D et al. The effects of aripiprazole and olanzapine on pupillary light reflex and its relationship with pharmacogenetics in a randomized multiple-dose trial. Br J Clin\\nPharmacol 2020;86:2051-62. en persoonlijke communicatie (supplementaire files)\\n3. \\nTveito M et al. Impact of age and CYP2D6 genetics on exposure of aripiprazole and dehydroaripiprazole in patients using long-acting injectable versus oral formulation:\\nrelevance of poor and intermediate metabolizer status. Eur J Clin Pharmacol 2020;76:41-9.\\n4. \\nJukic MM et al. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 2019;6(5):418-26. en\\npersoonlijke communicatie (correcte aantal patiënten per fenotype en per genotypegroep in de kineticasubgroup)\\n5. \\nBelmonte C et al. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 polymorphisms on pharmacokinetics and safety of aripiprazole in healthy volunteers. Basic Clin\\nPharmacol Toxicol 2018;122:596-605.\\n6. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n7. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\nDate 13-09-2021\\nCYP2D6 IM: atenolol\\n2454\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users. Clin Pharmacol Ther 2009;85:45-50.\\n2. \\nLewis RV et al. Timolol and atenolol: relationships between oxidation phenotype, pharmacokinetics and pharmacodynamics. Br J Clin Pharmacol 1985;19:329-33.\\n3. \\nDayer P et al. Interindividual variation of beta-adrenoceptor blocking drugs, plasma concentration and effect: influence of genetic status on behaviour of atenolol,\\nbopindolol and metoprolol. Eur J Clin Pharmacol 1985;28:149-53.\\nDate 12-09-2022\\nCYP2D6 PM: atenolol\\n2453\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users. Clin Pharmacol Ther 2009;85:45-50.\\n2. \\nLewis RV et al. Timolol and atenolol: relationships between oxidation phenotype, pharmacokinetics and pharmacodynamics. Br J Clin Pharmacol 1985;19:329-33.\\n3. \\nDayer P et al. Interindividual variation of beta-adrenoceptor blocking drugs, plasma concentration and effect: influence of genetic status on behaviour of atenolol,\\nbopindolol and metoprolol. Eur J Clin Pharmacol 1985;28:149-53.\\n4. \\nFreestone S et al. Comparison of two long-acting preparations of metoprolol with conventional metoprolol and atenolol in healthy men during chronic dosing. Br J Clin\\nPharmacol 1982;14:713-8.\\nDate 12-09-2022\\nCYP2D6 UM: atenolol\\n2455', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 9}),\n",
       " Document(page_content='This is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nLewis RV et al. Timolol and atenolol: relationships between oxidation phenotype, pharmacokinetics and pharmacodynamics. Br J Clin Pharmacol 1985;19:329-33.\\n2. \\nDayer P et al. Interindividual variation of beta-adrenoceptor blocking drugs, plasma concentration and effect: influence of genetic status on behaviour of atenolol,\\nbopindolol and metoprolol. Eur J Clin Pharmacol 1985;28:149-53.\\nDate 12-09-2022\\nCYP2D6 IM: atomoxetine\\n1599\\nThe dose requirement can be reduced, because the genetic variation results in a higher atomoxetine plasma concentration.\\nin the event of side effects occurring and/or a response later than 9 weeks: reduce the dose and check whether the effect is conserved\\nThe plasma concentration of atomoxetine is a factor of 2-3 times higher for IM than for NM at the same dose.\\nLiterature:\\n1. \\nJung EH et al. Effects of paroxetine on the pharmacokinetics of atomoxetine and its metabolites in different CYP2D6 genotypes. Arch Pharm Res 2020;43:1356-63.\\n2. \\nKim SH et al. Physiologically based pharmacokinetic modelling of atomoxetine with regard to CYP2D6 genotypes. Sci Rep 2018;8:12405.\\n3. \\nBrown JT et al. Single dose, CYP2D6 genotype-stratified pharmacokinetic study of atomoxetine in children with ADHD. Clin Pharmacol Ther 2016;99:642-50.\\n4. \\nByeon JY et al. Effects of the CYP2D6*10 allele on the pharmacokinetics of atomoxetine and its metabolites. Arch Pharm Res 2015;38:2083-91.\\n5. \\nFijal BA et al. CYP2D6 predicted metabolizer status and safety in adult patients with attention-deficit hyperactivity disorder participating in a large placebo-controlled\\natomoxetine maintenance of response clinical trial. J Clin Pharmacol 2015;55:1167-74.\\n6. \\nMatsui A et al. Pharmacokinetics, safety, and tolerability of atomoxetine and effect of CYP2D6*10/*10 genotype in healthy Japanese men. J Clin Pharmacol 2012;52:388-\\n403.\\n7. \\nter Laak MA et al. Recognition of impaired atomoxetine metabolism because of low CYP2D6 activity. Pediatr Neurol 2010;43:159-62.\\n8. \\nCui YM et al. Atomoxetine pharmacokinetics in healthy Chinese subjects and effect of the CYP2D6*10 allele. Br J Clin Pharmacol 2007;64:445-9.\\nDate 31-01-2022\\nCYP2D6 PM: atomoxetine\\n1598\\nThe risk of side effects is increased, because the genetic variation results in a higher atomoxetine plasma concentration.\\nstart with the normal initial dose, bearing in mind that an increase in this dose probably will not be required\\nadvise the patient to report side effects (such as decreased appetite, vomiting, abdominal pain, constipation, insomnia, early waking, drowsiness, irritability, pupil dilation\\nand itching)\\nif the medicine is effective, but side effects occur: reduce the dose and check whether the effect is conserved\\nThe plasma concentration of atomoxetine is a factor of 8-11 times higher for PM than for NM at the same dose.\\nLiterature:\\n1. \\nBrown JT et al. Single dose, CYP2D6 genotype-stratified pharmacokinetic study of atomoxetine in children with ADHD. Clin Pharmacol Ther 2016;99:642-50.\\n2. \\nFijal BA et al. CYP2D6 predicted metabolizer status and safety in adult patients with attention-deficit hyperactivity disorder participating in a large placebo-controlled\\natomoxetine maintenance of response clinical trial. J Clin Pharmacol 2015;55:1167-74.\\n3. \\nLoghin C et al. Effects of atomoxetine on the QT interval in healthy CYP2D6 poor metabolizers. Br J Clin Pharmacol 2013;75:538-49.\\n4. \\nMatsui A et al. Pharmacokinetics, safety, and tolerability of atomoxetine and effect of CYP2D6*10/*10 genotype in healthy Japanese men. J Clin Pharmacol 2012;52:388-\\n403.\\n5. \\nRamoz N et al. A Haplotype of the Norepinephrine Transporter (Net) Gene Slc6a2 is Associated with Clinical Response to Atomoxetine in Attention-Deficit Hyperactivity\\nDisorder (ADHD). Neuropsychopharmacology 2009;34:2135-42.\\n6. \\nTrzepacz PT et al. CYP2D6 metabolizer status and atomoxetine dosing in children and adolescents with ADHD. Eur Neuropsychopharmacol 2008;18:79-86.\\n7. \\nCui YM et al. Atomoxetine pharmacokinetics in healthy Chinese subjects and effect of the CYP2D6*10 Michelson D et al. CYP2D6 and clinical response to atomoxetine in\\nchildren and adolescents with ADHD. J Am Acad Child Adolesc Psychiatry 2007;46:242-51.\\n8. \\nSauer JM et al. Disposition and metabolic fate of atomoxetine hydrochloride: the role of CYP2D6 in human disposition and metabolism. Drug Metab Dispos 2003;31:98-\\n107.\\n9. \\nSmPC’s Strattera (NL en VS).\\n10. \\nData on file, Lilly Research Laboratories, 2006. Atomoxetine - comparison of data of extensive metaboliser and poor metaboliser patients.\\nDate 31-01-2022\\nCYP2D6 UM: atomoxetine\\n1600\\nEfficacy can be reduced due to the gene variation. The gene variation results in an increased conversion of atomoxetine to the active metabolite 4-hydroxyatomoxetine, which\\nhas a much lower plasma concentration.\\nAs a result, the plasma concentration of the active ingredients decreases.\\nbe extra alert to reduced efficacy of the treatment\\nadvise the patient to report an inadequate effect\\nan alternative can be selected as a precaution\\nClonidine is not metabolised by CYP2D6.\\nLiterature:\\n1. \\nBrown JT et al. Single dose, CYP2D6 genotype-stratified pharmacokinetic study of atomoxetine in children with ADHD. Clin Pharmacol Ther 2016;99:642-50.\\nDate 31-01-2022\\nSLCO1B1 521CC: atorvastatine\\n4058\\nThe risk of myopathy may be increased. The gene variation may lead to reduced atorvastatin transport to the liver, which may increase the atorvastatin plasma concentration.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 10}),\n",
       " Document(page_content='Patient has ADDITIONAL SIGNIFICANT RISK FACTORS for statin-induced myopathy:\\n1. \\nChoose an alternative\\nDo not select simvastatin, as this is also affected by the SLCO1B1 gene variation. Rosuvastatin and pravastatin are influenced to a similar extent by SLCO1B1\\npolymorphisms, but are not influenced by CYP3A4 inhibitors such as amiodarone, verapamil and diltiazem. Fluvastatin is not influenced significantly by SLCO1B1\\ngene variation or CYP3A4 inhibitors.\\n2. \\nIf an alternative is not an option: advise the patient to contact their doctor in the event of muscle symptoms.\\nPatient has NO additional significant risk factors for statin-induced myopathy:\\n1. \\nAdvise the patient to contact their doctor in the event of muscle symptoms.\\nLiterature:\\n1. \\nTurner RM et al. A genome-wide association study of circulating levels of atorvastatin and its major metabolites. Clin Pharmacol Ther 2020 Mar 3 [Epub ahead of print].\\n2. \\nLiu JE et al. SLCO1B1 521T > C polymorphism associated with rosuvastatin-induced myotoxicity in Chinese coronary artery disease patients: a nested case-control study.\\nEur J Clin Pharmacol 2017;73:1409-16.\\n3. \\nJiang J et al. Association between SLCO1B1 -521T>C and -388A>G polymorphisms and risk of statin-induced adverse drug reactions: a meta-analysis. Springerplus\\n2016;5:1368.\\n4. \\nFukunaga K et al. ABCB1 polymorphism is associated with atorvastatin-induced liver injury in Japanese population. BMC Genet 2016;17:79.\\n5. \\nHou Q et al. Association between SLCO1B1 gene T521C polymorphism and statin-related myopathy risk: a meta-analysis of case-control studies. Medicine (Baltimore)\\n2015;94:e1268.\\n6. \\nGiannakopoulou E et al. No impact of SLCO1B1 521 T>C, 388A>G and 411G>A polymorphisms on response to statin therapy in the Greek population. Mol Biol Rep\\n2014;41:4631-8.\\n7. \\nde Keyser CE et al. The SLCO1B1 c.521T>C polymorphism is associated with dose decrease or switching during statin therapy in the Rotterdam Study. Pharmacogenet\\nGenomics 2014;24:43-51.\\n8. \\nFu Q et al. Lack of association between SLCO1B1 polymorphism and the lipid-lowering effects of atorvastatin and simvastatin in Chinese individuals. Eur J Clin Pharmacol\\n2013;69:1269-74.\\n9. \\nCarr DF et al. SLCO1B1 genetic variant associated with statin-induced myopathy: a proof-of-concept study using the Clinical Practice Research Datalink. Clin Pharmacol\\nTher 2013;94:695-701.\\n10. \\nFrancesca Notarangelo M et al. Genetic predisposition to atorvastatin-induced myopathy: a case report. J Clin Pharm Ther 2012;37:604-6.\\n11. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and atorvastatin. Pharmacogenomics J 2012;12:233-\\n7.\\n12. \\nSantos PC et al. SLCO1B1 haplotypes are not associated with atorvastatin-induced myalgia in Brazilian patients with familial hypercholesterolemia. Eur J Clin Pharmacol\\n2012;68:273-9.\\n13. \\nRodrigues AC et al. Pharmacogenetics of OATP transporters reveals that SLCO1B1 c.388A>G variant is determinant of increased atorvastatin response. Int J Mol Sci\\n2011;12:5815-27.\\n14. \\nPuccetti L et al. Genetic involvement in statins induced myopathy. Preliminary data from an observational case-control study. Atherosclerosis 2010;211:28-9.\\n15. \\nLee YJ et al. Effects of SLCO1B1 and ABCB1 genotypes on the pharmacokinetics of atorvastatin and 2-hydroxyatorvastatin in healthy Korean subjects. Int J Clin\\nPharmacol Ther 2010;48:36-45.\\n16. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n17. \\nMega JL et al. Identification of genetic variants associated with response to statin therapy. Arterioscler Thromb Vasc Biol 2009;29:1310-5.\\n18. \\nHe YJ et al. Rifampicin alters atorvastatin plasma concentration on the basis of SLCO1B1 521T>C polymorphism. Clin Chim Acta 2009;405:49-52.\\n19. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with increased cholesterol synthesis rate.\\nPharmacogenet Genomics 2008;18:921-6.\\n20. \\nPasanen MK et al. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Ther 2007;82:726-3\\n21. \\nHermann M et al. Exposure of atorvastatin is unchanged but lactone and acid metabolites are increased several-fold in patients with atorvastatin-induced myopathy. Clin\\nPharmacol Ther 2006;79:532-9.\\n22. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J 2005;5:352-8.\\n23. \\nSmPC Lipitor.\\nDate 18-05-2020\\nSLCO1B1 521TC: atorvastatine\\n4057\\nThe risk of myopathy can be elevated. The gene variation may lead to reduced atorvastatin transport to the liver, which may increase atorvastatin plasma concentrations.\\nPatient has ADDITIONAL SIGNIFICANT RISK FACTORS for statin-induced myopathy:\\n1. \\nChoose an alternative\\nRosuvastatin and pravastatin are influenced to a similar extent by the SLCO1B1 gene variation, but are not influenced by CYP3A4 inhibitors such as amiodarone,\\nverapamil and diltiazem.\\nFluvastatin is not influenced significantly by the SLCO1B1 gene variation or CYP3A4 inhibitors.\\n2. \\nIf an alternative is not an option: advise the patient to contact their doctor in the event of muscle symptoms.\\nPatient has NO additional significant risk factors for statin-induced myopathy:\\n1. \\nAdvise the patient to contact their doctor in the event of muscle symptoms.\\nLiterature:\\n1. \\nTurner RM et al. A genome-wide association study of circulating levels of atorvastatin and its major metabolites. Clin Pharmacol Ther 2020 Mar 3 [Epub ahead of print].\\n2. \\nLiu JE et al. SLCO1B1 521T > C polymorphism associated with rosuvastatin-induced myotoxicity in Chinese coronary artery disease patients: a nested case-control study.\\nEur J Clin Pharmacol 2017;73:1409-16.\\n3. \\nJiang J et al. Association between SLCO1B1 -521T>C and -388A>G polymorphisms and risk of statin-induced adverse drug reactions: a meta-analysis. Springerplus\\n2016;5:1368.\\n4. \\nFukunaga K et al. ABCB1 polymorphism is associated with atorvastatin-induced liver injury in Japanese population. BMC Genet 2016;17:79.\\n5. \\nHou Q et al. Association between SLCO1B1 gene T521C polymorphism and statin-related myopathy risk: a meta-analysis of case-control studies. Medicine (Baltimore)\\n2015;94:e1268.\\n6. \\nGiannakopoulou E et al. No impact of SLCO1B1 521 T>C, 388A>G and 411G>A polymorphisms on response to statin therapy in the Greek population. Mol Biol Rep\\n2014;41:4631-8.\\n7. \\nde Keyser CE et al. The SLCO1B1 c.521T>C polymorphism is associated with dose decrease or switching during statin therapy in the Rotterdam Study. Pharmacogenet\\nGenomics 2014;24:43-51.\\n8. \\nFu Q et al. Lack of association between SLCO1B1 polymorphism and the lipid-lowering effects of atorvastatin and simvastatin in Chinese individuals. Eur J Clin Pharmacol\\n2013;69:1269-74.\\n9. \\nCarr DF et al. SLCO1B1 genetic variant associated with statin-induced myopathy: a proof-of-concept study using the Clinical Practice Research Datalink. Clin Pharmacol\\nTher 2013;94:695-701.\\n10. \\nFrancesca Notarangelo M et al. Genetic predisposition to atorvastatin-induced myopathy: a case report. J Clin Pharm Ther 2012;37:604-6.\\n11. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and atorvastatin. Pharmacogenomics J 2012;12:233-\\n7.\\n12. \\nSantos PC et al. SLCO1B1 haplotypes are not associated with atorvastatin-induced myalgia in Brazilian patients with familial hypercholesterolemia. Eur J Clin Pharmacol\\n2012;68:273-9.\\n13. \\nRodrigues AC et al. Pharmacogenetics of OATP transporters reveals that SLCO1B1 c.388A>G variant is determinant of increased atorvastatin response. Int J Mol Sci\\n2011;12:5815-27.\\n14. \\nPuccetti L et al. Genetic involvement in statins induced myopathy. Preliminary data from an observational case-control study. Atherosclerosis 2010;211:28-9.\\n15. \\nLee YJ et al. Effects of SLCO1B1 and ABCB1 genotypes on the pharmacokinetics of atorvastatin and 2-hydroxyatorvastatin in healthy Korean subjects. Int J Clin\\nPharmacol Ther 2010;48:36-45.\\n16. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n17. \\nMega JL et al. Identification of genetic variants associated with response to statin therapy. Arterioscler Thromb Vasc Biol 2009;29:1310-5.\\n18. \\nHe YJ et al. Rifampicin alters atorvastatin plasma concentration on the basis of SLCO1B1 521T>C polymorphism. Clin Chim Acta 2009;405:49-52.\\n19. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with increased cholesterol synthesis rate.\\nPharmacogenet Genomics 2008;18:921-6.\\n20. \\nPasanen MK et al. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Ther 2007;82:726-33.\\n21. \\nHermann M et al. Exposure of atorvastatin is unchanged but lactone and acid metabolites are increased several-fold in patients with atorvastatin-induced myopathy. Clin\\nPharmacol Ther 2006;79:532-9.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 11}),\n",
       " Document(page_content=\"22. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J 2005;5:352-8.\\nDate 18-05-2020\\nNUDT15 IM: azathioprine/mercaptopurine\\n7035\\nGrade ≥ 2 leukopaenia occurs in 42% of these patients with standard immunosuppression therapy. The gene variation increases the concentration of the fully activated\\nmetabolite of azathioprine and mercaptopurine.\\nIMMUNOSUPPRESSION:\\nstart with 50% of the standard dose\\nAdjustment of the initial dose should be performed based on toxicity (monitoring of the blood counts) and efficacy.\\nNote: The percentage of 50% is based on the analogy with TPMT, for which the gene variants have a comparable effect on toxicity to those of NUDT15. A percentage of < 70%\\nwas calculated for NUDT15, but there were insufficient data available to calculate the exact percentage.\\nNote: Dose adjustment based on the total of 6-TGN metabolites is not possible for these patients, as they develop toxicity within the therapeutic range that applies for patients\\nwithout gene variants.\\nLEUKAEMIA:\\nstart at 50% of the standard mercaptopurine dose, or start with the standard dose and reduce to 50% if side effects necessitate a dose reduction\\nIt is not known whether dose reduction in advance results in the same efficacy as dose reduction based on toxicity.\\nAdjustment of the initial dose should be performed based on toxicity (monitoring of the blood counts) and efficacy.\\nNote: The percentage of 50% is based on the analogy with TPMT, for which the gene variants have a comparable effect on toxicity to those of NUDT15. A percentage of < 70%\\nwas calculated for NUDT15, but there were insufficient data available to calculate the exact percentage.\\nNote: Dose adjustment based on the total of 6-TGN metabolites is not possible for these patients, as they develop toxicity within the therapeutic range that applies for patients\\nwithout gene variants.\\nNote: more stringent dose reductions are necessary if the patient is also TPMT IM or TPMT PM.\\nLiterature:\\n1. \\nZhu Y et al. Combination of common and novel rare NUDT15 variants improves predictive sensitivity of thiopurine-induced leukopenia in children with acute\\nlymphoblastic leukemia. Haematologica 2018 Mar 8 [Epub ahead of print].\\n2. \\nZhang AL et al. Association of NUDT15 c.415C>T allele and thiopurine-induced leukocytopenia in Asians: a systematic review and meta-analysis. Ir J Med Sci\\n2018;187:145-153.\\n3. \\nYi ES et al. NUDT15 variants cause hematopoietic toxicity with low 6-TGN levels in children with acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 13 [Epub\\nahead of print].\\n4. \\nKim H et al. APEX1 polymorphism and mercaptopurine-related early onset neutropenia in pediatric acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 4 [Epub\\nahead of print].\\n5. \\nChao K et al. Combined detection of NUDT15 variants could highly predict thiopurine-induced leukopenia in Chinese patients with inflammatory bowel disease: a\\nmulticenter analysis. Inflamm Bowel Dis 2017;23:1592-9.\\n6. \\nYin D et al. Impact of NUDT15 polymorphisms on thiopurines-induced myelotoxicity and thiopurines tolerance dose. Oncotarget 2017;8:13575-85.\\n7. \\nLiang DC et al. NUDT15 gene polymorphism related to mercaptopurine intolerance in Taiwan Chinese children with acute lymphoblastic leukemia. Pharmacogenomics J\\n2016;16:536-9.\\n8. \\nZhu X et al. NUDT15 polymorphisms are better than thiopurine S-methyltransferase as predictor of risk for thiopurine-induced leukopenia in Chinese patients with\\nCrohn's disease. Aliment Pharmacol Ther 2016;44:967-75.\\n9. \\nMoriyama T et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet 2016;48:367-73.\\n10. \\nYang JJ et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol 2015;33:1235-\\n42.\\n11. \\nYang SK et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet 2014;46:1017-20.\\n12. \\nSmPC’s Puri-Nethol en Imuran.\\nDate 17-09-2019\\nNUDT15 PM: azathioprine/mercaptopurine\\n7036\\nGrade ≥ 2 leukopaenia occurs in 96% of these patients with standard therapy. The gene variation increases the concentration of the fully activated metabolite of azathioprine\\nand mercaptopurine.\\navoid azathioprine and mercaptopurine\\nif it is not possible to avoid azathioprine and mercaptopurine: use 10% of the standard dose and advise patients to seek medical attention when symptoms of\\nmyelosuppression (such as severe sore throat in combination with fever, regular nosebleeds and tendency to bruising) occur\\nAny adjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and efficacy.\\nNote: The percentage of 10% is based on the analogy with TPMT, for which the gene variants have a comparable effect on toxicity to those of NUDT15. A percentage of <\\n20% was calculated for NUDT15 PM, but there were insufficient data available to calculate the exact percentage.\\nNote: Dose adjustment based on the total of 6-TGN metabolites is not possible for these patients, as they develop toxicity within the therapeutic range that applies for\\npatients without gene variants.\\nLiterature:\\n1. \\nZhu Y et al. Combination of common and novel rare NUDT15 variants improves predictive sensitivity of thiopurine-induced leukopenia in children with acute\\nlymphoblastic leukemia. Haematologica 2018 Mar 8 [Epub ahead of print].\\n2. \\nZhang AL et al. Association of NUDT15 c.415C>T allele and thiopurine-induced leukocytopenia in Asians: a systematic review and meta-analysis. Ir J Med Sci\\n2018;187:145-153.\\n3. \\nYi ES et al. NUDT15 variants cause hematopoietic toxicity with low 6-TGN levels in children with acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 13 [Epub\\nahead of print].\\n4. \\nKim H et al. APEX1 polymorphism and mercaptopurine-related early onset neutropenia in pediatric acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 4 [Epub\\nahead of print].\\n5. \\nChao K et al. Combined detection of NUDT15 variants could highly predict thiopurine-induced leukopenia in Chinese patients with inflammatory bowel disease: a\\nmulticenter analysis. Inflamm Bowel Dis 2017;23:1592-9.\\n6. \\nYin D et al. Impact of NUDT15 polymorphisms on thiopurines-induced myelotoxicity and thiopurines tolerance dose. Oncotarget 2017;8:13575-85.\\n7. \\nLiang DC et al. NUDT15 gene polymorphism related to mercaptopurine intolerance in Taiwan Chinese children with acute lymphoblastic leukemia. Pharmacogenomics J\\n2016;16:536-9.\\n8. \\nZhu X et al. NUDT15 polymorphisms are better than thiopurine S-methyltransferase as predictor of risk for thiopurine-induced leukopenia in Chinese patients with\\nCrohn's disease. Aliment Pharmacol Ther 2016;44:967-75.\\n9. \\nMoriyama T et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet 2016;48:367-73.\\n10. \\nYang JJ et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol 2015;33:1235-\\n42.\\n11. \\nYang SK et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet 2014;46:1017-20.\\n12. \\nSmPC’s Puri-Nethol en Imuran.\\nDate 17-09-2019\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 12}),\n",
       " Document(page_content=\"TPMT IM: azathioprine/mercaptopurine\\n1905\\nGrade ≥ 2 leukopaenia occurs in 23% of these patients with normal therapy for immunosuppression. The genetic variation increases the quantity of the active metabolites of\\nazathioprine and mercaptopurine.\\nRecommendation:\\nIMMUNOSUPPRESSION\\n-Start with 50% of the standard dose\\nAdjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and effectiveness.\\nDose adjustment is not required for doses lower than 1.5 mg/kg per day for azathioprine or 0.75 mg/kg per day for mercaptopurine.\\nLEUKAEMIA:\\nstart with 50% of the standard mercaptopurine dose, or start with the standard dose and reduce to 50% if side effects necessitate a dose reduction\\nIt is not known whether dose reduction in advance results in the same efficacy as dose reduction based on toxicity.\\nThe initial dose should be adjusted based on toxicity (monitoring of the blood counts) and efficacy.\\nNote: more stringent dose reductions are necessary if the patient is also NUDT15 IM or NUDT15 PM.\\nLiterature:\\n1. \\nFan X et al. NUDT15 polymorphism confer increased susceptibility to thiopurine-induced leukopenia in patients with autoimmune hepatitis and related cirrhosis. Front\\nPharmacol 2019;10:346.\\n2. \\nChoi R et al. Pathway genes and metabolites in thiopurine therapy in Korean children with acute lymphoblastic leukaemia. Br J Clin Pharmacol 2019 Mar 30 [Epub ahead\\nof print].\\n3. \\nEriksen PL et al. Enrichment of genetic variants in the glucocorticoid receptor signalling pathway in autoimmune hepatitis with failure of standard treatment. Basic Clin\\nPharmacol Toxicol 2017;121:189-94.\\n4. \\nLiu C et al. A genome-wide approach validates that thiopurine methyltransferase activity is a monogenic pharmacogenomic trait. Clin Pharmacol Ther 2017;101:373-381.\\n5. \\nCoenen MJ et al. Identification of patients with variants in TPMT and dose reduction reduces hematologic events during thiopurine treatment of inflammatory bowel\\ndisease. Gastroenterology 2015;149:907-17.\\n6. \\nLennard L et al. Thiopurine methyltransferase and treatment outcome in the UK acute lymphoblastic leukaemia trial ALL2003. Br J Haematol 2015;170:550-8.\\n7. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of thiopurine methyltransferase pharmacogenetics.\\nBr J Haematol 2015;169:228-40.\\n8. \\nKim MJ et al. Monitoring thiopurine metabolites in korean pediatric patients with inflammatory bowel disease. Yonsei Med J 2014;55:1289-96.\\n9. \\nLevinsen M et al. Pharmacogenetically based dosing of thiopurines in childhood acute lymphoblastic leukemia: influence on cure rates and risk of second cancer. Pediatr\\nBlood Cancer 2014;61:797-802.\\n10. \\nBooth RA et al. Assessment of thiopurine S-methyltransferase activity in patients prescribed thiopurines: a systematic review. Ann Intern Med 2011;154:814-23, W-295-8.\\n11. \\nNewman WG et al. A pragmatic randomized controlled trial of thiopurine methyltransferase genotyping prior to azathioprine treatment: the TARGET study.\\nPharmacogenomics 2011;12:815-26.\\n12. \\nDong XW et al. Thiopurine S-methyltransferase polymorphisms and thiopurine toxicity in treatment of inflammatory bowel disease. World J Gastroenterol 2010;16:3187-\\n95.\\n13. \\nHindorf U et al. Characterisation and utility of thiopurine methyltransferase and thiopurine metabolite measurements in autoimmune hepatitis. J Hepatol 2010;52:106-11.\\n14. \\nSheffield LJ et al. Thiopurine methyltransferase and thiopurine metabolite testing in patients with inflammatory bowel disease who are taking thiopurine drugs.\\nPharmacogenomics 2009;10:1091-9.\\n15. \\nAnsari A et al. Influence of xanthine oxidase on thiopurine metabolism in Crohn's disease. Aliment Pharmacol Ther 2008;28:749-57.\\n16. \\nGardiner SJ et al. Thiopurine dose in intermediate and normal metabolizers of thiopurine methyltransferase may differ three-fold. Clin Gastroenterol Hepatol 2008;6:654-\\n60.\\n17. \\nMoloney FJ et al. The frequency and significance of thiopurine S-methyltransferase gene polymorphisms in azathioprine-treated renal transplant recipients. Br J Dermatol\\n2006;154:1199-200.\\n18. \\nZelinkova Z et al. Inosine triphosphate pyrophosphatase and thiopurine s-methyltransferase genotypes relationship to azathioprine-induced myelosuppression. Clin\\nGastroenterol Hepatol 2006;4:44-9.\\n19. \\nJun JB et al. Thiopurine S-methyltransferase polymorphisms and the relationship between the mutant alleles and the adverse effects in systemic lupus erythematosus\\npatients taking azathioprine. Clin Exp Rheumatol 2005;23:873-6.\\n20. \\nStocco G et al. TPMT genotype and the use of thiopurines in paediatric inflammatory bowel disease. Dig Liver Dis 2005;37:940-5.\\n21. \\nKurzawski M et al. The impact of thiopurine s-methyltransferase polymorphism on azathioprine-induced myelotoxicity in renal transplant recipients. Ther Drug Monit\\n2005;27:435-41.\\n22. \\nFabre MA et al. The impact of thiopurine S-methyltransferase polymorphisms on azathioprine dose 1 year after renal transplantation. Transpl Int 2004;17:531-9.\\n23. \\nGearry RB et al. Thiopurine S-methyltransferase (TPMT) genotype does not predict adverse drug reactions to thiopurine drugs in patients with inflammatory bowel\\ndisease. Aliment Pharmacol Ther 2003;18:395-400.\\n24. \\nGilissen LP et al. Some cases demonstrating the clinical usefulness of therapeutic drug monitoring in thiopurine-treated inflammatory bowel disease patients. Eur J\\nGastroenterol Hepatol 2004;16:705-10.\\n25. \\nAnsari A et al. Thiopurine methyltransferase activity and the use of azathioprine in inflammatory bowel disease. Aliment Pharmacol Ther 2002;16:1743-50.\\n26. \\nLangley PG et al. Thiopurine methyltransferase phenotype and genotype in relation to azathioprine therapy in autoimmune hepatitis. J Hepatol 2002;37:441-7.\\n27. \\nRegueiro M et al. Determination of thiopurine methyltransferase genotype or phenotype optimizes initial dosing of azathioprine for the treatment of Crohn’s disease. J\\nClin Gastroenterol 2002;35:240-4.\\n28. \\nPandya B et al. Azathioprine toxicity and thiopurine methyltransferase genotype in renal transplant patients. Transplant Proc 2002;34:1642-5.\\n29. \\nCampbell S et al. Relevance of thiopurine methyltransferase activity in inflammatory bowel disease patients maintained on low-dose azathioprine. Aliment Pharmacol\\nTher 2002;16:389-98.\\n30. \\nColombel JF et al. Genotypic analysis of thiopurine S-methyltransferase in patients with Crohn’s disease and severe myelosuppression during azathioprine therapy.\\nGastroenterology 2000;118:1025-30.\\n31. \\nBlack AJ et al. Thiopurine methyltransferase genotype predicts therapy-limiting severe toxicity from azathioprine. Ann Intern Med 1998;129:716-8.\\n32. \\nHiggs JE et al. Are patients with intermediate TPMT activity at increased risk of myelosuppression when taking thiopurine medications? Pharmacogenomics 2010;11:177-\\n88.\\n33. \\nStanulla M et al. Thiopurine methyltransferase (TPMT) genotype and early treatment response to mercaptopurine in childhood acute lymphoblastic leukemia. JAMA\\n2005;293:1485-9.\\n34. \\nEvans WE et al. Preponderance of thiopurine S-methyltransferase deficiency and heterozygosity among patients intolerant to mercaptopurine or azathioprine. J Clin\\nOncol 2001;19:2293-301.\\n35. \\nRelling MV et al. Mercaptopurine therapy intolerance and heterozygosity at the thiopurine S-methyltransferase gene locus. J Natl Cancer Inst 1999;91:2001-8.\\n36. \\nMcLeod HL et al. Analysis of thiopurine methyltransferase variant alleles in childhood acute lymphoblastic leukaemia. Br J Haematol 1999;105:696-700.\\n37. \\nSmPC’s Imuran (VS) en Purixan (VS).\\nDate 17-09-2019\\nTPMT PM: azathioprine/mercaptopurine\\n1906\\nGrade ≥ 2 leukopaenia and intolerance occurred in 98% of these patients with standard therapy. The gene variation increases the quantities of the active metabolites of\\nazathioprine and mercaptopurine.\\nRecommendation:\\nChoose an alternative or use 10% of the standard dose.\\nAny adjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and effectiveness.\\nIf the dose is decreased: advise patients to seek medical attention when symptoms of myelosuppression (such as severe sore throat in combination with fever, regular\\nnosebleeds and tendency to bruising) occur\\nLiterature:\\n1. \\nFan X et al. NUDT15 polymorphism confer increased susceptibility to thiopurine-induced leukopenia in patients with autoimmune hepatitis and related cirrhosis. Front\\nPharmacol 2019;10:346.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 13}),\n",
       " Document(page_content=\"2. \\nLiu C et al. A genome-wide approach validates that thiopurine methyltransferase activity is a monogenic pharmacogenomic trait. Clin Pharmacol Ther 2017;101:373-381.\\n3. \\nvan Moorsel SA et al. Azathioprine therapy in a pediatric TPMT-deficient patient - still an option. Ther Drug Monit 2017;39:1-4.\\n4. \\nCoenen MJ et al. Identification of patients with variants in TPMT and dose reduction reduces hematologic events during thiopurine treatment of inflammatory bowel\\ndisease. Gastroenterology 2015;149:907-17.\\n5. \\nLennard L et al. Thiopurine methyltransferase and treatment outcome in the UK acute lymphoblastic leukaemia trial ALL2003. Br J Haematol 2015;170:550-8.\\n6. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of thiopurine methyltransferase pharmacogenetics.\\nBr J Haematol 2015;169:228-40.\\n7. \\nBelen BF et al. Severe myelotoxicity associated with thiopurine S-methyltransferase\\n3A/ \\n3C polymorphisms in a patient with pediatric leukemia and the effect of steroid\\ntherapy. Turk J Haematol 2014;31:399-402.\\n8. \\nKim MJ et al. Monitoring thiopurine metabolites in korean pediatric patients with inflammatory bowel disease. Yonsei Med J 2014;55:1289-96.\\n9. \\nLevinsen M et al. Pharmacogenetically based dosing of thiopurines in childhood acute lymphoblastic leukemia: influence on cure rates and risk of second cancer. Pediatr\\nBlood Cancer 2014;61:797-802.\\n10. \\nDemlova R et al. Augmenting clinical interpretability of thiopurine methyltransferase laboratory evaluation. Oncology 2014;86:152-8.\\n11. \\nLee MN et al. Successful azathioprine treatment with metabolite monitoring in a pediatric inflammatory bowel disease patient homozygous for TPMT*3C. Yonsei Med J\\n2013;54:1545-9.\\n12. \\nKim H et al. Pharmacogenetic analysis of pediatric patients with acute lymphoblastic leukemia: a possible association between survival rate and ITPA polymorphism.\\nPLoS One 2012;7:e45558.\\n13. \\nBooth RA et al. Assessment of thiopurine S-methyltransferase activity in patients prescribed thiopurines: a systematic review. Ann Intern Med 2011;154:814-23, W-295-8.\\n14. \\nNewman WG et al. A pragmatic randomized controlled trial of thiopurine methyltransferase genotyping prior to azathioprine treatment: the TARGET study.\\nPharmacogenomics 2011;12:815-26.\\n15. \\nDong XW et al. Thiopurine S-methyltransferase polymorphisms and thiopurine toxicity in treatment of inflammatory bowel disease. World J Gastroenterol 2010;16:3187-\\n95.\\n16. \\nAnsari A et al. Influence of xanthine oxidase on thiopurine metabolism in Crohn's disease. Aliment Pharmacol Ther 2008;28:749-57.\\n17. \\nZelinkova Z et al. Inosine triphosphate pyrophosphatase and thiopurine s-methyltransferase genotypes relationship to azathioprine-induced myelosuppression. Clin\\nGastroenterol Hepatol 2006;4:44-9.\\n18. \\nKurzawski M et al. The impact of thiopurine s-methyltransferase polymorphism on azathioprine-induced myelotoxicity in renal transplant recipients. Ther Drug Monit\\n2005;27:435-41.\\n19. \\nGardiner SJ et al. Two cases of thiopurine methyltransferase (TPMT) deficiency--a lucky save and a near miss with azathioprine. Br J Clin Pharmacol 2006;62:473-6.\\n20. \\nKurzawski et al. Severe azathioprine-induced myelotoxicity in a kidney transplant patient with thiopurine S-methyltransferase-deficient genotype (TPMT\\n3A/\\n3C). Transpl\\nInt 2005;18:623-5.\\n21. \\nGearry RB et al. Thiopurine S-methyltransferase (TPMT) genotype does not predict adverse drug reactions to thiopurine drugs in patients with inflammatory bowel\\ndisease. Aliment Pharmacol Ther 2003;18:395-400.\\n22. \\nGilissen LP et al. Some cases demonstrating the clinical usefulness of therapeutic drug monitoring in thiopurine-treated inflammatory bowel disease patients. Eur J\\nGastroenterol Hepatol 2004;16:705-10.\\n23. \\nKaskas BA et al. Safe treatment of thiopurine S-methyltransferase deficient Crohn’s disease patients with azathioprine. Gut 2003;52:140-2.\\n24. \\nColombel JF et al. Genotypic analysis of thiopurine S-methyltransferase in patients with Crohn’s disease and severe myelosuppression during azathioprine therapy.\\nGastroenterology 2000;118:1025-30.\\n25. \\nHiggs JE et al. Are patients with intermediate TPMT activity at increased risk of myelosuppression when taking thiopurine medications? Pharmacogenomics 2010;11:177-\\n88.\\n26. \\nStanulla M et al. Thiopurine methyltransferase (TPMT) genotype and early treatment response to mercaptopurine in childhood acute lymphoblastic leukemia. JAMA\\n2005;293:1485-9.\\n27. \\nSchaeffeler et al. A novel TPMT missense mutation associated with TPMT deficiency in a 5-year-old boy with ALL. Leukemia 2003;17:1422-4.\\n28. \\nEvans WE et al. Preponderance of thiopurine S-methyltransferase deficiency and heterozygosity among patients intolerant to mercaptopurine or azathioprine. J Clin\\nOncol 2001;19:2293-301.\\n29. \\nRelling MV et al. Mercaptopurine therapy intolerance and heterozygosity at the thiopurine S-methyltransferase gene locus. J Natl Cancer Inst 1999;91:2001-8.\\n30. \\nMcLeod HL et al. Analysis of thiopurine methyltransferase variant alleles in childhood acute lymphoblastic leukaemia. Br J Haematol 1999;105:696-700.\\n31. \\nAndersen JB et al. Pharmacokinetics, dose adjustments, and 6-mercaptopurine/methotrexate drug interactions in two patients with thiopurine methyltransferase\\ndeficiency. Acta Paediatr 1998;87:108-11.\\n32. \\nSmPC’s Imuran (NL en VS), Puri-Nethol (NL) en Purixan (VS).\\nDate 17-09-2019\\nCYP2D6 IM: bisoprolol\\n2457\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nFontana V et al. Chromosomal region 11p14.1 is associated with pharmacokinetics and pharmacodynamics of bisoprolol. Pharmgenomics Pers Med 2022;15:249-60.\\n2. \\nChan SW et al. Influence of CYP2D6 and CYP3A5 polymorphisms on the pharmacokinetics and pharmacodynamics of bisoprolol in hypertensive Chinese patients. Front\\nMed (Lausanne) 2021;8:683498.\\n3. \\nNozawa T et al. Influence of CYP2D6 genotype on metoprolol plasma concentration and beta-adrenergic inhibition during long-term treatment: a comparison with\\nbisoprolol. J Cardiovasc Pharmacol 2005;46:713-20.\\n4. \\nTaguchi M et al. Pharmacokinetic variability of routinely administered bisoprolol in middle-aged and elderly Japanese patients. Biol Pharm Bull 2005;28:876-81.\\nDate 12-09-2022\\nCYP2D6 PM: bisoprolol\\n2456\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nFontana V et al. Chromosomal region 11p14.1 is associated with pharmacokinetics and pharmacodynamics of bisoprolol. Pharmgenomics Pers Med 2022;15:249-60.\\n2. \\nChan SW et al. Influence of CYP2D6 and CYP3A5 polymorphisms on the pharmacokinetics and pharmacodynamics of bisoprolol in hypertensive Chinese patients. Front\\nMed (Lausanne) 2021;8:683498.\\n3. \\nDeroubaix X et al. Comparative bioavailability of a metoprolol controlled release formulation and a bisoprolol normal release tablet after single oral dose administration in\\nhealthy volunteers.\\nDate 12-09-2022\\nCYP2D6 UM: bisoprolol\\n2458\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nFontana V et al. Chromosomal region 11p14.1 is associated with pharmacokinetics and pharmacodynamics of bisoprolol. Pharmgenomics Pers Med 2022;15:249-60.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 14}),\n",
       " Document(page_content='Date 12-09-2022\\nCYP2D6 IM: brexpiprazol\\n7046\\nNO action is required for this gene-drug interaction.\\nThere are indications supporting an increase in the exposure to brexpiprazole, but no indications supporting an increase in side effects in patients with this gene variation.\\nLiterature:\\n1. \\nIshigooka J et al. Pharmacokinetics and safety of brexpiprazole following multiple-dose administration to Japanese patients with schizophrenia. J Clin Pharmacol\\n2018;58:74-80.\\n2. \\nEPAR Rxulti.\\nDate 13-09-2021\\nCYP2D6 PM: brexpiprazol\\n7047\\nThe risk of side effects is theoretically increased, because the gene variation reduces the metabolism of brexpiprazole.\\nuse half of the standard dose\\nLiterature:\\n1. \\nSmPC en EPAR Rxulti (NL) en SmPC Rexulti (VS).\\nDate 13-09-2021\\nCYP2D6 UM: brexpiprazol\\n7048\\nNO action is required for this gene-drug interaction.\\nThe gene variation results in a reduction of the exposure to brexpiprazole, but there are no indications supporting a decrease in efficacy.\\nLiterature:\\n1. \\nEPAR Rxulti.\\nDate 13-09-2021\\nHLA-A*3101: carbamazepine\\n6238\\nPatients with this genetic variation have an increased risk of experiencing the life-threatening cutaneous adverse events DRESS and Stevens-Johnson syndrome/toxic\\nepidermal necrolysis (SJS/TEN). The risk of carbamazepine-induced DRESS in these patients is 0.89%.\\nRecommendation:\\n1. \\ncarefully weigh the risk of DRESS and SJS/TEN against the benefits\\n2. \\nif an alternative is an option, choose an alternative\\nLiterature:\\n1. \\nNicoletti P et al. Shared genetic risk factors across carbamazepine-induced hypersensitivity reactions. Clin Pharmacol Ther 2019;106:1028-36.\\n2. \\nMushiroda T et al. Association of HLA-A*31:01 screening with the incidence of carbamazepine-induced cutaneous adverse reactions in a Japanese population. JAMA\\nNeurol 2018;75:842-9.\\n3. \\nGenin E et al. HLA-A*31:01 and different types of carbamazepine-induced severe cutaneous adverse reactions: an international study and meta-analysis.\\nPharmacogenomics J 2014;14:281-8.\\n4. \\nBloch KM et al. Pharmacogenetics of antiepileptic drug-induced hypersensitivity. Pharmacogenomics 2014;15:857-68.\\n5. \\nGrover S et al. HLA alleles and hypersensitivity to carbamazepine: an updated systematic review with meta-analysis. Pharmacogenet Genomics 2014;24:94-112.\\n6. \\nYip VL et al. HLA genotype and carbamazepine-induced cutaneous adverse drug reactions: a systematic review. Clin Pharmacol Ther 2012;92:757-65.\\n7. \\nSmPC’s Tegretol (NL en VS).\\nDate 23-05-2022\\nHLA-B*1502: carbamazepine\\n6237\\nPatients with this genetic variation have a severely increased risk of experiencing the life-threatening cutaneous adverse event Stevens-Johnson syndrome/toxic epidermal\\nnecrolysis (SJS/TEN). The risk of carbamazepine-induced SJS/TEN in these patients is 1.8-3.4%.\\nRecommendation:\\n1. \\nchoose an alternative if possible\\nPhenytoin, lamotrigine and oxcarbazepine also pose an increased risk of SJS/TEN in these patients, but the final risk is 10-fold lower for these medicines than for\\ncarbamazepine.\\nFurthermore, in the case of oxcarbazepine, the most severe forms (SJS/TEN overlap and TEN) have not been observed.\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 15}),\n",
       " Document(page_content='1. \\nSukasem C et al. Spectrum of cutaneous adverse reactions to aromatic antiepileptic drugs and human leukocyte antigen genotypes in Thai patients and meta-analysis.\\nPharmacogenomics J 2021;21:682-90.\\n2. \\nChouchi M et al. The HLA-B*15:02 polymorphism and Tegretol®-induced serious cutaneous reactions in epilepsy: an updated systematic review and meta-analysis. Rev\\nNeurol (Paris) 2018;174:278-91.\\n3. \\nBloch KM et al. Pharmacogenetics of antiepileptic drug-induced hypersensitivity. Pharmacogenomics 2014;15:857-68.\\n4. \\nGrover S et al. HLA alleles and hypersensitivity to carbamazepine: an updated systematic review with meta-analysis. Pharmacogenet Genomics 2014;24:94-112.\\n5. \\nTangamornsuksan W et al. Relationship between the HLA-B*1502 allele and carbamazepine-induced Stevens-Johnson syndrome and toxic epidermal necrolysis: a\\nsystematic review and meta-analysis. JAMA Dermatol 2013;149:1025-32.\\n6. \\nYip VL et al. HLA genotype and carbamazepine-induced cutaneous adverse drug reactions: a systematic review. Clin Pharmacol Ther 2012;92:757-65.\\n7. \\nChen P et al. Carbamazepine-induced toxic effects and HLA-B*1502 screening in Taiwan. N Engl J Med 2011;364:1126-33.\\n8. \\nSmPC’s Tegretol (NL en VS).\\n9. \\nHung SI et al. HLA-B genotyping to detect carbamazepine-induced Stevens-Johnson syndrome: implications for personalizing medicine. Personalized Med 2005;2:225–37.\\nDate 23-05-2022\\nHLA-B*1511: carbamazepine\\n6239\\nPatients with this genetic variation have an increased risk of experiencing the life-threatening cutaneous adverse event Stevens-Johnson syndrome/toxic epidermal necrolysis\\n(SJS/TEN). The risk of carbamazepine-induced SJS/TEN in patients with the HLA-B*1502 allele, which carries a 4.6-6.6 times higher risk than the HLA-B*1511 allele, is 1.8-\\n3.4%. This would equate to a risk of carbamazepine-induced SJS/TEN in these patients of 0.27-0.73%.\\nRecommendation:\\n1. \\ncarefully weigh the risk of SJS/TEN against the benefits\\n2. \\nif an alternative is an option, choose an alternative\\nLiterature:\\n1. \\nWang Q et al. Association between the HLA-B alleles and carbamazepine-induced SJS/TEN: a meta-analysis. Epilepsy Res 2017;135:19-28.\\n2. \\nGrover S et al. HLA alleles and hypersensitivity to carbamazepine: an updated systematic review with meta-analysis. Pharmacogenet Genomics 2014;24:94-112.\\nDate 23-05-2022\\nCYP2D6 IM: carvedilol\\n2345\\nNO action is required for this gene-drug interaction.\\nThe plasma concentration of carvedilol can be elevated. This does not, however, result in an increase in side effects.\\nLiterature:\\n1. \\nJung E et al. Influence of CYP2D6 polymorphism on the pharmacokinetic/pharmacodynamic characteristics of carvedilol in healthy Korean volunteers. J Korean Med Sci\\n2018;33:e182.\\n2. \\nLuzum JA et al. CYP2D6 genetic variation and beta-blocker maintenance dose in patients with heart failure. Pharm Res 2017;34:1615-25.\\n3. \\nShihmanter R et al. Variation in the CYP2D6 genotype is not associated with carvedilol dose changes in patients with heart failure. J Clin Pharm Ther 2014;39:432-8.\\n4. \\nNikolic VN et al. Population pharmacokinetics of carvedilol in patients with congestive heart failure. J Pharm Sci 2013;102:2851-8.\\n5. \\nSehrt D et al. Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics. Pharmacogenomics 2011;12:783-95.\\n6. \\nSaito M et al. Population pharmacokinetics of R- and S-carvedilol in Japanese patients with chronic heart failure. Biol Pharm Bull 2010;33:1378-84.\\n7. \\nBaudhuin LM et al. Relation of ADRB1, CYP2D6, and UGT1A1 polymorphisms with dose of, and response to, carvedilol or metoprolol therapy in patients with chronic\\nheart failure. Am J Cardiol 2010;106:402-8.\\n8. \\nHoriuchi I et al. Pharmacokinetics of R- and S-carvedilol in routinely treated Japanese patients with heart failure. Biol Pharm Bull 2008;31:976-80.\\n9. \\nTakekuma Y et al. Evaluation of effects of polymorphism for metabolic enzymes on pharmacokinetics of carvedilol by population pharmacokinetic analysis. Biol Pharm\\nBull 2007;30:537-42.\\n10. \\nHonda M et al. Multiple regression analysis of pharmacogenetic variability of carvedilol disposition in 54 healthy Japanese volunteers. Biol Pharm Bull 2006;29:772-8.\\n11. \\nTakekuma Y et al. Contribution of polymorphisms in UDP-glucuronosyltransferase and CYP2D6 to the individual variation in disposition of carvedilol. J Pharm Pharm Sci\\n2006;9:101-12.\\n12. \\nHonda M et al. Effect of CYP2D6*10 on the pharmacokinetics of R- and S-carvedilol in healthy Japanese volunteers. Biol Pharm Bull 2005;28:1476-9.\\nDate 12-09-2022\\nCYP2D6 PM: carvedilol\\n2344\\nNO action is required for this gene-drug interaction.\\nThe plasma concentration of carvedilol can be elevated. This does not, however, result in an increase in side effects.\\nLiterature:\\n1. \\nLuzum JA et al. CYP2D6 genetic variation and beta-blocker maintenance dose in patients with heart failure. Pharm Res 2017;34:1615-25.\\n2. \\nShihmanter R et al. Variation in the CYP2D6 genotype is not associated with carvedilol dose changes in patients with heart failure. J Clin Pharm Ther 2014;39:432-8.\\n3. \\nSehrt D et al. Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics. Pharmacogenomics 2011;12:783-95.\\n4. \\nBaudhuin LM et al. Relation of ADRB1, CYP2D6, and UGT1A1 polymorphisms with dose of, and response to, carvedilol or metoprolol therapy in patients with chronic\\nheart failure. Am J Cardiol 2010;106:402-8.\\n5. \\nGiessmann T et al. CYP2D6 genotype and induction of intestinal drug transporters by rifampin predict presystemic clearance of carvedilol in healthy subjects. Clin\\nPharmacol Ther 2004;75:213-22.\\n6. \\nZhou HH et al. Stereoselective disposition of carvedilol is determined by CYP2D6. Clin Pharmacol Ther 1995;57:518-24.\\n7. \\nSmPC’s Carvedilol Sandoz (Nederland) en Coreg (VS).\\nDate 12-09-2022\\nCYP2D6 UM: carvedilol\\n2346\\nNO action is required for this gene-drug interaction.\\nThe plasma concentration of carvedilol can be reduced. This does not, however, result in a decrease in the effectiveness.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 16}),\n",
       " Document(page_content='Literature:\\n1. \\nCabrera M et al. A use case to support precision medicine for frequently hospitalized older adults with polypharmacy. AMIA Jt Summits Transl Sci Proc 2016;2016:16-21.\\n2. \\nShihmanter R et al. Variation in the CYP2D6 genotype is not associated with carvedilol dose changes in patients with heart failure. J Clin Pharm Ther 2014;39:432-8.\\n3. \\nSehrt D et al. Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics. Pharmacogenomics 2011;12:783-95.\\n4. \\nBaudhuin LM et al. Relation of ADRB1, CYP2D6, and UGT1A1 polymorphisms with dose of, and response to, carvedilol or metoprolol therapy in patients with chronic\\nheart failure. Am J Cardiol 2010;106:402-8.\\nDate 12-09-2022\\nCYP2C19 IM: citalopram\\n4195\\nThe risk of QT prolongation and torsades de pointes is theoretically increased because the gene variation leads to an increased citalopram plasma concentration. If you follow\\nthe dose recommendation below, the increased plasma concentration and the theoretically increased risk of QT prolongation will be offset.\\nDo not exceed the following daily doses:\\n1. \\nadults up to 65 years: 30 mg as tablets or 22 mg as drops\\n2. \\nadults 65 years or older: 15 mg as tablets or 10 mg as drops\\nLiterature:\\n1. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol 2014;28:1143-8.\\n2. \\nChen B et al. Estimation of CYP2D6*10 genotypes on citalopram disposition in Chinese subjects by population pharmacokinetic assay. J Clin Pharm Ther 2013;38:504-11.\\n3. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n4. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;2:1-9.\\n5. \\nFudio S et al. Evaluation of the influence of sex and CYP2C19 and CYP2D6 polymorphisms in the disposition of citalopram. Eur J Pharmacol 2010;25:200-4.\\n6. \\nHilli J et al. MAO-A and COMT genotypes as possible regulators of perinatal serotonergic symptoms after in utero exposure to SSRIs. Eur Neuropsychopharmacol\\n2009;19:363-70.\\n7. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS ONE 2008;3:e1872.\\n8. \\nYin OQ et al. Phenotype-genotype relationship and clinical effects of citalopram in Chinese patients. J Clin Psychopharmacol 2006;26:367-72.\\n9. \\nRudberg I et al. Heterozygous mutation in CYP2C19 significantly increases the concentrations/dose ratio of racemic citalopram and escitalopram (S-citalopram). Ther\\nDrug Monitor 2006;28:102-5.\\n10. \\nYu BN et al. Pharmacokinetics of citalopram in relation to genetic polymorphism of CYP2C19. Drug Metab Dispos 2003;31:1255-9.\\n11. \\nJimmink A et al. Clinical toxicology of citalopram after acute intoxication with the sole drug or in combination with other drugs: overview of 26 cases. Ther Drug Monit\\n2008;30:365-71.\\nDate 14-05-2018\\nCYP2C19 PM: citalopram\\n4196\\nThe risk of QT prolongation and therefore also the theoretical risk of torsades de pointes is increased as the gene variation leads to an increased citalopram plasma\\nconcentration. If you follow the dose recommendation below, the increased plasma concentration and the increased risk of QT prolongation will be offset.\\ndo not exceed the following daily doses (50% of the standard maximum dose):\\n1. \\nadults up to 65 years: 20 mg as tablets or 16 mg as drops\\n2. \\nadults 65 years or older: 10 mg as tablets or 8 mg as drops\\nLiterature:\\n1. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol 2014;28:1143-8.\\n2. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n3. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;2:1-9.\\n4. \\nFudio S et al. Evaluation of the influence of sex and CYP2C19 and CYP2D6 polymorphisms in the disposition of citalopram. Eur J Pharmacol 2010;25:200-4.\\n5. \\nHilli J et al. MAO-A and COMT genotypes as possible regulators of perinatal serotonergic symptoms after in utero exposure to SSRIs. Eur Neuropsychopharmacol\\n2009;19:363-70.\\n6. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS ONE 2008;3:e1872.\\n7. \\nYin OQ et al. Phenotype-genotype relationship and clinical effects of citalopram in Chinese patients. J Clin Psychopharmacol 2006;26:367-72.\\n8. \\nHerrlin K et al. Metabolism of citalopram enantiomers in CYP2C19/CYP2D6 phenotyped panels of healthy Swedes. Br J Clin Pharmacol 2003;56:415-21.\\n9. \\nYu BN et al. Pharmacokinetics of citalopram in relation to genetic polymorphism of CYP2C19. Drug Metab Dispos 2003;31:1255-9.\\n10. \\nBaumann P et al. A double-blind, placebo-controlled study of citalopram with and without lithium in the treatment of therapy-resistant depressive patients: a clinical,\\npharmacokinetic, and pharmacogenetic investigation. J Clin Psychopharmacol 1996;16:307-14.\\n11. \\nSindrup SH et al. Pharmacokinetics of citalopram in relation to the sparteine and the mephenytoin oxidation polymorphisms. Ther Drug Monit 1993;15:11-7.\\n12. \\nJimmink A et al. Clinical toxicology of citalopram after acute intoxication with the sole drug or in combination with other drugs: overview of 26 cases. Ther Drug Monit\\n2008;30:365-71.\\n13. \\nSPC’s Cipramil en Celexa (VS).\\nDate 14-05-2018\\nCYP2C19 UM: citalopram\\n4197\\nNO action is needed for this gene-drug interaction.\\nThe gene variation increases conversion of citalopram to a weakly active metabolite. However, there is no significant effect on the plasma concentration of citalopram, the\\ntolerance or the response.\\nLiterature:\\n1. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n2. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;2:1-9.\\n3. \\nHilli J et al. MAO-A and COMT genotypes as possible regulators of perinatal serotonergic symptoms after in utero exposure to SSRIs. Eur Neuropsychopharmacol\\n2009;19:363-70.\\n4. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS ONE 2008;3:e1872.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 17}),\n",
       " Document(page_content='5. \\nJimmink A et al. Clinical toxicology of citalopram after acute intoxication with the sole drug or in combination with other drugs: overview of 26 cases. Ther Drug Monit\\n2008;30:365-71.\\nDate 14-05-2018\\nCYP2D6 IM: citalopram/escitalopram\\n1999\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nChen B et al. Estimation of CYP2D6*10 genotypes on citalopram disposition in Chinese subjects by population pharmacokinetic assay. J Clin Pharm Ther 2013;38:504-11.\\nPubMed PMID: 23981149.\\n2. \\nHan KM et al. CYP2D6 P34S polymorphism and outcomes of escitalopram treatment in Koreans with major depression. Psychiatry Investig 2013;10:286-93. PubMed\\nPMID: 24302953.\\n3. \\nHuezo-Diaz P et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol 2012;26:398-407. PubMed PMID: 21926427.\\n4. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67. PubMed PMID: 20531370.\\n5. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;21:1-9. PubMed PMID: 21192344.\\n6. \\nTsai MH et al. Genetic polymorphisms of cytochrome P450 enzymes influence metabolism of the antidepressant escitalopram and treatment response. Pharmacogenomics\\n2010;11:537-46. PubMed PMID: 20350136.\\n7. \\nFudio S et al. Evaluation of the influence of sex and CYP2C19 and CYP2D6 polymorphisms in the disposition of citalopram. Eur J Pharmacol 2010;626:200-4. PubMed\\nPMID: 19840783.\\n8. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS One 2008;3:e1872. PubMed PMID: 18382661.\\n9. \\nCarlsson B et al. Enantioselective analysis of citalopram and metabolites in adolescents. Ther Drug Monit 2001;23:658-64.\\n10. \\nSPC Cipramil.\\nDate 14-05-2018\\nCYP2D6 PM: citalopram/escitalopram\\n1998\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nHuezo-Diaz P et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol 2012;26:398-407. PubMed PMID: 21926427.\\n2. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67. PubMed PMID: 20531370.\\n3. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;21:1-9. PubMed PMID: 21192344.\\n4. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS One 2008;3:e1872. PubMed PMID: 18382661.\\n5. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and clinical effects of antidepressants in a\\nnaturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n6. \\nHerrlin K et al. Metabolism of citalopram enantiomers in CYP2C19/CYP2D6 phenotyped panels of healthy Swedes. Br J Clin Pharmacol 2003;56:415-21.\\n7. \\nBondolfi G et al. Non-response to citalopram in depressive patients: pharmacokinetic and clinical consequences of a fluvoxamine augmentation. Psychopharmacology\\n1996;128:421-5.\\n8. \\nSindrup SH et al. Pharmacokinetics of citalopram in relation to the sparteine and the mephenytoin oxidation polymorphisms. Ther Drug Monit 1993;15:11-7.\\n9. \\nSPC’s Cipramil, Lexapro (NL en VS) en Celexa (VS).\\nDate 14-05-2018\\nCYP2D6 UM: citalopram/escitalopram\\n2000\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nHuezo-Diaz P et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol 2012;26:398-407. PubMed PMID: 21926427.\\n2. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67. PubMed PMID: 20531370.\\n3. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;21:1-9. PubMed PMID: 21192344.\\n4. \\nCarlsson B et al. Enantioselective analysis of citalopram and metabolites in adolescents. Ther Drug Monit 2001;23:658-64.\\n5. \\nSPC Cipramil.\\nDate 14-05-2018\\nCYP2C19 IM: clomipramine\\n7027\\nNO action is required for this gene-drug interaction.\\nThe gene variation does increase clomipramine plasma concentrations, but not clomipramine+desmethylclomipramine plasma concentrations, which determines side effects\\nand efficacy in depression. The increase in the plasma concentration of clomipramine is favourable for the efficacy in anxiety and obsessive compulsive disorder.\\nLiterature:\\n1. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n2. \\nYokono A et al. The effect of CYP2C19 and CYP2D6 genotypes on the metabolism of clomipramine in Japanese psychiatric patients. J Clin Psychopharmacol 2001; 21:549-\\n55.\\nDate 04-03-2019', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 18}),\n",
       " Document(page_content='CYP2C19 PM: clomipramine\\n7028\\nNO action is required for this gene-drug interaction.\\nThe gene variation increases the plasma concentration of clomipramine. However, there is insufficient evidence to substantiate an increase of the plasma concentration of\\nclomipramine+desmethylclomipramine to such an extent that it increases the risk of side effects. The increase in the plasma concentration of clomipramine is favourable for\\nthe efficacy in anxiety and obsessive compulsive disorder.\\nLiterature:\\n1. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n2. \\nYokono A et al. The effect of CYP2C19 and CYP2D6 genotypes on the metabolism of clomipramine in Japanese psychiatric patients. J Clin Psychopharmacol 2001; 21:549-\\n55.\\n3. \\nNielsen KK et al. Single-dose kinetics of clomipramine: relationship to the sparteine and S-mephenytoin oxidation polymorphisms. Clin Pharmacol Ther 1994;55:518-27.\\nDate 04-03-2019\\nCYP2C19 UM: clomipramine\\n7029\\nThe gene variation increases the risk of ineffectiveness for obsessive compulsive disorder and anxiety disorders by reducing the plasma concentration of clomipramine. The\\ngene variation has little to no effect on the plasma concentration of clomipramine+desmethylclomipramine, which determines the efficacy for depression and side effects.\\nIndication OBSESSIVE COMPULSIVE DISORDER or ANXIETY DISORDERS:\\navoid clomipramine\\nAntidepressants that are not metabolised by CYP2C19 - or to a lesser extent - include, for example, fluoxetine, fluvoxamine and paroxetine.\\nif it is not possible to avoid clomipramine:\\nmonitor the effect and side effects or the plasma concentrations of clomipramine and desmethylclomipramine\\nFor obsessive compulsive disorder, the therapeutic plasma concentration of clomipramine is greater than 200 ng/mL in combination with a plasma\\nconcentration of desmethylclomipramine that is as low as possible.\\nFor anxiety disorders, the therapeutic plasma concentration of clomipramine is approximately 100 ng/mL in combination with a plasma concentration of\\ndesmethylclomipramine lower than 200 ng/mL.\\nA sum of the plasma concentrations of clomipramine and desmethylclomipramine exceeding 600 ng/mL is considered toxic.\\nadd a low dose of fluvoxamine if necessary, to inhibit CYP2C19 and CYP1A2 and thereby inhibit the conversion of clomipramine to desmethylclomipramine\\nIndication DEPRESSION:\\nno action required\\nLiterature:\\n1. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\nDate 04-03-2019\\nCYP2D6 IM: clomipramine\\n1481\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of clomipramine and the active metabolite\\ndesmethylclomipramine.\\nuse 70% of the standard dose and monitor the effect and side effects or the plasma concentrations of clomipramine and desmethylclomipramine\\nFor depression, the therapeutic range is 200-400 ng/mL for the sum of the plasma concentrations of clomipramine and desmethylclomipramine.\\nFor anxiety disorders, the therapeutic plasma concentration of clomipramine is approximately 100 ng/mL, in combination with a plasma concentration of\\ndesmethylclomipramine lower than 200 ng/mLFor obsessive compulsive disorder, the therapeutic plasma concentration of clomipramine is higher than 200 ng/mL, in\\ncombination with a plasma concentration of desmethylclomipramine that is as low as possible.\\nA sum of the plasma concentrations of clomipramine and desmethylclomipramine higher than 600 ng/mL is considered toxic.\\nLiterature:\\n1. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n2. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67, persoonlijke communicatie.\\n3. \\nVandel P et al. Clomipramine, fluoxetine and CYP2D6 metabolic capacity in depressed patients. Hum Psychopharmacol 2004;19:293-8.\\n4. \\nYokono A et al. The effect of CYP2C19 and CYP2D6 genotypes on the metabolism of clomipramine in Japanese psychiatric patients. Clin Psychopharmacol 2001;21:549-\\n55.\\n5. \\nBalant-Gorgia et al. High blood concentrations of imipramine or clomipramine and therapeutic failure: a case report study using drug monitoring data. Ther Drug Monit\\n1989;11:415-20.\\nDate 19-11-2018\\nCYP2D6 PM: clomipramine\\n1480\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of clomipramine and the active metabolite\\ndesmethylclomipramine.\\nIndication DEPRESSION:\\nuse 50% of the standard dose and monitor the effect and side effects or the plasma concentrations of clomipramine and desmethylclomipramine in order to set the\\nmaintenance dose.\\nThe therapeutic range is 200-400 ng/mL for the sum of the plasma concentrations of clomipramine and desmethylclomipramine. Values higher than 600 ng/mL are\\nconsidered toxic.\\nIndication ANXIETY DISORDERS or OBSESSIVE COMPULSIVE DISORDER:\\nif side effects occur: use 50% of the standard dose and monitor the effect and side effects or the plasma concentrations of clomipramine and desmethylclomipramine\\nin order to set the maintenance dose.\\nIt is not known whether it is possible to reduce the dose to such an extent that the side effects disappear while the effectiveness is retained. Clomipramine and\\ndesmethylclomipramine both contribute to the side effects. Only clomipramine contributes to the effectiveness.\\nFor anxiety disorders, the therapeutic plasma concentration of clomipramine is approximately 100 ng/mL, in combination with a plasma concentration of\\ndesmethylclomipramine lower than 200 ng/mL..\\nFor obsessive compulsive disorder, the therapeutic plasma concentration of clomipramine is higher than 200 ng/mL, in combination with a plasma concentration of\\ndesmethylclomipramine that is as low as possible.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 19}),\n",
       " Document(page_content='A sum of the plasma concentrations of clomipramine and desmethylclomipramine higher than 600 ng/mL is considered toxic, whilst the therapeutic upper limit for\\ndepression is 400 ng/mL. .\\nif dose reduction does not have the desired effect: avoid clomipramine\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include citalopram and sertraline.\\nLiterature:\\n1. \\nde Jong C et al. Clomipramine toxicity in a CYP 2D6 poor metabolizer patient who suddenly stopped smoking. J Clin Psychopharmacol 2018;38:389-391.\\n2. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n3. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67, persoonlijke communicatie.\\n4. \\nStephan PL et al. Adverse drug reactions following nonresponse in a depressed patient with CYP2D6 deficiency and low CYP 3A4/5 activity. Pharmacopsychiatry\\n2006;39:150-2.\\n5. \\nDanish University Antidepressant Group. Clomipramine dose-effect study in patients with depression: clinical end points and pharmacokinetics. Clin Pharmacol Ther\\n1999;66:152-65.\\n6. \\nNielsen KK et al. Single-dose kinetics of clomipramine: relationship to the sparteine and S-mephenytoin oxidation polymorphisms. Clin Pharmacol Ther 1994;55:518-27.\\n7. \\nTacke U et al. Debrisoquine hydroxylation phenotypes of patients with high versus low to normal serum antidepressant concentrations. J Clin Psychopharmacol\\n1992;12:262-7.\\n8. \\nNielsen KK et al. Steady-state plasma levels of clomipramine and its metabolites: impact of the sparteine/debrisoquine oxidation polymorphism. Eur J Clin Pharmacol\\n1992;43:405-11.\\n9. \\nBalant-Gorgia AE et al. High plasma concentrations of desmethylclomipramine after chronic administration of clomipramine to a poor metabolizer. Eur J Clin Pharmacol\\n1987;32:101-2.\\n10. \\nBalant-Gorgia AE et al. High blood concentrations of imipramine or clomipramine and therapeutic failure: a case report study using drug monitoring data. Ther Drug\\nMonit 1989;11:415-20.\\n11. \\nSmPC Anafranil (VS).\\nDate 19-11-2018\\nCYP2D6 UM: clomipramine\\n1482\\nThe risk of ineffectiveness and cardiotoxic side effects may be increased. The gene variation leads to reduced plasma concentrations of clomipramine and the active metabolite\\ndesmethylclomipramine and to increased concentrations of the potentially cardiotoxic hydroxy metabolites.\\nuse 1.5 times the standard dose and monitor the effect and side effects of the plasma concentrations of clomipramine and desmethylclomipramine to set the maintenance\\ndose.\\nFor depression, the therapeutic range is 200-400 ng/mL for the sum of the plasma concentrations of clomipramine and desmethylclomipramine.\\nFor anxiety disorders, the therapeutic plasma concentration of clomipramine is approximately 100 ng/mL, in combination with a plasma concentration of\\ndesmethylclomipramine lower than 200 ng/mL.\\nFor obsessive compulsive disorder, the therapeutic plasma concentration of clomipramine is higher than 200 ng/mL, in combination with a plasma concentration of\\ndesmethylclomipramine that is as low as possible.\\nif a dose increase is not wanted due to potential cardiotoxic hydroxy metabolites: avoid clomipramine.\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include citalopram and sertraline.\\nLiterature:\\n1. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67.\\n2. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch hospitalized patients. Pharmacogenomics J\\n2011;11:359-67, persoonlijke communicatie.\\n3. \\nBaumann P et al. Ultrarapid metabolism of clomipramine in a therapy-resistant depressive patient, as confirmed by CYP2 D6 genotyping. Pharmacopsychiatry\\n1998;31:72.\\n4. \\nBertilsson L et al. Molecular basis for rational megaprescribing in ultrarapid hydroxylators of debrisoquine. Lancet 1993;341:63.\\nDate 19-11-2018\\nCYP2D6 IM: clonidine\\n2531\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 31-01-2022\\nCYP2D6 PM: clonidine\\n2530\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 31-01-2022\\nCYP2D6 UM: clonidine\\n2532\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 20}),\n",
       " Document(page_content='Literature:\\n-\\nDate 31-01-2022\\nCYP2C19 IM: clopidogrel\\n2549\\nThe risk of serious cardiovascular and cerebrovascular events is increased in patients undergoing balloon angioplasty or stent placement (percutaneous coronary intervention)\\nand in patients with a stroke or TIA, as the genetic variation reduces the activation of clopidogrel. No negative clinical consequences have been observed in other patients.\\nPERCUTANEOUS CORONARY INTERVENTION, STROKE or TIA:\\nchoose an alternative or double the dose to 150 mg/day (600 mg loading dose)\\nPrasugrel, ticagrelor and acetylsalicylic acid/dipyridamole are not metabolised by CYP2C19 (or to a lesser extent).\\nOTHER INDICATIONS:\\nno action required\\nLiterature:\\n1. \\nLee J et al. CYP2C19 polymorphism is associated with amputation rates in patients taking clopidogrel after endovascular intervention for critical limb ischaemia. Eur J\\nVasc Endovasc Surg 2019;58:373-82.\\n2. \\nLan H et al. Anti-platelet therapy in mild cerebral infarction patients on the basis of CYP2C19 metabolizer status. Cell Transplant 2019;28:1039-44.\\n3. \\nKheiri B et al. CYP2C19 pharmacogenetics versus standard of care dosing for selecting antiplatelet therapy in patients with coronary artery disease: a meta-analysis of\\nrandomized clinical trials. Catheter Cardiovasc Interv 2019;93:1246-52.\\n4. \\nWilliams AK et al. CYP2C19 genotype-guided antiplatelet therapy and 30-day outcomes after percutaneous coronary intervention. Circ Genom Precis Med\\n2019;12:e002441.\\n5. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous coronary intervention. Circ Genom Precis\\nMed 2018;11:e002069.\\n6. \\nZhong Z et al. Effect of cytochrome P450 2C19 polymorphism on adverse cardiovascular events after drug-eluting stent implantation in a large Hakka population with\\nacute coronary syndrome receiving clopidogrel in southern China. Eur J Clin Pharmacol 2018;74:423-31.\\n7. \\nWu Y et al. Impact of CYP2C19 polymorphism in prognosis of minor stroke or TIA patients with declined eGFR on dual antiplatelet therapy: CHANCE substudy.\\nPharmacogenomics J 2018;18:713-20.\\n8. \\nCavallari LH et al. Multisite investigation of outcomes with implementation of CYP2C19 genotype-guided antiplatelet therapy after percutaneous coronary intervention.\\nJACC Cardiovasc Interv 2018;11:181-91.\\n9. \\nLin Y et al. Impact of glycemic control on efficacy of clopidogrel in transient ischemic attack or minor stroke patients with CYP2C19 genetic variants. Stroke 2017;48:998-\\n1004.\\n10. \\nPan Y et al. Genetic polymorphisms and clopidogrel efficacy for acute ischemic stroke or transient ischemic attack: a systematic review and meta-analysis. Circulation\\n2017;135:21-33.\\n11. \\nWang Y et al. Association between CYP2C19 loss-of-function allele status and efficacy of clopidogrel for risk reduction among patients with minor stroke or transient\\nischemic attack. JAMA 2016;316:70-8.\\n12. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events in Japanese patients with acute coronary\\nsyndrome: The PRASFIT-ACS study.\\n13. \\nShen DL et al. Clinical value of CYP2C19 genetic testing for guiding the antiplatelet therapy in a Chinese population. J Cardiovasc Pharmacol 2016;67:232-6.\\n14. \\nNiu X et al. CYP2C19 polymorphism and clinical outcomes among patients of different races treated with clopidogrel: a systematic review and meta-analysis. J Huazhong\\nUniv Sci Technolog Med Sci 2015;35:147-56.\\n15. \\nSorich MJ et al. CYP2C19 genotype has a greater effect on adverse cardiovascular outcomes following percutaneous coronary intervention and in Asian populations\\ntreated with clopidogrel: a meta-analysis. Circ Cardiovasc Genet 2014;7:895-902.\\n16. \\nMao L et al. Cytochrome CYP2C19 polymorphism and risk of adverse clinical events in clopidogrel-treated patients: a meta-analysis based on 23,035 subjects. Arch\\nCardiovasc Dis 2013;106:517-27.\\n17. \\nXie X et al. Personalized antiplatelet therapy according to CYP2C19 genotype after percutaneous coronary intervention: a randomized control trial. Int J Cardiol\\n2013;168:3736-40.\\n18. \\nJang JS et al. Meta-analysis of cytochrome P450 2C19 polymorphism and risk of adverse clinical outcomes among coronary artery disease patients of different ethnic\\ngroups treated with clopidogrel. Am J Cardiol 2012;110:502-8.\\n19. \\nHolmes MV et al. CYP2C19 genotype, clopidogrel metabolism, platelet function, and cardiovascular events: a systematic review and meta-analysis. JAMA 2011;306:2704-\\n14.\\n20. \\nLiu YP et al. Association of genetic variants in CYP2C19 and adverse clinical outcomes after treatment with clopidogrel: an updated meta-analysis. Thromb Res\\n2011;128:593-4.\\n21. \\nMega JL et al. Dosing clopidogrel based on CYP2C19 genotype and the effect on platelet reactivity in patients with stable cardiovascular disease. JAMA 2011;306:2221-8.\\n22. \\nSimon T et al. Genetic polymorphisms and the impact of a higher clopidogrel dose regimen on active metabolite exposure and antiplatelet response in healthy subjects.\\nClin Pharmacol Ther 2011;90:287-95.\\n23. \\nCollet JP et al. High doses of clopidogrel to overcome genetic resistance: the randomized cross-over CLOVIS-2 (Clopidogrel and Response Variability Investigation Study\\n2). JACC Cardiovasc Interv 2011;4:392-402.\\n24. \\nBonello-Palot N et al. Relation of body mass index to high on-treatment platelet reactivity and of failed clopidogrel dose adjustment according to platelet reactivity\\nmonitoring in patients undergoing percutaneous coronary intervention. Am J Cardiol 2009;104:1511-5.\\n25. \\nShuldiner AR et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA 2009;302:849-57.\\n26. \\nAleil B et al. CYP2C19*2 polymorphism is not the sole determinant of the response to clopidogrel: implications for its monitoring. J Thromb Haemost 2009;7:1747-9.\\n27. \\nPena A et al. Can we override clopidogrel resistance? Circulation 2009;119:2854-7.\\n28. \\nSibbing D et al. Cytochrome P450 2C19 loss-of-function polymorphism and stent thrombosis following percutaneous coronary intervention. Eur Heart J 2009;30:916-22.\\n29. \\nBrackbill ML et al. Frequency of CYP3A4, CYP3A5, CYP2C9, and CYP2C19 variant alleles in patients receiving clopidogrel that experience repeat acute coronary\\nsyndrome. Heart Vessels 2009;24:73-8.\\n30. \\nGiusti B et al. Relation of cytochrome P450 2C19 loss-of-function polymorphism to occurrence of drug-eluting coronary stent thrombosis. Am J Cardiol 2009;103:806-11.\\n31. \\nCollet JP et al. Cytochrome P450 2C19 polymorphism in young patients treated with clopidogrel after myocardial infarction: a cohort study. Lancet 2009;373:309-17.\\n32. \\nMega JL et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N Engl J Med 2009;360:354-62.\\n33. \\nSimon T et al. Genetic determinants of response to clopidogrel and cardiovascular events. N Engl J Med 2009;360:363-75.\\n34. \\nGeisler T et al. CYP2C19 and nongenetic factors predict poor responsiveness to clopidogrel loading dose after coronary stent implantation. Pharmacogenomics\\n2008;9:1251-9.\\n35. \\nUmemura K et al. The common gene variants of CYP2C19 affect pharmacokinetics and pharmacodynamics in an active metabolite of clopidogrel in healthy subjects. J\\nThromb Haemost 2008;6:1439-41.\\n36. \\nChen BL et al. Inhibition of ADP-induced platelet aggregation by clopidogrel is related to CYP2C19 genetic polymorphisms. Clin Exp Pharmacol Physiol 2008;35:904-8.\\n37. \\nKim KA et al. The effect of CYP2C19 polymorphism on the pharmacokinetics and pharmacodynamics of clopidogrel: a possible mechanism for clopidogrel resistance. Clin\\nPharmacol Ther 2008;84:236-42.\\n38. \\nMalek LA et al. Coexisting polymorphisms of P2Y12 and CYP2C19 genes as a risk factor for persistent platelet activation with clopidogrel. Circ J 2008;72:1165-9.\\n39. \\nTrenk D et al. Cytochrome P450 2C19 681G>A polymorphism and high on-clopidogrel platelet reactivity associated with adverse 1-year clinical outcome of elective\\npercutaneous coronary intervention with drug-eluting or bare-metal stents. J Am Coll Cardiol 2008;51:1925-34.\\n40. \\nFrere C et al. Effect of cytochrome p450 polymorphisms on platelet reactivity after treatment with clopidogrel in acute coronary syndrome. Am J Cardiol 2008;101:1088-\\n93.\\n41. \\nFontana P et al. Biological effect of increased maintenance dose of clopidogrel in cardiovascular outpatients and influence of the cytochrome P450 2C19*2 allele on\\nclopidogrel responsiveness. Thromb Res 2008;121:463-8.\\n42. \\nGiusti B et al. Cytochrome P450 2C19 loss-of-function polymorphism, but not CYP3A4 IVS10 + 12G/A and P2Y12 T744C polymorphisms, is associated with response\\nvariability to dual antiplatelet treatment in high-risk vascular patients. Pharmacogenet Genomics 2007;17:1057-64.\\n43. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response to clopidogrel but not prasugrel. J Thromb\\nHaemost 2007;5:2429-36.\\n44. \\nFontana P et al. Influence of CYP2C19 and CYP3A4 gene polymorphisms on clopidogrel responsiveness in healthy subjects. J Thromb Haemost 2007;5:2153-5.\\n45. \\nHulot JS et al. Cytochrome P450 2C19 loss-of-function polymorphism is a major determinant of clopidogrel responsiveness in healthy subjects. Blood 2006;108:2244-7.\\n46. \\nCBO Conceptrichtlijn Diagnostiek, Preventie en Behandeling van Veneuze Trombo-embolie en Secundaire Preventie Arteriële Trombose, 2008. www.cbo.nl.\\nDate 23-12-2019\\nCYP2C19 PM: clopidogrel\\n2548', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 21}),\n",
       " Document(page_content='The risk of serious cardiovascular and cerebrovascular events is increased in patients undergoing balloon angioplasty or stent placement (percutaneous coronary intervention)\\nand in patients with a stroke or TIA, because the genetic variation reduces the activation of clopidogrel. No negative clinical consequences have been proved in other patients.\\nPERCUTANEOUS CORONARY INTERVENTION, STROKE or TIA:\\navoid clopidogrel\\nPrasugrel, ticagrelor and acetylsalicylic acid/dipyridamole are not metabolised by CYP2C19 (or to a lesser extent).\\nOTHER INDICATIONS:\\ndetermine the level of inhibition of platelet aggregation by clopidogrel\\nconsider an alternative in poor responders\\nPrasugrel and ticagrelor are not metabolised by CYP2C19 (or to a lesser extent).\\nLiterature:\\n1. \\nLee J et al. CYP2C19 polymorphism is associated with amputation rates in patients taking clopidogrel after endovascular intervention for critical limb ischaemia. Eur J\\nVasc Endovasc Surg 2019;58:373-82.\\n2. \\nLan H et al. Anti-platelet therapy in mild cerebral infarction patients on the basis of CYP2C19 metabolizer status. Cell Transplant 2019;28:1039-44.\\n3. \\nKheiri B et al. CYP2C19 pharmacogenetics versus standard of care dosing for selecting antiplatelet therapy in patients with coronary artery disease: a meta-analysis of\\nrandomized clinical trials. Catheter Cardiovasc Interv 2019;93:1246-52.\\n4. \\nWilliams AK et al. CYP2C19 genotype-guided antiplatelet therapy and 30-day outcomes after percutaneous coronary intervention. Circ Genom Precis Med\\n2019;12:e002441.\\n5. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous coronary intervention. Circ Genom Precis\\nMed 2018;11:e002069.\\n6. \\nZhong Z et al. Effect of cytochrome P450 2C19 polymorphism on adverse cardiovascular events after drug-eluting stent implantation in a large Hakka population with\\nacute coronary syndrome receiving clopidogrel in southern China. Eur J Clin Pharmacol 2018;74:423-31.\\n7. \\nWu Y et al. Impact of CYP2C19 polymorphism in prognosis of minor stroke or TIA patients with declined eGFR on dual antiplatelet therapy: CHANCE substudy.\\nPharmacogenomics J 2018;18:713-20.\\n8. \\nCavallari LH et al. Multisite investigation of outcomes with implementation of CYP2C19 genotype-guided antiplatelet therapy after percutaneous coronary intervention.\\nJACC Cardiovasc Interv 2018;11:181-91.\\n9. \\nLin Y et al. Impact of glycemic control on efficacy of clopidogrel in transient ischemic attack or minor stroke patients with CYP2C19 genetic variants. Stroke 2017;48:998-\\n1004.\\n10. \\nPan Y et al. Genetic polymorphisms and clopidogrel efficacy for acute ischemic stroke or transient ischemic attack: a systematic review and meta-analysis. Circulation\\n2017;135:21-33.\\n11. \\nDeiman BA et al. Reduced number of cardiovascular events and increased cost-effectiveness by genotype-guided antiplatelet therapy in patients undergoing percutaneous\\ncoronary interventions in the Netherlands. Neth Heart J 2016;24:589-99.\\n12. \\nWang Y et al. Association between CYP2C19 loss-of-function allele status and efficacy of clopidogrel for risk reduction among patients with minor stroke or transient\\nischemic attack. JAMA 2016;316:70-8.\\n13. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events in Japanese patients with acute coronary\\nsyndrome: The PRASFIT-ACS study.\\n14. \\nShen DL et al. Clinical value of CYP2C19 genetic testing for guiding the antiplatelet therapy in a Chinese population. J Cardiovasc Pharmacol 2016;67:232-6.\\n15. \\nXiong R et al. A randomized controlled trial to assess the efficacy and safety of doubling dose clopidogrel versus ticagrelor for the treatment of acute coronary syndrome\\nin patients with CYP2C19*2 homozygotes. Int J Clin Exp Med 2015;8:13310-6.\\n16. \\nNiu X et al. CYP2C19 polymorphism and clinical outcomes among patients of different races treated with clopidogrel: a systematic review and meta-analysis. J Huazhong\\nUniv Sci Technolog Med Sci 2015;35:147-56.\\n17. \\nSorich MJ et al. CYP2C19 genotype has a greater effect on adverse cardiovascular outcomes following percutaneous coronary intervention and in Asian populations\\ntreated with clopidogrel: a meta-analysis. Circ Cardiovasc Genet 2014;7:895-902.\\n18. \\nMao L et al. Cytochrome CYP2C19 polymorphism and risk of adverse clinical events in clopidogrel-treated patients: a meta-analysis based on 23,035 subjects. Arch\\nCardiovasc Dis 2013;106:517-27.\\n19. \\nXie X et al. Personalized antiplatelet therapy according to CYP2C19 genotype after percutaneous coronary intervention: a randomized control trial. Int J Cardiol\\n2013;168:3736-40.\\n20. \\nJang JS et al. Meta-analysis of cytochrome P450 2C19 polymorphism and risk of adverse clinical outcomes among coronary artery disease patients of different ethnic\\ngroups treated with clopidogrel. Am J Cardiol 2012;110:502-8.\\n21. \\nHolmes MV et al. CYP2C19 genotype, clopidogrel metabolism, platelet function, and cardiovascular events: a systematic review and meta-analysis. JAMA 2011;306:2704-\\n14.\\n22. \\nLiu YP et al. Association of genetic variants in CYP2C19 and adverse clinical outcomes after treatment with clopidogrel: an updated meta-analysis. Thromb Res\\n2011;128:593-4.\\n23. \\nMega JL et al. Dosing clopidogrel based on CYP2C19 genotype and the effect on platelet reactivity in patients with stable cardiovascular disease. JAMA 2011;306:2221-8.\\n24. \\nSimon T et al. Genetic polymorphisms and the impact of a higher clopidogrel dose regimen on active metabolite exposure and antiplatelet response in healthy subjects.\\nClin Pharmacol Ther 2011;90:287-95.\\n25. \\nCollet JP et al. High doses of clopidogrel to overcome genetic resistance: the randomized cross-over CLOVIS-2 (Clopidogrel and Response Variability Investigation Study\\n2). JACC Cardiovasc Interv 2011;4:392-402.\\n26. \\nBonello-Palot N et al. Relation of body mass index to high on-treatment platelet reactivity and of failed clopidogrel dose adjustment according to platelet reactivity\\nmonitoring in patients undergoing percutaneous coronary intervention. Am J Cardiol 2009;104:1511-5.\\n27. \\nShuldiner AR et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA 2009;302:849-57.\\n28. \\nAleil B et al. CYP2C19*2 polymorphism is not the sole determinant of the response to clopidogrel: implications for its monitoring. J Thromb Haemost 2009;7:1747-9.\\n29. \\nPena A et al. Can we override clopidogrel resistance? Circulation 2009;119:2854-7.\\n30. \\nSibbing D et al. Cytochrome P450 2C19 loss-of-function polymorphism and stent thrombosis following percutaneous coronary intervention. Eur Heart J 2009;30:916-22.\\n31. \\nBrackbill ML et al. Frequency of CYP3A4, CYP3A5, CYP2C9, and CYP2C19 variant alleles in patients receiving clopidogrel that experience repeat acute coronary\\nsyndrome. Heart Vessels 2009;24:73-8.\\n32. \\nGiusti B et al. Relation of cytochrome P450 2C19 loss-of-function polymorphism to occurrence of drug-eluting coronary stent thrombosis. Am J Cardiol 2009;103:806-11.\\n33. \\nCollet JP et al. Cytochrome P450 2C19 polymorphism in young patients treated with clopidogrel after myocardial infarction: a cohort study. Lancet 2009;373:309-17.\\n34. \\nMega JL et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N Engl J Med 2009;360:354-62.\\n35. \\nSimon T et al. Genetic determinants of response to clopidogrel and cardiovascular events. N Engl J Med 2009;360:363-75.\\n36. \\nGeisler T et al. CYP2C19 and nongenetic factors predict poor responsiveness to clopidogrel loading dose after coronary stent implantation. Pharmacogenomics\\n2008;9:1251-9.\\n37. \\nUmemura K et al. The common gene variants of CYP2C19 affect pharmacokinetics and pharmacodynamics in an active metabolite of clopidogrel in healthy subjects. J\\nThromb Haemost 2008;6:1439-41.\\n38. \\nChen BL et al. Inhibition of ADP-induced platelet aggregation by clopidogrel is related to CYP2C19 genetic polymorphisms. Clin Exp Pharmacol Physiol 2008;35:904-8.\\n39. \\nKim KA et al. The effect of CYP2C19 polymorphism on the pharmacokinetics and pharmacodynamics of clopidogrel: a possible mechanism for clopidogrel resistance. Clin\\nPharmacol Ther 2008;84:236-42.\\n40. \\nMalek LA et al. Coexisting polymorphisms of P2Y12 and CYP2C19 genes as a risk factor for persistent platelet activation with clopidogrel. Circ J 2008;72:1165-9.\\n41. \\nTrenk D et al. Cytochrome P450 2C19 681G>A polymorphism and high on-clopidogrel platelet reactivity associated with adverse 1-year clinical outcome of elective\\npercutaneous coronary intervention with drug-eluting or bare-metal stents. J Am Coll Cardiol 2008;51:1925-34.\\n42. \\nFrere C et al. Effect of cytochrome p450 polymorphisms on platelet reactivity after treatment with clopidogrel in acute coronary syndrome. Am J Cardiol 2008;101:1088-\\n93.\\n43. \\nFontana P et al. Biological effect of increased maintenance dose of clopidogrel in cardiovascular outpatients and influence of the cytochrome P450 2C19*2 allele on\\nclopidogrel responsiveness. Thromb Res 2008;121:463-8.\\n44. \\nGiusti B et al. Cytochrome P450 2C19 loss-of-function polymorphism, but not CYP3A4 IVS10 + 12G/A and P2Y12 T744C polymorphisms, is associated with response\\nvariability to dual antiplatelet treatment in high-risk vascular patients. Pharmacogenet Genomics 2007;17:1057-64.\\n45. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response to clopidogrel but not prasugrel. J Thromb\\nHaemost 2007;5:2429-36.\\n46. \\nFDA Drug Safety Communication: Reduced effectiveness of Plavix (clopidogrel) in patients who are poor metabolizers of the drug. 03-12-10.\\n47. \\nSPC’s Clopidogrel Sandoz en Plavix (VS).\\n48. \\nCBO Conceptrichtlijn Diagnostiek, Preventie en Behandeling van Veneuze Trombo-embolie en Secundaire Preventie Arteriële Trombose, 2008. www.cbo.nl.\\nDate 23-12-2019\\nCYP2C19 UM: clopidogrel\\n2550\\nNO action is required for this gene-drug interaction.\\nThe genetic variation results in increased conversion of clopidogrel to the active metabolite. However, this can result in both positive effects (reduction in the risk of serious\\ncardiovascular and cerebrovascular events) and negative effects (increase in the risk of bleeding).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 22}),\n",
       " Document(page_content='Literature:\\n1. \\nLiu YP et al. Association of genetic variants in CYP2C19 and adverse clinical outcomes after treatment with clopidogrel: an updated meta-analysis. Thromb Res\\n2011;128:593-4.\\n2. \\nLi Y et al. The gain-of-function variant allele CYP2C19*17: a double-edged sword between thrombosis and bleeding in clopidogrel-treated patients. J Thromb Haemost\\n2012;10:199-206.\\n3. \\nSimon T et al. Genetic polymorphisms and the impact of a higher clopidogrel dose regimen on active metabolite exposure and antiplatelet response in healthy subjects.\\nClin Pharmacol Ther 2011;90:287-95.\\n4. \\nShuldiner AR et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA 2009;302:849-57.\\n5. \\nFrére C et al. The CYP2C19*17 allele is associated with better platelet response to clopidogrel in patients admitted for non-ST acute coronary syndrome. J Thromb\\nHaemost 2009;7:1409-11.\\n6. \\nMega JL et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N Engl J Med 2009;360:354-62.\\n7. \\nSimon T et al. Genetic determinants of response to clopidogrel and cardiovascular events. N Engl J Med 2009;360:363-75.\\n8. \\nGeisler T et al. CYP2C19 and nongenetic factors predict poor responsiveness to clopidogrel loading dose after coronary stent implantation. Pharmacogenomics\\n2008;9:1251-9.\\nDate 23-12-2019\\nCYP1A2 IM: clozapine\\n4644\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nAllorge D et al. Identification of a novel splice-site mutation in the CYP1A2 gene. Br J Clin Pharmacol 2003;56:341-4.\\nDate 13-09-2021\\nCYP1A2 NM: clozapine\\n4641\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nAmmar H et al. Clinical and genetic influencing factors on clozapine pharmacokinetics in Tunisian schizophrenic patients. Pharmacogenomics J 2021 Mar 17 (online\\nahead of print).\\n2. \\nOrtega-Vázquez A et al. Alcohol intake potentiates clozapine adverse effects associated to CYP1A2*1C in patients with refractory psychosis. Drug Dev Res 2020 Dec 17\\n(online ahead of print).\\n3. \\nLesche D et al. Impact of CYP1A2, CYP2C19, and CYP2D6 genotype- and phenoconversion-predicted enzyme activity on clozapine exposure and symptom severity.\\nPharmacogenomics J 2020;20:192-201.\\n4. \\nNa Takuathung M et al. Impact of CYP1A2 genetic polymorphisms on pharmacokinetics of antipsychotic drugs: a systematic review and meta-analysis. Acta Psychiatr\\nScand 2019;139:15-25.\\n5. \\nVasudev K et al. Genetic determinants of clozapine-induced metabolic side effects. Can J Psychiatry 2017;62:138-49.\\n6. \\nHuang HC et al. Cigarette smoking has a differential effect on the plasma level of clozapine in Taiwanese schizophrenic patients associated with the CYP1A2 gene\\n-163A/C single nucleotide polymorphism. Psychiatr Genet 2016;26:172-7.\\n7. \\nde Brito RB et al. The CYP1A2 -163C>A polymorphism is associated with super-refractory schizophrenia. Schizophr Res 2015;169:502-3.\\n8. \\nOlsson E et al. Genetic and clinical factors affecting plasma clozapine concentration. Prim Care Companion CNS Disord 2015;17;10.4088/ PCC.14m01704.\\n9. \\nViikki M et al. CYP1A2 polymorphism -1545C > T (rs2470890) is associated with increased side effects to clozapine. BMC Psychiatry 2014;14:50.\\n10. \\nLooman NMG et al. Associatie van genetische variatie in CYP1A2 en UGT1A4 met metabole stoornissen bij gebruikers van clozapine en olanzapine. PW Wetenschappelijk\\nPlatform 2013;7:a1310.\\n11. \\nKohlrausch FB et al. The CYP1A2 -163C>A polymorphism is associated with clozapine-induced generalized tonic-clonic seizures in Brazilian schizophrenia patients.\\nPsychiatry Res 2013;209:242-5.\\n12. \\nRajkumar AP et al. Association between CYP1A2 gene single nucleotide polymorphisms and clinical responses to clozapine in patients with treatment-resistant\\nschizophrenia. Acta Neuropsychiatr 2013;25:2-11.\\n13. \\nFerrari M et al. Association between CYP1A2 polymorphisms and clozapine-induced adverse reactions in patients with schizophrenia. Psychiatry Res 2012;200:1014-7.\\n14. \\nLee ST et al. Association study of 27 annotated genes for clozapine pharmacogenetics: validation of preexisting studies and identification of a new candidate gene,\\nABCB1, for treatment response. J Clin Psychopharmacol 2012;32:441-8.\\n15. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin Psychopharmacol 2009;29:319-26.\\n16. \\nMelkersson KI et al. Impact of CYP1A2 and CYP2D6 polymorphisms on drug metabolism and on insulin and lipid elevations and insulin resistance in clozapine-treated\\npatients. J Clin Psychiatry 2007;68:697-704.\\n17. \\nKootstra-Ros JE et al. The cytochrome P450 CYP1A2 genetic polymorphisms \\n1F and \\n1D do not affect clozapine clearance in a group of schizophrenic patients. Ann Clin\\nBiochem 2005;42:216-9.\\n18. \\nvan der Weide J et al. The effect of smoking and cytochrome P450 CYP1A2 genetic polymorphism on clozapine clearance and dose requirement. Pharmacogenetics\\n2003;13:169-72.\\n19. \\nBasile VS et al. Genetic dissection of atypical antipsychotic-induced weight gain: novel preliminary data on the pharmacogenetic puzzle. J Clin Psychiatry 2001;62:45-66.\\nDate 13-09-2021\\nCYP1A2 PM: clozapine\\n4645\\nThis is NOT a gene-drug interaction.\\nLiterature:\\nDate 13-09-2021\\nCYP1A2*1A/*1F: clozapine\\n4640\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 23}),\n",
       " Document(page_content='Literature:\\n1. \\nAmmar H et al. Clinical and genetic influencing factors on clozapine pharmacokinetics in Tunisian schizophrenic patients. Pharmacogenomics J 2021 Mar 17 (online\\nahead of print).\\n2. \\nOrtega-Vázquez A et al. Alcohol intake potentiates clozapine adverse effects associated to CYP1A2*1C in patients with refractory psychosis. Drug Dev Res 2020 Dec 17\\n(online ahead of print).\\n3. \\nLesche D et al. Impact of CYP1A2, CYP2C19, and CYP2D6 genotype- and phenoconversion-predicted enzyme activity on clozapine exposure and symptom severity.\\nPharmacogenomics J 2020;20:192-201.\\n4. \\nNa Takuathung M et al. Impact of CYP1A2 genetic polymorphisms on pharmacokinetics of antipsychotic drugs: a systematic review and meta-analysis. Acta Psychiatr\\nScand 2019;139:15-25.\\n5. \\nVasudev K et al. Genetic determinants of clozapine-induced metabolic side effects. Can J Psychiatry 2017;62:138-49.\\n6. \\nHuang HC et al. Cigarette smoking has a differential effect on the plasma level of clozapine in Taiwanese schizophrenic patients associated with the CYP1A2 gene\\n-163A/C single nucleotide polymorphism. Psychiatr Genet 2016;26:172-7.\\n7. \\nde Brito RB et al. The CYP1A2 -163C>A polymorphism is associated with super-refractory schizophrenia. Schizophr Res 2015;169:502-3.\\n8. \\nOlsson E et al. Genetic and clinical factors affecting plasma clozapine concentration. Prim Care Companion CNS Disord 2015;17;10.4088/ PCC.14m01704.\\n9. \\nViikki M et al. CYP1A2 polymorphism -1545C > T (rs2470890) is associated with increased side effects to clozapine. BMC Psychiatry 2014;14:50.\\n10. \\nLooman NMG et al. Associatie van genetische variatie in CYP1A2 en UGT1A4 met metabole stoornissen bij gebruikers van clozapine en olanzapine. PW Wetenschappelijk\\nPlatform 2013;7:a1310.\\n11. \\nKohlrausch FB et al. The CYP1A2 -163C>A polymorphism is associated with clozapine-induced generalized tonic-clonic seizures in Brazilian schizophrenia patients.\\nPsychiatry Res 2013;209:242-5.\\n12. \\nRajkumar AP et al. Association between CYP1A2 gene single nucleotide polymorphisms and clinical responses to clozapine in patients with treatment-resistant\\nschizophrenia. Acta Neuropsychiatr 2013;25:2-11.\\n13. \\nFerrari M et al. Association between CYP1A2 polymorphisms and clozapine-induced adverse reactions in patients with schizophrenia. Psychiatry Res 2012;200:1014-7.\\n14. \\nLee ST et al. Association study of 27 annotated genes for clozapine pharmacogenetics: validation of preexisting studies and identification of a new candidate gene,\\nABCB1, for treatment response. J Clin Psychopharmacol 2012;32:441-8.\\n15. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin Psychopharmacol 2009;29:319-26.\\n16. \\nMelkersson KI et al. Impact of CYP1A2 and CYP2D6 polymorphisms on drug metabolism and on insulin and lipid elevations and insulin resistance in clozapine-treated\\npatients. J Clin Psychiatry 2007;68:697-704.\\n17. \\nKootstra-Ros JE et al. The cytochrome P450 CYP1A2 genetic polymorphisms *1F and *1D do not affect clozapine clearance in a group of schizophrenic patients. Ann Clin\\nBiochem 2005;42:216-9.\\n18. \\nvan der Weide J et al. The effect of smoking and cytochrome P450 CYP1A2 genetic polymorphism on clozapine clearance and dose requirement. Pharmacogenetics\\n2003;13:169-72.\\n19. \\nBasile VS et al. Genetic dissection of atypical antipsychotic-induced weight gain: novel preliminary data on the pharmacogenetic puzzle. J Clin Psychiatry 2001;62:45-66.\\nDate 13-09-2021\\nCYP1A2*1C-heterozygoot: clozapine\\n4642\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nAmmar H et al. Clinical and genetic influencing factors on clozapine pharmacokinetics in Tunisian schizophrenic patients. Pharmacogenomics J 2021 Mar 17 (online\\nahead of print).\\n2. \\nOrtega-Vázquez A et al. Alcohol intake potentiates clozapine adverse effects associated to CYP1A2*1C in patients with refractory psychosis. Drug Dev Res 2020 Dec 17\\n(online ahead of print).\\n3. \\nVasudev K et al. Genetic determinants of clozapine-induced metabolic side effects. Can J Psychiatry 2017;62:138-49.\\n4. \\nHuang HC et al. Cigarette smoking has a differential effect on the plasma level of clozapine in Taiwanese schizophrenic patients associated with the CYP1A2 gene\\n-163A/C single nucleotide polymorphism. Psychiatr Genet 2016;26:172-7.\\n5. \\nKohlrausch FB et al. The CYP1A2 -163C>A polymorphism is associated with clozapine-induced generalized tonic-clonic seizures in Brazilian schizophrenia patients.\\nPsychiatry Res 2013;209:242-5.\\n6. \\nRajkumar AP et al. Association between CYP1A2 gene single nucleotide polymorphisms and clinical responses to clozapine in patients with treatment-resistant\\nschizophrenia. Acta Neuropsychiatr 2013;25:2-11.\\n7. \\nFerrari M et al. Association between CYP1A2 polymorphisms and clozapine-induced adverse reactions in patients with schizophrenia. Psychiatry Res 2012;200:1014-7.\\n8. \\nBolla E et al. Are CYP1A2*1F and *1C associated with clozapine tolerability?: a preliminary investigation. Psychiatry Res 2011;189:483.\\n9. \\nMelkersson KI et al. Impact of CYP1A2 and CYP2D6 polymorphisms on drug metabolism and on insulin and lipid elevations and insulin resistance in clozapine-treated\\npatients. J Clin Psychiatry 2007;68:697-704.\\n10. \\nKootstra-Ros JE et al. The cytochrome P450 CYP1A2 genetic polymorphisms *1F and *1D do not affect clozapine clearance in a group of schizophrenic patients. Ann Clin\\nBiochem 2005;42:216-9.\\nDate 13-09-2021\\nCYP1A2*1C/*1C: clozapine\\n4643\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nOrtega-Vázquez A et al. Alcohol intake potentiates clozapine adverse effects associated to CYP1A2*1C in patients with refractory psychosis. Drug Dev Res 2020 Dec 17\\n(online ahead of print).\\n2. \\nKohlrausch FB et al. The CYP1A2 -163C>A polymorphism is associated with clozapine-induced generalized tonic-clonic seizures in Brazilian schizophrenia patients.\\nPsychiatry Res 2013;209:242-5.\\n3. \\nRajkumar AP et al. Association between CYP1A2 gene single nucleotide polymorphisms and clinical responses to clozapine in patients with treatment-resistant\\nschizophrenia. Acta Neuropsychiatr 2013;25:2-11.\\n4. \\nFerrari M et al. Association between CYP1A2 polymorphisms and clozapine-induced adverse reactions in patients with schizophrenia. Psychiatry Res 2012;200:1014-7.\\n5. \\nBolla E et al. Are CYP1A2*1F and *1C associated with clozapine tolerability?: a preliminary investigation. Psychiatry Res 2011;189:483.\\nDate 13-09-2021\\nCYP2D6 IM: clozapine\\n1530\\nNO action is required for this gene-drug interaction.\\nThe genetic variation results in a slightly elevated plasma concentration of clozapine, but there are no clinical consequences.\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 24}),\n",
       " Document(page_content=\"1. \\nOrtega-Vázquez A et al. Alcohol intake potentiates clozapine adverse effects associated to CYP1A2*1C in patients with refractory psychosis. Drug Dev Res 2020 Dec 17\\n(online ahead of print).\\n2. \\nLesche D et al. Impact of CYP1A2, CYP2C19, and CYP2D6 genotype- and phenoconversion-predicted enzyme activity on clozapine exposure and symptom severity.\\nPharmacogenomics J 2020;20:192-201.\\n3. \\nTóth K et al. Potential role of patients' CYP3A-status in clozapine pharmacokinetics. Int J Neuropsychopharmacol 2017;20:529-37.\\n4. \\nAkamine Y et al. Quantification of the steady-state plasma concentrations of clozapine and N-desmethylclozapine in Japanese patients with schizophrenia using a novel\\nHPLC method and the effects of CYPs and ABC transporters polymorphisms. Ann Clin Biochem 2017;54:677-85.\\n5. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and chlorpromazine in the Chinese Han population.\\nPharmacogenomics J 2016;16:357-65.\\n6. \\nLee ST et al. Association study of 27 annotated genes for clozapine pharmacogenetic: validation of preexisting studies and identification of a new candidate gene, ABCB1,\\nfor treatment response. J Clin Psychopharmacol 2012; 32:441-8.\\n7. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin Psychopharmacol 2009;29:318-26.\\n8. \\nMelkersson KI et al. Impact of CYP1A2 and CYP2D6 polymorphisms on drug metabolism and on insulin and lipid elevations and insulin resistance in clozapine-treated\\npatients. J Clin Psychiatry 2007;68:697-704.\\n9. \\nDettling M et al. Long-term therapeutic drug monitoring of clozapine and metabolites in psychiatric in- and outpatients. Psychopharmacology 2000;152:80-86.\\nDate 13-09-2021\\nCYP2D6 PM: clozapine\\n1529\\nNO action is required for this gene-drug interaction.\\nThe genetic variation results in a slightly elevated plasma concentration of clozapine, but there are no clinical consequences.\\nLiterature:\\n1. \\nOrtega-Vázquez A et al. Alcohol intake potentiates clozapine adverse effects associated to CYP1A2*1C in patients with refractory psychosis. Drug Dev Res 2020 Dec 17\\n(online ahead of print).\\n2. \\nLesche D et al. Impact of CYP1A2, CYP2C19, and CYP2D6 genotype- and phenoconversion-predicted enzyme activity on clozapine exposure and symptom severity.\\nPharmacogenomics J 2020;20:192-201.\\n3. \\nTóth K et al. Potential role of patients' CYP3A-status in clozapine pharmacokinetics. Int J Neuropsychopharmacol 2017;20:529-37.\\n4. \\nAkamine Y et al. Quantification of the steady-state plasma concentrations of clozapine and N-desmethylclozapine in Japanese patients with schizophrenia using a novel\\nHPLC method and the effects of CYPs and ABC transporters polymorphisms. Ann Clin Biochem 2017;54:677-85.\\n5. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and chlorpromazine in the Chinese Han population.\\nPharmacogenomics J 2016;16:357-65.\\n6. \\nLee ST et al. Association study of 27 annotated genes for clozapine pharmacogenetic: validation of preexisting studies and identification of a new candidate gene, ABCB1,\\nfor treatment response. J Clin Psychopharmacol 2012; 32:441-8.\\n7. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin Psychopharmacol 2009;29:318-26.\\n8. \\nMelkersson KI et al. Impact of CYP1A2 and CYP2D6 polymorphisms on drug metabolism and on insulin and lipid elevations and insulin resistance in clozapine-treated\\npatients. J Clin Psychiatry 2007;68:697-704.\\n9. \\nDettling M et al. Long-term therapeutic drug monitoring of clozapine and metabolites in psychiatric in- and outpatients. Psychopharmacology 2000;152:80-86.\\n10. \\nDettling M et al. Clozapine-induced agranulocytosis and hereditary polymorphisms of clozapine metabolizing enzymes: no association with myeloperoxidase and\\ncytochrome P4502D6. Pharmacopsychiatry 2000;33:218-20.\\n11. \\nArranz MJ et al. Cytochrome P4502D6 genotype does not determine response to clozapine. Br J Clin Pharmacol 1995;39:417-20.\\n12. \\nDahl ML et al. Disposition of clozapine in man: lack of association with debrisoquine and S-mephenytoin hydroxylation polymorphisms. Br J Clin Pharmacol 1994;37:71-4.\\n13. \\nSmPC Clozaril (VS).\\nDate 13-09-2021\\nCYP2D6 UM: clozapine\\n1531\\nNO action is required for this gene-drug interaction.\\nThe genetic variation has a small effect on the plasma concentration of clozapine, but there are no clinical consequences.\\nLiterature:\\n1. \\nOrtega-Vázquez A et al. Alcohol intake potentiates clozapine adverse effects associated to CYP1A2*1C in patients with refractory psychosis. Drug Dev Res 2020 Dec 17\\n(online ahead of print).\\n2. \\nLesche D et al. Impact of CYP1A2, CYP2C19, and CYP2D6 genotype- and phenoconversion-predicted enzyme activity on clozapine exposure and symptom severity.\\nPharmacogenomics J 2020;20:192-201.\\n3. \\nTóth K et al. Potential role of patients' CYP3A-status in clozapine pharmacokinetics. Int J Neuropsychopharmacol 2017;20:529-37.\\n4. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin Psychopharmacol 2009;29:318-26.\\n5. \\nDettling M et al. Long-term therapeutic drug monitoring of clozapine and metabolites in psychiatric in- and outpatients. Psychopharmacology 2000;152:80-86.\\n6. \\nDettling M et al. Clozapine-induced agranulocytosis and hereditary polymorphisms of clozapine metabolizing enzymes: no association with myeloperoxidase and\\ncytochrome P4502D6. Pharmacopsychiatry 2000;33:218-20.\\nDate 13-09-2021\\nCYP2D6 IM: codeine\\n1584\\nThe genetic variation reduces the conversion of codeine to morphine. This can result in reduced analgesia.\\nRecommendation:\\nFor COUGH:\\n1. \\nno action required\\nFor PAIN:\\nIt is not possible to offer adequately substantiated advice for dose adjustment based on the limited available literature for this phenotype.\\n1. \\nbe alert to a reduced effectiveness\\n2. \\nin the case of inadequate effectiveness:\\n1. \\ntry a dose increase\\n2. \\nif this does not work: choose an alternative\\nDo not select tramadol, as this is also metabolised by CYP2D6\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n3. \\nif no alternative is selected: advise the patient to report inadequate analgesia\\nLiterature:\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 25}),\n",
       " Document(page_content='1. \\nBaber M et al. The pharmacogenetics of codeine pain relief in the postpartum period. Pharmacogenomics J 2015;15:430-5.\\n2. \\nProws CA et al. Codeine-related adverse drug reactions in children following tonsillectomy: a prospective study. Laryngoscope 2014;124:1242-50.\\n3. \\nKelly LE et al. A clinical tool for reducing central nervous system depression among neonates exposed to codeine through breast milk. PLoS One 2013;8:e70073.\\n4. \\nVanderVaart S et al. CYP2D6 polymorphisms and codeine analgesia in postpartum pain management: a pilot study. Ther Drug Monit 2011;33:425-32.\\n5. \\nWilliams DG et al. Pharmacogenetics of codeine metabolism in an urban population of children and its implications for analgesic reliability. Br J Anaesth 2002;89:839-45.\\n6. \\nTseng CY et al. Formation of morphine from codeine in Chinese subjects of different CYP2D6 genotypes. Clin Pharmacol Ther 1996;60:177-82.\\n7. \\nSPC Codeïnefosfaat Ratiopharm.\\nDate 20-11-2017\\nCYP2D6 PM: codeine\\n1583\\nThe genetic variation reduces the conversion of codeine to morphine. This can result in reduced analgesia.\\nRecommendation:\\nFor COUGH:\\n1. \\nno action required\\nFor PAIN:\\nIt is not possible to offer adequately substantiated advice for dose adjustment based on the limited available literature for this phenotype.\\n1. \\nchoose an alternative\\nDo not select tramadol, as this is also metabolised by CYP2D6\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n2. \\nif an alternative is not an option: advise the patient to report inadequate analgesia.\\nLiterature:\\n1. \\nBaber M et al. The pharmacogenetics of codeine pain relief in the postpartum period. Pharmacogenomics J 2015;15:430-5.\\n2. \\nProws CA et al. Codeine-related adverse drug reactions in children following tonsillectomy: a prospective study. Laryngoscope 2014;124:1242-50.\\n3. \\nKelly LE et al. A clinical tool for reducing central nervous system depression among neonates exposed to codeine through breast milk. PLoS One 2013;8:e70073.\\n4. \\nSistonen J et al. Prediction of codeine toxicity in infants and their mothers using a novel combination of maternal genetic markers. Clin Pharmacol Ther 2012;91:692-9.\\n5. \\nVanderVaart S et al. CYP2D6 polymorphisms and codeine analgesia in postpartum pain management: a pilot study. Ther Drug Monit 2011;33:425-32.\\n6. \\nKirchheiner J et al. Pharmacokinetics of codeine and its metabolite morphine in ultra-rapid metabolizers due to CYP2D6 duplication. Pharmacogenomics J 2007;7:257-65.\\n7. \\nWilliams DG et al. Pharmacogenetics of codeine metabolism in an urban population of children and its implications for analgesic reliability. Br J Anaesth 2002;89:839-45.\\n8. \\nPoulsen L et al. Codeine in post-operative pain. Study of the influence of sparteine phenotype and serum concentrations of morphine and morphine-6-glucuronide. Eur J\\nClin Pharmacol 1998;54:451-4.\\n9. \\nEckhardt K et al. Same incidence of adverse drug events after codeine administration irrespective of the genetically determined differences in morphine formation. Pain\\n1998;76:27-33.\\n10. \\nMikus G et al. Effect of codeine on gastrointestinal motility in relation to CYP2D6 phenotype. Clin Pharmacol Ther 1997;61:459-66.\\n11. \\nHasselstrom J et al. The effect of codeine on gastrointestinal transit in extensive and poor metabolisers of debrisoquine. Eur J Clin Pharmacol 1997;53:145-8.\\n12. \\nPoulsen L et al. Codeine and morphine in extensive and poor metabolizers of sparteine: pharmacokinetics, analgesic effect and side effects. Eur J Clin Pharmacol\\n1996;51:289-95.\\n13. \\nPersson K et al. Patient-controlled analgesia (PCA) with codeine for postoperative pain relief in ten extensive metabolisers and one poor metaboliser of\\ndextromethorphan. Br J Clin Pharmacol 1995;39:182-6.\\n14. \\nYue QY et al. Pharmacokinetics of codeine and its metabolites in Caucasian healthy volunteers: comparisons between extensive and poor hydroxylators of debrisoquine.\\nBr J Clin Pharmacol 1991;31:635-42.\\n15. \\nDesmeules J et al. Impact of environmental and genetic factors on codeine analgesia. Eur J Clin Pharmacol 1991;41:23-6.\\n16. \\nSindrup SH et al. Codeine increases pain thresholds to copper vapor laser stimuli in extensive but not poor metabolizers of sparteine. Clin Pharmacol Ther 1990;48:686-\\n93.\\n17. \\nSPC Codeïnefosfaat Ratiopharm.\\nDate 20-11-2017\\nCYP2D6 UM: codeine\\n1585\\nThe genetic variation increases the conversion of codeine to morphine. This can result in an increase in side effects. Death has occurred in children who received analgesic\\ndoses. One adult with reduced kidney function and co-medication with two CYP3A4 inhibitors became comatose after use of codeine for a cough.\\nRecommendation:\\nDOSES HIGHER THAN 20 mg every 6 hours for adults and 10 mg every 6 hours for children aged 12 years or older AND/OR ADDITIONAL RISK FACTORS, such as co-\\nmedication with CYP3A4 inhibitors and/or reduced kidney function:\\nCodeine is contra-indicated\\nif possible, select an alternative\\nFor PAIN: do not select tramadol, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6. Oxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in side effects in\\npatients.\\nFor COUGH: noscapine is not metabolised by CYP2D6.\\nDOSES LOWER THAN OR EQUAL TO 20 mg every 6 hours for adults and 10 mg every 6 hours for children aged 12 years or older AND NO ADDITIONAL RISK FACTORS,\\nsuch as co-medication with CYP3A4 inhibitors and/or reduced kidney function:\\nno action required\\nLiterature:\\n1. \\nBaber M et al. The pharmacogenetics of codeine pain relief in the postpartum period. Pharmacogenomics J 2015;15:430-5.\\n2. \\nRay JG et al. Risk of overdose and death following codeine prescription among immigrants. J Epidemiol Community Health 2014;68:1057-63.\\n3. \\nProws CA et al. Codeine-related adverse drug reactions in children following tonsillectomy: a prospective study. Laryngoscope 2014;124:1242-50.\\n4. \\nKelly LE et al. A clinical tool for reducing central nervous system depression among neonates exposed to codeine through breast milk. PLoS One 2013;8:e70073.\\n5. \\nKelly LE et al. More codeine fatalities after tonsillectomy in North American children. Pediatrics 2012;129:e1343-7.\\n6. \\nSistonen J et al. Prediction of codeine toxicity in infants and their mothers using a novel combination of maternal genetic markers. Clin Pharmacol Ther 2012;91:692-9.\\n7. \\nVanderVaart S et al. CYP2D6 polymorphisms and codeine analgesia in postpartum pain management: a pilot study. Ther Drug Monit 2011;33:425-32.\\n8. \\nCiszkowski C et al. Codeine, ultrarapid-metabolism genotype, and postoperative death. N Engl J Med 2009;361:827-8.\\n9. \\nMadadi P et al. Pharmacogenetics of neonatal opioid toxicity following maternal use of codeine during breastfeeding: a case-control study. Clin Pharmacol Ther\\n2009;85:31-5.\\n10. \\nKoren G et al. Pharmacogenetics of morphine poisoning in a breastfed neonate of a codeine-prescribed mother. Lancet 2006;368:704.\\n11. \\nKirchheiner J et al. Pharmacokinetics of codeine and its metabolite morphine in ultra-rapid metabolizers due to CYP2D6 duplication. Pharmacogenomics J 2007;7:257-65.\\n12. \\nGasche Y et al. Codeine intoxication associated with ultrarapid CYP2D6 metabolism. N Engl J Med 2004;351:2827-31.\\n13. \\nDalen P et al. Quick onset of severe abdominal pain after codeine in an ultrarapid metabolizer of debrisoquine. Ther Drug Monit 1997;19:543-4.\\n14. \\nEuropean Medicines Agency. Restrictions on use of codeine for pain relief in children – CMDh endorses PRAC recommendation. 28-06-13.\\n15. \\nSPC Codeïnefosfaat Ratiopharm.\\n16. \\nSPC Codeine Sulfate Tablets (VS).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 26}),\n",
       " Document(page_content='Date 20-11-2017\\nCYP2D6 IM: disopyramide\\n2537\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2D6 PM: disopyramide\\n2536\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2D6 UM: disopyramide\\n2538\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2C19 IM: doxepine\\n7021\\nNO action is required for this gene-drug interaction.\\nThe gene variation has an effect on the exposure to doxepin, but not on the exposure to doxepin + the active metabolite nordoxepin, which determines the effect and side\\neffects.\\nLiterature:\\n1. \\nKirchheiner J et al. Contributions of CYP2D6, CYP2C9 and CYP2C19 to the biotransformation of E- and Z-doxepin in healthy volunteers. Pharmacogenetics 2002;12:571-\\n80.\\nDate 04-03-2019\\nCYP2C19 PM: doxepine\\n7022\\nNO action is required for this gene-drug interaction.\\nThe gene variation has an effect on the exposure to doxepin, but not on the exposure to doxepin + the active metabolite nordoxepin, which determines the effect and side\\neffects.\\nLiterature:\\n1. \\nKirchheiner J et al. Contributions of CYP2D6, CYP2C9 and CYP2C19 to the biotransformation of E- and Z-doxepin in healthy volunteers. Pharmacogenetics 2002;12:571-\\n80.\\n2. \\nSmPC Silenor (VS).\\nDate 04-03-2019\\nCYP2C19 UM: doxepine\\n7023\\nNO action is required for this gene-drug interaction.\\nThe gene variations have an effect on the exposure to doxepin, but not on the exposure to doxepin + the active metabolite nordoxepin, which determines the effect and side\\neffects.\\nLiterature:\\n1. \\nKirchheiner J et al. Contributions of CYP2D6, CYP2C9 and CYP2C19 to the biotransformation of E- and Z-doxepin in healthy volunteers. Pharmacogenetics 2002;12:571-\\n80.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 27}),\n",
       " Document(page_content='Date 04-03-2019\\nCYP2D6 IM: doxepine\\n2015\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of doxepin and the active metabolite nordoxepin.\\nuse 80% of the standard dose and monitor the effect and side effects or the plasma concentrations of doxepin and nordoxepin in order to set the maintenance dose\\nThe therapeutic range is 100-250 ng/mL for the sum of doxepin and nordoxepin plasma concentrations. Values higher than 400 ng/mL are considered toxic.\\nLiterature:\\n1. \\nKirchheiner J et al. Contributions of CYP2D6, CYP2C9 and CYP2C19 to the biotransformation of E- and Z-doxepin in healthy volunteers. Pharmacogenetics 2002;12:571-\\n80.\\nDate 19-11-2018\\nCYP2D6 PM: doxepine\\n2016\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of doxepin and the active metabolite nordoxepin.\\nuse 40% of the standard dose and monitor the effect and side effects or the plasma concentrations of doxepin and nordoxepin in order to set the maintenance dose\\nThe therapeutic range is 100-250 ng/mL for the sum of doxepin and nordoxepin plasma concentrations. Values higher than 400 ng/mL are considered toxic.\\nLiterature:\\n1. \\nKoski A et al. A fatal doxepin poisoning associated with a defective CYP2D6 genotype. Am J Forensic Med Pathol 2007;28:259-61.\\n2. \\nKirchheiner J et al. Impact of the CYP2D6 ultra-rapid metabolizer genotype on doxepin pharmacokinetics and serotonin in platelets. Pharmacogenet Genomics\\n2005;15:579-87.\\n3. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and clinical effects of antidepressants in a\\nnaturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n4. \\nKirchheiner J et al. Contributions of CYP2D6, CYP2C9 and CYP2C19 to the biotransformation of E- and Z-doxepin in healthy volunteers. Pharmacogenetics 2002;12:571-\\n80.\\n5. \\nTacke U et al. Debrisoquine hydroxylation phenotypes of patients with high versus low to normal serum antidepressant concentrations. J Clin Psychopharmacol\\n1992;12:262-7.\\n6. \\nSmPC Silenor (VS).\\nDate 19-11-2018\\nCYP2D6 UM: doxepine\\n2017\\nThe risk of ineffectiveness and cardiotoxic side effects may be increased. The gene variation leads to reduced plasma concentrations of doxepin and the active metabolite\\nnordoxepin and an increase in the plasma concentrations of the potentially cardiotoxic hydroxy metabolites.\\ndouble the standard dose and monitor the effect and side effects or the plasma concentrations of doxepin and nordoxepin in order to set the maintenance dose\\nThe therapeutic range is 100-250 ng/mL for the sum of doxepin and nordoxepin plasma concentrations. Values higher than 400 ng/mL are considered toxic.\\nif a dose increase is not wanted due to the potentially cardiotoxic hydroxy metabolites: avoid doxepin.\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include citalopram and sertraline.\\nLiterature:\\n1. \\nKirchheiner J et al. Impact of the CYP2D6 ultra-rapid metabolizer genotype on doxepin pharmacokinetics and serotonin in platelets. Pharmacogenet Genomics\\n2005;15:579-87.\\nDate 19-11-2018\\nCYP2D6 IM: duloxetine\\n1673\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nZastrozhin et al. Impact of polymorphism of CYP2D6 on equilibrium concentration of duloxetine in patients suffering from major depressive disorder. Psychopharmacol\\nBull 2020;50:47-57.\\n2. \\nKamei S et al. Rapid onset of syndrome of inappropriate antidiuretic hormone secretion induced by duloxetine in an elderly type 2 diabetic patient with painful diabetic\\nneuropathy. J Diabetes Investig 2015;6:343-5.\\n3. \\nLobo ED et al. Pharmacokinetics of orally administered duloxetine in children and adolescents with major depressive disorder. Clin Pharmacokinet 2014;53:731-40.\\n4. \\nBeatty NC et al. Pharmacogenetic workup of perioperative serotonin syndrome. J Clin Anesth 2013;25:662-5.\\n5. \\nTianmei S et al. Pharmacokinetics and tolerability of duloxetine following oral administration to healthy Chinese subjects. Clin Pharmacokinet 2007;46:767-75.\\nDate 14-11-2022\\nCYP2D6 PM: duloxetine\\n1674\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nKuzin M et al. The role of the poor metabolizer genotype CYP2D6 and CYP1A2 phenotype in the pharmacokinetics of duloxetine and venlafaxine-a case report. Basic Clin', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 28}),\n",
       " Document(page_content=\"Pharmacol Toxicol 2020;127:354-7.\\n2. \\nAhmed AT et al. Pharmacokinetic-pharmacodynamic interaction associated with venlafaxine-XR remission in patients with major depressive disorder with history of\\ncitalopram / escitalopram treatment failure. J Affect Disord 2019;246:62-8.\\n3. \\nLobo ED et al. Pharmacokinetics of orally administered duloxetine in children and adolescents with major depressive disorder. Clin Pharmacokinet 2014;53:731-40.\\n4. \\nChan C et al. Duloxetine pharmacokinetics are similar in Japanese and Caucasian subjects. Br J Clin Pharmacol 2007;63:310-4.\\n5. \\nSmPC Cymbalta.\\nDate 14-11-2022\\nCYP2D6 UM: duloxetine\\n1675\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nAhmed AT et al. Pharmacokinetic-pharmacodynamic interaction associated with venlafaxine-XR remission in patients with major depressive disorder with history of\\ncitalopram / escitalopram treatment failure. J Affect Disord 2019;246:62-8.\\n2. \\nLobo ED et al. Pharmacokinetics of orally administered duloxetine in children and adolescents with major depressive disorder. Clin Pharmacokinet 2014;53:731-40.\\nDate 14-11-2022\\nCYP2B6 IM: efavirenz\\n4754\\nGenetic variations increase the efavirenz plasma concentration and therefore the risk of side effects. However, the efavirenz plasma concentration remains within the\\ntherapeutic range for the majority of patients.\\nRecommendation:\\n1. \\nDetermine the efavirenz plasma concentration if side effects occur and reduce the dose if needed.\\nIn 14 IM adults, a dose reduction to 400 mg/day (2/3rd of the standard dose) was sufficient to achieve therapeutic plasma concentrations and to reduce or resolve side\\neffects.\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nLiterature:\\n1. \\nVujkovic M et al. CYP2B6 516G>T minor allele protective of late virologic failure in efavirenz-treated HIV-infected patients in Botswana. J Acquir Immune Defic Syndr\\n2017 May 5 [Epub ahead of print].\\n2. \\nLeger P et al. Pharmacogenetics of efavirenz discontinuation for reported central nervous system symptoms appears to differ by race. Pharmacogenet Genomics\\n2016;26:473-80.\\n3. \\nCusato J et al. Efavirenz pharmacogenetics in a cohort of Italian patients. Int J Antimicrob Agents 2016;47:117-23.\\n4. \\nDickinson L et al. Comprehensive pharmacokinetic, pharmacodynamic and pharmacogenetic evaluation of once-daily efavirenz 400 and 600 mg in treatment-naïve HIV-\\ninfected patients at 96 weeks: results of the ENCORE1 study. Clin Pharmacokinet 2016;55:861-73.\\n5. \\nSwart M et al. An expanded analysis of pharmacogenetics determinants of efavirenz response that includes 3'-UTR single nucleotide polymorphisms among black South\\nAfrican HIV/AIDS patients. Front Genet 2016;6:356.\\n6. \\nMeng X et al. Effect of CYP2B6 gene polymorphisms on efavirenz plasma concentrations in Chinese patients with HIV infection. PLoS One 2015;10:e0130583.\\n7. \\nHaas DW et al. Functional CYP2B6 variants and virologic response to an efavirenz-containing regimen in Port-au-Prince, Haiti. J Antimicrob Chemother 2014;69:2187-90.\\n8. \\nMartín AS et al. Dose reduction of efavirenz: an observational study describing cost-effectiveness, pharmacokinetics and pharmacogenetics. Pharmacogenomics\\n2014;15:997-1006.\\n9. \\nBertrand J et al. Dependence of efavirenz- and rifampicin-isoniazid-based antituberculosis treatment drug-drug interaction on CYP2B6 and NAT2 genetic polymorphisms:\\nANRS 12154 study in Cambodia. J Infect Dis 2014;209:399-408.\\n10. \\nSarfo FS et al. Pharmacogenetic associations with plasma efavirenz concentrations and clinical correlates in a retrospective cohort of Ghanaian HIV-infected patients. J\\nAntimicrob Chemother 2014;69:491-9. PubMed PMID: 24080498.\\n11. \\nNgaimisi E et al. Importance of ethnicity, CYP2B6 and ABCB1 genotype for efavirenz pharmacokinetics and treatment outcomes: a parallel-group prospective cohort\\nstudy in two sub-Saharan Africa populations. PLoS One 2013;8:e67946. PubMed PMID: 23861838.\\n12. \\nYimer G et al. High plasma efavirenz level and CYP2B6*6 are associated with efavirenz-based HAART-induced liver injury in the treatment of naïve HIV patients from\\nEthiopia: a prospective cohort study. Pharmacogenomics J 2012;12:499-506. PubMed PMID: 21862974.\\n13. \\nMugusi S et al. Liver enzyme abnormalities and associated risk factors in HIV patients on efavirenz-based HAART with or without tuberculosis co-infection in Tanzania.\\nPLoS One 2012;7:e40180. PubMed PMID: 22808112.\\n14. \\nWyen C et al. Cytochrome P450 2B6 (CYP2B6) and constitutive androstane receptor (CAR) polymorphisms are associated with early discontinuation of efavirenz-\\ncontaining regimens. J Antimicrob Chemother 2011;66:2092-8. PubMed PMID: 21715435.\\n15. \\nCarr DF et al. Haplotype structure of CYP2B6 and association with plasma efavirenz concentrations in a Chilean HIV cohort. J Antimicrob Chemother 2010;65:1889-93.\\nPubMed PMID: 20639527.\\n16. \\nGatanaga H et al. Successful efavirenz dose reduction in HIV type 1-infected individuals with cytochrome P450 2B6 \\n6 and \\n26. Clin Infect Dis 2007;45:1230-7. PubMed\\nPMID: 17918089.\\n17. \\nHaas DW et al. Pharmacogenetics of long-term responses to antiretroviral regimens containing efavirenz and/or nelfinavir: an Adult Aids Clinical Trials Group study. J\\nInfect Dis 2005;192:1931-42. PubMed PMID: 16267764.\\nDate 05-03-2018\\nCYP2B6 PM: efavirenz\\n4755\\nGenetic variations increase the risk of side effects. The standard dose leads to an efavirenz concentration in the toxic range in the majority of patients with this genotype.\\nRecommendation:\\nEfavirenz in MONOpreparation, adults and children FROM 40 KG:\\nBody mass index LESS THAN or EQUAL to 25:\\n1. \\nThe recommended initial dose is 400 mg/day and this dose should be titrated to plasma concentration if needed (further reduction to 200 mg/day or in rare\\ncases an increase to 600 mg/day).\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nBody mass index GREATER than 25:\\n1. \\nThe recommended initial dose is 600 mg/day and this dose should be titrated to plasma concentration if needed (reduction to 400 or 200 mg/day).\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nEfavirenz in MONOpreparation, children LIGHTER THAN 40 KG:\\n1. \\nStart with the standard dose and titrate this dose to plasma concentration if needed. In adults, therapeutic plasma concentrations were achieved at either 2/3rd of\\nthe standard dose (1/3rd of the patients) or 1/3rd of the standard dose (2/3rd of the patients). In children younger than 3 years, therapeutic plasma concentrations\\nwere achieved at doses of approximately 10 mg/kg per day (as capsules) (100 mg/day for 7-14 kg and 150 mg/day for 14-17 kg; 50-75% of the standard dose).\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nEfavirenz in COMBINATION preparation:\\n1. \\nInitiate the combination preparation and titrate the efavirenz dose to plasma concentration if needed (reduction to 400 or 200 mg/day)\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nNote: the dosing recommendations above are based on PM patients with the *6/*6 genotype. There is evidence that the *18/*18 genotype in PM patients (only present in\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 29}),\n",
       " Document(page_content=\"negroid patients) may require greater dose reductions.\\nConsiderations:\\nDetailed justification for the recommendation is contained in the risk analysis. The considerations used for adults are also given below.\\nThe median or mean plasma concentrations or AUC in PM patients are above the therapeutic range, except in 3 studies with low efavirenz plasma concentrations in EM\\npatients (2 of the 3 studies performed in Africa and 1 study in the United States and Italy). A recent study showed a similar virological response for efavirenz 400 and 600\\nmg/day in patients not selected on genotype. The risk of underdose is therefore very small if the initial dose is reduced to 400 mg/day. Two small studies showed that dose\\nreductions did not reduce the efficacy (HIV remained undetectable), but side effects did reduce in 24 PM patients.\\nCompliance improves with administration of a combination preparation and the absence of unnecessary side effects due to excessive plasma concentrations.\\nConsideration to CYP2B6 inducers such as rifampicin is not needed in PM patients. The significantly low or absent metabolic capacity of CYP2B6 makes induction of little to no\\nrelevance. Moreover, the effects of enzyme induction by rifampicin and enzyme inhibition by isoniazid on efavirenz plasma concentrations seem to largely cancel each other\\nout, independent of the CYP2B6 phenotype of the patient.\\nLiterature:\\n1. \\nVujkovic M et al. CYP2B6 516G>T minor allele protective of late virologic failure in efavirenz-treated HIV-infected patients in Botswana. J Acquir Immune Defic Syndr\\n2017 May 5 [Epub ahead of print].\\n2. \\nBolton Moore C et al. CYP2B6 genotype-directed dosing is required for optimal efavirenz exposure in children 3-36 months with HIV infection. AIDS 2017;31:1129-1136.\\n3. \\nLeger P et al. Pharmacogenetics of efavirenz discontinuation for reported central nervous system symptoms appears to differ by race. Pharmacogenet Genomics\\n2016;26:473-80.\\n4. \\nCusato J et al. Efavirenz pharmacogenetics in a cohort of Italian patients. Int J Antimicrob Agents 2016;47:117-23.\\n5. \\nDickinson L et al. Comprehensive pharmacokinetic, pharmacodynamic and pharmacogenetic evaluation of once-daily efavirenz 400 and 600 mg in treatment-naïve HIV-\\ninfected patients at 96 weeks: results of the ENCORE1 study. Clin Pharmacokinet 2016;55:861-73.\\n6. \\nSwart M et al. An expanded analysis of pharmacogenetics determinants of efavirenz response that includes 3'-UTR single nucleotide polymorphisms among black South\\nAfrican HIV/AIDS patients. Front Genet 2016;6:356.\\n7. \\nMeng X et al. Effect of CYP2B6 gene polymorphisms on efavirenz plasma concentrations in Chinese patients with HIV infection. PLoS One 2015;10:e0130583.\\n8. \\nHaas DW et al. Functional CYP2B6 variants and virologic response to an efavirenz-containing regimen in Port-au-Prince, Haiti. J Antimicrob Chemother 2014;69:2187-90.\\n9. \\nMartín AS et al. Dose reduction of efavirenz: an observational study describing cost-effectiveness, pharmacokinetics and pharmacogenetics. Pharmacogenomics\\n2014;15:997-1006.\\n10. \\nBertrand J et al. Dependence of efavirenz- and rifampicin-isoniazid-based antituberculosis treatment drug-drug interaction on CYP2B6 and NAT2 genetic polymorphisms:\\nANRS 12154 study in Cambodia. J Infect Dis 2014;209:399-408.\\n11. \\nSarfo FS et al. Pharmacogenetic associations with plasma efavirenz concentrations and clinical correlates in a retrospective cohort of Ghanaian HIV-infected patients. J\\nAntimicrob Chemother 2014;69:491-9. PubMed PMID: 24080498.\\n12. \\nNgaimisi E et al. Importance of ethnicity, CYP2B6 and ABCB1 genotype for efavirenz pharmacokinetics and treatment outcomes: a parallel-group prospective cohort\\nstudy in two sub-Saharan Africa populations. PLoS One 2013;8:e67946. PubMed PMID: 23861838.\\n13. \\nYimer G et al. High plasma efavirenz level and CYP2B6*6 are associated with efavirenz-based HAART-induced liver injury in the treatment of naïve HIV patients from\\nEthiopia: a prospective cohort study. Pharmacogenomics J 2012;12:499-506. PubMed PMID: 21862974.\\n14. \\nMugusi S et al. Liver enzyme abnormalities and associated risk factors in HIV patients on efavirenz-based HAART with or without tuberculosis co-infection in Tanzania.\\nPLoS One 2012;7:e40180. PubMed PMID: 22808112.\\n15. \\nWyen C et al. Cytochrome P450 2B6 (CYP2B6) and constitutive androstane receptor (CAR) polymorphisms are associated with early discontinuation of efavirenz-\\ncontaining regimens. J Antimicrob Chemother 2011;66:2092-8. PubMed PMID: 21715435.\\n16. \\nRibaudo HJ et al. Effect of CYP2B6, ABCB1, and CYP3A5 polymorphisms on efavirenz pharmacokinetics and treatment response: an AIDS Clinical Trials Group study. J\\nInfect Dis 2010;202:717-22. PubMed PMID: 20662624.\\n17. \\nCarr DF et al. Haplotype structure of CYP2B6 and association with plasma efavirenz concentrations in a Chilean HIV cohort. J Antimicrob Chemother 2010;65:1889-93.\\nPubMed PMID: 20639527.\\n18. \\nGatanaga H et al. Successful efavirenz dose reduction in HIV type 1-infected individuals with cytochrome P450 2B6 *6 and *26. Clin Infect Dis 2007;45:1230-7. PubMed\\nPMID: 17918089.\\n19. \\nHaas DW et al. Pharmacogenetics of long-term responses to antiretroviral regimens containing efavirenz and/or nelfinavir: an Adult Aids Clinical Trials Group study. J\\nInfect Dis 2005;192:1931-42. PubMed PMID: 16267764.\\n20. \\nSPC’s Efavirenz Mylan en Sustiva (VS).\\n21. \\nENCORE1 Study Group. Efficacy of 400 mg efavirenz versus standard 600 mg dose in HIV-infected, antiretroviral-naive adults (ENCORE1): a randomised, double-blind,\\nplacebo-controlled, non-inferiority trial. Lancet 2014;383:1474-82.\\nDate 05-03-2018\\nCYP2B6*1/*5: efavirenz\\n6928\\nNO action is required for this gene-drug interaction.\\nGene variant *5 has no effect on the metabolism and consequently on the efficacy and side effects of efavirenz.\\nLiterature:\\n1. \\nBurger D et al. Interpatient variability in the pharmacokinetics of the HIV non-nucleoside\\nreverse transcriptase inhibitor efavirenz: the effect of gender, race, and CYP2B6 polymorphism. Br J Clin Pharmacol 2006;61:148-54.\\nDate 05-03-2018\\nCYP2B6*5/*5: efavirenz\\n6929\\nNO action is required for this gene-drug interaction.\\nGene variant *5 has no effect on the metabolism and consequently on the efficacy and side effects of efavirenz.\\nLiterature:\\n1. \\nBurger D et al. Interpatient variability in the pharmacokinetics of the HIV non-nucleoside\\nreverse transcriptase inhibitor efavirenz: the effect of gender, race, and CYP2B6 polymorphism. Br J Clin Pharmacol 2006;61:148-54.\\nDate 05-03-2018\\nCYP2B6*5/*6 of *5/*18: efavirenz\\n6930\\nThe genetic variation increases the plasma concentration of efavirenz and thereby the risk of side effects. However, the efavirenz plasma concentration remains within the\\ntherapeutic range for the majority of patients.\\nRecommendation:\\nDetermine the efavirenz plasma concentration if side effects occur and reduce the dose if needed.\\nIn 14 adults with a genotype with the same effect, a reduction of the dose to 400 mg/day (2/3 of the standard dose) was sufficient to achieve therapeutic plasma\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 30}),\n",
       " Document(page_content=\"concentrations and for the side effects to reduce or disappear.\\nThe therapeutic range established for efavirenz is 1000-4000 ng/mL.\\nLiterature:\\n1. \\nVujkovic M et al. CYP2B6 516G>T minor allele protective of late virologic failure in efavirenz-treated HIV-infected patients in Botswana. J Acquir Immune Defic Syndr\\n2017 May 5 [Epub ahead of print].\\n2. \\nLeger P et al. Pharmacogenetics of efavirenz discontinuation for reported central nervous system symptoms appears to differ by race. Pharmacogenet Genomics\\n2016;26:473-80.\\n3. \\nCusato J et al. Efavirenz pharmacogenetics in a cohort of Italian patients. Int J Antimicrob Agents 2016;47:117-23.\\n4. \\nDickinson L et al. Comprehensive pharmacokinetic, pharmacodynamic and pharmacogenetic evaluation of once-daily efavirenz 400 and 600 mg in treatment-naïve HIV-\\ninfected patients at 96 weeks: results of the ENCORE1 study. Clin Pharmacokinet 2016;55:861-73.\\n5. \\nSwart M et al. An expanded analysis of pharmacogenetics determinants of efavirenz response that includes 3'-UTR single nucleotide polymorphisms among black South\\nAfrican HIV/AIDS patients. Front Genet 2016;6:356.\\n6. \\nMeng X et al. Effect of CYP2B6 gene polymorphisms on efavirenz plasma concentrations in Chinese patients with HIV infection. PLoS One 2015;10:e0130583.\\n7. \\nHaas DW et al. Functional CYP2B6 variants and virologic response to an efavirenz-containing regimen in Port-au-Prince, Haiti. J Antimicrob Chemother 2014;69:2187-90.\\n8. \\nMartín AS et al. Dose reduction of efavirenz: an observational study describing cost-effectiveness, pharmacokinetics and pharmacogenetics. Pharmacogenomics\\n2014;15:997-1006.\\n9. \\nBertrand J et al. Dependence of efavirenz- and rifampicin-isoniazid-based antituberculosis treatment drug-drug interaction on CYP2B6 and NAT2 genetic polymorphisms:\\nANRS 12154 study in Cambodia. J Infect Dis 2014;209:399-408.\\n10. \\nSarfo FS et al. Pharmacogenetic associations with plasma efavirenz concentrations and clinical correlates in a retrospective cohort of Ghanaian HIV-infected patients. J\\nAntimicrob Chemother 2014;69:491-9. PubMed PMID: 24080498.\\n11. \\nNgaimisi E et al. Importance of ethnicity, CYP2B6 and ABCB1 genotype for efavirenz pharmacokinetics and treatment outcomes: a parallel-group prospective cohort\\nstudy in two sub-Saharan Africa populations. PLoS One 2013;8:e67946. PubMed PMID: 23861838.\\n12. \\nYimer G et al. High plasma efavirenz level and CYP2B6*6 are associated with efavirenz-based HAART-induced liver injury in the treatment of naïve HIV patients from\\nEthiopia: a prospective cohort study. Pharmacogenomics J 2012;12:499-506. PubMed PMID: 21862974.\\n13. \\nMugusi S et al. Liver enzyme abnormalities and associated risk factors in HIV patients on efavirenz-based HAART with or without tuberculosis co-infection in Tanzania.\\nPLoS One 2012;7:e40180. PubMed PMID: 22808112.\\n14. \\nWyen C et al. Cytochrome P450 2B6 (CYP2B6) and constitutive androstane receptor (CAR) polymorphisms are associated with early discontinuation of efavirenz-\\ncontaining regimens. J Antimicrob Chemother 2011;66:2092-8. PubMed PMID: 21715435.\\n15. \\nCarr DF et al. Haplotype structure of CYP2B6 and association with plasma efavirenz concentrations in a Chilean HIV cohort. J Antimicrob Chemother 2010;65:1889-93.\\nPubMed PMID: 20639527.\\n16. \\nGatanaga H et al. Successful efavirenz dose reduction in HIV type 1-infected individuals with cytochrome P450 2B6 \\n6 and \\n26. Clin Infect Dis 2007;45:1230-7. PubMed\\nPMID: 17918089.\\n17. \\nBurger D et al. Interpatient variability in the pharmacokinetics of the HIV non-nucleoside reverse transcriptase inhibitor efavirenz: the effect of gender, race, and CYP2B6\\npolymorphism. Br J Clin Pharmacol 2006;61:148-54.\\n18. \\nHaas DW et al. Pharmacogenetics of long-term responses to antiretroviral regimens containing efavirenz and/or nelfinavir: an Adult Aids Clinical Trials Group study. J\\nInfect Dis 2005;192:1931-42. PubMed PMID: 16267764.\\nDate 05-03-2018\\nCYP2D6 IM: eliglustat\\n6138\\nThis gene variation reduces the conversion of eliglustat to inactive metabolites. However, in the absence of CYP2D6 and CYP3A inhibitors, this does not result in a clinically\\nsignificant increased risk of side effects.\\nRecommendation:\\nCo-medication with BOTH a MODERATE to STRONG CYP2D6 INHIBITOR AND a MODERATE to STRONG CYP3A INHIBITOR:\\nEliglustat is contra-indicated.\\n1. \\nchoose an alternative if possible\\nStrong CYP2D6 inhibitor: for example paroxetine, fluoxetine, quinidine, bupropione.\\nModerate CYP2D6 inhibitor: for example duloxetine, terbinafine, moclobemide, mirabegron, cinacalcet, dronedarone. Strong CYP3A inhibitor: for example\\nketoconazole, clarithromycin, itraconazole, cobicistat, indinavir, lopinavir, ritonavir, saquinavir, telaprevir, tipranavir, posaconazole, voriconazole, telithromycin,\\nconivaptan, boceprevir.\\nModerate CYP3A inhibitor: for example erythromycin, ciprofloxacin, fluconazole, diltiazem, verapamil, aprepitant, atazanavir, darunavir, fosamprenavir, imatinib,\\ncimetidine.\\nCo-medication with a STRONG CYP2D6 INHIBITOR (e.g. paroxetine, fluoxetine, quinidine, bupropione):\\n1. \\nuse a dose of 84 mg eliglustat 1x daily\\nCo-medication with a MODERATE CYP2D6 INHIBITOR (for example duloxetine, terbinafine, moclobemide, mirabegron, cinacalcet, dronedarone):\\n1. \\nconsider a dose of 84 mg eliglustat 1x daily\\n2. \\nbe alert to side effects\\nCo-medication with a STRONG CYP3A INHIBITOR (for example ketoconazole, clarithromycin, itraconazole, cobicistat, indinavir, lopinavir, ritonavir, saquinavir,\\ntelaprevir, tipranavir, posaconazole, voriconazole, telithromycin, conivaptan, boceprevir):\\nchoose an alternative if possible\\nif an alternative is not an option:\\nconsider a dose of 84 mg eliglustat 1x daily\\nbe alert to side effects\\nCo-medication with a MODERATE CYP3A INHIBITOR (for example erythromycin, ciprofloxacin, fluconazole, diltiazem, verapamil, aprepitant, atazanavir, darunavir,\\nfosamprenavir, imatinib, cimetidine):\\n1. \\nchoose an alternative\\n2. \\nif an alternative is not an option:\\n1. \\nconsider a dose of 84 mg eliglustat 1x daily\\n2. \\nbe alert to side effects\\nCo-medication with a STRONG CYP3A INDUCER (for example rifampicin, carbamazepine, phenobarbital, phenytoin, rifabutine, hypericum):\\nEliglustat is not recommended. The plasma concentration may decrease so sharply that a therapeutic effect cannot be achieved.\\n1. \\nchoose an alternative if possible\\nNO co-medication with a moderate or strong CYP2D6 or CYP3A inhibitor or strong CYP3A inducer:\\n1. \\nuse the standard dose of 84 mg 2x daily\\nLiterature:\\n1. SPC’s Cerdelga (Nederland en VS).\\nDate 31-10-2016\\nCYP2D6 PM: eliglustat\\n6137\\nThis gene variation reduces the conversion of eliglustat to inactive metabolites. This increases the risk of side effects, such as a (small, dose-dependent) elongation of the QT\\ninterval. CYP3A inhibitors increase this risk even further.\\nRecommendation:\\nCo-medication with a STRONG CYP3A INHIBITOR (for example ketoconazole, clarithromycin, itraconazole, cobicistat, indinavir, lopinavir, ritonavir, saquinavir,\\ntelaprevir, tipranavir, posaconazole, voriconazole, telithromycin, conivaptan, boceprevir):\\nEliglustat is contra-indicated.\\n1. \\nchoose an alternative if possible\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 31}),\n",
       " Document(page_content='Co-medication with a MODERATE CYP3A INHIBITOR (for example erythromycin, ciprofloxacin, fluconazole, diltiazem, verapamil, aprepitant, atazanavir, darunavir,\\nfosamprenavir, imatinib, cimetidine):\\nEliglustat is not recommended.\\n1. \\nchoose an alternative if possible\\nCo-medication with a WEAK CYP3A INHIBITOR (for example amlopidine, cilostazole, fluvoxamine, goldenseal, isoniazide, ranitidine, ranolazine):\\n1. \\nchoose an alternative for the weak CYP3A inhibitor if possible\\n2. \\nif an alternative is not an option:\\n1. \\nuse a dose of 84 mg eliglustat 1x daily\\n2. \\nbe alert to side effects\\nCo-medication with a STRONG CYP3A INDUCER (for example rifampicin, carbamazepine, phenobarbital, phenytoin, rifabutine, hypericum):\\nEliglustat is not recommended. The plasma concentration may decrease so sharply that a therapeutic effect cannot be achieved.\\n1. \\nchoose an alternative if possible\\nNO co-medication with a CYP3A inhibitor or strong CYP3A inducer:\\n1. \\nuse a dose of 84 mg 1x daily\\nLiterature:\\n1. SPC’s Cerdelga (Nederland en VS).\\nDate 31-10-2016\\nCYP2D6 UM: eliglustat\\n6139\\nThis gene variation increases the conversion of eliglustat to inactive metabolites. As a result, a normal dose is not effective. There is not enough scientific substantiation to\\nsuggest an effective dose for all UM.\\nRecommendation:\\nEliglustat is contra-indicated.\\n1. choose an alternative if possible\\nLiterature:\\n1. SPC’s Cerdelga (Nederland en VS).\\nDate 31-10-2016\\nCYP2C19 IM: escitalopram\\n1821\\nThe risk of QT prolongation and torsades de pointes is theoretically increased because the gene variation leads to an increased escitalopram plasma concentration. If you\\nfollow the dose recommendation below, the increased plasma concentration and the theoretically increased risk of QT prolongation will be offset.\\nDo not exceed the following doses (75% of the standard maximum dose):\\nadults < 65 years 15 mg/day, ≥65 years 7.5 mg/day\\nLiterature:\\n1. \\nTsuchimine S et al. Effects of cytochrome P450 (CYP) 2C19 genotypes on steady-state plasma concentrations of escitalopram and its desmethyl metabolite in Japanese\\npatients with depression. Ther Drug Monit 2018 Mar 22 [Epub ahead of print].\\n2. \\nJukić MM et al. Impact of CYP2C19 genotype on escitalopram exposure and therapeutic failure: a retrospective study based on 2,087 patients. Am J Psychiatry 2018 Jan\\n12 [Epub ahead of print].\\n3. \\nHe Q et al. Correlation between cytochrome P450 2C19 genetic polymorphism and treatment response to escitalopram in panic disorder. Pharmacogenet Genomics\\n2017;27:279-284.\\n4. \\nBishop JR et al. Escitalopram pharmacogenetics: CYP2C19 relationships with dosing and clinical outcomes in autism spectrum disorder. Pharmacogenet Genomics\\n2015:548-54.\\n5. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl) 2015;232: 2609-17.\\n6. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol 2014;28:1143-8.\\n7. \\nWaade RB et al. Impact of age on serum concentrations of venlafaxine and escitalopram in different CYP2D6 and CYP2C19 genotype subgroups. Eur J Clin Pharmacol\\n2014;70:933-40.\\n8. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol 2014;28:133-41.\\n9. \\nNg C et al. Pharmacogenetic polymorphisms and response to escitalopram and venlafaxine over 8 weeks in major depression. Hum Psychopharmacol 2013;28:516-22.\\n10. \\nHuezo-Diaz et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol 2012;26:398-407.\\n11. \\nBrasch-Andersen C et al. A candidate gene study of serotonergic pathway genes and pain relief during treatment with escitalopram in patients with neuropathic pain\\nshows significant association to serotonin receptor2C (HTR2C). Eur J Clin Pharmacol 2011; 67:1131-7.\\n12. \\nTsai et al. Genetic polymorphisms of cytochrome P450 enzymes influence metabolism of the antidepressant escitalopram and treatment response. Pharmacogenomics\\n2010:11;537-46.\\n13. \\nJin Y et al. Effect of age, weight, and CYP2C19 genotype on escitalopram exposure. J Clin Pharmacol. 2010;50:62-72.\\n14. \\nRudberg I et al. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Clin Pharmacol Ther 2008;83:322-7.\\n15. \\nRudberg I et al. Heterozygous mutation in CYP2C19 significantly increases the concentrations/dose ratio of racemic citalopram and escitalopram (S-citalopram). Ther\\nDrug Monitor 2006;28:102-5.\\nDate 14-05-2018\\nCYP2C19 PM: escitalopram\\n1822\\nThe risk of switching to another antidepressant is increased. In addition, the risk of QT prolongation and torsades de pointes is theoretically increased because the gene\\nvariation leads to an increased escitalopram plasma concentration. If you follow the dose recommendation below, the increased plasma concentration, the theoretically\\nincreased risk of QT prolongation and the increased risk of switching to another antidepressant will be offset.\\nDo not exceed the following doses (50% of the standard maximum dose):\\nadults < 65 years 10 mg/day, ≥65 years 5 mg/day\\nLiterature:\\n1. \\nTsuchimine S et al. Effects of cytochrome P450 (CYP) 2C19 genotypes on steady-state plasma concentrations of escitalopram and its desmethyl metabolite in Japanese\\npatients with depression. Ther Drug Monit 2018 Mar 22 [Epub ahead of print].\\n2. \\nJukić MM et al. Impact of CYP2C19 genotype on escitalopram exposure and therapeutic failure: a retrospective study based on 2,087 patients. Am J Psychiatry 2018 Jan\\n12 [Epub ahead of print].', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 32}),\n",
       " Document(page_content='3. \\nHe Q et al. Correlation between cytochrome P450 2C19 genetic polymorphism and treatment response to escitalopram in panic disorder. Pharmacogenet Genomics\\n2017;27:279-284.\\n4. \\nAsakura S et al. Long-term administration of escitalopram in patients with social anxiety disorder in Japan. Neuropsychiatr Dis Treat 2016;12:1817-25.\\n5. \\nBishop JR et al. Escitalopram pharmacogenetics: CYP2C19 relationships with dosing and clinical outcomes in autism spectrum disorder. Pharmacogenet Genomics\\n2015:548-54.\\n6. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl) 2015;232: 2609-17.\\n7. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol 2014;28:1143-8.\\n8. \\nWaade RB et al. Impact of age on serum concentrations of venlafaxine and escitalopram in different CYP2D6 and CYP2C19 genotype subgroups. Eur J Clin Pharmacol\\n2014;70:933-40.\\n9. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol 2014;28:133-41.\\n10. \\nNg C et al. Pharmacogenetic polymorphisms and response to escitalopram and venlafaxine over 8 weeks in major depression. Hum Psychopharmacol 2013;28:516-22.\\n11. \\nHuezo-Diaz et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol 2012;26:398-407.\\n12. \\nBrasch-Andersen C et al. A candidate gene study of serotonergic pathway genes and pain relief during treatment with escitalopram in patients with neuropathic pain\\nshows significant association to serotonin receptor2C (HTR2C). Eur J Clin Pharmacol 2011; 67:1131-7.\\n13. \\nTsai et al. Genetic polymorphisms of cytochrome P450 enzymes influence metabolism of the antidepressant escitalopram and treatment response. Pharmacogenomics\\n2010:11;537-46.\\n14. \\nNoehr-Jensen et al. Impact of CYP2C19 phenotypes on escitalopram metabolism and an evaluation of pupillometry as a serotonergic biomarker. Eur J Clin Pharmacol\\n2009;65:887-94.\\n15. \\nJin Y et al. Effect of age, weight, and CYP2C19 genotype on escitalopram exposure. J Clin Pharmacol. 2010;50:62-72.\\n16. \\nRudberg I et al. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Clin Pharmacol Ther 2008;83:322-7.\\n17. \\nSPC’s Lexapro (NL en VS).\\nDate 14-05-2018\\nCYP2C19 UM: escitalopram\\n1820\\nThe risk of switching to another antidepressant is increased as the gene variation leads to a reduction in the escitalopram plasma concentration.\\navoid escitalopram\\nAntidepressants that are not metabolised or that are metabolised to a lesser extent by CYP2C19 are, for example, paroxetine or fluvoxamine.\\nLiterature:\\n1. \\nJukić MM et al. Impact of CYP2C19 genotype on escitalopram exposure and therapeutic failure: a retrospective study based on 2,087 patients. Am J Psychiatry 2018 Jan\\n12 [Epub ahead of print].\\n2. \\nBishop JR et al. Escitalopram pharmacogenetics: CYP2C19 relationships with dosing and clinical outcomes in autism spectrum disorder. Pharmacogenet Genomics\\n2015:548-54.\\n3. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl) 2015;232: 2609-17.\\n4. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol 2014;28:133-41.\\n5. \\nHuezo-Diaz et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol 2012;26:398-407.\\n6. \\nBrasch-Andersen C et al. A candidate gene study of serotonergic pathway genes and pain relief during treatment with escitalopram in patients with neuropathic pain\\nshows significant association to serotonin receptor2C (HTR2C). Eur J Clin Pharmacol 2011; 67:1131-7.\\n7. \\nJin Y et al. Effect of age, weight, and CYP2C19 genotype on escitalopram exposure. J Clin Pharmacol. 2010;50:62-72.\\n8. \\nOhlsson Rosenborg S et al. Kinetics of omeprazole and escitalopram in relation to the CYP2C19*17 allele in healthy subjects. Eur J Clin Pharmacol 2008 Jul 25. 4;1175-79.\\n9. \\nRudberg I et al. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Clin Pharmacol Ther 2008;83:322-7.\\nDate 14-05-2018\\nCYP2C19 IM: esomeprazol\\n1824\\nNO action is needed for this gene-drug interaction.\\nAlthough the genetic variation leads to a higher plasma concentration of esomeprazole, there is insufficient evidence to support an effect on the therapeutic effectiveness and\\nside effects.\\nLiterature:\\n1. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with proton pump inhibitors. Meta Gene\\n2016;9:159-64.\\n2. \\nHsu WH et al. Genetic polymorphisms of CYP2C19 and IL1B have no influence on esomeprazole treatment for mild erosive esophagitis. Kaohsiung J Med Sci 2015;31:255-\\n9.\\n3. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n4. \\nLee VW et al. Pharmacogenetics of esomeprazole or rabeprazole-based triple therapy in Helicobacter pylori eradication in Hong Kong non-ulcer dyspepsia Chinese\\nsubjects. J Clin Pharm Ther 2010;35:343-50.\\n5. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics and CYP2C19 polymorphism. Aliment\\nPharmacol Ther 2010;31:150-9.\\n6. \\nLou HY et al. Optimal dose regimens of esomeprazole for gastric acid suppression with minimal influence of the CYP2C19 polymorphism. Eur J Clin Pharmacol\\n2009;65:55-64.\\n7. \\nSheu BS et al. Body mass index can determine the healing of reflux esophagitis with Los Angeles Grades C and D by esomeprazole. Am J Gastroenterol 2008;103:2209-14.\\n8. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J Gastroenterol Hepatol 2007;22:815-20.\\n9. \\nSchwab M et al. Esomeprazole-induced healing of gastroesophageal reflux disease is unrelated to the genotype of CYP2C19: evidence from clinical and pharmacokinetic\\ndata. Clin Pharmacol Ther 2005;78:627-34.\\n10. \\nKuo CH et al. Efficacy of levofloxacin-based rescue therapy for Helicobacter pylori infection after standard triple therapy: a randomized controlled trial. J Antimicrob\\nChemother 2009;63:1017-24.\\n11. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple therapy with regular proton pump inhibitor\\ndosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n12. \\nMiehlke S et al. One-week once-daily triple therapy with esomeprazole, moxifloxacin, and rifabutin for eradication of persistent Helicobacter pylori resistant to both\\nmetronidazole and clarithromycin. Helicobacter 2008;13:69-74.\\n13. \\nMiehlke S et al. Randomized trial of rifabutin-based triple therapy and high-dose dual therapy for rescue treatment of Helicobacter pylori resistant to both metronidazole\\nand clarithromycin. Aliment Pharmacol Ther 2006;24:395-403.\\n14. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to CYP2C19 metabolism. Aliment Pharmacol\\nTher 2005;21:283-8.\\nDate 05-03-2018\\nCYP2C19 PM: esomeprazol\\n1825\\nNO action is needed for this gene-drug interaction.\\nAlthough the genetic variation leads to a higher plasma concentration of esomeprazole, there is insufficient evidence to support an effect on the therapeutic effectiveness and\\nside effects.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 33}),\n",
       " Document(page_content='Literature:\\n1. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with proton pump inhibitors. Meta Gene\\n2016;9:159-64.\\n2. \\nHsu WH et al. Genetic polymorphisms of CYP2C19 and IL1B have no influence on esomeprazole treatment for mild erosive esophagitis. Kaohsiung J Med Sci 2015;31:255-\\n9.\\n3. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n4. \\nLee VW et al. Pharmacogenetics of esomeprazole or rabeprazole-based triple therapy in Helicobacter pylori eradication in Hong Kong non-ulcer dyspepsia Chinese\\nsubjects. J Clin Pharm Ther 2010;35:343-50.\\n5. \\nLou HY et al. Optimal dose regimens of esomeprazole for gastric acid suppression with minimal influence of the CYP2C19 polymorphism. Eur J Clin Pharmacol\\n2009;65:55-64.\\n6. \\nSheu BS et al. Body mass index can determine the healing of reflux esophagitis with Los Angeles Grades C and D by esomeprazole. Am J Gastroenterol 2008;103:2209-14.\\n7. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J Gastroenterol Hepatol 2007;22:815-20.\\n8. \\nSchwab M et al. Esomeprazole-induced healing of gastroesophageal reflux disease is unrelated to he genotype of CYP2C19: evidence from clinical and pharmacokinetic\\ndata. Clin Pharmacol Ther 2005;78:627-34.\\n9. \\nKuo CH et al. Efficacy of levofloxacin-based rescue therapy for Helicobacter pylori infection after standard triple therapy: a randomized controlled trial. J Antimicrob\\nChemother 2009;63:1017-24.\\n10. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple therapy with regular proton pump inhibitor\\ndosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n11. \\nMiehlke S et al. One-week once-daily triple therapy with esomeprazole, moxifloxacin, and rifabutin for eradication of persistent Helicobacter pylori resistant to both\\nmetronidazole and clarithromycin. Helicobacter 2008;13:69-74.\\n12. \\nMiehlke S et al. Randomized trial of rifabutin-based triple therapy and high-dose dual therapy for rescue treatment of Helicobacter pylori resistant to both metronidazole\\nand clarithromycin. Aliment Pharmacol Ther 2006;24:395-403.\\n13. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to CYP2C19 metabolism. Aliment Pharmacol\\nTher 2005;21:283-8.\\n14. \\nSPC Nexium (Nederlands en Amerikaans).\\nDate 05-03-2018\\nCYP2C19 UM: esomeprazol\\n1826\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation may lead to faster inactivation of esomeprazole, there is insufficient evidence to support an effect on the therapeutic effectiveness and side\\neffects.\\nLiterature:\\n1. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with proton pump inhibitors. Meta Gene\\n2016;9:159-64.\\n2. \\nDury S et al. Agranulocytosis induced by proton pump inhibitors. J Clin Gastroenterol 2012;46:859.\\n3. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics and CYP2C19 polymorphism. Aliment\\nPharmacol Ther 2010;31:150-9.\\nDate 05-03-2018\\nCYP2C9 IM ANDERS: fenprocoumon\\n1875\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that this causes problems with normal\\ninitiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n3. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n4. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n5. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population\\npharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n7. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-genetic parameters. Eur J Clin Pharmacol\\n2011;67:371-81.\\n8. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n10. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol 2009;65:783-8.\\n11. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb Thrombolysis 2009;28:211-4.\\n12. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n13. \\nBohrer T et al. Left ventricular non-compaction associated with a genetic variant of the CYP2C9 gene. Heart Lung Circ 2006;15:269-71.\\n14. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9 genotypes are not. Eur J Clin Pharmacol\\n2006;62:713-20.\\n15. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n16. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol Ther 2004;76:409-17.\\n17. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and in vivo. Xenobiotica 2004;34:847-59.\\n18. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n19. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;14:27-33.\\n20. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy volunteers. Pharmacogenetics 2004;14:19-26.\\n21. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin Pharmacol 2003;59:213-9.\\nDate 14-05-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 34}),\n",
       " Document(page_content='CYP2C9 PM ANDERS: fenprocoumon\\n1876\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that this causes problems with normal\\ninitiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n3. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n4. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n5. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population\\npharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n7. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-genetic parameters. Eur J Clin Pharmacol\\n2011;67:371-81.\\n8. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n10. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol 2009;65:783-8.\\n11. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb Thrombolysis 2009;28:211-4.\\n12. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n13. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9 genotypes are not. Eur J Clin Pharmacol\\n2006;62:713-20.\\n14. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n15. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol Ther 2004;76:409-17.\\n16. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and in vivo. Xenobiotica 2004;34:847-59.\\n17. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n18. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;14:27-33.\\n19. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy volunteers. Pharmacogenetics 2004;14:19-26.\\n20. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin Pharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9*1/*2: fenprocoumon\\n1870\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that this causes problems with normal\\ninitiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n3. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n4. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n5. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population\\npharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n7. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-genetic parameters. Eur J Clin Pharmacol\\n2011;67:371-81.\\n8. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n10. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol 2009;65:783-8.\\n11. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb Thrombolysis 2009;28:211-4.\\n12. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n13. \\nBohrer T et al. Left ventricular non-compaction associated with a genetic variant of the CYP2C9 gene. Heart Lung Circ 2006;15:269-71.\\n14. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9 genotypes are not. Eur J Clin Pharmacol\\n2006;62:713-20.\\n15. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n16. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol Ther 2004;76:409-17.\\n17. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and in vivo. Xenobiotica 2004;34:847-59.\\n18. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n19. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;14:27-33.\\n20. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy volunteers. Pharmacogenetics 2004;14:19-26.\\n21. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin Pharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9*1/*3: fenprocoumon\\n1871\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that this causes problems with normal\\ninitiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 35}),\n",
       " Document(page_content='1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n3. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n4. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n5. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population\\npharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n7. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-genetic parameters. Eur J Clin Pharmacol\\n2011;67:371-81.\\n8. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n10. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol 2009;65:783-8.\\n11. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb Thrombolysis 2009;28:211-4.\\n12. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n13. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9 genotypes are not. Eur J Clin Pharmacol\\n2006;62:713-20.\\n14. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n15. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol Ther 2004;76:409-17.\\n16. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and in vivo. Xenobiotica 2004;34:847-59.\\n17. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n18. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;14:27-33.\\n19. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy volunteers. Pharmacogenetics 2004;14:19-26.\\n20. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin Pharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9*2/*2: fenprocoumon\\n1872\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose and possibly in an extension of the time required to achieve a stable INR. However, there is\\nnot enough evidence to confirm that this causes problems with normal initiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n3. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n4. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n5. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population\\npharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n7. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n8. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol 2009;65:783-8.\\n9. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb Thrombolysis 2009;28:211-4.\\n10. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n11. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9 genotypes are not. Eur J Clin Pharmacol\\n2006;62:713-20.\\n12. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin\\nPharmacol Ther 2005;77:479-85.\\n13. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol Ther 2004;76:409-17.\\n14. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and in vivo. Xenobiotica 2004;34:847-59.\\n15. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n16. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;14:27-33.\\n17. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy volunteers. Pharmacogenetics 2004;14:19-26.\\nDate 14-05-2018\\nCYP2C9*2/*3: fenprocoumon\\n1873\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that this causes problems with normal\\ninitiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n3. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n4. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n5. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population\\npharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n7. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-genetic parameters. Eur J Clin Pharmacol\\n2011;67:371-81.\\n8. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n10. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol 2009;65:783-8.\\n11. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n12. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9 genotypes are not. Eur J Clin Pharmacol\\n2006;62:713-20.\\n13. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and coumarin anticoagulants. Clin', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 36}),\n",
       " Document(page_content='Pharmacol Ther 2005;77:479-85.\\n14. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol Ther 2004;76:409-17.\\n15. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and in vivo. Xenobiotica 2004;34:847-59.\\n16. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon. Thromb\\nHaemost 2004;92:61-6.\\n17. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on acenocoumarol or phenprocoumon.\\nPharmacogenetics 2004;14:27-33.\\n18. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy volunteers. Pharmacogenetics 2004;14:19-26.\\n19. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin Pharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9*3/*3: fenprocoumon\\n1874\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that this causes problems with normal\\ninitiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n2. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n3. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol 2009;65:783-8.\\n4. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n5. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol Ther 2004;76:409-17.\\n6. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and in vivo. Xenobiotica 2004;34:847-59.\\n7. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy volunteers. Pharmacogenetics 2004;14:19-26.\\n8. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin Pharmacol 2003;59:213-9.\\nDate 14-05-2018\\nVKORC1 -1639 AA: fenprocoumon\\n1912\\nAn INR ≥ 6, resulting in an increased risk of bleeding, occurs in 17% of these patients with standard regulation by the Anticoagulation Clinic. The genetic variation increases\\nthe sensitivity to phenprocoumon.\\nMonitoring by a ANTICOAGULATION CLINIC:\\nrecommend to use 50% of the standard initial dose\\nNO monitoring by a anticoagulation clinic:\\nrecommend to use 50% of the standard initial dose\\nrecommend more frequent monitoring of the INR\\nFor patients younger than 75 years, the initial dose and the maintenance dose can be calculated using an algorithm as found in EU-PACT: see\\nhttps://www.knmp.nl/patientenzorg/medicatiebewaking/farmacogenetica for a calculation tool in the form of an Excel file. However, for patients aged 75 years and older, this\\nalgorithm increases the risk of an INR above the therapeutic range compared to an algorithm without gene variations. Therefore, use of this algorithm is not recommended for\\nthese patients.\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.\\n3. \\nBotton et al. A new algorithm for weekly phenprocoumon dose variation in a southern Brazilian population: role for CYP2C9, CYP3A4/5 and VKORC1 genes\\npolymorphisms. Basic Clin Pharmacol Toxicol 2014;114:323-9\\n4. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n5. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n6. \\nAbduljalil K et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population pharmacokinetic/\\npharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n7. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n8. \\nVerhoef TI et al. Long-term anticoagulant effects of CYP2C9 and VKORC1 genotypes in phenprocoumon users. J Thromb Haemost. 2012;10:2610-2.\\n9. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-genetic parameters. Eur J Clin Pharmacol\\n2011;67:371-81.\\n10. \\nTeichert M et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1, CYP2C9, and CYP4F2 genes. Pharmacogenet Genomics 2011;21:26-34.\\n11. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n12. \\nNowak-Göttl U et al. In pediatric patients, age has more impact on dosing of vitamin K antagonists than VKORC1 or CYP2C9 genotypes. Blood 2010;116:6101-5.\\n13. \\nPuehringer H et al. VKORC1 -1639G>A and CYP2C9*3 are the major genetic predictors of phenprocoumon dose requirement. Eur J Clin Pharmacol 2010;66:591-8.\\n14. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n15. \\nArnold ML et al. Pharmacogenetic testing for guiding de novo phenprocoumon therapy in stroke patients. Cerebrovasc Dis 2009;28:468-71.\\n16. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb Thrombolysis 2009;28:211-4.\\n17. \\nSchalekamp et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n18. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med 2005;2:e312.\\nDate 10-09-2018\\nVKORC1 -1639 GA: fenprocoumon\\n1911\\nNO action is needed for this gene-drug interaction.\\nThe gene variation leads to a lower dose requirement, but regular monitoring of patients ensures that this does not lead to a distinct increase in the risk of bleeding.\\nLiterature:\\n1. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb Haemost 2017;15:454-464.\\n2. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac Surg 2016;50:275-80.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 37}),\n",
       " Document(page_content='3. \\nBoton et al. A new algorithm for weekly phenprocoumon dose variation in a southern Brazilian population: role for CYP2C9, CYP3A4/5 and VKORC1 genes\\npolymorphisms. Basic Clin Pharmacol Toxicol 2014;114:323-9.\\n4. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med 2013;369:2304-12.\\n5. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost 2017;15:465-472.\\n6. \\nAbduljalil K et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a population pharmacokinetic/\\npharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n7. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon metabolism. Eur J Cardiothorac Surg\\n2013;44:309-14.\\n8. \\nVerhoef TI et al. Long-term anticoagulant effects of CYP2C9 and VKORC1 genotypes in phenprocoumon users. J Thromb Haemost. 2012;10:2610-2.\\n9. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-genetic parameters. Eur J Clin Pharmacol\\n2011;67:371-81.\\n10. \\nTeichert M et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1, CYP2C9, and CYP4F2 genes. Pharmacogenet Genomics 2011;21:26-34.\\n11. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene variants on the initiation and maintenance\\nphases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n12. \\nNowak-Göttl U et al. In pediatric patients, age has more impact on dosing of vitamin K antagonists than VKORC1 or CYP2C9 genotypes. Blood 2010;116:6101-5.\\n13. \\nPuehringer H et al. VKORC1 -1639G>A and CYP2C9*3 are the major genetic predictors of phenprocoumon dose requirement. Eur J Clin Pharmacol 2010;66:591-8.\\n14. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin Pharmacol 2010;66:253-60.\\n15. \\nArnold ML et al. Pharmacogenetic testing for guiding de novo phenprocoumon therapy in stroke patients. Cerebrovasc Dis 2009;28:468-71.\\n16. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb Thrombolysis 2009;28:211-4.\\n17. \\nSchmeits PC et al. Investigating unexpected INRs: in search of the culprit--adherence, interactions, genetics, and superwarfarin. Neth J Med 2009;67:76-8.\\n18. \\nSchalekamp et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both genotypes affects dose requirement. Clin\\nPharmacol Ther 2007;81:185-93.\\n19. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med 2005;2:e312.\\nDate 10-09-2018\\nCYP2C9 IM ANDERS: fenytoine\\n1676\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1. \\nThe loading dose does not need to be adjusted.\\n2. \\nFor the other doses, use 75% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1. \\nFohner AE et al. Assessing the clinical impact of CYP2C9 pharmacogenetic variation on phenytoin prescribing practice and patient response in an integrated health\\nsystem. Pharmacogenet Genomics 2019;29:192-9.\\n2. \\nSu SC et al. HLA alleles and CYP2C9*3 as predictors of phenytoin hypersensitivity in East Asians. Clin Pharmacol Ther 2019;105:476-85.\\n3. \\nWu X et al. Association of CYP2C9*3 with phenytoin-induced Stevens-Johnson syndrome and toxic epidermal necrolysis: a systematic review and meta-analysis. J Clin\\nPharm Ther 2018;43:408-13.\\n4. \\nYampayon K et al. Influence of genetic and non-genetic factors on phenytoin-induced severe cutaneous adverse drug reactions. Eur J Clin Pharmacol 2017;73:855-65.\\n5. \\nTassaneeyakul W et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-related severe cutaneous adverse reactions in\\na Thai population. Pharmacogenet Genomics 2016;26:225-34.\\n6. \\nOrtega-Vázquez A et al. CYP2C9, CYP2C19, ABCB1 genetic polymorphisms and phenytoin plasma concentrations in Mexican-Mestizo patients with epilepsy.\\nPharmacogenomics J 2016;16:286-92.\\n7. \\nYamamoto Y et al. Individualized phenytoin therapy for Japanese pediatric patients with epilepsy based on CYP2C9 and CYP2C19 genotypes. Ther Drug Monit\\n2015;37:229-35.\\n8. \\nChung WH et al. Genetic variants associated with phenytoin-related severe cutaneous adverse reactions. JAMA 2014;312:525-34.\\n9. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9 variants with phenytoin toxicity. Eur J Neurol\\n2011;18:1159-64.\\n10. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in Indian epileptic patients. Eur J Clin Pharmacol\\n2010;66:689-96.\\n11. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet Genomics 2010;20:58-63.\\n12. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol 2009;49:1483-7.\\n13. \\nMcCluggage LK et al. Phenytoin toxicity due to genetic polymorphism. Neurocrit Care 2009;10:222-4.\\n14. \\nLee SY et al. Contributions of CYP2C9/CYP2C19 genotypes and drug interaction to the phenytoin treatment in the Korean epileptic patients in the clinical setting. J\\nBiochem Mol Biol 2007;40:448-52.\\n15. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs carbamazepine and phenytoin. Proc Natl Acad Sci U\\nS A 2005;102:5507-12.\\n16. \\nHung CC et al. Dosage recommendation of phenytoin for patients with epilepsy with different CYP2C9/CYP2C19 polymorphisms. Ther Drug Monit 2004;26:534-40.\\n17. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement. Pharmacogenetics 2001;11:287-91.\\n18. \\nNinomiya H et al. Genetic polymorphism of the CYP2C subfamily and excessive serum phenytoin concentration with central nervous system intoxication. Ther Drug Monit\\n2000;22:230-2.\\n19. \\nMamiya K et al. The effects of genetic polymorphisms of CYP2C9 and CYP2C19 on phenytoin metabolism in Japanese adult patients with epilepsy: studies in\\nstereoselective hydroxylation and population pharmacokinetics. Epilepsia 1998;39:1317-23.\\n20. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 23-05-2022\\nCYP2C9 PM ANDERS: fenytoine\\n1677\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1. \\nThe loading dose does not need to be adjusted.\\n2. \\nFor the other doses, use 40-50% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1. \\nOrtega-Vázquez A et al. CYP2C9, CYP2C19, ABCB1 genetic polymorphisms and phenytoin plasma concentrations in Mexican-Mestizo patients with epilepsy.\\nPharmacogenomics J 2016;16:286-92.\\n2. \\nKidd RS et al. Identification of a null allele of CYP2C9 in an African-American exhibiting toxicity to phenytoin. Pharmacogenetics 2001;11:803-8.\\nDate 23-05-2022\\nCYP2C9*1/*2: fenytoine\\n1678\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 38}),\n",
       " Document(page_content='Recommendation:\\n1. \\nThe loading dose does not need to be adjusted.\\n2. \\nFor the other doses, use 75% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1. \\nFohner AE et al. Assessing the clinical impact of CYP2C9 pharmacogenetic variation on phenytoin prescribing practice and patient response in an integrated health\\nsystem. Pharmacogenet Genomics 2019;29:192-9.\\n2. \\nGuevara N et al. Role of CYP2C9, CYP2C19 and EPHX polymorphism in the pharmacokinetic of phenytoin: a study on Uruguayan Caucasian subjects. Pharmaceuticals\\n(Basel) 2017;10:73.\\n3. \\nOrtega-Vázquez A et al. CYP2C9, CYP2C19, ABCB1 genetic polymorphisms and phenytoin plasma concentrations in Mexican-Mestizo patients with epilepsy.\\nPharmacogenomics J 2016;16:286-92.\\n4. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9 variants with phenytoin toxicity. Eur J Neurol\\n2011;18:1159-64.\\n5. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in Indian epileptic patients. Eur J Clin Pharmacol\\n2010;66:689-96.\\n6. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet Genomics 2010;20:58-63.\\n7. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol 2009;49:1483-7.\\n8. \\nRosemary J et al. Influence of the CYP2C9 AND CYP2C19 polymorphisms on phenytoin hydroxylation in healthy individuals from south India. Indian J Med Res\\n2006;123:665-70.\\n9. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs carbamazepine and phenytoin. Proc Natl Acad Sci U\\nS A 2005;102:5507-12.\\n10. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n11. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement. Pharmacogenetics 2001;11:287-91.\\n12. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels. Pharmacogenomics J 2001;1:204-10.\\n13. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for phenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n14. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 23-05-2022\\nCYP2C9*1/*3: fenytoine\\n1679\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects. The life-threatening cutaneous side effects Stevens-Johnson\\nSyndrome and toxic epidermal necrolysis may occur, especially in Asian patients.\\nRecommendation:\\n1. \\nThe loading dose does not need to be adjusted.\\n2. \\nFor the other doses, use 75% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or, especially in Asian patients, rash) occur.\\nLiterature:\\n1. \\nSukasem C et al. Genetic and clinical risk factors associated with phenytoin-induced cutaneous adverse drug reactions in Thai population. Pharmacoepidemiol Drug Saf\\n2020;29:565-74.\\n2. \\nFohner AE et al. Assessing the clinical impact of CYP2C9 pharmacogenetic variation on phenytoin prescribing practice and patient response in an integrated health\\nsystem. Pharmacogenet Genomics 2019;29:192-9.\\n3. \\nSu SC et al. HLA alleles and CYP2C9*3 as predictors of phenytoin hypersensitivity in East Asians. Clin Pharmacol Ther 2019;105:476-85.\\n4. \\nWu X et al. Association of CYP2C9*3 with phenytoin-induced Stevens-Johnson syndrome and toxic epidermal necrolysis: a systematic review and meta-analysis. J Clin\\nPharm Ther 2018;43:408-13.\\n5. \\nGuevara N et al. Role of CYP2C9, CYP2C19 and EPHX polymorphism in the pharmacokinetic of phenytoin: a study on Uruguayan Caucasian subjects. Pharmaceuticals\\n(Basel) 2017;10:73.\\n6. \\nYampayon K et al. Influence of genetic and non-genetic factors on phenytoin-induced severe cutaneous adverse drug reactions. Eur J Clin Pharmacol 2017;73:855-65.\\n7. \\nTassaneeyakul W et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-related severe cutaneous adverse reactions in\\na Thai population. Pharmacogenet Genomics 2016;26:225-34.\\n8. \\nYamamoto Y et al. Individualized phenytoin therapy for Japanese pediatric patients with epilepsy based on CYP2C9 and CYP2C19 genotypes. Ther Drug Monit\\n2015;37:229-35.\\n9. \\nChung WH et al. Genetic variants associated with phenytoin-related severe cutaneous adverse reactions. JAMA 2014;312:525-34.\\n10. \\nHung CC et al. Effects of polymorphisms in six candidate genes on phenytoin maintenance therapy in Han Chinese patients. Pharmacogenomics 2012;13:1339-49.\\n11. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9 variants with phenytoin toxicity. Eur J Neurol\\n2011;18:1159-64.\\n12. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in Indian epileptic patients. Eur J Clin Pharmacol\\n2010;66:689-96.\\n13. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet Genomics 2010;20:58-63.\\n14. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol 2009;49:1483-7.\\n15. \\nMcCluggage LK et al. Phenytoin toxicity due to genetic polymorphism. Neurocrit Care 2009;10:222-4.\\n16. \\nLee SY et al. Contributions of CYP2C9/CYP2C19 genotypes and drug interaction to the phenytoin treatment in the Korean epileptic patients in the clinical setting. J\\nBiochem Mol Biol 2007;40:448-52.\\n17. \\nRosemary J et al. Influence of the CYP2C9 AND CYP2C19 polymorphisms on phenytoin hydroxylation in healthy individuals from south India. Indian J Med Res\\n2006;123:665-70.\\n18. \\nHung CC et al. Dosage recommendation of phenytoin for patients with epilepsy with different CYP2C9/CYP2C19 polymorphisms. Ther Drug Monit 2004;26:534-40.\\n19. \\nSoga Y et al. CYP2C polymorphisms, phenytoin metabolism and gingival overgrowth in epileptic subjects. Life Sci 2004;74:827-34.\\n20. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n21. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels. Pharmacogenomics J 2001;1:204-10.\\n22. \\nNinomiya H et al. Genetic polymorphism of the CYP2C subfamily and excessive serum phenytoin concentration with central nervous system intoxication. Ther Drug Monit\\n2000;22:230-2.\\n23. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for phenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n24. \\nMamiya K et al. The effects of genetic polymorphisms of CYP2C9 and CYP2C19 on phenytoin metabolism in Japanese adult patients with epilepsy: studies in\\nstereoselective hydroxylation and population pharmacokinetics. Epilepsia 1998;39:1317-23.\\n25. \\nOdani A et al. Genetic polymorphism of the CYP2C subfamily and its effect on the pharmacokinetics of phenytoin in Japanese patients with epilepsy. Clin Pharmacol Ther\\n1997;62:287-92.\\n26. \\nHashimoto Y et al. Effect of CYP2C polymorphisms on the pharmacokinetics of phenytoin in Japanese patients with epilepsy. Biol Pharm Bull 1996;19:1103-5.\\n27. \\nSmPC’s Diphantoïne-Z (NL) en Dilantin (VS).\\n28. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 23-05-2022\\nCYP2C9*2/*2: fenytoine\\n1680\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1. \\nThe loading dose does not need to be adjusted.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 39}),\n",
       " Document(page_content='2. \\nFor the other doses, use 50% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1. \\nFohner AE et al. Assessing the clinical impact of CYP2C9 pharmacogenetic variation on phenytoin prescribing practice and patient response in an integrated health\\nsystem. Pharmacogenet Genomics 2019;29:192-9.\\n2. \\nGuevara N et al. Role of CYP2C9, CYP2C19 and EPHX polymorphism in the pharmacokinetic of phenytoin: a study on Uruguayan Caucasian subjects. Pharmaceuticals\\n(Basel) 2017;10:73.\\n3. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9 variants with phenytoin toxicity. Eur J Neurol\\n2011;18:1159-64.\\n4. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet Genomics 2010;20:58-63.\\n5. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol 2009;49:1483-7.\\n6. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs carbamazepine and phenytoin. Proc Natl Acad Sci U\\nS A 2005;102:5507-12.\\n7. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n8. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement. Pharmacogenetics 2001;11:287-91.\\n9. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels. Pharmacogenomics J 2001;1:204-10.\\n10. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for phenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n11. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 23-05-2022\\nCYP2C9*2/*3: fenytoine\\n1681\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1. \\nThe loading dose does not need to be adjusted.\\n2. \\nFor the other doses, use 50% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1. \\nFohner AE et al. Assessing the clinical impact of CYP2C9 pharmacogenetic variation on phenytoin prescribing practice and patient response in an integrated health\\nsystem. Pharmacogenet Genomics 2019;29:192-9.\\n2. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in Indian epileptic patients. Eur J Clin Pharmacol\\n2010;66:689-96.\\n3. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet Genomics 2010;20:58-63.\\n4. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n5. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement. Pharmacogenetics 2001;11:287-91.\\n6. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 23-05-2022\\nCYP2C9*3/*3: fenytoine\\n1682\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects. The life-threatening cutaneous side effects Stevens-Johnson\\nSyndrome and toxic epidermal necrolysis may occur, especially in Asian patients.\\nRecommendation:\\n1. \\nThe loading dose does not need to be adjusted.\\n2. \\nFor the other doses, use 40% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or, especially in Asian patients, rash) occur.\\nLiterature:\\n1. \\nSukasem C et al. Genetic and clinical risk factors associated with phenytoin-induced cutaneous adverse drug reactions in Thai population. Pharmacoepidemiol Drug Saf\\n2020;29:565-74.\\n2. \\nFohner AE et al. Assessing the clinical impact of CYP2C9 pharmacogenetic variation on phenytoin prescribing practice and patient response in an integrated health\\nsystem. Pharmacogenet Genomics 2019;29:192-9.\\n3. \\nSu SC et al. HLA alleles and CYP2C9*3 as predictors of phenytoin hypersensitivity in East Asians. Clin Pharmacol Ther 2019;105:476-85.\\n4. \\nWu X et al. Association of CYP2C9*3 with phenytoin-induced Stevens-Johnson syndrome and toxic epidermal necrolysis: a systematic review and meta-analysis. J Clin\\nPharm Ther 2018;43:408-13.\\n5. \\nYampayon K et al. Influence of genetic and non-genetic factors on phenytoin-induced severe cutaneous adverse drug reactions. Eur J Clin Pharmacol 2017;73:855-65.\\n6. \\nTassaneeyakul W et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-related severe cutaneous adverse reactions in\\na Thai population. Pharmacogenet Genomics 2016;26:225-34.\\n7. \\nChung WH et al. Genetic variants associated with phenytoin-related severe cutaneous adverse reactions. JAMA 2014;312:525-34.\\n8. \\nHung CC et al. Effects of polymorphisms in six candidate genes on phenytoin maintenance therapy in Han Chinese patients. Pharmacogenomics 2012;13:1339-49.\\n9. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in Indian epileptic patients. Eur J Clin Pharmacol\\n2010;66:689-96.\\n10. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet Genomics 2010;20:58-63.\\n11. \\nJose L et al. Acenocoumarol and phenytoin toxicity in the presence of CYP2C9 mutation. J Assoc Physicians India 2008;56:250-2.\\n12. \\nRamasamy K et al. Severe phenytoin toxicity in a CYP2C9*3*3 homozygous mutant from India. Neurol India 2007;55:408-9.\\n13. \\nLee SY et al. Contributions of CYP2C9/CYP2C19 genotypes and drug interaction to the phenytoin treatment in the Korean epileptic patients in the clinical setting. J\\nBiochem Mol Biol 2007;40:448-52.\\n14. \\nRosemary J et al. Influence of the CYP2C9 AND CYP2C19 polymorphisms on phenytoin hydroxylation in healthy individuals from south India. Indian J Med Res\\n2006;123:665-70.\\n15. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs carbamazepine and phenytoin. Proc Natl Acad Sci U\\nS A 2005;102:5507-12.\\n16. \\nBrandolese R et al. Severe phenytoin intoxication in a subject homozygous for CYP2C9*3. Clin Pharmacol Ther 2001;70:391-4.\\n17. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels. Pharmacogenomics J 2001;1:204-10.\\n18. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for phenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n19. \\nKidd RS et al. Pharmacokinetics of chlorpheniramine, phenytoin, glipizide and nifedipine in an individual homozygous for the CYP2C9*3 allele. Pharmacogenetics\\n1999;9:71-80.\\n20. \\nSmPC’s Diphantoïne-Z (NL) en Dilantin (VS).\\n21. \\nwww.nvza.nl, TDM monografie voor fenytoïne.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 40}),\n",
       " Document(page_content='Date 23-05-2022\\nHLA-B*1502: fenytoine\\n6927\\nThe life-threatening cutaneous side effect Stevens-Johnson syndrome/toxic epidermal necrolysis (SJS/TEN) occurs more frequently in patients with this genetic variation. The\\ncalculated risk of phenytoin-induced SJS/TEN in patients with HLA-B*1502 is 0.65%.\\ncarefully weigh the risk of SJS/TEN against the benefits\\navoid phenytoin if an alternative is possible\\nCarbamazepine carries a 10-fold higher risk of SJS/TEN for these patients and is therefore not an alternative.\\nA comparable risk has been reported for lamotrigine as for phenytoin. The same applies for oxcarbazepine, but the most severe forms (SJS/TEN overlap and TEN) are not\\nobserved with oxcarbazepine.\\nif it is not possible to avoid this medication, then advise the patient to report any skin rash immediately\\nLiterature:\\n1. \\nPhung TH et al. The association between HLA-B*15:02 and phenytoin-induced severe cutaneous adverse reactions: a meta-analysis. Pharmacogenomics 2022;23:49-59.\\n2. \\nSukasem C et al. Spectrum of cutaneous adverse reactions to aromatic antiepileptic drugs and human leukocyte antigen genotypes in Thai patients and meta-analysis.\\nPharmacogenomics J 2021;21:682-90.\\n3. \\nManuyakorn W et al. Association of HLA genotypes with phenytoin induced severe cutaneous adverse drug reactions in Thai children. Epilepsy Res 2020;162:106321.\\n4. \\nSukasem C et al. Genetic and clinical risk factors associated with phenytoin-induced cutaneous adverse drug reactions in Thai population. Pharmacoepidemiol Drug Saf\\n2020;29:565-74.\\n5. \\nSu SC et al. HLA alleles and CYP2C9*3 as predictors of phenytoin hypersensitivity in East Asians. Clin Pharmacol Ther 2019;105:476-85.\\n6. \\nYampayon K et al. Influence of genetic and non-genetic factors on phenytoin-induced severe cutaneous adverse drug reactions. Eur J Clin Pharmacol 2017;73:855-865.\\n7. \\nShi YW et al. HLA-A*24:02 as a common risk factor for antiepileptic drug-induced cutaneous adverse reactions. Neurology 2017;88:2183-2191.\\n8. \\nChang CC et al. Association of HLA-B*15:13 and HLA-B*15:02 with phenytoin-induced severe cutaneous adverse reactions in a Malay population. Pharmacogenomics J\\n2017;17:170-173.\\n9. \\nTassaneeyakul W et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-related severe cutaneous adverse reactions in\\na Thai population. Pharmacogenet Genomics 2016;26:225-34.\\n10. \\nBloch KM et al. Pharmacogenetics of antiepileptic drug-induced hypersensitivity. Pharmacogenomics 2014;15:857-68.\\n11. \\nCheung YK et al. HLA-B alleles associated with severe cutaneous reactions to antiepileptic drugs in Han Chinese. Epilepsia 2013;54:1307-14.\\n12. \\nManuyakorn W et al. Phenobarbital-induced severe cutaneous adverse drug reactions are associated with CYP2C19*2 in Thai children. Pediatr Allergy Immunol\\n2013;24:299-303.\\n13. \\nHung SI et al. Common risk allele in aromatic antiepileptic-drug induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Han Chinese. Pharmacogenomics\\n2010;11:349-56.\\n14. \\nLocharernkul C et al. Carbamazepine and phenytoin induced Stevens-Johnson syndrome is associated with HLA-B*1502 allele in Thai population. Epilepsia 2008;49:2087-\\n91.\\n15. \\nSmPC Diphantoïne-Z.\\n16. \\nSmPC Dilantin (VS).\\nDate 23-05-2022\\nCYP2D6 IM: flecainide\\n1593\\nThe genetic variation reduces conversion of flecainide to inactive metabolites. This may increase the risk of side effects.\\nRecommendation:\\nIndications other than diagnosis of Brugada syndrome:\\n1. \\nreduce the dose to 75% of the standard dose and record an ECG and monitor the plasma concentration\\nProvocation test for diagnosis of Brugada syndrome:\\nNo action required.\\nAt a dose of 2.0 mg/kg body weight to a maximum of 150 mg, the response is better for patients with alleles that result in reduced activity.\\nAll 5 patients with these alleles and 20% of the patients with two fully active alleles exhibited a response within 30 minutes.\\nLiterature:\\n1. \\nPoh BH et al. Complete heart block secondary to flecainide toxicity: is it time for CYP2D6 genotype testing? Pediatrics 2020;146:e20192608.\\n2. \\nCalvo D et al. Time-dependent responses to provocative testing with flecainide in the diagnosis of Brugada syndrome. Heart Rhythm 2015;12:350-7.\\n3. \\nHu M et al. Effects of CYP2D6\\n10, CYP3A5\\n3, CYP1A2*1F, and ABCB1 C3435T polymorphisms on the pharmacokinetics of flecainide in healthy Chinese subjects. Drug\\nMetabol Drug Interact 2012;27:33-9.\\n4. \\nLim KS et al. Changes in the QTc interval after administration of flecainide acetate, with and without coadministered paroxetine, in relation to cytochrome P450 2D6\\ngenotype: data from an open-label, two-period, single-sequence crossover study in healthy Korean male subjects. Clin Ther 2010;32:659-66.\\n5. \\nLim KS et al. Pharmacokinetic interaction of flecainide and paroxetine in relation to the CYP2D6*10 allele in healthy Korean subjects. Br J Clin Pharmacol 2008;66:660-6.\\n6. \\nDoki K et al. Effect of CYP2D6 genotype on flecainide pharmacokinetics in Japanese patients with supraventricular tachyarrhythmia. Eur J Clin Pharmacol 2006;62:919-\\n26.\\nDate 12-09-2022\\nCYP2D6 PM: flecainide\\n1592\\nThe genetic variation reduces conversion of flecainide to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1. \\nreduce the dose to 50% of the standard dose and record an ECG and monitor the plasma concentration\\nLiterature:\\n1. \\nPalmiere C et al. Usefulness of post-mortem biochemistry in forensic pathology: illustrative case reports. Leg Med (Tokyo) 2012;14:27-35.\\n2. \\nTenneze L et al. Pharmacokinetics and electrocardiographic effects of a new controlled-release form of flecainide acetate: comparison with the standard form and\\ninfluence of the CYP2D6 polymorphism. Clin Pharmacol Ther 2002;72:112-22.\\n3. \\nFunck-Brentano C et al. Variable disposition kinetics and electrocardiographic effects of flecainide during repeated dosing in humans: contribution of genetic factors,\\ndose-dependent clearance, and interaction with amiodarone. Clin Pharmacol Ther 1994;55:256-69.\\n4. \\nGross AS et al. Polymorphic flecainide disposition under conditions of uncontrolled urine flow and pH. Eur J Clin Pharmacol 1991;40:155-62.\\n5. \\nGross AS et al. Stereoselective disposition of flecainide in relation to the sparteine/ debrisoquine metaboliser phenotype. Br J Clin Pharmacol 1989;28:555-66.\\n6. \\nMikus G et al. The influence of the sparteine/debrisoquin phenotype on the disposition of flecainide. Clin Pharmacol Ther 1989;45:562-7.\\nDate 12-09-2022', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 41}),\n",
       " Document(page_content='CYP2D6 UM: flecainide\\n1594\\nThe genetic variation increases conversion of flecainide to inactive metabolites. A higher dose is possibly required as a result.\\nRecommendation:\\nThere are no data about the pharmacokinetics and/or the effects of flecainide in UM.\\n1. \\nmonitor the plasma concentration as a precaution and record an ECG or select an alternative\\nExamples of anti-arrhythmic drugs that are not metabolised via CYP2D6 (or to a lesser extent) include sotalol, disopyramide, quinidine and amiodarone.\\nLiterature:\\n-\\nDate 12-09-2022\\nHLA-B*5701: flucloxacilline\\n4652\\nHLA-B*5701-positive patients have a 37-80-fold elevated risk of flucloxacillin-induced liver injury. However, the incidence is low (on average 1-2 per 1000 individuals; 7-14 per\\n1000 individuals older than 70 years).\\nregularly monitor the patient’s liver function\\nchoose an alternative if liver enzymes and/or bilirubin levels are elevated\\nLiterature:\\n1. \\nTeixeira M et al. Flucloxacillin-induced hepatotoxicity - association with HLA-B*5701. Rev Assoc Med Bras 2020;66:12-7.\\n2. \\nNicoletti P et al. Drug-induced liver injury due to flucloxacillin: relevance of multiple human leukocyte antigen alleles. Clin Pharmacol Ther 2019;106:245-53.\\n3. \\nVera JH et al. The safety of flucloxacillin in HIV-infected patients with positive HLA-B*5701 genotype. Aids 2013;27:484-5.\\n4. \\nPhilps EJ and Mallal SA. HLA-B*5701 and flucloxacillin associated drug-induced liver disease. Aids 2013;27:491-2.\\n5. \\nDaly AK et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nature Genetics 2009;41:816-9.\\n6. \\nSmPC Floxapen.\\nDate 15-11-2021\\nDPD AS 0: flucytosine\\n7209\\nA risk of life-threatening toxicity is increased by gene variation. A small proportion of flucytosine is converted to fluorouracil and patients with this gene variation are\\nintolerant even to small quantities of fluorouracil.\\navoid flucytosine\\nLiterature:\\n1. \\nMylan Healthcare LLC. Important risk information: Adapted recommendations for the use of flucytosine (Ancotil, solution for infusion 10 mg/ml, RVG 08533) in patients\\nwith dihydropyrimidine dehydrogenase (DPD) deficiency. Direct Healthcare Professional Communication (Orange Hand Letter) 05-06-20.\\n2. \\nSmPC Ancotil.\\nDate 15-11-2021\\nDPD AS 1,5: flucytosine\\n7210\\nA very low risk of severe toxicity is increased by gene variation. A small proportion of flucytosine is converted to fluorouracil. Patients with this gene variation are more likely\\nto have a reaction to fluorouracil, but generally tolerate low doses (50-75% of the standard fluorouracil dose).\\nBe alert to the occurrence of severe side effects, such as leukopaenia, neutropaenia, thrombocytopaenia and diarrhoea\\nIn the majority of cases, side effects of flucytosine occur in the first two to three weeks of the treatment. Flucytosine should be stopped if severe side effects occur.\\nLiterature:\\n1. \\nMylan Healthcare LLC. Important risk information: Adapted recommendations for the use of flucytosine (Ancotil, solution for infusion 10 mg/ml, RVG 08533) in patients\\nwith dihydropyrimidine dehydrogenase (DPD) deficiency. Direct Healthcare Professional Communication (Orange Hand Letter) 05-06-20.\\n2. \\nSmPC Ancotil.\\nDate 15-11-2021\\nDPD AS 1: flucytosine\\n7211\\nA very low risk of severe toxicity is increased by gene variation. A small proportion of flucytosine is converted to fluorouracil. Patients with this gene variation are more likely\\nto react to fluorouracil, but generally tolerate low doses (approximately 50% of the standard fluorouracil dose).\\nbe alert to the occurrence of severe side effects, such as leukopaenia, neutropaenia, thrombocytopaenia and diarrhoea\\nIn the majority of cases, side effects of flucytosine occur in the first two to three weeks of the treatment. Flucytosine should be stopped if severe side effects occur.\\nLiterature:\\n1. \\nMylan Healthcare LLC. Important risk information: Adapted recommendations for the use of flucytosine (Ancotil, solution for infusion 10 mg/ml, RVG 08533) in patients\\nwith dihydropyrimidine dehydrogenase (DPD) deficiency. Direct Healthcare Professional Communication (Orange Hand Letter) 05-06-20.\\n2. \\nSmPC Ancotil.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 42}),\n",
       " Document(page_content=\"Date 15-11-2021\\nDPD FENO: flucytosine\\n7208\\nA very low risk of severe toxicity is increased by gene variation. A small proportion of flucytosine is converted to fluorouracil. Patients with this gene variation are more likely\\nto have a reaction to fluorouracil, but generally tolerate low doses (15-50% of the standard fluorouracil dose).\\nbe alert to the occurrence of severe side effects, such as leukopaenia, neutropaenia, thrombocytopaenia and diarrhoea.\\nIn the majority of cases, side effects of flucytosine occur in the first two to three weeks of the treatment. Flucytosine should be stopped if severe side effects occur.\\nLiterature:\\n1. \\nMylan Healthcare LLC. Important risk information: Adapted recommendations for the use of flucytosine (Ancotil, solution for infusion 10 mg/ml, RVG 08533) in patients\\nwith dihydropyrimidine dehydrogenase (DPD) deficiency. Direct Healthcare Professional Communication (Orange Hand Letter) 05-06-20.\\n2. \\nSmPC Ancotil.\\nDate 15-11-2021\\nDPD AS 0: fluorouracil cutaan\\n6192\\nThe gene variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive metabolites means that the normal dose\\nis an overdose.\\navoid fluorouracil\\nNOTE: If a patient has two different genetic variations that lead to a non-functional DPD enzyme (e.g. *2A and *13), this recommendation only applies if the variations are\\non a different allele. If both variations are on the same allele, this patient actually has a gene activity score 1, for which no increased risk of severe, potentially fatal\\ntoxicity has been found with cutaneous use. These two situations can only be distinguished by determining the enzyme activity (phenotyping). This recommendation only\\napplies if the patient has virtually no enzyme activity.\\nLiterature:\\n1. \\nHenricks LM et al. Capecitabine-based treatment of a patient with a novel DPYD genotype and complete dihydropyrimidine dehydrogenase deficiency. Int J Cancer\\n2018;142:424-30.\\n2. \\nHenricks LM et al. Treatment algorithm for homozygous or compound heterozygous DPYD variant allele carriers with low-dose capecitabine. JCO Precis Oncol - published\\nonline 2017 Oct 6.\\n3. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-fluorouracil. J Gastrointest Cancer 2017;48:66-\\n69.\\n4. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-treated patients. Pharmacogenomics\\n2016;17:721-9.\\n5. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin Oncol 2016;34:227-34.\\n6. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the QUASAR2 study, systematic review, and meta-\\nanalysis. J Clin Oncol 2014; 32:1031-9.\\n7. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer.\\nClin Cancer Res 2011; 17:3455-68.\\n8. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-related toxicity in cancer patients. PLoS ONE\\n2008;3:e4003.\\n9. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection of dihydropyrimidine dehydrogenase\\ndeficiency. Cancer Lett 2007;249:271-82.\\n10. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-fluorouracil tolerance. Mol Cancer Ther\\n2006;5:2895-904.\\n11. \\nVan Kuilenburg AB et al. High prevalence of the IVS14 + 1G>A mutation in the dihydropyrimidine dehydrogenase gene of patients with severe 5-fluorouracil-associated\\ntoxicity. Pharmacogenetics 2002;12:555-8.\\n12. \\nRaida M et al. Prevalence of a common point mutation in the dihydropyrimidine dehydrogenase (DPD) gene within the 5'-splice donor site of intron 14 in patients with\\nsevere 5-fluorouracil (5-FU)-related toxicity compared with controls. Clin Cancer Res 2001;7:2832-9.\\n13. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-fluorouracil-associated toxicity:\\nidentification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n14. \\nJohnson MR et al. Life-threatening toxicity in a dihydropyrimidine dehydrogenase-deficient patient after treatment with topical 5-fluorouracil. Clin Cancer Res\\n1999;5:2006-11.\\n15. \\nSPC Efudix crème en Carac cream (VS).\\nDate 13-05-2019\\nDPD AS 1,5: fluorouracil/capecitabine\\n4894\\nThe gene variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive metabolites means that the normal dose\\nis an overdose.\\nStart with 50% of the standard dose or avoid fluorouracil and capecitabine.\\nAfter starting treatment, the dose should be adjusted based on toxicity and effectiveness. In a study involving 17 patients with genotype \\n1/2846T, the average dose after\\ntitration was 64% of the standard dose. For 51 patients with genotype \\n1/1236A, the average dose after titration was 74% of the standard dose. Tegafur is not an\\nalternative, as this is also metabolised by DPD.\\nLiterature:\\n1. \\nKleinjan JP et al. Tolerance-based capecitabine dose escalation after DPYD genotype-guided dosing in heterozygote DPYD variant carriers: a single-center observational\\nstudy. Anticancer Drugs 2019 Jan 8 [Epub ahead of print].\\n2. \\nLunenburg CATC et al. Diagnostic and therapeutic strategies for fluoropyrimidine treatment of patients carrying multiple DPYD variants. Genes (Basel) 2018;9:E585.\\n3. \\nLunenburg CATC et al. Standard fluoropyrimidine dosages in chemoradiation therapy result in an increased risk of severe toxicity in DPYD variant allele carriers. Eur J\\nCancer 2018;104:210-8.\\n4. \\nHenricks LM et al. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: a prospective safety analysis. Lancet Oncol\\n2018;19:1459-67 en persoonlijke communicatie (getitreerde dosis en mediane DPD-activiteit).\\n5. \\nMadi A et al. Pharmacogenetic analyses of 2183 patients with advanced colorectal cancer; potential role for common dihydropyrimidine dehydrogenase variants in\\ntoxicity to chemotherapy. Eur J Cancer 2018;102:31-9.\\n6. \\nMeulendijks D et al. Pretreatment serum uracil concentration as a predictor of severe and fatal fluoropyrimidine-associated toxicity. Br J Cancer 2017;116:1415-24.\\n7. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-treated patients. Pharmacogenomics\\n2016;17:721-9.\\n8. \\nLee AM et al. Association between DPYD c.1129-5923 C>G/hapB3 and severe toxicity to 5-fluorouracil-based chemotherapy in stage III colon cancer patients: NCCTG\\nN0147 (Alliance). Pharmacogenet Genomics 2016;26:133-7.\\n9. \\nMeulendijks D et al. Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe fluoropyrimidine-associated toxicity: a\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 43}),\n",
       " Document(page_content=\"systematic review and meta-analysis of individual patient data. Lancet Oncol 2015;16:1639-50.\\n10. \\nLee AM et al. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147). J Natl Cancer Inst 2014;106:dju298.\\n11. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the QUASAR2 study, systematic review, and meta-\\nanalysis. J Clin Oncol 2014; 32:1031-9.\\n12. \\nTerrazzino S et al. DPYD IVS14+1 G>A and 2846A>T genotyping for the prediction of severe fluoropyrimidine-related toxicity: a meta-analysis. Pharmacogenomics2013;\\n14:1255-72.\\n13. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer.\\nClin Cancer Res 2011; 17:3455-68.\\n14. \\nKristensen MH et al. Variants in the dihydropyrimidine dehydrogenase, methylenetetrahydrofolate reductase and thymidylate synthase genes predict early toxicity of 5-\\nfluorouracil in colorectal cancer patients. J Int Med Res 2010; 38:870-83.\\n15. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-related toxicity in cancer patients. PLoS ONE\\n2008;3:e4003.\\n16. \\nCapitain O et al. The influence of fluorouracil outcome parameters on tolerance and efficacy in patients with advanced colorectal cancer. Pharmacogenomics J\\n2008;8:256-67.\\n17. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection of dihydropyrimidine dehydrogenase\\ndeficiency. Cancer Lett 2007;249:271-82.\\n18. \\nCho HJ et al. Thymidylate synthase (TYMS) and dihydropyrimidine dehydrogenase (DPYD) polymorphisms in the Korean population for prediction of 5-fluorouracil-\\nassociated toxicity. Ther Drug Monit 2007;29:190-6.\\n19. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-fluorouracil tolerance. Mol Cancer Ther\\n2006;5:2895-904.\\n20. \\nYamaguchi K et al. Germline mutation of dihydropyrimidine dehydrogenese gene among a Japanese population in relation to toxicity to 5-fluorouracil. Jpn J Cancer Res\\n2001;92:337-42.\\n21. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-fluorouracil-associated toxicity:\\nidentification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n22. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS) en Xeloda (VS).\\nDate 13-05-2019\\nDPD AS 1: fluorouracil/capecitabine\\n2552\\nThe gene variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive metabolites means that the normal dose\\nis an overdose.\\nStart with 50% of the standard dose or avoid fluorouracil and capecitabine.\\nAdjustment of the subsequent dose should be guided by toxicity and effectiveness. However, in one study involving 17 patients with gene activity 1, the average dose\\nafter titration was 57% of the standard dose.\\nTegafur is not an alternative, as this is also metabolised by DPD.\\nLiterature:\\n1. \\nKleinjan JP et al. Tolerance-based capecitabine dose escalation after DPYD genotype-guided dosing in heterozygote DPYD variant carriers: a single-center observational\\nstudy. Anticancer Drugs 2019 Jan 8 [Epub ahead of print].\\n2. \\nHenricks LM et al. Effectiveness and safety of reduced-dose fluoropyrimidine therapy in patients carrying the DPYD*2A variant: a matched pair analysis. Int J Cancer\\n2018 Nov 28 [Epub ahead of print].\\n3. \\nLunenburg CATC et al. Standard fluoropyrimidine dosages in chemoradiation therapy result in an increased risk of severe toxicity in DPYD variant allele carriers. Eur J\\nCancer 2018;104:210-8.\\n4. \\nHenricks LM et al. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: a prospective safety analysis. Lancet Oncol\\n2018;19:1459-67 en persoonlijke communicatie (getitreerde dosis en mediane DPD-activiteit).\\n5. \\nMadi A et al. Pharmacogenetic analyses of 2183 patients with advanced colorectal cancer; potential role for common dihydropyrimidine dehydrogenase variants in\\ntoxicity to chemotherapy. Eur J Cancer 2018;102:31-9.\\n6. \\nMeulendijks D et al. Pretreatment serum uracil concentration as a predictor of severe and fatal fluoropyrimidine-associated toxicity. Br J Cancer 2017;116:1415-24.\\n7. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-fluorouracil. J Gastrointest Cancer 2017;48:66-\\n69.\\n8. \\nMeulendijks D et al. Patients homozygous for DPYD c.1129-5923C>G/ haplotype B3 have partial DPD deficiency and require a dose reduction when treated with\\nfluoropyrimidines. Cancer Chemother Pharmacol 2016;78:875-80.\\n9. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-treated patients. Pharmacogenomics\\n2016;17:721-9.\\n10. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin Oncol 2016;34:227-34.\\n11. \\nMeulendijks D et al. Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe fluoropyrimidine-associated toxicity: a\\nsystematic review and meta-analysis of individual patient data. Lancet Oncol 2015;16:1639-50.\\n12. \\nLee AM et al. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147). J Natl Cancer Inst 2014;106:dju298.\\n13. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the QUASAR2 study, systematic review, and meta-\\nanalysis. J Clin Oncol 2014; 32:1031-9.\\n14. \\nTerrazzino S et al. DPYD IVS14+1 G>A and 2846A>T genotyping for the prediction of severe fluoropyrimidine-related toxicity: a meta-analysis. Pharmacogenomics\\n2013;14:1255-72.\\n15. \\nMagnani E et al. Fluoropyrimidine toxicity in patients with dihydropyrimidine dehydrogenase splice site variant: the need for further revision of dose and schedule. Intern\\nEmerg Med 2013; 8:417-23.\\n16. \\nVulsteke C et al. Genetic variability in the multidrug resistance associated protein-1 (ABCC1/MRP1) predicts hematological toxicity in breast cancer patients receiving\\n(neo-)adjuvant chemotherapy with 5-fluorouracil, epirubicin and cyclophosphamide (FEC). Ann Oncol 2013; 24:1513-25.\\n17. \\nvan Kuilenburg AB et al. Evaluation of 5-fluorouracil pharmacokinetics in cancer patients with a c.1905+1 G>A mutation in DPYD by means of a Bayesian limited\\nsampling strategy. Clin Pharmacokinet 2012;51:163-74.\\n18. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer.\\nClin Cancer Res 2011; 17:3455-68.\\n19. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-related toxicity in cancer patients. PLoS ONE\\n2008;3:e4003.\\n20. \\nCapitain O et al. The influence of fluorouracil outcome parameters on tolerance and efficacy in patients with advanced colorectal cancer. Pharmacogenomics J\\n2008;8:256-67.\\n21. \\nSulzyc-Bielicka V et al. 5-Fluorouracil toxicity-attributable IVS14 + 1G > A mutation of the dihydropyrimidine dehydrogenase gene in Polish colorectal cancer patients.\\nPharmacol Rep 2008;60:238-42.\\n22. \\nSchwab M et al. Role of genetic and nongenetic factors for fluorouracil treatment-related severe toxicity: a prospective clinical trial by the German 5-FU Toxicity Study\\nGroup. J Clin Oncol 2008;26:2131-8.\\n23. \\nMercier C et al. Prospective phenotypic screening for DPD deficiency prior to 5-FU administration: Decrease in toxicity, not in efficacy. J Clin Oncol 2008;26(May 20\\nsuppl):abstr 14556. (meeting abstract)\\n24. \\nJatoi A et al. Paclitaxel, carboplatin, 5-fluorouracil, and radiation for locally advanced esophageal cancer: phase II results of preliminary pharmacologic and molecular\\nefforts to mitigate toxicity and predict outcomes: North Central Cancer Treatment Group (N0044). Am J Clin Oncol 2007;30:507-13.\\n25. \\nMagné N et al. Dihydropyrimidine dehydrogenase activity and the IVS14+1G>A mutation in patients developing 5FU-related toxicity. Br J Clin Pharmacol 2007;64:237-\\n40.\\n26. \\nSaif MW et al. Dihydropyrimidine dehydrogenase deficiency (GPD) in GI malignancies: experience of 4-years. Pak J Med Sci Q 2007;23:832-9.\\n27. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection of dihydropyrimidine dehydrogenase\\ndeficiency. Cancer Lett 2007;249:271-82.\\n28. \\nCho HJ et al. Thymidylate synthase (TYMS) and dihydropyrimidine dehydrogenase (DPYD) polymorphisms in the Korean population for prediction of 5-fluorouracil-\\nassociated toxicity. Ther Drug Monit 2007;29:190-6.\\n29. \\nSalgado J et al. Polymorphisms in the thymidylate synthase and dihydropyrimidine dehydrogenase genes predict response and toxicity to capecitabine-raltitrexed in\\ncolorectal cancer. Oncol Rep 2007;17:325-8.\\n30. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-fluorouracil tolerance. Mol Cancer Ther\\n2006;5:2895-904.\\n31. \\nLargillier R et al. Pharmacogenetics of capecitabine in advanced breast cancer patients. Clin Cancer Res 2006;12:5496-502.\\n32. \\nSalgueiro N et al. Mutations in exon 14 of dihydropyrimidine dehydrogenase and 5-Fluorouracil toxicity in Portuguese colorectal cancer patients. Genet Med 2004;6:102-\\n7.\\n33. \\nVan Kuilenburg AB et al. High prevalence of the IVS14 + 1G>A mutation in the dihydropyrimidine dehydrogenase gene of patients with severe 5-fluorouracil-associated\\ntoxicity. Pharmacogenetics 2002;12:555-8.\\n34. \\nRaida M et al. Prevalence of a common point mutation in the dihydropyrimidine dehydrogenase (DPD) gene within the 5'-splice donor site of intron 14 in patients with\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 44}),\n",
       " Document(page_content=\"severe 5-fluorouracil (5-FU)- related toxicity compared with controls. Clin Cancer Res 2001;7:2832-9.\\n35. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-fluorouracil-associated toxicity:\\nidentification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n36. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS) en Xeloda (VS).\\nDate 13-05-2019\\nDPD FENO: fluorouracil/capecitabine\\n4893\\nThe gene variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive metabolites means that the normal dose\\nis an overdose.\\nIt is not possible to recommend a dose adjustment for this patient based on the genotype only.\\ndetermine the residual DPD activity in mononuclear cells from peripheral blood and adjust the initial dose based on phenotype and genotype, or avoid fluorouracil and\\ncapecitabine.\\nTegafur is not an alternative, as this is also metabolised by DPD.\\nLiterature:\\n1. \\nLunenburg CATC et al. Diagnostic and therapeutic strategies for fluoropyrimidine treatment of patients carrying multiple DPYD variants. Genes (Basel) 2018;9:E585.\\n2. \\nLunenburg CATC et al. Standard fluoropyrimidine dosages in chemoradiation therapy result in an increased risk of severe toxicity in DPYD variant allele carriers. Eur J\\nCancer 2018;104:210-8.\\n3. \\nHenricks LM et al. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: a prospective safety analysis. Lancet Oncol\\n2018;19:1459-67 en persoonlijke communicatie (getitreerde dosis en mediane DPD-activiteit).\\n4. \\nHenricks LM et al. Treatment algorithm for homozygous or compound heterozygous DPYD variant allele carriers with low-dose capecitabine. JCO Precis Oncol 2017 -\\npublished online Oct 6.\\n5. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-fluorouracil. J Gastrointest Cancer 2017;48:66-\\n69.\\n6. \\nMeulendijks D et al. Patients homozygous for DPYD c.1129-5923C>G/ haplotype B3 have partial DPD deficiency and require a dose reduction when treated with\\nfluoropyrimidines. Cancer Chemother Pharmacol 2016;78:875-80.\\n7. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-treated patients. Pharmacogenomics\\n2016;17:721-9.\\n8. \\nLee AM et al. Association between DPYD c.1129-5923 C>G/hapB3 and severe toxicity to 5-fluorouracil-based chemotherapy in stage III colon cancer patients: NCCTG\\nN0147 (Alliance). Pharmacogenet Genomics 2016;26:133-7.\\n9. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin Oncol 2016;34:227-34.\\n10. \\nMeulendijks D et al. Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe fluoropyrimidine-associated toxicity: a\\nsystematic review and meta-analysis of individual patient data. Lancet Oncol 2015;16:1639-50.\\n11. \\nLee AM et al. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147). J Natl Cancer Inst 2014;106:dju298.\\n12. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer.\\nClin Cancer Res 2011; 17:3455-68.\\n13. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection of dihydropyrimidine dehydrogenase\\ndeficiency. Cancer Lett 2007;249:271-82.\\n14. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-fluorouracil tolerance. Mol Cancer Ther\\n2006;5:2895-904.\\n15. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS) en Xeloda (VS).\\nDate 13-05-2019\\nDPD AS 0: fluorouracil/capecitabine,systemisch\\n2551\\nThe gene variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive metabolites means that the standard\\ndose is a more than 100-fold overdose.\\nAvoid fluorouracil and capecitabine\\nTegafur is not an alternative, as this is also metabolised by DPD.\\nIf it is not possible to avoid fluorouracil and capecitabine: determine the residual DPD activity in mononuclear cells from peripheral blood and adjust the initial dose\\naccordingly.\\nA patient with 0.5% of the normal DPD activity tolerated 0.8% of the standard dose (150 mg capecitabine every 5 days). A patient with undetectable DPD activity\\ntolerated 0.43% of the standard dose (150 mg capecitabine every 5 days with every third dose skipped)\\nLiterature:\\n1. \\nHenricks LM et al. Capecitabine-based treatment of a patient with a novel DPYD genotype and complete dihydropyrimidine dehydrogenase deficiency. Int J Cancer\\n2018;142:424-30.\\n2. \\nHenricks LM et al. Treatment algorithm for homozygous or compound heterozygous DPYD variant allele carriers with low-dose capecitabine. JCO Precis Oncol - published\\nonline 2017 Oct 6.\\n3. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-fluorouracil. J Gastrointest Cancer 2017;48:66-\\n69.\\n4. \\nMeulendijks D et al. Patients homozygous for DPYD c.1129-5923C>G/ haplotype B3 have partial DPD deficiency and require a dose reduction when treated with\\nfluoropyrimidines. Cancer Chemother Pharmacol 2016;78:875-80.\\n5. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-treated patients. Pharmacogenomics\\n2016;17:721-9.\\n6. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin Oncol 2016;34:227-34.\\n7. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the QUASAR2 study, systematic review, and meta-\\nanalysis. J Clin Oncol 2014; 32:1031-9.\\n8. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer.\\nClin Cancer Res 2011; 17:3455-68.\\n9. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-related toxicity in cancer patients. PLoS ONE\\n2008;3:e4003.\\n10. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection of dihydropyrimidine dehydrogenase\\ndeficiency. Cancer Lett 2007;249:271-82.\\n11. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-fluorouracil tolerance. Mol Cancer Ther\\n2006;5:2895-904.\\n12. \\nVan Kuilenburg AB et al. High prevalence of the IVS14 + 1G>A mutation in the dihydropyrimidine dehydrogenase gene of patients with severe 5-fluorouracil-associated\\ntoxicity. Pharmacogenetics 2002;12:555-8.\\n13. \\nRaida M et al. Prevalence of a common point mutation in the dihydropyrimidine dehydrogenase (DPD) gene within the 5'-splice donor site of intron 14 in patients with\\nsevere 5-fluorouracil (5-FU)- related toxicity compared with controls. Clin Cancer Res 2001;7:2832-9.\\n14. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-fluorouracil-associated toxicity:\\nidentification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n15. \\nJohnson MR et al. Life-threatening toxicity in a dihydropyrimidine dehydrogenase-deficient patient after treatment with topical 5-fluorouracil. Clin Cancer Res\\n1999;5:2006-11.\\n16. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS), Xeloda (VS) en Carac cream (VS).\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 45}),\n",
       " Document(page_content='Date 13-05-2019\\nCYP2D6 IM: fluoxetine\\n5997\\nNO action is needed for this gene-drug interaction.\\nThe ratio of fluoxetine/norfluoxetine increases as a result of the reduced activity of CYP2D6. However, this does not affect the sum of the plasma concentrations of the active\\nsubstances (fluoxetine and norfluoxetine). There is no effect on adverse events or response.\\nLiterature:\\n1. \\nGassó P et al. Effect of CYP2D6, CYP2C9 and ABCB1 genotypes on fluoxetine plasma concentrations and clinical improvement in children and adolescent patients.\\nPharmacogenomics J 2014;14:457-62.\\n2. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n3. \\nScordo MG et al. Influence of CYP2C9, 2C19 and 2D6 genetic polymorphisms on the steady-state plasma concentrations of the enantiomers of fluoxetine and\\nnorfluoxetine. Basic Clin Pharmacol Toxicol 2005;97:296-301.\\n4. \\nLLerena A et al. Effect of CYP2D6 and CYP2C9 genotypes on fluoxetine and norfluoxetine plasma concentrations during steady-state conditions. Eur J Clin Pharmacol\\n2004;59:869-73.\\nDate 14-05-2018\\nCYP2D6 PM: fluoxetine\\n5996\\nNO action is needed for this gene-drug interaction.\\nThe ratio of fluoxetine/norfluoxetine increases as a result of the reduced activity of CYP2D6. However, this does not affect the sum of the plasma concentrations of the active\\nsubstances (fluoxetine and norfluoxetine). There is insufficient evidence to support an effect on adverse events or response.\\nLiterature:\\n1. \\nGassó P et al. Effect of CYP2D6, CYP2C9 and ABCB1 genotypes on fluoxetine plasma concentrations and clinical improvement in children and adolescent patients.\\nPharmacogenomics J 2014;14:457-62.\\n2. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n3. \\nScordo MG et al. Influence of CYP2C9, 2C19 and 2D6 genetic polymorphisms on the steady-state plasma concentrations of the enantiomers of fluoxetine and\\nnorfluoxetine. Basic Clin Pharmacol Toxicol 2005;97:296-301.\\n4. \\nLLerena A et al. Effect of CYP2D6 and CYP2C9 genotypes on fluoxetine and norfluoxetine plasma concentrations during steady-state conditions. Eur J Clin Pharmacol\\n2004;59:869-73.\\n5. \\nRoberts RL et al. No evidence of increased adverse drug reactions in cytochrome P450 CYP2D6 poor metabolizers treated with fluoxetine or nortriptyline. Hum\\nPsychopharmacol 2004;19:17-23.\\n6. \\nPerucca E et al. Fluoxetine-induced movement disorders and deficient CYP2D6 enzyme activity. Mov Disord 1997;12:624-5.\\n7. \\nSPC Prozac, USA, 30-01-09.\\nDate 14-05-2018\\nCYP2D6 UM: fluoxetine\\n5998\\nNO action is needed for this gene-drug interaction.\\nThe ratio of fluoxetine/norfluoxetine decreases as a result of the increased activity of CYP2D6. However, this does not affect the sum of the plasma concentrations of the active\\nsubstances (fluoxetine and norfluoxetine). There is no effect on adverse events or response.\\nLiterature:\\n1. \\nGassó P et al. Effect of CYP2D6, CYP2C9 and ABCB1 genotypes on fluoxetine plasma concentrations and clinical improvement in children and adolescent patients.\\nPharmacogenomics J 2014;14:457-62.\\n2. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n3. \\nScordo MG et al. Influence of CYP2C9, 2C19 and 2D6 genetic polymorphisms on the steady-state plasma concentrations of the enantiomers of fluoxetine and\\nnorfluoxetine. Basic Clin Pharmacol Toxicol 2005;97:296-301.\\n4. \\nLLerena A et al. Effect of CYP2D6 and CYP2C9 genotypes on fluoxetine and norfluoxetine plasma concentrations during steady-state conditions. Eur J Clin Pharmacol\\n2004;59:869-73.\\nDate 14-05-2018\\nCYP2D6 IM: flupentixol\\n1532\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nWaade RB et al. Impact of CYP2D6 on serum concentrations of flupentixol, haloperidol, perphenazine and zuclopenthixol. Br J Clin Pharmacol 2021;87:2228-35.\\nDate 13-09-2021\\nCYP2D6 PM: flupentixol\\n1534\\nThis is NOT a gene-drug interaction.\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 46}),\n",
       " Document(page_content='1. \\nWaade RB et al. Impact of CYP2D6 on serum concentrations of flupentixol, haloperidol, perphenazine and zuclopenthixol. Br J Clin Pharmacol 2021;87:2228-35.\\nDate 13-09-2021\\nCYP2D6 UM: flupentixol\\n1533\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nWaade RB et al. Impact of CYP2D6 on serum concentrations of flupentixol, haloperidol, perphenazine and zuclopenthixol. Br J Clin Pharmacol 2021;87:2228-35.\\nDate 13-09-2021\\nSLCO1B1 521CC: fluvastatine\\n4060\\nNO action is required for this gene-drug interaction.\\nThe gene variation increases the plasma concentration of fluvastatin, but there is insufficient evidence to prove an effect on efficacy or side effects.\\nLiterature:\\n1. \\nHirvensalo P et al. Enantiospecific pharmacogenomics of fluvastatin. Clin Pharmacol Ther 2019;106:668-80.\\n2. \\nMori D et al. Effect of OATP1B1 genotypes on plasma concentrations of endogenous OATP1B1 substrates and drugs, and their association in healthy volunteers. Drug\\nMetab Pharmacokinet 2019;34:78-86.\\n3. \\nXiang Q et al. The association between the SLCO1B1, apolipoprotein E, and CYP2C9 genes and lipid response to fluvastatin: a meta-analysis. Pharmacogenet Genomics\\n2018;28:261-7.\\n4. \\nMeyer zu Schwabedissen HE et al. Function-impairing polymorphisms of the hepatic uptake transporter SLCO1B1 modify the therapeutic efficacy of statins in a\\npopulation-based cohort. Pharmacogenet Genomics 2015;25:8-18.\\n5. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with increased cholesterol synthesis rate.\\nPharmacogenet Genomics 2008;18:921-6.\\n6. \\nCouvert P et al. Association between a frequent allele of the gene encoding OATP1B1 and enhanced LDL-lowering response to fluvastatin therapy. Pharmacogenomics\\n2008;9:1217-27.\\n7. \\nSinger JB et al. Genetic analysis of fluvastatin response and dyslipidemia in renal transplant recipients. J Lipid Res 2007;48:2072-8.\\n8. \\nNiemi M et al. SLCO1B1 polymorphism and sex affect the pharmacokinetics of pravastatin but not fluvastatin. Clin Pharmacol Ther 2006;80:356-66.\\n9. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J 2005;5:352-8.\\nDate 18-05-2020\\nSLCO1B1 521TC: fluvastatine\\n4059\\nNO action is required for this gene-drug interaction.\\nThe gene variation increases the plasma concentration of fluvastatin, but there is insufficient evidence to prove an effect on efficacy or side effects.\\nLiterature:\\n1. \\nHirvensalo P et al. Enantiospecific pharmacogenomics of fluvastatin. Clin Pharmacol Ther 2019;106:668-80.\\n2. \\nMori D et al. Effect of OATP1B1 genotypes on plasma concentrations of endogenous OATP1B1 substrates and drugs, and their association in healthy volunteers. Drug\\nMetab Pharmacokinet 2019;34:78-86.\\n3. \\nXiang Q et al. The association between the SLCO1B1, apolipoprotein E, and CYP2C9 genes and lipid response to fluvastatin: a meta-analysis. Pharmacogenet Genomics\\n2018;28:261-7.\\n4. \\nMeyer zu Schwabedissen HE et al. Function-impairing polymorphisms of the hepatic uptake transporter SLCO1B1 modify the therapeutic efficacy of statins in a\\npopulation-based cohort. Pharmacogenet Genomics 2015;25:8-18.\\n5. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with increased cholesterol synthesis rate.\\nPharmacogenet Genomics 2008;18:921-6.\\n6. \\nCouvert P et al. Association between a frequent allele of the gene encoding OATP1B1 and enhanced LDL-lowering response to fluvastatin therapy. Pharmacogenomics\\n2008;9:1217-27.\\n7. \\nSinger JB et al. Genetic analysis of fluvastatin response and dyslipidemia in renal transplant recipients. J Lipid Res 2007;48:2072-8.\\n8. \\nNiemi M et al. SLCO1B1 polymorphism and sex affect the pharmacokinetics of pravastatin but not fluvastatin. Clin Pharmacol Ther 2006;80:356-66.\\n9. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J 2005;5:352-8.\\nDate 18-05-2020\\nCYP2C19 IM: fluvoxamine\\n3510\\nThis is NOT a gene-drug interaction\\nLiterature:\\n1. \\nJan MW et al. Pharmacokinetics of fluvoxamine in relation to CYP2C19 phenotype and genotype. Drug Metabol Drug Interact 2002;19:1-11.\\n2. \\nSpigset O et al. The major fluvoxamine metabolite in urine is formed by CYP2D6. Eur J Clin Pharmacol 2001;57:653-8.\\n3. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J Clin Pharmacol 1997;52:129-33.\\nDate 14-05-2018\\nCYP2C19 PM: fluvoxamine\\n3509\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 47}),\n",
       " Document(page_content='Literature:\\n1. \\nJan MW et al. Pharmacokinetics of fluvoxamine in relation to CYP2C19 phenotype and genotype. Drug Metabol Drug Interact 2002;19:1-11.\\n2. \\nSpigset O et al. The major fluvoxamine metabolite in urine is formed by CYP2D6. Eur J Clin Pharmacol 2001;57:653-8.\\n3. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J Clin Pharmacol 1997;52:129-33.\\nDate 14-05-2018\\nCYP2C19 UM: fluvoxamine\\n3511\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nJan MW et al. Pharmacokinetics of fluvoxamine in relation to CYP2C19 phenotype and genotype. Drug Metabol Drug Interact 2002;19:1-11.\\n2. \\nSpigset O et al. The major fluvoxamine metabolite in urine is formed by CYP2D6. Eur J Clin Pharmacol 2001;57:653-8.\\n3. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J Clin Pharmacol 1997;52:129-33.\\nDate 14-05-2018\\nCYP2D6 IM: fluvoxamine\\n5994\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of fluvoxamine can increase as a result of the reduced activity of CYP2D6.\\nHowever, there is insufficient scientific substantiation of an increase in the risk of side effects.\\nLiterature:\\n1. \\nSuzuki Y et al. CYP2D6 genotype and smoking influence fluvoxamine steady-state concentration in Japanese psychiatric patients: lessons for genotype-phenotype\\nassociation study design in translational pharmacogenetics. J Psychopharmacol 2011;25:908-14.\\n2. \\nSugahara H et al. Effect of smoking and CYP2D6 polymorphisms on the extent of fluvoxamine-alprazolam interaction in patients with psychosomatic disease. Eur J Clin\\nPharmacol 2009;65:699-704.\\n3. \\nSuzuki Y et al. Polymorphisms in the 5-hydroxytryptamine 2A receptor and cytochromeP4502D6 genes synergistically predict fluvoxamine-induced side effects in\\nJapanese depressed patients. Neuropsychopharmacology 2006;31:825-31.\\n4. \\nGerstenberg G et al. Effects of the CYP2D6 genotype and cigarette smoking on the steady-state plasma concentrations of fluvoxamine and its major metabolite\\nfluvoxamino acid in Japanese depressed patients. Ther Drug Monit 2003;25:463-8.\\n5. \\nGerstenberg G et al. Relationship between clinical effects of fluvoxamine and the steady-state plasma concentrations of fluvoxamine and its major metabolite fluvoxamino\\nacid in Japanese depressed patients. Psychopharmacology (Berl) 2003;167:443-8.\\n6. \\nOhara K et al. CYP2D6*10 alleles do not determine plasma fluvoxamine concentration/dose ratio in Japanese subjects. Eur J Clin Pharmacol 2003;58:659-61.\\nDate 14-05-2018\\nCYP2D6 PM: fluvoxamine\\n5993\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of fluvoxamine can increase as a result of the reduced activity of CYP2D6.\\nHowever, there is no evidence to substantiate an increase in the risk of adverse events.\\nLiterature:\\n1. \\nChristensen M et al. Low daily 10-mg and 20-mg doses of fluvoxamine inhibit the metabolism of both caffeine (cytochrome P4501A2) and omeprazole (cytochrome\\nP4502C19). Clin Pharmacol Ther 2002;71:141-52.\\n2. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J Clin Pharmacol 1997;52:129-33.\\n3. \\nCarrillo JA et al. Disposition of fluvoxamine in humans is determined by the polymorphic CYP2D6 and also by the CYP1A2 activity. Clin Pharmacol Ther 1996;60:183-90.\\n4. \\nSPC’s Fevarin en Luvox (VS).\\nDate 14-05-2018\\nCYP2D6 UM: fluvoxamine\\n5995\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of fluvoxamine can decrease as a result of the increased activity of CYP2D6.\\nHowever, there is no scientific substantation of a reduced effectiveness.\\nLiterature:\\n1. \\nSuzuki Y et al. CYP2D6 genotype and smoking influence fluvoxamine steady-state concentration in Japanese psychiatric patients: lessons for genotype-phenotype\\nassociation study design in translational pharmacogenetics. J Psychopharmacol 2011;25:908-14.\\n2. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J Clin Pharmacol 1997;52:129-33.\\n3. \\nCarrillo JA et al. Disposition of fluvoxamine in humans is determined by the polymorphic CYP2D6 and also by the CYP1A2 activity. Clin Pharmacol Ther 1996;60:183-90.\\nDate 14-05-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 48}),\n",
       " Document(page_content=\"MTHFR 677CT: foliumzuur\\n7204\\nNO action is required for this gene-drug interaction.\\nThe gene variation has no effect on the treatment with folic acid. Treatment with folic acid decreases the reduction of folate concentrations caused by the gene variation.\\nLiterature:\\n1. \\nSong Chen MY et al. Defining the plasma folate concentration associated with the red blood cell folate concentration threshold for optimal neural tube defects prevention:\\na population-based, randomized trial of folic acid supplementation. Am J Clin Nutr 2019;109:1452-61.\\n2. \\nJin H et al. An evidence-based approach to globally assess the covariate-dependent effect of the MTHFR single nucleotide polymorphism rs1801133 on blood\\nhomocysteine: a systematic review and meta-analysis. Am J Clin Nutr 2018;107:817-25.\\n3. \\nDu B et al. Genetic polymorphisms of key enzymes in folate metabolism affect the efficacy of folate therapy in patients with hyperhomocysteinaemia. Br J Nutr\\n2018;119:887-95.\\n4. \\nHuang X et al. MTHFR gene and serum folate interaction on serum homocysteine lowering: prospect for precision folic acid treatment. Arterioscler Thromb Vasc Biol\\n2018;38:679-85.\\n5. \\nden Dekker HT et al. Maternal folic acid use during pregnancy, methylenatetrahydrofolate reductase gene polymorphism, and child's lung function and asthma. Clin Exp\\nAllergy 2018;48:175-85.\\n6. \\nShao W et al. Association between MTHFR C677T polymorphism and methotrexate treatment outcome in rheumatoid arthritis patients: a systematic review and meta-\\nanalysis. Genet Test Mol Biomarkers 2017;21:275-85.\\n7. \\nHuo Y et al. Efficacy of folic acid therapy in primary prevention of stroke among adults with hypertension in China: the CSPPT randomized clinical trial. JAMA\\n2015;313:1325-35.\\n8. \\nKondo A et al. C677T mutation in methylenetetrahydrofolate reductase gene and neural tube defects: should Japanese women undergo gene screening before pregnancy?\\nCongenit Anom (Kyoto) 2014;54:30-4.\\n9. \\nCrider KS et al. MTHFR 677C->T genotype is associated with folate and homocysteine concentrations in a large, population-based, double-blind trial of folic acid\\nsupplementation. Am J Clin Nutr 2011;93:1365-72.\\nDate 07-06-2021\\nMTHFR 677TT: foliumzuur\\n7205\\nNO action is required for this gene-drug interaction.\\nThe gene variation either has no effect or a positive effect on the treatment with folic acid. Treatment with folic acid decreases the reduction of folate concentrations caused\\nby the gene variation.\\nLiterature:\\n1. \\nSong Chen MY et al. Defining the plasma folate concentration associated with the red blood cell folate concentration threshold for optimal neural tube defects prevention:\\na population-based, randomized trial of folic acid supplementation. Am J Clin Nutr 2019;109:1452-61.\\n2. \\nJin H et al. An evidence-based approach to globally assess the covariate-dependent effect of the MTHFR single nucleotide polymorphism rs1801133 on blood\\nhomocysteine: a systematic review and meta-analysis. Am J Clin Nutr 2018;107:817-25.\\n3. \\nDu B et al. Genetic polymorphisms of key enzymes in folate metabolism affect the efficacy of folate therapy in patients with hyperhomocysteinaemia. Br J Nutr\\n2018;119:887-95.\\n4. \\nHuang X et al. MTHFR gene and serum folate interaction on serum homocysteine lowering: prospect for precision folic acid treatment. Arterioscler Thromb Vasc Biol\\n2018;38:679-85.\\n5. \\nden Dekker HT et al. Maternal folic acid use during pregnancy, methylenatetrahydrofolate reductase gene polymorphism, and child's lung function and asthma. Clin Exp\\nAllergy 2018;48:175-85.\\n6. \\nShao W et al. Association between MTHFR C677T polymorphism and methotrexate treatment outcome in rheumatoid arthritis patients: a systematic review and meta-\\nanalysis. Genet Test Mol Biomarkers 2017;21:275-85.\\n7. \\nColson NJ et al. The impact of MTHFR 677 C/T genotypes on folate status markers: a meta-analysis of folic acid intervention studies. Eur J Nutr 2017;56:247-60.\\n8. \\nHuo Y et al. Efficacy of folic acid therapy in primary prevention of stroke among adults with hypertension in China: the CSPPT randomized clinical trial. JAMA\\n2015;313:1325-35.\\n9. \\nKondo A et al. C677T mutation in methylenetetrahydrofolate reductase gene and neural tube defects: should Japanese women undergo gene screening before pregnancy?\\nCongenit Anom (Kyoto) 2014;54:30-4.\\n10. \\nCrider KS et al. MTHFR 677C->T genotype is associated with folate and homocysteine concentrations in a large, population-based, double-blind trial of folic acid\\nsupplementation. Am J Clin Nutr 2011;93:1365-72.\\nDate 07-06-2021\\nCYP2D6 IM: gefitinib\\n4871\\nNO action is needed for this gene-drug interaction.\\nSide effects can occur more frequently, as the gene variation increases the gefitinib plasma concentration. However, the side effects are reversible and manageable, to an\\nextent that adjustment of the therapy in advance is not necessary.\\nLiterature:\\n1. \\nHirose T et al. Association of pharmacokinetics and pharmacogenomics with safety and efficacy of gefitinib in patients with EGFR mutation positive advanced non-small\\ncell lung cancer. Lung Cancer 2016;93:69-76.\\n2. \\nSugiyama E et al. Impact of single nucleotide polymorphisms on severe hepatotoxicity induced by EGFR tyrosine kinase inhibitors in patients with non-small cell lung\\ncancer harboring EGFR mutations. Lung Cancer 2015;90:307-13.\\n3. \\nKobayashi H et al. Relationship among gefitinib exposure, polymorphisms of its metabolizing enzymes and transporters, and side effects in Japanese patients with non-\\nsmall-cell lung cancer. Clin Lung Cancer 2015;16:274-81.\\n4. \\nTakimoto T et al. Polymorphisms of CYP2D6 gene and gefitinib-induced hepatotoxicity. Clinical Lung Cancer 2013;14:502-7.\\n5. \\nSuzumura T et al. Reduced CYP2D6 function is associated with gefitinib-induced rash in patients with non-small cell lung cancer. BMC Cancer 2012;12:568.\\n6. \\nChhun S et al. Gefitinib-phenytoin interaction is not correlated with the C-erythromycin breath test in healthy male volunteers. Br J Clin Pharmacol 2009;68:226-37.\\nDate 19-11-2018\\nCYP2D6 PM: gefitinib\\n4872\\nNO action is needed for this gene-drug interaction.\\nThe gefitinib plasma concentration may increase due to reduced CYP2D6 activity. However, there is no evidence to suggest that side effects increase to an extent that\\nadjustment of therapy is needed.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 49}),\n",
       " Document(page_content='Literature:\\n1. \\nChhun S et al. Gefitinib-phenytoin interaction is not correlated with the C-erythromycin breath test in healthy male volunteers. Br J Clin Pharmacol 2009;68:226-37.\\n2. \\nSwaisland HC et al. Exploring the relationship between expression of cytochrome P450 enzymes and gefitinib pharmacokinetics. Clin Pharmacokinet 2006;45:633-44.\\n3. \\nSPC Iressa.\\nDate 19-11-2018\\nCYP2D6 UM: gefitinib\\n4873\\nNO action is needed for this gene-drug interaction.\\nThe gene variation may lead to a decrease in the gefitinib plasma concentration. In practice, an alternative is only chosen if non-response to gefitinib has been proved.\\nMoreover, dose adjustments guided by the gefitinib plasma concentration are rarely performed in clinical practice as the analytical method is not available in most hospitals.\\nLiterature:\\n1. \\nChhun S et al. Gefitinib-phenytoin interaction is not correlated with the C-erythromycin breath test in healthy male volunteers. Br J Clin Pharmacol 2009;68:226-37.\\n2. \\nSwaisland HC et al. Exploring the relationship between expression of cytochrome P450 enzymes and gefitinib pharmacokinetics. Clin Pharmacokinet 2006;45:633-44.\\n3. \\nSPC Iressa.\\nDate 19-11-2018\\nCYP2C9 IM ANDERS: glibenclamide\\n1882\\nNO action is required for this gene-drug interaction.\\nThe only relevant clinical consequence that was found is an increased effectiveness of glibenclamide without an increase in the frequency and severity of hypoglycaemia.\\nLiterature:\\n1. \\nSurendiran A et al. Influence of CYP2C9 gene polymorphisms on response to glibenclamide in type 2 diabetes mellitus patients. Eur J Clin Pharmacol 2011;67:797-801.\\n2. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n3. \\nYin OQ et al. CYP2C9, but not CYP2C19, polymorphisms affect the pharmacokinetics and pharmacodynamics of glyburide in Chinese subjects. Clin Pharmacol Ther\\n2005;78:370-7.\\n4. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\n5. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response in healthy volunteers. Clin Pharmacol\\nTher 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9 PM ANDERS: glibenclamide\\n1883\\nNO action is required for this gene-drug interaction.\\nNo relevant clinical consequences have been found for the genetic variation.\\nLiterature:\\n1. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n2. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n3. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response in healthy volunteers. Clin Pharmacol\\nTher 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*1/*2: glibenclamide\\n1877\\nNO action is required for this gene-drug interaction.\\nThe only relevant clinical consequence that was found is an increased effectiveness of glibenclamide without an increase in frequency and severity of hypoglycaemia for a\\ngroup of 1 *1/*2 and 15 *1/*3.\\nLiterature:\\n1. \\nSurendiran A et al. Influence of CYP2C9 gene polymorphisms on response to glibenclamide in type 2 diabetes mellitus patients. Eur J Clin Pharmacol 2011;67:797-801.\\n2. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\n4. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response in healthy volunteers. Clin Pharmacol\\nTher 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*1/*3: glibenclamide\\n1878', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 50}),\n",
       " Document(page_content=\"NO action is required for this gene-drug interaction.\\nThe only relevant clinical consequence is an increased effectiveness of glibenclamide without an increase in frequency and severity of hypoglycaemia.\\nLiterature:\\n1. \\nSurendiran A et al. Influence of CYP2C9 gene polymorphisms on response to glibenclamide in type 2 diabetes mellitus patients. Eur J Clin Pharmacol 2011;67:797-801.\\n2. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3. \\nYin OQ et al. CYP2C9, but not CYP2C19, polymorphisms affect the pharmacokinetics and pharmacodynamics of glyburide in Chinese subjects. Clin Pharmacol Ther\\n2005;78:370-7.\\n4. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\n5. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response in healthy volunteers. Clin Pharmacol\\nTher 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*2/*2: glibenclamide\\n1879\\nNO action is required for this gene-drug interaction.\\nNo significant clinical consequences have been found for the genetic variation.\\nLiterature:\\n1. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n2. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response in healthy volunteers. Clin Pharmacol\\nTher 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*2/*3: glibenclamide\\n1880\\nNO action is required for this gene-drug interaction.\\nNo significant kinetic or clinical consequences have been found for this genetic variation.\\nLiterature:\\n1. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n2. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n3. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response in healthy volunteers. Clin Pharmacol\\nTher 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*3/*3: glibenclamide\\n1881\\nNO action is required for this gene-drug interaction.\\nNo relevant clinical consequences have been found for this genetic variation.\\nLiterature:\\n1. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n2. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response in healthy volunteers. Clin Pharmacol\\nTher 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9 IM ANDERS: gliclazide\\n1889\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J Diabetes Investig 2016;7:764-8.\\n2. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n3. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS study. Clin Pharmacol Ther 2010;87:52-6.\\n4. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas. Pharmacogenomics 2009;10:1781-7.\\n5. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J Pharmacol 2008;153:1579-86.\\n6. \\nZhang Y et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on pharmacokinetics of gliclazide MR in Chinese subjects. Br J Clin Pharmacol 2007;64:67-74.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 51}),\n",
       " Document(page_content=\"Date 20-11-2017\\nCYP2C9 PM ANDERS: gliclazide\\n1890\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J Diabetes Investig 2016;7:764-8.\\n2. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n3. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS study. Clin Pharmacol Ther 2010;87:52-6.\\n4. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J Pharmacol 2008;153:1579-86.\\nDate 20-11-2017\\nCYP2C9*1/*2: gliclazide\\n1884\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide\\nLiterature:\\n1. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n2. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS study. Clin Pharmacol Ther 2010;87:52-6.\\n3. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas. Pharmacogenomics 2009;10:1781-7.\\n4. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J Pharmacol 2008;153:1579-86.\\nDate 20-11-2017\\nCYP2C9*1/*3: gliclazide\\n1885\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J Diabetes Investig 2016;7:764-8.\\n2. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n3. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS study. Clin Pharmacol Ther 2010;87:52-6.\\n4. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas. Pharmacogenomics 2009;10:1781-7.\\n5. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J Pharmacol 2008;153:1579-86.\\n6. \\nZhang Y et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on pharmacokinetics of gliclazide MR in Chinese subjects. Br J Clin Pharmacol 2007;64:67-74.\\nDate 20-11-2017\\nCYP2C9*2/*2: gliclazide\\n1886\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide.\\nLiterature:\\n1. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n2. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS study. Clin Pharmacol Ther 2010;87:52-6.\\n3. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J Pharmacol 2008;153:1579-86.\\nDate 20-11-2017\\nCYP2C9*2/*3: gliclazide\\n1887\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide.\\nLiterature:\\n1. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 52}),\n",
       " Document(page_content='2. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS study. Clin Pharmacol Ther 2010;87:52-6.\\nDate 20-11-2017\\nCYP2C9*3/*3: gliclazide\\n1888\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J Diabetes Investig 2016;7:764-8.\\n2. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n3. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS study. Clin Pharmacol Ther 2010;87:52-6.\\nDate 20-11-2017\\nCYP2C9 IM ANDERS: glimepiride\\n1896\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of glimepiride.\\nLiterature:\\n1. \\nBhatt D et al. Investigating the role of plasma glucose concentration as a phenotypic marker for CYP2C9 genetic variants, in the diabetic population of Gujarat. Indian J\\nPharm Sci 2014;76:72-7.\\n2. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n3. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n4. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas. Pharmacogenomics 2009;10:1781-7.\\n5. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n6. \\nSuzuki K et al. Effect of CYP2C9 genetic polymorphisms on the efficacy and pharmacokinetics of glimepiride in subjects with type 2 diabetes. Diabetes Res Clin Pract\\n2006;72:148-54.\\n7. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n8. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther 2005;78:90-2.\\n9. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9 PM ANDERS: glimepiride\\n1897\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation increases the risk of hypoglycaemia, this risk remains low. This also means that the genetic variation increases the effectiveness. A lack of\\neffectiveness is a much more common problem with sulphonyl urea derivatives than the occurrence of hypoglycaemia.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n3. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n4. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther 2005;78:90-2.\\n5. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*1/*2: glimepiride\\n1891\\nNO action is required for this gene-drug interaction.\\nNo significant kinetic or clinical consequences have been found for the genetic variation.\\nLiterature:\\n1. \\nBhatt D et al. Investigating the role of plasma glucose concentration as a phenotypic marker for CYP2C9 genetic variants, in the diabetic population of Gujarat. Indian J\\nPharm Sci 2014;76:72-7.\\n2. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n3. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n4. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas. Pharmacogenomics 2009;10:1781-7.\\n5. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n6. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 53}),\n",
       " Document(page_content='Pharmacol 2005;60:103-6.\\n7. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*1/*3: glimepiride\\n1892\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of glimepiride.\\nLiterature:\\n1. \\nBhatt D et al. Investigating the role of plasma glucose concentration as a phenotypic marker for CYP2C9 genetic variants, in the diabetic population of Gujarat. Indian J\\nPharm Sci 2014;76:72-7.\\n2. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8 polymorphisms in routine clinical setting. Eur J Clin\\nPharmacol 2011;67:1223-9.\\n3. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n4. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas. Pharmacogenomics 2009;10:1781-7.\\n5. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n6. \\nSuzuki K et al. Effect of CYP2C9 genetic polymorphisms on the efficacy and pharmacokinetics of glimepiride in subjects with type 2 diabetes. Diabetes Res Clin Pract\\n2006;72:148-54.\\n7. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n8. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther 2005;78:90-2.\\n9. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*2/*2: glimepiride\\n1893\\nNO action is required for this gene-drug interaction.\\nNo significant clinical effects were observed for this genetic variation.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\nDate 20-11-2017\\nCYP2C9*2/*3: glimepiride\\n1894\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation increases the risk of hypoglycaemia, this risk remains low. This also means that the genetic variation increases the effectiveness. A lack of\\neffectiveness is a much more common problem with sulphonyl urea derivatives than the occurrence of hypoglycaemia.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-\\n92.\\n3. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n4. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther 2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*3/*3: glimepiride\\n1895\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation increases the risk of hypoglycaemia, this risk remains low. This also means that the genetic variation increases the effectiveness. A lack of\\neffectiveness is a much more common problem with sulphonyl urea derivatives than the occurrence of hypoglycaemia.\\nLiterature:\\n1. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with sulphonylurea hypoglycaemic agents. Br J Clin\\nPharmacol 2005;60:103-6.\\n2. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther 2005;78:90-2.\\nDate 20-11-2017', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 54}),\n",
       " Document(page_content='CYP2D6 IM: haloperidol\\n1551\\nNO action is required for this gene-drug interaction.\\nThe genetic variation results in a higher plasma concentration, but the effect is small and no clinically significant effects were found.\\nLiterature:\\n1. \\nWaade RB et al. Impact of CYP2D6 on serum concentrations of flupentixol, haloperidol, perphenazine and zuclopenthixol. Br J Clin Pharmacol 2021;87:2228-35.\\n2. \\nTrogrlić Z et al. Pharmacogenomic response of low dose haloperidol in critically ill adults with delirium. J Crit Care 2020;57:203-7.\\n3. \\nSychev DA et al. The correlation between CYP2D6 isoenzyme activity and haloperidol efficacy and safety profile in patients with alcohol addiction during the exacerbation\\nof the addiction. Pharmgenomics Pers Med 2016;9:89-95.\\n4. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n5. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 and personal communication (mean dose-corrected trough concentrations).\\n6. \\nPanagiotidis G et al. Depot haloperidol treatment in outpatients with schizophrenia on monotherapy: impact of CYP2D6 polymorphism on pharmacokinetics and treatment\\noutcome. Ther Drug Monit 2007;29:417-22.\\n7. \\nPark JY et al. Combined effects of itraconazole and CYP2D6*10 genetic polymorphism on the pharmacokinetics and pharmacodynamics of haloperidol in healthy subjects.\\nJ Clin Psychopharmacol 2006;26:135-42.\\n8. \\nLLerena A et al. Relationship between haloperidol plasma concentration, debrisoquine metabolic ratio, CYP2D6 and CYP2C9 genotypes in psychiatric patients.\\nPharmacopsychiatry 2004;37:69-73.\\n9. \\nDesai M et al. Pharmacokinetics and QT interval pharmacodynamics of oral haloperidol in poor and extensive metabolizers of CYP2D6. Pharmacogenomics J 2003;3:105-\\n13.\\n10. \\nBrockmoller J et al. The impact of the CYP2D6 polymorphism on haloperidol pharmacokinetics and on the outcome of haloperidol treatment. Clin Pharmacol Ther\\n2002;72:438-52.\\n11. \\nYasui-Furukori N et al. Effect of the CYP2D6 genotype on prolactin concentration in schizophrenic patients treated with haloperidol. Schizophr Res 2001 1;52:139-42.\\n12. \\nPan L et al. Effects of smoking, CYP2D6 genotype, and concomitant drug intake on the steady state plasma concentrations of haloperidol and reduced haloperidol in\\nschizophrenic inpatients. Ther Drug Monit 1999;21:489-97.\\n13. \\nLlerena A et al. Haloperidol disposition is dependent on the debrisoquine hydroxylation phenotype: increased plasma levels of the reduced metabolite in poor\\nmetabolizers. Ther Drug Monit 1992;14:261-4.\\n14. \\nLlerena A. Haloperidol disposition is dependent on debrisoquine hydroxylation phenotype. Ther Drug Monit 1992;14:92-7.\\nDate 13-09-2021\\nCYP2D6 PM: haloperidol\\n1552\\nThere are indications for an increased risk of side effects. The genetic variation leads to decreased conversion of haloperidol, resulting in plasma concentrations that are\\napproximately 1.7-fold higher.\\nuse 60% of the standard\\nLiterature:\\n1. \\nWaade RB et al. Impact of CYP2D6 on serum concentrations of flupentixol, haloperidol, perphenazine and zuclopenthixol. Br J Clin Pharmacol 2021;87:2228-35.\\n2. \\nTrogrlić Z et al. Pharmacogenomic response of low dose haloperidol in critically ill adults with delirium. J Crit Care 2020;57:203-7.\\n3. \\nŠimić I et al. CYP2D6 \\n6/\\n6 genotype and drug interactions as cause of haloperidol-induced extrapyramidal symptoms. Pharmacogenomics 2016;17:1385-9.\\n4. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n5. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 and personal communication (mean dose-corrected trough concentrations).\\n6. \\nButwicka A et al. Neuroleptic malignant syndrome in an adolescent with CYP2D6 deficiency. Eur J Pediatr 2014;173:1639-42.\\n7. \\nGassó P et al. Relationship between CYP2D6 genotype and haloperidol pharmacokinetics and extrapyramidal symptoms in healthy volunteers. Pharmacogenomics\\n2013;14:1551-63.\\n8. \\nPanagiotidis G et al. Depot haloperidol treatment in outpatients with schizophrenia on monotherapy: impact of CYP2D6 polymorphism on pharmacokinetics and treatment\\noutcome. Ther Drug Monit 2007;29:417-22.\\n9. \\nLLerena A et al. Relationship between haloperidol plasma concentration, debrisoquine metabolic ratio, CYP2D6 and CYP2C9 genotypes in psychiatric patients.\\nPharmacopsychiatry 2004;37:69-73.\\n10. \\nDesai M et al. Pharmacokinetics and QT interval pharmacodynamics of oral haloperidol in poor and extensive metabolizers of CYP2D6. Pharmacogenomics J 2003;3:105-\\n13.\\n11. \\nBrockmoller J et al. The impact of the CYP2D6 polymorphism on haloperidol pharmacokinetics and on the outcome of haloperidol treatment. Clin Pharmacol Ther\\n2002;72:438-52.\\n12. \\nYasui-Furukori N et al. Effect of the CYP2D6 genotype on prolactin concentration in schizophrenic patients treated with haloperidol. Schizophr Res 2001 1;52:139-42.\\n13. \\nPan L et al. Effects of smoking, CYP2D6 genotype, and concomitant drug intake on the steady state plasma concentrations of haloperidol and reduced haloperidol in\\nschizophrenic inpatients. Ther Drug Monit 1999;21:489-97.\\n14. \\nLlerena A et al. Haloperidol disposition is dependent on the debrisoquine hydroxylation phenotype: increased plasma levels of the reduced metabolite in poor\\nmetabolizers. Ther Drug Monit 1992;14:261-4.\\n15. \\nLlerena A. Haloperidol disposition is dependent on debrisoquine hydroxylation phenotype. Ther Drug Monit 1992;14:92-7.\\n16. \\nSmPC’s Haldol en Haldol Decanoas.\\nDate 13-09-2021\\nCYP2D6 UM: haloperidol\\n1553\\nThere are indications of a risk of reduced effectiveness. The genetic variation leads to an increased conversion of haloperidol, resulting in a plasma concentration that is\\napproximately 40% lower.\\nuse 1.5 times the standard dose or choose an alternative. Antipsychotics that are not metabolised by CYP2D6 - or to a much lesser extent - include, for example,\\nflupentixol, penfluridol, quetiapine, olanzapine or clozapine.\\nLiterature:\\n1. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n2. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 and personal communication (mean dose-corrected trough concentrations).\\n3. \\nGassó P et al. Relationship between CYP2D6 genotype and haloperidol pharmacokinetics and extrapyramidal symptoms in healthy volunteers. Pharmacogenomics\\n2013;14:1551-63.\\n4. \\nPanagiotidis G et al. Depot haloperidol treatment in outpatients with schizophrenia on monotherapy: impact of CYP2D6 polymorphism on pharmacokinetics and treatment\\noutcome. Ther Drug Monit 2007;29:417-22.\\n5. \\nBrockmoller J et al. The impact of the CYP2D6 polymorphism on haloperidol pharmacokinetics and on the outcome of haloperidol treatment. Clin Pharmacol Ther\\n2002;72:438-52.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 55}),\n",
       " Document(page_content='Date 13-09-2021\\nCYP2C19 IM: imipramine\\n1913\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases imipramine plasma concentrations, but not imipramine+desipramine plasma concentrations, which govern effectiveness and side effects.\\nLiterature:\\n1. \\nSchenk PW et al. The CYP2C19*17 genotype is associated with lower imipramine plasma concentrations in a large group of depressed patients. Pharmacogenomics J\\n2010;10:219-25.\\n2. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large group of depressed patients. Mol\\nPsychiatry 2008;13:597-605.\\n3. \\nMadsen H et al. Imipramine demethylation in vivo: impact of CYP1A2, CYP2C19, and CYP3A4. Clin Pharmacol Ther 1997;61:319-24.\\n4. \\nKoyama E et al. Steady-state plasma concentrations of imipramine and desipramine in relation to S-mephenytoin 4-hydroxylation status in Japanese depressive patients. J\\nClin Psychopharmacol 1996;16:286-93.\\n5. \\nMadsen H et al. Imipramine metabolism in relation to the sparteine and mephenytoin oxidation polymorphisms--a population study. Br J Clin Pharmacol 1995;39:433-9.\\n6. \\nKoyama E et al. Metabolic disposition of imipramine in oriental subjects: relation to metoprolol alpha-hydroxylation and S-mephenytoin 4-hydroxylation phenotypes. J\\nPharmacol Exp Ther 1994;271:860-7.\\n7. \\nSkjelbo E et al. The N-demethylation of imipramine correlates with the oxidation of S-mephenytoin (S/R-ratio). A population study. Br J Clin Pharmacol 1993;35:331-4.\\nDate 10-09-2018\\nCYP2C19 PM: imipramine\\n1914\\nThe risk of side effects is increased. The gene variation results in an increase in the plasma concentration of imipramine+desipramine.\\nuse 70% of the standard dose and monitor the effect and side effects, or the imipramine and desipramine plasma concentrations to determine the maintenance dose.\\nor avoid imipramine\\nAntidepressants that are not or to a lesser extent metabolised by CYP2C19 include, for example, nortriptyline, fluvoxamine and mirtazapine.\\nLiterature:\\n1. \\nSchenk PW et al. The CYP2C19*17 genotype is associated with lower imipramine plasma concentrations in a large group of depressed patients. Pharmacogenomics J\\n2010;10:219-25.\\n2. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large group of depressed patients. Mol\\nPsychiatry 2008;13:597-605.\\n3. \\nMorinobu S et al. Effects of genetic defects in the CYP2C19 gene on the N-demethylation of imipramine, and clinical outcome of imipramine therapy. Psychiatry Clin\\nNeurosci 997;51:253-7.\\n4. \\nMadsen H et al. Imipramine demethylation in vivo: impact of CYP1A2, CYP2C19, and CYP3A4. Clin Pharmacol Ther 1997;61:319-24.\\n5. \\nKoyama E et al. Steady-state plasma concentrations of imipramine and desipramine in relation to S-mephenytoin 4-hydroxylation status in Japanese depressive patients. J\\nClin Psychopharmacol 1996;16:286-93.\\n6. \\nMadsen H et al. Imipramine metabolism in relation to the sparteine and mephenytoin oxidation polymorphisms--a population study. Br J Clin Pharmacol 1995;39:433-9.\\n7. \\nKoyama E et al. Metabolic disposition of imipramine in oriental subjects: relation to metoprolol alpha-hydroxylation and S-mephenytoin 4-hydroxylation phenotypes. J\\nPharmacol Exp Ther 1994;271:860-7.\\n8. \\nSkjelbo E et al. The N-demethylation of imipramine correlates with the oxidation of S-mephenytoin (S/R-ratio). A population study. Br J Clin Pharmacol 1993;35:331-4.\\n9. \\nSkjelbo E et al. The mephenytoin oxidation polymorphism is partially responsible for the N-demethylation of imipramine. Clin Pharmacol Ther 1991;49:18-23.\\nDate 10-09-2018\\nCYP2C19 UM: imipramine\\n1915\\nNO action is required for this gene-drug interaction.\\nThe genetic variation decreases imipramine plasma concentrations, but not imipramine+desipramine plasma concentrations, which govern effectiveness and side effects.\\nLiterature:\\n1. \\nSchenk PW et al. The CYP2C19*17 genotype is associated with lower imipramine plasma concentrations in a large group of depressed patients. Pharmacogenomics J\\n2010;10:219-25.\\nDate 10-09-2018\\nCYP2D6 IM: imipramine\\n1545\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of imipramine and desipramine.\\nuse 70% of the standard dose and monitor the effect and side effects or the plasma concentrations of imipramine and desipramine in order to set the maintenance dose\\nThe therapeutic range is 150-300 ng/mL for the sum of the imipramine and desipramine plasma concentrations. Values exceeding 500 ng/mL are considered toxic.\\nLiterature:\\n1. \\nSchliessbach J et al. Effect of single-dose imipramine on chronic low-back and experimental pain. A randomized controlled trial. PLoS One 2018;13:e0195776.\\n2. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large group of depressed patients. Mol\\nPsychiatry 2008;13:597-605.\\n3. \\nSindrup SH et al. Nonlinear kinetics of imipramine in low and medium plasma level ranges. Ther Drug Monit 1990;12:445-9.\\n4. \\nBrosen K et al. Imipramine demethylation and hydroxylation: impact of the sparteine oxidation phenotype. Clin Pharmacol Ther 1986;40:543-9.\\nDate 19-11-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 56}),\n",
       " Document(page_content='CYP2D6 PM: imipramine\\n1544\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of imipramine and the active metabolite desipramine.\\nuse 30% of the standard dose and monitor the effect and side effects or the plasma concentrations of imipramine and desipramine in order to set the maintenance dose\\nThe therapeutic range is 150-300 ng/mL for the sum of the imipramine and desipramine plasma concentrations. Values exceeding 500 ng/mL are considered toxic.\\nLiterature:\\n1. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large group of depressed patients. Mol\\nPsychiatry 2008;13:597-605.\\n2. \\nKoyama E et al. Metabolic disposition of imipramine in oriental subjects: relation to metoprolol alpha-hydroxylation and S-mephenytoin 4\"-hydroxylation phenotypes. J\\nPharmacol Exp Ther 1994;271:860-7.\\n3. \\nSindrup SH et al. Nonlinear kinetics of imipramine in low and medium plasma level ranges. Ther Drug Monit 1990;12:445-9.\\n4. \\nBalant-Gorgia AE et al. High blood concentrations of imipramine or clomipramine and therapeutic failure: a case report study using drug monitoring data. Ther Drug\\nMonit 1989;11:415-20.\\n5. \\nBrosen K et al. Imipramine demethylation and hydroxylation: impact of the sparteine oxidation phenotype. Clin Pharmacol Ther 1986;40:543-9.\\n6. \\nBrosen K et al. Steady-state concentrations of imipramine and its metabolites in relation to the sparteine/debrisoquine polymorphism. Eur J Clin Pharmacol 1986;30:679-\\n84.\\n7. \\nSmPC Tofranil-PM (VS).\\nDate 19-11-2018\\nCYP2D6 UM: imipramine\\n1546\\nThe risk of ineffectiveness and cardiotoxic side effects may be increased. The gene variation leads to reduced plasma concentrations of imipramine and the active metabolite\\ndesipramine and to increased plasma concentrations of the potentially cardiotoxic hydroxy metabolites.\\nuse 1.7 times the standard dose and monitor the effect and side effects or the plasma concentrations of imipramine and desipramine in order to set the maintenance dose\\nif a dose increase is not wanted due to the potentially cardiotoxic hydroxy metabolites: avoid imipramine.\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\nLiterature:\\n1. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large group of depressed patients. Mol\\nPsychiatry 2008;13:597-605.\\n2. \\nSindrup SH et al. Nonlinear kinetics of imipramine in low and medium plasma level ranges. Ther Drug Monit 1990;12:445-9.\\nDate 19-11-2018\\nUGT1A1 IM ANDERS: irinotecan\\n1691\\nNO action is needed for this gene-drug interaction.\\nThis genetic variation (IM) is more common in Western populations than the wild-type (\\n*1/\\n*1). This means that treatment is largely geared to patients with this genetic\\nvariation. Adjustment of the treatment is therefore not useful.\\nLiterature:\\n1. \\nYang Y et al. UGT1A1\\n6 and UGT1A1\\n28 polymorphisms are correlated with irinotecan-induced toxicity: A meta-analysis. Asia Pac J Clin Oncol 2018;14:e479-e489.\\n2. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a meta-analysis. Cancer Chemother\\nPharmacol 2017;79:1109-1117.\\n3. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis. Biomarkers. 2014;19:56-62.\\n4. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs 2013;31:1559-67.\\n5. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer Chemother Pharmacol 2009;65:97-105.\\n6. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of UGT1A1\\n6 and \\n28. Pharmacogenet Genomics\\n2007;17:497-504.\\n7. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype screening: a double-blind, randomized, placebo-\\ncontrolled study. Oncologist 2006;11:944-54.\\n8. \\nHan JY et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in patients with non-small-cell lung cancer\\ntreated with irinotecan and cisplatin. J Clin Oncol 2006;24:2237-44.\\n9. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol 2004;44:854-60.\\n10. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered Japanese patients with cancer. Clin\\nPharmacol Ther 2004;75:501-15.\\n11. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of irinotecan. J Clin Oncol 2004;22:1382-8.\\n12. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n13. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer Res 2000;60:6921-6.\\nDate 07-06-2021\\nUGT1A1 PM ANDERS: irinotecan\\n1692\\nSerious, life-threatening adverse events occur more often in patients with this genetic variation. The genetic variation reduces conversion of irinotecan to inactive metabolites.\\nStart with 70% of the standard dose\\nIf the patient tolerates this initial dose, the dose can be increased, guided by the neutrophil count.\\nLiterature:\\n1. \\nYang Y et al. UGT1A1\\n6 and UGT1A1\\n28 polymorphisms are correlated with irinotecan-induced toxicity: A meta-analysis. Asia Pac J Clin Oncol 2018;14:e479-e489.\\n2. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a meta-analysis. Cancer Chemother\\nPharmacol 2017;79:1109-1117.\\n3. \\nHan FF et al. Associations between UGT1A1\\n6 or UGT1A1\\n6/*28 polymorphisms and irinotecan-induced neutropenia in Asian cancer patients. Cancer Chemother\\nPharmacol 2014;73:779-88.\\n4. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis. Biomarkers. 2014;19:56-62.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 57}),\n",
       " Document(page_content=\"5. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs 2013;31:1559-67.\\n6. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 polymorphism and irinotecan-induced diarrhoea: a meta-analysis. Eur J Cancer 2010;46:1856-65.\\n7. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer Chemother Pharmacol 2009;65:97-105.\\n8. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of UGT1A1\\n6 and \\n28. Pharmacogenet Genomics\\n2007;17:497-504.\\n9. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype screening: a double-blind, randomized, placebo-\\ncontrolled study. Oncologist 2006;11:944-54.\\n10. \\nHan JY et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in patients with non-small-cell lung cancer\\ntreated with irinotecan and cisplatin. J Clin Oncol 2006;24:2237-44.\\n11. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol 2004;44:854-60.\\n12. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered Japanese patients with cancer. Clin\\nPharmacol Ther 2004;75:501-15.\\n13. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of irinotecan. J Clin Oncol 2004;22:1382-8.\\n14. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n15. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer Res 2000;60:6921-6.\\nDate 07-06-2021\\nUGT1A1*1/*28: irinotecan\\n1693\\nNO action is needed for this gene-drug interaction.\\nThis genetic variation (*1/*28) is more common in Western populations than the wild-type (*1/*1). This means that treatment is largely geared to patients with this genetic\\nvariation. Adjustment of the treatment is therefore not useful.\\nLiterature:\\n1. \\nYang Y et al. UGT1A1\\n6 and UGT1A1\\n28 polymorphisms are correlated with irinotecan-induced toxicity: A meta-analysis. Asia Pac J Clin Oncol 2018;14:e479-e489.\\n2. \\nTejpar S et al. Clinical and pharmacogenetic determinants of 5-fluorouracyl/ leucovorin/irinotecan toxicity: results of the PETACC-3 trial. Eur J Cancer. 2018;99:66-77.\\n3. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a meta-analysis. Cancer Chemother\\nPharmacol 2017;79:1109-1117.\\n4. \\nLiu XH et al. Predictive value of UGT1A1*28 polymorphism in irinotecan-based chemotherapy. J Cancer 2017;8:691-703.\\n5. \\nDias MM et al. The effect of the UGT1A1*28 allele on survival after irinotecan-based chemotherapy: a collaborative meta-analysis. Pharmacogenomics J 2014;14:424-31.\\n6. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis. Biomarkers. 2014;19:56-62.\\n7. \\nLiu X et al. Association of UGT1A1*28 polymorphisms with irinotecan-induced toxicities in colorectal cancer: a meta-analysis in Caucasians. Pharmacogenomics J\\n2014;14:120-9.\\n8. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs 2013;31:1559-67.\\n9. \\nLiu X et al. Association between UGT1A1*28 polymorphisms and clinical outcomes of irinotecan-based chemotherapies in colorectal cancer: a meta-analysis in\\nCaucasians. PLoS One 2013;8:e58489.\\n10. \\nDias MM et al. Impact of the UGT1A1*28 allele on response to irinotecan: a systematic review and meta-analysis. Pharmacogenomics 2012;13:889-99.\\n11. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 genotype and irinotecan-induced neutropenia: low doses also increase risk. Clin Cancer Res 2010;16:3832-\\n42.\\n12. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 polymorphism and irinotecan-induced diarrhoea: a meta-analysis. Eur J Cancer 2010;46:1856-65.\\n13. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer Chemother Pharmacol 2009;65:97-\\n105.Kweekel DM et al. UGT1A1*28 genotype and irinotecan dosage in patients with metastatic colorectal cancer: a Dutch Colorectal Cancer Group study. Br J Cancer\\n2008;99:275-82.\\n14. \\nLiu CY et al. UGT1A1*28 polymorphism predicts irinotecan-induced severe toxicities without affecting treatment outcome and survival in patients with metastatic\\ncolorectal carcinoma. Cancer 2008;112:1932-40.\\n15. \\nLankisch TO et al. Gilbert's Syndrome and irinotecan toxicity: combination with UDP-glucuronosyl-transferase 1A7 variants increases risk. Cancer Epidemiol Biomarkers\\nPrev 2008;17:695-701.\\n16. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of UGT1A1\\n6 and \\n28. Pharmacogenet Genomics\\n2007;17:497-504.\\n17. \\nStewart CF et al. UGT1A1 promoter genotype correlates with SN-38 pharmacokinetics, but not severe toxicity in patients receiving low-dose irinotecan. J Clin Oncol\\n2007;25:2594-600.\\n18. \\nCôté JF et al. UGT1A1 polymorphism can predict hematologic toxicity in patients treated with irinotecan. Clin Cancer Res 2007;13:3269-75.\\n19. \\nRamchandani RP et al. The role of SN-38 exposure, UGT1A1*28 polymorphism, and baseline bilirubin level in predicting severe irinotecan toxicity. J Clin Pharmacol\\n2007;47:78-86.\\n20. \\nZárate Romero R et al. Potential application of GSTT1-null genotype in predicting toxicity associated to 5-fluouracil irinotecan and leucovorin regimen in advanced stage\\ncolorectal cancer patients. Oncol Rep 2006;16:497-503.\\n21. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype screening: a double-blind, randomized, placebo-\\ncontrolled study. Oncologist 2006;11:944-54.\\n22. \\nToffoli G et al. The role of UGT1A1*28 polymorphism in the pharmacodynamics and pharmacokinetics of irinotecan in patients with metastatic colorectal cancer. J Clin\\nOncol 2006;24:3061-8.\\n23. \\nHan JY et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in patients with non-small-cell lung cancer\\ntreated with irinotecan and cisplatin. J Clin Oncol 2006;24:2237-44.\\n24. \\nMcLeod HL et al. UGT1A1*28, toxicity and outcome in advanced colorectal cancer: results from Trial N9741. J Clin Oncol 2006;24 (suppl. abstr. 3520).\\n25. \\nMassacesi C et al. Uridine diphosphate glucuronosyl transferase 1A1 promoter polymorphism predicts the risk of gastrointestinal toxicity and fatigue induced by\\nirinotecan-based chemotherapy. Cancer 2006;106:1007-16.\\n26. \\nWright MA et al. A phase I pharmacologic and pharmacogenetic trial of sequential 24-hour infusion of irinotecan followed by leucovorin and a 48-hour infusion of\\nfluorouracil in adult patients with solid tumors. Clin Cancer Res 2005;11:4144-50.\\n27. \\nKweekel DM et al. Ondersteuning van de chemotherapiekeuze. Pharm Weekblad 2005;20:685-7.\\n28. \\nSteiner M et al. 5-fluorouracil/irinotecan induced lethal toxicity as a result of a combined pharmacogenetic syndrome: report of a case. J Clin Pathol 2005;58:553-5.\\n29. \\nSoepenberg O et al. Phase I pharmacokinetic, food effect, and pharmacogenetic study of oral irinotecan given as semisolid matrix capsules in patients with solid tumors.\\nClin Cancer Res 2005;11:1504-11.\\n30. \\nZhou Q et al. Pharmacogenetic profiling across the irinotecan pathway in Asian patients with cancer. Br J Clin Pharmacol 2005;59:415-24.\\n31. \\nMarcuello E et al. UGT1A1 gene variations and irinotecan treatment in patients with metastatic colorectal cancer. Br J Cancer 2004;91:678-82.\\n32. \\nRouits E et al. Relevance of different UGT1A1 polymorphisms in irinotecan-induced toxicity: a molecular and clinical study of 75 patients. Clin Cancer Res 2004;10:5151-\\n9.\\n33. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol 2004;44:854-60.\\n34. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered Japanese patients with cancer. Clin\\nPharmacol Ther 2004;75:501-15.\\n35. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of irinotecan. J Clin Oncol 2004;22:1382-8.\\n36. \\nFont A et al. Weekly regimen of irinotecan/docetaxel in previously treated non-small cell lung cancer patients and correlation with uridine diphosphate glucuronosyl-\\ntransferase 1A1 (UGT1A1) polymorphism. Invest New Drugs 2003;21:435-43.\\n37. \\nMathijssen RH et al. Irinotecan pathway genotype analysis to predict pharmacokinetics. Clin Cancer Res 2003;9:3246-53.\\n38. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n39. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer Res 2000;60:6921-6.\\n40. \\nWasserman E et al. Severe CPT-11 toxicity in patients with Gilbert’s syndrome: two case reports. Ann Oncol 1997;8:1049-51.\\nDate 07-06-2021\\nUGT1A1*28/*28: irinotecan\\n1694\\nSerious, life-threatening adverse events occur more often in patients with this genetic variation. The genetic variation reduces conversion of irinotecan to inactive metabolites.\\nStart with 70% of the standard dose\\nIf the patient tolerates this initial dose, the dose can be increased, guided by the neutrophil count.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 58}),\n",
       " Document(page_content=\"Literature:\\n1. \\nYang Y et al. UGT1A1\\n6 and UGT1A1\\n28 polymorphisms are correlated with irinotecan-induced toxicity: A meta-analysis. Asia Pac J Clin Oncol 2018;14:e479-e489.\\n2. \\nTejpar S et al. Clinical and pharmacogenetic determinants of 5-fluorouracyl/ leucovorin/irinotecan toxicity: results of the PETACC-3 trial. Eur J Cancer. 2018;99:66-77.\\n3. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a meta-analysis. Cancer Chemother\\nPharmacol 2017;79:1109-1117.\\n4. \\nLiu XH et al. Predictive value of UGT1A1*28 polymorphism in irinotecan-based chemotherapy. J Cancer 2017;8:691-703.\\n5. \\nLu CY et al. Clinical implication of UGT1A1 promoter polymorphism for irinotecan dose escalation in metastatic colorectal cancer patients treated with bevacizumab\\ncombined with FOLFIRI in the first-line setting. Transl Oncol 2015;8:474-9.\\n6. \\nDias MM et al. The effect of the UGT1A1*28 allele on survival after irinotecan-based chemotherapy: a collaborative meta-analysis. Pharmacogenomics J 2014;14:424-31.\\n7. \\nHan FF et al. Associations between UGT1A1\\n6 or UGT1A1\\n6/*28 polymorphisms and irinotecan-induced neutropenia in Asian cancer patients. Cancer Chemother\\nPharmacol 2014;73:779-88.\\n8. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis. Biomarkers. 2014;19:56-62.\\n9. \\nLiu X et al. Association of UGT1A1*28 polymorphisms with irinotecan-induced toxicities in colorectal cancer: a meta-analysis in Caucasians. Pharmacogenomics J\\n2014;14:120-9.\\n10. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs 2013;31:1559-67.\\n11. \\nLiu X et al. Association between UGT1A1*28 polymorphisms and clinical outcomes of irinotecan-based chemotherapies in colorectal cancer: a meta-analysis in\\nCaucasians. PLoS One 2013;8:e58489.\\n12. \\nDias MM et al. Impact of the UGT1A1*28 allele on response to irinotecan: a systematic review and meta-analysis. Pharmacogenomics 2012;13:889-99.\\n13. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 genotype and irinotecan-induced neutropenia: low doses also increase risk. Clin Cancer Res 2010;16:3832-\\n42.\\n14. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 polymorphism and irinotecan-induced diarrhoea: a meta-analysis. Eur J Cancer 2010;46:1856-65.\\n15. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer Chemother Pharmacol 2009;65:97-105.\\n16. \\nKweekel DM et al. UGT1A1*28 genotype and irinotecan dosage in patients with metastatic colorectal cancer: a Dutch Colorectal Cancer Group study. Br J Cancer\\n2008;99:275-82.\\n17. \\nLiu CY et al. UGT1A1*28 polymorphism predicts irinotecan-induced severe toxicities without affecting treatment outcome and survival in patients with metastatic\\ncolorectal carcinoma. Cancer 2008;112:1932-40.\\n18. \\nLankisch TO et al. Gilbert's Syndrome and irinotecan toxicity: combination with UDP-glucuronosyl-transferase 1A7 variants increases risk. Cancer Epidemiol Biomarkers\\nPrev 2008;17:695-701.\\n19. \\nHoskins JM et al. UGT1A1*28 genotype and irinotecan-induced neutropenia: dose matters. J Natl Cancer Inst 2007;99:1290-5.\\n20. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of UGT1A1\\n6 and \\n28. Pharmacogenet Genomics\\n2007;17:497-504.\\n21. \\nStewart CF et al. UGT1A1 promoter genotype correlates with SN-38 pharmacokinetics, but not severe toxicity in patients receiving low-dose irinotecan. J Clin Oncol\\n2007;25:2594-600.\\n22. \\nCôté JF et al. UGT1A1 polymorphism can predict hematologic toxicity in patients treated with irinotecan. Clin Cancer Res 2007;13:3269-75.\\n23. \\nRamchandani RP et al. The role of SN-38 exposure, UGT1A1*28 polymorphism, and baseline bilirubin level in predicting severe irinotecan toxicity. J Clin Pharmacol\\n2007;47:78-86.\\n24. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype screening: a double-blind, randomized, placebo-\\ncontrolled study. Oncologist 2006;11:944-54.\\n25. \\nToffoli G et al. The role of UGT1A1*28 polymorphism in the pharmacodynamics and pharmacokinetics of irinotecan in patients with metastatic colorectal cancer. J Clin\\nOncol 2006;24:3061-8.\\n26. \\nMcLeod HL et al. UGT1A1*28, toxicity and outcome in advanced colorectal cancer: results from Trial N9741. J Clin Oncol 2006;24 (suppl. abstr. 3520).\\n27. \\nMassacesi C et al. Uridine diphosphate glucuronosyl transferase 1A1 promoter polymorphism predicts the risk of gastrointestinal toxicity and fatigue induced by\\nirinotecan-based chemotherapy. Cancer 2006;106:1007-16.\\n28. \\nKweekel DM et al. Ondersteuning van de chemotherapiekeuze. Pharm Weekblad 2005;20:685-7.\\n29. \\nSoepenberg O et al. Phase I pharmacokinetic, food effect, and pharmacogenetic study of oral irinotecan given as semisolid matrix capsules in patients with solid tumors.\\nClin Cancer Res 2005;11:1504-11.\\n30. \\nZhou Q et al. Pharmacogenetic profiling across the irinotecan pathway in Asian patients with cancer. Br J Clin Pharmacol 2005;59:415-24.\\n31. \\nCarlini LE et al. UGT1A7 and UGT1A9 polymorphisms predict response and toxicity in colorectal cancer patients treated with capecitabine/ irinotecan. Clin Cancer Res\\n2005;11:1226-36.\\n32. \\nKitagawa C et al. Genetic polymorphism in the phenobarbital-responsive enhancer module of the UDP-glucuronosyltransferase 1A1 gene and irinotecan toxicity.\\nPharmacogenet Genomics 2005;15:35-41.\\n33. \\nMarcuello E et al. UGT1A1 gene variations and irinotecan treatment in patients with metastatic colorectal cancer. Br J Cancer 2004;91:678-82.\\n34. \\nRouits E et al. Relevance of different UGT1A1 polymorphisms in irinotecan-induced toxicity: a molecular and clinical study of 75 patients. Clin Cancer Res 2004;10:5151-\\n9.\\n35. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol 2004;44:854-60.\\n36. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered Japanese patients with cancer. Clin\\nPharmacol Ther 2004;75:501-15.\\n37. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of irinotecan. J Clin Oncol 2004;22:1382-8.\\n38. \\nFont A et al. Weekly regimen of irinotecan/docetaxel in previously treated non-small cell lung cancer patients and correlation with uridine diphosphate glucuronosyl-\\ntransferase 1A1 (UGT1A1) polymorphism. Invest New Drugs 2003;21:435-43.\\n39. \\nMathijssen RH et al. Irinotecan pathway genotype analysis to predict pharmacokinetics. Clin Cancer Res 2003;9:3246-53.\\n40. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n41. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer Res 2000;60:6921-6.\\n42. \\nWasserman E et al. Severe CPT-11 toxicity in patients with Gilbert’s syndrome: two case reports. Ann Oncol 1997;8:1049-51.\\n43. \\nSmPC’s Campto en Camptosar (VS).\\nDate 07-06-2021\\nCYP2D6 IM: kinidine\\n2534\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nNielsen F et al. Lack of relationship between quinidine pharmacokinetics and the sparteine oxidation polymorphism. Eur J Clin Pharmacol 1995;48:501-4.\\n2. \\nBrøsen K et al. Quinidine kinetics after a single oral dose in relation to the sparteine oxidation polymorphism in man. Br J Clin Pharmacol 1990;29:248-53.\\n3. \\nMikus G et al. Pharmacokinetics and metabolism of quinidine in extensive and poor metabolisers of sparteine. Eur J Clin Pharmacol 1986;31:69-72.\\nDate 12-09-2022\\nCYP2D6 PM: kinidine\\n2533\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nNielsen F et al. Lack of relationship between quinidine pharmacokinetics and the sparteine oxidation polymorphism. Eur J Clin Pharmacol 1995;48:501-4.\\n2. \\nBrøsen K et al. Quinidine kinetics after a single oral dose in relation to the sparteine oxidation polymorphism in man. Br J Clin Pharmacol 1990;29:248-53.\\n3. \\nMikus G et al. Pharmacokinetics and metabolism of quinidine in extensive and poor metabolisers of sparteine. Eur J Clin Pharmacol 1986;31:69-72.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 59}),\n",
       " Document(page_content='Date 12-09-2022\\nCYP2D6 UM: kinidine\\n2535\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nNielsen F et al. Lack of relationship between quinidine pharmacokinetics and the sparteine oxidation polymorphism. Eur J Clin Pharmacol 1995;48:501-4.\\n2. \\nBrøsen K et al. Quinidine kinetics after a single oral dose in relation to the sparteine oxidation polymorphism in man. Br J Clin Pharmacol 1990;29:248-53.\\n3. \\nMikus G et al. Pharmacokinetics and metabolism of quinidine in extensive and poor metabolisers of sparteine. Eur J Clin Pharmacol 1986;31:69-72.\\nDate 12-09-2022\\nHLA-B*1502: lamotrigine\\n6932\\nThe life-threatening cutaneous side effect Stevens-Johnson Syndrome/Toxic Epidermal Necrolysis (SJS/TEN) occurs more often in patients with this genetic variation. Based on\\nthe estimated risk for all patients and the increase by a factor 3.6 for patients with this genetic variation, the risk of lamotrigine-induced SJS/TEN in patients with HLA-B*1502\\nis estimated at 0.4%.\\ncarefully weigh the risk of SJS/TEN against the benefits\\navoid lamotrigine if an alternative is available\\nCarbamazepine carries a much higher risk of SJS/TEN in these patients and is therefore not an alternative.\\nA similar risk has been reported for phenytoin as for lamotrigine. The same applies to oxcarbazepine, but the most severe forms (SJS/TEN overlap and TEN) have not\\nbeen observed with oxcarbazepine.\\nif it is not possible to avoid these products, advise the patient to report any rash immediately\\nLiterature:\\n1. \\nSukasem C et al. Spectrum of cutaneous adverse reactions to aromatic antiepileptic drugs and human leukocyte antigen genotypes in Thai patients and meta-analysis.\\nPharmacogenomics J 2021;21:682-90.\\n2. \\nSabourirad S et al. Investigating the association of lamotrigine and phenytoin-induced Stevens-Johnson syndrome/Toxic Epidermal Necrolysis with HLA-B*1502 in Iranian\\npopulation. Exp Dermatol 2021;30:284-7.\\n3. \\nDeng Y et al. Association between HLA alleles and lamotrigine-induced cutaneous adverse drug reactions in Asian populations: a meta-analysis. Seizure 2018;60:163-71.\\n4. \\nShi YW et al. HLA-A*24:02 as a common risk factor for antiepileptic drug-induced cutaneous adverse reactions. Neurology 2017;88:2183-2191.\\n5. \\nZeng T et al. Association of HLA-B*1502 allele with lamotrigine-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Han Chinese subjects: a meta-\\nanalysis. Int J Dermatol 2015;54:488-93.\\n6. \\nKwan PK et al. Association between HLA-B*15:02 allele and antiepileptic drug-induced severe cutaneous reactions in Hong Kong Chinese: a population-based study. Hong\\nKong Med J 2014;20 Suppl 7:16-8.\\n7. \\nWang W et al. Genetic predictors of Stevens-Johnson syndrome and toxic epidermal necrolysis induced by aromatic antiepileptic drugs among the Chinese Han\\npopulation. Epilepsy Behav 2014;37:16-9.\\n8. \\nBloch KM et al. Pharmacogenetics of antiepileptic drug-induced hypersensitivity. Pharmacogenomics 2014;15:857-68.\\n9. \\nCheung YK et al. HLA-B alleles associated with severe cutaneous reactions to antiepileptic drugs in Han Chinese. Epilepsia 2013;54:1307-14.\\n10. \\nHung SI et al. Common risk allele in aromatic antiepileptic-drug induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Han Chinese. Pharmacogenomics\\n2010;11:349-56.\\nDate 23-05-2022\\nCYP2C19 IM: lansoprazol\\n1831\\nNO action is needed for this gene-drug interaction.\\nThe higher plasma concentration of lansoprazole results in an increase in the therapeutic effectiveness, without an increase in the incidence of side effects.\\nLiterature:\\n1. \\nLang JE et al. Lansoprazole is associated with worsening asthma control in children with the CYP2C19 poor metabolizer phenotype. Ann Am Thorac Soc 2015 Jun;12:878-\\n85.\\n2. \\nLi CY et al. A correlative study of polymorphisms of CYP2C19 and MDR1 C3435T with the pharmacokinetic profiles of lansoprazole and its main metabolites following\\nsingle oral administration in healthy adult Chinese subjects. Eur J Drug Metab Pharmacokinet 2014;39:121-8.\\n3. \\nLima JJ et al. Association of CYP2C19 polymorphisms and lansoprazole-associated respiratory adverse effects in children. J Pediatr 2013;163:686-91.\\n4. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n5. \\nZhang D et al. Effects of CYP2C19 polymorphism on the pharmacokinetics of lansoprazole and its main metabolites in healthy Chinese subjects. Xenobiotica 2011;41:511-\\n7.\\n6. \\nTamura T et al. Improvements in Helicobacter pylori eradication rates through clinical CYP2C19 genotyping. Nagoya J Med Sci 2011;73:25-31.\\n7. \\nXu HR et al. The effect of CYP2C19 activity on pharmacokinetics of lansoprazole and its active metabolites in healthy subjects. Pharm Biol 2010;48:947-52.\\n8. \\nLee JH et al. The influence of CYP2C19 polymorphism on eradication of Helicobacter pylori: a prospective randomized study of lansoprazole and rabeprazole. Gut Liver\\n2010;4:201-6.\\n9. \\nFuruta T et al. CYP2C19 genotype is associated with symptomatic recurrence of GERD during maintenance therapy with low-dose lansoprazole. Eur J Clin Pharmacol\\n2009;65:693-8.\\n10. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n11. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in Caucasians. Br J Clin Pharmacol 2008;65:752-60.\\n12. \\nSakurai Y et al. Population pharmacokinetics and proton pump inhibitory effects of intravenous lansoprazole in healthy Japanese males. Biol Pharm Bull 2007;30:2238-43.\\n13. \\nKawamura M et al. Cytochrome P450 2C19 polymorphism influences the preventive effect of lansoprazole on the recurrence of erosive reflux esophagitis. J Gastroenterol\\nHepatol 2007;22:222-6.\\n14. \\nFuruta T et al. Effect of concomitant dosing of famotidine with lansoprazole on gastric acid secretion in relation to CYP2C19 genotype status. Aliment Pharmacol Ther\\n2005;22:67-74.\\n15. \\nKawamura M et al. Study Group of GERD. The effects of lansoprazole on erosive reflux oesophagitis are influenced by CYP2C19 polymorphism. Aliment Pharmacol Ther\\n2003;17:965-73.\\n16. \\nFuruta T et al. Effect of cytochrome P4502C19 genotypic differences on cure rates for gastroesophageal reflux disease by lansoprazole. Clin Pharmacol Ther\\n2002;72:453-60.\\n17. \\nShirai N et al. Comparison of lansoprazole and famotidine for gastric acid inhibition during the daytime and night-time in different CYP2C19 genotype groups. Aliment\\nPharmacol Ther 2002;16:837-46.\\n18. \\nFuruta T et al. Effect of high-dose lansoprazole on intragastric pH in subjects who are homozygous extensive metabolizers of cytochrome P4502C19. Clin Pharmacol Ther\\n2001;70:484-92.\\n19. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol Ther 2000;14:1259-66.\\n20. \\nHowden CW et al. Dose-response evaluation of the antisecretory effect of continuous infusion intravenous lansoprazole regimens over 48 h. Aliment Pharmacol Ther\\n2006;23:975-84.\\n21. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 60}),\n",
       " Document(page_content='Helicobacter 2008;13:532-41.\\n22. \\nFuruta T et al. Effect of MDR1 C3435T polymorphism on cure rates of Helicobacter pylori infection by triple therapy with lansoprazole, amoxicillin and clarithromycin in\\nrelation to CYP 2C19 genotypes and 23S rRNA genotypes of H. pylori. Aliment Pharmacol Ther 2007;26:693-703.\\n23. \\nSugimoto M et al. Evidence that the degree and duration of acid suppression are related to Helicobacter pylori eradication by triple therapy. Helicobacter 2007;12:317-\\n23.\\n24. \\nFuruta T et al. Pharmacogenomics-based tailored versus standard therapeutic regimen for eradication of H. pylori. Clin Pharmacol Ther 2007;81:521-8.\\n25. \\nPadol S et al. The effect of CYP2C19 polymorphisms on H. pylori eradication rate in dual and triple first-line PPI therapies: a meta-analysis. Am J Gastroenterol\\n2006;101:1467-75.\\n26. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of clarithromycin-sensitive strains of Helicobacter\\npylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n27. \\nFuruta T et al. Influence of CYP2C19 polymorphism and Helicobacter pylori genotype determined from gastric tissue samples on response to triple therapy for H pylori\\ninfection. Clin Gastroenterol Hepatol 2005;3:564-73.\\n28. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin Gastroenterol Hepatol 2004;2:22-30.\\n29. \\nKawabata H et al. Effect of different proton pump inhibitors, differences in CYP2C19 genotype and antibiotic resistance on the eradication rate of Helicobacter pylori\\ninfection by a 1-week regimen of proton pump inhibitor, amoxicillin and clarithromycin. Aliment Pharmacol Ther 2003;17:259-64.\\n30. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori infection with lansoprazole- or\\nrabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n31. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a proton pump inhibitor, amoxicillin, and\\nclarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n32. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n33. \\nQiao HL et al. Pharmacokinetics of three proton pump inhibitors in Chinese subjects in relation to the CYP2C19 genotype. Eur J Clin Pharmacol 2006;62:107-12.\\n34. \\nMiura M et al. Pharmacokinetic differences between the enantiomers of lansoprazole and its metabolite, 5-hydroxylansoprazole, in relation to CYP2C19 genotypes. Eur J\\nClin Pharmacol 2004;60:623-8.\\n35. \\nIeiri I et al. Comparison of the kinetic disposition of and serum gastrin change by lansoprazole versus rabeprazole during an 8-day dosing scheme in relation to CYP2C19\\npolymorphism. Eur J Clin Pharmacol 2001;57:485-92.\\n36. \\nSakai T et al. CYP2C19 genotype and pharmacokinetics of three proton pump inhibitors in healthy subjects. Pharm Res 2001;18:721-7.\\n37. \\nKatsuki H et al. Genetic polymorphism of CYP2C19 and lansoprazole pharmacokinetics in Japanese subjects. Eur J Clin Pharmacol 1997;52:391-6.\\nDate 05-03-2018\\nCYP2C19 PM: lansoprazol\\n1832\\nNO action is needed for this gene-drug interaction.\\nThe higher plasma concentration of lansoprazole results in an increase in the therapeutic effectiveness, without an increase in the incidence of side effects.\\nLiterature:\\n1. \\nLiou JM et al. Sequential therapy for 10 days versus triple therapy for 14 days in the eradication of Helicobacter pylori in the community and hospital populations: a\\nrandomised trial. Gut 2016;65:1784-1792.\\n2. \\nLiou JM et al. Levofloxacin sequential therapy vs levofloxacin triple therapy in the second-line treatment of Helicobacter pylori: a randomized trial. Am J Gastroenterol\\n2016;111:381-7.\\n3. \\nLang JE et al. Lansoprazole is associated with worsening asthma control in children with the CYP2C19 poor metabolizer phenotype. Ann Am Thorac Soc 2015 Jun;12:878-\\n85.\\n4. \\nLi CY et al. A correlative study of polymorphisms of CYP2C19 and MDR1 C3435T with the pharmacokinetic profiles of lansoprazole and its main metabolites following\\nsingle oral administration in healthy adult Chinese subjects. Eur J Drug Metab Pharmacokinet 2014;39:121-8.\\n5. \\nLima JJ et al. Association of CYP2C19 polymorphisms and lansoprazole-associated respiratory adverse effects in children. J Pediatr 2013;163:686-91.\\n6. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n7. \\nZhang D et al. Effects of CYP2C19 polymorphism on the pharmacokinetics of lansoprazole and its main metabolites in healthy Chinese subjects. Xenobiotica 2011;41:511-\\n7.\\n8. \\nTamura T et al. Improvements in Helicobacter pylori eradication rates through clinical CYP2C19 genotyping. Nagoya J Med Sci 2011;73:25-31.\\n9. \\nXu HR et al. The effect of CYP2C19 activity on pharmacokinetics of lansoprazole and its active metabolites in healthy subjects. Pharm Biol 2010;48:947-52.\\n10. \\nLee JH et al. The influence of CYP2C19 polymorphism on eradication of Helicobacter pylori: a prospective randomized study of lansoprazole and rabeprazole. Gut Liver\\n2010;4:201-6.\\n11. \\nFuruta T et al. CYP2C19 genotype is associated with symptomatic recurrence of GERD during maintenance therapy with low-dose lansoprazole. Eur J Clin Pharmacol\\n2009;65:693-8.\\n12. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n13. \\nSakurai Y et al. Population pharmacokinetics and proton pump inhibitory effects of intravenous lansoprazole in healthy Japanese males. Biol Pharm Bull 2007;30:2238-43.\\n14. \\nKawamura M et al. Cytochrome P450 2C19 polymorphism influences the preventive effect of lansoprazole on the recurrence of erosive reflux esophagitis. J Gastroenterol\\nHepatol 2007;22:222-6.\\n15. \\nFuruta T et al. Effect of concomitant dosing of famotidine with lansoprazole on gastric acid secretion in relation to CYP2C19 genotype status. Aliment Pharmacol Ther\\n2005;22:67-74.\\n16. \\nKawamura M et al. Study Group of GERD. The effects of lansoprazole on erosive reflux oesophagitis are influenced by CYP2C19 polymorphism. Aliment Pharmacol Ther\\n2003;17:965-73.\\n17. \\nFuruta T et al. Effect of cytochrome P4502C19 genotypic differences on cure rates for gastroesophageal reflux disease by lansoprazole. Clin Pharmacol Ther\\n2002;72:453-60.\\n18. \\nShirai N et al. Comparison of lansoprazole and famotidine for gastric acid inhibition during the daytime and night-time in different CYP2C19 genotype groups. Aliment\\nPharmacol Ther 2002;16:837-46.\\n19. \\nFuruta T et al. Effect of high-dose lansoprazole on intragastric pH in subjects who are homozygous extensive metabolizers of cytochrome P4502C19. Clin Pharmacol Ther\\n2001;70:484-92.\\n20. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol Ther 2000;14:1259-66.\\n21. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n22. \\nFuruta T et al. Effect of MDR1 C3435T polymorphism on cure rates of Helicobacter pylori infection by triple therapy with lansoprazole, amoxicillin and clarithromycin in\\nrelation to CYP 2C19 genotypes and 23S rRNA genotypes of H. pylori. Aliment Pharmacol Ther 2007;26:693-703.\\n23. \\nSugimoto M et al. Evidence that the degree and duration of acid suppression are related to Helicobacter pylori eradication by triple therapy. Helicobacter 2007;12:317-\\n23.\\n24. \\nFuruta T et al. Pharmacogenomics-based tailored versus standard therapeutic regimen for eradication of H. pylori. Clin Pharmacol Ther 2007;81:521-8.\\n25. \\nPadol S et al. The effect of CYP2C19 polymorphisms on H. pylori eradication rate in dual and triple first-line PPI therapies: a meta-analysis. Am J Gastroenterol\\n2006;101:1467-75.\\n26. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of clarithromycin-sensitive strains of Helicobacter\\npylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n27. \\nFuruta T et al. Influence of CYP2C19 polymorphism and Helicobacter pylori genotype determined from gastric tissue samples on response to triple therapy for H pylori\\ninfection. Clin Gastroenterol Hepatol 2005;3:564-73.\\n28. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin Gastroenterol Hepatol 2004;2:22-30.\\n29. \\nKawabata H et al. Effect of different proton pump inhibitors, differences in CYP2C19 genotype and antibiotic resistance on the eradication rate of Helicobacter pylori\\ninfection by a 1-week regimen of proton pump inhibitor, amoxicillin and clarithromycin. Aliment Pharmacol Ther 2003;17:259-64.\\n30. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori infection with lansoprazole- or\\nrabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n31. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a proton pump inhibitor, amoxicillin, and\\nclarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n32. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n33. \\nQiao HL et al. Pharmacokinetics of three proton pump inhibitors in Chinese subjects in relation to the CYP2C19 genotype. Eur J Clin Pharmacol 2006;62:107-12.\\n34. \\nMiura M et al. Pharmacokinetic differences between the enantiomers of lansoprazole and its metabolite, 5-hydroxylansoprazole, in relation to CYP2C19 genotypes. Eur J\\nClin Pharmacol 2004;60:623-8.\\n35. \\nHu YR et al. Pharmacokinetics of lansoprazole in Chinese healthy subjects in relation to CYP2C19 genotypes. Acta Pharmacol Sin 2004;25:986-90.\\n36. \\nIeiri I et al. Comparison of the kinetic disposition of and serum gastrin change by lansoprazole versus rabeprazole during an 8-day dosing scheme in relation to CYP2C19\\npolymorphism. Eur J Clin Pharmacol 2001;57:485-92.\\n37. \\nSakai T et al. CYP2C19 genotype and pharmacokinetics of three proton pump inhibitors in healthy subjects. Pharm Res 2001;18:721-7.\\n38. \\nKatsuki H et al. Genetic polymorphism of CYP2C19 and lansoprazole pharmacokinetics in Japanese subjects. Eur J Clin Pharmacol 1997;52:391-6.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 61}),\n",
       " Document(page_content='39. \\nSPC Prezal.\\nDate 05-03-2018\\nCYP2C19 UM: lansoprazol\\n1833\\nThe genetic variation may reduce lansoprazole plasma concentrations and therefore lansoprazole effectiveness.\\nRecommendation:\\nFor Helicobacter pylori ERADICATION THERAPY:\\n1. \\nUse a 4-fold higher dose\\n2. \\nAdvise the patient to contact their doctor if symptoms of dyspepsia persist\\nOTHER INDICATIONS:\\n1. \\nBe alert to reduced effectiveness\\n2. \\nIf necessary, use a 4-fold higher dose\\n3. \\nAdvise the patient to report persisting symptoms of dyspepsia\\nLiterature:\\n1. \\nLiou JM et al. Sequential therapy for 10 days versus triple therapy for 14 days in the eradication of Helicobacter pylori in the community and hospital populations: a\\nrandomised trial. Gut 2016;65:1784-1792.\\n2. \\nLiou JM et al. Levofloxacin sequential therapy vs levofloxacin triple therapy in the second-line treatment of Helicobacter pylori: a randomized trial. Am J Gastroenterol\\n2016;111:381-7.\\n3. \\nLang JE et al. Lansoprazole is associated with worsening asthma control in children with the CYP2C19 poor metabolizer phenotype. Ann Am Thorac Soc 2015 Jun;12:878-\\n85.\\n4. \\nLi CY et al. A correlative study of polymorphisms of CYP2C19 and MDR1 C3435T with the pharmacokinetic profiles of lansoprazole and its main metabolites following\\nsingle oral administration in healthy adult Chinese subjects. Eur J Drug Metab Pharmacokinet 2014;39:121-8.\\n5. \\nLima JJ et al. Association of CYP2C19 polymorphisms and lansoprazole-associated respiratory adverse effects in children. J Pediatr 2013;163:686-91.\\n6. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n7. \\nZhang D et al. Effects of CYP2C19 polymorphism on the pharmacokinetics of lansoprazole and its main metabolites in healthy Chinese subjects. Xenobiotica 2011;41:511-\\n7.\\n8. \\nTamura T et al. Improvements in Helicobacter pylori eradication rates through clinical CYP2C19 genotyping. Nagoya J Med Sci 2011;73:25-31.\\n9. \\nXu HR et al. The effect of CYP2C19 activity on pharmacokinetics of lansoprazole and its active metabolites in healthy subjects. Pharm Biol 2010;48:947-52.\\n10. \\nLee JH et al. The influence of CYP2C19 polymorphism on eradication of Helicobacter pylori: a prospective randomized study of lansoprazole and rabeprazole. Gut Liver\\n2010;4:201-6.\\n11. \\nFuruta T et al. CYP2C19 genotype is associated with symptomatic recurrence of GERD during maintenance therapy with low-dose lansoprazole. Eur J Clin Pharmacol\\n2009;65:693-8.\\n12. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n13. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in Caucasians. Br J Clin Pharmacol 2008;65:752-60.\\n14. \\nSakurai Y et al. Population pharmacokinetics and proton pump inhibitory effects of intravenous lansoprazole in healthy Japanese males. Biol Pharm Bull 2007;30:2238-43.\\n15. \\nKawamura M et al. Cytochrome P450 2C19 polymorphism influences the preventive effect of lansoprazole on the recurrence of erosive reflux esophagitis. J Gastroenterol\\nHepatol 2007;22:222-6.\\n16. \\nFuruta T et al. Effect of concomitant dosing of famotidine with lansoprazole on gastric acid secretion in relation to CYP2C19 genotype status. Aliment Pharmacol Ther\\n2005;22:67-74.\\n17. \\nKawamura M et al. Study Group of GERD. The effects of lansoprazole on erosive reflux oesophagitis are influenced by CYP2C19 polymorphism. Aliment Pharmacol Ther\\n2003;17:965-73.\\n18. \\nFuruta T et al. Effect of cytochrome P4502C19 genotypic differences on cure rates for gastroesophageal reflux disease by lansoprazole. Clin Pharmacol Ther\\n2002;72:453-60.\\n19. \\nShirai N et al. Comparison of lansoprazole and famotidine for gastric acid inhibition during the daytime and night-time in different CYP2C19 genotype groups. Aliment\\nPharmacol Ther 2002;16:837-46.\\n20. \\nFuruta T et al. Effect of high-dose lansoprazole on intragastric pH in subjects who are homozygous extensive metabolizers of cytochrome P4502C19. Clin Pharmacol Ther\\n2001;70:484-92.\\n21. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol Ther 2000;14:1259-66.\\n22. \\nHowden CW et al. Dose-response evaluation of the antisecretory effect of continuous infusion intravenous lansoprazole regimens over 48 h. Aliment Pharmacol Ther\\n2006;23:975-84.\\n23. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n24. \\nFuruta T et al. Effect of MDR1 C3435T polymorphism on cure rates of Helicobacter pylori infection by triple therapy with lansoprazole, amoxicillin and clarithromycin in\\nrelation to CYP 2C19 genotypes and 23S rRNA genotypes of H. pylori. Aliment Pharmacol Ther 2007;26:693-703.\\n25. \\nSugimoto M et al. Evidence that the degree and duration of acid suppression are related to Helicobacter pylori eradication by triple therapy. Helicobacter 2007;12:317-\\n23.\\n26. \\nFuruta T et al. Pharmacogenomics-based tailored versus standard therapeutic regimen for eradication of H. pylori. Clin Pharmacol Ther 2007;81:521-8.\\n27. \\nPadol S et al. The effect of CYP2C19 polymorphisms on H. pylori eradication rate in dual and triple first-line PPI therapies: a meta-analysis. Am J Gastroenterol\\n2006;101:1467-75.\\n28. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of clarithromycin-sensitive strains of Helicobacter\\npylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n29. \\nFuruta T et al. Influence of CYP2C19 polymorphism and Helicobacter pylori genotype determined from gastric tissue samples on response to triple therapy for H pylori\\ninfection. Clin Gastroenterol Hepatol 2005;3:564-73.\\n30. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin Gastroenterol Hepatol 2004;2:22-30.\\n31. \\nKawabata H et al. Effect of different proton pump inhibitors, differences in CYP2C19 genotype and antibiotic resistance on the eradication rate of Helicobacter pylori\\ninfection by a 1-week regimen of proton pump inhibitor, amoxicillin and clarithromycin. Aliment Pharmacol Ther 2003;17:259-64.\\n32. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori infection with lansoprazole- or\\nrabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n33. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a proton pump inhibitor, amoxicillin, and\\nclarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n34. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n35. \\nQiao HL et al. Pharmacokinetics of three proton pump inhibitors in Chinese subjects in relation to the CYP2C19 genotype. Eur J Clin Pharmacol 2006;62:107-12.\\n36. \\nMiura M et al. Pharmacokinetic differences between the enantiomers of lansoprazole and its metabolite, 5-hydroxylansoprazole, in relation to CYP2C19 genotypes. Eur J\\nClin Pharmacol 2004;60:623-8.\\n37. \\nHu YR et al. Pharmacokinetics of lansoprazole in Chinese healthy subjects in relation to CYP2C19 genotypes. Acta Pharmacol Sin 2004;25:986-90.\\n38. \\nIeiri I et al. Comparison of the kinetic disposition of and serum gastrin change by lansoprazole versus rabeprazole during an 8-day dosing scheme in relation to CYP2C19\\npolymorphism. Eur J Clin Pharmacol 2001;57:485-92.\\n39. \\nSakai T et al. CYP2C19 genotype and pharmacokinetics of three proton pump inhibitors in healthy subjects. Pharm Res 2001;18:721-7.\\n40. \\nKatsuki H et al. Genetic polymorphism of CYP2C19 and lansoprazole pharmacokinetics in Japanese subjects. Eur J Clin Pharmacol 1997;52:391-6.\\n41. \\nSPC Prezal.\\nDate 05-03-2018\\nMTHFR 677CT: methotrexaat\\n5599\\nThis is NOT a gene-drug interaction.\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 62}),\n",
       " Document(page_content='1. \\nHuang J et al. Are gene polymorphisms related to adverse events of methotrexate in patients with rheumatoid arthritis? A retrospective cohort study based on an updated\\nmeta-analysis. Ther Adv Chronic Dis 2020;11:2040622320916026.\\n2. \\nYao P et al. The influence of MTHFR genetic polymorphisms on adverse reactions after methotrexate in patients with hematological malignancies: a meta-analysis.\\nHematology 2019;24:10-9.\\n3. \\nZhu C et al. Associations between the C677T and A1298C polymorphisms of MTHFR and the toxicity of methotrexate in childhood malignancies: a meta-analysis.\\nPharmacogenomics J 2018;18:450-9.\\n4. \\nShao W et al. Association between MTHFR C677T polymorphism and methotrexate treatment outcome in rheumatoid arthritis patients: a systematic review and meta-\\nanalysis. Genet Test Mol Biomarkers 2017;21:275-85.\\n5. \\nQiu Q et al. Polymorphisms and pharmacogenomics for the clinical efficacy of methotrexate in patients with rheumatoid arthritis: a systematic review and meta-analysis.\\nSci Rep 2017;7:44015.\\n6. \\nQiu Q et al. Polymorphisms and pharmacogenomics for the toxicity of methotrexate monotherapy in patients with rheumatoid arthritis: a systematic review and meta-\\nanalysis. Medicine (Baltimore) 2017;96:e6337.\\n7. \\nChen Y et al. Are gene polymorphisms related to treatment outcomes of methotrexate in patients with rheumatoid arthritis? A systematic review and meta-analysis.\\nPharmacogenomics 2017;18:175-95.\\n8. \\nSong GG et al. Association of the MTHFR C677T and A1298C polymorphisms with methotrexate toxicity in rheumatoid arthritis: a meta-analysis. Clin Rheumatol\\n2014;33:1715-24.\\n9. \\nHagleitner MM et al. The role of the MTHFR 677C>T polymorphism in methotrexate-induced liver toxicity: a meta-analysis in patients with cancer. Pharmacogenomics J\\n2014;14:115-9.\\n10. \\nMorgan MD et al. MTHFR functional genetic variation and methotrexate treatment response in rheumatoid arthritis: a meta-analysis. Pharmacogenomics 2014;15:467-75.\\n11. \\nOwen SA et al. MTHFR gene polymorphisms and outcome of methotrexate treatment in patients with rheumatoid arthritis: analysis of key polymorphisms and meta-\\nanalysis of C677T and A1298C polymorphisms. Pharmacogenomics J 2013;13:137-47.\\n12. \\nYang L et al. Impact of methylenetetrahydrofolate reductase (MTHFR) polymorphisms on methotrexate-induced toxicities in acute lymphoblastic leukemia: a meta-\\nanalysis. Tumour Biol 2012;33:1445-54.\\n13. \\nSpyridopoulou KP et al. Methylene tetrahydrofolate reductase gene polymorphisms and their association with methotrexate toxicity: a meta-analysis. Pharmacogenet\\nGenomics 2012;22:117-33.\\n14. \\nFisher MC et al. Meta-analysis of methylenetetrahydrofolate reductase (MTHFR) polymorphisms affecting methotrexate toxicity. J Rheumatol 2009;36:539-45.\\nDate 07-06-2021\\nMTHFR 677TT: methotrexaat\\n5600\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nHuang J et al. Are gene polymorphisms related to adverse events of methotrexate in patients with rheumatoid arthritis? A retrospective cohort study based on an updated\\nmeta-analysis. Ther Adv Chronic Dis 2020;11:2040622320916026.\\n2. \\nYao P et al. The influence of MTHFR genetic polymorphisms on adverse reactions after methotrexate in patients with hematological malignancies: a meta-analysis.\\nHematology 2019;24:10-9.\\n3. \\nZhu C et al. Associations between the C677T and A1298C polymorphisms of MTHFR and the toxicity of methotrexate in childhood malignancies: a meta-analysis.\\nPharmacogenomics J 2018;18:450-9.\\n4. \\nShao W et al. Association between MTHFR C677T polymorphism and methotrexate treatment outcome in rheumatoid arthritis patients: a systematic review and meta-\\nanalysis. Genet Test Mol Biomarkers 2017;21:275-85.\\n5. \\nQiu Q et al. Polymorphisms and pharmacogenomics for the clinical efficacy of methotrexate in patients with rheumatoid arthritis: a systematic review and meta-analysis.\\nSci Rep 2017;7:44015.\\n6. \\nQiu Q et al. Polymorphisms and pharmacogenomics for the toxicity of methotrexate monotherapy in patients with rheumatoid arthritis: a systematic review and meta-\\nanalysis. Medicine (Baltimore) 2017;96:e6337.\\n7. \\nChen Y et al. Are gene polymorphisms related to treatment outcomes of methotrexate in patients with rheumatoid arthritis? A systematic review and meta-analysis.\\nPharmacogenomics 2017;18:175-95.\\n8. \\nSong GG et al. Association of the MTHFR C677T and A1298C polymorphisms with methotrexate toxicity in rheumatoid arthritis: a meta-analysis. Clin Rheumatol\\n2014;33:1715-24.\\n9. \\nHagleitner MM et al. The role of the MTHFR 677C>T polymorphism in methotrexate-induced liver toxicity: a meta-analysis in patients with cancer. Pharmacogenomics J\\n2014;14:115-9.\\n10. \\nMorgan MD et al. MTHFR functional genetic variation and methotrexate treatment response in rheumatoid arthritis: a meta-analysis. Pharmacogenomics 2014;15:467-75.\\n11. \\nLopez-Lopez E et al. A systematic review and meta-analysis of MTHFR polymorphisms in methotrexate toxicity prediction in pediatric acute lymphoblastic leukemia.\\nPharmacogenomics J 2013;13:498-506.\\n12. \\nOwen SA et al. MTHFR gene polymorphisms and outcome of methotrexate treatment in patients with rheumatoid arthritis: analysis of key polymorphisms and meta-\\nanalysis of C677T and A1298C polymorphisms. Pharmacogenomics J 2013;13:137-47.\\n13. \\nYang L et al. Impact of methylenetetrahydrofolate reductase (MTHFR) polymorphisms on methotrexate-induced toxicities in acute lymphoblastic leukemia: a meta-\\nanalysis. Tumour Biol 2012;33:1445-54.\\n14. \\nSpyridopoulou KP et al. Methylene tetrahydrofolate reductase gene polymorphisms and their association with methotrexate toxicity: a meta-analysis. Pharmacogenet\\nGenomics 2012;22:117-33.\\n15. \\nFisher MC et al. Meta-analysis of methylenetetrahydrofolate reductase (MTHFR) polymorphisms affecting methotrexate toxicity. J Rheumatol 2009;36:539-45.\\nDate 07-06-2021\\nCOMT Met/Met: methylfenidaat\\n6433\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nFageera W et al. COMT by DRD3 epistatic interaction in modulating behaviors in children with ADHD: a pharmaco-dynamic behavioral approach. J Atten Disord\\n2021;25:1720-30.\\n2. \\nMyer NM et al. Pharmacogenetics predictors of methylphenidate efficacy in childhood ADHD. Mol Psychiatry 2018;23:1929-36.\\n3. \\nPagerols M et al. Pharmacogenetics of methylphenidate response and tolerability in attention-deficit/hyperactivity disorder. Pharmacogenomics J 2017;17:98-104.\\n4. \\nUnal D et al. Genetic variations in attention deficit hyperactivity disorder subtypes and treatment resistant cases. Psychiatry Investig 2016;13:427-33.\\n5. \\nPark S et al. Catechol-O-methyl-transferase Val158Met polymorphism and a response of hyperactive-impulsive symptoms to methylphenidate: a replication study from\\nSouth Korea. J Psychopharmacol 2014;28:671-6.\\n6. \\nContini V et al. No significant association between genetic variants in 7 candidate genes and response to methylphenidate treatment in adult patients with ADHD. J Clin\\nPsychopharmacol 2012;32:820-3.\\n7. \\nFroehlich TE et al. Pharmacogenetic predictors of methylphenidate dose-response in attention-deficit/hyperactivity disorder. J Am Acad Child Adolesc Psychiatry\\n2011;50:1129-1139.\\n8. \\nSalatino-Oliveira A et al. Catechol-O-methyltransferase Valine158Methionine polymorphism moderates methylphenidate effects on oppositional symptoms in boys with\\nattention-deficit/hyperactivity disorder. Biol Psychiatry 2011;70:216-21.\\n9. \\nMcGough JJ et al. A candidate gene analysis of methylphenidate response in attention-deficit/hyperactivity disorder. J Am Acad Child Adolesc Psychiatry 2009;48:1155-64.\\n10. \\nKereszturi E et al. Catechol-O-methyl-transferase Val158Met polymorphism is associated with methylphenidate response in ADHD children. Am J Med Genet B\\nNeuropsychiatr Genet 2008;147B:1431-5.\\n11. \\nSengupta S et al. COMT Val108/158Met polymorphism and the modulation of task-oriented behavior in children with ADHD. Neuropsychopharmacology 2008;33:3069-77.\\n12. \\nCheon KA et al. Association of the catechol-O-methyltransferase polymorphism with methylphenidate response in a classroom setting in children with attention-deficit\\nhyperactivity disorder. Int Clin Psychopharmacol 2008;23:291-8.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 63}),\n",
       " Document(page_content='Date 31-01-2022\\nCOMT Val/Met: methylfenidaat\\n6432\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nFageera W et al. COMT by DRD3 epistatic interaction in modulating behaviors in children with ADHD: a pharmaco-dynamic behavioral approach. J Atten Disord\\n2021;25:1720-30.\\n2. \\nMyer NM et al. Pharmacogenetics predictors of methylphenidate efficacy in childhood ADHD. Mol Psychiatry 2018;23:1929-36.\\n3. \\nPagerols M et al. Pharmacogenetics of methylphenidate response and tolerability in attention-deficit/hyperactivity disorder. Pharmacogenomics J 2017;17:98-104.\\n4. \\nUnal D et al. Genetic variations in attention deficit hyperactivity disorder subtypes and treatment resistant cases. Psychiatry Investig 2016;13:427-33.\\n5. \\nPark S et al. Catechol-O-methyl-transferase Val158Met polymorphism and a response of hyperactive-impulsive symptoms to methylphenidate: a replication study from\\nSouth Korea. J Psychopharmacol 2014;28:671-6.\\n6. \\nContini V et al. No significant association between genetic variants in 7 candidate genes and response to methylphenidate treatment in adult patients with ADHD. J Clin\\nPsychopharmacol 2012;32:820-3.\\n7. \\nFroehlich TE et al. Pharmacogenetic predictors of methylphenidate dose-response in attention-deficit/hyperactivity disorder. J Am Acad Child Adolesc Psychiatry\\n2011;50:1129-1139.\\n8. \\nSalatino-Oliveira A et al. Catechol-O-methyltransferase Valine158Methionine polymorphism moderates methylphenidate effects on oppositional symptoms in boys with\\nattention-deficit/hyperactivity disorder. Biol Psychiatry 2011;70:216-21.\\n9. \\nMcGough JJ et al. A candidate gene analysis of methylphenidate response in attention-deficit/hyperactivity disorder. J Am Acad Child Adolesc Psychiatry 2009;48:1155-64.\\n10. \\nKereszturi E et al. Catechol-O-methyl-transferase Val158Met polymorphism is associated with methylphenidate response in ADHD children. Am J Med Genet B\\nNeuropsychiatr Genet 2008;147B:1431-5.\\n11. \\nSengupta S et al. COMT Val108/158Met polymorphism and the modulation of task-oriented behavior in children with ADHD. Neuropsychopharmacology 2008;33:3069-77.\\n12. \\nCheon KA et al. Association of the catechol-O-methyltransferase polymorphism with methylphenidate response in a classroom setting in children with attention-deficit\\nhyperactivity disorder. Int Clin Psychopharmacol 2008;23:291-8.\\nDate 31-01-2022\\nCYP2D6 IM: methylfenidaat\\n2528\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nDeVane CL et al. Single-dose pharmacokinetics of methylphenidate in CYP2D6 extensive and poor metabolizers. J Clin Psychopharmacol 2000;20:347-9.\\nDate 31-01-2022\\nCYP2D6 PM: methylfenidaat\\n2527\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nDeVane CL et al. Single-dose pharmacokinetics of methylphenidate in CYP2D6 extensive and poor metabolizers. J Clin Psychopharmacol 2000;20:347-9.\\nDate 31-01-2022\\nCYP2D6 UM: methylfenidaat\\n2529\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nDeVane CL et al. Single-dose pharmacokinetics of methylphenidate in CYP2D6 extensive and poor metabolizers. J Clin Psychopharmacol 2000;20:347-9.\\nDate 31-01-2022\\nCYP2D6 IM: metoprolol\\n1554\\nThe gene variation reduces the conversion of metoprolol to inactive metabolites. However, the clinical consequences are limited mainly to the occurrence of asymptomatic\\nbradycardia.\\nRecommendation:\\nIf a GRADUAL REDUCTION in HEART RATE is desired, or in the event of SYMPTOMATIC BRADYCARDIA:\\n1. \\nuse smaller steps in dose titration and/or prescribe no more than 50% of the standard dose\\nOTHER CASES:\\n1. \\nno action required\\nLiterature:\\n1. \\nChen J et al. Impact of the CYP2D6 genotype on metoprolol tolerance and adverse events in elderly Chinese patients with cardiovascular diseases. Front Pharmacol\\n2022;13:876392.\\n2. \\nMeloche M et al. Leveraging large observational studies to discover genetic determinants of drug concentrations: a proof-of-concept study. Clin Transl Sci 2022;15:1063-\\n73.\\n3. \\nMeloche M et al. CYP2D6 polymorphism and its impact on the clinical response to metoprolol: a systematic review and meta-analysis. Br J Clin Pharmacol 2020;86:1015-', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 64}),\n",
       " Document(page_content='33.\\n4. \\nChen L et al. The association of ADRB1 and CYP2D6 polymorphisms with antihypertensive effects and analysis of their contribution to hypertension risk. Am J Med Sci\\n2018;355:235-9.\\n5. \\nGao X et al. Impact of CYP2D6 and ADRB1 polymorphisms on heart rate of post-PCI patients treated with metoprolol. Pharmacogenomics 2017 Nov 2 (online ahead of\\nprint).\\n6. \\nHamadeh IS et al. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol Ther 2014;96:175-81.\\n7. \\nBatty JA et al. An investigation of CYP2D6 genotype and response to metoprolol CR/XL during dose titration in patients with heart failure: a MERIT-HF substudy. Clin\\nPharmacol Ther 2014;95:321-30.\\n8. \\nRau T et al. Impact of the CYP2D6 genotype on the clinical effects of metoprolol: a prospective longitudinal study. Clin Pharmacol Ther 2009;85:269-72.\\n9. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users. Clin Pharmacol Ther 2009;85:45-50.\\n10. \\nGoryachkina K et al. CYP2D6 is a major determinant of metoprolol disposition and effects in hospitalized Russian patients treated for acute myocardial infarction. Eur J\\nClin Pharmacol 2008;64:1163-73.\\n11. \\nYuan H et al. Effects of polymorphism of the beta(1) adrenoreceptor and CYP2D6 on the therapeutic effects of metoprolol. J Int Med Res 2008;36:1354-62.\\n12. \\nJin SK et al. Influence of CYP2D6*10 on the pharmacokinetics of metoprolol in healthy Korean volunteers. J Clin Pharm Ther 200;33:567-73.\\n13. \\nIsmail R et al. The relevance of CYP2D6 genetic polymorphism on chronic metoprolol therapy in cardiovascular patients. J Clin Pharm Ther 2006;31:99-109.\\n14. \\nTerra SG et al. Beta1-adrenergic receptor polymorphisms and left ventricular remodeling changes in response to beta-blocker therapy. Clin Pharmacol Ther 2005;77:127-\\n137.\\n15. \\nNozawa T et al. Influence of CYP2D6 genotype on metoprolol plasma concentration and beta-adrenergic inhibition during long-term treatment: a comparison with\\nbisoprolol. J Cardiovasc Pharmacol 2005;46:713-20.\\n16. \\nFux R et al. Impact of CYP2D6 genotype on adverse effects during treatment with metoprolol: a prospective clinical study. Clin Pharmacol Ther 2005;78:378-87.\\n17. \\nZineh I et al. Pharmacokinetics and CYP2D6 genotypes do not predict metoprolol adverse events or efficacy in hypertension. Clin Pharmacol Ther 2004;76:536-44.\\n18. \\nTaguchi M et al. Effect of CYP2D6*10 on pharmacokinetic variability of routinely administered metoprolol in middle-aged and elderly Japanese patients. Eur J Clin\\nPharmacol 2003;59:385-8.\\n19. \\nRau T et al. Effect of the CYP2D6 genotype on metoprolol metabolism persists during long-term treatment. Pharmacogenetics 2002;12:465-72.\\n20. \\nHuang JD et al. Pharmacokinetics of metoprolol enantiomers in Chinese subjects of major CYP2D6 genotypes. Clin Pharmacol Ther 1999;65:402-7.\\n21. \\nKoytchev R et al. Influence of the cytochrome P4502D6*4 allele on the pharmacokinetics of controlled-release metoprolol. Eur J Clin Pharmacol 1998;54:469-74.\\nDate 12-09-2022\\nCYP2D6 PM: metoprolol\\n1555\\nThe gene variation reduces the conversion of metoprolol to inactive metabolites. However, the clinical consequences are limited mainly to the occurrence of asymptomatic\\nbradycardia.\\nRecommendation:\\nIf a GRADUAL REDUCTION in HEART RATE is desired, or in the event of SYMPTOMATIC BRADYCARDIA:\\n1. \\nuse smaller steps in dose titration and/or prescribe no more than 25% of the standard dose\\nOTHER CASES:\\n1. \\nno action required\\nLiterature:\\n1. \\nMeloche M et al. Leveraging large observational studies to discover genetic determinants of drug concentrations: a proof-of-concept study. Clin Transl Sci 2022;15:1063-\\n73.\\n2. \\nMeloche M et al. CYP2D6 polymorphism and its impact on the clinical response to metoprolol: a systematic review and meta-analysis. Br J Clin Pharmacol 2020;86:1015-\\n33.\\n3. \\nHamadeh IS et al. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol Ther 2014;96:175-81.\\n4. \\nBatty JA et al. An investigation of CYP2D6 genotype and response to metoprolol CR/XL during dose titration in patients with heart failure: a MERIT-HF substudy. Clin\\nPharmacol Ther 2014;95:321-30.\\n5. \\nRau T et al. Impact of the CYP2D6 genotype on the clinical effects of metoprolol: a prospective longitudinal study. Clin Pharmacol Ther 2009;85:269-72.\\n6. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users. Clin Pharmacol Ther 2009;85:45-50.\\n7. \\nGoryachkina K et al. CYP2D6 is a major determinant of metoprolol disposition and effects in hospitalized Russian patients treated for acute myocardial infarction. Eur J\\nClin Pharmacol 2008;64:1163-73.\\n8. \\nYuan H et al. Effects of polymorphism of the beta(1) adrenoreceptor and CYP2D6 on the therapeutic effects of metoprolol. J Int Med Res 2008;36:1354-62.\\n9. \\nSeeringer A et al. Enantiospecific pharmacokinetics of metoprolol in CYP2D6 ultra-rapid metabolizers and correlation with exercise-induced heart rate. Eur J Clin\\nPharmacol 2008;64:883-8.\\n10. \\nIsmail R et al. The relevance of CYP2D6 genetic polymorphism on chronic metoprolol therapy in cardiovascular patients. J Clin Pharm Ther 2006;31:99-109.\\n11. \\nTerra SG et al. Beta1-adrenergic receptor polymorphisms and left ventricular remodeling changes in response to beta-blocker therapy. Clin Pharmacol Ther 2005;77:127-\\n137.\\n12. \\nFux R et al. Impact of CYP2D6 genotype on adverse effects during treatment with metoprolol: a prospective clinical study. Clin Pharmacol Ther 2005;78:378-87.\\n13. \\nZineh I et al. Pharmacokinetics and CYP2D6 genotypes do not predict metoprolol adverse events or efficacy in hypertension. Clin Pharmacol Ther 2004;76:536-44.\\n14. \\nKirchheiner J et al. Impact of the ultrarapid metabolizer genotype of cytochrome P450 2D6 on metoprolol pharmacokinetics and pharmacodynamics. Clin Pharmacol Ther\\n2004;76:302-12.\\n15. \\nWuttke H et al. Increased frequency of cytochrome P450 2D6 poor metabolizers among patients with metoprolol-associated adverse effects. Clin Pharmacol Ther\\n2002;72:429-37.\\n16. \\nRau T et al. Effect of the CYP2D6 genotype on metoprolol metabolism persists during long-term treatment. Pharmacogenetics 2002;12:465-72.\\n17. \\nLaurent-Kenesi MA et al. Influence of CYP2D6-dependent metabolism on the steady-state pharmacokinetics and pharmacodynamics of metoprolol and nicardipine, alone\\nand in combination. Br J Clin Pharmacol 1993;36:531-8.\\n18. \\nLewis RV et al. Influence of debrisoquine oxidation phenotype on exercise tolerance and subjective fatigue after metoprolol and atenolol in healthy subjects. Br J Clin\\nPharmacol 1991;31:391-8.\\n19. \\nClark DWJ et al. Adverse effects from metoprolol are not generally associated with oxidation status. Br J Clin Pharmacol 1984;18:965-966.\\n20. \\nLennard MS et al. Differential stereoselective metabolism of metoprolol in extensive and poor debrisoquin metabolizers. Clin Pharmacol Ther 1983;34:732-7.\\n21. \\nSmPC Toprol-XL (VS).\\nDate 12-09-2022\\nCYP2D6 UM: metoprolol\\n1556\\nThe gene variation increases the conversion of metoprolol to inactive metabolites. This can increase the dose requirement. However, with a target dose of 200 mg/day, there\\nwas no effect on the blood pressure and hardly any effect on the reduction of the heart rate.\\nRecommendation:\\n1. \\nuse the maximum dose for the relevant indication as a target dose\\n2. \\nif the effectiveness is still insufficient: increase the dose based on effectiveness and side effects to 2.5 times the standard dose or select an alternative\\nPossible alternatives include:\\nHEART FAILURE: bisoprolol or carvedilol. Bisoprolol: advantage: not metabolised by CYP2D6; disadvantage: elimination depends on the kidney function. Carvedilol:\\nadvantage: elimination does not depend on the kidney function; disadvantage: is metabolised (to a lesser extent than metoprolol) by CYP2D6.\\nOTHER INDICATIONS: atenolol or bisoprolol. Neither is metabolised by CYP2D6.\\nLiterature:\\n1. \\nMeloche M et al. Leveraging large observational studies to discover genetic determinants of drug concentrations: a proof-of-concept study. Clin Transl Sci 2022;15:1063-', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 65}),\n",
       " Document(page_content='73.\\n2. \\nMeloche M et al. CYP2D6 polymorphism and its impact on the clinical response to metoprolol: a systematic review and meta-analysis. Br J Clin Pharmacol 2020;86:1015-\\n33.\\n3. \\nHamadeh IS et al. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol Ther 2014;96:175-81.\\n4. \\nGoryachkina K et al. CYP2D6 is a major determinant of metoprolol disposition and effects in hospitalized Russian patients treated for acute myocardial infarction. Eur J\\nClin Pharmacol 2008;64:1163-73.\\n5. \\nSeeringer A et al. Enantiospecific pharmacokinetics of metoprolol in CYP2D6 ultra-rapid metabolizers and correlation with exercise-induced heart rate. Eur J Clin\\nPharmacol 2008;64:883-8.\\n6. \\nIsmail R et al. The relevance of CYP2D6 genetic polymorphism on chronic metoprolol therapy in cardiovascular patients. J Clin Pharm Ther 2006;31:99-109.\\n7. \\nFux R et al. Impact of CYP2D6 genotype on adverse effects during treatment with metoprolol: a prospective clinical study. Clin Pharmacol Ther 2005;78:378-87.\\n8. \\nKirchheiner J et al. Impact of the ultrarapid metabolizer genotype of cytochrome P450 2D6 on metoprolol pharmacokinetics and pharmacodynamics. Clin Pharmacol Ther\\n2004;76:302-12.\\nDate 12-09-2022\\nCYP2C19 IM: mirtazapine\\n3507\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nGrasmäder K et al. Population pharmacokinetic analysis of mirtazapine. Eur J Clin Pharmacol 2004;60:473-80.\\nDate 14-11-2022\\nCYP2C19 PM: mirtazapine\\n3506\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nJohnson M et al. A poor metabolizer for cytochromes P450 2D6 and 2C19: a case report on antidepressant treatment. CNS Spectr 2006;11:757-60.\\nDate 14-11-2022\\nCYP2C19 UM: mirtazapine\\n3508\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\nDate 14-11-2022\\nCYP2D6 IM: mirtazapine\\n2002\\nNO action is required for this gene-drug interaction.\\nThere is insufficient evidence to suggest that the higher plasma concentration of mirtazapine results in an increase in the side effects or efficacy.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nZastrozhin MS et al. The Influence of concentration of micro-RNA hsa-miR-370-3p and CYP2D6*4 on equilibrium concentration of mirtazapine in patients with major\\ndepressive disorder. Psychopharmacol Bull 2020;50:58-75.\\n3. \\nZastrozhin MS et al. Effects of CYP2D6 activity on the efficacy and safety of mirtazapine in patients with depressive disorders and comorbid alcohol use disorder. Can J\\nPhysiol Pharmacol 2019;97:781-5.\\n4. \\nHayashi Y et al. Factors affecting steady-state plasma concentrations of enantiomeric mirtazapine and its desmethylated metabolites in Japanese psychiatric patients.\\nPharmacopsychiatry 2015;48:279-85.\\n5. \\nOkubo M et al. Effects of cytochrome P450 2D6 and 3A5 genotypes and possible coadministered medicines on the metabolic clearance of antidepressant mirtazapine in\\nJapanese patients. Biochem Pharmacol 2015;93:104-9.\\n6. \\nJaquenoud Sirot E et al. Multicenter study on the clinical effectiveness, pharmacokinetics, and pharmacogenetics of mirtazapine in depression. J Clin Psychopharmacol\\n2012;32:622-9.\\n7. \\nBorobia AM et al. Influence of sex and CYP2D6 genotype on mirtazapine disposition, evaluated in Spanish healthy volunteers. Pharmacol Res 2009;59:393-8.\\n8. \\nLind AB et al. Steady-state concentrations of mirtazapine, N-desmethylmirtazapine, 8-hydroxymirtazapine and their enantiomers in relation to cytochrome P450 2D6\\ngenotype, age and smoking behaviour. Clin Pharmacokinet 2009;48:63-70.\\n9. \\nGrasmader K et al. Population pharmacokinetic analysis of mirtazapine. Eur J Clin Pharmacol 2004;60:473-80.\\n10. \\nMurphy GM Jr et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\nDate 14-11-2022\\nCYP2D6 PM: mirtazapine\\n2001', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 66}),\n",
       " Document(page_content='NO action is required for this gene-drug interaction.\\nThe higher plasma concentration of mirtazapine does not result - or hardly results - in an increase in the side effects.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nMilosavljevic F et al. Association of CYP2C19 and CYP2D6 poor and intermediate metabolizer status with antidepressant and antipsychotic exposure: a systematic review\\nand meta-analysis. JAMA Psychiatry 2021;78:270-80.\\n3. \\nJaquenoud Sirot E et al. Multicenter study on the clinical effectiveness, pharmacokinetics, and pharmacogenetics of mirtazapine in depression. J Clin Psychopharmacol\\n2012;32:622-9.\\n4. \\nRamaekers JG et al. Residual effects of esmirtazapine on actual driving performance: overall findings and an exploratory analysis into the role of CYP2D6 phenotype.\\nPsychopharmacology 2011;215:321-32.\\n5. \\nBorobia AM et al. Influence of sex and CYP2D6 genotype on mirtazapine disposition, evaluated in Spanish healthy volunteers. Pharmacol Res 2009;59:393-8.\\n6. \\nLind AB et al. Steady-state concentrations of mirtazapine, N-desmethylmirtazapine, 8-hydroxymirtazapine and their enantiomers in relation to cytochrome P450 2D6\\ngenotype, age and smoking behaviour. Clin Pharmacokinet 2009;48:63-70.\\n7. \\nBrockmoller J et al. Pharmacokinetics of mirtazapine: enantioselective effects of the CYP2D6 ultra rapid metabolizer genotype and correlation with adverse effects. Clin\\nPharmacol Ther 2007;81:699-707.\\n8. \\nJohnson M et al. A poor metabolizer for cytochromes P450 2D6 and 2C19: a case report on antidepressant treatment. CNS Spectr 2006;11:757-60.\\n9. \\nStephan PL et al. Adverse drug reactions following nonresponse in a depressed patient with CYP2D6 deficiency and low CYP 3A4/5 activity. Pharmacopsychiatry\\n2006;39:150-2.\\n10. \\nKirchheiner J et al. Impact of the CYP2D6 ultrarapid metabolizer genotype on mirtazapine pharmacokinetics and adverse events in healthy volunteers. J Clin\\nPsychopharmacol 2004;24:647-52.\\n11. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and clinical effects of antidepressants in a\\nnaturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n12. \\nMurphy GM Jr et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\nDate 14-11-2022\\nCYP2D6 UM: mirtazapine\\n2003\\nNO action is required for this gene-drug interaction.\\nThe effect on the plasma concentration of mirtazapine is small. No effect has been demonstrated with regard to effectiveness or side effects.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nJaquenoud Sirot E et al. Multicenter study on the clinical effectiveness, pharmacokinetics, and pharmacogenetics of mirtazapine in depression. J Clin Psychopharmacol\\n2012;32:622-9.\\n3. \\nLind AB et al. Steady-state concentrations of mirtazapine, N-desmethylmirtazapine, 8-hydroxymirtazapine and their enantiomers in relation to cytochrome P450 2D6\\ngenotype, age and smoking behaviour. Clin Pharmacokinet 2009;48:63-70.\\n4. \\nBrockmoller J et al. Pharmacokinetics of mirtazapine: enantioselective effects of the CYP2D6 ultra rapid metabolizer genotype and correlation with adverse effects. Clin\\nPharmacol Ther 2007;81:699-707.\\n5. \\nKirchheiner J et al. Impact of the CYP2D6 ultrarapid metabolizer genotype on mirtazapine pharmacokinetics and adverse events in healthy volunteers. J Clin\\nPsychopharmacol 2004;24:647-52.\\n6. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and clinical effects of antidepressants in a\\nnaturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\nDate 14-11-2022\\nCYP2C19 IM: moclobemide\\n1991\\nNO action is needed for this gene-drug interaction.\\nAlthough the moclobemide plasma concentration may increase as a result of the decreased CYP2C19 activity, this does not lead to an increased incidence of side effects, in as\\nfar as is known.\\nLiterature:\\n1. \\nGram LF et al. Moclobemide, a substrate of CYP2C19 and an inhibitor of CYP2C19, CYP2D6, and CYP1A2: a panel study. Clin Pharmacol Ther 1995;57:670-7.\\n2. \\nSmPC Moclobemide Mylan.\\nDate 14-11-2022\\nCYP2C19 PM: moclobemide\\n1992\\nNO action is needed for this gene-drug interaction.\\nAlthough the moclobemide plasma concentration may increase as a result of the decreased CYP2C19 activity, this does not lead to an increased incidence of side effects, in as\\nfar as is known.\\nLiterature:\\n1. \\nYu KS et al. Effect of omeprazole on the pharmacokinetics of moclobemide according to the genetic polymorphism of CYP2C19. Clin Pharmacol Ther 2001;69:266-73.\\n2. \\nGram LF et al. Moclobemide, a substrate of CYP2C19 and an inhibitor of CYP2C19, CYP2D6, and CYP1A2: a panel study. Clin Pharmacol Ther 1995;57:670-7.\\n3. \\nSmPC Moclobemide Mylan.\\nDate 14-11-2022\\nCYP2C19 UM: moclobemide\\n1993', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 67}),\n",
       " Document(page_content='NO action is needed for this gene-drug interaction.\\nAlthough the moclobemide plasma concentration may decrease as a result of increased CYP2C19 activity, this does not lead to increased effectiveness, in as far as is known.\\nLiterature:\\n1. \\nGram LF et al. Moclobemide, a substrate of CYP2C19 and an inhibitor of CYP2C19, CYP2D6, and CYP1A2: a panel study. Clin Pharmacol Ther 1995;57:670-7.\\n2. \\nSmPC Moclobemide Mylan.\\nDate 14-11-2022\\nCYP2D6 IM: nortriptyline\\n1557\\nThe risk of side effects may be increased, because the gene variation leads to an increased plasma concentration of nortriptyline.\\nuse 60% of the standard dose and monitor the effect and side effects or the plasma concentration of nortriptyline in order to set the maintenance dose\\nThe therapeutic range of nortriptyline is 50-150 ng/mL. Values exceeding 250 ng/mL are considered toxic.\\nLiterature:\\n1. \\nBerm E et al. Relation between CYP2D6 genotype, phenotype and therapeutic drug concentrations among nortriptyline and venlafaxine users in old age psychiatry.\\nPharmacopsychiatry 2016;49:186-190.\\n2. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl) 2015;232:2609-17.\\n3. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol 2014;28:133-41.\\n4. \\nLee SY et al. Sequence-based CYP2D6 genotyping in the Korean population. Ther Drug Monit 2006;28:382-7.\\n5. \\nLee S et al. A case report of a poor metabolizer of CYP2D6 presented with unusual responses to nortriptyline medication. J Korean Med Sci 2004;19:750-2.\\n6. \\nDalen P et al. Disposition of debrisoquine and nortriptyline in Korean subjects in relation to CYP2D6 genotypes, and comparison with Caucasians. Br J Clin Pharmacol\\n2003;55:630-4.\\n7. \\nKvist EE et al. Quantitative pharmacogenetics of nortriptyline: a novel approach. Clin Pharmacokinet 2001;40:869-77.\\n8. \\nMorita S et al. Steady-state plasma levels of nortriptyline and its hydroxylated metabolites in Japanese patients: impact of CYP2D6 genotype on the hydroxylation of\\nnortriptyline. J Clin Psychopharmacol 2000;20:141-9.\\n9. \\nYue QJ et al. Pharmacokinetics of nortriptyline and its 10-hydroxymetabolite in Chinese subjects of different CYP2D6 genotypes. Clin Pharmacol Ther 1998;64:384-90.\\n10. \\nChen S et al. The cytochrome P450 2D6 (CYP2D6) enzyme polymorphism: screening costs and influence on clinical outcomes in psychiatry. Clin Pharmacol Ther\\n1996;60:522-34.\\nDate 19-11-2018\\nCYP2D6 PM: nortriptyline\\n1558\\nThe risk of side effects may be increased, because the gene variation leads to an increased plasma concentration of nortriptyline.\\nuse 40% of the standard dose and monitor the effect and side effects or the plasma concentration of nortriptyline in order to set the maintenance dose The therapeutic\\nrange of nortriptyline is 50-150 ng/mL. Values exceeding 250 ng/mL are considered toxic.\\nLiterature:\\n1. \\nBerm E et al. Relation between CYP2D6 genotype, phenotype and therapeutic drug concentrations among nortriptyline and venlafaxine users in old age psychiatry.\\nPharmacopsychiatry 2016;49:186-190.\\n2. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl) 2015;232:2609-17.\\n3. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol 2014;28:133-41.\\n4. \\nRoberts et al. No evidence of increased adverse drug reactions in cytochrome P450 CYP2D6 poor metabolizers treated with fluoxetine or nortriptyline. Hum\\nPsychopharmacol 2004 Jan;19:17-23.\\n5. \\nKvist EE et al. Quantitative pharmacogenetics of nortriptyline: a novel approach. Clin Pharmacokinet 2001;40:869-77.\\n6. \\nDalen P et al. 10-Hydroxylation of nortriptyline in white persons with 0, 1, 2, 3, and 13 functional CYP2D6 genes. Clin Pharmacol Ther 1998;63:444-52.\\n7. \\nDahl M et al. Steady-state plasma levels of nortriptyline and its 10-hydroxymetabolite: relationship to the CYP2D6 genotype. Psychopharmacol 1996;123:315-9.\\n8. \\nChen S et al. The cytochrome P450 2D6 (CYP2D6) enzyme polymorphism: screening costs and influence on clinical outcomes in psychiatry. Clin Pharmacol Ther\\n1996;60:522-34.\\n9. \\nBertilsson L et al. Slow hydroxylation of nortriptyline and concomitant poor debrisoquine hydroxylation: clinical implications. Lancet 1981;1:560-1.\\n10. \\nSmPC Pamelor (VS).\\nDate 19-11-2018\\nCYP2D6 UM: nortriptyline\\n1559\\nThe risk of ineffectiveness and cardiotoxic effects may be increased. The gene variation leads to a decrease in the plasma concentration of nortriptyline and an increase in the\\nplasma concentration of the cardiotoxic metabolite Z-10-hydroxynortriptyline.\\nuse 1.7 times the standard dose and monitor the effect and side effects or the plasma concentration of nortriptyline and be alert to an increase in the plasma\\nconcentration of the cardiotoxic metabolite Z-10-hydroxynortriptyline\\nPlasma concentrations of Z-hydroxynortriptyline exceeding 40 ng/mL are considered toxic.\\nif a dose increase is not wanted due to the cardiotoxic hydroxy metabolite: avoid nortriptyline\\nAntidepressants that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\nLiterature:\\n1. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl) 2015;232:2609-17.\\n2. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol 2014;28:133-41.\\n3. \\nLee SY et al. Sequence-based CYP2D6 genotyping in the Korean population. Ther Drug Monit 2006;28:382-7.\\n4. \\nKvist EE et al. Quantitative pharmacogenetics of nortriptyline: a novel approach. Clin Pharmacokinet 2001;40:869-77.\\n5. \\nDalen P et al. 10-Hydroxylation of nortriptyline in white persons with 0, 1, 2, 3, and 13 functional CYP2D6 genes. Clin Pharmacol Ther 1998;63:444-52.\\n6. \\nBertilsson L et al. Molecular basis for rational megaprescribing in ultrarapid hydroxylators of debrisoquine. Lancet 1993;341:63.\\nDate 19-11-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 68}),\n",
       " Document(page_content='CYP1A2 IM: olanzapine\\n4650\\nThis is NOT a gene-drug interaction.\\nLiterature:\\nDate 13-09-2021\\nCYP1A2 NM: olanzapine\\n4647\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nKoller D et al. Safety and cardiovascular effects of multiple-dose administration of aripiprazole and olanzapine in a randomised clinical trial. Hum Psychopharmacol\\n2021;36:1-12.\\n2. \\nYan P et al. Association of the genetic polymorphisms of metabolizing enzymes, transporters, target receptors and their interactions with treatment response to\\nolanzapine in chinese han schizophrenia patients. Psychiatry Res 2020;293:113470.\\n3. \\nKoller D et al. The effects of aripiprazole and olanzapine on pupillary light reflex and its relationship with pharmacogenetics in a randomized multiple-dose trial. Br J Clin\\nPharmacol 2020;86:2051-62 en persoonlijke communicatie (supplementaire files).\\n4. \\nHattori S et al. The association of genetic polymorphisms in CYP1A2, UGT1A4, and ABCB1 with autonomic nervous system dysfunction in schizophrenia patients treated\\nwith olanzapine. BMC Psychiatry 2020;20:72.\\n5. \\nDjordjevic N et al. Cigarette smoking and heavy coffee consumption affecting response to olanzapine: The role of genetic polymorphism. World J Biol Psychiatry\\n2020;21:29-52.\\n6. \\nNa Takuathung M et al. Impact of CYP1A2 genetic polymorphisms on pharmacokinetics of antipsychotic drugs: a systematic review and meta-analysis. Acta Psychiatr\\nScand 2019;139:15-25.\\n7. \\nCzerwensky F et al. CYP1A2*1D and *1F polymorphisms have a significant impact on olanzapine serum concentrations. Ther Drug Monit 2015; 37:152-60.\\n8. \\nLooman NMG et al. Associatie van genetische variatie in CYP1A2 en UGT1A4 met metabole stoornissen bij gebruikers van clozapine en olanzapine. PW Wetenschappelijk\\nPlatform 2013; 7:a1310.\\n9. \\nSöderberg MM et al. Influence of CYP1A1/CYP1A2 and AHR polymorphisms on systemic olanzapine exposure. Pharmacogenet Genomics 2013;23:279-85.\\n10. \\nCabaleiro T et al. Polymorphisms influencing olanzapine metabolism and adverse effects in healthy subjects. Hum Psychopharmacol 2013;28:205-14.\\n11. \\nSkogh E et al. High correlation between serum and cerebrospinal fluid olanzapine concentrations in patients with schizophrenia or schizoaffective disorder medicating\\nwith oral olanzapine as the only antipsychotic drug. J Clin Psychopharmacol 2011;31:4-9.\\n12. \\nGhotbi R et al. Carriers of the UGT1A4 142T>G gene variant are predisposed to reduced olanzapine exposure--an impact similar to male gender or smoking in\\nschizophrenic patients. Eur J Clin Pharmacol 2010;66:465-74.\\n13. \\nNozawa M et al. The relationship between the response of clinical symptoms and plasma olanzapine concentration, based on pharmacogenetics: Juntendo University\\nSchizophrenia Projects (JUSP). Ther Drug Monit 2008;30:35-40.\\n14. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\nDate 13-09-2021\\nCYP1A2 PM: olanzapine\\n4651\\nThis is NOT a gene-drug interaction.\\nThis is NOT a gene-drug interaction.\\nLiterature:\\nDate 13-09-2021\\nCYP1A2*1A/*1F: olanzapine\\n4646\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nYan P et al. Association of the genetic polymorphisms of metabolizing enzymes, transporters, target receptors and their interactions with treatment response to\\nolanzapine in chinese han schizophrenia patients. Psychiatry Res 2020;293:113470.\\n2. \\nHattori S et al. The association of genetic polymorphisms in CYP1A2, UGT1A4, and ABCB1 with autonomic nervous system dysfunction in schizophrenia patients treated\\nwith olanzapine. BMC Psychiatry 2020;20:72.\\n3. \\nNa Takuathung M et al. Impact of CYP1A2 genetic polymorphisms on pharmacokinetics of antipsychotic drugs: a systematic review and meta-analysis. Acta Psychiatr\\nScand 2019;139:15-25.\\n4. \\nCzerwensky F et al. CYP1A2*1D and *1F polymorphisms have a significant impact on olanzapine serum concentrations. Ther Drug Monit 2015; 37:152-60.\\n5. \\nLooman NMG et al. Associatie van genetische variatie in CYP1A2 en UGT1A4 met metabole stoornissen bij gebruikers van clozapine en olanzapine. PW Wetenschappelijk\\nPlatform 2013; 7:a1310.\\n6. \\nSöderberg MM et al. Influence of CYP1A1/CYP1A2 and AHR polymorphisms on systemic olanzapine exposure. Pharmacogenet Genomics 2013;23:279-85.\\n7. \\nCabaleiro T et al. Polymorphisms influencing olanzapine metabolism and adverse effects in healthy subjects. Hum Psychopharmacol 2013;28:205-14.\\n8. \\nSkogh E et al. High correlation between serum and cerebrospinal fluid olanzapine concentrations in patients with schizophrenia or schizoaffective disorder medicating\\nwith oral olanzapine as the only antipsychotic drug. J Clin Psychopharmacol 2011;31:4-9.\\n9. \\nGhotbi R et al. Carriers of the UGT1A4 142T>G gene variant are predisposed to reduced olanzapine exposure--an impact similar to male gender or smoking in\\nschizophrenic patients. Eur J Clin Pharmacol 2010;66:465-74.\\n10. \\nNozawa M et al. The relationship between the response of clinical symptoms and plasma olanzapine concentration, based on pharmacogenetics: Juntendo University\\nSchizophrenia Projects (JUSP). Ther Drug Monit 2008;30:35-40.\\n11. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\nDate 13-09-2021', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 69}),\n",
       " Document(page_content='CYP1A2*1C-heterozygoot: olanzapine\\n4648\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nKoller D et al. Safety and cardiovascular effects of multiple-dose administration of aripiprazole and olanzapine in a randomised clinical trial. Hum Psychopharmacol\\n2021;36:1-12.\\n2. \\nYan P et al. Association of the genetic polymorphisms of metabolizing enzymes, transporters, target receptors and their interactions with treatment response to\\nolanzapine in chinese han schizophrenia patients. Psychiatry Res 2020;293:113470.\\n3. \\nKoller D et al. The effects of aripiprazole and olanzapine on pupillary light reflex and its relationship with pharmacogenetics in a randomized multiple-dose trial. Br J Clin\\nPharmacol 2020;86:2051-62 en persoonlijke communicatie (supplementaire files).\\n4. \\nHattori S et al. The association of genetic polymorphisms in CYP1A2, UGT1A4, and ABCB1 with autonomic nervous system dysfunction in schizophrenia patients treated\\nwith olanzapine. BMC Psychiatry 2020;20:72.\\n5. \\nLooman NMG et al. Associatie van genetische variatie in CYP1A2 en UGT1A4 met metabole stoornissen bij gebruikers van clozapine en olanzapine. PW Wetenschappelijk\\nPlatform 2013; 7:a1310.\\n6. \\nCabaleiro T et al. Polymorphisms influencing olanzapine metabolism and adverse effects in healthy subjects. Hum Psychopharmacol 2013;28:205-14.\\n7. \\nNozawa M et al. The relationship between the response of clinical symptoms and plasma olanzapine concentration, based on pharmacogenetics: Juntendo University\\nSchizophrenia Projects (JUSP). Ther Drug Monit 2008;30:35-40.\\n8. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\nDate 13-09-2021\\nCYP1A2*1C/*1C: olanzapine\\n4649\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nYan P et al. Association of the genetic polymorphisms of metabolizing enzymes, transporters, target receptors and their interactions with treatment response to\\nolanzapine in chinese han schizophrenia patients. Psychiatry Res 2020;293:113470.\\n2. \\nHattori S et al. The association of genetic polymorphisms in CYP1A2, UGT1A4, and ABCB1 with autonomic nervous system dysfunction in schizophrenia patients treated\\nwith olanzapine. BMC Psychiatry 2020;20:72.\\n3. \\nNozawa M et al. The relationship between the response of clinical symptoms and plasma olanzapine concentration, based on pharmacogenetics: Juntendo University\\nSchizophrenia Projects (JUSP). Ther Drug Monit 2008;30:35-40.\\n4. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\nDate 13-09-2021\\nCYP2D6 IM: olanzapine\\n1560\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nZubiaur P et al. Impact of polymorphisms in transporter and metabolizing enzyme genes on olanzapine pharmacokinetics and safety in healthy volunteers. Biomed\\nPharmacother 2021;133:111087.\\n2. \\nMiroshnichenko II et al. Therapeutic drug monitoring of olanzapine and cytochrome P450 genotyping in nonsmoking subjects. Ther Drug Monit 2020;42:325-9.\\n3. \\nDjordjevic N et al. Cigarette smoking and heavy coffee consumption affecting response to olanzapine: The role of genetic polymorphism. World J Biol Psychiatry\\n2020;21:29-52.\\n4. \\nOkubo M et al. Individual differences in in vitro and in vivo metabolic clearances of the antipsychotic drug olanzapine from non-smoking and smoking Japanese subjects\\ngenotyped for cytochrome P4502D6 and flavincontaining monooxygenase 3. Hum Psychopharmacol 2016;31:83-92.\\n5. \\nCabaleiro T et al. Polymorphisms influencing olanzapine metabolism and adverse effects in healthy subjects. Hum Psychopharmacol 2013;28:205-14.\\n6. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\n7. \\nNozawa M et al. The relationship between the response of clinical symptoms and plasma olanzapine concentration, based on pharmacogenetics: Juntendo University\\nSchizophrenia Projects (JUSP). Ther Drug Monit 2008;30:35-40.\\n8. \\nIwahashi K. Olanzapine metabolism by CYP1A2/CYP2D6 and hyperglycaemia. Acta Neuropsychiatrica 2004;16:229-230.\\n9. \\nEllingrod VL et al. CYP2D6 polymorphisms and atypical antipsychotic weight gain. Psychiatr Genet 2002;12:55-8.\\nDate 13-09-2021\\nCYP2D6 PM: olanzapine\\n1561\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nZubiaur P et al. Impact of polymorphisms in transporter and metabolizing enzyme genes on olanzapine pharmacokinetics and safety in healthy volunteers. Biomed\\nPharmacother 2021;133:111087.\\n2. \\nDjordjevic N et al. Cigarette smoking and heavy coffee consumption affecting response to olanzapine: The role of genetic polymorphism. World J Biol Psychiatry\\n2020;21:29-52.\\n3. \\nCabaleiro T et al. Polymorphisms influencing olanzapine metabolism and adverse effects in healthy subjects. Hum Psychopharmacol 2013;28:205-14.\\n4. \\nSkogh E et al. High correlation between serum and cerebrospinal fluid olanzapine concentrations in patients with schizophrenia or schizoaffective disorder medicating\\nwith oral olanzapine as the only antipsychotic drug. J Clin Psychopharmacol 2011;31:4-9.\\n5. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\n6. \\nCarrillo JA et al. Role of the smoking-induced cytochrome P450 (CYP)1A2 and polymorphic CYP2D6 in steady-state concentration of olanzapine. J Clin Psychopharmacol\\n2003;23:119-27.\\n7. \\nHagg S et al. Olanzapine disposition in humans is unrelated to CYP1A2 and CYP2D6 phenotypes. Eur J Clin Pharmacol 2001;57:493-7.\\nDate 13-09-2021\\nCYP2D6 UM: olanzapine\\n1562', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 70}),\n",
       " Document(page_content=\"This is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nZubiaur P et al. Impact of polymorphisms in transporter and metabolizing enzyme genes on olanzapine pharmacokinetics and safety in healthy volunteers. Biomed\\nPharmacother 2021;133:111087.\\n2. \\nCabaleiro T et al. Polymorphisms influencing olanzapine metabolism and adverse effects in healthy subjects. Hum Psychopharmacol 2013;28:205-14.\\n3. \\nSkogh E et al. High correlation between serum and cerebrospinal fluid olanzapine concentrations in patients with schizophrenia or schizoaffective disorder medicating\\nwith oral olanzapine as the only antipsychotic drug. J Clin Psychopharmacol 2011;31:4-9.\\nDate 13-09-2021\\nCYP2C19 IM: omeprazol\\n1839\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of omeprazole results in an increase in the therapeutic effectiveness, without an increase in the side effects.\\nLiterature:\\n1. \\nPark S et al. Effects of CYP2C19 genetic polymorphisms on PK/PD responses of omeprazole in Korean healthy volunteers. J Korean Med Sci 2017;32:729-736.\\n2. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3. \\nWang L et al. Ilaprazole for the treatment of duodenal ulcer: a randomized, double-blind and controlled phase III trial. Curr Med Res Opin 2012;28:101-9.\\n4. \\nZendehdel N et al. Role of cytochrome P450 2C19 genetic polymorphisms in the therapeutic efficacy of omeprazole in Iranian patients with erosive reflux esophagitis.\\nArch Iran Med 2010;13:406-12.\\n5. \\nHelsby NA et al. Omeprazole-induced acute interstitial nephritis is not related to CYP2C19 genotype or CYP2C19 phenotype. Br J Clin Pharmacol 2010;69:516-9.\\n6. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n7. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in Caucasians. Br J Clin Pharmacol 2008;65:752-60.\\n8. \\nOhkusa T et al. Effect of CYP2C19 polymorphism on the safety and efficacy of omeprazole in Japanese patients with recurrent reflux esophagitis. Aliment Pharmacol Ther\\n2005;21:1331-9.\\n9. \\nRoh HK et al. Omeprazole treatment of Korean patients: effects on gastric pH and gastrin release in relation to CYP2C19 geno- and phenotypes. Basic Clin Pharmacol\\nToxicol 2004;95:112-9.\\n10. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH. Aliment Pharmacol Ther 2001;15:1929-37.\\n11. \\nSagar M et al. Effects of omeprazole on intragastric pH and plasma gastrin are dependent on the CYP2C19 polymorphism. Gastroenterology 2000;119:670-6.\\n12. \\nFuruta T et al. CYP2C19 genotype status and effect of omeprazole on intragastric pH in humans. Clin Pharmacol Ther 1999;65:552-61.\\n13. \\nShimatani T et al. Effect of omeprazole 10 mg on intragastric pH in three different CYP2C19 genotypes, compared with omeprazole 20 mg and lafutidine 20 mg, a new\\nH2-receptor antagonist. Aliment Pharmacol Ther 2003;18:1149–1157.\\n14. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to CYP2C19 genotype. Dig Dis Sci 2008;53:933-\\n7.\\n15. \\nSugimoto M et al. Initial 48-hour acid inhibition by intravenous infusion of omeprazole, famotidine, or both in relation to cytochrome P450 2C19 genotype status. Clin\\nPharmacol Ther 2006;80:539-48.\\n16. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19 genotypic differences. Dig Dis Sci\\n2005;50:1625-31.\\n17. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n18. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of clarithromycin-sensitive strains of Helicobacter\\npylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n19. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders with Helicobacter pylori infection. Eur J Clin\\nPharmacol 2005;61:375-9.\\n20. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to CYP2C19 metabolism. Aliment Pharmacol\\nTher 2005;21:283-8.\\n21. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin Gastroenterol Hepatol 2004;2:22-30.\\n22. \\nSapone A et al. The clinical role of cytochrome p450 genotypes in Helicobacter pylori management. Am J Gastroenterol 2003;98:1010-5.\\n23. \\nMiwa H et al. Clarithromycin resistance, but not CYP2C-19 polymorphism, has a major impact on treatment success in 7-day treatment regimen for cure of H. pylori\\ninfection: a multiple logistic regression analysis. Dig Dis Sci 2001;46:2445-50.\\n24. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton pump inhibitor (omeprazole or\\nrabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n25. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a proton pump inhibitor, amoxicillin, and\\nclarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n26. \\nAoyama N et al. Sufficient effect of 1-week omeprazole and amoxicillin dual treatment for Helicobacter pylori eradication in cytochrome P450 2C19 poor metabolizers. J\\nGastroenterol 1999;34 Suppl 11:80-3.\\n27. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n28. \\nHu XP et al. Effects of CYP2C19 genetic polymorphism on the pharmacokinetics and pharmacodynamics of omeprazole in Chinese people. J Clin Pharm Ther 2007;32:517-\\n24.\\n29. \\nSmPC Prilosec (VS).\\nDate 05-03-2018\\nCYP2C19 PM: omeprazol\\n1840\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of omeprazole results in an increase in the therapeutic effectiveness, without an increase in the side effects.\\nLiterature:\\n1. \\nPark S et al. Effects of CYP2C19 genetic polymorphisms on PK/PD responses of omeprazole in Korean healthy volunteers. J Korean Med Sci 2017;32:729-736.\\n2. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3. \\nWang L et al. Ilaprazole for the treatment of duodenal ulcer: a randomized, double-blind and controlled phase III trial. Curr Med Res Opin 2012;28:101-9.\\n4. \\nHelsby NA et al. Omeprazole-induced acute interstitial nephritis is not related to CYP2C19 genotype or CYP2C19 phenotype. Br J Clin Pharmacol 2010;69:516-9.\\n5. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n6. \\nOhkusa T et al. Effect of CYP2C19 polymorphism on the safety and efficacy of omeprazole in Japanese patients with recurrent reflux esophagitis. Aliment Pharmacol Ther\\n2005;21:1331-9.\\n7. \\nRoh HK et al. Omeprazole treatment of Korean patients: effects on gastric pH and gastrin release in relation to CYP2C19 geno- and phenotypes. Basic Clin Pharmacol\\nToxicol 2004;95:112-9.\\n8. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH. Aliment Pharmacol Ther 2001;15:1929-37.\\n9. \\nSagar M et al. Effects of omeprazole on intragastric pH and plasma gastrin are dependent on the CYP2C19 polymorphism. Gastroenterology 2000;119:670-6.\\n10. \\nFuruta T et al. CYP2C19 genotype status and effect of omeprazole on intragastric pH in humans. Clin Pharmacol Ther 1999;65:552-61.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 71}),\n",
       " Document(page_content=\"11. \\nShimatani T et al. Effect of omeprazole 10 mg on intragastric pH in three different CYP2C19 genotypes, compared with omeprazole 20 mg and lafutidine 20 mg, a new\\nH2-receptor antagonist. Aliment Pharmacol Ther 2003;18:1149–1157.\\n12. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to CYP2C19 genotype. Dig Dis Sci 2008;53:933-\\n7.\\n13. \\nSugimoto M et al. Initial 48-hour acid inhibition by intravenous infusion of omeprazole, famotidine, or both in relation to cytochrome P450 2C19 genotype status. Clin\\nPharmacol Ther 2006;80:539-48.\\n14. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases. J Gastroenterol Hepatol 2006;21:1381-\\n7.\\n15. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19 genotypic differences. Dig Dis Sci\\n2005;50:1625-31.\\n16. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n17. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of clarithromycin-sensitive strains of Helicobacter\\npylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n18. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to CYP2C19 metabolism. Aliment Pharmacol\\nTher 2005;21:283-8.\\n19. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin Gastroenterol Hepatol 2004;2:22-30.\\n20. \\nSapone A et al. The clinical role of cytochrome p450 genotypes in Helicobacter pylori management. Am J Gastroenterol 2003;98:1010-5.\\n21. \\nMiwa H et al. Clarithromycin resistance, but not CYP2C-19 polymorphism, has a major impact on treatment success in 7-day treatment regimen for cure of H. pylori\\ninfection: a multiple logistic regression analysis. Dig Dis Sci 2001;46:2445-50.\\n22. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton pump inhibitor (omeprazole or\\nrabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n23. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a proton pump inhibitor, amoxicillin, and\\nclarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n24. \\nTanigawara Y et al. CYP2C19 genotype-related efficacy of omeprazole for the treatment of infection caused by Helicobacter pylori. Clin Pharmacol Ther 1999;66:528-34.\\n25. \\nAoyama N et al. Sufficient effect of 1-week omeprazole and amoxicillin dual treatment for Helicobacter pylori eradication in cytochrome P450 2C19 poor metabolizers. J\\nGastroenterol 1999;34 Suppl 11:80-3.\\n26. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n27. \\nHu XP et al. Effects of CYP2C19 genetic polymorphism on the pharmacokinetics and pharmacodynamics of omeprazole in Chinese people. J Clin Pharm Ther 2007;32:517-\\n24.\\n28. \\nSmPC’s Losec en Prilosec (VS).\\nDate 05-03-2018\\nCYP2C19 UM: omeprazol\\n1841\\nThe genetic variation may lead to a reduced omeprazole plasma concentration and therefore reduced effectiveness.\\nRecommendation:\\nFor Helicobacter pylori ERADICATION THERAPY:\\n1. \\nuse a 3-fold higher dose\\n2. \\nadvise the patient to contact their doctor if symptoms of dyspepsia persist\\nOTHER INDICATIONS:\\n1. \\nbe alert to reduced effectiveness\\n2. \\nif necessary, use a 3-fold higher dose\\n3. \\nadvise the patient to report persisting symptoms of dyspepsia\\nLiterature:\\n1. \\nPark S et al. Effects of CYP2C19 genetic polymorphisms on PK/PD responses of omeprazole in Korean healthy volunteers. J Korean Med Sci 2017;32:729-736.\\n2. \\nChwiesko A et al. Effects of different omeprazole dosing on gastric pH in non-variceal upper gastrointestinal bleeding: a randomized prospective study. J Dig Dis\\n2016;17:588-599.\\n3. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n4. \\nDury S et al. Agranulocytosis induced by proton pump inhibitors. J Clin Gastroenterol 2012;46:859.\\n5. \\nWang L et al. Ilaprazole for the treatment of duodenal ulcer: a randomized, double-blind and controlled phase III trial. Curr Med Res Opin 2012;28:101-9.\\n6. \\nZendehdel N et al. Role of cytochrome P450 2C19 genetic polymorphisms in the therapeutic efficacy of omeprazole in Iranian patients with erosive reflux esophagitis.\\nArch Iran Med 2010;13:406-12.\\n7. \\nHelsby NA et al. Omeprazole-induced acute interstitial nephritis is not related to CYP2C19 genotype or CYP2C19 phenotype. Br J Clin Pharmacol 2010;69:516-9.\\n8. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n9. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in Caucasians. Br J Clin Pharmacol 2008;65:752-60.\\n10. \\nOhkusa T et al. Effect of CYP2C19 polymorphism on the safety and efficacy of omeprazole in Japanese patients with recurrent reflux esophagitis. Aliment Pharmacol Ther\\n2005;21:1331-9.\\n11. \\nRoh HK et al. Omeprazole treatment of Korean patients: effects on gastric pH and gastrin release in relation to CYP2C19 geno- and phenotypes. Basic Clin Pharmacol\\nToxicol 2004;95:112-9.\\n12. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH. Aliment Pharmacol Ther 2001;15:1929-37.\\n13. \\nSagar M et al. Effects of omeprazole on intragastric pH and plasma gastrin are dependent on the CYP2C19 polymorphism. Gastroenterology 2000;119:670-6.\\n14. \\nFuruta T et al. CYP2C19 genotype status and effect of omeprazole on intragastric pH in humans. Clin Pharmacol Ther 1999;65:552-61.\\n15. \\nShimatani T et al. Effect of omeprazole 10 mg on intragastric pH in three different CYP2C19 genotypes, compared with omeprazole 20 mg and lafutidine 20 mg, a new\\nH2-receptor antagonist. Aliment Pharmacol Ther 2003;18:1149–1157.\\n16. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to CYP2C19 genotype. Dig Dis Sci 2008;53:933-\\n7.\\n17. \\nSugimoto M et al. Initial 48-hour acid inhibition by intravenous infusion of omeprazole, famotidine, or both in relation to cytochrome P450 2C19 genotype status. Clin\\nPharmacol Ther 2006;80:539-48.\\n18. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases. J Gastroenterol Hepatol 2006;21:1381-\\n7.\\n19. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19 genotypic differences. Dig Dis Sci\\n2005;50:1625-31.\\n20. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n21. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of clarithromycin-sensitive strains of Helicobacter\\npylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n22. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders with Helicobacter pylori infection. Eur J Clin\\nPharmacol 2005;61:375-9.\\n23. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to CYP2C19 metabolism. Aliment Pharmacol\\nTher 2005;21:283-8.\\n24. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin Gastroenterol Hepatol 2004;2:22-30.\\n25. \\nSapone A et al. The clinical role of cytochrome p450 genotypes in Helicobacter pylori management. Am J Gastroenterol 2003;98:1010-5.\\n26. \\nMiwa H et al. Clarithromycin resistance, but not CYP2C-19 polymorphism, has a major impact on treatment success in 7-day treatment regimen for cure of H. pylori\\ninfection: a multiple logistic regression analysis. Dig Dis Sci 2001;46:2445-50.\\n27. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton pump inhibitor (omeprazole or\\nrabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n28. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a proton pump inhibitor, amoxicillin, and\\nclarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n29. \\nTanigawara Y et al. CYP2C19 genotype-related efficacy of omeprazole for the treatment of infection caused by Helicobacter pylori. Clin Pharmacol Ther 1999;66:528-34.\\n30. \\nAoyama N et al. Sufficient effect of 1-week omeprazole and amoxicillin dual treatment for Helicobacter pylori eradication in cytochrome P450 2C19 poor metabolizers. J\\nGastroenterol 1999;34 Suppl 11:80-3.\\n31. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n32. \\nRocha A et al. Investigation of the in vivo activity of CYP3A in Brazilian volunteers: comparison of midazolam and omeprazole as drug markers. Eur J Clin Pharmacol\\n2008;64:901-6.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 72}),\n",
       " Document(page_content='33. \\nBaldwin RM et al. Increased omeprazole metabolism in carriers of the CYP2C19*17 allele; a pharmacokinetic study in healthy volunteers. Br J Clin Pharmacol\\n2008;65:767-74.\\n34. \\nHu XP et al. Effects of CYP2C19 genetic polymorphism on the pharmacokinetics and pharmacodynamics of omeprazole in Chinese people. J Clin Pharm Ther 2007;32:517-\\n24.\\n35. \\nSim SC et al. A common novel CYP2C19 gene variant causes ultrarapid drug metabolism relevant for the drug response to proton pump inhibitors and antidepressants.\\nClin Pharmacol Ther 2006;79:103-13.\\n36. \\nSmPC’s Losec en Prilosec (VS).\\nDate 05-03-2018\\nHLA-B*1502: oxcarbazepine\\n6931\\nStevens-Johnson syndrome, the severe cutaneous side effect that can potentially result in permanent damage, occurs more often in patients with this genetic variation. The\\ncalculated risk of oxcarbazepine-induced SJS in patients with HLA-B*1502 is 0.73%.\\ncarefully weigh the risk of SJS against the benefits\\navoid oxcarbazepine if an alternative is available\\nCarbamazepine carries a 10-fold higher risk of SJS/TEN in these patients and is therefore not an alternative.\\nIn these patients, phenytoin and lamotrigine carry a similar risk of SJS/TEN as oxcarbazepine, but more severe forms of SJS/TEN (SJS/TEN overlap and TEN) are also\\nobserved with these medicines. Therefore, they are also not suitable as alternatives.\\nif it is not possible to avoid oxcarbazepine, advise the patient to report any rash immediately\\nLiterature:\\n1. \\nLee KH et al. Reducing severe cutaneous adverse and type B adverse drug reactions using pre-stored human leukocyte antigen genotypes. Clin Transl Allergy\\n2022;12:e12098.\\n2. \\nSukasem C et al. Spectrum of cutaneous adverse reactions to aromatic antiepileptic drugs and human leukocyte antigen genotypes in Thai patients and meta-analysis.\\nPharmacogenomics J 2021;21:682-90.\\n3. \\nLiu H et al. Association of the HLA-B*15:02 allele with pure cervical lymphadenopathy induced by carbamazepine and oxcarbazepine: a case report and literature review.\\nBasic Clin Pharmacol Toxicol 2021;128:348-51.\\n4. \\nLiu Y et al. Association between HLA-B*15:02 and oxcarbazepine-induced cutaneous adverse reaction: a meta-analysis. Pharmacogenomics 2018;19:547-52.\\n5. \\nTangamornsuksan W et al. Association between HLA genotypes and oxcarbazepine-induced cutaneous adverse drug reactions: a systematic review and meta-analysis. J\\nPharm Pharm Sci 2018;21:1-18.\\n6. \\nChen CB et al. Risk and association of HLA with oxcarbazepine-induced cutaneous adverse reactions in Asians. Neurology 2017;88:78-86.\\n7. \\nSun D et al. Association of HLA-B*1502 and *1511 allele with antiepileptic drug-induced Stevens-Johnson syndrome in central China. J Huazhong Univ Sci Technolog Med\\nSci 2014;34:146-50.\\n8. \\nAmstutz U et al. HLA-A*31:01 and HLA-B*15:02 as genetic markers for carbamazepine hypersensitivity in children. Clin Pharmacol Ther 2013;94:142-9.\\n9. \\nWal P et al. Genetic predisposition to oxcarbazepine induced Stevens-Johnson syndrome. Indian J Crit Care Med 2011;15:173-5.\\n10. \\nHung SI et al. Common risk allele in aromatic antiepileptic-drug induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Han Chinese. Pharmacogenomics\\n2010;11:349-56.\\n11. \\nShankarkumar U et al. HLA B*1502 allele association with oxcarbamazepine-induced skin reactions in epilepsy patient from India. Epilepsia 2009;50:1837-8.\\n12. \\nLin LC et al. Oxcarbazepine-induced Stevens-Johnson syndrome: a case report. Kaohsiung J Med Sci 2009;25:82-6.\\n13. \\nSmPC’s Trileptal (NL) en Oxtellar XR (VS).\\nDate 23-05-2022\\nCYP2D6 IM: oxycodon\\n1587\\nNO action is required for this gene-drug interaction.\\nThe reduced conversion of oxycodone to the more active metabolite oxymorphone does not result in reduced analgesia for patients.\\nLiterature:\\n1. \\nCajanus K et al. Analgesic plasma concentrations of oxycodone after surgery for breast cancer-which factors matter? Clin Pharmacol Ther 2017 Jun 23 [Epub ahead of\\nprint].\\n2. \\nStamer UM et al. CYP2D6 genotype dependent oxycodone metabolism in postoperative patients. PloS One 2013;8:e60239.\\n3. \\nNaito T et al. CYP3A5*3 affects plasma disposition of noroxycodone and dose escalation in cancer patients receiving oxycodone. J Clin Pharmacol 2011;51:1529-38.\\n4. \\nLemberg KK et al. Does co-administration of paroxetine change oxycodone analgesia: an interaction study in chronic pain patients. Scan Jour Pain 2010:1;24-33.\\n5. \\nComer SD et al. Abuse liability of oxycodone as a function of pain and drug use history. Drug Alcohol Depend 2010;109:130-8.\\n6. \\nJannetto PJ et al. Utilization of pharmacogenomics and therapeutic drug monitoring for opioid pain management. Pharmacogenomics 2009;10:1157-67.\\nDate 20-11-2017\\nCYP2D6 PM: oxycodon\\n1586\\nNO action is required for this gene-drug interaction.\\nThe reduced conversion of oxycodone to the more active metabolite oxymorphone does not result in reduced analgesia in patients.\\nLiterature:\\n1. \\nCajanus K et al. Analgesic plasma concentrations of oxycodone after surgery for breast cancer-which factors matter? Clin Pharmacol Ther 2017 Jun 23 [Epub ahead of\\nprint].\\n2. \\nLam J et al. Putative association of ABCB1 2677G>T/A with oxycodone-induced central nervous system depression in breastfeeding mothers. Ther Drug Monit\\n2013;35:466-72.\\n3. \\nStamer UM et al. CYP2D6 genotype dependent oxycodone metabolism in postoperative patients. PloS One 2013;8:e60239.\\n4. \\nAndreassen TN et al. Do CYP2D6 genotypes reflect oxycodone requirements for cancer patients treated for cancer pain? A cross-sectional multicentre study. Eur J Clin\\nPharmacol 2012;68:55-64.\\n5. \\nZwisler ST et al. Impact of CYP2D6 genotype on post-operative intravenous oxycodone analgesia. Acta Anaesthesiol Scan 2010;54:232-40.\\n6. \\nSamer CF et al. Genetic polymorphisms and drug interactions modulating CYP2D6 and CYP3A activities have a major effect on oxycodone analgesic efficacy and safety.\\nBr J Pharmacol 2010;16:919-30.\\n7. \\nComer SD et al. Abuse liability of oxycodone as a function of pain and drug use history. Drug Alcohol Depend 2010;109:130-8.\\n8. \\nJannetto PJ et al. Utilization of pharmacogenomics and therapeutic drug monitoring for opioid pain management. Pharmacogenomics 2009;10:1157-67.\\n9. \\nZwisler ST et al. The hypoalgesic effect of oxycodone in human experimental pain models in relation to the CYP2D6 oxidation polymorphism. Basic Clin Pharmacol Toxicol\\n2009;104:335-44.\\n10. \\nFoster A et al. Complicated pain management in a CYP450 2D6 poor metabolizer. Pain Pract 2007;7:352-6.\\n11. \\nSusce MT et al. Response to hydrocodone, codeine and oxycodone in a CYP2D6 poor metabolizer. Prog Neuropsychopharmacol Biol Psychiatry 2006;30:1356-8.\\n12. \\nMaddocks I et al. Attenuation of morphine-induced delirium in palliative care by substitution with infusion of oxycodone. J Pain Symptom Manage 1996;12:182-9.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 73}),\n",
       " Document(page_content='Date 20-11-2017\\nCYP2D6 UM: oxycodon\\n1588\\nNO action is required for this gene-drug interaction.\\nThe increased conversion of oxycodone to the more active metabolite oxymorphone does not result in an increase in side effects in patients.\\nLiterature:\\n1. \\nCajanus K et al. Analgesic plasma concentrations of oxycodone after surgery for breast cancer-which factors matter? Clin Pharmacol Ther 2017 Jun 23 [Epub ahead of\\nprint].\\n2. \\nLam J et al. Putative association of ABCB1 2677G>T/A with oxycodone-induced central nervous system depression in breastfeeding mothers. Ther Drug Monit\\n2013;35:466-72.\\n3. \\nStamer UM et al. CYP2D6 genotype dependent oxycodone metabolism in postoperative patients. PloS One 2013;8:e60239.\\n4. \\nAndreassen TN et al. Do CYP2D6 genotypes reflect oxycodone requirements for cancer patients treated for cancer pain? A cross-sectional multicentre study. Eur J Clin\\nPharmacol 2012;68:55-64.\\n5. \\nLemberg KK et al. Does co-administration of paroxetine change oxycodone analgesia: an interaction study in chronic pain patients. Scan Jour Pain 2010:1;24-33.\\n6. \\nSamer CF et al. Genetic polymorphisms and drug interactions modulating CYP2D6 and CYP3A activities have a major effect on oxycodone analgesic efficacy and safety.\\nBr J Pharmacol 2010;16:919-30.\\n7. \\nde Leon J et al. Adverse drug reactions to oxycodone and hydrocodone in CYP2D6 ultrarapid metabolizers. J Clin Psychopharmacol 2003;23:420-1.\\nDate 20-11-2017\\nCYP3A4 IM: paclitaxel\\n7107\\nNO action is required for this gene-drug interaction.\\nThe genetic variation has a slight effect on the metabolism of paclitaxel, and there is insufficient evidence of an increase in adverse events or reduced efficacy.\\nLiterature:\\n1. \\nLam SW et al. Genotypes of CYP2C8 and FGD4 and their association with peripheral neuropathy or early dose reduction in paclitaxel-treated breast cancer patients. Br J\\nCancer 2016;115:1335-42.\\n2. \\nApellániz-Ruiz M et al. Whole-exome sequencing reveals defective CYP3A4 variants predictive of paclitaxel dose-limiting neuropathy. Clin Cancer Res 2015;21:322-8.\\n3. \\nAssis J et al. Influence of CYP3A4 genotypes in the outcome of serous ovarian cancer patients treated with first-line chemotherapy: implication of a CYP3A4 activity\\nprofile. Int J Clin Exp Med 2013;6:552-61.\\n4. \\nde Graan AJ et al. CYP3A4*22 genotype and systemic exposure affect paclitaxel-induced neurotoxicity. Clin Cancer Res 2013;19:3316-24.\\n5. \\nNakajima Y et al. Impact of the haplotype CYP3A4*16B harboring the Thr185Ser substitution on paclitaxel metabolism in Japanese patients with cancer. Clin Pharmacol\\nTher 2006;80:179-91.\\n6. \\nNakajima M et al. Pharmacokinetics of paclitaxel in ovarian cancer patients and genetic polymorphisms of CYP2C8, CYP3A4, and MDR1. J Clin Pharmacol 2005;45: 674-\\n82.\\nDate 04-11-2019\\nCYP3A4 PM: paclitaxel\\n7113\\nNO action is required for this gene-drug interaction.\\nThe genetic variation has a slight effect on the metabolism of paclitaxel, and there is insufficient evidence of an increase in adverse events or reduced efficacy.\\nLiterature:\\n1. \\nLam SW et al. Genotypes of CYP2C8 and FGD4 and their association with peripheral neuropathy or early dose reduction in paclitaxel-treated breast cancer patients. Br J\\nCancer 2016;115:1335-42.\\n2. \\nAssis J et al. Influence of CYP3A4 genotypes in the outcome of serous ovarian cancer patients treated with first-line chemotherapy: implication of a CYP3A4 activity\\nprofile. Int J Clin Exp Med 2013;6:552-61.\\n3. \\nde Graan AJ et al. CYP3A4*22 genotype and systemic exposure affect paclitaxel-induced neurotoxicity. Clin Cancer Res 2013;19:3316-24.\\nDate 04-11-2019\\nCYP2C19 IM: pantoprazol\\n1847\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of pantoprazole results in an increase in the therapeutic effectiveness, without an increase in the side effects.\\nLiterature:\\n1. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with proton pump inhibitors. Meta Gene\\n2016;9:159-64.\\n2. \\nOrmeci A et al. Effect of cytochrome P450 2C19 polymorphisms on the Helicobacter pylori eradication rate following two-week triple therapy with pantoprazole or\\nrabeprazole. Eur Rev Med Pharmacol Sci 2016;20:879-85.\\n3. \\nHsu PI et al. A randomized controlled study comparing reverse hybrid therapy and standard triple therapy for Helicobacter pylori infection. Medicine (Baltimore)\\n2015;94:e2104.\\n4. \\nRomán M et al. Evaluation of the relationship between polymorphisms in CYP2C19 and the pharmacokinetics of omeprazole, pantoprazole and rabeprazole.\\nPharmacogenomics 2014;15:1893-901.\\n5. \\nGawrońska-Szklarz B et al. CYP2C19 polymorphism affects single-dose pharmacokinetics of oral pantoprazole in healthy volunteers. Eur J Clin Pharmacol 2012;68:1267-\\n74.\\n6. \\nSheu BS et al. CYP2C19 genotypes determine the efficacy of on-demand therapy of pantoprazole for reflux esophagitis as Los-Angeles grades C and D. J Gastroenterol\\nHepatol 2012;27:104-9.\\n7. \\nThacker DL et al. Stereoselective pharmacokinetics of stable isotope (+/-)-[13C]-pantoprazole: Implications for a rapid screening phenotype test of CYP2C19 activity.\\nChirality 2011;23:904-9.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 74}),\n",
       " Document(page_content='8. \\nChen WY et al. Double-dosed pantoprazole accelerates the sustained symptomatic response in overweight and obese patients with reflux esophagitis in Los Angeles\\ngrades A and B. Am J Gastroenterol 2010;105:1046-52.\\n9. \\nGawrońska-Szklarz B et al. Effects of CYP2C19, MDR1, and interleukin 1-B gene variants on the eradication rate of Helicobacter pylori infection by triple therapy with\\npantoprazole, amoxicillin, and metronidazole. Eur J Clin Pharmacol 2010;66:681-7.\\n10. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics and CYP2C19 polymorphism. Aliment\\nPharmacol Ther 2010;31:150-9.\\n11. \\nHunfeld NG et al. Effect of CYP2C19 \\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in Caucasians. Br J Clin Pharmacol 2008;65:752-\\n60.\\n12. \\nChoi KD et al. Optimal dose of intravenous pantoprazole in patients with peptic ulcer bleeding requiring endoscopic hemostasis in Korea. J Gastroenterol Hepatol\\n2009;24:1617-24.\\n13. \\nOh JH et al. Low-dose intravenous pantoprazole for optimal inhibition of gastric acid in Korean patients. J Gastroenterol Hepatol 2007;22:1429-34.\\n14. \\nOh JH et al. Effects of CYP2C19 and MDR1 genotype on the eradication rate of Helicobacter pylori infection by triple therapy with pantoprazole, amoxycillin and\\nclarithromycin. J Gastroenterol Hepatol 2009;24:294-8.\\n15. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple therapy with regular proton pump inhibitor\\ndosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n16. \\nKurzawski M et al. Effect of CYP2C19*17 gene variant on Helicobacter pylori eradication in peptic ulcer patients. Eur J Clin Pharmacol 2006;62:877-80.\\n17. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders with Helicobacter pylori infection. Eur J Clin\\nPharmacol 2005;61:375-9.\\n18. \\nKearns GL et al. Single-dose pharmacokinetics of oral and intravenous pantoprazole in children and adolescents. J Clin Pharmacol 2008;48:1356-65.\\nDate 05-03-2018\\nCYP2C19 PM: pantoprazol\\n1848\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of pantoprazole results in an increase in the therapeutic effectiveness, without an increase in the side effects.\\nLiterature:\\n1. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with proton pump inhibitors. Meta Gene\\n2016;9:159-64.\\n2. \\nOrmeci A et al. Effect of cytochrome P450 2C19 polymorphisms on the Helicobacter pylori eradication rate following two-week triple therapy with pantoprazole or\\nrabeprazole. Eur Rev Med Pharmacol Sci 2016;20:879-85.\\n3. \\nHsu PI et al. A randomized controlled study comparing reverse hybrid therapy and standard triple therapy for Helicobacter pylori infection. Medicine (Baltimore)\\n2015;94:e2104.\\n4. \\nGawrońska-Szklarz B et al. CYP2C19 polymorphism affects single-dose pharmacokinetics of oral pantoprazole in healthy volunteers. Eur J Clin Pharmacol 2012;68:1267-\\n74.\\n5. \\nSheu BS et al. CYP2C19 genotypes determine the efficacy of on-demand therapy of pantoprazole for reflux esophagitis as Los-Angeles grades C and D. J Gastroenterol\\nHepatol 2012;27:104-9.\\n6. \\nThacker DL et al. Stereoselective pharmacokinetics of stable isotope (+/-)-[13C]-pantoprazole: Implications for a rapid screening phenotype test of CYP2C19 activity.\\nChirality 2011;23:904-9.\\n7. \\nChen WY et al. Double-dosed pantoprazole accelerates the sustained symptomatic response in overweight and obese patients with reflux esophagitis in Los Angeles\\ngrades A and B. Am J Gastroenterol 2010;105:1046-52.\\n8. \\nGawrońska-Szklarz B et al. Effects of CYP2C19, MDR1, and interleukin 1-B gene variants on the eradication rate of Helicobacter pylori infection by triple therapy with\\npantoprazole, amoxicillin, and metronidazole. Eur J Clin Pharmacol 2010;66:681-7.\\n9. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19 genotypes. J Clin Gastroenterol 2009;43:920-5.\\n10. \\nHunfeld NG et al. Effect of CYP2C19 \\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in Caucasians. Br J Clin Pharmacol 2008;65:752-\\n60.\\n11. \\nChoi KD et al. Optimal dose of intravenous pantoprazole in patients with peptic ulcer bleeding requiring endoscopic hemostasis in Korea. J Gastroenterol Hepatol\\n2009;24:1617-24.\\n12. \\nOh JH et al. Low-dose intravenous pantoprazole for optimal inhibition of gastric acid in Korean patients. J Gastroenterol Hepatol 2007;22:1429-34.\\n13. \\nOh JH et al. Effects of CYP2C19 and MDR1 genotype on the eradication rate of Helicobacter pylori infection by triple therapy with pantoprazole, amoxycillin and\\nclarithromycin. J Gastroenterol Hepatol 2009;24:294-8.\\n14. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple therapy with regular proton pump inhibitor\\ndosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n15. \\nKurzawski M et al. Effect of CYP2C19*17 gene variant on Helicobacter pylori eradication in peptic ulcer patients. Eur J Clin Pharmacol 2006;62:877-80.\\n16. \\nKearns GL et al. Single-dose pharmacokinetics of oral and intravenous pantoprazole in children and adolescents. J Clin Pharmacol 2008;48:1356-65.\\n17. \\nSPC’s Pantozol en Protonix I.V. (VS).\\nDate 05-03-2018\\nCYP2C19 UM: pantoprazol\\n1849\\nThe genetic variation may lead to reduced pantoprazole plasma concentrations and therefore reduced pantoprazole effectiveness.\\nRecommendation:\\nFor Helicobacter pylori ERADICATION THERAPY:\\n1. \\nuse a 5-fold higher dose\\n2. \\nadvise the patient to contact their doctor if symptoms of dyspepsia persist\\nOTHER INDICATIONS:\\n1. \\nbe alert to reduced effectiveness\\n2. \\nif necessary, use a 5-fold higher dose\\n3. \\nadvise the patient to report persisting symptoms of dyspepsia\\nLiterature:\\n1. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with proton pump inhibitors. Meta Gene\\n2016;9:159-64.\\n2. \\nOrmeci A et al. Effect of cytochrome P450 2C19 polymorphisms on the Helicobacter pylori eradication rate following two-week triple therapy with pantoprazole or\\nrabeprazole. Eur Rev Med Pharmacol Sci 2016;20:879-85.\\n3. \\nHsu PI et al. A randomized controlled study comparing reverse hybrid therapy and standard triple therapy for Helicobacter pylori infection. Medicine (Baltimore)\\n2015;94:e2104.\\n4. \\nRomán M et al. Evaluation of the relationship between polymorphisms in CYP2C19 and the pharmacokinetics of omeprazole, pantoprazole and rabeprazole.\\nPharmacogenomics 2014;15:1893-901.\\n5. \\nGawrońska-Szklarz B et al. CYP2C19 polymorphism affects single-dose pharmacokinetics of oral pantoprazole in healthy volunteers. Eur J Clin Pharmacol 2012;68:1267-\\n74.\\n6. \\nSheu BS et al. CYP2C19 genotypes determine the efficacy of on-demand therapy of pantoprazole for reflux esophagitis as Los-Angeles grades C and D. J Gastroenterol\\nHepatol 2012;27:104-9.\\n7. \\nThacker DL et al. Stereoselective pharmacokinetics of stable isotope (+/-)-[13C]-pantoprazole: Implications for a rapid screening phenotype test of CYP2C19 activity.\\nChirality 2011;23:904-9.\\n8. \\nChen WY et al. Double-dosed pantoprazole accelerates the sustained symptomatic response in overweight and obese patients with reflux esophagitis in Los Angeles\\ngrades A and B. Am J Gastroenterol 2010;105:1046-52.\\n9. \\nGawrońska-Szklarz B et al. Effects of CYP2C19, MDR1, and interleukin 1-B gene variants on the eradication rate of Helicobacter pylori infection by triple therapy with\\npantoprazole, amoxicillin, and metronidazole. Eur J Clin Pharmacol 2010;66:681-7.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 75}),\n",
       " Document(page_content='10. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics and CYP2C19 polymorphism. Aliment\\nPharmacol Ther 2010;31:150-9.\\n11. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19 genotypes. J Clin Gastroenterol 2009;43:920-5.\\n12. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in Caucasians. Br J Clin Pharmacol 2008;65:752-60.\\n13. \\nChoi KD et al. Optimal dose of intravenous pantoprazole in patients with peptic ulcer bleeding requiring endoscopic hemostasis in Korea. J Gastroenterol Hepatol\\n2009;24:1617-24.\\n14. \\nOh JH et al. Low-dose intravenous pantoprazole for optimal inhibition of gastric acid in Korean patients. J Gastroenterol Hepatol 2007;22:1429-34.\\n15. \\nOh JH et al. Effects of CYP2C19 and MDR1 genotype on the eradication rate of Helicobacter pylori infection by triple therapy with pantoprazole, amoxycillin and\\nclarithromycin. J Gastroenterol Hepatol 2009;24:294-8.\\n16. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple therapy with regular proton pump inhibitor\\ndosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n17. \\nKurzawski M et al. Effect of CYP2C19*17 gene variant on Helicobacter pylori eradication in peptic ulcer patients. Eur J Clin Pharmacol 2006;62:877-80.\\n18. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders with Helicobacter pylori infection. Eur J Clin\\nPharmacol 2005;61:375-9.\\n19. \\nKearns GL et al. Single-dose pharmacokinetics of oral and intravenous pantoprazole in children and adolescents. J Clin Pharmacol 2008;48:1356-65.\\n20. \\nSPC’s Pantozol en Protonix I.V. (VS).\\nDate 05-03-2018\\nCYP2D6 IM: paroxetine\\n1563\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of paroxetine can increase as a result of the reduced activity of CYP2D6. However, studies did not find any clinical effects.\\nLiterature:\\n1. \\nJanssen PK et al. Nonresponders to daily paroxetine and another SSRI in men with lifelong premature ejaculation: a pharmacokinetic dose-escalation study for a rare\\nphenomenon. Korean J Urol 2014;55:599-607.\\n2. \\nSaruwatari J et al. Possible impact of the CYP2D6*10 polymorphism on the nonlinear pharmacokinetic parameter estimates of paroxetine in Japanese patients with major\\ndepressive disorders. Pharmgenomics Pers Med 2014;7:121-7.\\n3. \\nMurata Y et al. Severe sleepiness and excess sleep duration induced by paroxetine treatment is a beneficial pharmacological effect, not an adverse reaction. J Affect\\nDisord 2013;150:1209-12.\\n4. \\nMurata Y et al. Effects of the serotonin 1A, 2A, 2C, 3A, and 3B and serotonin transporter gene polymorphisms on the occurrence of paroxetine discontinuation syndrome.\\nJ Clin Psychopharmacol 2010;30:11-7.\\n5. \\nVervers FF et al. Effect of cytochrome P450 2D6 genotype on maternal paroxetine plasma concentrations during pregnancy. Clin Pharmacokinet 2009;48:677-83.\\n6. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6 polymorphisms. Int J Psychiatry Clin Pract\\n2009;13:345-348.\\n7. \\nGex-Fabry M et al. CYP2D6 and ABCB1 genetic variability: influence on paroxetine plasma level and therapeutic response. Ther Drug Monit 2008;30:474-82.\\n8. \\nKuhn UD et al. Reboxetine and cytochrome P450--comparison with paroxetine treatment in humans. Int J Clin Pharmacol Ther 2007;45:36-46.\\n9. \\nSugai T et al. The effect of 5-hydroxytryptamine 3A and 3B receptor genes on nausea induced by paroxetine. Pharmacogenomics J 2006;6:351-6.\\n10. \\nFindling RL et al. Multiple dose pharmacokinetics of paroxetine in children and adolescents with major depressive disorder or obsessive-compulsive disorder.\\nNeuropsychopharmacology 2006;31:1274-85.\\n11. \\nFeng Y et al. Paroxetine: population pharmacokinetic analysis in late-life depression using sparse concentration sampling. Br J Clin Pharmacol 2006;61:558-69.\\n12. \\nUeda M et al. The impact of CYP2D6 genotypes on the plasma concentration of paroxetine in Japanese psychiatric patients. Prog Neuro-psychopharmacol Biol Psychiatry\\n2006;30:486-91.\\n13. \\nSawamura K ea. Effects of dosage and CYP2D6-mutated allele on plasma concentration of paroxetine. Eur J Clin Pharmacol 2004;60:553-7.\\n14. \\nMurphy G ea. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\n15. \\nOzdemir V ea. Paroxetine steady-state plasma concentration in relation to CYP2D6 genotype in extensive metabolizers. J Clin Psychopharmacol 1999;19:472-5.\\nDate 14-05-2018\\nCYP2D6 PM: paroxetine\\n1564\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of paroxetine can increase as a result of the reduced activity of CYP2D6. However, studies did not find any clinical effects.\\nLiterature:\\n1. \\nVervers FF et al. Effect of cytochrome P450 2D6 genotype on maternal paroxetine plasma concentrations during pregnancy. Clin Pharmacokinet 2009;48:677-83.\\n2. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6 polymorphisms. Int J Psychiatry Clin Pract\\n2009;13:345-348.\\n3. \\nGex-Fabry M et al. CYP2D6 and ABCB1 genetic variability: influence on paroxetine plasma level and therapeutic response. Ther Drug Monit 2008;30:474-82.\\n4. \\nKuhn UD et al. Reboxetine and cytochrome P450--comparison with paroxetine treatment in humans. Int J Clin Pharmacol Ther 2007;45:36-46.\\n5. \\nFindling RL et al. Multiple dose pharmacokinetics of paroxetine in children and adolescents with major depressive disorder or obsessive-compulsive disorder.\\nNeuropsychopharmacology 2006;31:1274-85.\\n6. \\nFeng Y et al. Paroxetine: population pharmacokinetic analysis in late-life depression using sparse concentration sampling. Br J Clin Pharmacol 2006;61:558-69.\\n7. \\nSawamura K et al. Effects of dosage and CYP2D6-mutated allele on plasma concentration of paroxetine. Eur J Clin Pharmacol 2004;60:553-7.\\n8. \\nCharlier C et al. Polymorphisms in the CYP 2D6 gene: association with plasma concentrations of fluoxetine and paroxetine. Ther Drug Monit 2003;25:738-42.\\n9. \\nMurphy G et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\n10. \\nSindrup SH et al. Pharmacokinetics of the selective serotonin reuptake inhibitor paroxetine: nonlinearity and relation to the sparteine oxidation polymorphism. Clin\\nPharmacol Ther 1992;51:288-95.\\n11. \\nSindrup SH et al. The relationship between paroxetine and the sparteine oxidation polymorphism. Clin Pharmacol Ther 1992;51:278-87.\\nDate 14-05-2018\\nCYP2D6 UM: paroxetine\\n1565\\nEfficacy will probably be lacking. The genetic variation increases the conversion of paroxetine.\\nIt is not possible to offer substantiated advice for dose adjustment based on the literature.\\navoid paroxetine\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include for example citalopram or sertraline.\\nLiterature:\\n1. \\nGex-Fabry M et al. CYP2D6 and ABCB1 genetic variability: influence on paroxetine plasma level and therapeutic response. Ther Drug Monit 2008;30:474-82.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 76}),\n",
       " Document(page_content='2. \\nFindling RL et al. Multiple dose pharmacokinetics of paroxetine in children and adolescents with major depressive disorder or obsessive-compulsive disorder.\\nNeuropsychopharmacology 2006;31:1274-85.\\n3. \\nFeng Y et al. Paroxetine: population pharmacokinetic analysis in late-life depression using sparse concentration sampling. Br J Clin Pharmacol 2006;61:558-69.\\n4. \\nGüzey C et al. Low serum concentrations of paroxetine in CYP2D6 ultrarapid metabolizers. J Clin Psychopharmacol 2006;26:211-2.\\n5. \\nCharlier C et al. Polymorphisms in the CYP 2D6 gene: association with plasma concentrations of fluoxetine and paroxetine. Ther Drug Monit 2003;25:738-42.\\n6. \\nMurphy G et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\nDate 14-05-2018\\nCYP2D6 IM: pimozide\\n2448\\nThe risk of QT-prolongation – and thereby also the risk of torsade de points – is theoretically increased, because the genetic variation results in an increase in the plasma\\nconcentration of pimozide. The elevated plasma concentration and associated theoretical increased risk of QT elongation can be negated by following the dose\\nrecommendations provided below.\\n- use no more than the following doses (80% of the standard maximum dose):     - 12 years and older: 16 mg/day    - younger than 12 years: 0.08 mg/kg per day to a maximum of 3 mg/day\\nLiterature:\\n1. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 en persoonlijke communicatie (gemiddelde dosisgecorrigeerde pimozidedalconcentraties).\\n2. \\nNucci G et al. Population pharmacokinetic modelling of pimozide and its relation to CYP2D6 genotype. Poster presented at the annual meeting of population approach\\ngroup in Europe 2007.\\nDate 13-09-2021\\nCYP2D6 PM: pimozide\\n2447\\nThe risk of QT-prolongation – and thereby also the risk of torsade de points – is theoretically increased, because the genetic variation results in an increase in the plasma\\nconcentration of pimozide. The elevated plasma concentration and associated theoretical increased risk of QT elongation can be negated by following the dose\\nrecommendations provided below.\\n- use no more than the following doses (50% of the standard maximum dose):     - 12 years and older: 10 mg/day    - younger than 12 years: 0.05 mg/kg per day to a maximum of 2 mg/day\\nLiterature:\\n1. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 en persoonlijke communicatie (gemiddelde dosisgecorrigeerde pimozidedalconcentraties).\\n2. \\nNucci G et al. Population pharmacokinetic modelling of pimozide and its relation to CYP2D6 genotype. Poster presented at the annual meeting of population approach\\ngroup in Europe 2007.\\n3. \\nDesta Z et al. Effect of clarithromycin on the pharmacokinetics and pharmacodynamics of pimozide in healthy poor and extensive metabolizers of cytochrome P450 2D6\\n(CYP2D6). Clin Pharmacol Ther 1999;65:10-20.\\n4. \\nPharmacogenetic changes to the FDA-approved Orap (pimozide) label include adult and pediatric dosing recommendations for CYP2D6 poor metabolizers. FDA-\\nnieuwsbericht 27-09-11.\\n5. \\nSmPC Orap (NL en VS).\\nDate 13-09-2021\\nCYP2D6 UM: pimozide\\n2449\\nNO action is required for this gene-drug interaction.\\nThis gene variation can result in lower pimozide concentrations. However, there is no evidence of reduced effectiveness.\\nLiterature:\\n1. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 en persoonlijke communicatie (gemiddelde dosisgecorrigeerde pimozidedalconcentraties).\\nDate 13-09-2021\\nCYP2C19 IM: prasugrel\\n2546\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous coronary intervention. Circ Genom Precis\\nMed 2018;11:e002069.\\n2. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events in Japanese patients with acute coronary\\nsyndrome: The PRASFIT-ACS study. J Cardiol 2016;68:29-36.\\n3. \\nDoll JA et al. Impact of CYP2C19 metabolizer status on patients with ACS treated with prasugrel versus clopidogrel. J Am Coll Cardiol 2016;67:936-47.\\n4. \\nVarenhorst C et al. Genetic variation of CYP2C19 affects both pharmacokinetic and pharmacodynamic responses to clopidogrel but not prasugrel in aspirin-treated\\npatients with coronary artery disease. Eur Heart J 2009;30:1744-52.\\n5. \\nMega JL et al. Cytochrome P450 genetic polymorphisms and the response to prasugrel: relationship to pharmacokinetic, pharmacodynamic, and clinical outcomes.\\nCirculation 2009;119:2553-60.\\n6. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response to clopidogrel but not prasugrel. J Thromb\\nHaemost 2007;5:2429-36.\\n7. \\nSPC Efient (NL en VS).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 77}),\n",
       " Document(page_content='Date 19-11-2018\\nCYP2C19 PM: prasugrel\\n2545\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous coronary intervention. Circ Genom Precis\\nMed 2018;11:e002069.\\n2. \\nDeiman BA et al. Reduced number of cardiovascular events and increased cost-effectiveness by genotype-guided antiplatelet therapy in patients undergoing percutaneous\\ncoronary interventions in the Netherlands. Neth Heart J 2016;24:589-99.\\n3. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events in Japanese patients with acute coronary\\nsyndrome: The PRASFIT-ACS study. J Cardiol 2016;68:29-36.\\n4. \\nDoll JA et al. Impact of CYP2C19 metabolizer status on patients with ACS treated with prasugrel versus clopidogrel. J Am Coll Cardiol 2016;67:936-47.\\n5. \\nVarenhorst C et al. Genetic variation of CYP2C19 affects both pharmacokinetic and pharmacodynamic responses to clopidogrel but not prasugrel in aspirin-treated\\npatients with coronary artery disease. Eur Heart J 2009;30:1744-52.\\n6. \\nMega JL et al. Cytochrome P450 genetic polymorphisms and the response to prasugrel: relationship to pharmacokinetic, pharmacodynamic, and clinical outcomes.\\nCirculation 2009;119:2553-60.\\n7. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response to clopidogrel but not prasugrel. J Thromb\\nHaemost 2007;5:2429-36.\\n8. \\nSPC Efient (NL en VS).\\nDate 19-11-2018\\nCYP2C19 UM: prasugrel\\n2547\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nMega JL et al. Cytochrome P450 genetic polymorphisms and the response to prasugrel: relationship to pharmacokinetic, pharmacodynamic, and clinical outcomes.\\nCirculation 2009;119:2553-60.\\n2. \\nSPC Efient (NL en VS).\\nDate 19-11-2018\\nCYP2D6 IM: propafenon\\n1596\\nGenetic variation increases the sum of the plasma concentrations of propafenone and the active metabolite 5-hydroxypropafenone. This may increase the risk of side effects.\\nRecommendation:\\nIt is not possible to offer adequately substantiated recommendations for dose adjustment based on the literature.\\n1. \\nEither guide the dose by therapeutic drug monitoring, perform an ECG and be alert to side effects\\n2. \\nOr choose an alternative\\nAntiarrhythmic drugs that are hardly if at all metabolised by CYP2D6 include, for example, sotalol, disopyramide, quinidine and amiodarone.\\nLiterature:\\n1. \\nSunthankar SD et al. Influence of CYP2D6 genetic variation on adverse events with propafenone in the pediatric and young adult population. Clin Transl Sci 2022 May 5\\n[online ahead of print].\\n2. \\nDoki K et al. Effect of CYP2D6 genetic polymorphism on peak propafenone concentration: no significant effect of CYP2D6*10. Pharmacogenomics 2020;21:1279-88.\\n3. \\nMörike K et al. Propafenone for the prevention of atrial tachyarrhythmias after cardiac surgery: a randomized, double-blind placebo-controlled trial. Clin Pharmacol Ther\\n2008;84:104-10.\\n4. \\nChen B et al. Influence of CYP2D6*10B genotype on pharmacokinetics of propafenone enantiomers in Chinese subjects. Acta Pharmacol Sin 2003;24:1277-80.\\n5. \\nCai WM et al. Effect of CYP2D6*10 genotype on propafenone pharmacodynamics in Chinese patients with ventricular arrhythmia. Acta Pharmacol Sin 2002;23:1040-4.\\n6. \\nCai WM et al. Simultaneous modeling of pharmacokinetics and pharmacodynamics of propafenone in healthy subjects. Acta Pharmacol Sin 2001;22:956-60.\\nDate 12-09-2022\\nCYP2D6 PM: propafenon\\n1595\\nGenetic variation increases the sum of the plasma concentrations of propafenone and the active metabolite 5-hydroxypropafenone. This increases the risk of side effects.\\nRecommendation:\\n1. \\nReduce the dose to 30% of the standard dose, perform an ECG and monitor plasma concentrations\\nLiterature:\\n1. \\nSunthankar SD et al. Influence of CYP2D6 genetic variation on adverse events with propafenone in the pediatric and young adult population. Clin Transl Sci 2022 May 5\\n[online ahead of print].\\n2. \\nMörike K et al. Propafenone for the prevention of atrial tachyarrhythmias after cardiac surgery: a randomized, double-blind placebo-controlled trial. Clin Pharmacol Ther\\n2008;84:104-10.\\n3. \\nJazwinska-Tarnawska E et al. The influence of CYP2D6 polymorphism on the antiarrhythmic efficacy of propafenone in patients with paroxysmal atrial fibrillation during 3\\nmonths propafenone prophylactic treatment. Int J Clin Pharmacol Ther 2001;39:288-92.\\n4. \\nChow MS et al. Evaluation of CYP2D6 oxidation of dextromethorphan and propafenone in a Chinese population with atrial fibrillation. J Clin Pharmacol 2001;41:92-6.\\n5. \\nLabbe L et al. Pharmacokinetic and pharmacodynamic interaction between mexiletine and propafenone in human beings. Clin Pharmacol Ther 2000;68:44-57.\\n6. \\nDilger K et al. Consequences of rifampicin treatment on propafenone disposition in extensive and poor metabolizers of CYP2D6. Pharmacogenetics 1999;9:551-9.\\n7. \\nCai WM et al. The influence of CYP2D6 activity on the kinetics of propafenone enantiomers in Chinese subjects. Br J Clin Pharmacol 1999;47:553-6.\\n8. \\nMorike K et al. Propafenone in a usual dose produces severe side-effects: the impact of genetically determined metabolic status on drug therapy. J Intern Med\\n1995;238:469-72.\\n9. \\nMorike KE et al. Quinidine-enhanced beta-blockade during treatment with propafenone in extensive metabolizer human subjects. Clin Pharmacol Ther 1994;55:28-34.\\n10. \\nLee JT et al. The role of genetically determined polymorphic drug metabolism in the beta-blockade produced by propafenone. N Engl J Med 1990;21:1764-8.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 78}),\n",
       " Document(page_content='11. \\nSiddoway LA et al. Polymorphism of propafenone metabolism and disposition in man: clinical and pharmacokinetic consequences. Circulation 1987;75:785-91.\\n12. \\nSmPC Rytmonorm.\\n13. \\nSmPC Rythmol SR (VS).\\nDate 12-09-2022\\nCYP2D6 UM: propafenon\\n1597\\nGenetic variation decreases the sum of the plasma concentrations of propafenone and the active metabolite 5-hydroxypropafenone. This increases the risk of reduced or no\\nefficacy.\\nRecommendation:\\nIt is not possible to offer adequately substantiated recommendations for dose adjustment based on the literature.\\n1. \\nEither monitor plasma concentrations, perform an ECG and be alert to reduced efficacy of the therapy.\\n2. \\nOr choose an alternative\\nAntiarrhythmic drugs that are hardly if at all metabolised by CYP2D6 include, for example, sotalol, disopyramide, quinidine and amiodarone.\\nLiterature:\\n1. \\nSunthankar SD et al. Influence of CYP2D6 genetic variation on adverse events with propafenone in the pediatric and young adult population. Clin Transl Sci 2022 May 5\\n[online ahead of print].\\n2. \\nMörike K et al. Propafenone for the prevention of atrial tachyarrhythmias after cardiac surgery: a randomized, double-blind placebo-controlled trial. Clin Pharmacol Ther\\n2008;84:104-10.\\n3. \\nJazwinska-Tarnawska E et al. The influence of CYP2D6 polymorphism on the antiarrhythmic efficacy of propafenone in patients with paroxysmal atrial fibrillation during 3\\nmonths propafenone prophylactic treatment. Int J Clin Pharmacol Ther 2001;39:288-92.\\nDate 12-09-2022\\nCYP2D6 IM: quetiapine\\n2394\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nStäuble CK et al. Severe adverse drug reactions to quetiapine in two patients carrying CYP2D6*4 variants: a case report. Int J Mol Sci 2021;22:6480.\\n2. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and chlorpromazine in the Chinese Han population.\\nPharmacogenomics J 2016;16:357-65.\\n3. \\nBakken GV et al. Impact of genetic variability in CYP2D6, CYP3A5, and ABCB1 on serum concentrations of quetiapine and N-desalkylquetiapine in psychiatric patients.\\nTher Drug Monit 2015;37:256-61.\\n4. \\nKato D et al. Delirium resolving upon switching from risperidone to quetiapine: implication of CYP2D6 genotype. Psychosomatics 2005;46:374-5.\\nDate 13-09-2021\\nCYP2D6 PM: quetiapine\\n2393\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and chlorpromazine in the Chinese Han population.\\nPharmacogenomics J 2016;16:357-65.\\n2. \\nBakken GV et al. Impact of genetic variability in CYP2D6, CYP3A5, and ABCB1 on serum concentrations of quetiapine and N-desalkylquetiapine in psychiatric patients.\\nTher Drug Monit 2015;37:256-61.\\nDate 13-09-2021\\nCYP2D6 UM: quetiapine\\n2395\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nBakken GV et al. Impact of genetic variability in CYP2D6, CYP3A5, and ABCB1 on serum concentrations of quetiapine and N-desalkylquetiapine in psychiatric patients.\\nTher Drug Monit 2015;37:256-61.\\n2. \\nKhazaal Y et al. Use of high doses of quetiapine in bipolar disorder episodes are not linked to high activity of cytochrome P450 3A4 and/or cytochrome P4502D6.\\nPsychiatr Q 2013;84:329-35.\\nDate 13-09-2021\\nCYP3A4 IM: quetiapine\\n5992\\nNO action is needed for this gene-drug interaction.\\nThis gene variation reduces the conversion of quetiapine to inactive metabolites and a metabolite with anti-depressant effect. However, the effect on the plasma concentration\\nof quetiapine is limited (20% increase) and it is not known whether this has any clinical consequences. The relationship between the plasma concentration and clinical effect is\\nweak for quetiapine.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 79}),\n",
       " Document(page_content=\"Literature:\\n1. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism on serum concentration of quetiapine in psychiatric patients. J Clin Psychopharmacol 2014;34:256-\\n60. En persoonlijke communicatie.\\nDate 13-09-2021\\nCYP3A4 PM: quetiapine\\n5991\\nThe plasma concentration of quetiapine is 3.2-fold higher in these patients. In addition, the formation of the active metabolite N-desalkylquetiapine, which is probably\\nresponsible for the antidepressant effect, should be reduced. The gene variation results in reduced activity of the enzyme CYP3A4, which converts quetiapine to N-\\ndesalkylquetiapine and an inactive metabolite.\\nindication DEPRESSION\\nchoose an alternative\\nAripiprazole appears to be less dependent on CYP3A4 for metabolism. Olanzapine is not metabolised by CYP3A4.\\nOTHER INDICATIONS\\nuse 30% of the standard dose\\nLiterature:\\n1. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism on serum concentration of quetiapine in psychiatric patients. J Clin Psychopharmacol 2014;34:256-\\n60. En persoonlijke communicatie.\\nDate 13-09-2021\\nCYP2C19 IM: rabeprazol\\n1856\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of rabeprazole does not result in an increase in side effects.\\nLiterature:\\n1. \\nNakamura K et al. Limited effect of rebamipide in addition to proton pump inhibitor (PPI) in the treatment of post-endoscopic submucosal dissection gastric ulcers: a\\nrandomized controlled trial comparing PPI plus rebamipide combination therapy with PPI monotherapy. Gut Liver 2016;10:917-924.\\n2. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3. \\nKinoshita Y et al. Randomised clinical trial: a multicentre, double-blind, placebo- controlled study on the efficacy and safety of rabeprazole 5 mg or 10 mg once daily in\\npatients with non-erosive reflux disease. Aliment Pharmacol Ther 2011;33:213-24.\\n4. \\nLay CS et al. Correlation of CYP2C19 genetic polymorphisms with Helicobacter pylori eradication in patients with cirrhosis and peptic ulcer. J Chin Med Assoc\\n2010;73:188-93.\\n5. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n6. \\nYamano HO et al. Plasma concentration of rabeprazole after 8-week administration in gastroesophageal reflux disease patients and intragastric pH elevation. J\\nGastroenterol Hepatol 2008;23:534-40.\\n7. \\nLee YC et al. Influence of cytochrome P450 2C19 genetic polymorphism and dosage of rabeprazole on accuracy of proton-pump inhibitor testing in Chinese patients with\\ngastroesophageal reflux disease. J Gastroenterol Hepatol 2007;22:1286-92.\\n8. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J Gastroenterol Hepatol 2007;22:815-20.\\n9. \\nAriizumi K et al. Therapeutic effects of 10 mg/day rabeprazole administration on reflux esophagitis was not influenced by the CYP2C19 polymorphism. J Gastroenterol\\nHepatol 2006;21:1428-34.\\n10. \\nHu YM et al. Pharmacodynamic and kinetic effect of rabeprazole on serum gastrin level in relation to CYP2C19 polymorphism in Chinese Hans. World J Gastroenterol\\n2006;12:4750-3.\\n11. \\nSugimoto M et al. Comparison of an increased dosage regimen of rabeprazole versus a concomitant dosage regimen of famotidine with rabeprazole for nocturnal gastric\\nacid inhibition in relation to cytochrome P450 2C19 genotypes. Clin Pharmacol Ther 2005;77:302-11.\\n12. \\nSugimoto M et al. Different dosage regimens of rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450 2C19 genotype status. Clin Pharmacol\\nTher 2004;76:290-301.\\n13. \\nShimatani T et al. Rabeprazole 10 mg twice daily is superior to 20 mg once daily for night-time gastric acid suppression. Aliment Pharmacol Ther 2004;19:113-22.\\n14. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH. Aliment Pharmacol Ther 2001;15:1929-37.\\n15. \\nHorai Y et al. Pharmacodynamic effects and kinetic disposition of rabeprazole in relation to CYP2C19 genotypes. Aliment Pharmacol Ther 2001;15:793-803.\\n16. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol Ther 2000;14:1259-66.\\n17. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to CYP2C19 genotype. Dig Dis Sci 2008;53:933-\\n7.\\n18. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19 genotypic differences. Dig Dis Sci\\n2005;50:1625-31.\\n19. \\nYang JC et al. Pharmacokinetic- pharmacodynamic analysis of the role of CYP2C19 genotypes in short-term rabeprazole-based triple therapy against Helicobacter pylori.\\nBr J Clin Pharmacol 2009;67:503-10.\\n20. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n21. \\nKuwayama H et al. Rabeprazole-based eradication therapy for Helicobacter pylori: a large-scale study in Japan. Aliment Pharmacol Ther 2007;25:1105-13.\\n22. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori infection with lansoprazole- or\\nrabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n23. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton pump inhibitor (omeprazole or\\nrabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n24. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n25. \\nSPC Pariet.\\nDate 05-03-2018\\nCYP2C19 PM: rabeprazol\\n1857\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of rabeprazole does not result in an increase in side effects.\\nLiterature:\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 80}),\n",
       " Document(page_content=\"1. \\nNakamura K et al. Limited effect of rebamipide in addition to proton pump inhibitor (PPI) in the treatment of post-endoscopic submucosal dissection gastric ulcers: a\\nrandomized controlled trial comparing PPI plus rebamipide combination therapy with PPI monotherapy. Gut Liver 2016;10:917-924.\\n2. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3. \\nKinoshita Y et al. Randomised clinical trial: a multicentre, double-blind, placebo- controlled study on the efficacy and safety of rabeprazole 5 mg or 10 mg once daily in\\npatients with non-erosive reflux disease. Aliment Pharmacol Ther 2011;33:213-24.\\n4. \\nLay CS et al. Correlation of CYP2C19 genetic polymorphisms with Helicobacter pylori eradication in patients with cirrhosis and peptic ulcer. J Chin Med Assoc\\n2010;73:188-93.\\n5. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19 genotypes. J Clin Gastroenterol 2009;43:920-5.\\n6. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n7. \\nYamano HO et al. Plasma concentration of rabeprazole after 8-week administration in gastroesophageal reflux disease patients and intragastric pH elevation. J\\nGastroenterol Hepatol 2008;23:534-40.\\n8. \\nLee YC et al. Influence of cytochrome P450 2C19 genetic polymorphism and dosage of rabeprazole on accuracy of proton-pump inhibitor testing in Chinese patients with\\ngastroesophageal reflux disease. J Gastroenterol Hepatol 2007;22:1286-92.\\n9. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J Gastroenterol Hepatol 2007;22:815-20.\\n10. \\nAriizumi K et al. Therapeutic effects of 10 mg/day rabeprazole administration on reflux esophagitis was not influenced by the CYP2C19 polymorphism. J Gastroenterol\\nHepatol 2006;21:1428-34.\\n11. \\nHu YM et al. Pharmacodynamic and kinetic effect of rabeprazole on serum gastrin level in relation to CYP2C19 polymorphism in Chinese Hans. World J Gastroenterol\\n2006;12:4750-3.\\n12. \\nSugimoto M et al. Comparison of an increased dosage regimen of rabeprazole versus a concomitant dosage regimen of famotidine with rabeprazole for nocturnal gastric\\nacid inhibition in relation to cytochrome P450 2C19 genotypes. Clin Pharmacol Ther 2005;77:302-11.\\n13. \\nSugimoto M et al. Different dosage regimens of rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450 2C19 genotype status. Clin Pharmacol\\nTher 2004;76:290-301.\\n14. \\nShimatani T et al. Rabeprazole 10 mg twice daily is superior to 20 mg once daily for night-time gastric acid suppression. Aliment Pharmacol Ther 2004;19:113-22.\\n15. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH. Aliment Pharmacol Ther 2001;15:1929-37.\\n16. \\nHorai Y et al. Pharmacodynamic effects and kinetic disposition of rabeprazole in relation to CYP2C19 genotypes. Aliment Pharmacol Ther 2001;15:793-803.\\n17. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol Ther 2000;14:1259-66.\\n18. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to CYP2C19 genotype. Dig Dis Sci 2008;53:933-\\n7.\\n19. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases. J Gastroenterol Hepatol 2006;21: 1381-\\n7.\\n20. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19 genotypic differences. Dig Dis Sci\\n2005;50:1625-31.\\n21. \\nYang JC et al. Pharmacokinetic- pharmacodynamic analysis of the role of CYP2C19 genotypes in short-term rabeprazole-based triple therapy against Helicobacter pylori.\\nBr J Clin Pharmacol 2009;67:503-10.\\n22. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n23. \\nKuwayama H et al. Rabeprazole-based eradication therapy for Helicobacter pylori: a large-scale study in Japan. Aliment Pharmacol Ther 2007;25:1105-13.\\n24. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori infection with lansoprazole- or\\nrabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n25. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton pump inhibitor (omeprazole or\\nrabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n26. \\nHokari K et al. Efficacy of triple therapy with rabeprazole for Helicobacter pylori infection and CYP2C19 genetic polymorphism. Aliment Pharmacol Ther 2001;15:1479-\\n84.\\n27. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n28. \\nSPC’s Pariet en Aciphex (VS).\\nDate 05-03-2018\\nCYP2C19 UM: rabeprazol\\n1858\\nNO action is required for this gene-drug interaction.\\nThere is currently insufficient information about this gene variation to recommend any action. Moreover, the fact that there are no differences in effectiveness between PM\\nand EM patients also makes differences in effectiveness between UM and EM patients less likely.\\nLiterature:\\n1. \\nNakamura K et al. Limited effect of rebamipide in addition to proton pump inhibitor (PPI) in the treatment of post-endoscopic submucosal dissection gastric ulcers: a\\nrandomized controlled trial comparing PPI plus rebamipide combination therapy with PPI monotherapy. Gut Liver 2016;10:917-924.\\n2. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with proton pump inhibitor-based triple therapy\\nregimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3. \\nKinoshita Y et al. Randomised clinical trial: a multicentre, double-blind, placebo- controlled study on the efficacy and safety of rabeprazole 5 mg or 10 mg once daily in\\npatients with non-erosive reflux disease. Aliment Pharmacol Ther 2011;33:213-24.\\n4. \\nLay CS et al. Correlation of CYP2C19 genetic polymorphisms with Helicobacter pylori eradication in patients with cirrhosis and peptic ulcer. J Chin Med Assoc\\n2010;73:188-93.\\n5. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19 genotypes. J Clin Gastroenterol 2009;43:920-5.\\n6. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor maintenance therapy. Hepatogastroenterology\\n2009;56:703-6.\\n7. \\nYamano HO et al. Plasma concentration of rabeprazole after 8-week administration in gastroesophageal reflux disease patients and intragastric pH elevation. J\\nGastroenterol Hepatol 2008;23:534-40.\\n8. \\nLee YC et al. Influence of cytochrome P450 2C19 genetic polymorphism and dosage of rabeprazole on accuracy of proton-pump inhibitor testing in Chinese patients with\\ngastroesophageal reflux disease. J Gastroenterol Hepatol 2007;22:1286-92.\\n9. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J Gastroenterol Hepatol 2007;22:815-20.\\n10. \\nAriizumi K et al. Therapeutic effects of 10 mg/day rabeprazole administration on reflux esophagitis was not influenced by the CYP2C19 polymorphism. J Gastroenterol\\nHepatol 2006;21:1428-34.\\n11. \\nHu YM et al. Pharmacodynamic and kinetic effect of rabeprazole on serum gastrin level in relation to CYP2C19 polymorphism in Chinese Hans. World J Gastroenterol\\n2006;12:4750-3.\\n12. \\nSugimoto M et al. Comparison of an increased dosage regimen of rabeprazole versus a concomitant dosage regimen of famotidine with rabeprazole for nocturnal gastric\\nacid inhibition in relation to cytochrome P450 2C19 genotypes. Clin Pharmacol Ther 2005;77:302-11.\\n13. \\nSugimoto M et al. Different dosage regimens of rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450 2C19 genotype status. Clin Pharmacol\\nTher 2004;76:290-301.\\n14. \\nShimatani T et al. Rabeprazole 10 mg twice daily is superior to 20 mg once daily for night-time gastric acid suppression. Aliment Pharmacol Ther 2004;19:113-22.\\n15. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH. Aliment Pharmacol Ther 2001;15:1929-37.\\n16. \\nHorai Y et al. Pharmacodynamic effects and kinetic disposition of rabeprazole in relation to CYP2C19 genotypes. Aliment Pharmacol Ther 2001;15:793-803.\\n17. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol Ther 2000;14:1259-66.\\n18. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to CYP2C19 genotype. Dig Dis Sci 2008;53:933-\\n7.\\n19. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases. J Gastroenterol Hepatol 2006;21: 1381-\\n7.\\n20. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19 genotypic differences. Dig Dis Sci\\n2005;50:1625-31.\\n21. \\nYang JC et al. Pharmacokinetic- pharmacodynamic analysis of the role of CYP2C19 genotypes in short-term rabeprazole-based triple therapy against Helicobacter pylori.\\nBr J Clin Pharmacol 2009;67:503-10.\\n22. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for Helicobacter pylori eradication: a meta-analysis.\\nHelicobacter 2008;13:532-41.\\n23. \\nKuwayama H et al. Rabeprazole-based eradication therapy for Helicobacter pylori: a large-scale study in Japan. Aliment Pharmacol Ther 2007;25:1105-13.\\n24. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori infection with lansoprazole- or\\nrabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n25. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton pump inhibitor (omeprazole or\\nrabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 81}),\n",
       " Document(page_content='26. \\nHokari K et al. Efficacy of triple therapy with rabeprazole for Helicobacter pylori infection and CYP2C19 genetic polymorphism. Aliment Pharmacol Ther 2001;15:1479-\\n84.\\n27. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n28. \\nSPC’s Pariet en Aciphex (VS).\\nDate 05-03-2018\\nHLA-B44 negatief: ribavirine\\n2343\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nVidal-Castiñeira JR et al. Diversity of killer cell immunoglobulin-like receptor (KIR) genotypes and KIR2DL2/3 variants in HCV treatment outcome. PLoS One\\n2014;9:e99426.\\n2. \\nVidal-Castiñeira JR et al. A predictive model of treatment outcome in patients with chronic HCV infection using IL28B and PD-1 genotyping. J Hepatol 2012;56:1230-8.\\n3. \\nRomero-Gómez M et al. HLA class I B44 is associated with sustained response to interferon + ribavirin therapy in patients with chronic hepatitis C. Am J Gastroenterol\\n2003;98:1621-6.\\nDate 13-05-2019\\nCYP2D6 IM: risperidon\\n1536\\nNO action is needed for this gene-drug interaction.\\nThere is little evidence to support an increase in side effects caused by the gene variation. The gene variation may lead to a decrease in the required maintenance dose.\\nHowever, as the effect on the dose is smaller than that of the normal biological variation, action is not useful.\\nLiterature:\\n1. \\nGanoci L et al. ABCB1, ABCG2 and CYP2D6 polymorphism effects on disposition and response to long-acting risperidone. Prog Neuropsychopharmacol Biol Psychiatry\\n2021;104:110042.\\n2. \\nCui Y et al. CYP2D6 genotype-based dose recommendations for risperidone in Asian people. Front Pharmacol 2020;11:936.\\n3. \\nJukic MM et al. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 2019;6:418-26.\\n4. \\nOshikoya KA et al. CYP2D6 genotype and adverse events to risperidone in children and adolescents. Pediatr Res 2019;85:602-6.\\n5. \\nSchoretsanitis G et al. Prolactin levels: sex differences in the effects of risperidone, 9-hydroxyrisperidone levels, CYP2D6 and ABCB1 variants. Pharmacogenomics\\n2018;19:815-823.\\n6. \\nKaur G et al. Identification of genetic correlates of response to risperidone: findings of a multicentric schizophrenia study from India. Asian J Psychiatr 2017;29:174-82.\\n7. \\nIvaturi V et al. Exposure-response analysis after subcutaneous administration of RBP-7000, a once-a-month long-acting Atrigel formulation of risperidone. Br J Clin\\nPharmacol 2017;83:1476-98.\\n8. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and chlorpromazine in the Chinese Han population.\\nPharmacogenomics J 2016;16:357-65.\\n9. \\nSukasem C et al. Impact of pharmacogenetic markers of CYP2D6 and DRD2 on prolactin response in risperidone-treated Thai children and adolescents with autism\\nspectrum disorders. J Clin Psychopharmacol 2016;36:141-6.\\n10. \\nDos Santos-Júnior A et al. Pharmacogenetics of risperidone and cardiovascular risk in children and adolescents. Int J Endocrinol 2016;2016:5872423.\\n11. \\ndos Santos Júnior A et al. Hyperprolactinemia in children and adolescents with use of risperidone: clinical and molecular genetics aspects. J Child Adolesc\\nPsychopharmacol 2015;25:738-48.\\n12. \\nVandenberghe F et al. Genetics-based population pharmacokinetics and pharmacodynamics of risperidone in a psychiatric cohort. Clin Pharmacokinet 2015;54:1259-72.\\n13. \\nvan der Weide K et al. The influence of the CYP3A4 *22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\n14. \\nSuzuki Y et al. Effect of risperidone metabolism and P-glycoprotein gene polymorphism on QT interval in patients with schizophrenia. Pharmacogenomics J 2014;14:452-\\n6.\\n15. \\nAlmoguera B et al. CYP2D6 poor metabolizer status might be associated with better response to risperidone treatment. Pharmacogenet Genomics 2013 ;23:627-30.\\n16. \\nAlmoguera B et al. Association of common genetic variants with risperidone adverse events in a Spanish schizophrenic population. Pharmacogenomics J 2013; 13:197-\\n204. PubMed PMID: 22212732.\\n17. \\nMas S et al. Intuitive pharmacogenetics: spontaneous risperidone dosage is related to CYP2D6, CYP3A5 and ABCB1 genotypes. Pharmacogenomics J 2012; 12:255-9.\\n18. \\nJovanović N et al. The role of CYP2D6 and ABCB1 pharmacogenetics in drug-naïve patients with first-episode schizophrenia treated with risperidone. Eur J Clin\\nPharmacol 2010; 66:1109-17.\\n19. \\nNovalbos J et al. Effects of CYP2D6 genotype on the pharmacokinetics, pharmacodynamics, and safety of risperidone in healthy volunteers. J Clin Psychopharmacol\\n2010;30:504-11.\\n20. \\nWang L et al. Serum prolactin levels, plasma risperidone levels, polymorphism of cytochrome P450 2D6 and clinical response in patients with schizophrenia. J\\nPsychopharmacol 2007;21:837-42.\\n21. \\nde Leon J et al. A study of genetic (CYP2D6 and ABCB1) and environmental (drug inhibitors and inducers) variables that may influence plasma risperidone levels.\\nPharmacopsychiatry 2007;40:93-102.\\n22. \\nCho HY et al. Pharmacokinetics and bioequivalence evaluation of risperidone in healthy male subjects with different CYP2D6 genotypes. Arch Pharm Res 2006;29:525-33.\\n23. \\nLane HY et al. Risperidone-related weight gain: genetic and nongenetic predictors. J Clin Psychopharmacol 2006;26:128-34.\\n24. \\nRiedel M et al. Risperidone plasma levels, clinical response and side-effects. Eur Arch Psychiatry Clin Neurosci 2005;255:261-8.\\n25. \\nKato D et al. Delirium resolving upon switching from risperidone to quetiapine: implication of CYP2D6 genotype. Psychosomatics 2005;46:374-5.\\n26. \\nKakihara S et al. Prediction of response to risperidone treatment with respect to plasma concencentrations of risperidone, catecholamine metabolites, and polymorphism\\nof cytochrome P450 2D6. Int Clin Psychopharmacol 2005;20:71-8.\\n27. \\nde Leon J et al. The CYP2D6 poor metabolizer phenotype may be associated with risperidone adverse drug reactions and discontinuation. J Clin Psychiatry 2005;66:15-27.\\n28. \\nLlerena A et al. QTc interval, CYP2D6 and CYP2C9 genotypes and risperidone plasma concentrations. J Psychopharmacol 2004;18:189-93.\\n29. \\nYasui-Furukori N et al. Effects of various factors on steady-state plasma concentrations of risperidone and 9-hydroxyrisperidone: lack of impact of MDR-1 genotypes. Br J\\nClin Pharmacol 2004;57:569-75.\\n30. \\nRoh HK et al. Risperidone metabolism in relation to CYP2D6*10 allele in Korean schizophrenic patients. Eur J Clin Pharmacol 2001;57:671-5.\\n31. \\nScordo MG et al. Cytochrome P450 2D6 genotype and steady state plasma levels of risperidone and 9-hydroxyrisperidone. Psychopharmacology 1999;147:300-5.\\n32. \\nBork JA et al. A pilot study on risperidone metabolism: the role of cytochromes P450 2D6 and 3A. J Clin Psychiatry 1999;60:469-76.\\nDate 13-09-2021\\nCYP2D6 PM: risperidon\\n1537\\nThe percentage of patients with therapy failure increased from 16% to 26%. The gene variation increases the plasma concentration of risperidone plus the active metabolite\\nand increases the proportion of risperidone in this ratio, which is more effective at crossing the blood-brain barrier.\\nuse 67% of the standard dose\\nif problematic side effects originating in the central nervous system occur despite this reduced dose, then reduce the dose further to 50% of the standard dose\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 82}),\n",
       " Document(page_content='1. \\nGanoci L et al. ABCB1, ABCG2 and CYP2D6 polymorphism effects on disposition and response to long-acting risperidone. Prog Neuropsychopharmacol Biol Psychiatry\\n2021;104:110042.\\n2. \\nJukic MM et al. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 2019;6:418-26.\\n3. \\nOshikoya KA et al. CYP2D6 genotype and adverse events to risperidone in children and adolescents. Pediatr Res 2019;85:602-6.\\n4. \\nSchoretsanitis G et al. Prolactin levels: sex differences in the effects of risperidone, 9-hydroxyrisperidone levels, CYP2D6 and ABCB1 variants. Pharmacogenomics\\n2018;19:815-823.\\n5. \\nKaur G et al. Identification of genetic correlates of response to risperidone: findings of a multicentric schizophrenia study from India. Asian J Psychiatr 2017;29:174-82.\\n6. \\nIvaturi V et al. Exposure-response analysis after subcutaneous administration of RBP-7000, a once-a-month long-acting Atrigel formulation of risperidone. Br J Clin\\nPharmacol 2017;83:1476-98.\\n7. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and chlorpromazine in the Chinese Han population.\\nPharmacogenomics J 2016;16:357-65.\\n8. \\nDos Santos-Júnior A et al. Pharmacogenetics of risperidone and cardiovascular risk in children and adolescents. Int J Endocrinol 2016;2016:5872423.\\n9. \\ndos Santos Júnior A et al. Hyperprolactinemia in children and adolescents with use of risperidone: clinical and molecular genetics aspects. J Child Adolesc\\nPsychopharmacol 2015;25:738-48.\\n10. \\nVandenberghe F et al. Genetics-based population pharmacokinetics and pharmacodynamics of risperidone in a psychiatric cohort. Clin Pharmacokinet 2015;54:1259-72.\\n11. \\nvan der Weide K et al. The influence of the CYP3A4 *22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\n12. \\nSuzuki Y et al. Effect of risperidone metabolism and P-glycoprotein gene polymorphism on QT interval in patients with schizophrenia. Pharmacogenomics J 2014;14:452-\\n6.\\n13. \\nGassó P et al. Effect of CYP2D6 on risperidone pharmacokinetics and extrapyramidal symptoms in healthy volunteers: results from a pharmacogenetic clinical trial.\\nPharmacogenomics 2014;15:17-28.\\n14. \\nAlmoguera B et al. CYP2D6 poor metabolizer status might be associated with better response to risperidone treatment. Pharmacogenet Genomics 2013 ;23:627-30.\\n15. \\nAlmoguera B et al. Association of common genetic variants with risperidone adverse events in a Spanish schizophrenic population. Pharmacogenomics J 2013; 13:197-\\n204. PubMed PMID: 22212732.\\n16. \\nMas S et al. Intuitive pharmacogenetics: spontaneous risperidone dosage is related to CYP2D6, CYP3A5 and ABCB1 genotypes. Pharmacogenomics J 2012; 12:255-9.\\n17. \\nNovalbos J et al. Effects of CYP2D6 genotype on the pharmacokinetics, pharmacodynamics, and safety of risperidone in healthy volunteers. J Clin Psychopharmacol\\n2010;30:504-11.\\n18. \\nJovanović N et al. The role of CYP2D6 and ABCB1 pharmacogenetics in drug-naïve patients with first-episode schizophrenia treated with risperidone. Eur J Clin\\nPharmacol 2010; 66:1109-17.\\n19. \\nde Leon J et al. A study of genetic (CYP2D6 and ABCB1) and environmental (drug inhibitors and inducers) variables that may influence plasma risperidone levels.\\nPharmacopsychiatry 2007;40:93-102.\\n20. \\nde Leon J et al. Polymorphic variations in GSTM1, GSTT1, PgP, CYP2D6, CYP3A5, and dopamine D2 and D3 receptors and their association with tardive dyskinesia in\\nsevere mental illness. J Clin Psychopharmacol 2005;25:448-56.\\n21. \\nde Leon J et al. The CYP2D6 poor metabolizer phenotype may be associated with risperidone adverse drug reactions and discontinuation. J Clin Psychiatry 2005;66:15-27.\\n22. \\nLlerena A et al. QTc interval, CYP2D6 and CYP2C9 genotypes and risperidone plasma concentrations. J Psychopharmacol 2004;18:189-93.\\n23. \\nKohnke MD et al. Cytochrome P450 2D6 deficiency and its clinical relevance in a patient treated with risperidone. Pharmacopsychiatry 2002;35(3):116-8.\\n24. \\nScordo MG et al. Cytochrome P450 2D6 genotype and steady state plasma levels of risperidone and 9-hydroxyrisperidone. Psychopharmacology 1999;147:300-5.\\n25. \\nBork JA et al. A pilot study on risperidone metabolism: the role of cytochromes P450 2D6 and 3A. J Clin Psychiatry 1999;60:469-76.\\n26. \\nSmPC’s Risperdal (NL en VS) en Risperdal Consta (NL).\\nDate 13-09-2021\\nCYP2D6 UM: risperidon\\n1535\\nThe percentage of patients with therapy failure increases from 16% to 37%. The gene variation leads to a high ratio of the active metabolite (9-hydroxyrisperidone\\n(paliperidone)) compared to risperidone, which crosses the blood-brain barrier more effectively.\\nchoose an alternative or titrate the dose according to the maximum dose for the active metabolite (paliperidone) (oral 12 mg/day for adults and children from 15 years of\\nage weighing at least 51 kg and 6 mg/day for children from 15 years of age weighing less than 51 kg; intramuscular 75 mg per 2 weeks)\\nLiterature:\\n1. \\nJukic MM et al. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 2019;6:418-26.\\n2. \\nOshikoya KA et al. CYP2D6 genotype and adverse events to risperidone in children and adolescents. Pediatr Res 2019;85:602-6.\\n3. \\nSchoretsanitis G et al. Prolactin levels: sex differences in the effects of risperidone, 9-hydroxyrisperidone levels, CYP2D6 and ABCB1 variants. Pharmacogenomics\\n2018;19:815-823.\\n4. \\nvan der Weide K et al. The influence of the CYP3A4 *22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and\\nrisperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\n5. \\nGassó P et al. Effect of CYP2D6 on risperidone pharmacokinetics and extrapyramidal symptoms in healthy volunteers: results from a pharmacogenetic clinical trial.\\nPharmacogenomics 2014;15:17-28.\\n6. \\nAlmoguera B et al. CYP2D6 poor metabolizer status might be associated with better response to risperidone treatment. Pharmacogenet Genomics 2013 ;23:627-30.\\n7. \\nAlmoguera B et al. Association of common genetic variants with risperidone adverse events in a Spanish schizophrenic population. Pharmacogenomics J 2013; 13:197-\\n204. PubMed PMID: 22212732.\\n8. \\nMas S et al. Intuitive pharmacogenetics: spontaneous risperidone dosage is related to CYP2D6, CYP3A5 and ABCB1 genotypes. Pharmacogenomics J 2012; 12:255-9.\\n9. \\nNovalbos J et al. Effects of CYP2D6 genotype on the pharmacokinetics, pharmacodynamics, and safety of risperidone in healthy volunteers. J Clin Psychopharmacol\\n2010;30:504-11.\\n10. \\nde Leon J et al. A study of genetic (CYP2D6 and ABCB1) and environmental (drug inhibitors and inducers) variables that may influence plasma risperidone levels.\\nPharmacopsychiatry 2007;40:93-102.\\n11. \\nde Leon J et al. The CYP2D6 poor metabolizer phenotype may be associated with risperidone adverse drug reactions and discontinuation. J Clin Psychiatry 2005;66:15-27.\\n12. \\nLlerena A et al. QTc interval, CYP2D6 and CYP2C9 genotypes and risperidone plasma concentrations. J Psychopharmacol 2004;18:189-93.\\n13. \\nGuzey C et al. Risperidone metabolism and the impact of being a cytochrome P450 2D6 ultrarapid metabolizer. J Clin Psychiatry 2000;61:600-1.\\n14. \\nScordo MG et al. Cytochrome P450 2D6 genotype and steady state plasma levels of risperidone and 9-hydroxyrisperidone. Psychopharmacology 1999;147:300-5.\\nDate 13-09-2021\\nCYP2C19 IM: sertraline\\n2008\\nNO action is needed for this gene-drug interaction.\\nThe gene variation has a minor effect on the sertraline plasma concentration. No effect on side effects was found.\\nLiterature:\\n1. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy volunteers. Basic Clin Pharmacol Toxicol\\n2018;122:501-511.\\n2. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome. Brain Dev 2017;39:483-492.\\n3. \\nYuce-Artun N et al. Influence of CYP2B6 and CYP2C19 polymorphisms on sertraline metabolism in major depression patients. Int J Clin Pharm 2016;38:388-94.\\n4. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n5. \\nRudberg I et al. Serum concentrations of sertraline and N-desmethyl sertraline in relation to CYP2C19 genotype in psychiatric patients. Eur J Clin Pharmacol\\n2008;64:1181-8.\\n6. \\nWang JH et al. Pharmacokinetics of sertraline in relation to genetic polymorphism of CYP2C19. Clin Pharmacol Ther 2001;70:42-7.\\nDate 14-05-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 83}),\n",
       " Document(page_content='CYP2C19 PM: sertraline\\n2009\\nThe risk of side effects is increased. The gene variation leads to increased plasma concentrations of sertraline\\nDo not give doses exceeding 75 mg/day\\nGuide the dose by response and side effects and/or sertraline plasma concentration.\\nLiterature:\\n1. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome. Brain Dev 2017;39:483-492.\\n2. \\nYuce-Artun N et al. Influence of CYP2B6 and CYP2C19 polymorphisms on sertraline metabolism in major depression patients. Int J Clin Pharm 2016;38:388-94.\\n3. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n4. \\nRudberg I et al. Serum concentrations of sertraline and N-desmethyl sertraline in relation to CYP2C19 genotype in psychiatric patients. Eur J Clin Pharmacol\\n2008;64:1181-8.\\n5. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and clinical effects of antidepressants in a\\nnaturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n6. \\nWang JH et al. Pharmacokinetics of sertraline in relation to genetic polymorphism of CYP2C19. Clin Pharmacol Ther 2001;70:42-7.\\n7. \\nSPC Zoloft.\\nDate 14-05-2018\\nCYP2C19 UM: sertraline\\n2010\\nNO action is needed for this gene-drug interaction.\\nThe gene variation has a negligible effect on the plasma concentration of sertraline. Moreover, no significant effect on response and side effects has been found.\\nLiterature:\\n1. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome. Brain Dev 2017;39:483-492.\\n2. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n3. \\nRudberg I et al. Serum concentrations of sertraline and N-desmethyl sertraline in relation to CYP2C19 genotype in psychiatric patients. Eur J Clin Pharmacol\\n2008;64:1181-8.\\nDate 14-05-2018\\nCYP2D6 IM: sertraline\\n3513\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy volunteers. Basic Clin Pharmacol Toxicol\\n2018;122:501-511.\\n2. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome. Brain Dev 2017;39:483-492.\\n3. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\nDate 14-05-2018\\nCYP2D6 PM: sertraline\\n3512\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy volunteers. Basic Clin Pharmacol Toxicol\\n2018;122:501-511.\\n2. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome. Brain Dev 2017;39:483-492.\\n3. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n4. \\nHamelin BA et al. The disposition of fluoxetine but not sertraline is altered in poor metabolizers of debrisoquin. Clin Pharmacol Ther 1996;60:512-21.\\nDate 14-05-2018\\nCYP2D6 UM: sertraline\\n3514\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy volunteers. Basic Clin Pharmacol Toxicol\\n2018;122:501-511.\\n2. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\nDate 14-05-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 84}),\n",
       " Document(page_content='SLCO1B1 521CC: simvastatine\\n4056\\nWhen using simvastatin 80 mg/day, the risk of myopathy is increased 30-fold to 18% and the risk of severe myopathy is increased 48-fold to 12%. When using 40 mg/day, this\\nrisk is increased 7-fold to 1% and 11-fold to 0.68% respectively. The gene variation leads to reduced simvastatin transport to the liver, which increases the simvastatin plasma\\nconcentration and therefore the risk of side effects.\\n1. \\nChoose an alternative\\nConsider any additional risk factors for statin-induced myopathy.\\nAtorvastatin is affected less severely by the SLCO1B1 gene variation, but is also affected by CYP3A4 inhibitors such as amiodarone, verapamil and diltiazem. Use of\\natorvastatin is not recommended for patients with additional risk factors for statin-induced myopathy.\\nRosuvastatin and pravastatin are influenced to a lesser extent by the SLCO1B1 gene variation. They are also not influenced by CYP3A4 inhibitors such as amiodarone,\\nverapamil and diltiazem.\\nFluvastatin is not significantly influenced by the SLCO1B1 gene variation or CYP3A4 inhibitors.\\nLiterature:\\n1. \\nXiang Q et al. Association between SLCO1B1 T521C polymorphism and risk of statin-induced myopathy: a meta-analysis. Pharmacogenomics J 2018;18:721-9.\\n2. \\nWu X et al. Associations of the SLCO1B1 polymorphisms with hepatic function, baseline lipid levels, and lipid-lowering response to simvastatin in patients with\\nhyperlipidemia. Clin Appl Thromb Hemost 2018;24:240S-247S.\\n3. \\nKitzmiller JP et al. Candidate-gene study of functional polymorphisms in SLCO1B1 and CYP3A4/5 and the cholesterol-lowering response to simvastatin. Clin Transl Sci\\n2017;10:172-7.\\n4. \\nJiang J et al. Association between SLCO1B1 -521T>C and -388A>G polymorphisms and risk of statin-induced adverse drug reactions: a meta-analysis. Springerplus\\n2016;5:1368.\\n5. \\nHou Q et al. Association between SLCO1B1 gene T521C polymorphism and statin-related myopathy risk: a meta-analysis of case-control studies. Medicine (Baltimore)\\n2015;94:e1268.\\n6. \\nLuzum JA et al. Individual and combined associations of genetic variants in CYP3A4, CYP3A5, and SLCO1B1 with simvastatin and simvastatin acid plasma concentrations.\\nJ Cardiovasc Pharmacol 2015;66:80-5.\\n7. \\nDou Y et al. Meta-analysis of the SLCO1B1 c.521T>C variant reveals slight influence on the lipid-lowering efficacy of statins. Ann Lab Med 2015;35:329-35.\\n8. \\nde Keyser CE et al. The SLCO1B1 c.521T>C polymorphism is associated with dose decrease or switching during statin therapy in the Rotterdam Study. Pharmacogenet\\nGenomics 2014;24:43-51.\\n9. \\nCarr DF et al. SLCO1B1 genetic variant associated with statin-induced myopathy: a proof-of-concept study using the Clinical Practice Research Datalink. Clin Pharmacol\\nTher 2013;94:695-701.\\n10. \\nHopewell JC et al. Impact of common genetic variation on response to simvastatin therapy among 18 705 participants in the Heart Protection Study. Eur Heart J\\n2013;34:982-92.\\n11. \\nHu M et al. Intronic variants in SLCO1B1 related to statin-induced myopathy are associated with the low-density lipoprotein cholesterol response to statins in Chinese\\npatients with hyperlipidaemia. Pharmacogenet Genomics 2012;22:803-6.\\n12. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and atorvastatin. Pharmacogenomics J 2012;12:233-\\n7.\\n13. \\nSortica VA et al. SLCO1B1 gene variability influences lipid-lowering efficacy on simvastatin therapy in Southern Brazilians. Clin Chem Lab Med 2012;50:441-8.\\n14. \\nBailey KM et al. Hepatic metabolism and transporter gene variants enhance response to rosuvastatin in patients with acute myocardial infarction: the GEOSTAT-1 Study.\\nCirc Cardiovasc Genet 2010;3:276-85.\\n15. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n16. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with increased cholesterol synthesis rate.\\nPharmacogenet Genomics 2008;18:921-6.\\n17. \\nSEARCH Collaborative Group. SLCO1B1 variants and statin-induced myopathy -- a genomewide study. N Engl J Med 2008;359:789-99.\\n18. \\nPasanen MK et al. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genomics 2006;16:873-9.\\n19. \\nRamsey LB et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin-induced myopathy: 2014 update. Clin Pharmacol Ther\\n2014;96:423-8.\\n20. \\nSmPC Zocor.\\nDate 18-05-2020\\nSLCO1B1 521TC: simvastatine\\n4055\\nWhen using simvastatin 80 mg/day, the risk of myopathy is increased 5-fold to 3% for moderately severe to severe myopathy and 1.3% for severe myopathy. When using 40\\nmg/day, this risk is increased 2.6-fold to 0.39% and 0.17% respectively. The gene variation may lead to reduced simvastatin transport to the liver, which may increase\\nsimvastatin plasma concentrations and therefore the risk of side effects.\\n1. \\nChoose an alternative\\nConsider any additional risk factors for statin-induced myopathy.\\nAtorvastatin is affected less severely by the SLCO1B1 gene variation, but is also affected by CYP3A4 inhibitors such as amiodarone, verapamil and diltiazem. Use of\\natorvastatin is not recommended for patients with additional risk factors for statin-induced myopathy.\\nRosuvastatin and pravastatin are influenced to a lesser extent by the SLCO1B1 gene variation. They are also not influenced by CYP3A4 inhibitors such as amiodarone,\\nverapamil and diltiazem.\\nFluvastatin is not significantly influenced by the SLCO1B1 gene variation or CYP3A4 inhibitors.\\n2. \\nIf an alternative is not an option:\\n1. \\nAvoid simvastatin doses exceeding 40 mg/day\\n2. \\nAdvise the patient to contact their doctor in the event of muscle symptoms.\\nLiterature:\\n1. \\nXiang Q et al. Association between SLCO1B1 T521C polymorphism and risk of statin-induced myopathy: a meta-analysis. Pharmacogenomics J 2018;18:721-9.\\n2. \\nWu X et al. Associations of the SLCO1B1 polymorphisms with hepatic function, baseline lipid levels, and lipid-lowering response to simvastatin in patients with\\nhyperlipidemia. Clin Appl Thromb Hemost 2018;24:240S-247S.\\n3. \\nKitzmiller JP et al. Candidate-gene study of functional polymorphisms in SLCO1B1 and CYP3A4/5 and the cholesterol-lowering response to simvastatin. Clin Transl Sci\\n2017;10:172-7.\\n4. \\nJiang J et al. Association between SLCO1B1 -521T>C and -388A>G polymorphisms and risk of statin-induced adverse drug reactions: a meta-analysis. Springerplus\\n2016;5:1368.\\n5. \\nHou Q et al. Association between SLCO1B1 gene T521C polymorphism and statin-related myopathy risk: a meta-analysis of case-control studies. Medicine (Baltimore)\\n2015;94:e1268.\\n6. \\nLuzum JA et al. Individual and combined associations of genetic variants in CYP3A4, CYP3A5, and SLCO1B1 with simvastatin and simvastatin acid plasma concentrations.\\nJ Cardiovasc Pharmacol 2015;66:80-5.\\n7. \\nDou Y et al. Meta-analysis of the SLCO1B1 c.521T>C variant reveals slight influence on the lipid-lowering efficacy of statins. Ann Lab Med 2015;35:329-35.\\n8. \\nde Keyser CE et al. The SLCO1B1 c.521T>C polymorphism is associated with dose decrease or switching during statin therapy in the Rotterdam Study. Pharmacogenet\\nGenomics 2014;24:43-51.\\n9. \\nCarr DF et al. SLCO1B1 genetic variant associated with statin-induced myopathy: a proof-of-concept study using the Clinical Practice Research Datalink. Clin Pharmacol\\nTher 2013;94:695-701.\\n10. \\nHopewell JC et al. Impact of common genetic variation on response to simvastatin therapy among 18 705 participants in the Heart Protection Study. Eur Heart J\\n2013;34:982-92.\\n11. \\nHu M et al. Intronic variants in SLCO1B1 related to statin-induced myopathy are associated with the low-density lipoprotein cholesterol response to statins in Chinese\\npatients with hyperlipidaemia. Pharmacogenet Genomics 2012;22:803-6.\\n12. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and atorvastatin. Pharmacogenomics J 2012;12:233-\\n7.\\n13. \\nSortica VA et al. SLCO1B1 gene variability influences lipid-lowering efficacy on simvastatin therapy in Southern Brazilians. Clin Chem Lab Med 2012;50:441-8.\\n14. \\nBailey KM et al. Hepatic metabolism and transporter gene variants enhance response to rosuvastatin in patients with acute myocardial infarction: the GEOSTAT-1 Study.\\nCirc Cardiovasc Genet 2010;3:276-85.\\n15. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n16. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with increased cholesterol synthesis rate.\\nPharmacogenet Genomics 2008;18:921-6.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 85}),\n",
       " Document(page_content='17. \\nSEARCH Collaborative Group. SLCO1B1 variants and statin-induced myopathy -- a genomewide study. N Engl J Med 2008;359:789-99.\\n18. \\nPasanen MK et al. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genomics 2006;16:873-9.\\n19. \\nRamsey LB et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin-induced myopathy: 2014 update. Clin Pharmacol Ther\\n2014;96:423-8.\\n20. \\nSmPC Zocor.\\nDate 18-05-2020\\nCYP2C9 IM ANDERS: siponimod\\n7165\\nTheoretically, the risk of adverse effects in increased, as the genetic variation results in higher plasma concentrations of siponimod.\\n- use 50% of the normal maintenance dose    - reconsider the choice and the potential benefit of siponimod if the patient is also using a moderate CYP3A4 inducer, such as modafinil    \\nFor the comparable genetic variation *1/*3, the moderate CYP3A4 inducer results in a reduction in the exposure of siponimod by 49%, according to a pharmacokinetic model.\\nLiterature:\\n1. \\nGardin A et al. Siponimod pharmacokinetics, safety, and tolerability in combination with the potent CYP3A4 inhibitor itraconazole in healthy subjects with different\\nCYP2C9 genotypes. Eur J Clin Pharmacol 2019;75:1565-74.\\n2. \\nSmPC Mayzent (NL en VS).\\nDate 12-03-2020\\nCYP2C9 PM ANDERS: siponimod\\n7166\\nSiponimod is contraindicated in patients with the comparable genetic variation *3/*3. Theoretically, the risk of adverse effects is greatly increased, as the genetic variation\\nresults in much higher plasma concentrations of siponimod.\\n- avoid siponimod\\nLiterature:\\n1. \\nGardin A et al. Effect of fluconazole coadministration and CYP2C9 genetic polymorphism on siponimod pharmacokinetics in healthy subjects. Clin Pharmacokinet\\n2019;58:349-61.\\n2. \\nSmPC Mayzent (NL en VS).\\nDate 12-03-2020\\nCYP2C9*1/*2: siponimod\\n7160\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can slightly increase the exposure to siponimod. However, the effect is too small to expect any impact on efficacy or adverse effects.\\nLiterature:\\n1. \\nGardin A et al. Siponimod pharmacokinetics, safety, and tolerability in combination with the potent CYP3A4 inhibitor itraconazole in healthy subjects with different\\nCYP2C9 genotypes. Eur J Clin Pharmacol 2019;75:1565-74.\\n2. \\nSmPC Mayzent (NL en VS).\\nDate 12-03-2020\\nCYP2C9*1/*3: siponimod\\n7161\\nTheoretically, the risk of adverse effects in increased, as the genetic variation results in higher plasma concentrations of siponimod.\\n- use 50% of the normal maintenance dose    - reconsider the choice and the potential benefit of siponimod if the patient is also using a moderate CYP3A4 inducer, such as modafinil    \\nFor this genetic variation, a moderate CYP3A4 inducer results in a reduction in the exposure of siponimod by 49%, according to a pharmacokinetic model.\\nLiterature:\\n1. \\nGardin A et al. Siponimod pharmacokinetics, safety, and tolerability in combination with the potent CYP3A4 inhibitor itraconazole in healthy subjects with different\\nCYP2C9 genotypes. Eur J Clin Pharmacol 2019;75:1565-74.\\n2. \\nSmPC Mayzent (NL en VS).\\nDate 12-03-2020\\nCYP2C9*2/*2: siponimod\\n7162\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can slightly increase the exposure to siponimod. However, the effect is too small to expect any impact on efficacy or adverse effects.\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 86}),\n",
       " Document(page_content='1. \\nSmPC Mayzent (NL en VS).\\nDate 12-03-2020\\nCYP2C9*2/*3: siponimod\\n7163\\nTheoretically, the risk of adverse effects in increased, as the genetic variation results in higher plasma concentrations of siponimod.\\n- use 50% of the normal maintenance dose    - reconsider the choice and the potential benefit of siponimod if the patient is also using a moderate CYP3A4 inducer, such as modafinil    \\nFor this genetic variation, a moderate CYP3A4 inducer results in a reduction in the exposure of siponimod by 49%, according to a pharmacokinetic model.\\nLiterature:\\n1. \\nGardin A et al. Effect of fluconazole coadministration and CYP2C9 genetic polymorphism on siponimod pharmacokinetics in healthy subjects. Clin Pharmacokinet\\n2019;58:349-61.\\n2. \\nSmPC Mayzent (NL en VS).\\nDate 12-03-2020\\nCYP2C9*3/*3: siponimod\\n7164\\nSiponimod is contraindicated in patients with this genetic variation. Theoretically, the risk of adverse effects is greatly increased, as the genetic variation results in much\\nhigher plasma concentrations of siponimod.\\n- avoid siponimod\\nLiterature:\\n1. \\nGardin A et al. Effect of fluconazole coadministration and CYP2C9 genetic polymorphism on siponimod pharmacokinetics in healthy subjects. Clin Pharmacokinet\\n2019;58:349-61.\\n2. \\nSmPC Mayzent (NL en VS).\\nDate 12-03-2020\\nCYP2D6 IM: sotalol\\n2540\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2D6 PM: sotalol\\n2539\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP2D6 UM: sotalol\\n2541\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 12-09-2022\\nCYP3A5 heterozygote expresser: tacrolimus\\n2358\\nAn increase of the initial dose can result in an increased chance of reaching a tacrolimus concentration within the target range before the start of therapeutic drug monitoring.\\nHowever, there is no direct evidence that this results in improved clinical results. The genetic variation results in increased conversion of tacrolimus to inactive metabolites\\nand therefore in a higher required dose.\\n- Indications OTHER than liver transplantation:        - Use 1.5 times the initial dose that would yield the desired result in non-expressers    \\nAdjustment of the dose should then be based on therapeutic drug monitoring.\\nFor example: A Dutch study found a median trough concentration for tacrolimus of 14.7 ng/mL after 3 days at an initial dose of 0.15 mg/kg twice daily for 29 kidney transplant\\npatients who were heterozygous expressers. Their target value was 10 - 15 ng/mL.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 87}),\n",
       " Document(page_content='- LIVER transplantation:    \\nIn addition to the patient’s genotype, the metabolism of tacrolimus is also determined by the genotype of the transplanted liver.\\n- LIVER is also of the genotype HETEROZYGOUS EXPRESSER:\\n- Use 1.5 times the normal initial dose\\nAdjustment of the dose should then be based on therapeutic drug monitoring.\\n- LIVER has a DIFFERENT genotype:\\nThere is insufficient evidence in the literature to support a dose recommendation.\\nLiterature:\\n1. \\nLargeau B et al. Comparison of tacrolimus starting doses based on CYP3A5 phenotype or genotype in kidney transplant recipients. Prog Transplant\\n2019:1526924819873905 [Epub ahead of print].\\n2. \\nFernando ME et al. Influence of CYP3A5 and ABCB1 polymorphism on tacrolimus drug dosing in South Indian renal allograft recipients. Indian J Nephrol 2019;29:261-6.\\n3. \\nSeibert SR et al. Tacrolimus trough and dose intra-patient variability and CYP3A5 genotype: effects on acute rejection and graft failure in European American and African\\nAmerican kidney transplant recipients. Clin Transplant 2018;32: e13424.\\n4. \\nLiu F et al. Long-term influence of CYP3A5, CYP3A4, ABCB1, and NR1I2 polymorphisms on tacrolimus concentration in Chinese renal transplant recipients. Genet Test\\nMol Biomarkers 2017;21:663-73.\\n5. \\nPallet N et al. Long-term clinical impact of adaptation of initial tacrolimus dosing to CYP3A5 genotype. Am J Transplant 2016;16:2670-5.\\n6. \\nShuker N et al. A randomized controlled trial comparing the efficacy of Cyp3a5 genotype-based with body-weight-based tacrolimus dosing after living donor kidney\\ntransplantation. Am J Transplant 2016;16:2085-96.\\n7. \\nDe Meyer M et al. Pharmacogenetic-based strategy using de novo tacrolimus once daily after kidney transplantation: prospective pilot study. Pharmacogenomics\\n2016;17:1019-27.\\n8. \\nYaowakulpatana K et al. Impact of CYP3A5 polymorphism on trough concentrations and outcomes of tacrolimus minimization during the early period after kidney\\ntransplantation. Eur J Clin Pharmacol 2016;72:277-83.\\n9. \\nPulk RA et al. Multigene predictors of tacrolimus exposure in kidney transplant recipients. Pharmacogenomics 2015;16:841-54.\\n10. \\nWang L et al. Benefits of minimizing immunosuppressive dosage according to cytochrome P450 3A5 genotype in liver transplant patients: findings from a single-center\\nstudy. Genet Mol Res 2015;14:3191-9.\\n11. \\nRojas L et al. Effect of CYP3A5*3 on kidney transplant recipients treated with tacrolimus: a systematic review and meta-analysis of observational studies.\\nPharmacogenomics J 2015;15:38-48.\\n12. \\nBuendia JA et al. Effects of combinational CYP3A5 6986A>G polymorphism in graft liver and native intestine on the pharmacokinetics of tacrolimus in liver transplant\\npatients: a meta-analysis. Ther Drug Monit 2014;36:442-7.\\n13. \\nUesugi M et al. Impact of cytochrome P450 3A5 polymorphism in graft livers on the frequency of acute cellular rejection in living-donor liver transplantation.\\nPharmacogenet Genomics 2014;24:356-66.\\n14. \\nTerrazzino S et al. The effect of CYP3A5 6986A>G and ABCB1 3435C>T on tacrolimus dose-adjusted trough levels and acute rejection rates in renal transplant patients: a\\nsystematic review and meta-analysis. Pharmacogenet Genomics 2012;22:642-5.\\n15. \\nTang HL et al. Lower tacrolimus daily dose requirements and acute rejection rates in the CYP3A5 non-expressers than expressers. Pharmacogenet Genomics\\n2011;21:713-20.\\n16. \\nThervet E et al. Optimization of initial tacrolimus dose using pharmacogenetic testing. Clin Pharmacol Ther 2010; 87: 721-726.\\n17. \\nSatoh S et al. Lack of tacrolimus circadian pharmacokinetics and CYP3A5 pharmacogenetics in the early and maintenance stages in Japanese renal transplant recipients.\\nBr J Clin Pharmacol 2008;66:207-14.\\n18. \\nKlauke B et al. No association between single nucleotide polymorphisms and the development of nephrotoxicity after orthotopic heart transplantation. J Heart Lung\\nTransplant 2008;27:741-5.\\n19. \\nFukudo M et al. Impact of MDR1 and CYP3A5 on the oral clearance of tacrolimus and tacrolimus-related renal dysfunction in adult living-donor liver transplant patients.\\nPharmacogenet Genomics 2008;18:413-23.\\n20. \\nHesselink DA et al. CYP3A5 genotype is not associated with a higher risk of acute rejection in tacrolimus-treated renal transplant recipients. Pharmacogenet Genomics\\n2008;18:339-48.\\n21. \\nKuypers DR et al. CYP3A5 and CYP3A4 but not MDR1 single-nucleotide polymorphisms determine long-term tacrolimus disposition and drug-related nephrotoxicity in\\nrenal recipients. Clin Pharmacol Ther 2007;82:711-25.\\n22. \\nRenders L et al. CYP3A5 genotype markedly influences the pharmacokinetics of tacrolimus and sirolimus in kidney transplant recipients. Clin Pharmacol Ther\\n2007;81:228-34.\\n23. \\nMourad M et al. The influence of genetic polymorphisms of cytochrome P450 3A5 and ABCB1 on starting dose- and weight-standardized tacrolimus trough concentrations\\nafter kidney transplantation in relation to renal function. Clin Chem Lab Med 2006;44:1192-8.\\n24. \\nRoy JN et al. Cyp3A4, Cyp3A5, and MDR-1 genetic influences on tacrolimus pharmacokinetics in renal transplant recipients. Pharmacogenet Genomics 2006;16:659-65.\\n25. \\nCheung CY et al. Influence of different allelic variants of the CYP3A and ABCB1 genes on the tacrolimus pharmacokinetic profile of Chinese renal transplant recipients.\\nPharmacogenomics 2006;7:563-74.\\n26. \\nUesugi M et al. Effect of intestinal CYP3A5 on postoperative tacrolimus trough levels in living-donor liver transplant recipients. Pharmacogenet Genomics 2006;16:119-\\n27.\\n27. \\nZhang X et al. Influence of CYP3A5 and MDR1 polymorphisms on tacrolimus concentration in the early stage after renal transplantation. Clin Transplant 2005;19:638-43.\\n28. \\nTada H et al. Impact of CYP3A5 and MDR1(ABCB1) C3435T polymorphisms on the pharmacokinetics of tacrolimus in renal transplant recipients. Transplant Proc\\n2005;37:1730-2.\\n29. \\nMacphee IA et al. Tacrolimus pharmacogenetics: the CYP3A5*1 allele predicts low dose-normalized tacrolimus blood concentrations in whites and South Asians.\\nTransplantation 2005;79:499-502.\\n30. \\nHesselink DA et al. Genetic polymorphisms of the CYP3A4, CYP3A5, and MDR-1 genes and pharmacokinetics of the calcineurin inhibitors cyclosporine and tacrolimus.\\nClin Pharmacol Ther 2003;74:245-54.\\nDate 03-02-2020\\nCYP3A5 homozygote expresser: tacrolimus\\n2357\\nAn increase of the initial dose can result in an increased chance of reaching a tacrolimus concentration within the target range before the start of therapeutic drug monitoring.\\nHowever, there is no direct evidence that this results in improved clinical results. The genetic variation results in an increased conversion of tacrolimus to inactive metabolites\\nand therefore a higher required dose.\\n- Indications OTHER than liver transplantation:        - Use 2.5 times the initial dose that would yield the desired result in non-expressers    \\nAdjustment of the dose should then be based on therapeutic drug monitoring.\\nFor example: One Dutch study found a median trough concentration for tacrolimus after three days of 9.4 ng/mL at an initial dose of 0.15 mg/kg twice daily for 5 homozygous\\nkidney transplant patients. Their target value was 10 - 15 ng/mL.\\n- LIVER transplantation:    \\nIn addition to the patient’s genotype, the metabolism of tacrolimus is also determined by the genotype of the transplanted liver.\\n- LIVER is also of the genotype HOMOZYGOUS EXPRESSER:\\n- Use 2.5 times the normal initial dose\\nAdjustment of the dose should then be based on therapeutic drug monitoring.\\n- LIVER has a DIFFERENT genotype:\\nThere is insufficient evidence in the literature to support a dose recommendation.\\nLiterature:\\n1. \\nLargeau B et al. Comparison of tacrolimus starting doses based on CYP3A5 phenotype or genotype in kidney transplant recipients. Prog Transplant\\n2019:1526924819873905 [Epub ahead of print].\\n2. \\nFernando ME et al. Influence of CYP3A5 and ABCB1 polymorphism on tacrolimus drug dosing in South Indian renal allograft recipients. Indian J Nephrol 2019;29:261-6.\\n3. \\nSeibert SR et al. Tacrolimus trough and dose intra-patient variability and CYP3A5 genotype: effects on acute rejection and graft failure in European American and African\\nAmerican kidney transplant recipients. Clin Transplant 2018;32: e13424.\\n4. \\nLiu F et al. Long-term influence of CYP3A5, CYP3A4, ABCB1, and NR1I2 polymorphisms on tacrolimus concentration in Chinese renal transplant recipients. Genet Test\\nMol Biomarkers 2017;21:663-73.\\n5. \\nPallet N et al. Long-term clinical impact of adaptation of initial tacrolimus dosing to CYP3A5 genotype. Am J Transplant 2016;16:2670-5.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 88}),\n",
       " Document(page_content='6. \\nShuker N et al. A randomized controlled trial comparing the efficacy of Cyp3a5 genotype-based with body-weight-based tacrolimus dosing after living donor kidney\\ntransplantation. Am J Transplant 2016;16:2085-96.\\n7. \\nDe Meyer M et al. Pharmacogenetic-based strategy using de novo tacrolimus once daily after kidney transplantation: prospective pilot study. Pharmacogenomics\\n2016;17:1019-27.\\n8. \\nYaowakulpatana K et al. Impact of CYP3A5 polymorphism on trough concentrations and outcomes of tacrolimus minimization during the early period after kidney\\ntransplantation. Eur J Clin Pharmacol 2016;72:277-83.\\n9. \\nPulk RA et al. Multigene predictors of tacrolimus exposure in kidney transplant recipients. Pharmacogenomics 2015;16:841-54.\\n10. \\nWang L et al. Benefits of minimizing immunosuppressive dosage according to cytochrome P450 3A5 genotype in liver transplant patients: findings from a single-center\\nstudy. Genet Mol Res 2015;14:3191-9.\\n11. \\nRojas L et al. Effect of CYP3A5*3 on kidney transplant recipients treated with tacrolimus: a systematic review and meta-analysis of observational studies.\\nPharmacogenomics J 2015;15:38-48.\\n12. \\nBuendia JA et al. Effects of combinational CYP3A5 6986A>G polymorphism in graft liver and native intestine on the pharmacokinetics of tacrolimus in liver transplant\\npatients: a meta-analysis. Ther Drug Monit 2014;36:442-7.\\n13. \\nUesugi M et al. Impact of cytochrome P450 3A5 polymorphism in graft livers on the frequency of acute cellular rejection in living-donor liver transplantation.\\nPharmacogenet Genomics 2014;24:356-66.\\n14. \\nTerrazzino S et al. The effect of CYP3A5 6986A>G and ABCB1 3435C>T on tacrolimus dose-adjusted trough levels and acute rejection rates in renal transplant patients: a\\nsystematic review and meta-analysis. Pharmacogenet Genomics 2012;22:642-5.\\n15. \\nTang HL et al. Lower tacrolimus daily dose requirements and acute rejection rates in the CYP3A5 non-expressers than expressers. Pharmacogenet Genomics\\n2011;21:713-20.\\n16. \\nThervet E et al. Optimization of initial tacrolimus dose using pharmacogenetic testing. Clin Pharmacol Ther 2010; 87: 721-726.\\n17. \\nSatoh S et al. Lack of tacrolimus circadian pharmacokinetics and CYP3A5 pharmacogenetics in the early and maintenance stages in Japanese renal transplant recipients.\\nBr J Clin Pharmacol 2008;66:207-14.\\n18. \\nKlauke B et al. No association between single nucleotide polymorphisms and the development of nephrotoxicity after orthotopic heart transplantation. J Heart Lung\\nTransplant 2008;27:741-5.\\n19. \\nFukudo M et al. Impact of MDR1 and CYP3A5 on the oral clearance of tacrolimus and tacrolimus-related renal dysfunction in adult living-donor liver transplant patients.\\nPharmacogenet Genomics 2008;18:413-23.\\n20. \\nHesselink DA et al. CYP3A5 genotype is not associated with a higher risk of acute rejection in tacrolimus-treated renal transplant recipients. Pharmacogenet Genomics\\n2008;18:339-48.\\n21. \\nKuypers DR et al. CYP3A5 and CYP3A4 but not MDR1 single-nucleotide polymorphisms determine long-term tacrolimus disposition and drug-related nephrotoxicity in\\nrenal recipients. Clin Pharmacol Ther 2007;82:711-25.\\n22. \\nRenders L et al. CYP3A5 genotype markedly influences the pharmacokinetics of tacrolimus and sirolimus in kidney transplant recipients. Clin Pharmacol Ther\\n2007;81:228-34.\\n23. \\nMourad M et al. The influence of genetic polymorphisms of cytochrome P450 3A5 and ABCB1 on starting dose- and weight-standardized tacrolimus trough concentrations\\nafter kidney transplantation in relation to renal function. Clin Chem Lab Med 2006;44:1192-8.\\n24. \\nRoy JN et al. Cyp3A4, Cyp3A5, and MDR-1 genetic influences on tacrolimus pharmacokinetics in renal transplant recipients. Pharmacogenet Genomics 2006;16:659-65.\\n25. \\nCheung CY et al. Influence of different allelic variants of the CYP3A and ABCB1 genes on the tacrolimus pharmacokinetic profile of Chinese renal transplant recipients.\\nPharmacogenomics 2006;7:563-74.\\n26. \\nUesugi M et al. Effect of intestinal CYP3A5 on postoperative tacrolimus trough levels in living-donor liver transplant recipients. Pharmacogenet Genomics 2006;16:119-\\n27.\\n27. \\nZhang X et al. Influence of CYP3A5 and MDR1 polymorphisms on tacrolimus concentration in the early stage after renal transplantation. Clin Transplant 2005;19:638-43.\\n28. \\nTada H et al. Impact of CYP3A5 and MDR1(ABCB1) C3435T polymorphisms on the pharmacokinetics of tacrolimus in renal transplant recipients. Transplant Proc\\n2005;37:1730-2.\\n29. \\nMacphee IA et al. Tacrolimus pharmacogenetics: the CYP3A5*1 allele predicts low dose-normalized tacrolimus blood concentrations in whites and South Asians.\\nTransplantation 2005;79:499-502.\\n30. \\nHesselink DA et al. Genetic polymorphisms of the CYP3A4, CYP3A5, and MDR-1 genes and pharmacokinetics of the calcineurin inhibitors cyclosporine and tacrolimus.\\nClin Pharmacol Ther 2003;74:245-54.\\nDate 03-02-2020\\nCYP2D6 IM: tamoxifen\\n1602\\nThis gene variation reduces the conversion of tamoxifen to the active metabolite endoxifen. This can result in reduced effectiveness.\\nRecommendation:\\n1. \\nselect an alternative or measure the endoxifen concentration and increase the dose if necessary by a factor of 1.5-2\\nAromatase inhibitors are a possible alternative for post-menopausal women.\\n2. \\nif TAMOXIFEN is selected: avoid co-medication with CYP2D6 inhibitors such as paroxetine and fluoxetine\\nLiterature:\\n1. \\nWelzen ME et al. The effect of tamoxifen dose increment in patients with impaired CYP2D6 activity. Ther Drug Monit 2015;37:501-7.\\n2. \\nMartinez de Dueñas E et al. Adjusting the dose of tamoxifen in patients with early breast cancer and CYP2D6 poor metabolizer phenotype. Breast 2014;23:400-6.\\n3. \\nProvince MA et al. CYP2D6 genotype and adjuvant tamoxifen: meta-analysis of heterogeneous study populations. Clin Pharmacol Ther 2014;95:216-27.\\n4. \\nJung JA et al. Association between CYP2D6 genotypes and the clinical outcomes of adjuvant tamoxifen for breast cancer: a meta-analysis. Pharmacogenomics2014;15:49-\\n60.\\n5. \\nCronin-Fenton DP et al. Metabolism and transport of tamoxifen in relation to its effectiveness: new perspectives on an ongoing controversy. Future Oncol 2014;10:107-22.\\n6. \\nLum DW et al. CYP2D6 genotype and tamoxifen response for breast cancer: a systematic review and meta-analysis. PLoS One 2013;8:e76648.\\n7. \\nZeng Z et al. CYP2D6 polymorphisms influence tamoxifen treatment outcomes in breast cancer patients: a meta-analysis. Cancer Chemother Pharmacol 2013;72:287-303.\\n8. \\nRegan MM et al. CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer: the breast international group 1-98 trial.\\nJ Natl Cancer Inst 2012;104:441-51.\\n9. \\nRae JM et al. CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen-treated breast cancer patients. J Natl Cancer Inst 2012;104:452-60.\\n10. \\nPark IH et al. Lack of any association between functionally significant CYP2D6 polymorphisms and clinical outcomes in early breast cancer patients receiving adjuvant\\ntamoxifen treatment. Breast Cancer Res Treat 2012;131:455-61.\\n11. \\nKiyotani K et al. Dose-adjustment study of tamoxifen based on CYP2D6 genotypes in Japanese breast cancer patients. Breast Cancer Res Treat 2012;131:137-45.\\n12. \\nBarginear MF et al. Increasing tamoxifen dose in breast cancer patients based on CYP2D6 genotypes and endoxifen levels: effect on active metabolite isomers and the\\nantiestrogenic activity score. Clin Pharmacol Ther 2011;90:605-11.\\n13. \\nIrvin WJ Jr et al. Genotype-guided tamoxifen dosing increases active metabolite exposure in women with reduced CYP2D6 metabolism: a multicenter study. J Clin Oncol\\n2011;29:3232-9.\\n14. \\nLash TL et al. CYP2D6 inhibition and breast cancer recurrence in a population-based study in Denmark. J Natl Cancer Inst 2011;103:489-500.\\n15. \\nThompson AM et al. Comprehensive CYP2D6 genotype and adherence affect outcome in breast cancer patients treated with tamoxifen monotherapy. Breast Cancer Res\\nTreat 2011;125:279-87.\\n16. \\nStingl JC et al. Impact of CYP2D6 *4 genotype on progression free survival in tamoxifen breast cancer treatment. Curr Med Res Opin 2010;26:2535-42.\\n17. \\nLammers LA et al. The impact of CYP2D6-predicted phenotype on tamoxifen treatment outcome in patients with metastatic breast cancer. Br J Cancer 2010;103:765-71.\\n18. \\nSchroth W et al. CYP2D6 polymorphisms as predictors of outcome in breast cancer patients treated with tamoxifen: expanded polymorphism coverage improves risk\\nstratification. Clin Cancer Res 2010;16:4468-77.\\n19. \\nSeruga B et al. Cytochrome P450 2D6 and outcomes of adjuvant tamoxifen therapy: results of a meta-analysis. Breast Cancer Res Treat 2010;122:609-17.\\n20. \\nKiyotani K et al. Lessons for pharmacogenomics studies: association study between CYP2D6 genotype and tamoxifen response. Pharmacogenet Genomics 2010;20:565-8.\\n21. \\nAbraham JE et al. CYP2D6 gene variants: association with breast cancer specific survival in a cohort of breast cancer patients from the United Kingdom treated with\\nadjuvant tamoxifen. Breast Cancer Res 2010;12:R64.\\n22. \\nSchroth W et al. Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated with tamoxifen. JAMA 2009;302:1429-\\n36.\\n23. \\nGjerde J et al. Effects of CYP2D6 and SULT1A1 genotypes including SULT1A1 gene copy number on tamoxifen metabolism. Ann Oncol 2008;19:56-61.\\n24. \\nSchroth W et al. Breast cancer treatment outcome with adjuvant tamoxifen relative to patient CYP2D6 and CYP2C19 genotypes. J Clin Oncol 2007;25:5187-93.\\n25. \\nLim HS et al. Clinical implications of CYP2D6 genotypes predictive of tamoxifen pharmacokinetics in metastatic breast cancer. J Clin Oncol 2007;25:3837-45.\\n26. \\nGonzalez-Santiago S et al. CYP2D6*4 polymorphism as blood predictive biomarker of breast cancer relapse in patients receiving adjuvant tamoxifen. J Clin Oncol\\n2007;25(18S):590.\\n27. \\nWegman P et al. Genetic variants of CYP3A5, CYP2D6, SULT1A1, UGT2B15 and tamoxifen response in postmenopausal patients with breast cancer. Breast Cancer Res\\n2007;9:R7.\\n28. \\nGoetz MP et al. The impact of cytochrome P450 2D6 metabolism in women receiving adjuvant tamoxifen. Breast Cancer Res Treat 2007;101:113-21.\\n29. \\nBorges S et al. Quantitative effect of CYP2D6 genotype and inhibitors on tamoxifen metabolism: implication for optimization of breast cancer treatment. Clin Pharmacol\\nTher 2006;80:61-74.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 89}),\n",
       " Document(page_content='30. \\nNowell SA et al. Association of genetic variation in tamoxifen-metabolizing enzymes with overall survival and recurrence of disease in breast cancer patients. Breast\\nCancer Res Treat 2005;91:249-58.\\n31. \\nJin Y et al. CYP2D6 genotype, antidepressant use, and tamoxifen metabolism during adjuvant breast cancer treatment. J Natl Cancer Inst 2005;97:30-9.\\n32. \\nWegman P et al. Genotype of metabolic enzymes and the benefit of tamoxifen in postmenopausal breast cancer patients. Breast Cancer Res 2005;7:R284-90.\\nDate 09-11-2015\\nCYP2D6 PM: tamoxifen\\n1601\\nThis gene variation reduces the conversion of tamoxifen to the active metabolite endoxifen. This can result in reduced effectiveness.\\nRecommendation:\\n1. \\nselect an alternative or increase the dose to 40 mg/day and monitor the endoxifen concentration\\nStudies have demonstrated that PM can achieve an adequate endoxifen concentration when the dose is increased to 40-60 mg/day.\\nAromatase inhibitors are a possible alternative for post-menopausal women.\\nLiterature:\\n1. \\nWelzen ME et al. The effect of tamoxifen dose increment in patients with impaired CYP2D6 activity. Ther Drug Monit 2015;37:501-7.\\n2. \\nMartinez de Dueñas E et al. Adjusting the dose of tamoxifen in patients with early breast cancer and CYP2D6 poor metabolizer phenotype. Breast 2014;23:400-6.\\n3. \\nProvince MA et al. CYP2D6 genotype and adjuvant tamoxifen: meta-analysis of heterogeneous study populations. Clin Pharmacol Ther 2014;95:216-27.\\n4. \\nJung JA et al. Association between CYP2D6 genotypes and the clinical outcomes of adjuvant tamoxifen for breast cancer: a meta-analysis. Pharmacogenomics2014;15:49-\\n60.\\n5. \\nCronin-Fenton DP et al. Metabolism and transport of tamoxifen in relation to its effectiveness: new perspectives on an ongoing controversy. Future Oncol 2014;10:107-22.\\n6. \\nLum DW et al. CYP2D6 genotype and tamoxifen response for breast cancer: a systematic review and meta-analysis. PLoS One 2013;8:e76648.\\n7. \\nZeng Z et al. CYP2D6 polymorphisms influence tamoxifen treatment outcomes in breast cancer patients: a meta-analysis. Cancer Chemother Pharmacol 2013;72:287-303.\\n8. \\nRegan MM et al. CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer: the breast international group 1-98 trial.\\nJ Natl Cancer Inst 2012;104:441-51.\\n9. \\nRae JM et al. CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen-treated breast cancer patients. J Natl Cancer Inst 2012;104:452-60.\\n10. \\nBarginear MF et al. Increasing tamoxifen dose in breast cancer patients based on CYP2D6 genotypes and endoxifen levels: effect on active metabolite isomers and the\\nantiestrogenic activity score. Clin Pharmacol Ther 2011;90:605-11.\\n11. \\nIrvin WJ Jr et al. Genotype-guided tamoxifen dosing increases active metabolite exposure in women with reduced CYP2D6 metabolism: a multicenter study. J Clin Oncol\\n2011;29:3232-9.\\n12. \\nLash TL et al. CYP2D6 inhibition and breast cancer recurrence in a population-based study in Denmark. J Natl Cancer Inst 2011;103:489-500.\\n13. \\nThompson AM et al. Comprehensive CYP2D6 genotype and adherence affect outcome in breast cancer patients treated with tamoxifen monotherapy. Breast Cancer Res\\nTreat 2011;125:279-87.\\n14. \\nStingl JC et al. Impact of CYP2D6 *4 genotype on progression free survival in tamoxifen breast cancer treatment. Curr Med Res Opin 2010;26:2535-42.\\n15. \\nLammers LA et al. The impact of CYP2D6-predicted phenotype on tamoxifen treatment outcome in patients with metastatic breast cancer. Br J Cancer 2010;103:765-71.\\n16. \\nSchroth W et al. CYP2D6 polymorphisms as predictors of outcome in breast cancer patients treated with tamoxifen: expanded polymorphism coverage improves risk\\nstratification. Clin Cancer Res 2010;16:4468-77.\\n17. \\nSeruga B et al. Cytochrome P450 2D6 and outcomes of adjuvant tamoxifen therapy: results of a meta-analysis. Breast Cancer Res Treat 2010;122:609-17.\\n18. \\nKiyotani K et al. Lessons for pharmacogenomics studies: association study between CYP2D6 genotype and tamoxifen response. Pharmacogenet Genomics 2010;20:565-8.\\n19. \\nAbraham JE et al. CYP2D6 gene variants: association with breast cancer specific survival in a cohort of breast cancer patients from the United Kingdom treated with\\nadjuvant tamoxifen. Breast Cancer Res 2010;12:R64.\\n20. \\nSchroth W et al. Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated with tamoxifen. JAMA 2009;302:1429-\\n36.\\n21. \\nGjerde J et al. Effects of CYP2D6 and SULT1A1 genotypes including SULT1A1 gene copy number on tamoxifen metabolism. Ann Oncol 2008;19:56-61.\\n22. \\nSchroth W et al. Breast cancer treatment outcome with adjuvant tamoxifen relative to patient CYP2D6 and CYP2C19 genotypes. J Clin Oncol 2007;25:5187-93.\\n23. \\nGonzalez-Santiago S et al. CYP2D6*4 polymorphism as blood predictive biomarker of breast cancer relapse in patients receiving adjuvant tamoxifen. J Clin Oncol\\n2007;25(18S):590.\\n24. \\nWegman P et al. Genetic variants of CYP3A5, CYP2D6, SULT1A1, UGT2B15 and tamoxifen response in postmenopausal patients with breast cancer. Breast Cancer Res\\n2007;9:R7.\\n25. \\nGoetz MP et al. The impact of cytochrome P450 2D6 metabolism in women receiving adjuvant tamoxifen. Breast Cancer Res Treat 2007;101:113-21.\\n26. \\nBonanni B ea. Polymorphism in the CYP2D6 tamoxifen-metabolizing gene influences clinical effect but not hot flashes: data from the Italian Tamoxifen Trial. J Clin Oncol\\n2006;24:3708-9.\\n27. \\nBorges S et al. Quantitative effect of CYP2D6 genotype and inhibitors on tamoxifen metabolism: implication for optimization of breast cancer treatment. Clin Pharmacol\\nTher 2006;80:61-74.\\n28. \\nGoetz MP et al. Pharmacogenetics of tamoxifen biotransformation is associated with clinical outcomes of efficacy and hot flashes. J Clin Oncol 2005;23:9312-8.\\n29. \\nNowell SA et al. Association of genetic variation in tamoxifen-metabolizing enzymes with overall survival and recurrence of disease in breast cancer patients. Breast\\nCancer Res Treat 2005;91:249-58.\\n30. \\nJin Y et al. CYP2D6 genotype, antidepressant use, and tamoxifen metabolism during adjuvant breast cancer treatment. J Natl Cancer Inst 2005;97:30-9.\\n31. \\nWegman P et al. Genotype of metabolic enzymes and the benefit of tamoxifen in postmenopausal breast cancer patients. Breast Cancer Res 2005;7:R284-90.\\n32. \\nSPC Tamoxifen PCH.\\nDate 09-11-2015\\nCYP2D6 UM: tamoxifen\\n1603\\nNO action is needed for this gene-drug interaction.\\nAs a result of the genetic variation, the plasma concentration of the active metabolites 4-hydroxytamoxifen and endoxifen can increase. However, there is no evidence that this\\nresults in an increase in the side effects.\\nLiterature:\\n1. \\nMartinez de Dueñas E et al. Adjusting the dose of tamoxifen in patients with early breast cancer and CYP2D6 poor metabolizer phenotype. Breast 2014;23:400-6.\\n2. \\nSchroth W et al. CYP2D6 polymorphisms as predictors of outcome in breast cancer patients treated with tamoxifen: expanded polymorphism coverage improves risk\\nstratification. Clin Cancer Res 2010;16:4468-77.\\n3. \\nGjerde J et al. Effects of CYP2D6 and SULT1A1 genotypes including SULT1A1 gene copy number on tamoxifen metabolism. Ann Oncol 2008;19:56-61.\\n4. \\nLim HS et al. Clinical implications of CYP2D6 genotypes predictive of tamoxifen pharmacokinetics in metastatic breast cancer. J Clin Oncol 2007;25:3837-45.\\nDate 09-11-2015\\nDPD AS 0: tegafur\\n2553\\nThe gene variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur to inactive metabolites means that the normal dose is an overdose.\\navoid tegafur\\nFluorouracil and capecitabine are not suitable alternatives, as these are also metabolised by DPD.\\nIf it is not possible to avoid tegafur: start with a very low dose and adjust the initial dose based on toxicity and efficacy. A substantiated recommendation for dose\\nreduction cannot be made based on the literature. The recommendation for fluorouracil and capecitabine is to determine the residual DPD activity in mononuclear cells\\nfrom peripheral blood and to adjust the initial dose accordingly. A patient with 0.5% of the normal DPD activity tolerated 0.8% of the standard capecitabine dose (150 mg', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 90}),\n",
       " Document(page_content='every 5 days). A patient with undetectable DPD activity tolerated 0.43% of the standard capecitabine dose (150 mg every 5 days with every third dose skipped)\\nLiterature:\\n1. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or capecitabine. Ann Intern Med\\n2010;153:767-8.\\n2. \\nSPC Teysuno.\\nDate 13-05-2019\\nDPD AS 1,5: tegafur\\n4892\\nThe gene variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur into inactive metabolites means that the normal dose is an overdose.\\nAvoid tegafur or start with a low dose and adjust the initial dose based on toxicity and efficacy\\nFluorouracil and capecitabine are not alternatives, as these are also metabolised by DPD.\\nIt is not possible to offer substantiated advice for dose reduction based on the literature.\\nFor fluorouracil and capecitabine, starting with 50 % of the standard dose is recommended and the dose should then be adjusted based on toxicity and effectiveness.\\nIn one study, the average dose of fluorouracil/capecitabine after titration was 64% of the standard dose for 17 patients with genotype *1/2846T and 74% of the standard\\ndose for 51 patients with genotype *1/1236A.\\nLiterature:\\n1. \\nCubero DI et al. Tegafur-uracil is a safe alternative for the treatment of colorectal cancer in patients with partial dihydropyrimidine dehydrogenase deficiency: a proof of\\nprinciple. Ther Adv Med Oncol 2012;4:167-72.\\n2. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or capecitabine. Ann Intern Med\\n2010;153:767-8.\\n3. \\nSPC Teysuno.\\nDate 13-05-2019\\nDPD AS 1: tegafur\\n2554\\nThe gene variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur into inactive metabolites means that the normal dose is an overdose.\\nAvoid tegafur or start with a low dose and adjust the initial dose based on toxicity and efficacy\\nFluorouracil and capecitabine are not alternatives, as these are also metabolised by DPD.\\nIt is not possible to offer substantiated advice for dose reduction based on the literature.\\nFor fluorouracil and capecitabine, starting with 50 % of the standard dose is recommended.\\nLiterature:\\n1. \\nCubero DI et al. Tegafur-uracil is a safe alternative for the treatment of colorectal cancer in patients with partial dihydropyrimidine dehydrogenase deficiency: a proof of\\nprinciple. Ther Adv Med Oncol 2012;4:167-72.\\n2. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or capecitabine. Ann Intern Med\\n2010;153:767-8.\\n3. \\nSPC Teysuno.\\nDate 13-05-2019\\nDPD FENO: tegafur\\n4891\\nThe gene variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur to inactive metabolites means that the normal dose is an overdose.\\nAvoid tegafur or start with a low dose and adjust the initial dose based on toxicity and efficacy\\nFluorouracil and capecitabine are not alternatives, as these are also metabolised by DPD.\\nIt is not possible to offer substantiated advice for dose reduction based on the literature.\\nFor fluorouracil and capecitabine, it is recommended to determine the residual DPD activity in mononuclear cells from peripheral blood and adjust the initial dose based\\non phenotype and genotype.\\nLiterature:\\n1. \\nCubero DI et al. Tegafur-uracil is a safe alternative for the treatment of colorectal cancer in patients with partial dihydropyrimidine dehydrogenase deficiency: a proof of\\nprinciple. Ther Adv Med Oncol 2012;4:167-72.\\n2. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or capecitabine. Ann Intern Med\\n2010;153:767-8.\\n3. \\nSPC Teysuno.\\nDate 13-05-2019\\nCYP2C19 IM: ticagrelor\\n3516\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nRath PC et al. A study on the impact of CYP2C19 genotype and platelet reactivity assay on patients undergoing PCI. Indian Heart J 2015;67:114-21.\\n2. \\nSteg PG et al. Stent thrombosis with ticagrelor versus clopidogrel in patients with acute coronary syndromes: an analysis from the prospective, randomized PLATO trial.\\nCirculation 2013;128:1055-65.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 91}),\n",
       " Document(page_content=\"3. \\nTantry US et al. First analysis of the relation between CYP2C19 genotype and pharmacodynamics in patients treated with ticagrelor versus clopidogrel: the\\nONSET/OFFSET and RESPOND genotype studies. Circ Cardiovasc Genet 2010;3:556-66.\\n4. \\nWallentin L et al. Effect of CYP2C19 and ABCB1 single nucleotide polymorphisms on outcomes of treatment with ticagrelor versus clopidogrel for acute coronary\\nsyndromes: a genetic substudy of the PLATO trial. Lancet 2010;376:1320-8.\\n5. \\nSPC’s Brilique (NL) en Brilinta (VS).\\nDate 19-11-2018\\nCYP2C19 PM: ticagrelor\\n3515\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nZhong Z et al. Effect of cytochrome P450 2C19 polymorphism on adverse cardiovascular events after drug-eluting stent implantation in a large Hakka population with\\nacute coronary syndrome receiving clopidogrel in southern China. Eur J Clin Pharmacol 2018;74:423-31.\\n2. \\nShen DL et al. Clinical value of CYP2C19 genetic testing for guiding the antiplatelet therapy in a Chinese population. J Cardiovasc Pharmacol 2016;67:232-6.\\n3. \\nXiong R et al. A randomized controlled trial to assess the efficacy and safety of doubling dose clopidogrel versus ticagrelor for the treatment of acute coronary syndrome\\nin patients with CYP2C19*2 homozygotes. Int J Clin Exp Med 2015;8:13310-6.\\n4. \\nRath PC et al. A study on the impact of CYP2C19 genotype and platelet reactivity assay on patients undergoing PCI. Indian Heart J 2015;67:114-21.\\n5. \\nSteg PG et al. Stent thrombosis with ticagrelor versus clopidogrel in patients with acute coronary syndromes: an analysis from the prospective, randomized PLATO trial.\\nCirculation 2013;128:1055-65.\\n6. \\nTantry US et al. First analysis of the relation between CYP2C19 genotype and pharmacodynamics in patients treated with ticagrelor versus clopidogrel: the\\nONSET/OFFSET and RESPOND genotype studies. Circ Cardiovasc Genet 2010;3:556-66.\\n7. \\nWallentin L et al. Effect of CYP2C19 and ABCB1 single nucleotide polymorphisms on outcomes of treatment with ticagrelor versus clopidogrel for acute coronary\\nsyndromes: a genetic substudy of the PLATO trial. Lancet 2010;376:1320-8.\\n8. \\nSPC’s Brilique (NL) en Brilinta (VS).\\nDate 19-11-2018\\nCYP2C19 UM: ticagrelor\\n3517\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1. \\nRath PC et al. A study on the impact of CYP2C19 genotype and platelet reactivity assay on patients undergoing PCI. Indian Heart J 2015;67:114-21.\\n2. \\nSteg PG et al. Stent thrombosis with ticagrelor versus clopidogrel in patients with acute coronary syndromes: an analysis from the prospective, randomized PLATO trial.\\nCirculation 2013;128:1055-65.\\n3. \\nTantry US et al. First analysis of the relation between CYP2C19 genotype and pharmacodynamics in patients treated with ticagrelor versus clopidogrel: the\\nONSET/OFFSET and RESPOND genotype studies. Circ Cardiovasc Genet 2010;3:556-66.\\nDate 19-11-2018\\nNUDT15 IM: tioguanine\\n7033\\nGrade ≥ 2 leukopaenia occurs in an estimated 40% of these patients with standard therapy. The genetic variation increases the concentration of the fully activated metabolite\\nof tioguanine.\\nIMMUNOSUPPRESSION:\\nstart with 75% of the standard dose\\nAdjustment of the initial dose should be performed based on toxicity (monitoring of the blood counts) and efficacy.\\nMonitoring should be performed at an increased frequency.\\nNOTE: The percentage of 75% is based on the analogy with TPMT, for which the gene variants have a comparable effect on toxicity to those of NUDT15.\\nNOTE: Dose adjustment based on the total of 6-TGN metabolites is not possible for these patients, as they develop toxicity within the therapeutic range that applies\\nfor patients without gene variants.\\nLEUKAEMIA:\\nstart with 75% of the standard tioguanine dose or start with the standard dose and reduce to 75% if side effects necessitate a dose reduction\\nIt is not known whether dose reduction in advance results in the same efficacy as dose reduction based on toxicity.\\nAdjustment of the initial dose should be performed based on toxicity (monitoring of the blood counts) and efficacy.\\nMonitoring should be performed at an increased frequency.\\nNOTE: The percentage of 75% is based on the analogy with TPMT, for which the gene variants have a comparable effect on toxicity to those of NUDT15.\\nNOTE: Dose adjustment based on the total of 6-TGN metabolites is not possible for these patients, as they develop toxicity within the therapeutic range that applies\\nfor patients without gene variants.\\nNote: more stringent dose reductions are necessary if the patient is also TPMT IM.\\nLiterature:\\n1. \\nZhu Y et al. Combination of common and novel rare NUDT15 variants improves predictive sensitivity of thiopurine-induced leukopenia in children with acute\\nlymphoblastic leukemia. Haematologica 2018 Mar 8 [Epub ahead of print].\\n2. \\nZhang AL et al. Association of NUDT15 c.415C>T allele and thiopurine-induced leukocytopenia in Asians: a systematic review and meta-analysis. Ir J Med Sci\\n2018;187:145-153.\\n3. \\nYi ES et al. NUDT15 variants cause hematopoietic toxicity with low 6-TGN levels in children with acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 13 [Epub\\nahead of print].\\n4. \\nKim H et al. APEX1 polymorphism and mercaptopurine-related early onset neutropenia in pediatric acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 4 [Epub\\nahead of print].\\n5. \\nChao K et al. Combined detection of NUDT15 variants could highly predict thiopurine-induced leukopenia in Chinese patients with inflammatory bowel disease: a\\nmulticenter analysis. Inflamm Bowel Dis 2017;23:1592-9.\\n6. \\nYin D et al. Impact of NUDT15 polymorphisms on thiopurines-induced myelotoxicity and thiopurines tolerance dose. Oncotarget 2017;8:13575-85.\\n7. \\nLiang DC et al. NUDT15 gene polymorphism related to mercaptopurine intolerance in Taiwan Chinese children with acute lymphoblastic leukemia. Pharmacogenomics J\\n2016;16:536-9.\\n8. \\nZhu X et al. NUDT15 polymorphisms are better than thiopurine S-methyltransferase as predictor of risk for thiopurine-induced leukopenia in Chinese patients with\\nCrohn's disease. Aliment Pharmacol Ther 2016;44:967-75.\\n9. \\nMoriyama T et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet 2016;48:367-73.\\n10. \\nYang JJ et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol 2015;33:1235-\\n42.\\n11. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of thiopurine methyltransferase pharmacogenetics.\\nBr J Haematol. 2015;169:228-40. PubMed PMID: 25441457.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 92}),\n",
       " Document(page_content=\"12. \\nYang SK et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet 2014;46:1017-20.\\n13. \\nSmPC Lanvis.\\nDate 04-03-2019\\nNUDT15 PM: tioguanine\\n7034\\nGrade ≥ 2 leukopaenia occurs in an estimated 95% of these patients with standard therapy. The genetic variation increases the concentration of the fully activated metabolite\\nof tioguanine.\\navoid tioguanine\\nif it is not possible to avoid tioguanine: use 10% of the standard dose and advise patients to seek medical attention when symptoms of myelosuppression (such as severe\\nsore throat in combination with fever, regular nosebleeds and tendency to bruising) occur\\nAny adjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and efficacy.\\nMonitoring should be performed at an increased frequency.\\nNOTE: The percentage of 10% is based on the analogy with azathioprine and mercaptopurine and the analogy with TPMT, for which the gene variants have a comparable\\neffect on toxicity to those of NUDT15. For NUDT15 PM, a percentage of < 20% was calculated for azathioprine and mercaptopurine, but there were insufficient data\\navailable to calculate the exact percentage.\\nNOTE: Dose adjustment based on the total of 6-TGN metabolites is not possible for these patients, as they develop toxicity within the therapeutic range that applies for\\npatients without gene variants.\\nLiterature:\\n1. \\nZhu Y et al. Combination of common and novel rare NUDT15 variants improves predictive sensitivity of thiopurine-induced leukopenia in children with acute\\nlymphoblastic leukemia. Haematologica 2018 Mar 8 [Epub ahead of print].\\n2. \\nZhang AL et al. Association of NUDT15 c.415C>T allele and thiopurine-induced leukocytopenia in Asians: a systematic review and meta-analysis. Ir J Med Sci\\n2018;187:145-153.\\n3. \\nYi ES et al. NUDT15 variants cause hematopoietic toxicity with low 6-TGN levels in children with acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 13 [Epub\\nahead of print].\\n4. \\nKim H et al. APEX1 polymorphism and mercaptopurine-related early onset neutropenia in pediatric acute lymphoblastic leukemia. Cancer Res Treat 2017 Sep 4 [Epub\\nahead of print].\\n5. \\nChao K et al. Combined detection of NUDT15 variants could highly predict thiopurine-induced leukopenia in Chinese patients with inflammatory bowel disease: a\\nmulticenter analysis. Inflamm Bowel Dis 2017;23:1592-9.\\n6. \\nYin D et al. Impact of NUDT15 polymorphisms on thiopurines-induced myelotoxicity and thiopurines tolerance dose. Oncotarget 2017;8:13575-85.\\n7. \\nLiang DC et al. NUDT15 gene polymorphism related to mercaptopurine intolerance in Taiwan Chinese children with acute lymphoblastic leukemia. Pharmacogenomics J\\n2016;16:536-9.\\n8. \\nZhu X et al. NUDT15 polymorphisms are better than thiopurine S-methyltransferase as predictor of risk for thiopurine-induced leukopenia in Chinese patients with\\nCrohn's disease. Aliment Pharmacol Ther 2016;44:967-75.\\n9. \\nMoriyama T et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet 2016;48:367-73.\\n10. \\nYang JJ et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol 2015;33:1235-\\n42.\\n11. \\nYang SK et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet 2014;46:1017-20.\\n12. \\nSmPC’s Lanvis.\\nDate 04-03-2019\\nTPMT IM: tioguanine\\n1907\\nThe risk of serious adverse events such as myelosuppression is increased. The genetic variation increases the concentration of the active metabolites of thioguanine.\\nIMMUNOSUPPRESSION:\\nStart with 75% of the standard dose\\nAdjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and efficacy.\\nLEUKAEMIA:\\nstart with 75% of the standard tioguanine dose, or start with the standard dose and reduce to 75% if side effects necessitate a dose reduction\\nIt is not known whether dose reduction in advance results in the same efficacy as dose reduction based on toxicity.\\nThe initial dose should be adjusted based on toxicity (monitoring of the blood counts) and efficacy.\\nNote: more stringent dose reductions are necessary if the patient is also NUDT15 IM or NUDT15 PM.\\nLiterature:\\n1. \\nMcAtee CL et al. Treatment-related sinusoidal obstruction syndrome in children with de novo acute lymphoblastic leukemia during intensification. Cancer Chemother\\nPharmacol 2017;80:1261-4.\\n2. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of thiopurine methyltransferase pharmacogenetics.\\nBr J Haematol 2015;169:228-40.\\n3. \\nWray L et al. TPMT and MTHFR genotype is not associated with altered risk of thioguanine-related sinusoidal obstruction syndrome in pediatric acute lymphoblastic\\nleukemia: a report from the Children's Oncology Group. Pediatr Blood Cancer 2014;61:2086-8.\\n4. \\nLennard L et al. The thiopurine methyltransferase genetic polymorphism is associated with thioguanine-related veno-occlusive disease of the liver in children with acute\\nlymphoblastic leukemia. Clin Pharmacol Ther 2006;80:375-83.\\n5. \\nStanden GR et al. Heterozygosity for the thiopurine methyltransferase *3A allele in an acute non-lymphoblastic leukaemia patient with delayed marrow regeneration\\nfollowing H-DAT chemotherapy. Br J Haematol 2001;112:1089.\\n6. \\nTeml A et al. A prospective, open-label trial of 6-thioguanine in patients with ulcerative or indeterminate colitis. Scand J Gastroenterol 2005;40:1205-13.\\n7. \\nHerrlinger KR et al. Thioguanine-nucleotides do not predict efficacy of tioguanine in Crohn's disease. Aliment Pharmacol Ther 2004;19:1269-76.\\nDate 04-11-2019\\nTPMT PM: tioguanine\\n1908\\nThe risk of serious, life-threatening adverse events such as myelosuppression is strongly increased. The genetic variation increases the concentration of the active metabolites\\nof thioguanine.\\nChoose an alternative or use 6-7% of the standard dose\\nAny adjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and effectiveness.\\nIf the dose is decreased: advise patients to seek medical attention when symptoms of myelosuppression (such as severe sore throat in combination with fever, regular\\nnosebleeds and tendency to bruising) develop\\nLiterature:\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 93}),\n",
       " Document(page_content=\"1. \\nvan der Burg M and Gerding MN. Pancytopenie bij tioguaninegebruik. Ned Tijdschr Geneeskd 2018;162:D2839.\\n2. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of thiopurine methyltransferase pharmacogenetics.\\nBr J Haematol 2015;169:228-40.\\n3. \\nWray L et al. TPMT and MTHFR genotype is not associated with altered risk of thioguanine-related sinusoidal obstruction syndrome in pediatric acute lymphoblastic\\nleukemia: a report from the Children's Oncology Group. Pediatr Blood Cancer 2014;61:2086-8.\\n4. \\nMares WG et al. Safe 6-thioguanine therapy of a TPMT deficient Crohn's disease patient by using therapeutic drug monitoring. J Crohns Colitis 2009;3:128-30.\\n5. \\nMcBride KL et al. Severe 6-thioguanine-induced marrow aplasia in a child with acute lymphoblastic leukemia and inherited thiopurine methyltransferase deficiency. J\\nPediatr Hematol Oncol 2000;22:441-5.\\n6. \\nSmPC’s Lanvis (NL) and Tabloid (VS).\\nDate 04-11-2019\\nCYP2C9 IM ANDERS: tolbutamide\\n1903\\nNO action is required for this gene-drug interaction.\\nThere is insufficient evidence to state that the increased tolbutamide plasma concentration has any clinical consequences.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy volunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm Ther 2005;30:241-9.\\n5. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n6. \\nLee CR et al. Tolbutamide, flurbiprofen, and losartan as probes of CYP2C9 activity in humans. J Clin Pharmacol 2003;43:84-91.\\n7. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol Ther 2002;72:562-71.\\n8. \\nShon JH et al. Effects of CYP2C19 and CYP2C9 genetic polymorphisms on the disposition of and blood glucose lowering response to tolbutamide in humans.\\nPharmacogenetics 2002;12:111-9.\\n9. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose response in healthy volunteers.\\nPharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9 PM ANDERS: tolbutamide\\n1904\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy volunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm Ther 2005;30:241-9.\\n5. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n6. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol Ther 2002;72:562-71.\\n7. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose response in healthy volunteers.\\nPharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*1/*2: tolbutamide\\n1898\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy volunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n5. \\nLee CR et al. Tolbutamide, flurbiprofen, and losartan as probes of CYP2C9 activity in humans. J Clin Pharmacol 2003;43:84-91.\\n6. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol Ther 2002;72:562-71.\\n7. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose response in healthy volunteers.\\nPharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*1/*3: tolbutamide\\n1899\\nNO action is required for this gene-drug interaction.\\nThere is insufficient evidence to state that the increased tolbutamide plasma concentration has any clinical consequences.\\nLiterature:\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 94}),\n",
       " Document(page_content='1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy volunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm Ther 2005;30:241-9.\\n5. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n6. \\nLee CR et al. Tolbutamide, flurbiprofen, and losartan as probes of CYP2C9 activity in humans. J Clin Pharmacol 2003;43:84-91.\\n7. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol Ther 2002;72:562-71.\\n8. \\nShon JH et al. Effects of CYP2C19 and CYP2C9 genetic polymorphisms on the disposition of and blood glucose lowering response to tolbutamide in humans.\\nPharmacogenetics 2002;12:111-9.\\n9. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose response in healthy volunteers.\\nPharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*2/*2: tolbutamide\\n1900\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n4. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol Ther 2002;72:562-71.\\n5. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose response in healthy volunteers.\\nPharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*2/*3: tolbutamide\\n1901\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy volunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose response in healthy volunteers.\\nPharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*3/*3: tolbutamide\\n1902\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care patients with Type 2 diabetes mellitus.\\nPharmacogenomics 2010;11:1517-23.\\n2. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy volunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm Ther 2005;30:241-9.\\n5. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose response in healthy volunteers.\\nPharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2D6 IM: tramadol\\n1590\\nThe genetic variation reduces the conversion of tramadol to a metabolite with a higher activity. This can result in reduced analgesia.\\nRecommendation:\\nIt is not possible to provide a specific recommendation for dose adjustment, because the total analgesic effect changes when the ratio between the mother compound and the\\nactive metabolite changes.\\n1. \\nbe alert to a reduced effectiveness\\n2. \\nin the case of inadequate effectiveness:\\n1. \\ntry a dose increase\\n2. \\nif this does not work: choose an alternative\\nDo not select codeine, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n3. \\nif no alternative is selected: advise the patient to report inadequate analgesia', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 95}),\n",
       " Document(page_content='Literature:\\n1. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2. \\nDong H et al. Effect of the CYP2D6 gene polymorphism on postoperative analgesia of tramadol in Han nationality nephrectomy patients. Eur J Clin Pharmacol\\n2015;71:681-6.\\n3. \\nZhao Q et al. A logistic equation to determine the validity of tramadol from related gene polymorphisms and psychological factors. Pharmacogenomics 2014;15:487-95.\\n4. \\nMatouskova O et al. Pupillometry in healthy volunteers as a biomarker of tramadol efficacy. J Clin Pharm Ther 2011;36:513-7.\\n5. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain 2010;11:1274-81.\\n6. \\nKim E et al. Adverse events in analgesic treatment with tramadol associated with CYP2D6 extensive-metaboliser and OPRM1 high-expression variants. Ann Rheum Dis\\n2010;69:1889-90.\\n7. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates and infants. Pediatr Res 2008;63:674-9.\\n8. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin Pharmacol Ther 2007;82:41-7.\\n9. \\nSlanar O et al. Miotic action of tramadol is determined by CYP2D6 genotype. Physiol Res 2007;56:129-36.\\n10. \\nWang G et al. Effect of the CYP2D6*10 C188T polymorphism on postoperative tramadol analgesia in a Chinese population. Eur J Clin Pharmacol 2006;62:927-31.\\n11. \\nBorlak J et al. A rapid and simple CYP2D6 genotyping assay: case study with the analgetic tramadol. Metabolism 2003;52:1439-43.\\n12. \\nGan SH et al. Correlation of tramadol pharmacokinetics and CYP2D6*10 genotype in Malaysian subjects. J Pharm Biomed Anal 2002;30:189-195.\\n13. \\nAbdel-Rahman SM et al. Concordance between tramadol and dextromethorphan parent/metabolite ratios: the influence of CYP2D6 and non-CYP2D6 pathways on\\nbiotransformation. J Clin Pharmacol 2002;42:24-9.\\nDate 20-11-2017\\nCYP2D6 PM: tramadol\\n1589\\nThe genetic variation reduces the conversion of tramadol to a metabolite with a higher activity. This can result in reduced analgesia.\\nRecommendation:\\nIt is not possible to provide a specific recommendation for dose adjustment, because the total analgesic effect changes when the ratio between the mother compound and the\\nactive metabolite changes.\\n1. \\nbe alert to a reduced effectiveness\\n2. \\nin the case of inadequate effectiveness:\\n1. \\ntry a dose increase.\\n2. \\nif this does not work: choose an alternative\\nDo not select codeine, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n3. \\nif no alternative is selected: advise the patient to report inadequate analgesia\\nLiterature:\\n1. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2. \\nMatouskova O et al. Pupillometry in healthy volunteers as a biomarker of tramadol efficacy. J Clin Pharm Ther 2011;36:513-7.\\n3. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain 2010;11:1274-81.\\n4. \\nHalling J et al. CYP2D6 polymorphism in relation to tramadol metabolism: a study of faroese patients. Ther Drug Monit 2008;30:271-5.\\n5. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates and infants. Pediatr Res 2008;63:674-9.\\n6. \\nKirchheiner J et al. Effects of the CYP2D6 gene duplication on the pharmacokinetics and pharmacodynamics of tramadol. J Clin Psychopharmacol 2008;28:78-83.\\n7. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin Pharmacol Ther 2007;82:41-7.\\n8. \\nGarcía-Quetglas E et al. Pharmacokinetics of tramadol enantiomers and their respective phase I metabolites in relation to CYP2D6 phenotype. Pharmacol Res\\n2007;55:122-30.\\n9. \\nSlanar O et al. Miotic action of tramadol is determined by CYP2D6 genotype. Physiol Res 2007;56:129-36.\\n10. \\nPedersen RS et al. Enantioselective pharmacokinetics of tramadol in CYP2D6 extensive and poor metabolizers. Eur J Clin Pharmacol 2006;62:513-21.\\n11. \\nEnggaard TP et al. The analgesic effect of tramadol after intravenous injection in healthy volunteers in relation to CYP2D6. Anesth Analg 2006;102:146-50.\\n12. \\nFliegert F et al. The effects of tramadol on static and dynamic pupillometry in healthy subjects--the relationship between pharmacodynamics, pharmacokinetics and\\nCYP2D6 metaboliser status. Eur J Clin Pharmacol 2005;61:257-66.\\n13. \\nBorlak J et al. A rapid and simple CYP2D6 genotyping assay: case study with the analgetic tramadol. Metabolism 2003;52:1439-43.\\n14. \\nStamer UM et al. Impact of CYP2D6 genotype on postoperative tramadol analgesia. Pain 2003;105:231-8.\\n15. \\nAbdel-Rahman SM et al. Concordance between tramadol and dextromethorphan parent/metabolite ratios: the influence of CYP2D6 and non-CYP2D6 pathways on\\nbiotransformation. J Clin Pharmacol 2002;42:24-9.\\n16. \\nPaar WD et al. Polymorphic CYP2D6 mediates O-demethylation of the opioid analgesic tramadol. Eur J Clin Pharmacol 1997;53:235-9.\\n17. \\nPoulsen L et al. The hypoalgesic effect of tramadol in relation to CYP2D6. Clin Pharmacol Ther 1996;60:636-44.\\n18. \\nSPC Ultram (VS).\\nDate 20-11-2017\\nCYP2D6 UM: tramadol\\n1591\\nThe genetic variation increases the conversion of tramadol to a metabolite with a stronger opioid effect. This can result in an increase in potentially life-threatening side\\neffects.\\nRecommendation:\\nAs the total analgesic effect changes when the ratio between the mother compound and the active metabolite changes, the effect of a dose reduction cannot be predicted with\\ncertainty.\\nselect an alternative\\nDo not choose codeine, as it is contra-indicated for CYP2D6 UM.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in side effects in patients.\\nif an alternative is not possible:\\nuse 40% of the standard dose\\nadvise the patient to report side effects (such as drowsiness, confusion, constipation, nausea and vomiting, respiratory depression or urine retention).\\nLiterature:\\n1. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2. \\nOrliaguet G et al. A case of respiratory depression in a child with ultrarapid CYP2D6 metabolism after tramadol. Pediatrics 2015;135:e753-5.\\n3. \\nSaarikoski T et al. Effects of terbinafine and itraconazole on the pharmacokinetics of orally administered tramadol. Eur J Clin Pharmacol 2015;71:321-7.\\n4. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain 2010;11:1274-81.\\n5. \\nStamer UM et al. Respiratory depression with tramadol in a patient with renal impairment and CYP2D6 gene duplication. Anesth Analg 2008;107:926-9.\\n6. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates and infants. Pediatr Res 2008;63:674-9.\\n7. \\nKirchheiner J et al. Effects of the CYP2D6 gene duplication on the pharmacokinetics and pharmacodynamics of tramadol. J Clin Psychopharmacol 2008;28:78-83.\\n8. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin Pharmacol Ther 2007;82:41-7.\\n9. \\nGleason PP et al. Debilitating reaction following the initial dose of tramadol. Ann Pharmacother 1997;31:1150-2.\\n10. \\nSPC Ultram (VS).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 96}),\n",
       " Document(page_content='Date 20-11-2017\\nCYP2D6 IM: venlafaxine\\n1539\\nThere are indications of an increased risk of side effects and a reduced chance of efficacy. The gene variation reduces the conversion of venlafaxine to the active metabolite O-\\ndesmethylvenlafaxine, whilst an association between high O-desmethylvenlafaxine/venlafaxine ratios and response without side effects was found.\\nIt is not possible to offer adequately substantiated advice for dose reduction based on the literature.\\navoid venlafaxine\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, duloxetine, mirtazapine, citalopram and sertraline.\\nif it is not possible to avoid venlafaxine and side effects occur:\\n1. \\nreduce the dose\\n2. \\nmonitor the effect and side effects or check the plasma concentrations of venlafaxine and O-desmethylvenlafaxine\\nIt is not known whether it is possible to reduce the dose to such an extent that the side effects disappear, while the effectiveness is maintained. In general, it is\\nassumed that the effectiveness is determined by the sum of the plasma concentrations of venlafaxine and O-desmethylvenlafaxine. However, the side effects do not\\nappear to be related to this sum.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nAustin-Zimmerman I et al. The influence of CYP2D6 and CYP2C19 genetic variation on diabetes mellitus risk in people taking antidepressants and antipsychotics. Genes\\n(Basel) 2021;12:1758.\\n3. \\nGanesh SV et al. Therapeutic drug monitoring of psychotropics as a diagnostic tool for CYP2D6 poor metabolizer phenotype. Ther Drug Monit 2021;43:672-80.\\n4. \\nKringen MK et al. The influence of combined CYP2D6 and CYP2C19 genotypes on venlafaxine and O-desmethylvenlafaxine concentrations in a large patient cohort. J Clin\\nPsychopharmacol 2020;40:137-44.\\n5. \\nLin XQ et al. The associations between CYP2D6 metabolizer status and pharmacokinetics and clinical outcomes of venlafaxine: a systematic review and meta-analysis.\\nPharmacopsychiatry 2019;52:222-31.\\n6. \\nWatanabe Y et al. Factors impacting the efficacy of venlafaxine extended release 75-225 mg/day in patients with major depressive disorder: exploratory post hoc\\nsubgroup analyses of a randomized, double-blind, placebo-controlled study in Japan. Neuropsychiatr Dis Treat 2018;14:1261-72.\\n7. \\nTaranu A et al. Should a routine genotyping of CYP2D6 and CYP2C19 genetic polymorphisms be recommended to predict venlafaxine efficacy in depressed patients\\ntreated in psychiatric settings? Pharmacogenomics 2017;18:639-50.\\n8. \\nBerm E et al. Relation between CYP2D6 genotype, phenotype and therapeutic drug concentrations among nortriptyline and venlafaxine users in old age psychiatry.\\nPharmacopsychiatry 2016;49:186-190 en persoonlijke communicatie (correctie: totaal aantal patiënten met sub- and supra-therapeutische plasmaconcentraties na 3\\nweken waren omgewisseld in tabel 3, en het totale aantal patiënten met TDM na 3 weken was 37).\\n9. \\nJiang F et al. The influences of CYP2D6 genotypes and drug interactions on the pharmacokinetics of venlafaxine: exploring predictive biomarkers for treatment outcomes.\\nPsychopharmacology (Berl) 2015;232:1899-909.\\n10. \\nWaade RB et al. Impact of age on serum concentrations of venlafaxine and escitalopram in different CYP2D6 and CYP2C19 genotype subgroups. Eur J Clin Pharmacol\\n2014;70:933-40.\\n11. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n12. \\nNg C et al. Pharmacogenetic polymorphisms and response to escitalopram and venlafaxine over 8 weeks in major depression. Hum Psychopharmacol 2013;28:516-22.\\n13. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6 polymorphisms. Int J Psychiatry Clin Pract\\n2009;13:345-8.\\n14. \\nHermann M et al. Serum concentrations of venlafaxine and its metabolites O-desmethylvenlafaxine and N-desmethylvenlafaxine in heterozygous carriers of the\\nCYP2D6*3, *4 or *5 allele. Eur J Clin Pharmacol 2008;64:483-7.\\n15. \\nMcAlpine DE et al. Cytochrome P450 2D6 genotype variation and venlafaxine dosage. Mayo Clin Proc 2007;82:1065-8.\\n16. \\nShams ME et al. CYP2D6 polymorphism and clinical effect of the antidepressant venlafaxine. J Clin Pharm Ther 2006;31:493-502.\\n17. \\nWhyte EM et al. CYP2D6 genotype and venlafaxine-XR concentrations in depressed elderly. Int J Geriatr Psychiatry 2006;21:542-9.\\n18. \\nFukuda T et al. The impact of the CYP2D6 and CYP2C19 genotypes on venlafaxine pharmacokinetics in a Japanese population. Eur J Clin Pharmacol 2000;56:175-80.\\n19. \\nVeefkind AH et al. Venlafaxine serum levels and CYP2D6 genotype. Ther Drug Monit 2000;22:202-8.\\n20. \\nFukuda T et al. Effect of the CYP2D6*10 genotype on venlafaxine pharmacokinetics in healthy adult volunteers. Br J Clin Pharmacol 1999;47:450-3.\\nDate 14-11-2022\\nCYP2D6 PM: venlafaxine\\n1538\\nThere are indications of an increased risk of side effects and a reduced chance of efficacy. The gene variation reduces the conversion of venlafaxine to the active metabolite O-\\ndesmethylvenlafaxine, whilst an association between high O-desmethylvenlafaxine/venlafaxine ratios and response without side effects was found.\\nIt is not possible to offer adequately substantiated advice for dose reduction based on the literature.\\navoid venlafaxine\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, duloxetine, mirtazapine, citalopram and sertraline.\\nIf it is not possible to avoid venlafaxine and side effects occur:\\n1. \\nreduce the dose\\n2. \\nmonitor the effect and side effects or check the plasma concentrations of venlafaxine and O-desmethylvenlafaxine\\nIt is not known whether it is possible to reduce the dose to such an extent that the side effects disappear, while the effectiveness is maintained. In general, it is\\nassumed that the effectiveness is determined by the sum of the plasma concentrations of venlafaxine and O-desmethylvenlafaxine. However, the side effects do not\\nappear to be related to this sum.\\nFurthermore, a reduced effectiveness of venlafaxine has been observed in depression patients with this gene variation.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nAustin-Zimmerman I et al. The influence of CYP2D6 and CYP2C19 genetic variation on diabetes mellitus risk in people taking antidepressants and antipsychotics. Genes\\n(Basel) 2021;12:1758.\\n3. \\nGanesh SV et al. Therapeutic drug monitoring of psychotropics as a diagnostic tool for CYP2D6 poor metabolizer phenotype. Ther Drug Monit 2021;43:672-80.\\n4. \\nKringen MK et al. The influence of combined CYP2D6 and CYP2C19 genotypes on venlafaxine and O-desmethylvenlafaxine concentrations in a large patient cohort. J Clin\\nPsychopharmacol 2020;40:137-44.\\n5. \\nLin XQ et al. The associations between CYP2D6 metabolizer status and pharmacokinetics and clinical outcomes of venlafaxine: a systematic review and meta-analysis.\\nPharmacopsychiatry 2019;52:222-31.\\n6. \\nAhmed AT et al. Pharmacokinetic-pharmacodynamic interaction associated with venlafaxine-XR remission in patients with major depressive disorder with history of\\ncitalopram / escitalopram treatment failure. J Affect Disord 2019;246:62-8.\\n7. \\nTaranu A et al. Should a routine genotyping of CYP2D6 and CYP2C19 genetic polymorphisms be recommended to predict venlafaxine efficacy in depressed patients\\ntreated in psychiatric settings? Pharmacogenomics 2017;18:639-50.\\n8. \\nBerm E et al. Relation between CYP2D6 genotype, phenotype and therapeutic drug concentrations among nortriptyline and venlafaxine users in old age psychiatry.\\nPharmacopsychiatry 2016;49:186-190 en persoonlijke communicatie (correctie: totaal aantal patiënten met sub- and supra-therapeutische plasmaconcentraties na 3\\nweken waren omgewisseld in tabel 3, en het totale aantal patiënten met TDM na 3 weken was 37).\\n9. \\nWaade RB et al. Impact of age on serum concentrations of venlafaxine and escitalopram in different CYP2D6 and CYP2C19 genotype subgroups. Eur J Clin Pharmacol\\n2014;70:933-40.\\n10. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n11. \\nNg C et al. Pharmacogenetic polymorphisms and response to escitalopram and venlafaxine over 8 weeks in major depression. Hum Psychopharmacol 2013;28:516-22.\\n12. \\nLobello KW et al. Cytochrome P450 2D6 phenotype predicts antidepressant efficacy of venlafaxine: a secondary analysis of 4 studies in major depressive disorder. J Clin\\nPsychiatry 2010;71:1482-7.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 97}),\n",
       " Document(page_content='13. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6 polymorphisms. Int J Psychiatry Clin Pract\\n2009;13:345-8.\\n14. \\nPreskorn S et al. Comparison of the pharmacokinetics of venlafaxine extended release and desvenlafaxine in extensive and poor cytochrome P450 2D6 metabolizers. J\\nClin Psychopharmacol 2009;29:39-43.\\n15. \\nHermann M et al. Serum concentrations of venlafaxine and its metabolites O-desmethylvenlafaxine and N-desmethylvenlafaxine in heterozygous carriers of the\\nCYP2D6*3, *4 or *5 allele. Eur J Clin Pharmacol 2008;64:483-7.\\n16. \\nShams ME et al. CYP2D6 polymorphism and clinical effect of the antidepressant venlafaxine. J Clin Pharm Ther 2006;31:493-502.\\n17. \\nWhyte EM et al. CYP2D6 genotype and venlafaxine-XR concentrations in depressed elderly. Int J Geriatr Psychiatry 2006;21:542-9.\\n18. \\nEap CB et al. Role of CYP2D6 in the stereoselective disposition of venlafaxine in humans. Pharmacogenetics 2003;13:39-47.\\n19. \\nLessard E et al. Influence of CYP2D6 activity on the disposition and cardiovascular toxicity of the antidepressant agent venlafaxine in humans. Pharmacogenetics\\n1999;9:435-43.\\n20. \\nVeefkind AH et al. Venlafaxine serum levels and CYP2D6 genotype. Ther Drug Monit 2000;22:202-8.\\nDate 14-11-2022\\nCYP2D6 UM: venlafaxine\\n1540\\nIt may be difficult to adjust the dose for patients due to altered metabolism between venlafaxine and the active metabolite O-desmethylvenlafaxine. The gene variation\\nincreases the conversion of venlafaxine to O-desmethylvenlafaxine and reduces the sum of venlafaxine plus O-desmethylvenlafaxine.\\n1. \\nbe alert to a possible decrease in the sum of the plasma concentrations of venlafaxine and the active metabolite O-desmethylvenlafaxine\\n2. \\nif necessary, increase the dose to 150% of the standard dose\\n3. \\nif dose adjustment does not result in efficacy without unacceptable side effects or if dose adjustment based on therapeutic drug monitoring is not possible, then\\nvenlafaxine should be avoided\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, duloxetine, mirtazapine, citalopram and sertraline.\\nLiterature:\\n1. \\nScherf-Clavel M et al. Effects of pharmacokinetic gene variation on therapeutic drug levels and anti-depressant treatment response. Pharmacopsychiatry 2022 Jul 15.\\nOnline ahead of print.\\n2. \\nGanesh SV et al. Therapeutic drug monitoring of psychotropics as a diagnostic tool for CYP2D6 poor metabolizer phenotype. Ther Drug Monit 2021;43:672-80.\\n3. \\nKringen MK et al. The influence of combined CYP2D6 and CYP2C19 genotypes on venlafaxine and O-desmethylvenlafaxine concentrations in a large patient cohort. J Clin\\nPsychopharmacol 2020;40:137-44.\\n4. \\nAhmed AT et al. Pharmacokinetic-pharmacodynamic interaction associated with venlafaxine-XR remission in patients with major depressive disorder with history of\\ncitalopram / escitalopram treatment failure. J Affect Disord 2019;246:62-8.\\n5. \\nTaranu A et al. Should a routine genotyping of CYP2D6 and CYP2C19 genetic polymorphisms be recommended to predict venlafaxine efficacy in depressed patients\\ntreated in psychiatric settings? Pharmacogenomics 2017;18:639-50.\\n6. \\nJiang F et al. The influences of CYP2D6 genotypes and drug interactions on the pharmacokinetics of venlafaxine: exploring predictive biomarkers for treatment outcomes.\\nPsychopharmacology (Berl) 2015;232:1899-909.\\n7. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder. Pharmacogenomics J 2014;14:176-81.\\n8. \\nShams ME et al. CYP2D6 polymorphism and clinical effect of the antidepressant venlafaxine. J Clin Pharm Ther 2006;31:493-502.\\n9. \\nVeefkind AH et al. Venlafaxine serum levels and CYP2D6 genotype. Ther Drug Monit 2000;22:202-8.\\nDate 14-11-2022\\nCYP2C19 IM: voriconazol\\n1683\\nThe gene variation can reduce the conversion of voriconazole and consequently increase the plasma concentration. This could result in improved efficacy or an increase in the\\nrisk of side effects.\\nmonitor the plasma concentration\\nLiterature:\\n1. \\nTian X et al. Impact of CYP2C19 phenotype and drug-drug interactions on voriconazole concentration in pediatric patients. Antimicrob Agents Chemother\\n2021;65:e0020721.\\n2. \\nZhao YC et al. Predictors of adverse events and determinants of the voriconazole trough concentration in kidney transplantation recipients. Clin Transl Sci 2021;14:702-\\n11.\\n3. \\nShang S et al. Effect of CYP2C19 polymorphism on the plasma voriconazole concentration and voriconazole-to-voriconazole-N-oxide concentration ratio in elderly\\npatients. Mycoses 2020 May 16 (online ahead of print).\\n4. \\nHicks JK et al. Prospective CYP2C19-guided voriconazole prophylaxis in patients with neutropenic acute myeloid leukemia reduces the incidence of subtherapeutic\\nantifungal plasma concentrations. Clin Pharmacol Ther 2020;107:563-70.\\n5. \\nPatel JN et al. Evaluation of CYP2C19 genotype-guided voriconazole prophylaxis after allogeneic hematopoietic cell transplant. Clin Pharmacol Ther 2020;107:571-9.\\n6. \\nSong Y et al. Association of CYP2C19 and UGT1A4 polymorphisms with voriconazole-induced liver injury. Per Med 2020;17:15-22.\\n7. \\nBlanco-Dorado S et al. Impact of CYP2C19 genotype and drug interactions on voriconazole plasma concentrations: a Spain pharmacogenetic-pharmacokinetic prospective\\nmulticenter study. Pharmacotherapy 2020;40:17-25.\\n8. \\nYamada T et al. Impact of flavin-containing monooxygenase 3 and CYP2C19 genotypes on plasma disposition and adverse effects of voriconazole administered orally in\\nimmunocompromised patients. J Infect Chemother 2019;25:1019-25.\\n9. \\nMafuru M et al. The influence of proinflammatory cytokines on voriconazole trough concentration in patients with different forms of hematologic disorders. J Clin\\nPharmacol 2019;59:1340-50.\\n10. \\nMiao Q et al. Correlation of CYP2C19 genotype with plasma voriconazole exposure in South-western Chinese Han patients with invasive fungal infections. Medicine\\n(Baltimore) 2019;98:e14137.\\n11. \\nSienkiewicz B et al. Influence of CYP2C19 genotypes on the occurrence of adverse drug reactions of voriconazole among hematological patients after allo-HSCT. Pathol\\nOncol Res 2018;24:541-5.\\n12. \\nHamadeh IS et al. Impact of the CYP2C19 genotype on voriconazole exposure in adults with invasive fungal infections. Pharmacogenet Genomics 2017;27:190-6.\\n13. \\nLi X et al. Effect of cytochrome P450 2C19 polymorphisms on the clinical outcomes of voriconazole: a systematic review and meta-analysis. Eur J Clin Pharmacol\\n2016;72:1185-93.\\n14. \\nWang Y et al. Risk factors for voriconazole-associated hepatotoxicity in patients in the intensive care unit. Pharmacotherapy 2016;36:757-65.\\n15. \\nChuwongwattana S et al. A prospective observational study of CYP2C19 polymorphisms and voriconazole plasma level in adult Thai patients with invasive aspergillosis.\\nDrug Metab Pharmacokinet 2016;31:117-22.\\n16. \\nTeusink A et al. Genotype-directed dosing leads to optimized voriconazole levels in pediatric patients receiving hematopoietic stem cell transplantation. Biol Blood\\nMarrow Transplant 2016;22:482-6.\\n17. \\nLamoureux F et al. Impact of CYP2C19 genetic polymorphisms on voriconazole dosing and exposure in adult patients with invasive fungal infections. Int J Antimicrob\\nAgents 2016;47:124-31.\\n18. \\nChawla PK et al. Correlation of CYP2C19 genotype with plasma voriconazole levels: a preliminary retrospective study in Indians. Int J Clin Pharm 2015;37:925-30.\\n19. \\nYamada T et al. Saturated metabolism of voriconazole N-oxidation resulting in nonlinearity of pharmacokinetics of voriconazole at clinical doses. Biol Pharm Bull\\n2015;38:1496-503.\\n20. \\nMori M et al. Pharmacokinetics and safety of voriconazole intravenous-to-oral switch regimens in immunocompromised Japanese pediatric patients. Antimicrob Agents\\nChemother 2015;59:1004-13.\\n21. \\nWang T et al. Efficacy and safety of voriconazole and CYP2C19 polymorphism for optimised dosage regimens in patients with invasive fungal infections. Int J Antimicrob\\nAgents 2014;44:436-42.\\n22. \\nLiu P et al. Population pharmacokinetic-pharmacodynamic analysis of voriconazole and anidulafungin in adult patients with invasive aspergillosis. Antimicrob Agents\\nChemother 2014;58:4727-36.\\n23. \\nZonios D et al. Voriconazole metabolism, toxicity, and the effect of cytochrome P450 2C19 genotype. J Infect Dis 2014;209:1941-8.\\n24. \\nHicks JK et al. Voriconazole plasma concentrations in immunocompromised pediatric patients vary by CYP2C19 diplotypes. Pharmacogenomics 2014;15:1065-78.\\n25. \\nKim SH et al. Clinical impact of cytochrome P450 2C19 genotype on the treatment of invasive aspergillosis under routine therapeutic drug monitoring of voriconazole in a', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 98}),\n",
       " Document(page_content='Korean population. Infect Chemother 2013;45:406-14.\\n26. \\nRacil Z et al. Monitoring trough voriconazole plasma concentrations in haematological patients: real life multicentre experience. Mycoses 2012;55:483-92.\\n27. \\nKim SH et al. Voriconazole-related severe adverse events: clinical application of therapeutic drug monitoring in Korean patients. Int J Infect Dis 2011;15:e753-8.\\n28. \\nBerge M et al. Effect of cytochrome P450 2C19 genotype on voriconazole exposure in cystic fibrosis lung transplant patients. Eur J Clin Pharmacol 2011;67:253-60.\\n29. \\nBrüggemann RJ et al. Pharmacokinetics and safety of 14 days intravenous voriconazole in allogeneic haematopoietic stem cell transplant recipients. J Antimicrob\\nChemother 2010;65:107-13.\\n30. \\nMatsumoto K et al. Correlation between voriconazole trough plasma concentration and hepatotoxicity in patients with different CYP2C19 genotypes. Int J Antimicrob\\nAgents 2009;34:91-4.\\n31. \\nKarlsson MO et al. Population pharmacokinetic analysis of voriconazole plasma concentration data from pediatric studies. Antimicrob Agents Chemother 2009;53:935-44.\\n32. \\nWeiss J et al. CYP2C19 genotype is a major factor contributing to the highly variable pharmacokinetics of voriconazole. J Clin Pharmacol 2009;49:196-204.\\n33. \\nLevin MD et al. Hepatotoxicity of oral and intravenous voriconazole in relation to cytochrome P450 polymorphisms. J Antimicrob Chemother 2007;60:1104-7.\\n34. \\nMikus G et al. Potent cytochrome P450 2C19 genotype-related interaction between voriconazole and the cytochrome P450 3A4 inhibitor ritonavir. Clin Pharmacol Ther\\n2006;80:126-35.\\n35. \\nRengelshausen J et al. Opposite effects of short-term and long-term St John’s wort intake on voriconazole pharmacokinetics. Clin Pharmacol Ther 2005;78:25-33.\\n36. \\nIkeda Y et al. Pharmacokinetics of voriconazole and cytochrome P450 2C19 genetic status. Clin Pharmacol Ther 2004;75:587-8.\\n37. \\nSPC Vfend (NL en VS).\\nDate 15-11-2021\\nCYP2C19 PM: voriconazol\\n1684\\nThe gene variation can reduce the conversion of voriconazole and consequently increase the plasma concentration. This could result in improved efficacy or an increase in the\\nrisk of side effects.\\nInitially, the risk of side effects is of particular interest.\\nuse 50% of the standard dose and monitor the plasma concentration\\nLiterature:\\n1. \\nTian X et al. Impact of CYP2C19 phenotype and drug-drug interactions on voriconazole concentration in pediatric patients. Antimicrob Agents Chemother\\n2021;65:e0020721.\\n2. \\nZhao YC et al. Predictors of adverse events and determinants of the voriconazole trough concentration in kidney transplantation recipients. Clin Transl Sci 2021;14:702-\\n11.\\n3. \\nShang S et al. Effect of CYP2C19 polymorphism on the plasma voriconazole concentration and voriconazole-to-voriconazole-N-oxide concentration ratio in elderly\\npatients. Mycoses 2020 May 16 (online ahead of print).\\n4. \\nYuan ZQ et al. The impact of plasma protein binding characteristics and unbound concentration of voriconazole on its adverse drug reactions. Front Pharmacol\\n2020;11:505.\\n5. \\nHicks JK et al. Prospective CYP2C19-guided voriconazole prophylaxis in patients with neutropenic acute myeloid leukemia reduces the incidence of subtherapeutic\\nantifungal plasma concentrations. Clin Pharmacol Ther 2020;107:563-70.\\n6. \\nPatel JN et al. Evaluation of CYP2C19 genotype-guided voriconazole prophylaxis after allogeneic hematopoietic cell transplant. Clin Pharmacol Ther 2020;107:571-9.\\n7. \\nSong Y et al. Association of CYP2C19 and UGT1A4 polymorphisms with voriconazole-induced liver injury. Per Med 2020;17:15-22.\\n8. \\nBlanco-Dorado S et al. Impact of CYP2C19 genotype and drug interactions on voriconazole plasma concentrations: a Spain pharmacogenetic-pharmacokinetic prospective\\nmulticenter study. Pharmacotherapy 2020;40:17-25.\\n9. \\nYamada T et al. Impact of flavin-containing monooxygenase 3 and CYP2C19 genotypes on plasma disposition and adverse effects of voriconazole administered orally in\\nimmunocompromised patients. J Infect Chemother 2019;25:1019-25.\\n10. \\nMafuru M et al. The influence of proinflammatory cytokines on voriconazole trough concentration in patients with different forms of hematologic disorders. J Clin\\nPharmacol 2019;59:1340-50.\\n11. \\nMiao Q et al. Correlation of CYP2C19 genotype with plasma voriconazole exposure in South-western Chinese Han patients with invasive fungal infections. Medicine\\n(Baltimore) 2019;98:e14137.\\n12. \\nHamadeh IS et al. Impact of the CYP2C19 genotype on voriconazole exposure in adults with invasive fungal infections. Pharmacogenet Genomics 2017;27:190-6.\\n13. \\nLi X et al. Effect of cytochrome P450 2C19 polymorphisms on the clinical outcomes of voriconazole: a systematic review and meta-analysis. Eur J Clin Pharmacol\\n2016;72:1185-93.\\n14. \\nWang Y et al. Risk factors for voriconazole-associated hepatotoxicity in patients in the intensive care unit. Pharmacotherapy 2016;36:757-65.\\n15. \\nChuwongwattana S et al. A prospective observational study of CYP2C19 polymorphisms and voriconazole plasma level in adult Thai patients with invasive aspergillosis.\\nDrug Metab Pharmacokinet 2016;31:117-22.\\n16. \\nTeusink A et al. Genotype-directed dosing leads to optimized voriconazole levels in pediatric patients receiving hematopoietic stem cell transplantation. Biol Blood\\nMarrow Transplant 2016;22:482-6.\\n17. \\nLamoureux F et al. Impact of CYP2C19 genetic polymorphisms on voriconazole dosing and exposure in adult patients with invasive fungal infections. Int J Antimicrob\\nAgents 2016;47:124-31.\\n18. \\nChawla PK et al. Correlation of CYP2C19 genotype with plasma voriconazole levels: a preliminary retrospective study in Indians. Int J Clin Pharm 2015;37:925-30.\\n19. \\nYamada T et al. Saturated metabolism of voriconazole N-oxidation resulting in nonlinearity of pharmacokinetics of voriconazole at clinical doses. Biol Pharm Bull\\n2015;38:1496-503.\\n20. \\nMori M et al. Pharmacokinetics and safety of voriconazole intravenous-to-oral switch regimens in immunocompromised Japanese pediatric patients. Antimicrob Agents\\nChemother 2015;59:1004-13.\\n21. \\nWang T et al. Efficacy and safety of voriconazole and CYP2C19 polymorphism for optimised dosage regimens in patients with invasive fungal infections. Int J Antimicrob\\nAgents 2014;44:436-42.\\n22. \\nLiu P et al. Population pharmacokinetic-pharmacodynamic analysis of voriconazole and anidulafungin in adult patients with invasive aspergillosis. Antimicrob Agents\\nChemother 2014;58:4727-36.\\n23. \\nZonios D et al. Voriconazole metabolism, toxicity, and the effect of cytochrome P450 2C19 genotype. J Infect Dis 2014;209:1941-8.\\n24. \\nHicks JK et al. Voriconazole plasma concentrations in immunocompromised pediatric patients vary by CYP2C19 diplotypes. Pharmacogenomics 2014;15:1065-78.\\n25. \\nKim SH et al. Clinical impact of cytochrome P450 2C19 genotype on the treatment of invasive aspergillosis under routine therapeutic drug monitoring of voriconazole in a\\nKorean population. Infect Chemother 2013;45:406-14.\\n26. \\nDriscoll TA et al. Comparison of pharmacokinetics and safety of voriconazole intravenous-to-oral switch in immunocompromised adolescents and healthy adults.\\nAntimicrob Agents Chemother 2011;55:5780-9.\\n27. \\nKim SH et al. Voriconazole-related severe adverse events: clinical application of therapeutic drug monitoring in Korean patients. Int J Infect Dis 2011;15:e753-8.\\n28. \\nLei HP et al. Lack of effect of Ginkgo biloba on voriconazole pharmacokinetics in Chinese volunteers identified as CYP2C19 poor and extensive metabolizers. Ann\\nPharmacother 2009;43:726-31.\\n29. \\nWang G et al. The CYP2C19 ultra-rapid metabolizer genotype influences the pharmacokinetics of voriconazole in healthy male volunteers. Eur J Clin Pharmacol\\n2009;65:281-5.\\n30. \\nWeiss J et al. CYP2C19 genotype is a major factor contributing to the highly variable pharmacokinetics of voriconazole. J Clin Pharmacol 2009;49:196-204.\\n31. \\nMikus G et al. Potent cytochrome P450 2C19 genotype-related interaction between voriconazole and the cytochrome P450 3A4 inhibitor ritonavir. Clin Pharmacol Ther\\n2006;80:126-35.\\n32. \\nRengelshausen J et al. Opposite effects of short-term and long-term St John\"s wort intake on voriconazole pharmacokinetics. Clin Pharmacol Ther 2005;78:25-33.\\n33. \\nIkeda Y et al. Pharmacokinetics of voriconazole and cytochrome P450 2C19 genetic status. Clin Pharmacol Ther 2004;75:587-8.\\n34. \\nSPC Vfend (NL en VS).\\nDate 15-11-2021\\nCYP2C19 UM: voriconazol\\n1685\\nThe gene variation increases the conversion of voriconazole, which increases the risk of ineffectiveness.\\nuse an initial dose that is 1.5x higher and monitor the plasma concentration\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 99}),\n",
       " Document(page_content='1. \\nPatel JN et al. Evaluation of CYP2C19 genotype-guided voriconazole prophylaxis after allogeneic hematopoietic cell transplant. Clin Pharmacol Ther 2020;107:571-9.\\n2. \\nBlanco-Dorado S et al. Impact of CYP2C19 genotype and drug interactions on voriconazole plasma concentrations: a Spain pharmacogenetic-pharmacokinetic prospective\\nmulticenter study. Pharmacotherapy 2020;40:17-25.\\n3. \\nSienkiewicz B et al. Influence of CYP2C19 genotypes on the occurrence of adverse drug reactions of voriconazole among hematological patients after allo-HSCT. Pathol\\nOncol Res 2018;24:541-5.\\n4. \\nHamadeh IS et al. Impact of the CYP2C19 genotype on voriconazole exposure in adults with invasive fungal infections. Pharmacogenet Genomics 2017;27:190-6.\\n5. \\nWilliams K et al. Association of CYP2C19 *17/*17 genotype with the risk of voriconazole-associated squamous cell carcinoma. JAMA Dermatol 2016;152:719-20.\\n6. \\nLamoureux F et al. Impact of CYP2C19 genetic polymorphisms on voriconazole dosing and exposure in adult patients with invasive fungal infections. Int J Antimicrob\\nAgents 2016;47:124-31.\\n7. \\nChawla PK et al. Correlation of CYP2C19 genotype with plasma voriconazole levels: a preliminary retrospective study in Indians. Int J Clin Pharm 2015;37:925-30.\\n8. \\nZonios D et al. Voriconazole metabolism, toxicity, and the effect of cytochrome P450 2C19 genotype. J Infect Dis 2014;209:1941-8.\\n9. \\nHicks JK et al. Voriconazole plasma concentrations in immunocompromised pediatric patients vary by CYP2C19 diplotypes. Pharmacogenomics 2014;15:1065-78.\\n10. \\nDriscoll TA et al. Comparison of pharmacokinetics and safety of voriconazole intravenous-to-oral switch in immunocompromised children and healthy adults. Antimicrob\\nAgents Chemother 2011;55:5770-9.\\n11. \\nDriscoll TA et al. Comparison of pharmacokinetics and safety of voriconazole intravenous-to-oral switch in immunocompromised adolescents and healthy adults.\\nAntimicrob Agents Chemother 2011;55:5780-9.\\n12. \\nBerge M et al. Effect of cytochrome P450 2C19 genotype on voriconazole exposure in cystic fibrosis lung transplant patients. Eur J Clin Pharmacol 2011;67:253-60.\\n13. \\nWang G et al. The CYP2C19 ultra-rapid metabolizer genotype influences the pharmacokinetics of voriconazole in healthy male volunteers. Eur J Clin Pharmacol\\n2009;65:281-5.\\n14. \\nWeiss J et al. CYP2C19 genotype is a major factor contributing to the highly variable pharmacokinetics of voriconazole. J Clin Pharmacol 2009;49:196-204.\\n15. \\nLi X et al. Effect of cytochrome P450 2C19 polymorphisms on the clinical outcomes of voriconazole: a systematic review and meta-analysis. Eur J Clin Pharmacol\\n2016;72:1185-93.\\nDate 15-11-2021\\nCYP2C9 IM ANDERS: warfarine\\n6233\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1. \\nuse 65% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm. Algorithms for Caucasian patients usually contain only the *2 and *3 allele. If\\nthe activity of the reduced-activity alleles is comparable to the activity of *2 or *3, then the algorithm can be completed as if *1/*2 or *1/*3 is present. See\\nhttps://www.knmp.nl/producten-en-diensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-pharmacogenetics for Excel files containing\\ncalculation modules for oral and equivalent intravenous doses. From day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nModified dose algorithms have been developed for patients of African or (East) Asian heritage.\\nLiterature:\\n1. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015; 39:228-34.\\n2. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014; 177:654-7.\\n3. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014; 12:1480-7.\\n4. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014; 174:1330-8.\\n5. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013; 168:4234-43.\\n6. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012; 7:e44064.\\n7. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 2009; 65:365-75.\\n8. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review and meta-analysis. Genet Med 2005;\\n7:97-104.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9 PM ANDERS: warfarine\\n6234\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1. \\nuse 20% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm.\\nAlgorithms for Caucasian patients usually contain only the *2 and *3 allele. If the activity of the reduced-activity alleles is comparable to the activity of *2 or *3, then the\\nalgorithm can be completed as if *2 or *3 is present. See https://www.knmp.nl/producten-en-diensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-\\ninformation-pharmacogenetics for Excel files containing calculation modules for oral and equivalent intravenous doses. From day 6 on the standard algorithm without\\ngenotype information can be used to calculate the dose.\\nModified dose algorithms have been developed for patients of African or (East) Asian heritage.\\nLiterature:\\n1. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015; 39:228-34.\\n2. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014; 177:654-7.\\n3. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014; 12:1480-7.\\n4. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014; 174:1330-8.\\n5. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013; 168:4234-43.\\n6. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012; 7:e44064.\\n7. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 2009; 65:365-75.\\n8. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review and meta-analysis. Genet Med 2005;\\n7:97-104.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*1/*2: warfarine\\n6228\\nNO action is required for this gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 100}),\n",
       " Document(page_content='Genetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes problems when therapy is initiated as\\nusual.\\nLiterature:\\n1. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015; 39:228-34.\\n2. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014; 177:654-7.\\n3. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014; 12:1480-7.\\n4. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014; 174:1330-8.\\n5. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013; 168:4234-43.\\n6. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012; 7:e44064.\\n7. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 2009; 65:365-75.\\n8. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review and meta-analysis. Genet Med 2005;\\n7:97-104.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*1/*3: warfarine\\n6229\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1. \\nuse 65% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see https://www.knmp.nl/producten-en-\\ndiensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-pharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015; 39:228-34.\\n2. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014; 177:654-7.\\n3. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014; 12:1480-7.\\n4. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014; 174:1330-8.\\n5. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013; 168:4234-43.\\n6. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012; 7:e44064.\\n7. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 2009; 65:365-75.\\n8. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review and meta-analysis. Genet Med 2005;\\n7:97-104.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*2/*2: warfarine\\n6230\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1. \\nuse 65% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see https://www.knmp.nl/producten-en-\\ndiensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-pharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015; 39:228-34.\\n2. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014; 177:654-7.\\n3. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014; 12:1480-7.\\n4. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014; 174:1330-8.\\n5. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013; 168:4234-43.\\n6. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012; 7:e44064.\\n7. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 2009; 65:365-75.\\n8. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review and meta-analysis. Genet Med 2005;\\n7:97-104.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*2/*3: warfarine\\n6231\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1. \\nuse 45% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see https://www.knmp.nl/producten-en-', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 101}),\n",
       " Document(page_content='diensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-pharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015; 39:228-34.\\n2. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014; 177:654-7.\\n3. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014; 12:1480-7.\\n4. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014; 174:1330-8.\\n5. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013; 168:4234-43.\\n6. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012; 7:e44064.\\n7. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 2009; 65:365-75.\\n8. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review and meta-analysis. Genet Med 2005;\\n7:97-104.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*3/*3: warfarine\\n6232\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1. \\nuse 20% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see https://www.knmp.nl/producten-en-\\ndiensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-pharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015; 39:228-34.\\n2. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014; 177:654-7.\\n3. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014; 12:1480-7.\\n4. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014; 174:1330-8.\\n5. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013; 168:4234-43.\\n6. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012; 7:e44064.\\n7. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 2009; 65:365-75.\\n8. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review and meta-analysis. Genet Med 2005;\\n7:97-104.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nVKORC1 -1639 AA: warfarine\\n6236\\nThe genetic variation results in increased sensitivity to warfarin. This results in an increase in the risk of excessively severe inhibition of blood clotting (INR > 4) during the\\nfirst month of the treatment.\\nRecommendation:\\n1. \\nuse 60% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see\\nhttps://www.knmp.nl/patientenzorg/medicatiebewaking/farmacogenetica.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1. \\nZhang J et al. The influence of VKORC1 gene polymorphism on warfarin maintenance dosage in pediatric patients: A systematic review and meta-analysis. Thromb Res\\n2015;136:955-61.\\n2. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015;39:228-34.\\n3. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmaco-genetic dosing of warfarin. Int J Cardiol 2014;177:654-7.\\n4. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014;12:1480-7.\\n5. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014;174:1330-8.\\n6. \\nJin B et al. The impact of VKORC1-1639G\\u2009>\\u2009A genetic polymorphism upon warfarin dose requirement in different ethnic populations. Curr Med Res Opin 2014;30:1505-11.\\n7. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013;168:4234-43.\\n8. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012;7:e44064.\\n9. \\nYang L et al. Impact of VKORC1 gene polymorphism on interindividual and interethnic warfarin dosage requirement--a systematic review and meta analysis. Thromb Res\\n2010;125:e159-66.\\n10. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nVKORC1 -1639 GA: warfarine\\n6235\\nNO action is required for this gene-drug interaction.\\nThe genetic variation results in a reduction in the required dose and an increase in the risk of excessively severe inhibition of blood clotting during the first month of the\\ntreatment. However, the effect is small and GA is also the most common genotype, meaning that the standard treatment will primarily be based on patients with this genotype.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 102}),\n",
       " Document(page_content='Literature:\\n1. \\nZhang J et al. The influence of VKORC1 gene polymorphism on warfarin maintenance dosage in pediatric patients: A systematic review and meta-analysis. Thromb Res\\n2015;136:955-61.\\n2. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical algorithm for warfarin dosing. J Thromb\\nThrombolysis 2015;39:228-34.\\n3. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;177:654-7.\\n4. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J\\nThromb Haemost 2014;12:1480-7.\\n5. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. JAMA Intern Med 2014;174:1330-8.\\n6. \\nJin B et al. The impact of VKORC1-1639G\\u2009>\\u2009A genetic polymorphism upon warfarin dose requirement in different ethnic populations. Curr Med Res Opin 2014;30:1505-11.\\n7. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients: a systematic review and meta-analysis.\\nInt J Cardiol 2013;168:4234-43.\\n8. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 2012;7:e44064.\\n9. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2D6 IM: zuclopentixol\\n1548\\nThe risk of side effects may be elevated. The genetic variation leads to decreased conversion of zuclopentixol, which causes the plasma concentration to be approximately\\n1.35-fold higher.\\nuse 75% of the standard dose\\nLiterature:\\n1. \\nWaade RB et al. Impact of CYP2D6 on serum concentrations of flupentixol, haloperidol, perphenazine and zuclopenthixol. Br J Clin Pharmacol 2021;87:2228-35.\\n2. \\nTveito M et al. Impact of age and CYP2D6 genotype on exposure of zuclopenthixol in patients using long-acting injectable versus oral formulation-an observational study\\nincluding 2044 patients. Eur J Clin Pharmacol 2021;77:215-21.\\n3. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n4. \\nVan Berlo-van de Laar et al. Dosering aanpassen aan eliminatiesnelheid. Pharm Weekblad 2004; 139:740-43.\\n5. \\nJaanson P et al. Maintenance therapy with zuclopentixol decanoate: associations between plasma concentrations, neurological side effects and CYP2D6 genotype.\\nPsychopharmacology 2002;162:67-73.\\n6. \\nJerling M et al. The CYP2D6 genotype predicts the oral clearance of the neuroleptic agents perphenazine and zuclopentixol. Clin Pharmacol Ther 1996;59:423-8.\\n7. \\nSmPC’s Cisordinol en Cisordinol Depot.\\nDate 13-09-2021\\nCYP2D6 PM: zuclopentixol\\n1547\\nThe risk of side effects may be elevated. The genetic variation results in a decreased conversion of zuclopentixol, which causes the plasma concentration to be approximately\\n1.6-fold higher.\\nuse 50% of the standard dose\\nLiterature:\\n1. \\nWaade RB et al. Impact of CYP2D6 on serum concentrations of flupentixol, haloperidol, perphenazine and zuclopenthixol. Br J Clin Pharmacol 2021;87:2228-35.\\n2. \\nTveito M et al. Impact of age and CYP2D6 genotype on exposure of zuclopenthixol in patients using long-acting injectable versus oral formulation-an observational study\\nincluding 2044 patients. Eur J Clin Pharmacol 2021;77:215-21.\\n3. \\nVan Berlo-van de Laar et al. Dosering aanpassen aan eliminatiesnelheid. Pharm Weekblad 2004; 139:740-43.\\n4. \\nJaanson P et al. Maintenance therapy with zuclopentixol decanoate: associations between plasma concentrations, neurological side effects and CYP2D6 genotype.\\nPsychopharmacology 2002;162:67-73.\\n5. \\nLinnet K et al. Influence of Cyp2D6 genetic polymorphism on ratios of steady-state serum concentration to dose of the neuroleptic zuclopentixol. Ther Drug Monit\\n1996;18:629-34.\\n6. \\nJerling M et al. The CYP2D6 genotype predicts the oral clearance of the neuroleptic agents perphenazine and zuclopentixol. Clin Pharmacol Ther 1996;59:423-8.\\n7. \\nDahl ML et al. Disposition of the neuroleptic zuclopentixol cosegregates with the polymorphic hydroxylation of debrisoquine in humans. Acta Psychiatr Scand 1991;84:99-\\n102.\\n8. \\nSmPC’s Cisordinol en Cisordinol Depot.\\nDate 13-09-2021\\nCYP2D6 UM: zuclopentixol\\n1549\\nThe risk of ineffectiveness may be elevated. The genetic variation leads to an increased conversion of zuclopentixol, which can result in a reduction of the plasma\\nconcentration.\\nThere is insufficient information available to make a dosage recommendation. - if the effectiveness is insufficient: try a dose increase Do not exceed 1.5 times the standard\\ndose\\nLiterature:\\n1. \\nTveito M et al. Impact of age and CYP2D6 genotype on exposure of zuclopenthixol in patients using long-acting injectable versus oral formulation-an observational study\\nincluding 2044 patients. Eur J Clin Pharmacol 2021;77:215-21.\\n2. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of aripiprazole, haloperidol, risperidone, paliperidone\\nand zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n3. \\nSmPC’s Cisordinol en Cisordinol Depot.\\nDate 13-09-2021', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/farmacogenetica_engels_20230501.pdf', 'page': 103}),\n",
       " Document(page_content=\"Table of Pharmacogenetic Associations\\nPharmacogenetic tests, along with other information about patients and their disease or condition, can\\nplay an important role in drug therapy . When a health care provider is considering prescribing a drug,\\nknowledge of a patient's genotype may be used to aid in determining a therapeutic strategy ,\\ndetermining an appropriate dosage, or assessing the likelihood of benefit or toxicity .\\nOn this page:\\nAbout the Table\\nSection 1: Pharmacogenetic Associations for which the Data Support Therapeutic Management\\nRecommendations\\nSection 2: Pharmacogenetic Associations for which the Data Indicate a Potential Impact on\\nSafety or Response\\nSection 3: Pharmacogenetic Associations for which the Data Demonstrate a Potential Impact on\\nPharmacokinetic Properties Only\\nUpdates to the Table\\nAdditional Resources\\nAbout the Table\\nFor the pharmacogenetic associations listed in this table, the FDA  has evaluated and believes there is\\nsufficient scientific evidence to suggest that subgroups of patients with certain genetic variants, or\\ngenetic variant-inferred phenotypes (such as af fected subgroup in the table below), are likely to have\\naltered drug metabolism, and in certain cases, dif ferential therapeutic ef fects, including dif ferences in\\nrisks of adverse events.\\nThe fact that the\\xa0FDA  has included a particular gene-drug interaction in the table does not necessarily\\nmean the FDA  advocates using a pharmacogenetic test before prescribing the corresponding\\nmedication, unless the test is a companion diagnostic. Tests that are essential for the safe and ef fective\\nuse of a therapeutic product, including those that identify patients for which the drug is\\ncontraindicated, are companion diagnostics.\\nThis table is not intended to af fect current regulatory requirements or policies, including the FDA's\\npolicy regarding companion diagnostics: Guidance for Industry and FDA  Staff: In Vitro Companion\\nDiagnostic Devices  (/regulatory-information/search-fda-guidance-documents/in-vitro-companion-\\ndiagnostic-devices) . Nor is the table intended to make an assessment on the safe and ef fective use of,\\nor regulatory requirements for , tests that detect variants in the referenced genes, or to provide\\ncomprehensive information on the described gene-drug interactions.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 1/14\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 0}),\n",
       " Document(page_content=\"Specific information regarding therapeutic management is provided for some pharmacogenetic\\nassociations listed in the table, but most of the associations listed have not been evaluated in terms of\\nthe impact of genetic testing on clinical outcomes, such as improved therapeutic ef fectiveness or\\nincreased risk of specific adverse events. In addition, clinical studies, if available, may only have\\nlinked genetic variation to a drug's pharmacokinetics (such as the way in which the drug is\\nmetabolized), and dif ferences in drug ef ficacy or safety across dif ferent genotype subgroups may not\\nbe known. If no statements related to ef ficacy or toxicity are provided, the scientific evidence the FDA\\nreviewed was considered insuf ficient to support such associations.\\nThe FDA  recognizes that practitioners will take into account dif ferent sources and strengths of\\nevidence and will make prescribing decisions based on their judgment about which treatments are\\nappropriate for individual patients. In particular , each patient's genetic makeup is only one of many\\nfactors that may impact drug concentrations and response, highlighting the fact that information\\nprovided in this table is limited to certain pharmacogenetic associations only and does not provide\\ncomprehensive information needed for safe and ef fective use of a drug. Accordingly , health care\\nproviders should refer to FDA-approved labeling for prescribing information, including monitoring\\ninstructions and information on other factors that may af fect drug concentrations, benefits, and risks.\\nIn this context, the information in this Table is intended primarily for prescribers, and patients should\\nnot adjust their medications without consulting their prescriber .\\nThis version of the table is limited to pharmacogenetic associations that are related to drug\\nmetabolizing enzyme gene variants, drug transporter gene variants, and gene variants that have been\\nrelated to a predisposition for certain adverse events. The FDA  recognizes that various other\\npharmacogenetic associations exist that are not listed here, and this table will be updated periodically\\nwith additional pharmacogenetic associations supported by suf ficient scientific evidence.\\nThe FDA  has opened a docket  (https://www .regulations.gov/docket?D=FDA-2020-N-0839)  for\\nstakeholders—including scientific and medical communities, patients, providers, and industry—to\\noffer specific comments on pharmacogenetic associations that the FDA  should or should not include in\\nthis table, along with the rationale and underlying evidence that supports the pharmacogenetic\\nassociation.\\nPlease submit all comments to the open docket  (https://www .regulations.gov/docket?D=FDA-2020-N-\\n0839) . Please submit any questions to PGx@fda.hhs.gov  (mailto:PGx@fda.hhs.gov) .\\nSection 1: Pharmacogenetic Associations for which the Data\\nSupport Therapeutic Management Recommendations\\nDrug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nAbaca vir HLA-B *57:01 allele\\npositiv eResults in higher adv erse r eaction risk (hypersensitivity\\nreactions). Do not use abaca vir in patients positiv e for HL A-\\nB*57:01.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 2/14\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 1}),\n",
       " Document(page_content='Drug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nAbrocitinib CYP2C19 poor\\nmetaboliz ersResults in higher systemic concentr ations and ma y result in\\nhigher adv erse r eaction risk. Dosage adjustment is\\nrecommended. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations.\\nAmifampridine NAT2 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk. Use lowest r ecommended star ting dosage and\\nmonit or for adv erse r eactions. Ref er to FD A labeling for speci\\x00c\\ndosing r ecommendations.\\nAmifampridine\\nPhosphateNAT2 poor\\nmetaboliz ersResults in higher systemic concentr ations. Use lowest\\nrecommended star ting dosage (15 mg/da y) and monit or for\\nadverse r eactions.\\nAmphetamine CYP2D6 poor\\nmetaboliz ersMay aff ect systemic concentr ations and adv erse r eaction risk.\\nConsider lower star ting dosage or use alternativ e agent.\\nAripipr azole CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk. Dosage adjustment is r ecommended. Ref er to FD A\\nlabeling for speci\\x00c dosing r ecommendations.\\nAripipr azole\\nLaur oxilCYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations. Dosage adjustment is\\nrecommended. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations.\\nAtomoxetine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk. Adjust titr ation inter val and incr ease dosage if\\ntolerated. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations.\\nAzathioprine TPM T\\nand/or\\nNUD T15intermediate or\\npoor\\nmetaboliz ersAlters systemic activ e metabolite concentr ation and dosage\\nrequir ements. Results in higher adv erse r eaction risk\\n(myelosuppr ession). Consider alternativ e ther apy in poor\\nmetaboliz ers. Dosage r eduction is r ecommended in intermediate\\nmetaboliz ers for NUD T15 or TPM T. Intermediate metaboliz ers for\\nboth genes ma y requir e mor e substantial dosage r eductions.\\nRefer to FD A labeling for speci\\x00c dosing r ecommendations.\\nBelinostat UGT1A1 *28/*28 (poor\\nmetaboliz ers)May result in higher systemic concentr ations and higher adv erse\\nreaction risk. Reduce star ting dose t o 750 mg/m2 in poor\\nmetaboliz ers.\\nBelzutifan CYP2C19\\nand/or\\nUGT2B17poor\\nmetaboliz ersResults in higher systemic concentr ations and ma y result in\\nhigher adv erse r eaction risk (anemia, hypo xia). Monit or patients\\nwho ar e poor metaboliz ers for both genes for adv erse r eactions.\\nBrexpipr azole CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations. Dosage adjustment is\\nrecommended. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations.\\nBrivaracetam CYP2C19 intermediate or\\npoor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk. Consider dosage r eductions in poor metaboliz ers.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 3/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 2}),\n",
       " Document(page_content='Drug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nCapecitabine DPYD intermediate or\\npoor\\nmetaboliz ersResults in higher adv erse r eaction risk (se vere, life-thr eatening, or\\nfatal t oxicities). No dosage has pr oven saf e in poor metaboliz ers,\\nand insu\\x00cient data ar e available t o recommend a dosage in\\nintermediate metaboliz ers. Withhold or discontinue in the\\npresence of early-onset or unusually se vere toxicity .\\nCarbamaz epine HLA-B *15:02 allele\\npositiv eResults in higher adv erse r eaction risk (se vere skin r eactions).\\nAvoid use unless potential bene\\x00ts outweigh risks and consider\\nrisks of alternativ e ther apies. P atients positiv e for HL A-B*15:02\\nmay be at incr eased risk of se vere skin r eactions with other drugs\\nthat ar e associated with a risk of Ste vens Johnson\\nSyndr ome/T oxic Epidermal necr olysis (SJS/TEN). Genotyping is\\nnot a substitute for clinical vigilance.\\nCeleco xib CYP2C9 poor\\nmetaboliz ers or\\n*3 carriersResults in higher systemic concentr ations. Reduce star ting dose\\nto half of the lowest r ecommended dose in poor metaboliz ers.\\nConsider alternativ e ther apy in poor metaboliz ers\\xa0with juv enile\\nrheumat oid ar thritis.\\nCitalopr am CYP2C19 poor\\nmetaboliz ersResults in higher systemic concentr ations and adv erse r eaction\\nrisk (Q T prolongation). The maximum r ecommended dose is 20\\nmg.\\nClobazam CYP2C19 intermediate or\\npoor\\nmetaboliz ersResults in higher systemic activ e metabolite concentr ations. P oor\\nmetabolism r esults in higher adv erse r eaction risk. Dosage\\nadjustment is r ecommended. Ref er to FD A labeling for speci\\x00c\\ndosing r ecommendations.\\nClopidogr el CYP2C19 intermediate or\\npoor\\nmetaboliz ersResults in lower systemic activ e metabolite concentr ations, lower\\nantiplatelet r esponse, and ma y result in higher car diovascular\\nrisk. Consider use of another platelet P2Y12 inhibit or.\\nClozapine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations. Dosage r eductions\\nmay be necessar y.\\nCodeine CYP2D6 ultrarapid\\nmetaboliz ersResults in higher systemic activ e metabolite concentr ations and\\nhigher adv erse r eaction risk (lif e-thr eatening r espir atory\\ndepr ession and death). Codeine is contr aindicated in childr en\\nunder 12 y ears of age.\\nDeutetr abenazine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and adv erse r eaction\\nrisk (Q T prolongation). The maximum r ecommended dosage\\nshould not ex ceed 36 mg (maximum single dose of 18 mg).\\nDronabinol CYP2C9 intermediate or\\npoor\\nmetaboliz ersMay result in higher systemic concentr ations and higher adv erse\\nreaction risk. Monit or for adv erse r eactions.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 4/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 3}),\n",
       " Document(page_content='Drug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nEliglustat CYP2D6 ultrarapid,\\nnormal,\\nintermediate, or\\npoor\\nmetaboliz ersAlters systemic concentr ations, eff ectiv eness, and adv erse\\nreaction risk (Q T prolongation). Indicated for normal,\\nintermediate, and poor metaboliz er patients. Ultr arapid\\nmetaboliz ers ma y not achie ve adequate concentr ations t o\\nachie ve a ther apeutic eff ect. The r ecommended dosages ar e\\nbased on CYP2D6 metaboliz er status. Coadministr ation with\\nstrong CYP3A inhibit ors is contr aindicated in intermediate and\\npoor CYP2D6 metaboliz ers. Ref er to FD A labeling for speci\\x00c\\ndosing r ecommendations.\\nErda\\x00tinib CYP2C9 *3/*3 (poor\\nmetaboliz ers)May result in higher systemic concentr ations and higher adv erse\\nreaction risk. Monit or for adv erse r eactions.\\nFlibanserin CYP2C19 poor\\nmetaboliz ersMay result in higher systemic concentr ations and higher adv erse\\nreaction risk. Monit or patients for adv erse r eactions.\\nFlurbipr ofen CYP2C9 poor\\nmetaboliz ers or\\n*3 carriersResults in higher systemic concentr ations. Use a r educed dosage\\nin poor metaboliz ers.\\nFluor ouracil DPYD intermediate or\\npoor\\nmetaboliz erResults in higher adv erse r eaction risk (se vere, life-thr eatening, or\\nfatal t oxicities). No dosage has pr oven saf e in poor metaboliz ers\\nand insu\\x00cient data ar e available t o recommend a dosage in\\nintermediate metaboliz ers. Withhold or discontinue in the\\npresence of early-onset or unusually se vere toxicity .\\nFosphenyt oin CYP2C9 intermediate or\\npoor\\nmetaboliz ersMay result in higher systemic concentr ations and higher adv erse\\nreaction risk (centr al ner vous system t oxicity). Consider star ting\\nat the lower end of the dosage r ange and monit or serum\\nconcentr ations. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations. Carriers of CYP2C9*3 alleles ma y be at\\nincreased risk of se vere cutaneous adv erse r eactions. Consider\\navoiding fosphenyt oin as an alternativ e to carbamaz epine in\\npatients who ar e CYP2C9*3 carriers. Genotyping is not a\\nsubstitute for clinical vigilance and patient management.\\nFosphenyt oin HLA-B *15:02 allele\\npositiv eMay result in higher adv erse r eaction risk (se vere cutaneous\\nreactions). P atients positiv e for HL A-B*15:02 ma y be at\\nincreased risk of Ste vens Johnson Syndr ome/T oxic Epidermal\\nnecrolysis (SJS/TEN). Consider a voiding fosphenyt oin as an\\nalternativ e to carbamaz epine in patients who ar e positiv e for\\nHLA-B*15:02. Genotyping is not a substitute for clinical vigilance\\nand patient management.\\nGe\\x00tinib CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk. Monit or for adv erse r eactions.\\nIloperidone CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk (Q T prolongation). Reduce dosage b y 50%.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 5/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 4}),\n",
       " Document(page_content='Drug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nIrinotecan UGT1A1 *1/*6, *1/*28\\n(intermediate\\nmetaboliz ers)\\nor\\xa0\\n*6/*6, *6/*28,\\n*28/*28 (poor\\nmetaboliz ers)Results in higher systemic activ e metabolite concentr ations and\\nhigher adv erse r eaction risk (se vere or lif e-thr eatening\\nneutr openia, se vere diarrhea). Closely monit or for neutr openia\\nduring and after tr eatment. Consider r educing the star ting\\ndosage b y at least one le vel in poor metaboliz ers and modify the\\ndosage based on individual patient t olerance. Ref er to FD A\\nlabeling for speci\\x00c dosing r ecommendations.\\nLofexidine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk. Monit or for or thostatic hypotension and\\nbradycardia.\\nMeclizine CYP2D6 ultrarapid,\\nintermediate, or\\npoor\\nmetaboliz ersMay aff ect systemic concentr ations. Monit or for adv erse\\nreactions and clinical eff ect.\\nMelo xicam CYP2C9 poor\\nmetaboliz ers or\\n*3 carriersResults in higher systemic concentr ations. Consider dose\\nreductions in poor metaboliz ers. Monit or patients for adv erse\\nreactions.\\nMetoclopr amide CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk. The r ecommended dosage is lower . Ref er to FD A\\nlabeling for speci\\x00c dosing r ecommendations.\\nMercaptopurine TPM T\\nand/or\\nNUD T15intermediate or\\npoor\\nmetaboliz ersAlters systemic activ e metabolite concentr ation and dosage\\nrequir ements. Results in higher adv erse r eaction risk\\n(myelosuppr ession). Initial dosages should be r educed in poor\\nmetaboliz ers; poor metaboliz ers gener ally t olerate 10% or less of\\nthe r ecommended dosage. Intermediate metaboliz ers ma y\\nrequir e dosage r eductions based on t olerability . Intermediate\\nmetaboliz ers for both genes ma y requir e mor e substantial\\ndosage r eductions. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations.\\nMivacurium BCHE intermediate or\\npoor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk (pr olonged neur omuscular blockade). A void use in\\npoor metaboliz ers.\\nNateglinide CYP2C9 poor\\nmetaboliz ersResults in higher systemic concentr ations and ma y result in\\nhigher adv erse r eaction risk (hypogly cemia). Dosage r eduction is\\nrecommended. Incr ease monit oring fr equency for adv erse\\nreactions. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations.\\nOliceridine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk (r espir atory depr ession and sedation). Ma y requir e\\nless fr equent dosing.\\nPantoprazole CYP2C19 intermediate or\\npoor\\nmetaboliz ersResults in higher systemic concentr ations. Consider dosage\\nreduction in childr en who ar e poor metaboliz ers. No dosage\\nadjustment is needed for adult patients who ar e intermediate or\\npoor metaboliz ers.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 6/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 5}),\n",
       " Document(page_content='Drug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nPhenyt oin CYP2C9 intermediate or\\npoor\\nmetaboliz ersMay result in higher systemic concentr ations and higher adv erse\\nreaction risk (centr al ner vous system t oxicity). Ref er to FD A\\nlabeling for speci\\x00c dosing r ecommendations. Carriers of\\nCYP2C9*3 alleles ma y be at incr eased risk of se vere cutaneous\\nadverse r eactions. Consider a voiding phenyt oin as an alternativ e\\nto carbamaz epine in patients who ar e CYP2C9*3 carriers.\\nGenotyping is not a substitute for clinical vigilance and patient\\nmanagement.\\nPhenyt oin HLA-B *15:02 allele\\npositiv eMay result in higher adv erse r eaction risk (se vere cutaneous\\nreactions). P atients positiv e for HL A-B*15:02 ma y be at\\nincreased risk of Ste vens Johnson Syndr ome/T oxic Epidermal\\nnecrolysis (SJS/TEN). Consider a voiding phenyt oin as an\\nalternativ e to carbamaz epine in patients who ar e positiv e for\\nHLA-B*15:02. Genotyping is not a substitute for clinical vigilance\\nand patient management.\\nPimo zide CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations. Dosages should not\\nexceed 0.05 mg/k g in childr en or 4 mg/da y in adults who ar e\\npoor metaboliz ers and dosages should not be incr eased earlier\\nthan 14 da ys.\\nPiroxicam CYP2C9 intermediate or\\npoor\\nmetaboliz ersResults in higher systemic concentr ations. Consider r educing\\ndosage in poor metaboliz ers.\\nPitolisant CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations. Use lowest\\nrecommended star ting dosage. Ref er to FD A labeling for speci\\x00c\\ndosing r ecommendations.\\nPropaf enone CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk (arrhythmia). A void use in poor metaboliz ers taking\\na CYP3A4 inhibit or.\\nSacituzumab\\nGovitecan-hziyUGT1A1 *28/*28 (poor\\nmetaboliz ers)May result in higher systemic concentr ations and adv erse\\nreaction risk (neutr openia). Monit or for adv erse r eactions and\\ntolerance t o treatment.\\nSiponimod CYP2C9 intermediate or\\npoor\\nmetaboliz ersResults in higher systemic concentr ations. Adjust dosage based\\non genotype. Do not use in patients with CYP2C9 *3/*3 genotype.\\nRefer to FD A labeling for speci\\x00c dosing r ecommendations.\\nSuccinylcholine BCHE intermediate or\\npoor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk (pr olonged neur omuscular blockade). A void use in\\npoor metaboliz ers. Ma y administer test dose t o assess\\nsensitivity and administer cautiously via slow infusion.\\nTacrolimus CYP3A5 intermediate or\\nnormal\\nmetaboliz ersResults in lower systemic concentr ations, lower pr obability of\\nachie ving tar get concentr ations and ma y result in higher r ejection\\nrisk. Measur e drug concentr ations and adjust dosage based on\\ntrough whole blood tacr olimus concentr ations.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 7/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 6}),\n",
       " Document(page_content='Drug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nTetrabenazine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations. The maximum\\nrecommended single dose is 25 mg and should not ex ceed 50\\nmg/da y.\\nThioguanine TPM T\\nand/or\\nNUD T15intermediate or\\npoor\\nmetaboliz ersAlters systemic activ e metabolite concentr ation and dosage\\nrequir ements. Results in higher adv erse r eaction risk\\n(myelosuppr ession). Initial dosages should be r educed in poor\\nmetaboliz ers; poor metaboliz ers gener ally t olerate 10% or less of\\nthe r ecommended dosage. Intermediate metaboliz ers ma y\\nrequir e dosage r eductions based on t olerability . Intermediate\\nmetaboliz ers for both genes ma y requir e mor e substantial\\ndosage r eductions. Ref er to FD A labeling for speci\\x00c dosing\\nrecommendations.\\nThioridazine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations and higher adv erse\\nreaction risk (Q T prolongation). Pr edicted eff ect based on\\nexperience with CYP2D6 inhibit ors. Contr aindicated in poor\\nmetaboliz ers.\\nTramadol CYP2D6 Ultrarapid\\nmetaboliz ersResults in higher systemic and br east milk activ e metabolite\\nconcentr ations, which ma y result in r espir atory depr ession and\\ndeath. Contr aindicated in childr en under 12 and in adolescents\\nfollowing t onsillect omy/adenoidect omy. Breastf eeding is not\\nrecommended during tr eatment.\\nValbenazine CYP2D6 poor\\nmetaboliz ersResults in higher systemic activ e metabolite concentr ations and\\nhigher adv erse r eaction risk (Q T prolongation). Dosage\\nreductions ma y be necessar y.\\nVenlafaxine CYP2D6 poor\\nmetaboliz ersAlters systemic par ent drug and metabolite concentr ations.\\nConsider dosage r eductions.\\nVortioxetine CYP2D6 poor\\nmetaboliz ersResults in higher systemic concentr ations. The maximum\\nrecommended dose is 10 mg.\\nWarfarin CYP2C9 intermediate or\\npoor\\nmetaboliz ersAlters systemic concentr ations and dosage r equir ements. Select\\ninitial dosage, taking int o account clinical and genetic fact ors.\\nMonit or and adjust dosages based on INR.\\nWarfarin CYP4F2 V433M v ariant\\ncarriersMay aff ect dosage r equir ements. Monit or and adjust doses\\nbased on INR.\\nWarfarin VKORC1 -1639G>A\\nvariant carriersAlters dosage r equir ements. Select initial dosage, taking int o\\naccount clinical and genetic fact ors. Monit or and adjust dosages\\nbased on INR.316/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 8/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 7}),\n",
       " Document(page_content='Section 2: Pharmacogenetic Associations for which the Data\\nIndicate a Potential Impact on Safety or Response\\nDrug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nAllopurinol HLA-B *58:01 allele\\npositiv eResults in higher adv erse r eaction risk (se vere skin\\nreactions).\\nCarbamaz epine HLA-A *31:01 allele\\npositiv eResults in higher adv erse r eaction risk (se vere skin\\nreactions). Consider risk and bene\\x00t of carbamaz epine use\\nin patients positiv e for HL A-A*31:01. Genotyping is not a\\nsubstitute for clinical vigilance.\\nCarvedilol CYP2D6 poor metaboliz ers Results in higher systemic concentr ations and higher\\nadverse r eaction risk (dizziness).\\nCevimeline CYP2D6 poor metaboliz ers May result in higher adv erse r eaction risk. Use with caution.\\nCodeine CYP2D6 poor metaboliz ers Results in lower systemic activ e metabolite concentr ations\\nand ma y result in r educed e\\x00cacy .\\nEfavirenz CYP2B6 poor metaboliz ers Results in higher systemic concentr ations and higher\\nadverse r eaction risk (Q T prolongation).\\nIsoniazid Nonspeci\\x00c\\n(NAT)poor metaboliz ers May result in higher systemic concentr ations and adv erse\\nreaction risk.\\nLapatinib HLA-DRB1 *07:01 allele\\npositiv eResults in higher adv erse r eaction risk (hepat otoxicity).\\nMonit or liv er function tests r egar dless of genotype.\\nLapatinib HLA-DQA1 *02:01 allele\\npositiv eResults in higher adv erse r eaction risk (hepat otoxicity).\\nMonit or liv er function tests r egar dless of genotype.\\nMavacamten CYP2C19 intermediate or\\npoor metaboliz ersResults in higher systemic concentr ations and ma y have\\nhigher adv erse r eaction risk (hear t failur e). Dosage is\\nbased on individual r esponse. The dose titr ation and\\nmonit oring schedule accounts for diff erences due t o\\nCYP2C19 genetic v ariation, so adjustments based on\\nCYP2C19 genotype ar e not necessar y. Ref er to FD A labeling\\nfor speci\\x00c dosing r ecommendations and monit oring.\\nNilotinib UGT1A1 *28/*28 (poor\\nmetaboliz ers)Results in higher adv erse r eaction risk (hyperbilirubinemia).\\nOxcarbaz epine HLA-B *15:02 allele\\npositiv eResults in higher adv erse r eaction risk (se vere skin\\nreactions). P atients positiv e for HL A-B*15:02 ma y be at\\nincreased risk of se vere skin r eactions with other drugs\\nthat ar e associated with a risk of Ste vens Johnson\\nSyndr ome/T oxic Epidermal necr olysis (SJS/TEN).\\nGenotyping is not a substitute for clinical vigilance.\\nPazopanib HLA-B *57:01 allele\\npositiv eMay result in higher adv erse r eaction risk (liv er enzyme\\nelevations). Monit or liv er function tests r egar dless of\\ngenotype.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 9/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 8}),\n",
       " Document(page_content='Drug GeneAffected\\nSubgr oups+ Description of Gene-Drug Inter action\\nPazopanib UGT1A1 *28/*28 (poor\\nmetaboliz ers)Results in higher adv erse r eaction risk (hyperbilirubinemia).\\nPerphenazine CYP2D6 poor metaboliz ers Results in higher systemic concentr ations and higher\\nadverse r eaction risk.\\nProcainamide Nonspeci\\x00c\\n(NAT)poor metaboliz ers Alters systemic par ent drug and metabolite concentr ations.\\nMay result in higher adv erse r eaction risk.\\nSimv astatin SLCO1B1 521 TC or 521 CC\\n(intermediate or\\npoor function\\ntranspor ters)Results in higher systemic concentr ations and higher\\nadverse r eaction risk (my opathy). The risk of adv erse\\nreaction (my opathy) is higher for patients on 80 mg than\\nfor those on lower doses.\\nSulfametho xazole\\nand TrimethoprimNonspeci\\x00c\\n(NAT)poor metaboliz ers May result in higher adv erse r eaction risk.\\nSulfasalazine Nonspeci\\x00c\\n(NAT)poor metaboliz ers Results in higher systemic metabolite concentr ations and\\nhigher adv erse r eaction risk.\\nTolter odine CYP2D6 poor metaboliz ers Results in higher systemic concentr ations and higher\\nadverse r eaction risk (Q T prolongation).\\nTramadol CYP2D6 poor metaboliz ers Results in lower systemic activ e metabolite concentr ations\\nand ma y result in r educed e\\x00cacy .\\nVoriconaz ole CYP2C19 intermediate or\\npoor metaboliz ersResults in higher systemic concentr ations and ma y result in\\nhigher adv erse r eaction risk.\\nSection 3: Pharmacogenetic Associations for which the Data\\nDemonstrate a Potential Impact on Pharmacokinetic\\nProperties Only\\nThe impact of these genetic variants or genetic variant inferred phenotypes on the safety or response\\nof the corresponding drug has not been established.\\nDrug Gene Affected Subgr oups+ Description of Gene-Drug Inter action\\nAmitriptyline CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nAmoxapine CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nAtorvastatin SLCO1B1 521 CC (poor function\\ntranspor ters)May result in higher systemic concentr ations.\\nAvatrombopag CYP2C9 intermediate or poor\\nmetaboliz ersResults in higher systemic concentr ations.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 10/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 9}),\n",
       " Document(page_content='Drug Gene Affected Subgr oups+ Description of Gene-Drug Inter action\\nCarisopr odol CYP2C19 poor metaboliz ers Results in higher systemic concentr ations. Use with\\ncaution.\\nClomipr amine CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nDarif enacin CYP2D6 poor metaboliz ers Results in higher systemic concentr ations.\\nDesipr amine CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nDexlansopr azole CYP2C19 intermediate or poor\\nmetaboliz ersResults in higher systemic concentr ations.\\nDiaz epam CYP2C19 poor metaboliz ers May aff ect systemic concentr ations.\\nDolutegr avir UGT1A1 poor metaboliz ers Results in higher systemic concentr ations.\\nDonepezil CYP2D6 ultrarapid or poor\\nmetaboliz ersAlters systemic concentr ations.\\nDoxepin CYP2C19 intermediate or poor\\nmetaboliz ersResults in higher systemic concentr ations.\\nDoxepin CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nElagolix SLCO1B1 521 CC (poor function\\ntranspor ters)Results in higher systemic concentr ations.\\nEscitalopr am CYP2C19 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nEsomepr azole CYP2C19 poor metaboliz ers Results in higher systemic concentr ations.\\nFesoter odine CYP2D6 poor metaboliz ers Results in higher systemic activ e metabolite\\nconcentr ations.\\nFluvoxamine CYP2D6 poor metaboliz ers Results in higher systemic concentr ations. Use with\\ncaution.\\nGalantamine CYP2D6 poor metaboliz ers Results in higher systemic concentr ations. Titrate\\ndosage based on t olerability .\\nHydralazine Nonspeci\\x00c\\n(NAT)poor metaboliz ers Results in higher systemic concentr ations.\\nIbupr ofen CYP2C9 poor metaboliz ers or\\n*3 carriersMay result in higher systemic concentr ations.\\nImipr amine CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 11/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 10}),\n",
       " Document(page_content='Drug Gene Affected Subgr oups+ Description of Gene-Drug Inter action\\nLansopr azole CYP2C19 poor metaboliz ers Results in higher systemic concentr ations.\\nMetoprolol CYP2D6 poor metaboliz ers Results in higher systemic concentr ations.\\nMirabegr on CYP2D6 poor metaboliz ers Results in higher systemic concentr ations.\\nNebiv olol CYP2D6 poor metaboliz ers May result in higher systemic concentr ations.\\nNortriptyline CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nOmepr azole CYP2C19 intermediate or poor\\nmetaboliz ersResults in higher systemic concentr ations.\\nParoxetine CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nPropranolol CYP2D6 poor metaboliz ers May aff ect systemic concentr ations.\\nProtriptyline CYP2D6 poor metaboliz ers Results in higher systemic concentr ations.\\nRabepr azole CYP2C19 poor metaboliz ers Results in higher systemic concentr ations.\\nRaltegr avir UGT1A1 *28/*28 (poor\\nmetaboliz ers)Results in higher systemic concentr ations.\\nRisperidone CYP2D6 poor metaboliz ers Alters systemic par ent drug and metabolite\\nconcentr ations.\\nRosuv astatin SLCO1B1 521 CC (poor function\\ntranspor ters)Results in higher systemic concentr ations.\\nTamoxifen CYP2D6 intermediate or poor\\nmetaboliz ersResults in lower systemic activ e metabolite\\nconcentr ations. The impact of CYP2D6 intermediate or\\npoor metabolism on e\\x00cacy is not well established.\\nTamsulosin CYP2D6 poor metaboliz ers Results in higher systemic concentr ations. Pr edicted\\neffect based on experience with CYP2D6 inhibit ors. Use\\nwith caution.\\nTrimipr amine CYP2D6 ultrarapid,\\nintermediate, or poor\\nmetaboliz ersMay alter systemic concentr ations.\\nViloxazine CYP2D6 poor metaboliz ers May result in higher systemic concentr ations.\\n+ The table describes gene-drug interactions and indicates specific af fected subgroup(s) to which the\\ninteraction applies. The af fected subgroup(s) may be carriers of a specific genetic variant (for\\nexample, HLA-B*15:02), or a genotype-inferred phenotype, ultrarapid, normal, intermediate, or poor\\nmetabolizers/function transporters of a drug metabolizing enzyme/drug transporter . Normal\\nmetabolizers or normal transporters do not have genetic variants that are expected to impact\\nmetabolism or transport function. In general, ultrarapid metabolizers have two or more copies of a\\ngenetic variant that increases metabolic function; intermediate metabolizers or reduced function\\ntransporters are individuals who have one or two copies of a genetic variant that reduces the ability to16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 12/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 11}),\n",
       " Document(page_content='metabolize or transport a drug; and poor metabolizers or poor function transporters are individuals\\nwho generally have two copies of a genetic variant that results in little to no ability to metabolize or\\ntransport a drug.\\nWhen evaluating if a gene-drug association is supported, the FDA  believes it is important to ensure\\nthat there is suf ficient scientific evidence to support:\\nThe relationship between the detected genetic variant and as appropriate, diplotypes, and the\\naffected subgroup, and\\nThe association between the af fected subgroup and the claimed gene-drug interaction.\\nIn some cases, a specific genetic variant may af fect the metabolism of dif ferent drugs in dif ferent\\nways. In cases where the association is limited to specific genetic variants and does not apply to all\\nindividuals with the genotype-inferred phenotype, the specific variants are provided in the table. In\\ncases where individual genetic variants are not listed in the table, the FDA  believes there is suf ficient\\nscientific evidence to generally support the described association for the genotype-inferred phenotype\\nsubgroup, provided specific genetic variants are determined to confer the genotype-inferred phenotype\\nbased on suf ficient scientific evidence.\\nFor example, when considering, as described in the table, that poor and intermediate metabolizers of\\nCYP2C19 have higher systemic active metabolite concentrations, higher adverse reaction risk, and\\ndosage adjustments are recommended when taking clobazam, suf ficient scientific evidence supports\\nthe following, with respect to the *2 allele:\\nThe functionality of the *2 variant is known, such as *2 variant results in a loss of CYP2C19\\nenzyme function, and\\n*1/*2 confers an intermediate metabolizer phenotype and *2/*2 confers a poor metabolizer\\nphenotype.\\nUpdates to the Table\\nOctober  26, 2022:  The FDA  made these changes:\\nSection 1:  Added abrocitinib and nateglinide. Updated the Affected Subgroups and\\nDescription of Gene-Drug Interaction for irinotecan.\\nSection 2:\\xa0 Added mavacamten.\\nMay 24, 2022: The FDA  made these changes:\\nSection 1:  Added belzutifan and updated the Affected Subgroups and Description of\\nGene-Drug Interaction for pantoprazole.\\nSection 2:  Added tramadol.\\nSection 3:  Added lansoprazole.\\nNovember  8, 2021: \\xa0 The FDA  made these additions:\\nSection 1: Added fosphenytoin and phenytoin.16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 13/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 12}),\n",
       " Document(page_content='Section 3:\\xa0 Added viloxazine.\\nMay 24, 2021:  The FDA  made these changes:\\nSection 1:  Added oliceridine and updated the Affected Subgroups and Description of\\nGene-Drug Interaction for celecoxib, flurbiprofen, and meloxicam.\\nSection 3: Added Ibuprofen.\\nMarch 18, 2021:  The FDA  made these changes:\\nSection 1:  Added meloxicam, pitolisant, and sacituzumab. Updated the Description of\\nGene-Drug Interaction for tacrolimus.\\nSection 2:  Moved voriconazole from Section 3 to Section 2 and updated the Description\\nof Gene-Drug Interaction.\\nSection 3:  Added atorvastatin and updated the Affected Subgroups and Description of\\nGene-Drug Interaction for escitalopram.\\nFebruary 25, 2020:  The FDA  corrected an error in the tramadol pharmacogenetic association\\nlisted in Section 1. The af fected subgroup was previously listed incorrectly as poor metabolizers\\nin the initial version.\\nAdditional Resources\\nFDA  Statement: FDA\\xa0Announces Collaborative Review of Scientific Evidence to Support\\nAssociations Between Genetic Information and Specific Medications  (/news-events/press-\\nannouncements/fda-announces-collaborative-review-scientific-evidence-support-associations-\\nbetween-genetic)\\n\\xa0\\nWas this helpful? Yes No16/05/2024, 00:13 Table of Pharmacogenetic Associations | FDA\\nhttps://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations 14/14', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/Table_of_Pharmacogenetic_Associations _ FDA.pdf', 'page': 13}),\n",
       " Document(page_content='HLA-B*5701: abacavir\\n2356\\nHLA-B*5701-positive patients have a strongly increased risk of a hypersensitivity reaction to abacavir.\\nRecommendation:\\nAbacavir is contra-indicated for HLA-B*5701-positive patients.\\n1\\n. \\nAdvise the prescriber to prescribe an alternative according to the current guidelines.\\nLiterature:\\n1\\n. \\nSaag M et al. High sensitivity of human leukocyte antigen-b*5701 as a marker for immunologically confirmed abacavir\\nhypersensitivity in white and black patients. Clin Infect Dis 2008;46:1111-8.\\n2\\n. \\nMallal S et al. HLA-B*5701 screening for hypersensitivity to abacavir. N Engl J Med 2008;358:568-79.\\n3\\n. \\nWaters LJ et al. Prospective HLA-B*5701 screening and abacavir hypersensitivity: a single centre experience. AIDS\\n2007;21:2533-4.\\n4\\n. \\nRodríguez-Nóvoa S et al. Value of the HLA-B*5701 allele to predict abacavir hypersensitivity in Spaniards. AIDS Res Hum\\nRetroviruses 2007;23:1374-6.\\n5\\n. \\nZucman D et al. Prospective screening for human leukocyte antigen-B*5701 avoids abacavir hypersensitivity reaction in the\\nethnically mixed French HIV population. J Acquir Immune Defic Syndr 2007;45:1-3.\\n6\\n. \\nRauch A et al. Prospective genetic screening decreases the incidence of abacavir hypersensitivity reactions in the Western\\nAustralian HIV cohort study. Clin Infect Dis 2006;43:99-102.\\n7\\n. \\nStekler J et al. Abacavir hypersensitivity reaction in primary HIV infection. AIDS 2006;20:1269-74.\\n8\\n. \\nPhillips EJ et al. Clinical and immunogenetic correlates of abacavir hypersensitivity. AIDS 2005;19:979-81.\\n9\\n. \\nHughes DA et al. Cost-effectiveness analysis of HLA B*5701 genotyping in preventing abacavir hypersensitivity.\\nPharmacogenetics 2004;14:335-42.\\n10\\n. \\nMartin AM et al. Predisposition to abacavir hypersensitivity conferred by HLA-B*5701 and a haplotypic Hsp70-Hom variant.\\nProc Natl Acad Sci U S A 2004;101:4180-5.\\n11\\n. \\nHughes AR et al. Association of genetic variations in HLA-B region with hypersensitivity to abacavir in some, but not all,\\npopulations. Pharmacogenomics 2004;5:203-11.\\n12\\n. \\nHetherington S et al. Genetic variations in HLA-B region and hypersensitivity reactions to abacavir. Lancet 2002;359:1121-2.\\n13\\n. \\nMallal S et al. Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-\\ntranscriptase inhibitor abacavir. Lancet 2002;359:727-32.\\n14\\n. \\nSPC Ziagen.\\nDate 11-12-2008\\nCYP2C9 IM: acenocoumarol\\n1868\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes\\nproblems when therapy is initiated as usual (i.e. with frequent INR monitoring).\\nLiterature:\\n1\\n. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical\\npractice in Hungary. Environ Toxicol Pharmacol 2017;56:282-289.\\n2\\n. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian\\npatients with mechanical heart valves. Clin Appl Thromb Hemost 2017;23: 876-882.\\n3\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n4\\n. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective\\nrandomised study. Thromb Haemost 2016;115:117-25.\\n5\\n. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian)\\npopulation. Eur J Clin Pharmacol 2015;71:173-81.\\n6\\n. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014;15:987-96.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 0}),\n",
       " Document(page_content='8\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n9\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n10\\n. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during\\nacenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n11\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n12\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n13\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n14\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n16\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n17\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n18\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.\\n19\\n. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the\\nanticoagulant response of acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n20\\n. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip\\nreplacement surgery. Mol Diagn Ther 2007;11:123-8.\\n21\\n. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n23\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n24\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n25\\n. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26\\n. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;1427-33.\\n28\\n. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther\\n2003;7461-8.\\n29\\n. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30\\n. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and\\nstability of anticoagulation. Haematologica 2002;87:1185-91.\\n31\\n. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol.\\nBlood 2002;99:4237-9.\\n32\\n. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol.\\nPharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9 PM: acenocoumarol\\n1869\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes\\nproblems when therapy is initiated as usual (i.e. with frequent INR monitoring).\\nLiterature:\\n1\\n. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical\\npractice in Hungary. Environ Toxicol Pharmacol 2017;56:282-289.\\n2\\n. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian\\npatients with mechanical heart valves. Clin Appl Thromb Hemost 2017;23: 876-882.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 1}),\n",
       " Document(page_content='3\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n4\\n. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective\\nrandomised study. Thromb Haemost 2016;115:117-25.\\n5\\n. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian)\\npopulation. Eur J Clin Pharmacol 2015;71:173-81.\\n6\\n. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014;15:987-96.\\n8\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n9\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.Gschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug\\ninteractions during acenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n10\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n11\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n12\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n13\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n14\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n15\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n16\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n17\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.\\n18\\n. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the\\nanticoagulant response of acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n19\\n. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip\\nreplacement surgery. Mol Diagn Ther 2007;11:123-8.\\n20\\n. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n21\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n22\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n23\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n24\\n. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n25\\n. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n26\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;1427-33.\\n27\\n. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther\\n2003;7461-8.\\n28\\n. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n29\\n. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and\\nstability of anticoagulation. Haematologica 2002;87:1185-91.\\n30\\n. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol.\\nBlood 2002;99:4237-9.\\n31\\n. \\nVerstuyft C et al. Early acenocoumarol overanticoagulation among cytochrome P450 2C9 poor metabolizers. Pharmacogenetics\\n2001;11:735-7.\\n32\\n. \\nThijssen HH et al. Altered pharmacokinetics of R- and S-acenocoumarol in a subject heterozygous for CYP2C9*3. Clin Pharmacol\\nTher 2001;70:292-8.\\n33\\n. \\nRettie AE et al. A case study of acenocoumarol sensitivity and genotype-phenotype discordancy explained by combinations of\\npolymorphisms in VKORC1 and CYP2C9. Br J Clin Pharmacol 2006;62:617-20.\\n34\\n. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol.\\nPharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*1/*2: acenocoumarol\\n1863', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 2}),\n",
       " Document(page_content='NO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes\\nproblems when therapy is initiated as usual (i.e. with frequent INR monitoring).\\nLiterature:\\n1\\n. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical\\npractice in Hungary. Environ Toxicol Pharmacol 2017;56:282-289.\\n2\\n. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian\\npatients with mechanical heart valves. Clin Appl Thromb Hemost 2017;23: 876-882.\\n3\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n4\\n. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective\\nrandomised study. Thromb Haemost 2016;115:117-25.\\n5\\n. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian)\\npopulation. Eur J Clin Pharmacol 2015;71:173-81.\\n6\\n. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014;15:987-96.\\n8\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n9\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n10\\n. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during\\nacenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n11\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n12\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n13\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n14\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n16\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n17\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n18\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.\\n19\\n. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the\\nanticoagulant response of acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n20\\n. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip\\nreplacement surgery. Mol Diagn Ther 2007;11:123-8.\\n21\\n. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n23\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n24\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n25\\n. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26\\n. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;1427-33.\\n28\\n. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther\\n2003;7461-8.\\n29\\n. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30\\n. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and\\nstability of anticoagulation. Haematologica 2002;87:1185-91.\\n31\\n. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol.\\nBlood 2002;99:4237-9.\\n32\\n. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 3}),\n",
       " Document(page_content='Pharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*1/*3: acenocoumarol\\n1864\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes\\nproblems when therapy is initiated as usual (i.e. with frequent INR monitoring).\\nLiterature:\\n1\\n. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical\\npractice in Hungary. Environ Toxicol Pharmacol 2017;56:282-289.\\n2\\n. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian\\npatients with mechanical heart valves. Clin Appl Thromb Hemost 2017;23: 876-882.\\n3\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n4\\n. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective\\nrandomised study. Thromb Haemost 2016;115:117-25.\\n5\\n. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian)\\npopulation. Eur J Clin Pharmacol 2015;71:173-81.\\n6\\n. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014;15:987-96.\\n8\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n9\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n10\\n. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during\\nacenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n11\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n12\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n13\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n14\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n16\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n17\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n18\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.\\n19\\n. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the\\nanticoagulant response of acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n20\\n. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip\\nreplacement surgery. Mol Diagn Ther 2007;11:123-8.\\n21\\n. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n23\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n24\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n25\\n. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26\\n. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 4}),\n",
       " Document(page_content='acenocoumarol or phenprocoumon. Pharmacogenetics 2004;1427-33.\\n28\\n. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther\\n2003;7461-8.\\n29\\n. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30\\n. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and\\nstability of anticoagulation. Haematologica 2002;87:1185-91.\\n31\\n. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol.\\nBlood 2002;99:4237-9.\\n32\\n. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol.\\nPharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*2/*2: acenocoumarol\\n1865\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes\\nproblems when therapy is initiated as usual (i.e. with frequent INR monitoring).\\nLiterature:\\n1\\n. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical\\npractice in Hungary. Environ Toxicol Pharmacol 2017;56:282-289.\\n2\\n. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian\\npatients with mechanical heart valves. Clin Appl Thromb Hemost 2017;23: 876-882.\\n3\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n4\\n. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective\\nrandomised study. Thromb Haemost 2016;115:117-25.\\n5\\n. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n6\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014;15:987-96.\\n7\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n8\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n9\\n. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during\\nacenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n10\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n11\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n12\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n13\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n14\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n15\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n16\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n17\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.\\n18\\n. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the\\nanticoagulant response of acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n19\\n. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip\\nreplacement surgery. Mol Diagn Ther 2007;11:123-8.\\n20\\n. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n21\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 5}),\n",
       " Document(page_content='22\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n23\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n24\\n. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n25\\n. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n26\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;1427-33.\\n27\\n. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther\\n2003;7461-8.\\n28\\n. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n29\\n. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and\\nstability of anticoagulation. Haematologica 2002;87:1185-91.\\n30\\n. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol.\\nBlood 2002;99:4237-9.\\nDate 14-05-2018\\nCYP2C9*2/*3: acenocoumarol\\n1866\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes\\nproblems when therapy is initiated as usual (i.e. with frequent INR monitoring).\\nLiterature:\\n1\\n. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical\\npractice in Hungary. Environ Toxicol Pharmacol 2017;56:282-289.\\n2\\n. \\nKalpana SR et al. Influence of VKORC1 and CYP2C9 polymorphisms on daily acenocoumarol dose requirement in South Indian\\npatients with mechanical heart valves. Clin Appl Thromb Hemost 2017;23: 876-882.\\n3\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n4\\n. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective\\nrandomised study. Thromb Haemost 2016;115:117-25.\\n5\\n. \\nKrishna Kumar D et al. An acenocoumarol dosing algorithm exploiting clinical and genetic factors in South Indian (Dravidian)\\npopulation. Eur J Clin Pharmacol 2015;71:173-81.\\n6\\n. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n7\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014;15:987-96.\\n8\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n9\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n10\\n. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during\\nacenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n11\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n12\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n13\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n14\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n15\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n16\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n17\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n18\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 6}),\n",
       " Document(page_content='19\\n. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the\\nanticoagulant response of acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n20\\n. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip\\nreplacement surgery. Mol Diagn Ther 2007;11:123-8.\\n21\\n. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n22\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n23\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n24\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n25\\n. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n26\\n. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n27\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;1427-33.\\n28\\n. \\nThijssen HH et al. Acenocoumarol pharmacokinetics in relation to cytochrome P450 2C9 genotype. Clin Pharmacol Ther\\n2003;7461-8.\\n29\\n. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n30\\n. \\nTassies D et al. Pharmacogenetics of acenocoumarol: cytochrome P450 CYP2C9 polymorphisms influence dose requirements and\\nstability of anticoagulation. Haematologica 2002;87:1185-91.\\n31\\n. \\nThijssen HH et al. The possession of the CYP2C9*3 allele is associated with low dose requirement of acenocoumarol.\\nPharmacogenetics 2000;10:757-60.\\nDate 14-05-2018\\nCYP2C9*3/*3: acenocoumarol\\n1867\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to a decrease in the required maintenance dose and possibly an increase in the time needed to reach a stable\\nINR. However, there is insufficient evidence that this causes problems when therapy is initiated as usual (i.e. with frequent INR\\nmonitoring).\\nLiterature:\\n1\\n. \\nVarnai R et al. CYP2C9 and VKORC1 in therapeutic dosing and safety of acenocoumarol treatment: implication for clinical\\npractice in Hungary. Environ Toxicol Pharmacol 2017;56:282-289.\\n2\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n3\\n. \\nCerezo-Manchado JJ et al. Genotype-guided therapy improves initial acenocoumarol dosing. Results from a prospective\\nrandomised study. Thromb Haemost 2016;115:117-25.\\n4\\n. \\nJiménez-Varo E et al. Pharmacogenetics role in the safety of acenocoumarol therapy. Thromb Haemost 2014;112:522-36.\\n5\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n6\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n7\\n. \\nGschwind L et al. Impact of CYP2C9 polymorphisms on the vulnerability to pharmacokinetic drug-drug interactions during\\nacenocoumarol treatment. Pharmacogenomics 2013:14;745-53.\\n8\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n9\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n10\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n11\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n12\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n13\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n14\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 7}),\n",
       " Document(page_content='15\\n. \\nGonzález-Conejero R et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the\\nanticoagulant response of acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n16\\n. \\nBeinema MJ et al. The influence of NSAIDs on coumarin sensitivity in patients with CYP2C9 polymorphism after total hip\\nreplacement surgery. Mol Diagn Ther 2007;11:123-8.\\n17\\n. \\nMark L et al. Cytochrome P450 2C9 polymorphism and acenocoumarol therapy. Kardiol Pol 2006;64:397-402.\\n18\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n19\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n20\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n21\\n. \\nMorin S et al. Pharmacogenetics of acenocoumarol pharmacodynamics. Clin Pharmacol Ther 2004;75:403-14.\\n22\\n. \\nSchalekamp T et al. Acenocoumarol stabilization is delayed in CYP2C9*3 carriers. Clin Pharmacol Ther 2004;75:394-402.\\n23\\n. \\nVerstuyft C et al. Genetic and environmental risk factors for oral anticoagulant overdose. Eur J Clin Pharmacol 2003;58:739-45.\\n24\\n. \\nHermida J et al. Differential effects of 2C9*3 and 2C9*2 variants of cytochrome P-450 CYP2C9 on sensitivity to acenocoumarol.\\nBlood 2002;99:4237-9.\\n25\\n. \\nVerstuyft C et al. Early acenocoumarol overanticoagulation among cytochrome P450 2C9 poor metabolizers. Pharmacogenetics\\n2001;11:735-7.\\nDate 14-05-2018\\nVKORC1 -1639 AA: acenocoumarol\\n1910\\nAn INR ≥ 6, resulting in an increased risk of bleeding, occurs in 8-12% of these patients during the first weeks of treatment with standard\\nregulation by the Anticoagulation Clinic. The genetic variation increases the sensitivity to acenocoumarol.\\nMonitoring by the ANTICOAGULATION CLINIC (National INR Monitoring Service):\\nrecommend to use 50% of the standard initial dose\\nOTHERWISE:\\nrecommend to use 50% of the standard initial dose\\nrecommend more frequent monitoring of the INR\\nThe initial dose and the maintenance dose can be calculated using an algorithm.\\nHowever, for patients with two or more VKORC1 and/or CYP2C9 variations, the algorithm used in EU-PACT (see\\nhttps://www.knmp.nl/patientenzorg/medicatiebewaking/farmacogenetica for a calculation tool in the form of an Excel file) did not result\\nin a significant reduction in the incidence of INRs above the target range when compared to an algorithm without genetic information.\\nWe are therefore unable to recommend the use of this algorithm at this time.\\nA (non-validated) algorithm has been prescribed for children that should result in a better prediction of the maintenance dose for AA\\nthan the current guideline used by the Anticoagulation Clinic (Maagdenberg H et al. The pediatric acenocoumarol dosing algorithm: The\\nChildren Anticoagulation and Pharmacogenetics Study. J Thromb Haemost 2018 Jun 23 [Epub ahead of print]. PubMed PMID:\\n29935043).\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014; 15: 987-96.\\n3\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. NEJM 2013;369: 2304-\\n12.\\n4\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n5\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n6\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n7\\n. \\nKovac MK et al. The c.-1639G>A polymorphism of the VKORC1 gene in Serbian population: retrospective study of the\\nvariability in response to oral anticoagulant therapy. Blood Coagul Fibrinolysis 2010;21:558-63.\\n8\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n9\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 8}),\n",
       " Document(page_content='10\\n. \\nStepien E et al. A vitamin K epoxide reductase-oxidase complex gene polymorphism (-1639G>A) and interindividual variability\\nin the dose-effect of vitamin K antagonists. J Appl Genet 2009;50:399-403.\\n11\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n12\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n13\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n14\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.\\n15\\n. \\nGonzalez-Conejero R et al. The genetic interaction of VKORC1 c1173t/calumenin a29809g modulates the anticoagulant response\\nof acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n16\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n17\\n. \\nMontes R et al. The c.-1639G > A polymorphism of the VKORC1 gene is a major determinant of the response to acenocoumarol\\nin anticoagulated patients. Br J Haematol 2006;133:183-7.\\n18\\n. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med\\n2005;2:e312.\\n19\\n. \\nBodin L et al. Cytochrome P450 2C9 (CYP2C9) and vitamin K epoxide reductase (VKORC1) genotypes as determinants of\\nacenocoumarol sensitivity. Blood 2005;106:135-140.\\nDate 10-09-2018\\nVKORC1 -1639 GA: acenocoumarol\\n1909\\nNO action is needed for this gene-drug interaction.\\nThe genetic variation results in a reduction of the required dose, but with the current practice of initiating or reviewing treatment this\\nresults in little or no increased risk of bleeding or excessive anticoagulation.\\nLiterature:\\n1\\n. \\nCerezo-Manchado JJ et al. Effect of VKORC1, CYP2C9 and CYP4F2 genetic variants in early outcomes during acenocoumarol\\ntreatment. Pharmacogenomics 2014; 15: 987-96.\\n2\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. NEJM 2013;369: 2304-\\n12.\\n3\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of the CYP2C9 and VKORC1 genotypes in acenocoumarol users. J Thromb\\nHaemost 2012;10:606-14.\\n4\\n. \\nEsmerian MO et al. Influence of CYP2C9 and VKORC1 polymorphisms on warfarin and acenocoumarol in a sample of Lebanese\\npeople. J Clin Pharmacol 2011;51:1418-28.\\n5\\n. \\nKovac MK et al. The c.-1639G>A polymorphism of the VKORC1 gene in Serbian population: retrospective study of the\\nvariability in response to oral anticoagulant therapy. Blood Coagul Fibrinolysis 2010;21:558-63.\\n6\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n7\\n. \\nWijnen PA et al. Variant VKORC1 and CYP2C9 alleles in patients with diffuse alveolar hemorrhage caused by oral\\nanticoagulants. Mol Diagn Ther 2010;14:23-30.\\n8\\n. \\nStepien E et al. A vitamin K epoxide reductase-oxidase complex gene polymorphism (-1639G>A) and interindividual variability\\nin the dose-effect of vitamin K antagonists. J Appl Genet 2009;50:399-403.\\n9\\n. \\nTeichert M et al. Genotypes associated with reduced activity of VKORC1 and CYP2C9 and their modification of acenocoumarol\\nanticoagulation during the initial treatment period. Clin Pharmacol Ther 2009;85:379-86.\\n10\\n. \\nMontes R et al. The influence of polymorphisms of VKORC1 and CYP2C9 on major gastrointestinal bleeding risk in\\nanticoagulated patients. Br J Haematol 2008;143:727-33.\\n11\\n. \\nMarkatos CN et al. VKORC1 and CYP2C9 allelic variants influence acenocoumarol dose requirements in Greek patients.\\nPharmacogenomics 2008;9:1631-8.\\n12\\n. \\nSpreafico M et al. Effects of CYP2C9 and VKORC1 on INR variations and dose requirements during initial phase of anticoagulant\\ntherapy. Pharmacogenomics 2008;9:1237-50.\\n13\\n. \\nGonzalez-Conejero R et al. The genetic interaction of VKORC1 c1173t/calumenin a29809g modulates the anticoagulant response\\nof acenocoumarol. J Thromb Haemost 2007;5:1701-6.\\n14\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and acenocoumarol anticoagulation status: interaction between both\\ngenotypes affects overanticoagulation. Clin Pharmacol Ther 2006;80:13-22.\\n15\\n. \\nMontes R et al. The c.-1639G > A polymorphism of the VKORC1 gene is a major determinant of the response to acenocoumarol\\nin anticoagulated patients. Br J Haematol 2006;133:183-7.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 9}),\n",
       " Document(page_content='16\\n. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med\\n2005;2:e312.\\n17\\n. \\nBodin L et al. Cytochrome P450 2C9 (CYP2C9) and vitamin K epoxide reductase (VKORC1) genotypes as determinants of\\nacenocoumarol sensitivity. Blood 2005;106:135-140.\\nDate 10-09-2018\\nCYP2D6 IM: amiodaron\\n2543\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 PM: amiodaron\\n2542\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 UM: amiodaron\\n2544\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 IM: amitriptyline\\n1920\\nThe genetic polymorphism leads to decreased metabolic capacity of CYP2D6, which may cause an increase in the plasma concentrations\\nof amitriptyline and its active metabolite nortriptyline and decreased plasma concentrations of the active metabolites E-10-OH-\\namitriptyline and E-10-OH-nortriptyline.\\nRecommendation:\\n1\\n. \\nChoose an alternative if possible\\nAntidepressants that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\n2\\n. \\nIf an alternative is not an option: use 60% of the standard dose and monitor the plasma concentrations of amitriptyline and\\nnortriptyline', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 10}),\n",
       " Document(page_content='As side effects are related to nortriptyline plasma concentrations and the efficacy to amitriptyline plus nortriptyline plasma\\nconcentrations, which are influenced to a lesser extent by CYP2D6, it is not known whether it is possible to reduce the dose to\\nsuch an extent that the side effects disappear, but the efficacy is maintained.\\nLiterature:\\n1\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2010 Jun 8 [Epub ahead of print]\\n2\\n. \\nKoski A et al. CYP2D6 and CYP2C19 genotypes and amitriptyline metabolite ratios in a series of medicolegal autopsies. Forensic\\nSci Int 2006;158:177-83.\\n3\\n. \\nSteimer W et al. Amitriptyline or not, that is the question: pharmacogenetic testing of CYP2D6 and CYP2C19 identifies patients\\nwith low or high risk for side effects in amitriptyline therapy. Clin Chem 2005;51:376-85.\\n4\\n. \\nSteimer W et al. Allele-specific change of concentration and functional gene dose for the prediction of steady-state serum\\nconcentrations of amitriptyline and nortriptyline in CYP2C19 and CYP2D6 extensive and intermediate metabolizers. Clin Chem\\n2004;50:1623-33.\\n5\\n. \\nShimoda K et al. The impact of CYP2C19 and CYP2D6 genotypes on metabolism of amitriptyline in Japanese psychiatric\\npatients. J Clin Psychopharmacol 2002;22:371-8.\\n6\\n. \\nBreyer-Pfaff U et al. Enantioselective amitriptyline metabolism in patients phenotyped for two cytochrome P450 isozymes. Clin\\nPharmacol Ther 1992;52:350-8.\\n7\\n. \\nMellstrom B et al. Amitriptyline metabolism: association with debrisoquin hydroxylation in nonsmokers. Clin Pharmacol Ther\\n1986;39:369-71.\\n8\\n. \\nBaumann P et al. Amitriptyline pharmacokinetics and clinical response: II. Metabolic polymorphism assessed by hydroxylation of\\ndebrisoquine and mephenytoin. Int Clin Psychopharmacol 1986;1:102-12.\\nDate 19-09-2007\\nCYP2D6 PM: amitriptyline\\n1921\\nThe genetic polymorphism leads to decreased metabolic capacity of CYP2D6, which may cause an increase in the plasma concentrations\\nof amitriptyline and its active metabolite nortriptyline and decreased plasma concentrations of the active metabolites E-10-OH-\\namitriptyline and E-10-OH-nortriptyline.\\nRecommendation:\\n1\\n. \\nChoose an alternative if possible\\nAntidepressants that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\n2\\n. \\nIf an alternative is not an option: use 50% of the standard dose and monitor the plasma concentrations of amitriptyline and\\nnortriptyline\\nAs side effects are related to nortriptyline plasma concentrations and the efficacy to amitriptyline plus nortriptyline plasma\\nconcentrations, which are influenced to a lesser extent by CYP2D6, it is not known whether it is possible to reduce the dose to\\nsuch an extent that the side effects disappear, but the efficacy is maintained.\\nLiterature:\\n1\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2010 Jun 8 [Epub ahead of print]\\n2\\n. \\nHalling J et al. The CYP2D6 polymorphism in relation to the metabolism of amitriptyline and nortriptyline in the Faroese\\npopulation. Br J Clin Pharmacol 2008;65:134-8.\\n3\\n. \\nKoski A et al. CYP2D6 and CYP2C19 genotypes and amitriptyline metabolite ratios in a series of medicolegal autopsies. Forensic\\nSci Int 2006;158:177-83.\\n4\\n. \\nSteimer W et al. Amitriptyline or not, that is the question: pharmacogenetic testing of CYP2D6 and CYP2C19 identifies patients\\nwith low or high risk for side effects in amitriptyline therapy. Clin Chem 2005;51:376-85.\\n5\\n. \\nBreyer-Pfaff U et al. Enantioselective amitriptyline metabolism in patients phenotyped for two cytochrome P450 isozymes. Clin\\nPharmacol Ther 1992;52:350-8.\\n6\\n. \\nMellstrom B et al. Amitriptyline metabolism: association with debrisoquin hydroxylation in nonsmokers. Clin Pharmacol Ther\\n1986;39:369-71.\\n7\\n. \\nBaumann P et al. Amitriptyline pharmacokinetics and clinical response: II. Metabolic polymorphism assessed by hydroxylation of\\ndebrisoquine and mephenytoin. Int Clin Psychopharmacol 1986;1:102-12.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 11}),\n",
       " Document(page_content='Date 19-09-2007\\nCYP2D6 UM: amitriptyline\\n1922\\nThe genetic polymorphism leads to increased metabolic capacity of CYP2D6, which may cause a decrease in the plasma concentrations\\nof amitriptyline and its active metabolite nortriptyline and increased plasma concentrations of the active metabolites E-10-OH-\\namitriptyline and E-10-OH-nortriptyline.\\nRecommendation:\\n1\\n. \\nChoose an alternative if possible\\nAntidepressants that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\n2\\n. \\nIf an alternative is not an option: increase the dose to 1.25 times the standard dose, monitor the plasma concentrations and be alert\\nto potential therapy failure due to decreased amitriptyline plus nortriptyline plasma concentrations and to increased plasma\\nconcentrations of the potentially cardiotoxic, active hydroxy metabolites.\\nLiterature:\\n1\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2010 Jun 8 [Epub ahead of print]\\n2\\n. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and\\nclinical effects of antidepressants in a naturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n3\\n. \\nBertilsson L et al. Extremely rapid hydroxylation of debrisoquine: a case report with implication for treatment with nortriptyline\\nand other tricyclic antidepressants. Ther Drug Monit 1985;7:478-80.\\n4\\n. \\nBaumann P et al. Amitriptyline pharmacokinetics and clinical response: II. Metabolic polymorphism assessed by hydroxylation of\\ndebrisoquine and mephenytoin. Int Clin Psychopharmacol 1986;1:102-12.\\nDate 19-09-2007\\nFact. V Leiden heterozyg: anticoncept. met oestr.\\n1567\\nThe heterozygously present genetic polymorphism “factor V Leiden” causes an increased tendency to coagulation, resulting in an\\nincreased risk of venous thromboembolism. Contraceptives containing oestrogens can increase this risk even further.\\nRecommendation:\\nIf the patient has a FAMILY HISTORY WITH A LOT OF THROMBOSIS, or has had a PREVIOUS THROMBOSIS:\\n1\\n. \\nAdvise the prescriber to avoid the use of contraceptives that contain oestrogens and prescribe a non-hormone contraceptive -\\nsuch as a copper IUD - as an alternative. One could also opt for a progestogen-only contraceptive method, such as the depot\\ninjection, an IUD with levonorgestrel or an implant with etonogestrel.\\nOTHER CASES:\\n1\\n. \\nAdvise the patient to avoid additional risk factors for thrombosis (obesity, smoking, etc.).\\nLiterature:\\n1\\n. \\nDulicek P et al. Venous thromboembolism in young female while on oral contraceptives: frequency of inherited thrombophilia and\\nanalysis of thrombosis in 400 Czech women. Clin Appl Thromb Hemost 2008 Dec 30 (Epub ahead of print).\\n2\\n. \\nCelikel S et al. Hereditary angioedema associated with heterozygous factor V Leiden mutation in a patient with Purpura fulminans.\\nInt Arch Allergy Immunol 2007;142:175-8.\\n3\\n. \\nCouturaud F et al. Incidence of venous thromboembolism in first-degree relatives of patients with venous thromboembolism who\\nhave factor V Leiden. Thromb Haemost 2006;96:744-9.\\n4\\n. \\nMartinelli I et al. Oral contraceptive use, thrombophilia and their interaction in young women with ischemic stroke. Haematologica\\n2006;91:844-7.\\n5\\n. \\nWu O et al. Screening for thrombophilia in high-risk situations: systematic review and cost-effectiveness analysis. The\\nThrombosis: Risk and Economic Assessment of Thrombophilia Screening (TREATS) study. Health Technol Assess 2006;10:1-\\n110.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 12}),\n",
       " Document(page_content='6\\n. \\nSchreijer AJ et al. Activation of coagulation system during air travel: a crossover study. Lancet 2006;367:832-8.\\n7\\n. \\nOsmanağaoğlu MA et al. Skin venous thromboembolism by combined oral contraceptive in a woman with acquired angioedema\\nand Factor V Leiden mutation. Contraception 2006;73:311-4.\\n8\\n. \\nSlooter AJ et al. Prothrombotic conditions, oral contraceptives, and the risk of ischemic stroke. J Thromb Haemost 2005;3:1213-7.\\n9\\n. \\nGirolami A et al. Long term use of oral contraceptives without thrombosis in patients with FV Leiden polymorphism: a study of 37\\npatients (2 homozygous and 35 heterozygous). J Thromb Thrombolysis 2004;17:145-9.\\n10\\n. \\nMartinelli I et al. Risk factors and recurrence rate of primary deep vein thrombosis of the upper extremities. Circulation\\n2004;110:566-70.\\n11\\n. \\nSidney S et al. Venous thromboembolic disease in users of low-estrogen combined estrogen-progestin oral contraceptives.\\nContraception 2004;70:3-10.\\n12\\n. \\nAment L. Factor V Leiden and contraception. J Midwifery Womens Health 2004;49:51-2.\\n13\\n. \\nKemmeren JM et al. Effect of second- and third-generation oral contraceptives on the protein C system in the absence or presence\\nof the factor V Leiden mutation: a randomized trial. Blood 2004;103:927-33.\\n14\\n. \\nKemmeren JM et al. Effect of second- and third-generation oral contraceptives on fibrinolysis in the absence or presence of the\\nfactor V Leiden mutation. Blood Coagul Fibrinolysis 2002;13:373-81.\\n15\\n. \\nKemmeren JM et al. Effects of second and third generation oral contraceptives and their respective progestagens on the\\ncoagulation system in the absence or presence of the factor V Leiden mutation. Thromb Haemost 2002;87:199-205.\\n16\\n. \\nLegnani C et al. Venous thromboembolism in young women: role of thrombophilic mutations and oral contraceptive use. Eur\\nHeart J 2002;23:984-90.\\n17\\n. \\nEmmerich J et al. Combined effect of factor V Leiden and prothrombin 20210A on the risk of venous thromboembolism - pooled\\nanalysis of 8 case-control studies including 2310 cases and 3204 controls. Thromb Haemost 2001;86:809-16.\\n18\\n. \\nBloemenkamp KW et al. Enhancement by factor V Leiden mutation of risk of deep-vein thrombosis associated with oral\\ncontraceptives containing a third-generation progestagen. Lancet 1995;346:1593-6.\\n19\\n. \\nVandenbroucke JP et al. Increased risk of venous thrombosis in oral-contraceptive users who are carriers of factor V Leiden\\nmutation. Lancet 1994;344:1453-7.\\n20\\n. \\nCBO Richtlijn Diagnostiek, Preventie en Behandeling van Veneuze Trombo-embolie en Secundaire Preventie Arteriële Trombose\\n2009. www.cbo.nl.\\nDate 08-06-2005\\nFact. V Leiden homozyg: anticoncept. met oestr.\\n1566\\nThe homozygously present genetic polymorphism “factor V Leiden” causes an increased tendency to coagulation, resulting in an\\nincreased risk of venous thromboembolism. Contraceptives containing oestrogens can increase this risk even further.\\nRecommendation:\\nIf the patient has a FAMILY HISTORY WITH A LOT OF THROMBOSIS, or has had a PREVIOUS THROMBOSIS:\\n1\\n. \\nAdvise the prescriber to avoid the use of contraceptives that contain oestrogens and prescribe a non-hormone contraceptive -\\nsuch as a copper IUD - as an alternative. One could also opt for a progestogen-only contraceptive method, such as the depot\\ninjection, an IUD with levonorgestrel or an implant with etonogestrel.\\nOTHER CASES:\\n1\\n. \\nAdvise the patient to avoid additional risk factors for thrombosis (obesity, smoking, etc.).\\nLiterature:\\n1\\n. \\nDulicek P et al. Venous thromboembolism in young female while on oral contraceptives: frequency of inherited thrombophilia and\\nanalysis of thrombosis in 400 Czech women. Clin Appl Thromb Hemost 2008 Dec 30 (Epub ahead of print).\\n2\\n. \\nCouturaud F et al. Incidence of venous thromboembolism in first-degree relatives of patients with venous thromboembolism who\\nhave factor V Leiden. Thromb Haemost 2006;96:744-9.\\n3\\n. \\nWu O et al. Screening for thrombophilia in high-risk situations: systematic review and cost-effectiveness analysis. The\\nThrombosis: Risk and Economic Assessment of Thrombophilia Screening (TREATS) study. Health Technol Assess 2006;10:1-\\n110.\\n4\\n. \\nSlooter AJ et al. Prothrombotic conditions, oral contraceptives, and the risk of ischemic stroke. J Thromb Haemost 2005;3:1213-7.\\n5\\n. \\nGirolami A et al. Long term use of oral contraceptives without thrombosis in patients with FV Leiden polymorphism: a study of 37\\npatients (2 homozygous and 35 heterozygous). J Thromb Thrombolysis 2004;17:145-9.\\n6\\n. \\nSidney S et al. Venous thromboembolic disease in users of low-estrogen combined estrogen-progestin oral contraceptives.\\nContraception 2004;70:3-10.\\n7\\n. \\nEhrenforth et al. Impact of environmental and hereditary risk factors on the clinical manifestation of thrombophilia in homozygous\\ncarriers of factor V:G1691A. J Thromb Haemost 2004;2:430-6.\\n8\\n. \\nLegnani C et al. Venous thromboembolism in young women: role of thrombophilic mutations and oral contraceptive use. Eur\\nHeart J 2002;23:984-90.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 13}),\n",
       " Document(page_content='9\\n. \\nEmmerich J et al. Combined effect of factor V Leiden and prothrombin 20210A on the risk of venous thromboembolism - pooled\\nanalysis of 8 case-control studies including 2310 cases and 3204 controls. Thromb Haemost 2001;86:809-16.\\n10\\n. \\nBloemenkamp KW et al. Enhancement by factor V Leiden mutation of risk of deep-vein thrombosis associated with oral\\ncontraceptives containing a third-generation progestagen. Lancet 1995;346:1593-6.\\n11\\n. \\nCBO Richtlijn Diagnostiek, Preventie en Behandeling van Veneuze Trombo-embolie en Secundaire Preventie Arteriële Trombose\\n2009. www.cbo.nl.\\nDate 08-06-2005\\nCYP2D6 IM: aripiprazol\\n1541\\nNO action is needed for this gene-drug interaction.\\nThe genetic variation increases the plasma concentration of the sum of aripiprazole and the active metabolite dehydroaripiprazole to a\\nlimited degree. There is insufficient evidence that this increases the risk of side effects.\\nLiterature:\\n1\\n. \\nBelmonte C et al. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 polymorphisms on pharmacokinetics and safety of\\naripiprazole in healthy volunteers. Basic Clin Pharmacol Toxicol 2018 Jan 11 [Epub ahead of print].\\n2\\n. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of\\naripiprazole, haloperidol, risperidone, paliperidone and zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n3\\n. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of\\naripiprazole, haloperidol, pimozide, and risperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\n4\\n. \\nSuzuki T et al. Effects of genetic polymorphisms of CYP2D6, CYP3A5, and ABCB1 on the steady-state plasma concentrations of\\naripiprazole and its active metabolite, dehydroaripiprazole, in Japanese patients with schizophrenia. Ther Drug Monit 2014;36:651-\\n5.\\n5\\n. \\nSuzuki T et al. Effects of the CYP-2D6*10 allele on the steady-state plasma concentrations of aripiprazole and its active\\nmetabolite, dehydroaripiprazole, in Japanese patients with schizophrenia. Ther Drug Monit 2011;33:21-4.\\n6\\n. \\nHendset M et al. Impact of the CYP2D6 genotype on steady-state serum concentrations of aripiprazole and dehydroaripiprazole.\\nEur J Clin Pharmacol 2007;63:1147-51.\\n7\\n. \\nKubo M et al. Pharmacokinetics of aripiprazole, a new antipsychotic, following oral dosing in healthy adult Japanese volunteers:\\ninfluence of CYP2D6 polymorphism. Drug Metab Pharmacokinet 2007;22:358-66.\\n8\\n. \\nKim E et al. Effects of DRD2 and CYP2D6 genotypes on delta EEG power response to aripiprazole in healthy male volunteers: a\\npreliminary study. Hum Psychopharmacol 2006;21:519-28.\\n9\\n. \\nKubo M et al. Influence of itraconazole co-administration and CYP2D6 genotype on the pharmacokinetics of the new\\nantipsychotic aripiprazole. Drug Metab Pharmacokinet 2005;20:55-64.\\nDate 14-05-2018\\nCYP2D6 PM: aripiprazol\\n1542\\nThe risk of side effects is increased. The genetic variation leads to an increase in the sum of the plasma concentrations of aripiprazole and\\nthe active metabolite.\\nadminister no more than 10 mg/day or 300 mg/month (67-75% of the standard maximum dose of aripiprazole).\\nLiterature:\\n1\\n. \\nBelmonte C et al. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 polymorphisms on pharmacokinetics and safety of\\naripiprazole in healthy volunteers. Basic Clin Pharmacol Toxicol 2018 Jan 11 [Epub ahead of print].\\n2\\n. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of\\naripiprazole, haloperidol, risperidone, paliperidone and zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n3\\n. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of\\naripiprazole, haloperidol, pimozide, and risperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\n4\\n. \\nHendset M et al. Impact of the CYP2D6 genotype on steady-state serum concentrations of aripiprazole and dehydroaripiprazole.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 14}),\n",
       " Document(page_content='Eur J Clin Pharmacol 2007;63:1147-51.\\n5\\n. \\nOosterhuis M et al. Safety of aripiprazole: high serum levels in a CYP2D6 mutated patient. Am J Psychiatry 2007;164:175.\\n6\\n. \\nSPC’s Abilify, Abilify Maintena, Abilify (USA), Aristad (USA).\\nDate 14-05-2018\\nCYP2D6 UM: aripiprazol\\n1543\\nNO action is needed for this gene-drug interaction.\\nThe genetic variation decreases the plasma concentration of the sum of aripiprazole and the active metabolite dehydroaripiprazole to a\\nlimited degree. There is no evidence that this increases the risk of reduced effectiveness.\\nLiterature:\\n1\\n. \\nBelmonte C et al. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 polymorphisms on pharmacokinetics and safety of\\naripiprazole in healthy volunteers. Basic Clin Pharmacol Toxicol 2018 Jan 11 [Epub ahead of print].\\n2\\n. \\nPatteet L et al. Genotype and co-medication dependent CYP2D6 metabolic activity: effects on serum concentrations of\\naripiprazole, haloperidol, risperidone, paliperidone and zuclopenthixol. Eur J Clin Pharmacol 2016;72:175-84.\\n3\\n. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of\\naripiprazole, haloperidol, pimozide, and risperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36.\\nDate 14-05-2018\\nCYP2D6 IM: atenolol\\n2454\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users.\\nClin Pharmacol Ther 2009;85:45-50.\\nDate 26-05-2009\\nCYP2D6 PM: atenolol\\n2453\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users.\\nClin Pharmacol Ther 2009;85:45-50.\\n2\\n. \\nLewis RV et al. Timolol and atenolol: relationships between oxidation phenotype, pharmacokinetics and pharmacodynamics. Br J\\nClin Pharmacol 1985;19:329-33.\\n3\\n. \\nDayer P et al. Interindividual variation of beta-adrenoceptor blocking drugs, plasma concentration and effect: influence of genetic\\nstatus on behaviour of atenolol, bopindolol and metoprolol. Eur J Clin Pharmacol 1985;28:149-53.\\n4\\n. \\nFreestone S et al. Comparison of two long-acting preparations of metoprolol with conventional metoprolol and atenolol in healthy\\nmen during chronic dosing. Br J Clin Pharmacol 1982;14:713-8.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 15}),\n",
       " Document(page_content='Date 26-05-2009\\nCYP2D6 UM: atenolol\\n2455\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 26-05-2009\\nCYP2D6 IM: atomoxetine\\n1599\\nThe genetic variation increases the plasma concentration of atomoxetine and can thereby reduce the dose requirement.\\nRecommendation:\\n1\\n. \\nin the event of side effects occurring and/or a response later than 9 weeks: reduce the dose and check whether the effect is\\nconserved\\nThe plasma concentration of atomoxetine is a factor of 2-3 times higher for IM than for EM at the same dose.\\nLiterature:\\n1\\n. \\nBrown JT et al. Single dose, CYP2D6 genotype-stratified pharmacokinetic study of atomoxetine in children with ADHD. Clin\\nPharmacol Ther 2016;99:642-50.\\n2\\n. \\nByeon JY et al. Effects of the CYP2D6*10 allele on the pharmacokinetics of atomoxetine and its metabolites. Arch Pharm Res\\n2015;38:2083-91.\\n3\\n. \\nFijal BA et al. CYP2D6 predicted metabolizer status and safety in adult patients with attention-deficit hyperactivity disorder\\nparticipating in a large placebo-controlled atomoxetine maintenance of response clinical trial. J Clin Pharmacol 2015;55:1167-74.\\n4\\n. \\nMatsui A et al. Pharmacokinetics, safety, and tolerability of atomoxetine and effect of CYP2D6\\n10/\\n10 genotype in healthy Japanese\\nmen. J Clin Pharmacol 2012;52:388-403.\\n5\\n. \\nter Laak MA et al. Recognition of impaired atomoxetine metabolism because of low CYP2D6 activity. Pediatr Neurol\\n2010;43:159-62.\\n6\\n. \\nCui YM et al. Atomoxetine pharmacokinetics in healthy Chinese subjects and effect of the CYP2D6*10 allele. Br J Clin\\nPharmacol 2007;64:445-9.\\nDate 31-10-2016\\nCYP2D6 PM: atomoxetine\\n1598\\nThe genetic variation increases the plasma concentration of atomoxetine and thereby the risk of side effects.\\nRecommendation:\\n1\\n. \\nstart with the normal initial dose, bearing in mind that an increase in this dose probably will not be required\\n2\\n. \\nadvise the patient to seek contact if side effects occur (such as decreased appetite, vomiting, abdominal pain, constipation,\\ninsomnia, early waking, drowsiness, irritability, pupil dilation and itching)\\n3\\n. \\nif the medicine is effective, but side effects occur: reduce the dose and check whether the effect is conserved\\nThe plasma concentration of atomoxetine is a factor of 8-11 times higher for PM than for EM at the same dose.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 16}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nBrown JT et al. Single dose, CYP2D6 genotype-stratified pharmacokinetic study of atomoxetine in children with ADHD. Clin\\nPharmacol Ther 2016;99:642-50.\\n2\\n. \\nFijal BA et al. CYP2D6 predicted metabolizer status and safety in adult patients with attention-deficit hyperactivity disorder\\nparticipating in a large placebo-controlled atomoxetine maintenance of response clinical trial. J Clin Pharmacol 2015;55:1167-74.\\n3\\n. \\nLoghin C et al. Effects of atomoxetine on the QT interval in healthy CYP2D6 poor metabolizers. Br J Clin Pharmacol\\n2013;75:538-49.\\n4\\n. \\nMatsui A et al. Pharmacokinetics, safety, and tolerability of atomoxetine and effect of CYP2D6\\n10/\\n10 genotype in healthy Japanese\\nmen. J Clin Pharmacol 2012;52:388-403.\\n5\\n. \\nRamoz N et al. A Haplotype of the Norepinephrine Transporter (Net) Gene Slc6a2 is Associated with Clinical Response to\\nAtomoxetine in Attention-Deficit Hyperactivity Disorder (ADHD). Neuropsychopharmacology 2009;34:2135-42.\\n6\\n. \\nTrzepacz PT et al. CYP2D6 metabolizer status and atomoxetine dosing in children and adolescents with ADHD. Eur\\nNeuropsychopharmacol 2008;18:79-86.\\n7\\n. \\nCui YM et al. Atomoxetine pharmacokinetics in healthy Chinese subjects and effect of the CYP2D6*10 Michelson D et al.\\nCYP2D6 and clinical response to atomoxetine in children and adolescents with ADHD. J Am Acad Child Adolesc Psychiatry\\n2007;46:242-51.\\n8\\n. \\nSauer JM et al. Disposition and metabolic fate of atomoxetine hydrochloride: the role of CYP2D6 in human disposition and\\nmetabolism. Drug Metab Dispos 2003;31:98-107.\\n9\\n. \\nSPC’s Strattera (NL en VS).\\n10\\n. \\nData on file, Lilly Research Laboratories, 2006. Atomoxetine - comparison of data of extensive metaboliser and poor metaboliser\\npatients.\\nDate 31-10-2016\\nCYP2D6 UM: atomoxetine\\n1600\\nThe genetic variation results in an increased conversion of atomoxetine to the active metabolite 4-hydroxyatomoxetine, which has a much\\nlower plasma concentration. As the plasma concentration of the active ingredients decreases as a result, this gene variation can result in\\nreduced efficacy.\\nRecommendation:\\n1\\n. \\nbe extra alert to reduced efficacy of the treatment\\n2\\n. \\nadvise the patient to contact their doctor in the event of inadequate effect\\n3\\n. \\nan alternative can be selected as a precaution\\nClonidine is not metabolised by CYP2D6.\\nLiterature:\\n1\\n. \\nBrown JT et al. Single dose, CYP2D6 genotype-stratified pharmacokinetic study of atomoxetine in children with ADHD. Clin\\nPharmacol Ther 2016;99:642-50.\\nDate 31-10-2016\\nSLCO1B1 521CC: atorvastatine\\n4058\\nThe genetic polymorphism may lead to reduced atorvastatin transport to the liver. This may increase atorvastatin plasma concentrations\\nand therefore the risk of myopathy.\\nRecommendation:\\nPatient has ADDITIONAL SIGNIFICANT RISK FACTORS for statin-induced myopathy:\\n1\\n. \\nChoose an alternative\\nRosuvastatin and pravastatin are influenced to a similar extent by SLCO1B1 polymorphisms but are not influenced by\\nCYP3A4 inhibitors such as amiodarone, verapamil and diltiazem.\\nFluvastatin is not influenced by SLCO1B1 polymorphisms or CYP3A4 inhibitors.\\n2\\n. \\nIf an alternative is not an option: advise the patient to contact their doctor in the event of muscle symptoms.\\nPatient has NO additional significant risk factors for statin-induced myopathy:\\n1\\n. \\nAdvise the patient to contact their doctor in the event of muscle symptoms.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 17}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and\\natorvastatin. Pharmacogenomics J 2012;12:233-7.\\n2\\n. \\nFrancesca Notarangelo M et al. Genetic predisposition to atorvastatin-induced myopathy: a case report. J Clin Pharm Ther 2012\\nMay 14. [Epub ahead of print]\\n3\\n. \\nSantos PC et al. SLCO1B1 haplotypes are not associated with atorvastatin-induced myalgia in Brazilian patients with familial\\nhypercholesterolemia. Eur J Clin Pharmacol 2012;68:273-9.\\n4\\n. \\nRodrigues AC et al. Pharmacogenetics of OATP transporters reveals that SLCO1B1 c.388A>G variant is determinant of increased\\natorvastatin response. Int J Mol Sci 2011;12:5815-27.\\n5\\n. \\nPuccetti L et al. Genetic involvement in statins induced myopathy. Preliminary data from an observational case-control study.\\nAtherosclerosis 2010;211:28-9.\\n6\\n. \\nLee YJ et al. Effects of SLCO1B1 and ABCB1 genotypes on the pharmacokinetics of atorvastatin and 2-hydroxyatorvastatin in\\nhealthy Korean subjects. Int J Clin Pharmacol Ther 2010;48:36-45.\\n7\\n. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n8\\n. \\nMega JL et al. Identification of genetic variants associated with response to statin therapy. Arterioscler Thromb Vasc Biol\\n2009;29:1310-5.\\n9\\n. \\nHe YJ et al. Rifampicin alters atorvastatin plasma concentration on the basis of SLCO1B1 521T>C polymorphism. Clin Chim\\nActa 2009;405:49-52.\\n10\\n. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with\\nincreased cholesterol synthesis rate. Pharmacogenet Genomics 2008;18:921-6.\\n11\\n. \\nPasanen MK et al. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin\\nPharmacol Ther 2007;82:726-33.\\n12\\n. \\nHermann M et al. Exposure of atorvastatin is unchanged but lactone and acid metabolites are increased several-fold in patients\\nwith atorvastatin-induced myopathy. Clin Pharmacol Ther 2006;79:532-9.\\n13\\n. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J\\n2005;5:352-8.\\nDate 27-11-2012\\nSLCO1B1 521TC: atorvastatine\\n4057\\nThe genetic polymorphism may lead to reduced atorvastatin transport to the liver. This may increase atorvastatin plasma concentrations\\nand therefore the risk of myopathy.\\nRecommendation:\\nPatient has ADDITIONAL SIGNIFICANT RISK FACTORS for statin-induced myopathy:\\n1\\n. \\nChoose an alternative\\nRosuvastatin and pravastatin are influenced to a similar extent by SLCO1B1 polymorphisms but are not influenced by\\nCYP3A4 inhibitors such as amiodarone, verapamil and diltiazem.\\nFluvastatin is not influenced by SLCO1B1 polymorphisms or CYP3A4 inhibitors.\\n2\\n. \\nIf an alternative is not an option: advise the patient to contact their doctor in the event of muscle symptoms.\\nPatient has NO additional significant risk factors for statin-induced myopathy:\\n1\\n. \\nAdvise the patient to contact their doctor in the event of muscle symptoms.\\nLiterature:\\n1\\n. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and\\natorvastatin. Pharmacogenomics J 2012;12:233-7.\\n2\\n. \\nFrancesca Notarangelo M et al. Genetic predisposition to atorvastatin-induced myopathy: a case report. J Clin Pharm Ther 2012\\nMay 14. [Epub ahead of print]\\n3\\n. \\nSantos PC et al. SLCO1B1 haplotypes are not associated with atorvastatin-induced myalgia in Brazilian patients with familial\\nhypercholesterolemia. Eur J Clin Pharmacol 2012;68:273-9.\\n4\\n. \\nRodrigues AC et al. Pharmacogenetics of OATP transporters reveals that SLCO1B1 c.388A>G variant is determinant of increased\\natorvastatin response. Int J Mol Sci 2011;12:5815-27.\\n5\\n. \\nPuccetti L et al. Genetic involvement in statins induced myopathy. Preliminary data from an observational case-control study.\\nAtherosclerosis 2010;211:28-9.\\n6\\n. \\nLee YJ et al. Effects of SLCO1B1 and ABCB1 genotypes on the pharmacokinetics of atorvastatin and 2-hydroxyatorvastatin in', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 18}),\n",
       " Document(page_content=\"healthy Korean subjects. Int J Clin Pharmacol Ther 2010;48:36-45.\\n7\\n. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n8\\n. \\nMega JL et al. Identification of genetic variants associated with response to statin therapy. Arterioscler Thromb Vasc Biol\\n2009;29:1310-5.\\n9\\n. \\nHe YJ et al. Rifampicin alters atorvastatin plasma concentration on the basis of SLCO1B1 521T>C polymorphism. Clin Chim\\nActa 2009;405:49-52.\\n10\\n. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with\\nincreased cholesterol synthesis rate. Pharmacogenet Genomics 2008;18:921-6.\\n11\\n. \\nPasanen MK et al. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin\\nPharmacol Ther 2007;82:726-33.\\n12\\n. \\nHermann M et al. Exposure of atorvastatin is unchanged but lactone and acid metabolites are increased several-fold in patients\\nwith atorvastatin-induced myopathy. Clin Pharmacol Ther 2006;79:532-9.\\n13\\n. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J\\n2005;5:352-8.\\nDate 27-11-2012\\nTPMT IM: azathioprine/mercaptopurine\\n1905\\nThe genetic variation reduces the conversion of azathioprine and mercaptopurine to mainly inactive metabolites. This increases the risk\\nof serious, life-threatening adverse events such as myelosuppression.\\nRecommendation:\\n1\\n. \\nStart with 50% of the standard dose\\nAdjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and effectiveness.\\nThe frequency of monitoring should be increased.\\nDose adjustment is not required for doses lower than 1.5 mg/kg per day for azathioprine or 0.75 mg/kg per day for mercaptopurine.\\nLiterature:\\n1\\n. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of\\nthiopurine methyltransferase pharmacogenetics. Br J Haematol 2014 Nov 29 [Epub ahead of print].\\n2\\n. \\nKim MJ et al. Monitoring thiopurine metabolites in korean pediatric patients with inflammatory bowel disease. Yonsei Med J\\n2014;55:1289-96.\\n3\\n. \\nLevinsen M et al. Pharmacogenetically based dosing of thiopurines in childhood acute lymphoblastic leukemia: influence on cure\\nrates and risk of second cancer. Pediatr Blood Cancer 2014;61:797-802.\\n4\\n. \\nBooth RA et al. Assessment of thiopurine S-methyltransferase activity in patients prescribed thiopurines: a systematic review. Ann\\nIntern Med 2011;154:814-23, W-295-8.\\n5\\n. \\nDong XW et al. Thiopurine S-methyltransferase polymorphisms and thiopurine toxicity in treatment of inflammatory bowel\\ndisease. World J Gastroenterol 2010;16:3187-95.\\n6\\n. \\nHindorf U et al. Characterisation and utility of thiopurine methyltransferase and thiopurine metabolite measurements in\\nautoimmune hepatitis. J Hepatol 2010;52:106-11.\\n7\\n. \\nSheffield LJ et al. Thiopurine methyltransferase and thiopurine metabolite testing in patients with inflammatory bowel disease\\nwho are taking thiopurine drugs. Pharmacogenomics 2009;10:1091-9.\\n8\\n. \\nAnsari A et al. Influence of xanthine oxidase on thiopurine metabolism in Crohn's disease. Aliment Pharmacol Ther 2008;28:749-\\n57.\\n9\\n. \\nGardiner SJ et al. Thiopurine dose in intermediate and normal metabolizers of thiopurine methyltransferase may differ three-fold.\\nClin Gastroenterol Hepatol 2008;6:654-60.\\n10\\n. \\nMoloney FJ et al. The frequency and significance of thiopurine S-methyltransferase gene polymorphisms in azathioprine-treated\\nrenal transplant recipients. Br J Dermatol 2006;154:1199-200.\\n11\\n. \\nZelinkova Z et al. Inosine triphosphate pyrophosphatase and thiopurine s-methyltransferase genotypes relationship to azathioprine-\\ninduced myelosuppression. Clin Gastroenterol Hepatol 2006;4:44-9.\\n12\\n. \\nJun JB et al. Thiopurine S-methyltransferase polymorphisms and the relationship between the mutant alleles and the adverse\\neffects in systemic lupus erythematosus patients taking azathioprine. Clin Exp Rheumatol 2005;23:873-6.\\n13\\n. \\nStocco G et al. TPMT genotype and the use of thiopurines in paediatric inflammatory bowel disease. Dig Liver Dis 2005;37:940-\\n5.\\n14\\n. \\nKurzawski M et al. The impact of thiopurine s-methyltransferase polymorphism on azathioprine-induced myelotoxicity in renal\\ntransplant recipients. Ther Drug Monit 2005;27:435-41.\\n15\\n. \\nFabre MA et al. The impact of thiopurine S-methyltransferase polymorphisms on azathioprine dose 1 year after renal\\ntransplantation. Transpl Int 2004;17:531-9.\\n16\\n. \\nGearry RB et al. Thiopurine S-methyltransferase (TPMT) genotype does not predict adverse drug reactions to thiopurine drugs in\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 19}),\n",
       " Document(page_content='patients with inflammatory bowel disease. Aliment Pharmacol Ther 2003;18:395-400.\\n17\\n. \\nGilissen LP et al. Some cases demonstrating the clinical usefulness of therapeutic drug monitoring in thiopurine-treated\\ninflammatory bowel disease patients. Eur J Gastroenterol Hepatol 2004;16:705-10.\\n18\\n. \\nAnsari A et al. Thiopurine methyltransferase activity and the use of azathioprine in inflammatory bowel disease. Aliment\\nPharmacol Ther 2002;16:1743-50.\\n19\\n. \\nLangley PG et al. Thiopurine methyltransferase phenotype and genotype in relation to azathioprine therapy in autoimmune\\nhepatitis. J Hepatol 2002;37:441-7.\\n20\\n. \\nRegueiro M et al. Determination of thiopurine methyltransferase genotype or phenotype optimizes initial dosing of azathioprine\\nfor the treatment of Crohn’s disease. J Clin Gastroenterol 2002;35:240-4.\\n21\\n. \\nPandya B et al. Azathioprine toxicity and thiopurine methyltransferase genotype in renal transplant patients. Transplant Proc\\n2002;34:1642-5.\\n22\\n. \\nCampbell S et al. Relevance of thiopurine methyltransferase activity in inflammatory bowel disease patients maintained on low-\\ndose azathioprine. Aliment Pharmacol Ther 2002;16:389-98.\\n23\\n. \\nColombel JF et al. Genotypic analysis of thiopurine S-methyltransferase in patients with Crohn’s disease and severe\\nmyelosuppression during azathioprine therapy. Gastroenterology 2000;118:1025-30.\\n24\\n. \\nBlack AJ et al. Thiopurine methyltransferase genotype predicts therapy-limiting severe toxicity from azathioprine. Ann Intern\\nMed 1998;129:716-8.\\n25\\n. \\nHiggs JE et al. Are patients with intermediate TPMT activity at increased risk of myelosuppression when taking thiopurine\\nmedications? Pharmacogenomics 2010;11:177-88.\\n26\\n. \\nStanulla M et al. Thiopurine methyltransferase (TPMT) genotype and early treatment response to mercaptopurine in childhood\\nacute lymphoblastic leukemia. JAMA 2005;293:1485-9.\\n27\\n. \\nEvans WE et al. Preponderance of thiopurine S-methyltransferase deficiency and heterozygosity among patients intolerant to\\nmercaptopurine or azathioprine. J Clin Oncol 2001;19:2293-301.\\n28\\n. \\nRelling MV et al. Mercaptopurine therapy intolerance and heterozygosity at the thiopurine S-methyltransferase gene locus. J Natl\\nCancer Inst 1999;91:2001-8.\\n29\\n. \\nMcLeod HL et al. Analysis of thiopurine methyltransferase variant alleles in childhood acute lymphoblastic leukaemia. Br J\\nHaematol 1999;105:696-700.\\nDate 27-05-2015\\nTPMT PM: azathioprine/mercaptopurine\\n1906\\nThe genetic variation reduces the conversion of azathioprine and mercaptopurine to mainly inactive metabolites. This increases the risk\\nof serious, life-threatening adverse events such as myelosuppression.\\nRecommendation:\\n1\\n. \\nChoose an alternative or start with 10% of the standard dose.\\nAny adjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and effectiveness.\\nThe frequency of monitoring should be increased.\\n2\\n. \\nIf the dose is decreased: advise patients to seek medical attention when symptoms of myelosuppression (such as severe sore throat\\nin combination with fever, regular nosebleeds and tendency to bruising) occur\\nLiterature:\\n1\\n. \\nYang JJ et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute\\nlymphoblastic leukemia. J Clin Oncol 2015 Jan 26. [Epub ahead of print].\\n2\\n. \\nBelen BF et al. Severe myelotoxicity associated with thiopurine S-methyltransferase\\n3A/ \\n3C polymorphisms in a patient with\\npediatric leukemia and the effect of steroid therapy. Turk J Haematol 2014;31:399-402.\\n3\\n. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of\\nthiopurine methyltransferase pharmacogenetics. Br J Haematol 2014 Nov 29 [Epub ahead of print].\\n4\\n. \\nKim MJ et al. Monitoring thiopurine metabolites in korean pediatric patients with inflammatory bowel disease. Yonsei Med J\\n2014;55:1289-96.\\n5\\n. \\nLevinsen M et al. Pharmacogenetically based dosing of thiopurines in childhood acute lymphoblastic leukemia: influence on cure\\nrates and risk of second cancer. Pediatr Blood Cancer 2014;61:797-802.\\n6\\n. \\nDemlova R et al. Augmenting clinical interpretability of thiopurine methyltransferase laboratory evaluation. Oncology\\n2014;86:152-8.\\n7\\n. \\nLee MN et al. Successful azathioprine treatment with metabolite monitoring in a pediatric inflammatory bowel disease patient\\nhomozygous for TPMT*3C. Yonsei Med J 2013;54:1545-9.\\n8\\n. \\nKim H et al. Pharmacogenetic analysis of pediatric patients with acute lymphoblastic leukemia: a possible association between\\nsurvival rate and ITPA polymorphism. PLoS One 2012;7:e45558.\\n9\\n. \\nBooth RA et al. Assessment of thiopurine S-methyltransferase activity in patients prescribed thiopurines: a systematic review. Ann', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 20}),\n",
       " Document(page_content=\"Intern Med 2011;154:814-23, W-295-8.\\n10\\n. \\nDong XW et al. Thiopurine S-methyltransferase polymorphisms and thiopurine toxicity in treatment of inflammatory bowel\\ndisease. World J Gastroenterol 2010;16:3187-95.\\n11\\n. \\nAnsari A et al. Influence of xanthine oxidase on thiopurine metabolism in Crohn's disease. Aliment Pharmacol Ther 2008;28:749-\\n57.\\n12\\n. \\nZelinkova Z et al. Inosine triphosphate pyrophosphatase and thiopurine s-methyltransferase genotypes relationship to azathioprine-\\ninduced myelosuppression. Clin Gastroenterol Hepatol 2006;4:44-9.\\n13\\n. \\nKurzawski M et al. The impact of thiopurine s-methyltransferase polymorphism on azathioprine-induced myelotoxicity in renal\\ntransplant recipients. Ther Drug Monit 2005;27:435-41.\\n14\\n. \\nGardiner SJ et al. Two cases of thiopurine methyltransferase (TPMT) deficiency--a lucky save and a near miss with azathioprine.\\nBr J Clin Pharmacol 2006;62:473-6.\\n15\\n. \\nKurzawski et al. Severe azathioprine-induced myelotoxicity in a kidney transplant patient with thiopurine S-methyltransferase-\\ndeficient genotype (TPMT\\n3A/\\n3C). Transpl Int 2005;18:623-5.\\n16\\n. \\nGearry RB et al. Thiopurine S-methyltransferase (TPMT) genotype does not predict adverse drug reactions to thiopurine drugs in\\npatients with inflammatory bowel disease. Aliment Pharmacol Ther 2003;18:395-400.\\n17\\n. \\nGilissen LP et al. Some cases demonstrating the clinical usefulness of therapeutic drug monitoring in thiopurine-treated\\ninflammatory bowel disease patients. Eur J Gastroenterol Hepatol 2004;16:705-10.\\n18\\n. \\nKaskas BA et al. Safe treatment of thiopurine S-methyltransferase deficient Crohn’s disease patients with azathioprine. Gut\\n2003;52:140-2.\\n19\\n. \\nColombel JF et al. Genotypic analysis of thiopurine S-methyltransferase in patients with Crohn’s disease and severe\\nmyelosuppression during azathioprine therapy. Gastroenterology 2000;118:1025-30.\\n20\\n. \\nHiggs JE et al. Are patients with intermediate TPMT activity at increased risk of myelosuppression when taking thiopurine\\nmedications? Pharmacogenomics 2010;11:177-88.\\n21\\n. \\nStanulla M et al. Thiopurine methyltransferase (TPMT) genotype and early treatment response to mercaptopurine in childhood\\nacute lymphoblastic leukemia. JAMA 2005;293:1485-9.\\n22\\n. \\nSchaeffeler et al. A novel TPMT missense mutation associated with TPMT deficiency in a 5-year-old boy with ALL. Leukemia\\n2003;17:1422-4.\\n23\\n. \\nEvans WE et al. Preponderance of thiopurine S-methyltransferase deficiency and heterozygosity among patients intolerant to\\nmercaptopurine or azathioprine. J Clin Oncol 2001;19:2293-301.\\n24\\n. \\nRelling MV et al. Mercaptopurine therapy intolerance and heterozygosity at the thiopurine S-methyltransferase gene locus. J Natl\\nCancer Inst 1999;91:2001-8.\\n25\\n. \\nMcLeod HL et al. Analysis of thiopurine methyltransferase variant alleles in childhood acute lymphoblastic leukaemia. Br J\\nHaematol 1999;105:696-700.\\n26\\n. \\nAndersen JB et al. Pharmacokinetics, dose adjustments, and 6-mercaptopurine/methotrexate drug interactions in two patients with\\nthiopurine methyltransferase deficiency. Acta Paediatr 1998;87:108-11.\\n27\\n. \\nSPC’s Imuran en Puri-Nethol.\\nDate 27-05-2015\\nCYP2D6 IM: bisoprolol\\n2457\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nNozawa T et al. Influence of CYP2D6 genotype on metoprolol plasma concentration and beta-adrenergic inhibition during long-\\nterm treatment: a comparison with bisoprolol. J Cardiovasc Pharmacol 2005;46:713-20.\\n2\\n. \\nTaguchi M et al. Pharmacokinetic variability of routinely administered bisoprolol in middle-aged and elderly Japanese patients.\\nBiol Pharm Bull 2005;28:876-81.\\nDate 26-05-2009\\nCYP2D6 PM: bisoprolol\\n2456\\nThis is NOT a gene-drug interaction.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 21}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nDeroubaix X et al. Comparative bioavailability of a metoprolol controlled release formulation and a bisoprolol normal release\\ntablet after single oral dose administration in healthy volunteers.\\nDate 26-05-2009\\nCYP2D6 UM: bisoprolol\\n2458\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 26-05-2009\\nCYP2D6 IM: carvedilol\\n2345\\nNO action is required for this gene-drug interaction.\\nThe plasma concentration of carvedilol can be elevated. This does not, however, result in an increase in side effects.\\nLiterature:\\n1\\n. \\nShihmanter R et al. Variation in the CYP2D6 genotype is not associated with carvedilol dose changes in patients with heart\\nfailure. J Clin Pharm Ther 2014;39:432-8.\\n2\\n. \\nNikolic VN et al. Population pharmacokinetics of carvedilol in patients with congestive heart failure. J Pharm Sci 2013;102:2851-\\n8.\\n3\\n. \\nSehrt D et al. Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics.\\nPharmacogenomics 2011;12:783-95.\\n4\\n. \\nSaito M et al. Population pharmacokinetics of R- and S-carvedilol in Japanese patients with chronic heart failure. Biol Pharm Bull\\n2010;33:1378-84.\\n5\\n. \\nBaudhuin LM et al. Relation of ADRB1, CYP2D6, and UGT1A1 polymorphisms with dose of, and response to, carvedilol or\\nmetoprolol therapy in patients with chronic heart failure. Am J Cardiol 2010;106:402-8.\\n6\\n. \\nHoriuchi I et al. Pharmacokinetics of R- and S-carvedilol in routinely treated Japanese patients with heart failure. Biol Pharm Bull\\n2008;31:976-80.\\n7\\n. \\nTakekuma Y et al. Evaluation of effects of polymorphism for metabolic enzymes on pharmacokinetics of carvedilol by population\\npharmacokinetic analysis. Biol Pharm Bull 2007;30:537-42.\\n8\\n. \\nHonda M et al. Multiple regression analysis of pharmacogenetic variability of carvedilol disposition in 54 healthy Japanese\\nvolunteers. Biol Pharm Bull 2006;29:772-8.\\n9\\n. \\nTakekuma Y et al. Contribution of polymorphisms in UDP-glucuronosyltransferase and CYP2D6 to the individual variation in\\ndisposition of carvedilol. J Pharm Pharm Sci 2006;9:101-12.\\n10\\n. \\nHonda M et al. Effect of CYP2D6*10 on the pharmacokinetics of R- and S-carvedilol in healthy Japanese volunteers. Biol Pharm\\nBull 2005;28:1476-9.\\nDate 24-08-2016\\nCYP2D6 PM: carvedilol\\n2344\\nNO action is required for this gene-drug interaction.\\nThe plasma concentration of carvedilol can be elevated. This does not, however, result in an increase in side effects.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 22}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nShihmanter R et al. Variation in the CYP2D6 genotype is not associated with carvedilol dose changes in patients with heart\\nfailure. J Clin Pharm Ther 2014;39:432-8.\\n2\\n. \\nSehrt D et al. Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics.\\nPharmacogenomics 2011;12:783-95.\\n3\\n. \\nBaudhuin LM et al. Relation of ADRB1, CYP2D6, and UGT1A1 polymorphisms with dose of, and response to, carvedilol or\\nmetoprolol therapy in patients with chronic heart failure. Am J Cardiol 2010;106:402-8.\\n4\\n. \\nGiessmann T et al. CYP2D6 genotype and induction of intestinal drug transporters by rifampin predict presystemic clearance of\\ncarvedilol in healthy subjects. Clin Pharmacol Ther 2004;75:213-22.\\n5\\n. \\nZhou HH et al. Stereoselective disposition of carvedilol is determined by CYP2D6. Clin Pharmacol Ther 1995;57:518-24.\\n6\\n. \\nSPC’s Carvedilol Sandoz (Nederland) en Coreg (VS).\\nDate 24-08-2016\\nCYP2D6 UM: carvedilol\\n2346\\nNO action is required for this gene-drug interaction.\\nThe plasma concentration of carvedilol can be reduced. This does not, however, result in a decrease in the effectiveness.\\nLiterature:\\n1\\n. \\nShihmanter R et al. Variation in the CYP2D6 genotype is not associated with carvedilol dose changes in patients with heart\\nfailure. J Clin Pharm Ther 2014;39:432-8.\\n2\\n. \\nSehrt D et al. Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics.\\nPharmacogenomics 2011;12:783-95.\\n3\\n. \\nBaudhuin LM et al. Relation of ADRB1, CYP2D6, and UGT1A1 polymorphisms with dose of, and response to, carvedilol or\\nmetoprolol therapy in patients with chronic heart failure. Am J Cardiol 2010;106:402-8.\\nDate 24-08-2016\\nCYP2C19 IM: citalopram\\n4195\\nThe risk of QT prolongation and torsades de pointes is theoretically increased because the gene variation leads to an increased citalopram\\nplasma concentration. If you follow the dose recommendation below, the increased plasma concentration and the theoretically increased\\nrisk of QT prolongation will be offset.\\nDo not exceed the following daily doses:\\n1\\n. \\nadults up to 65 years: 30 mg as tablets or 22 mg as drops\\n2\\n. \\nadults 65 years or older: 15 mg as tablets or 10 mg as drops\\nLiterature:\\n1\\n. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol\\n2014;28:1143-8.\\n2\\n. \\nChen B et al. Estimation of CYP2D6*10 genotypes on citalopram disposition in Chinese subjects by population pharmacokinetic\\nassay. J Clin Pharm Ther 2013;38:504-11.\\n3\\n. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67.\\n4\\n. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;2:1-9.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 23}),\n",
       " Document(page_content='5\\n. \\nFudio S et al. Evaluation of the influence of sex and CYP2C19 and CYP2D6 polymorphisms in the disposition of citalopram. Eur J\\nPharmacol 2010;25:200-4.\\n6\\n. \\nHilli J et al. MAO-A and COMT genotypes as possible regulators of perinatal serotonergic symptoms after in utero exposure to\\nSSRIs. Eur Neuropsychopharmacol 2009;19:363-70.\\n7\\n. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS ONE\\n2008;3:e1872.\\n8\\n. \\nYin OQ et al. Phenotype-genotype relationship and clinical effects of citalopram in Chinese patients. J Clin Psychopharmacol\\n2006;26:367-72.\\n9\\n. \\nRudberg I et al. Heterozygous mutation in CYP2C19 significantly increases the concentrations/dose ratio of racemic citalopram\\nand escitalopram (S-citalopram). Ther Drug Monitor 2006;28:102-5.\\n10\\n. \\nYu BN et al. Pharmacokinetics of citalopram in relation to genetic polymorphism of CYP2C19. Drug Metab Dispos 2003;31:1255-\\n9.\\n11\\n. \\nJimmink A et al. Clinical toxicology of citalopram after acute intoxication with the sole drug or in combination with other drugs:\\noverview of 26 cases. Ther Drug Monit 2008;30:365-71.\\nDate 14-05-2018\\nCYP2C19 PM: citalopram\\n4196\\nThe risk of QT prolongation and therefore also the theoretical risk of torsades de pointes is increased as the gene variation leads to an\\nincreased citalopram plasma concentration. If you follow the dose recommendation below, the increased plasma concentration and the\\nincreased risk of QT prolongation will be offset.\\ndo not exceed the following daily doses (50% of the standard maximum dose):\\n1\\n. \\nadults up to 65 years: 20 mg as tablets or 16 mg as drops\\n2\\n. \\nadults 65 years or older: 10 mg as tablets or 8 mg as drops\\nLiterature:\\n1\\n. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol\\n2014;28:1143-8.\\n2\\n. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67.\\n3\\n. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;2:1-9.\\n4\\n. \\nFudio S et al. Evaluation of the influence of sex and CYP2C19 and CYP2D6 polymorphisms in the disposition of citalopram. Eur J\\nPharmacol 2010;25:200-4.\\n5\\n. \\nHilli J et al. MAO-A and COMT genotypes as possible regulators of perinatal serotonergic symptoms after in utero exposure to\\nSSRIs. Eur Neuropsychopharmacol 2009;19:363-70.\\n6\\n. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS ONE\\n2008;3:e1872.\\n7\\n. \\nYin OQ et al. Phenotype-genotype relationship and clinical effects of citalopram in Chinese patients. J Clin Psychopharmacol\\n2006;26:367-72.\\n8\\n. \\nHerrlin K et al. Metabolism of citalopram enantiomers in CYP2C19/CYP2D6 phenotyped panels of healthy Swedes. Br J Clin\\nPharmacol 2003;56:415-21.\\n9\\n. \\nYu BN et al. Pharmacokinetics of citalopram in relation to genetic polymorphism of CYP2C19. Drug Metab Dispos 2003;31:1255-\\n9.\\n10\\n. \\nBaumann P et al. A double-blind, placebo-controlled study of citalopram with and without lithium in the treatment of therapy-\\nresistant depressive patients: a clinical, pharmacokinetic, and pharmacogenetic investigation. J Clin Psychopharmacol\\n1996;16:307-14.\\n11\\n. \\nSindrup SH et al. Pharmacokinetics of citalopram in relation to the sparteine and the mephenytoin oxidation polymorphisms. Ther\\nDrug Monit 1993;15:11-7.\\n12\\n. \\nJimmink A et al. Clinical toxicology of citalopram after acute intoxication with the sole drug or in combination with other drugs:\\noverview of 26 cases. Ther Drug Monit 2008;30:365-71.\\n13\\n. \\nSPC’s Cipramil en Celexa (VS).\\nDate 14-05-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 24}),\n",
       " Document(page_content='CYP2C19 UM: citalopram\\n4197\\nNO action is needed for this gene-drug interaction.\\nThe gene variation increases conversion of citalopram to a weakly active metabolite. However, there is no significant effect on the\\nplasma concentration of citalopram, the tolerance or the response.\\nLiterature:\\n1\\n. \\nDe Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67.\\n2\\n. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;2:1-9.\\n3\\n. \\nHilli J et al. MAO-A and COMT genotypes as possible regulators of perinatal serotonergic symptoms after in utero exposure to\\nSSRIs. Eur Neuropsychopharmacol 2009;19:363-70.\\n4\\n. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS ONE\\n2008;3:e1872.\\n5\\n. \\nJimmink A et al. Clinical toxicology of citalopram after acute intoxication with the sole drug or in combination with other drugs:\\noverview of 26 cases. Ther Drug Monit 2008;30:365-71.\\nDate 14-05-2018\\nCYP2D6 IM: citalopram/escitalopram\\n1999\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nChen B et al. Estimation of CYP2D6*10 genotypes on citalopram disposition in Chinese subjects by population pharmacokinetic\\nassay. J Clin Pharm Ther 2013;38:504-11. PubMed PMID: 23981149.\\n2\\n. \\nHan KM et al. CYP2D6 P34S polymorphism and outcomes of escitalopram treatment in Koreans with major depression.\\nPsychiatry Investig 2013;10:286-93. PubMed PMID: 24302953.\\n3\\n. \\nHuezo-Diaz P et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol\\n2012;26:398-407. PubMed PMID: 21926427.\\n4\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67. PubMed PMID: 20531370.\\n5\\n. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;21:1-9. PubMed PMID:\\n21192344.\\n6\\n. \\nTsai MH et al. Genetic polymorphisms of cytochrome P450 enzymes influence metabolism of the antidepressant escitalopram and\\ntreatment response. Pharmacogenomics 2010;11:537-46. PubMed PMID: 20350136.\\n7\\n. \\nFudio S et al. Evaluation of the influence of sex and CYP2C19 and CYP2D6 polymorphisms in the disposition of citalopram. Eur J\\nPharmacol 2010;626:200-4. PubMed PMID: 19840783.\\n8\\n. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS One\\n2008;3:e1872. PubMed PMID: 18382661.\\n9\\n. \\nCarlsson B et al. Enantioselective analysis of citalopram and metabolites in adolescents. Ther Drug Monit 2001;23:658-64.\\n10\\n. \\nSPC Cipramil.\\nDate 14-05-2018\\nCYP2D6 PM: citalopram/escitalopram\\n1998\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 25}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nHuezo-Diaz P et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol\\n2012;26:398-407. PubMed PMID: 21926427.\\n2\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67. PubMed PMID: 20531370.\\n3\\n. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;21:1-9. PubMed PMID:\\n21192344.\\n4\\n. \\nPeters EJ et al. Pharmacokinetic genes do not influence response or tolerance to citalopram in the STAR*D sample. PLoS One\\n2008;3:e1872. PubMed PMID: 18382661.\\n5\\n. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and\\nclinical effects of antidepressants in a naturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n6\\n. \\nHerrlin K et al. Metabolism of citalopram enantiomers in CYP2C19/CYP2D6 phenotyped panels of healthy Swedes. Br J Clin\\nPharmacol 2003;56:415-21.\\n7\\n. \\nBondolfi G et al. Non-response to citalopram in depressive patients: pharmacokinetic and clinical consequences of a fluvoxamine\\naugmentation. Psychopharmacology 1996;128:421-5.\\n8\\n. \\nSindrup SH et al. Pharmacokinetics of citalopram in relation to the sparteine and the mephenytoin oxidation polymorphisms. Ther\\nDrug Monit 1993;15:11-7.\\n9\\n. \\nSPC’s Cipramil, Lexapro (NL en VS) en Celexa (VS).\\nDate 14-05-2018\\nCYP2D6 UM: citalopram/escitalopram\\n2000\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nHuezo-Diaz P et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol\\n2012;26:398-407. PubMed PMID: 21926427.\\n2\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67. PubMed PMID: 20531370.\\n3\\n. \\nMrazek DA et al. CYP2C19 variation and citalopram response. Pharmacogenet Genomics 2011;21:1-9. PubMed PMID:\\n21192344.\\n4\\n. \\nCarlsson B et al. Enantioselective analysis of citalopram and metabolites in adolescents. Ther Drug Monit 2001;23:658-64.\\n5\\n. \\nSPC Cipramil.\\nDate 14-05-2018\\nCYP2D6 IM: clomipramine\\n1481\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of clomipramine and the\\nactive metabolite desmethylclomipramine.\\nuse 70% of the standard dose and monitor the effect and side effects or the plasma concentrations of clomipramine and\\ndesmethylclomipramine\\nFor depression, the therapeutic range is 200-400 ng/mL for the sum of the plasma concentrations of clomipramine and\\ndesmethylclomipramine.\\nFor anxiety disorders, the therapeutic plasma concentration of clomipramine is approximately 100 ng/mL, in combination with a\\nplasma concentration of desmethylclomipramine lower than 200 ng/mLFor obsessive compulsive disorder, the therapeutic plasma\\nconcentration of clomipramine is higher than 200 ng/mL, in combination with a plasma concentration of desmethylclomipramine\\nthat is as low as possible.\\nA sum of the plasma concentrations of clomipramine and desmethylclomipramine higher than 600 ng/mL is considered toxic.\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 26}),\n",
       " Document(page_content='1\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67.\\n2\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67, persoonlijke communicatie.\\n3\\n. \\nVandel P et al. Clomipramine, fluoxetine and CYP2D6 metabolic capacity in depressed patients. Hum Psychopharmacol\\n2004;19:293-8.\\n4\\n. \\nYokono A et al. The effect of CYP2C19 and CYP2D6 genotypes on the metabolism of clomipramine in Japanese psychiatric\\npatients. Clin Psychopharmacol 2001;21:549-55.\\n5\\n. \\nBalant-Gorgia et al. High blood concentrations of imipramine or clomipramine and therapeutic failure: a case report study using\\ndrug monitoring data. Ther Drug Monit 1989;11:415-20.\\nDate 19-11-2018\\nCYP2D6 PM: clomipramine\\n1480\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of clomipramine and the\\nactive metabolite desmethylclomipramine.\\nIndication DEPRESSION:\\nuse 50% of the standard dose and monitor the effect and side effects or the plasma concentrations of clomipramine and\\ndesmethylclomipramine in order to set the maintenance dose.\\nThe therapeutic range is 200-400 ng/mL for the sum of the plasma concentrations of clomipramine and\\ndesmethylclomipramine. Values higher than 600 ng/mL are considered toxic.\\nIndication ANXIETY DISORDERS or OBSESSIVE COMPULSIVE DISORDER:\\nif side effects occur: use 50% of the standard dose and monitor the effect and side effects or the plasma concentrations of\\nclomipramine and desmethylclomipramine in order to set the maintenance dose.\\nIt is not known whether it is possible to reduce the dose to such an extent that the side effects disappear while the\\neffectiveness is retained. Clomipramine and desmethylclomipramine both contribute to the side effects. Only clomipramine\\ncontributes to the effectiveness.\\nFor anxiety disorders, the therapeutic plasma concentration of clomipramine is approximately 100 ng/mL, in combination\\nwith a plasma concentration of desmethylclomipramine lower than 200 ng/mL..\\nFor obsessive compulsive disorder, the therapeutic plasma concentration of clomipramine is higher than 200 ng/mL, in\\ncombination with a plasma concentration of desmethylclomipramine that is as low as possible.\\nA sum of the plasma concentrations of clomipramine and desmethylclomipramine higher than 600 ng/mL is considered\\ntoxic, whilst the therapeutic upper limit for depression is 400 ng/mL. .\\nif dose reduction does not have the desired effect: avoid clomipramine\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include citalopram and sertraline.\\nLiterature:\\n1\\n. \\nde Jong C et al. Clomipramine toxicity in a CYP 2D6 poor metabolizer patient who suddenly stopped smoking. J Clin\\nPsychopharmacol 2018;38:389-391.\\n2\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67.\\n3\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67, persoonlijke communicatie.\\n4\\n. \\nStephan PL et al. Adverse drug reactions following nonresponse in a depressed patient with CYP2D6 deficiency and low CYP\\n3A4/5 activity. Pharmacopsychiatry 2006;39:150-2.\\n5\\n. \\nDanish University Antidepressant Group. Clomipramine dose-effect study in patients with depression: clinical end points and\\npharmacokinetics. Clin Pharmacol Ther 1999;66:152-65.\\n6\\n. \\nNielsen KK et al. Single-dose kinetics of clomipramine: relationship to the sparteine and S-mephenytoin oxidation\\npolymorphisms. Clin Pharmacol Ther 1994;55:518-27.\\n7\\n. \\nTacke U et al. Debrisoquine hydroxylation phenotypes of patients with high versus low to normal serum antidepressant\\nconcentrations. J Clin Psychopharmacol 1992;12:262-7.\\n8\\n. \\nNielsen KK et al. Steady-state plasma levels of clomipramine and its metabolites: impact of the sparteine/debrisoquine oxidation\\npolymorphism. Eur J Clin Pharmacol 1992;43:405-11.\\n9\\n. \\nBalant-Gorgia AE et al. High plasma concentrations of desmethylclomipramine after chronic administration of clomipramine to a\\npoor metabolizer. Eur J Clin Pharmacol 1987;32:101-2.\\n10\\n. \\nBalant-Gorgia AE et al. High blood concentrations of imipramine or clomipramine and therapeutic failure: a case report study\\nusing drug monitoring data. Ther Drug Monit 1989;11:415-20.\\n11\\n. \\nSmPC Anafranil (VS).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 27}),\n",
       " Document(page_content='Date 19-11-2018\\nCYP2D6 UM: clomipramine\\n1482\\nThe risk of ineffectiveness and cardiotoxic side effects may be increased. The gene variation leads to reduced plasma concentrations of\\nclomipramine and the active metabolite desmethylclomipramine and to increased concentrations of the potentially cardiotoxic hydroxy\\nmetabolites.\\nuse 1.5 times the standard dose and monitor the effect and side effects of the plasma concentrations of clomipramine and\\ndesmethylclomipramine to set the maintenance dose.\\nFor depression, the therapeutic range is 200-400 ng/mL for the sum of the plasma concentrations of clomipramine and\\ndesmethylclomipramine.\\nFor anxiety disorders, the therapeutic plasma concentration of clomipramine is approximately 100 ng/mL, in combination with a\\nplasma concentration of desmethylclomipramine lower than 200 ng/mL.\\nFor obsessive compulsive disorder, the therapeutic plasma concentration of clomipramine is higher than 200 ng/mL, in\\ncombination with a plasma concentration of desmethylclomipramine that is as low as possible.\\nif a dose increase is not wanted due to potential cardiotoxic hydroxy metabolites: avoid clomipramine.\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include citalopram and sertraline.\\nLiterature:\\n1\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67.\\n2\\n. \\nde Vos A et al. Association between CYP2C19*17 and metabolism of amitriptyline, citalopram and clomipramine in Dutch\\nhospitalized patients. Pharmacogenomics J 2011;11:359-67, persoonlijke communicatie.\\n3\\n. \\nBaumann P et al. Ultrarapid metabolism of clomipramine in a therapy-resistant depressive patient, as confirmed by CYP2 D6\\ngenotyping. Pharmacopsychiatry 1998;31:72.\\n4\\n. \\nBertilsson L et al. Molecular basis for rational megaprescribing in ultrarapid hydroxylators of debrisoquine. Lancet 1993;341:63.\\nDate 19-11-2018\\nCYP2D6 IM: clonidine\\n2531\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 PM: clonidine\\n2530\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 28}),\n",
       " Document(page_content='Date 24-08-2016\\nCYP2D6 UM: clonidine\\n2532\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2C19 IM: clopidogrel\\n2549\\nThe risk of serious cardiovascular and cerebrovascular events is increased in patients undergoing balloon angioplasty or stent placement\\n(percutaneous coronary intervention) and in patients with a stroke or TIA, as the genetic variation reduces the activation of clopidogrel.\\nNo negative clinical consequences have been observed in other patients.\\nPERCUTANEOUS CORONARY INTERVENTION, STROKE or TIA:\\nchoose an alternative or double the dose to 150 mg/day (600 mg loading dose)\\nPrasugrel, ticagrelor and acetylsalicylic acid/dipyridamole are not metabolised by CYP2C19 (or to a lesser extent).\\nOTHER INDICATIONS:\\nno action required\\nLiterature:\\n1\\n. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous\\ncoronary intervention. Circ Genom Precis Med 2018;11:e002069.\\n2\\n. \\nZhong Z et al. Effect of cytochrome P450 2C19 polymorphism on adverse cardiovascular events after drug-eluting stent\\nimplantation in a large Hakka population with acute coronary syndrome receiving clopidogrel in southern China. Eur J Clin\\nPharmacol 2018;74:423-31.\\n3\\n. \\nWu Y et al. Impact of CYP2C19 polymorphism in prognosis of minor stroke or TIA patients with declined eGFR on dual\\nantiplatelet therapy: CHANCE substudy. Pharmacogenomics J 2018 Mar 8 [Epub ahead of print].\\n4\\n. \\nCavallari LH et al. Multisite investigation of outcomes with implementation of CYP2C19 genotype-guided antiplatelet therapy\\nafter percutaneous coronary intervention. JACC Cardiovasc Interv 2018;11:181-91.\\n5\\n. \\nLin Y et al. Impact of glycemic control on efficacy of clopidogrel in transient ischemic attack or minor stroke patients with\\nCYP2C19 genetic variants. Stroke 2017;48:998-1004.\\n6\\n. \\nPan Y et al. Genetic polymorphisms and clopidogrel efficacy for acute ischemic stroke or transient ischemic attack: a systematic\\nreview and meta-analysis. Circulation 2017;135:21-33.\\n7\\n. \\nWang Y et al. Association between CYP2C19 loss-of-function allele status and efficacy of clopidogrel for risk reduction among\\npatients with minor stroke or transient ischemic attack. JAMA 2016;316:70-8.\\n8\\n. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events\\nin Japanese patients with acute coronary syndrome: The PRASFIT-ACS study.\\n9\\n. \\nShen DL et al. Clinical value of CYP2C19 genetic testing for guiding the antiplatelet therapy in a Chinese population. J\\nCardiovasc Pharmacol 2016;67:232-6.\\n10\\n. \\nNiu X et al. CYP2C19 polymorphism and clinical outcomes among patients of different races treated with clopidogrel: a\\nsystematic review and meta-analysis. J Huazhong Univ Sci Technolog Med Sci 2015;35:147-56.\\n11\\n. \\nSorich MJ et al. CYP2C19 genotype has a greater effect on adverse cardiovascular outcomes following percutaneous coronary\\nintervention and in Asian populations treated with clopidogrel: a meta-analysis. Circ Cardiovasc Genet 2014;7:895-902.\\n12\\n. \\nMao L et al. Cytochrome CYP2C19 polymorphism and risk of adverse clinical events in clopidogrel-treated patients: a meta-\\nanalysis based on 23,035 subjects. Arch Cardiovasc Dis 2013;106:517-27.\\n13\\n. \\nXie X et al. Personalized antiplatelet therapy according to CYP2C19 genotype after percutaneous coronary intervention: a\\nrandomized control trial. Int J Cardiol 2013;168:3736-40.\\n14\\n. \\nJang JS et al. Meta-analysis of cytochrome P450 2C19 polymorphism and risk of adverse clinical outcomes among coronary artery\\ndisease patients of different ethnic groups treated with clopidogrel. Am J Cardiol 2012;110:502-8.\\n15\\n. \\nHolmes MV et al. CYP2C19 genotype, clopidogrel metabolism, platelet function, and cardiovascular events: a systematic review\\nand meta-analysis. JAMA 2011;306:2704-14.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 29}),\n",
       " Document(page_content='16\\n. \\nLiu YP et al. Association of genetic variants in CYP2C19 and adverse clinical outcomes after treatment with clopidogrel: an\\nupdated meta-analysis. Thromb Res 2011;128:593-4.\\n17\\n. \\nMega JL et al. Dosing clopidogrel based on CYP2C19 genotype and the effect on platelet reactivity in patients with stable\\ncardiovascular disease. JAMA 2011;306:2221-8.\\n18\\n. \\nSimon T et al. Genetic polymorphisms and the impact of a higher clopidogrel dose regimen on active metabolite exposure and\\nantiplatelet response in healthy subjects. Clin Pharmacol Ther 2011;90:287-95.\\n19\\n. \\nCollet JP et al. High doses of clopidogrel to overcome genetic resistance: the randomized cross-over CLOVIS-2 (Clopidogrel and\\nResponse Variability Investigation Study 2). JACC Cardiovasc Interv 2011;4:392-402.\\n20\\n. \\nBonello-Palot N et al. Relation of body mass index to high on-treatment platelet reactivity and of failed clopidogrel dose\\nadjustment according to platelet reactivity monitoring in patients undergoing percutaneous coronary intervention. Am J Cardiol\\n2009;104:1511-5.\\n21\\n. \\nShuldiner AR et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel\\ntherapy. JAMA 2009;302:849-57.\\n22\\n. \\nAleil B et al. CYP2C19*2 polymorphism is not the sole determinant of the response to clopidogrel: implications for its\\nmonitoring. J Thromb Haemost 2009;7:1747-9.\\n23\\n. \\nPena A et al. Can we override clopidogrel resistance? Circulation 2009;119:2854-7.\\n24\\n. \\nSibbing D et al. Cytochrome P450 2C19 loss-of-function polymorphism and stent thrombosis following percutaneous coronary\\nintervention. Eur Heart J 2009;30:916-22.\\n25\\n. \\nBrackbill ML et al. Frequency of CYP3A4, CYP3A5, CYP2C9, and CYP2C19 variant alleles in patients receiving clopidogrel\\nthat experience repeat acute coronary syndrome. Heart Vessels 2009;24:73-8.\\n26\\n. \\nGiusti B et al. Relation of cytochrome P450 2C19 loss-of-function polymorphism to occurrence of drug-eluting coronary stent\\nthrombosis. Am J Cardiol 2009;103:806-11.\\n27\\n. \\nCollet JP et al. Cytochrome P450 2C19 polymorphism in young patients treated with clopidogrel after myocardial infarction: a\\ncohort study. Lancet 2009;373:309-17.\\n28\\n. \\nMega JL et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N Engl J Med 2009;360:354-62.\\n29\\n. \\nSimon T et al. Genetic determinants of response to clopidogrel and cardiovascular events. N Engl J Med 2009;360:363-75.\\n30\\n. \\nGeisler T et al. CYP2C19 and nongenetic factors predict poor responsiveness to clopidogrel loading dose after coronary stent\\nimplantation. Pharmacogenomics 2008;9:1251-9.\\n31\\n. \\nUmemura K et al. The common gene variants of CYP2C19 affect pharmacokinetics and pharmacodynamics in an active\\nmetabolite of clopidogrel in healthy subjects. J Thromb Haemost 2008;6:1439-41.\\n32\\n. \\nChen BL et al. Inhibition of ADP-induced platelet aggregation by clopidogrel is related to CYP2C19 genetic polymorphisms. Clin\\nExp Pharmacol Physiol 2008;35:904-8.\\n33\\n. \\nKim KA et al. The effect of CYP2C19 polymorphism on the pharmacokinetics and pharmacodynamics of clopidogrel: a possible\\nmechanism for clopidogrel resistance. Clin Pharmacol Ther 2008;84:236-42.\\n34\\n. \\nMalek LA et al. Coexisting polymorphisms of P2Y12 and CYP2C19 genes as a risk factor for persistent platelet activation with\\nclopidogrel. Circ J 2008;72:1165-9.\\n35\\n. \\nTrenk D et al. Cytochrome P450 2C19 681G>A polymorphism and high on-clopidogrel platelet reactivity associated with adverse\\n1-year clinical outcome of elective percutaneous coronary intervention with drug-eluting or bare-metal stents. J Am Coll Cardiol\\n2008;51:1925-34.\\n36\\n. \\nFrere C et al. Effect of cytochrome p450 polymorphisms on platelet reactivity after treatment with clopidogrel in acute coronary\\nsyndrome. Am J Cardiol 2008;101:1088-93.\\n37\\n. \\nFontana P et al. Biological effect of increased maintenance dose of clopidogrel in cardiovascular outpatients and influence of the\\ncytochrome P450 2C19*2 allele on clopidogrel responsiveness. Thromb Res 2008;121:463-8.\\n38\\n. \\nGiusti B et al. Cytochrome P450 2C19 loss-of-function polymorphism, but not CYP3A4 IVS10 + 12G/A and P2Y12 T744C\\npolymorphisms, is associated with response variability to dual antiplatelet treatment in high-risk vascular patients. Pharmacogenet\\nGenomics 2007;17:1057-64.\\n39\\n. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response\\nto clopidogrel but not prasugrel. J Thromb Haemost 2007;5:2429-36.\\n40\\n. \\nFontana P et al. Influence of CYP2C19 and CYP3A4 gene polymorphisms on clopidogrel responsiveness in healthy subjects. J\\nThromb Haemost 2007;5:2153-5.\\n41\\n. \\nHulot JS et al. Cytochrome P450 2C19 loss-of-function polymorphism is a major determinant of clopidogrel responsiveness in\\nhealthy subjects. Blood 2006;108:2244-7.\\n42\\n. \\nCBO Conceptrichtlijn Diagnostiek, Preventie en Behandeling van Veneuze Trombo-embolie en Secundaire Preventie Arteriële\\nTrombose, 2008. www.cbo.nl.\\nDate 19-11-2018\\nCYP2C19 PM: clopidogrel\\n2548\\nThe risk of serious cardiovascular and cerebrovascular events is increased in patients undergoing balloon angioplasty or stent placement\\n(percutaneous coronary intervention) and in patients with a stroke or TIA, because the genetic variation reduces the activation of\\nclopidogrel. No negative clinical consequences have been proved in other patients.\\nPERCUTANEOUS CORONARY INTERVENTION, STROKE or TIA:\\navoid clopidogrel', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 30}),\n",
       " Document(page_content='Prasugrel, ticagrelor and acetylsalicylic acid/dipyridamole are not metabolised by CYP2C19 (or to a lesser extent).\\nOTHER INDICATIONS:\\ndetermine the level of inhibition of platelet aggregation by clopidogrel\\nconsider an alternative in poor responders\\nPrasugrel and ticagrelor are not metabolised by CYP2C19 (or to a lesser extent).\\nLiterature:\\n1\\n. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous\\ncoronary intervention. Circ Genom Precis Med 2018;11:e002069.\\n2\\n. \\nZhong Z et al. Effect of cytochrome P450 2C19 polymorphism on adverse cardiovascular events after drug-eluting stent\\nimplantation in a large Hakka population with acute coronary syndrome receiving clopidogrel in southern China. Eur J Clin\\nPharmacol 2018;74:423-31.\\n3\\n. \\nWu Y et al. Impact of CYP2C19 polymorphism in prognosis of minor stroke or TIA patients with declined eGFR on dual\\nantiplatelet therapy: CHANCE substudy. Pharmacogenomics J 2018 Mar 8 [Epub ahead of print].\\n4\\n. \\nCavallari LH et al. Multisite investigation of outcomes with implementation of CYP2C19 genotype-guided antiplatelet therapy\\nafter percutaneous coronary intervention. JACC Cardiovasc Interv 2018;11:181-91.\\n5\\n. \\nLin Y et al. Impact of glycemic control on efficacy of clopidogrel in transient ischemic attack or minor stroke patients with\\nCYP2C19 genetic variants. Stroke 2017;48:998-1004.\\n6\\n. \\nPan Y et al. Genetic polymorphisms and clopidogrel efficacy for acute ischemic stroke or transient ischemic attack: a systematic\\nreview and meta-analysis. Circulation 2017;135:21-33.\\n7\\n. \\nDeiman BA et al. Reduced number of cardiovascular events and increased cost-effectiveness by genotype-guided antiplatelet\\ntherapy in patients undergoing percutaneous coronary interventions in the Netherlands. Neth Heart J 2016;24:589-99.\\n8\\n. \\nWang Y et al. Association between CYP2C19 loss-of-function allele status and efficacy of clopidogrel for risk reduction among\\npatients with minor stroke or transient ischemic attack. JAMA 2016;316:70-8.\\n9\\n. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events\\nin Japanese patients with acute coronary syndrome: The PRASFIT-ACS study.\\n10\\n. \\nShen DL et al. Clinical value of CYP2C19 genetic testing for guiding the antiplatelet therapy in a Chinese population. J\\nCardiovasc Pharmacol 2016;67:232-6.\\n11\\n. \\nXiong R et al. A randomized controlled trial to assess the efficacy and safety of doubling dose clopidogrel versus ticagrelor for the\\ntreatment of acute coronary syndrome in patients with CYP2C19*2 homozygotes. Int J Clin Exp Med 2015;8:13310-6.\\n12\\n. \\nNiu X et al. CYP2C19 polymorphism and clinical outcomes among patients of different races treated with clopidogrel: a\\nsystematic review and meta-analysis. J Huazhong Univ Sci Technolog Med Sci 2015;35:147-56.\\n13\\n. \\nSorich MJ et al. CYP2C19 genotype has a greater effect on adverse cardiovascular outcomes following percutaneous coronary\\nintervention and in Asian populations treated with clopidogrel: a meta-analysis. Circ Cardiovasc Genet 2014;7:895-902.\\n14\\n. \\nMao L et al. Cytochrome CYP2C19 polymorphism and risk of adverse clinical events in clopidogrel-treated patients: a meta-\\nanalysis based on 23,035 subjects. Arch Cardiovasc Dis 2013;106:517-27.\\n15\\n. \\nXie X et al. Personalized antiplatelet therapy according to CYP2C19 genotype after percutaneous coronary intervention: a\\nrandomized control trial. Int J Cardiol 2013;168:3736-40.\\n16\\n. \\nJang JS et al. Meta-analysis of cytochrome P450 2C19 polymorphism and risk of adverse clinical outcomes among coronary artery\\ndisease patients of different ethnic groups treated with clopidogrel. Am J Cardiol 2012;110:502-8.\\n17\\n. \\nHolmes MV et al. CYP2C19 genotype, clopidogrel metabolism, platelet function, and cardiovascular events: a systematic review\\nand meta-analysis. JAMA 2011;306:2704-14.\\n18\\n. \\nLiu YP et al. Association of genetic variants in CYP2C19 and adverse clinical outcomes after treatment with clopidogrel: an\\nupdated meta-analysis. Thromb Res 2011;128:593-4.\\n19\\n. \\nMega JL et al. Dosing clopidogrel based on CYP2C19 genotype and the effect on platelet reactivity in patients with stable\\ncardiovascular disease. JAMA 2011;306:2221-8.\\n20\\n. \\nSimon T et al. Genetic polymorphisms and the impact of a higher clopidogrel dose regimen on active metabolite exposure and\\nantiplatelet response in healthy subjects. Clin Pharmacol Ther 2011;90:287-95.\\n21\\n. \\nCollet JP et al. High doses of clopidogrel to overcome genetic resistance: the randomized cross-over CLOVIS-2 (Clopidogrel and\\nResponse Variability Investigation Study 2). JACC Cardiovasc Interv 2011;4:392-402.\\n22\\n. \\nBonello-Palot N et al. Relation of body mass index to high on-treatment platelet reactivity and of failed clopidogrel dose\\nadjustment according to platelet reactivity monitoring in patients undergoing percutaneous coronary intervention. Am J Cardiol\\n2009;104:1511-5.\\n23\\n. \\nShuldiner AR et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel\\ntherapy. JAMA 2009;302:849-57.\\n24\\n. \\nAleil B et al. CYP2C19*2 polymorphism is not the sole determinant of the response to clopidogrel: implications for its\\nmonitoring. J Thromb Haemost 2009;7:1747-9.\\n25\\n. \\nPena A et al. Can we override clopidogrel resistance? Circulation 2009;119:2854-7.\\n26\\n. \\nSibbing D et al. Cytochrome P450 2C19 loss-of-function polymorphism and stent thrombosis following percutaneous coronary\\nintervention. Eur Heart J 2009;30:916-22.\\n27\\n. \\nBrackbill ML et al. Frequency of CYP3A4, CYP3A5, CYP2C9, and CYP2C19 variant alleles in patients receiving clopidogrel\\nthat experience repeat acute coronary syndrome. Heart Vessels 2009;24:73-8.\\n28\\n. \\nGiusti B et al. Relation of cytochrome P450 2C19 loss-of-function polymorphism to occurrence of drug-eluting coronary stent\\nthrombosis. Am J Cardiol 2009;103:806-11.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 31}),\n",
       " Document(page_content='29\\n. \\nCollet JP et al. Cytochrome P450 2C19 polymorphism in young patients treated with clopidogrel after myocardial infarction: a\\ncohort study. Lancet 2009;373:309-17.\\n30\\n. \\nMega JL et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N Engl J Med 2009;360:354-62.\\n31\\n. \\nSimon T et al. Genetic determinants of response to clopidogrel and cardiovascular events. N Engl J Med 2009;360:363-75.\\n32\\n. \\nGeisler T et al. CYP2C19 and nongenetic factors predict poor responsiveness to clopidogrel loading dose after coronary stent\\nimplantation. Pharmacogenomics 2008;9:1251-9.\\n33\\n. \\nUmemura K et al. The common gene variants of CYP2C19 affect pharmacokinetics and pharmacodynamics in an active\\nmetabolite of clopidogrel in healthy subjects. J Thromb Haemost 2008;6:1439-41.\\n34\\n. \\nChen BL et al. Inhibition of ADP-induced platelet aggregation by clopidogrel is related to CYP2C19 genetic polymorphisms. Clin\\nExp Pharmacol Physiol 2008;35:904-8.\\n35\\n. \\nKim KA et al. The effect of CYP2C19 polymorphism on the pharmacokinetics and pharmacodynamics of clopidogrel: a possible\\nmechanism for clopidogrel resistance. Clin Pharmacol Ther 2008;84:236-42.\\n36\\n. \\nMalek LA et al. Coexisting polymorphisms of P2Y12 and CYP2C19 genes as a risk factor for persistent platelet activation with\\nclopidogrel. Circ J 2008;72:1165-9.\\n37\\n. \\nTrenk D et al. Cytochrome P450 2C19 681G>A polymorphism and high on-clopidogrel platelet reactivity associated with adverse\\n1-year clinical outcome of elective percutaneous coronary intervention with drug-eluting or bare-metal stents. J Am Coll Cardiol\\n2008;51:1925-34.\\n38\\n. \\nFrere C et al. Effect of cytochrome p450 polymorphisms on platelet reactivity after treatment with clopidogrel in acute coronary\\nsyndrome. Am J Cardiol 2008;101:1088-93.\\n39\\n. \\nFontana P et al. Biological effect of increased maintenance dose of clopidogrel in cardiovascular outpatients and influence of the\\ncytochrome P450 2C19*2 allele on clopidogrel responsiveness. Thromb Res 2008;121:463-8.\\n40\\n. \\nGiusti B et al. Cytochrome P450 2C19 loss-of-function polymorphism, but not CYP3A4 IVS10 + 12G/A and P2Y12 T744C\\npolymorphisms, is associated with response variability to dual antiplatelet treatment in high-risk vascular patients. Pharmacogenet\\nGenomics 2007;17:1057-64.\\n41\\n. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response\\nto clopidogrel but not prasugrel. J Thromb Haemost 2007;5:2429-36.\\n42\\n. \\nFDA Drug Safety Communication: Reduced effectiveness of Plavix (clopidogrel) in patients who are poor metabolizers of the\\ndrug. 03-12-10.\\n43\\n. \\nSPC’s Clopidogrel Sandoz en Plavix (VS).\\n44\\n. \\nCBO Conceptrichtlijn Diagnostiek, Preventie en Behandeling van Veneuze Trombo-embolie en Secundaire Preventie Arteriële\\nTrombose, 2008. www.cbo.nl.\\nDate 19-11-2018\\nCYP2C19 UM: clopidogrel\\n2550\\nNO action is required for this gene-drug interaction.\\nThe genetic variation results in increased conversion of clopidogrel to the active metabolite. However, this can result in both positive\\neffects (reduction in the risk of serious cardiovascular and cerebrovascular events) and negative effects (increase in the risk of bleeding).\\nLiterature:\\n1\\n. \\nLiu YP et al. Association of genetic variants in CYP2C19 and adverse clinical outcomes after treatment with clopidogrel: an\\nupdated meta-analysis. Thromb Res 2011;128:593-4.\\n2\\n. \\nLi Y et al. The gain-of-function variant allele CYP2C19*17: a double-edged sword between thrombosis and bleeding in\\nclopidogrel-treated patients. J Thromb Haemost 2012;10:199-206.\\n3\\n. \\nSimon T et al. Genetic polymorphisms and the impact of a higher clopidogrel dose regimen on active metabolite exposure and\\nantiplatelet response in healthy subjects. Clin Pharmacol Ther 2011;90:287-95.\\n4\\n. \\nShuldiner AR et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel\\ntherapy. JAMA 2009;302:849-57.\\n5\\n. \\nFrére C et al. The CYP2C19*17 allele is associated with better platelet response to clopidogrel in patients admitted for non-ST\\nacute coronary syndrome. J Thromb Haemost 2009;7:1409-11.\\n6\\n. \\nMega JL et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N Engl J Med 2009;360:354-62.\\n7\\n. \\nSimon T et al. Genetic determinants of response to clopidogrel and cardiovascular events. N Engl J Med 2009;360:363-75.\\n8\\n. \\nGeisler T et al. CYP2C19 and nongenetic factors predict poor responsiveness to clopidogrel loading dose after coronary stent\\nimplantation. Pharmacogenomics 2008;9:1251-9.\\nDate 19-11-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 32}),\n",
       " Document(page_content='CYP2D6 IM: clozapine\\n1530\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and\\nchlorpromazine in the Chinese Han population. Pharmacogenomics J 2015 Aug 18 [Epub ahead of print].\\n2\\n. \\nLee ST et al. Association study of 27 annotated genes for clozapine pharmacogenetic: validation of preexisting studies and\\nidentification of a new candidate gene, ABCB1, for treatment response. J Clin Psychopharmacol 2012; 32:441-8.\\n3\\n. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin\\nPsychopharmacol 2009;29:318-26.\\n4\\n. \\nMelkersson KI et al. Impact of CYP1A2 and CYP2D6 polymorphisms on drug metabolism and on insulin and lipid elevations and\\ninsulin resistance in clozapine-treated patients. J Clin Psychiatry 2007;68:697-704.\\n5\\n. \\nDettling M et al. Long-term therapeutic drug monitoring of clozapine and metabolites in psychiatric in- and outpatients.\\nPsychopharmacology 2000;152:80-86.\\nDate 25-05-2016\\nCYP2D6 PM: clozapine\\n1529\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nXu Q et al. Association studies of genomic variants with treatment response to risperidone, clozapine, quetiapine and\\nchlorpromazine in the Chinese Han population. Pharmacogenomics J 2015 Aug 18 [Epub ahead of print].\\n2\\n. \\nLee ST et al. Association study of 27 annotated genes for clozapine pharmacogenetic: validation of preexisting studies and\\nidentification of a new candidate gene, ABCB1, for treatment response. J Clin Psychopharmacol 2012; 32:441-8.\\n3\\n. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin\\nPsychopharmacol 2009;29:318-26.\\n4\\n. \\nMelkersson KI et al. Impact of CYP1A2 and CYP2D6 polymorphisms on drug metabolism and on insulin and lipid elevations and\\ninsulin resistance in clozapine-treated patients. J Clin Psychiatry 2007;68:697-704.\\n5\\n. \\nDettling M et al. Long-term therapeutic drug monitoring of clozapine and metabolites in psychiatric in- and outpatients.\\nPsychopharmacology 2000;152:80-86.\\n6\\n. \\nDettling M et al. Clozapine-induced agranulocytosis and hereditary polymorphisms of clozapine metabolizing enzymes: no\\nassociation with myeloperoxidase and cytochrome P4502D6. Pharmacopsychiatry 2000;33:218-20.\\n7\\n. \\nArranz MJ et al. Cytochrome P4502D6 genotype does not determine response to clozapine. Br J Clin Pharmacol 1995;39:417-20.\\n8\\n. \\nDahl ML et al. Disposition of clozapine in man: lack of association with debrisoquine and S-mephenytoin hydroxylation\\npolymorphisms. Br J Clin Pharmacol 1994;37:71-4.\\nDate 25-05-2016\\nCYP2D6 UM: clozapine\\n1531\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nJaquenoud Sirot E et al. ABCB1 and cytochrome P450 polymorphisms: clinical pharmacogenetics of clozapine. J Clin\\nPsychopharmacol 2009;29:318-26.\\n2\\n. \\nDettling M et al. Long-term therapeutic drug monitoring of clozapine and metabolites in psychiatric in- and outpatients.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 33}),\n",
       " Document(page_content='Psychopharmacology 2000;152:80-86.\\n3\\n. \\nDettling M et al. Clozapine-induced agranulocytosis and hereditary polymorphisms of clozapine metabolizing enzymes: no\\nassociation with myeloperoxidase and cytochrome P4502D6. Pharmacopsychiatry 2000;33:218-20.\\nDate 25-05-2016\\nCYP2D6 IM: codeine\\n1584\\nThe genetic variation reduces the conversion of codeine to morphine. This can result in reduced analgesia.\\nRecommendation:\\nFor COUGH:\\n1\\n. \\nno action required\\nFor PAIN:\\nIt is not possible to offer adequately substantiated advice for dose adjustment based on the limited available literature for this\\nphenotype.\\n1\\n. \\nbe alert to a reduced effectiveness\\n2\\n. \\nin the case of inadequate effectiveness:\\n1\\n. \\ntry a dose increase\\n2\\n. \\nif this does not work: choose an alternative\\nDo not select tramadol, as this is also metabolised by CYP2D6\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in\\npatients.\\n3\\n. \\nif no alternative is selected: advise the patient to report inadequate analgesia\\nLiterature:\\n1\\n. \\nBaber M et al. The pharmacogenetics of codeine pain relief in the postpartum period. Pharmacogenomics J 2015;15:430-5.\\n2\\n. \\nProws CA et al. Codeine-related adverse drug reactions in children following tonsillectomy: a prospective study. Laryngoscope\\n2014;124:1242-50.\\n3\\n. \\nKelly LE et al. A clinical tool for reducing central nervous system depression among neonates exposed to codeine through breast\\nmilk. PLoS One 2013;8:e70073.\\n4\\n. \\nVanderVaart S et al. CYP2D6 polymorphisms and codeine analgesia in postpartum pain management: a pilot study. Ther Drug\\nMonit 2011;33:425-32.\\n5\\n. \\nWilliams DG et al. Pharmacogenetics of codeine metabolism in an urban population of children and its implications for analgesic\\nreliability. Br J Anaesth 2002;89:839-45.\\n6\\n. \\nTseng CY et al. Formation of morphine from codeine in Chinese subjects of different CYP2D6 genotypes. Clin Pharmacol Ther\\n1996;60:177-82.\\n7\\n. \\nSPC Codeïnefosfaat Ratiopharm.\\nDate 20-11-2017\\nCYP2D6 PM: codeine\\n1583\\nThe genetic variation reduces the conversion of codeine to morphine. This can result in reduced analgesia.\\nRecommendation:\\nFor COUGH:\\n1\\n. \\nno action required\\nFor PAIN:\\nIt is not possible to offer adequately substantiated advice for dose adjustment based on the limited available literature for this\\nphenotype.\\n1\\n. \\nchoose an alternative\\nDo not select tramadol, as this is also metabolised by CYP2D6\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n2\\n. \\nif an alternative is not an option: advise the patient to report inadequate analgesia.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 34}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nBaber M et al. The pharmacogenetics of codeine pain relief in the postpartum period. Pharmacogenomics J 2015;15:430-5.\\n2\\n. \\nProws CA et al. Codeine-related adverse drug reactions in children following tonsillectomy: a prospective study. Laryngoscope\\n2014;124:1242-50.\\n3\\n. \\nKelly LE et al. A clinical tool for reducing central nervous system depression among neonates exposed to codeine through breast\\nmilk. PLoS One 2013;8:e70073.\\n4\\n. \\nSistonen J et al. Prediction of codeine toxicity in infants and their mothers using a novel combination of maternal genetic markers.\\nClin Pharmacol Ther 2012;91:692-9.\\n5\\n. \\nVanderVaart S et al. CYP2D6 polymorphisms and codeine analgesia in postpartum pain management: a pilot study. Ther Drug\\nMonit 2011;33:425-32.\\n6\\n. \\nKirchheiner J et al. Pharmacokinetics of codeine and its metabolite morphine in ultra-rapid metabolizers due to CYP2D6\\nduplication. Pharmacogenomics J 2007;7:257-65.\\n7\\n. \\nWilliams DG et al. Pharmacogenetics of codeine metabolism in an urban population of children and its implications for analgesic\\nreliability. Br J Anaesth 2002;89:839-45.\\n8\\n. \\nPoulsen L et al. Codeine in post-operative pain. Study of the influence of sparteine phenotype and serum concentrations of\\nmorphine and morphine-6-glucuronide. Eur J Clin Pharmacol 1998;54:451-4.\\n9\\n. \\nEckhardt K et al. Same incidence of adverse drug events after codeine administration irrespective of the genetically determined\\ndifferences in morphine formation. Pain 1998;76:27-33.\\n10\\n. \\nMikus G et al. Effect of codeine on gastrointestinal motility in relation to CYP2D6 phenotype. Clin Pharmacol Ther 1997;61:459-\\n66.\\n11\\n. \\nHasselstrom J et al. The effect of codeine on gastrointestinal transit in extensive and poor metabolisers of debrisoquine. Eur J Clin\\nPharmacol 1997;53:145-8.\\n12\\n. \\nPoulsen L et al. Codeine and morphine in extensive and poor metabolizers of sparteine: pharmacokinetics, analgesic effect and side\\neffects. Eur J Clin Pharmacol 1996;51:289-95.\\n13\\n. \\nPersson K et al. Patient-controlled analgesia (PCA) with codeine for postoperative pain relief in ten extensive metabolisers and\\none poor metaboliser of dextromethorphan. Br J Clin Pharmacol 1995;39:182-6.\\n14\\n. \\nYue QY et al. Pharmacokinetics of codeine and its metabolites in Caucasian healthy volunteers: comparisons between extensive\\nand poor hydroxylators of debrisoquine. Br J Clin Pharmacol 1991;31:635-42.\\n15\\n. \\nDesmeules J et al. Impact of environmental and genetic factors on codeine analgesia. Eur J Clin Pharmacol 1991;41:23-6.\\n16\\n. \\nSindrup SH et al. Codeine increases pain thresholds to copper vapor laser stimuli in extensive but not poor metabolizers of\\nsparteine. Clin Pharmacol Ther 1990;48:686-93.\\n17\\n. \\nSPC Codeïnefosfaat Ratiopharm.\\nDate 20-11-2017\\nCYP2D6 UM: codeine\\n1585\\nThe genetic variation increases the conversion of codeine to morphine. This can result in an increase in side effects. Death has occurred\\nin children who received analgesic doses. One adult with reduced kidney function and co-medication with two CYP3A4 inhibitors\\nbecame comatose after use of codeine for a cough.\\nRecommendation:\\nDOSES HIGHER THAN 20 mg every 6 hours for adults and 10 mg every 6 hours for children aged 12 years or older AND/OR\\nADDITIONAL RISK FACTORS, such as co-medication with CYP3A4 inhibitors and/or reduced kidney function:\\nCodeine is contra-indicated\\nif possible, select an alternative\\nFor PAIN: do not select tramadol, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6. Oxycodone is metabolised by CYP2D6 to a limited extent, but this does not\\nresult in differences in side effects in patients.\\nFor COUGH: noscapine is not metabolised by CYP2D6.\\nDOSES LOWER THAN OR EQUAL TO 20 mg every 6 hours for adults and 10 mg every 6 hours for children aged 12 years or\\nolder AND NO ADDITIONAL RISK FACTORS, such as co-medication with CYP3A4 inhibitors and/or reduced kidney function:\\nno action required\\nLiterature:\\n1\\n. \\nBaber M et al. The pharmacogenetics of codeine pain relief in the postpartum period. Pharmacogenomics J 2015;15:430-5.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 35}),\n",
       " Document(page_content='2\\n. \\nRay JG et al. Risk of overdose and death following codeine prescription among immigrants. J Epidemiol Community Health\\n2014;68:1057-63.\\n3\\n. \\nProws CA et al. Codeine-related adverse drug reactions in children following tonsillectomy: a prospective study. Laryngoscope\\n2014;124:1242-50.\\n4\\n. \\nKelly LE et al. A clinical tool for reducing central nervous system depression among neonates exposed to codeine through breast\\nmilk. PLoS One 2013;8:e70073.\\n5\\n. \\nKelly LE et al. More codeine fatalities after tonsillectomy in North American children. Pediatrics 2012;129:e1343-7.\\n6\\n. \\nSistonen J et al. Prediction of codeine toxicity in infants and their mothers using a novel combination of maternal genetic markers.\\nClin Pharmacol Ther 2012;91:692-9.\\n7\\n. \\nVanderVaart S et al. CYP2D6 polymorphisms and codeine analgesia in postpartum pain management: a pilot study. Ther Drug\\nMonit 2011;33:425-32.\\n8\\n. \\nCiszkowski C et al. Codeine, ultrarapid-metabolism genotype, and postoperative death. N Engl J Med 2009;361:827-8.\\n9\\n. \\nMadadi P et al. Pharmacogenetics of neonatal opioid toxicity following maternal use of codeine during breastfeeding: a case-\\ncontrol study. Clin Pharmacol Ther 2009;85:31-5.\\n10\\n. \\nKoren G et al. Pharmacogenetics of morphine poisoning in a breastfed neonate of a codeine-prescribed mother. Lancet\\n2006;368:704.\\n11\\n. \\nKirchheiner J et al. Pharmacokinetics of codeine and its metabolite morphine in ultra-rapid metabolizers due to CYP2D6\\nduplication. Pharmacogenomics J 2007;7:257-65.\\n12\\n. \\nGasche Y et al. Codeine intoxication associated with ultrarapid CYP2D6 metabolism. N Engl J Med 2004;351:2827-31.\\n13\\n. \\nDalen P et al. Quick onset of severe abdominal pain after codeine in an ultrarapid metabolizer of debrisoquine. Ther Drug Monit\\n1997;19:543-4.\\n14\\n. \\nEuropean Medicines Agency. Restrictions on use of codeine for pain relief in children – CMDh endorses PRAC recommendation.\\n28-06-13.\\n15\\n. \\nSPC Codeïnefosfaat Ratiopharm.\\n16\\n. \\nSPC Codeine Sulfate Tablets (VS).\\nDate 20-11-2017\\nCYP2D6 IM: disopyramide\\n2537\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 PM: disopyramide\\n2536\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 UM: disopyramide\\n2538\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 36}),\n",
       " Document(page_content='Literature:\\n-\\nDate 24-08-2016\\nCYP2D6 IM: doxepine\\n2015\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of doxepin and the active\\nmetabolite nordoxepin.\\nuse 80% of the standard dose and monitor the effect and side effects or the plasma concentrations of doxepin and nordoxepin in\\norder to set the maintenance dose\\nThe therapeutic range is 100-250 ng/mL for the sum of doxepin and nordoxepin plasma concentrations. Values higher than 400\\nng/mL are considered toxic.\\nLiterature:\\n1\\n. \\nKirchheiner J et al. Contributions of CYP2D6, CYP2C9 and CYP2C19 to the biotransformation of E- and Z-doxepin in healthy\\nvolunteers. Pharmacogenetics 2002;12:571-80.\\nDate 19-11-2018\\nCYP2D6 PM: doxepine\\n2016\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of doxepin and the active\\nmetabolite nordoxepin.\\nuse 40% of the standard dose and monitor the effect and side effects or the plasma concentrations of doxepin and nordoxepin in\\norder to set the maintenance dose\\nThe therapeutic range is 100-250 ng/mL for the sum of doxepin and nordoxepin plasma concentrations. Values higher than 400\\nng/mL are considered toxic.\\nLiterature:\\n1\\n. \\nKoski A et al. A fatal doxepin poisoning associated with a defective CYP2D6 genotype. Am J Forensic Med Pathol 2007;28:259-\\n61.\\n2\\n. \\nKirchheiner J et al. Impact of the CYP2D6 ultra-rapid metabolizer genotype on doxepin pharmacokinetics and serotonin in\\nplatelets. Pharmacogenet Genomics 2005;15:579-87.\\n3\\n. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and\\nclinical effects of antidepressants in a naturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n4\\n. \\nKirchheiner J et al. Contributions of CYP2D6, CYP2C9 and CYP2C19 to the biotransformation of E- and Z-doxepin in healthy\\nvolunteers. Pharmacogenetics 2002;12:571-80.\\n5\\n. \\nTacke U et al. Debrisoquine hydroxylation phenotypes of patients with high versus low to normal serum antidepressant\\nconcentrations. J Clin Psychopharmacol 1992;12:262-7.\\n6\\n. \\nSmPC Silenor (VS).\\nDate 19-11-2018\\nCYP2D6 UM: doxepine\\n2017\\nThe risk of ineffectiveness and cardiotoxic side effects may be increased. The gene variation leads to reduced plasma concentrations of', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 37}),\n",
       " Document(page_content='doxepin and the active metabolite nordoxepin and an increase in the plasma concentrations of the potentially cardiotoxic hydroxy\\nmetabolites.\\ndouble the standard dose and monitor the effect and side effects or the plasma concentrations of doxepin and nordoxepin in order\\nto set the maintenance dose\\nThe therapeutic range is 100-250 ng/mL for the sum of doxepin and nordoxepin plasma concentrations. Values higher than 400\\nng/mL are considered toxic.\\nif a dose increase is not wanted due to the potentially cardiotoxic hydroxy metabolites: avoid doxepin.\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include citalopram and sertraline.\\nLiterature:\\n1\\n. \\nKirchheiner J et al. Impact of the CYP2D6 ultra-rapid metabolizer genotype on doxepin pharmacokinetics and serotonin in\\nplatelets. Pharmacogenet Genomics 2005;15:579-87.\\nDate 19-11-2018\\nCYP2D6 IM: duloxetine\\n1673\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nKamei S et al. Rapid onset of syndrome of inappropriate antidiuretic hormone secretion induced by duloxetine in an elderly type 2\\ndiabetic patient with painful diabetic neuropathy. J Diabetes Investig 2015;6:343-5.\\n2\\n. \\nLobo ED et al. Pharmacokinetics of orally administered duloxetine in children and adolescents with major depressive disorder.\\nClin Pharmacokinet 2014;53:731-40.\\n3\\n. \\nBeatty NC et al. Pharmacogenetic workup of perioperative serotonin syndrome. J Clin Anesth 2013;25:662-5.\\n4\\n. \\nTianmei S et al. Pharmacokinetics and tolerability of duloxetine following oral administration to healthy Chinese subjects. Clin\\nPharmacokinet 2007;46:767-75.\\nDate 30-01-2017\\nCYP2D6 PM: duloxetine\\n1674\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nLobo ED et al. Pharmacokinetics of orally administered duloxetine in children and adolescents with major depressive disorder.\\nClin Pharmacokinet 2014;53:731-40.\\n2\\n. \\nChan C et al. Duloxetine pharmacokinetics are similar in Japanese and Caucasian subjects. Br J Clin Pharmacol 2007;63:310-4.\\n3\\n. \\nSPC Cymbalta.\\nDate 30-01-2017\\nCYP2D6 UM: duloxetine\\n1675\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 38}),\n",
       " Document(page_content=\"Literature:\\n1\\n. \\nLobo ED et al. Pharmacokinetics of orally administered duloxetine in children and adolescents with major depressive disorder.\\nClin Pharmacokinet 2014;53:731-40.\\nDate 30-01-2017\\nCYP2B6 IM: efavirenz\\n4754\\nGenetic variations increase the efavirenz plasma concentration and therefore the risk of side effects. However, the efavirenz plasma\\nconcentration remains within the therapeutic range for the majority of patients.\\nRecommendation:\\n1\\n. \\nDetermine the efavirenz plasma concentration if side effects occur and reduce the dose if needed.\\nIn 14 IM adults, a dose reduction to 400 mg/day (2/3rd of the standard dose) was sufficient to achieve therapeutic plasma\\nconcentrations and to reduce or resolve side effects.\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nLiterature:\\n1\\n. \\nVujkovic M et al. CYP2B6 516G>T minor allele protective of late virologic failure in efavirenz-treated HIV-infected patients in\\nBotswana. J Acquir Immune Defic Syndr 2017 May 5 [Epub ahead of print].\\n2\\n. \\nLeger P et al. Pharmacogenetics of efavirenz discontinuation for reported central nervous system symptoms appears to differ by\\nrace. Pharmacogenet Genomics 2016;26:473-80.\\n3\\n. \\nCusato J et al. Efavirenz pharmacogenetics in a cohort of Italian patients. Int J Antimicrob Agents 2016;47:117-23.\\n4\\n. \\nDickinson L et al. Comprehensive pharmacokinetic, pharmacodynamic and pharmacogenetic evaluation of once-daily efavirenz\\n400 and 600 mg in treatment-naïve HIV-infected patients at 96 weeks: results of the ENCORE1 study. Clin Pharmacokinet\\n2016;55:861-73.\\n5\\n. \\nSwart M et al. An expanded analysis of pharmacogenetics determinants of efavirenz response that includes 3'-UTR single\\nnucleotide polymorphisms among black South African HIV/AIDS patients. Front Genet 2016;6:356.\\n6\\n. \\nMeng X et al. Effect of CYP2B6 gene polymorphisms on efavirenz plasma concentrations in Chinese patients with HIV infection.\\nPLoS One 2015;10:e0130583.\\n7\\n. \\nHaas DW et al. Functional CYP2B6 variants and virologic response to an efavirenz-containing regimen in Port-au-Prince, Haiti. J\\nAntimicrob Chemother 2014;69:2187-90.\\n8\\n. \\nMartín AS et al. Dose reduction of efavirenz: an observational study describing cost-effectiveness, pharmacokinetics and\\npharmacogenetics. Pharmacogenomics 2014;15:997-1006.\\n9\\n. \\nBertrand J et al. Dependence of efavirenz- and rifampicin-isoniazid-based antituberculosis treatment drug-drug interaction on\\nCYP2B6 and NAT2 genetic polymorphisms: ANRS 12154 study in Cambodia. J Infect Dis 2014;209:399-408.\\n10\\n. \\nSarfo FS et al. Pharmacogenetic associations with plasma efavirenz concentrations and clinical correlates in a retrospective cohort\\nof Ghanaian HIV-infected patients. J Antimicrob Chemother 2014;69:491-9. PubMed PMID: 24080498.\\n11\\n. \\nNgaimisi E et al. Importance of ethnicity, CYP2B6 and ABCB1 genotype for efavirenz pharmacokinetics and treatment outcomes:\\na parallel-group prospective cohort study in two sub-Saharan Africa populations. PLoS One 2013;8:e67946. PubMed PMID:\\n23861838.\\n12\\n. \\nYimer G et al. High plasma efavirenz level and CYP2B6*6 are associated with efavirenz-based HAART-induced liver injury in\\nthe treatment of naïve HIV patients from Ethiopia: a prospective cohort study. Pharmacogenomics J 2012;12:499-506. PubMed\\nPMID: 21862974.\\n13\\n. \\nMugusi S et al. Liver enzyme abnormalities and associated risk factors in HIV patients on efavirenz-based HAART with or\\nwithout tuberculosis co-infection in Tanzania. PLoS One 2012;7:e40180. PubMed PMID: 22808112.\\n14\\n. \\nWyen C et al. Cytochrome P450 2B6 (CYP2B6) and constitutive androstane receptor (CAR) polymorphisms are associated with\\nearly discontinuation of efavirenz-containing regimens. J Antimicrob Chemother 2011;66:2092-8. PubMed PMID: 21715435.\\n15\\n. \\nCarr DF et al. Haplotype structure of CYP2B6 and association with plasma efavirenz concentrations in a Chilean HIV cohort. J\\nAntimicrob Chemother 2010;65:1889-93. PubMed PMID: 20639527.\\n16\\n. \\nGatanaga H et al. Successful efavirenz dose reduction in HIV type 1-infected individuals with cytochrome P450 2B6 \\n6 and \\n26.\\nClin Infect Dis 2007;45:1230-7. PubMed PMID: 17918089.\\n17\\n. \\nHaas DW et al. Pharmacogenetics of long-term responses to antiretroviral regimens containing efavirenz and/or nelfinavir: an\\nAdult Aids Clinical Trials Group study. J Infect Dis 2005;192:1931-42. PubMed PMID: 16267764.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 39}),\n",
       " Document(page_content=\"Date 05-03-2018\\nCYP2B6 PM: efavirenz\\n4755\\nGenetic variations increase the risk of side effects. The standard dose leads to an efavirenz concentration in the toxic range in the\\nmajority of patients with this genotype.\\nRecommendation:\\nEfavirenz in MONOpreparation, adults and children FROM 40 KG:\\nBody mass index LESS THAN or EQUAL to 25:\\n1\\n. \\nThe recommended initial dose is 400 mg/day and this dose should be titrated to plasma concentration if needed\\n(further reduction to 200 mg/day or in rare cases an increase to 600 mg/day).\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nBody mass index GREATER than 25:\\n1\\n. \\nThe recommended initial dose is 600 mg/day and this dose should be titrated to plasma concentration if needed\\n(reduction to 400 or 200 mg/day).\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nEfavirenz in MONOpreparation, children LIGHTER THAN 40 KG:\\n1\\n. \\nStart with the standard dose and titrate this dose to plasma concentration if needed. In adults, therapeutic plasma\\nconcentrations were achieved at either 2/3rd of the standard dose (1/3rd of the patients) or 1/3rd of the standard dose (2/3rd\\nof the patients). In children younger than 3 years, therapeutic plasma concentrations were achieved at doses of\\napproximately 10 mg/kg per day (as capsules) (100 mg/day for 7-14 kg and 150 mg/day for 14-17 kg; 50-75% of the\\nstandard dose).\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nEfavirenz in COMBINATION preparation:\\n1\\n. \\nInitiate the combination preparation and titrate the efavirenz dose to plasma concentration if needed (reduction to 400 or 200\\nmg/day)\\nThe therapeutic range established for efavirenz is 1000-4000 ng/ml.\\nNote: the dosing recommendations above are based on PM patients with the *6/*6 genotype. There is evidence that the *18/*18\\ngenotype in PM patients (only present in negroid patients) may require greater dose reductions.\\nConsiderations:\\nDetailed justification for the recommendation is contained in the risk analysis. The considerations used for adults are also given below.\\nThe median or mean plasma concentrations or AUC in PM patients are above the therapeutic range, except in 3 studies with low\\nefavirenz plasma concentrations in EM patients (2 of the 3 studies performed in Africa and 1 study in the United States and Italy). A\\nrecent study showed a similar virological response for efavirenz 400 and 600 mg/day in patients not selected on genotype. The risk of\\nunderdose is therefore very small if the initial dose is reduced to 400 mg/day. Two small studies showed that dose reductions did not\\nreduce the efficacy (HIV remained undetectable), but side effects did reduce in 24 PM patients.\\nCompliance improves with administration of a combination preparation and the absence of unnecessary side effects due to excessive\\nplasma concentrations.\\nConsideration to CYP2B6 inducers such as rifampicin is not needed in PM patients. The significantly low or absent metabolic capacity of\\nCYP2B6 makes induction of little to no relevance. Moreover, the effects of enzyme induction by rifampicin and enzyme inhibition by\\nisoniazid on efavirenz plasma concentrations seem to largely cancel each other out, independent of the CYP2B6 phenotype of the patient.\\nLiterature:\\n1\\n. \\nVujkovic M et al. CYP2B6 516G>T minor allele protective of late virologic failure in efavirenz-treated HIV-infected patients in\\nBotswana. J Acquir Immune Defic Syndr 2017 May 5 [Epub ahead of print].\\n2\\n. \\nBolton Moore C et al. CYP2B6 genotype-directed dosing is required for optimal efavirenz exposure in children 3-36 months with\\nHIV infection. AIDS 2017;31:1129-1136.\\n3\\n. \\nLeger P et al. Pharmacogenetics of efavirenz discontinuation for reported central nervous system symptoms appears to differ by\\nrace. Pharmacogenet Genomics 2016;26:473-80.\\n4\\n. \\nCusato J et al. Efavirenz pharmacogenetics in a cohort of Italian patients. Int J Antimicrob Agents 2016;47:117-23.\\n5\\n. \\nDickinson L et al. Comprehensive pharmacokinetic, pharmacodynamic and pharmacogenetic evaluation of once-daily efavirenz\\n400 and 600 mg in treatment-naïve HIV-infected patients at 96 weeks: results of the ENCORE1 study. Clin Pharmacokinet\\n2016;55:861-73.\\n6\\n. \\nSwart M et al. An expanded analysis of pharmacogenetics determinants of efavirenz response that includes 3'-UTR single\\nnucleotide polymorphisms among black South African HIV/AIDS patients. Front Genet 2016;6:356.\\n7\\n. \\nMeng X et al. Effect of CYP2B6 gene polymorphisms on efavirenz plasma concentrations in Chinese patients with HIV infection.\\nPLoS One 2015;10:e0130583.\\n8\\n. \\nHaas DW et al. Functional CYP2B6 variants and virologic response to an efavirenz-containing regimen in Port-au-Prince, Haiti. J\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 40}),\n",
       " Document(page_content='Antimicrob Chemother 2014;69:2187-90.\\n9\\n. \\nMartín AS et al. Dose reduction of efavirenz: an observational study describing cost-effectiveness, pharmacokinetics and\\npharmacogenetics. Pharmacogenomics 2014;15:997-1006.\\n10\\n. \\nBertrand J et al. Dependence of efavirenz- and rifampicin-isoniazid-based antituberculosis treatment drug-drug interaction on\\nCYP2B6 and NAT2 genetic polymorphisms: ANRS 12154 study in Cambodia. J Infect Dis 2014;209:399-408.\\n11\\n. \\nSarfo FS et al. Pharmacogenetic associations with plasma efavirenz concentrations and clinical correlates in a retrospective cohort\\nof Ghanaian HIV-infected patients. J Antimicrob Chemother 2014;69:491-9. PubMed PMID: 24080498.\\n12\\n. \\nNgaimisi E et al. Importance of ethnicity, CYP2B6 and ABCB1 genotype for efavirenz pharmacokinetics and treatment outcomes:\\na parallel-group prospective cohort study in two sub-Saharan Africa populations. PLoS One 2013;8:e67946. PubMed PMID:\\n23861838.\\n13\\n. \\nYimer G et al. High plasma efavirenz level and CYP2B6*6 are associated with efavirenz-based HAART-induced liver injury in\\nthe treatment of naïve HIV patients from Ethiopia: a prospective cohort study. Pharmacogenomics J 2012;12:499-506. PubMed\\nPMID: 21862974.\\n14\\n. \\nMugusi S et al. Liver enzyme abnormalities and associated risk factors in HIV patients on efavirenz-based HAART with or\\nwithout tuberculosis co-infection in Tanzania. PLoS One 2012;7:e40180. PubMed PMID: 22808112.\\n15\\n. \\nWyen C et al. Cytochrome P450 2B6 (CYP2B6) and constitutive androstane receptor (CAR) polymorphisms are associated with\\nearly discontinuation of efavirenz-containing regimens. J Antimicrob Chemother 2011;66:2092-8. PubMed PMID: 21715435.\\n16\\n. \\nRibaudo HJ et al. Effect of CYP2B6, ABCB1, and CYP3A5 polymorphisms on efavirenz pharmacokinetics and treatment\\nresponse: an AIDS Clinical Trials Group study. J Infect Dis 2010;202:717-22. PubMed PMID: 20662624.\\n17\\n. \\nCarr DF et al. Haplotype structure of CYP2B6 and association with plasma efavirenz concentrations in a Chilean HIV cohort. J\\nAntimicrob Chemother 2010;65:1889-93. PubMed PMID: 20639527.\\n18\\n. \\nGatanaga H et al. Successful efavirenz dose reduction in HIV type 1-infected individuals with cytochrome P450 2B6 *6 and *26.\\nClin Infect Dis 2007;45:1230-7. PubMed PMID: 17918089.\\n19\\n. \\nHaas DW et al. Pharmacogenetics of long-term responses to antiretroviral regimens containing efavirenz and/or nelfinavir: an\\nAdult Aids Clinical Trials Group study. J Infect Dis 2005;192:1931-42. PubMed PMID: 16267764.\\n20\\n. \\nSPC’s Efavirenz Mylan en Sustiva (VS).\\n21\\n. \\nENCORE1 Study Group. Efficacy of 400 mg efavirenz versus standard 600 mg dose in HIV-infected, antiretroviral-naive adults\\n(ENCORE1): a randomised, double-blind, placebo-controlled, non-inferiority trial. Lancet 2014;383:1474-82.\\nDate 05-03-2018\\nCYP2D6 IM: eliglustat\\n6138\\nThis gene variation reduces the conversion of eliglustat to inactive metabolites. However, in the absence of CYP2D6 and CYP3A\\ninhibitors, this does not result in a clinically significant increased risk of side effects.\\nRecommendation:\\nCo-medication with BOTH a MODERATE to STRONG CYP2D6 INHIBITOR AND a MODERATE to STRONG CYP3A\\nINHIBITOR:\\nEliglustat is contra-indicated.\\n1\\n. \\nchoose an alternative if possible\\nStrong CYP2D6 inhibitor: for example paroxetine, fluoxetine, quinidine, bupropione.\\nModerate CYP2D6 inhibitor: for example duloxetine, terbinafine, moclobemide, mirabegron, cinacalcet, dronedarone.\\nStrong CYP3A inhibitor: for example ketoconazole, clarithromycin, itraconazole, cobicistat, indinavir, lopinavir, ritonavir,\\nsaquinavir, telaprevir, tipranavir, posaconazole, voriconazole, telithromycin, conivaptan, boceprevir.\\nModerate CYP3A inhibitor: for example erythromycin, ciprofloxacin, fluconazole, diltiazem, verapamil, aprepitant,\\natazanavir, darunavir, fosamprenavir, imatinib, cimetidine.\\nCo-medication with a STRONG CYP2D6 INHIBITOR (e.g. paroxetine, fluoxetine, quinidine, bupropione):\\n1\\n. \\nuse a dose of 84 mg eliglustat 1x daily\\nCo-medication with a MODERATE CYP2D6 INHIBITOR (for example duloxetine, terbinafine, moclobemide, mirabegron,\\ncinacalcet, dronedarone):\\n1\\n. \\nconsider a dose of 84 mg eliglustat 1x daily\\n2\\n. \\nbe alert to side effects\\nCo-medication with a STRONG CYP3A INHIBITOR (for example ketoconazole, clarithromycin, itraconazole, cobicistat,\\nindinavir, lopinavir, ritonavir, saquinavir, telaprevir, tipranavir, posaconazole, voriconazole, telithromycin, conivaptan,\\nboceprevir):\\nchoose an alternative if possible\\nif an alternative is not an option:\\nconsider a dose of 84 mg eliglustat 1x daily\\nbe alert to side effects\\nCo-medication with a MODERATE CYP3A INHIBITOR (for example erythromycin, ciprofloxacin, fluconazole, diltiazem,\\nverapamil, aprepitant, atazanavir, darunavir, fosamprenavir, imatinib, cimetidine):\\n1\\n. \\nchoose an alternative\\n2\\n. \\nif an alternative is not an option:\\n1\\n. \\nconsider a dose of 84 mg eliglustat 1x daily', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 41}),\n",
       " Document(page_content='2\\n. \\nbe alert to side effects\\nCo-medication with a STRONG CYP3A INDUCER (for example rifampicin, carbamazepine, phenobarbital, phenytoin, rifabutine,\\nhypericum):\\nEliglustat is not recommended. The plasma concentration may decrease so sharply that a therapeutic effect cannot be achieved.\\n1\\n. \\nchoose an alternative if possible\\nNO co-medication with a moderate or strong CYP2D6 or CYP3A inhibitor or strong CYP3A inducer:\\n1\\n. \\nuse the standard dose of 84 mg 2x daily\\nLiterature:\\n1. SPC’s Cerdelga (Nederland en VS).\\nDate 31-10-2016\\nCYP2D6 PM: eliglustat\\n6137\\nThis gene variation reduces the conversion of eliglustat to inactive metabolites. This increases the risk of side effects, such as a (small,\\ndose-dependent) elongation of the QT interval. CYP3A inhibitors increase this risk even further.\\nRecommendation:\\nCo-medication with a STRONG CYP3A INHIBITOR (for example ketoconazole, clarithromycin, itraconazole, cobicistat,\\nindinavir, lopinavir, ritonavir, saquinavir, telaprevir, tipranavir, posaconazole, voriconazole, telithromycin, conivaptan,\\nboceprevir):\\nEliglustat is contra-indicated.\\n1\\n. \\nchoose an alternative if possible\\nCo-medication with a MODERATE CYP3A INHIBITOR (for example erythromycin, ciprofloxacin, fluconazole, diltiazem,\\nverapamil, aprepitant, atazanavir, darunavir, fosamprenavir, imatinib, cimetidine):\\nEliglustat is not recommended.\\n1\\n. \\nchoose an alternative if possible\\nCo-medication with a WEAK CYP3A INHIBITOR (for example amlopidine, cilostazole, fluvoxamine, goldenseal, isoniazide,\\nranitidine, ranolazine):\\n1\\n. \\nchoose an alternative for the weak CYP3A inhibitor if possible\\n2\\n. \\nif an alternative is not an option:\\n1\\n. \\nuse a dose of 84 mg eliglustat 1x daily\\n2\\n. \\nbe alert to side effects\\nCo-medication with a STRONG CYP3A INDUCER (for example rifampicin, carbamazepine, phenobarbital, phenytoin, rifabutine,\\nhypericum):\\nEliglustat is not recommended. The plasma concentration may decrease so sharply that a therapeutic effect cannot be achieved.\\n1\\n. \\nchoose an alternative if possible\\nNO co-medication with a CYP3A inhibitor or strong CYP3A inducer:\\n1\\n. \\nuse a dose of 84 mg 1x daily\\nLiterature:\\n1. SPC’s Cerdelga (Nederland en VS).\\nDate 31-10-2016\\nCYP2D6 UM: eliglustat\\n6139\\nThis gene variation increases the conversion of eliglustat to inactive metabolites. As a result, a normal dose is not effective. There is not\\nenough scientific substantiation to suggest an effective dose for all UM.\\nRecommendation:\\nEliglustat is contra-indicated.\\n1. choose an alternative if possible', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 42}),\n",
       " Document(page_content='Literature:\\n1. SPC’s Cerdelga (Nederland en VS).\\nDate 31-10-2016\\nCYP2C19 IM: escitalopram\\n1821\\nThe risk of QT prolongation and torsades de pointes is theoretically increased because the gene variation leads to an increased\\nescitalopram plasma concentration. If you follow the dose recommendation below, the increased plasma concentration and the\\ntheoretically increased risk of QT prolongation will be offset.\\nDo not exceed the following doses (75% of the standard maximum dose):\\nadults < 65 years 15 mg/day, ≥65 years 7.5 mg/day\\nLiterature:\\n1\\n. \\nTsuchimine S et al. Effects of cytochrome P450 (CYP) 2C19 genotypes on steady-state plasma concentrations of escitalopram and\\nits desmethyl metabolite in Japanese patients with depression. Ther Drug Monit 2018 Mar 22 [Epub ahead of print].\\n2\\n. \\nJukić MM et al. Impact of CYP2C19 genotype on escitalopram exposure and therapeutic failure: a retrospective study based on\\n2,087 patients. Am J Psychiatry 2018 Jan 12 [Epub ahead of print].\\n3\\n. \\nHe Q et al. Correlation between cytochrome P450 2C19 genetic polymorphism and treatment response to escitalopram in panic\\ndisorder. Pharmacogenet Genomics 2017;27:279-284.\\n4\\n. \\nBishop JR et al. Escitalopram pharmacogenetics: CYP2C19 relationships with dosing and clinical outcomes in autism spectrum\\ndisorder. Pharmacogenet Genomics 2015:548-54.\\n5\\n. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl)\\n2015;232: 2609-17.\\n6\\n. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol\\n2014;28:1143-8.\\n7\\n. \\nWaade RB et al. Impact of age on serum concentrations of venlafaxine and escitalopram in different CYP2D6 and CYP2C19\\ngenotype subgroups. Eur J Clin Pharmacol 2014;70:933-40.\\n8\\n. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol\\n2014;28:133-41.\\n9\\n. \\nNg C et al. Pharmacogenetic polymorphisms and response to escitalopram and venlafaxine over 8 weeks in major depression.\\nHum Psychopharmacol 2013;28:516-22.\\n10\\n. \\nHuezo-Diaz et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol\\n2012;26:398-407.\\n11\\n. \\nBrasch-Andersen C et al. A candidate gene study of serotonergic pathway genes and pain relief during treatment with escitalopram\\nin patients with neuropathic pain shows significant association to serotonin receptor2C (HTR2C). Eur J Clin Pharmacol 2011;\\n67:1131-7.\\n12\\n. \\nTsai et al. Genetic polymorphisms of cytochrome P450 enzymes influence metabolism of the antidepressant escitalopram and\\ntreatment response. Pharmacogenomics 2010:11;537-46.\\n13\\n. \\nJin Y et al. Effect of age, weight, and CYP2C19 genotype on escitalopram exposure. J Clin Pharmacol. 2010;50:62-72.\\n14\\n. \\nRudberg I et al. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Clin\\nPharmacol Ther 2008;83:322-7.\\n15\\n. \\nRudberg I et al. Heterozygous mutation in CYP2C19 significantly increases the concentrations/dose ratio of racemic citalopram\\nand escitalopram (S-citalopram). Ther Drug Monitor 2006;28:102-5.\\nDate 14-05-2018\\nCYP2C19 PM: escitalopram\\n1822\\nThe risk of conversion to another antidepressant is increased. In addition, the risk of QT prolongation and torsades de pointes is\\ntheoretically increased because the gene variation leads to an increased escitalopram plasma concentration. If you follow the dose\\nrecommendation below, the increased plasma concentration, the theoretically increased risk of QT prolongation and the increased risk of\\nconversion to another antidepressant will be offset.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 43}),\n",
       " Document(page_content='Do not exceed the following doses (50% of the standard maximum dose):\\nadults < 65 years 10 mg/day, ≥65 years 5 mg/day\\nLiterature:\\n1\\n. \\nTsuchimine S et al. Effects of cytochrome P450 (CYP) 2C19 genotypes on steady-state plasma concentrations of escitalopram and\\nits desmethyl metabolite in Japanese patients with depression. Ther Drug Monit 2018 Mar 22 [Epub ahead of print].\\n2\\n. \\nJukić MM et al. Impact of CYP2C19 genotype on escitalopram exposure and therapeutic failure: a retrospective study based on\\n2,087 patients. Am J Psychiatry 2018 Jan 12 [Epub ahead of print].\\n3\\n. \\nHe Q et al. Correlation between cytochrome P450 2C19 genetic polymorphism and treatment response to escitalopram in panic\\ndisorder. Pharmacogenet Genomics 2017;27:279-284.\\n4\\n. \\nAsakura S et al. Long-term administration of escitalopram in patients with social anxiety disorder in Japan. Neuropsychiatr Dis\\nTreat 2016;12:1817-25.\\n5\\n. \\nBishop JR et al. Escitalopram pharmacogenetics: CYP2C19 relationships with dosing and clinical outcomes in autism spectrum\\ndisorder. Pharmacogenet Genomics 2015:548-54.\\n6\\n. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl)\\n2015;232: 2609-17.\\n7\\n. \\nKumar Y et al. CYP2C19 variation, not citalopram dose nor serum level, is associated with QTc prolongation. J Psychopharmacol\\n2014;28:1143-8.\\n8\\n. \\nWaade RB et al. Impact of age on serum concentrations of venlafaxine and escitalopram in different CYP2D6 and CYP2C19\\ngenotype subgroups. Eur J Clin Pharmacol 2014;70:933-40.\\n9\\n. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol\\n2014;28:133-41.\\n10\\n. \\nNg C et al. Pharmacogenetic polymorphisms and response to escitalopram and venlafaxine over 8 weeks in major depression.\\nHum Psychopharmacol 2013;28:516-22.\\n11\\n. \\nHuezo-Diaz et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol\\n2012;26:398-407.\\n12\\n. \\nBrasch-Andersen C et al. A candidate gene study of serotonergic pathway genes and pain relief during treatment with escitalopram\\nin patients with neuropathic pain shows significant association to serotonin receptor2C (HTR2C). Eur J Clin Pharmacol 2011;\\n67:1131-7.\\n13\\n. \\nTsai et al. Genetic polymorphisms of cytochrome P450 enzymes influence metabolism of the antidepressant escitalopram and\\ntreatment response. Pharmacogenomics 2010:11;537-46.\\n14\\n. \\nNoehr-Jensen et al. Impact of CYP2C19 phenotypes on escitalopram metabolism and an evaluation of pupillometry as a\\nserotonergic biomarker. Eur J Clin Pharmacol 2009;65:887-94.\\n15\\n. \\nJin Y et al. Effect of age, weight, and CYP2C19 genotype on escitalopram exposure. J Clin Pharmacol. 2010;50:62-72.\\n16\\n. \\nRudberg I et al. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Clin\\nPharmacol Ther 2008;83:322-7.\\n17\\n. \\nSPC’s Lexapro (NL en VS).\\nDate 14-05-2018\\nCYP2C19 UM: escitalopram\\n1820\\nThe risk of conversion to another antidepressant is increased as the gene variation leads to a reduction in the escitalopram plasma\\nconcentration.\\navoid escitalopram\\nAntidepressants that are not metabolised or that are metabolised to a lesser extent by CYP2C19 are, for example, paroxetine or\\nfluvoxamine.\\nLiterature:\\n1\\n. \\nJukić MM et al. Impact of CYP2C19 genotype on escitalopram exposure and therapeutic failure: a retrospective study based on\\n2,087 patients. Am J Psychiatry 2018 Jan 12 [Epub ahead of print].\\n2\\n. \\nBishop JR et al. Escitalopram pharmacogenetics: CYP2C19 relationships with dosing and clinical outcomes in autism spectrum\\ndisorder. Pharmacogenet Genomics 2015:548-54.\\n3\\n. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl)\\n2015;232: 2609-17.\\n4\\n. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 44}),\n",
       " Document(page_content='2014;28:133-41.\\n5\\n. \\nHuezo-Diaz et al. CYP2C19 genotype predicts steady state escitalopram concentration in GENDEP. J Psychopharmacol\\n2012;26:398-407.\\n6\\n. \\nBrasch-Andersen C et al. A candidate gene study of serotonergic pathway genes and pain relief during treatment with escitalopram\\nin patients with neuropathic pain shows significant association to serotonin receptor2C (HTR2C). Eur J Clin Pharmacol 2011;\\n67:1131-7.\\n7\\n. \\nJin Y et al. Effect of age, weight, and CYP2C19 genotype on escitalopram exposure. J Clin Pharmacol. 2010;50:62-72.\\n8\\n. \\nOhlsson Rosenborg S et al. Kinetics of omeprazole and escitalopram in relation to the CYP2C19*17 allele in healthy subjects. Eur\\nJ Clin Pharmacol 2008 Jul 25. 4;1175-79.\\n9\\n. \\nRudberg I et al. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Clin\\nPharmacol Ther 2008;83:322-7.\\nDate 14-05-2018\\nCYP2C19 IM: esomeprazol\\n1824\\nNO action is needed for this gene-drug interaction.\\nAlthough the genetic variation leads to a higher plasma concentration of esomeprazole, there is insufficient evidence to support an effect\\non the therapeutic effectiveness and side effects.\\nLiterature:\\n1\\n. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with\\nproton pump inhibitors. Meta Gene 2016;9:159-64.\\n2\\n. \\nHsu WH et al. Genetic polymorphisms of CYP2C19 and IL1B have no influence on esomeprazole treatment for mild erosive\\nesophagitis. Kaohsiung J Med Sci 2015;31:255-9.\\n3\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n4\\n. \\nLee VW et al. Pharmacogenetics of esomeprazole or rabeprazole-based triple therapy in Helicobacter pylori eradication in Hong\\nKong non-ulcer dyspepsia Chinese subjects. J Clin Pharm Ther 2010;35:343-50.\\n5\\n. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics\\nand CYP2C19 polymorphism. Aliment Pharmacol Ther 2010;31:150-9.\\n6\\n. \\nLou HY et al. Optimal dose regimens of esomeprazole for gastric acid suppression with minimal influence of the CYP2C19\\npolymorphism. Eur J Clin Pharmacol 2009;65:55-64.\\n7\\n. \\nSheu BS et al. Body mass index can determine the healing of reflux esophagitis with Los Angeles Grades C and D by\\nesomeprazole. Am J Gastroenterol 2008;103:2209-14.\\n8\\n. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J\\nGastroenterol Hepatol 2007;22:815-20.\\n9\\n. \\nSchwab M et al. Esomeprazole-induced healing of gastroesophageal reflux disease is unrelated to the genotype of CYP2C19:\\nevidence from clinical and pharmacokinetic data. Clin Pharmacol Ther 2005;78:627-34.\\n10\\n. \\nKuo CH et al. Efficacy of levofloxacin-based rescue therapy for Helicobacter pylori infection after standard triple therapy: a\\nrandomized controlled trial. J Antimicrob Chemother 2009;63:1017-24.\\n11\\n. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple\\ntherapy with regular proton pump inhibitor dosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n12\\n. \\nMiehlke S et al. One-week once-daily triple therapy with esomeprazole, moxifloxacin, and rifabutin for eradication of persistent\\nHelicobacter pylori resistant to both metronidazole and clarithromycin. Helicobacter 2008;13:69-74.\\n13\\n. \\nMiehlke S et al. Randomized trial of rifabutin-based triple therapy and high-dose dual therapy for rescue treatment of Helicobacter\\npylori resistant to both metronidazole and clarithromycin. Aliment Pharmacol Ther 2006;24:395-403.\\n14\\n. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to\\nCYP2C19 metabolism. Aliment Pharmacol Ther 2005;21:283-8.\\nDate 05-03-2018\\nCYP2C19 PM: esomeprazol\\n1825\\nNO action is needed for this gene-drug interaction.\\nAlthough the genetic variation leads to a higher plasma concentration of esomeprazole, there is insufficient evidence to support an effect', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 45}),\n",
       " Document(page_content='on the therapeutic effectiveness and side effects.\\nLiterature:\\n1\\n. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with\\nproton pump inhibitors. Meta Gene 2016;9:159-64.\\n2\\n. \\nHsu WH et al. Genetic polymorphisms of CYP2C19 and IL1B have no influence on esomeprazole treatment for mild erosive\\nesophagitis. Kaohsiung J Med Sci 2015;31:255-9.\\n3\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n4\\n. \\nLee VW et al. Pharmacogenetics of esomeprazole or rabeprazole-based triple therapy in Helicobacter pylori eradication in Hong\\nKong non-ulcer dyspepsia Chinese subjects. J Clin Pharm Ther 2010;35:343-50.\\n5\\n. \\nLou HY et al. Optimal dose regimens of esomeprazole for gastric acid suppression with minimal influence of the CYP2C19\\npolymorphism. Eur J Clin Pharmacol 2009;65:55-64.\\n6\\n. \\nSheu BS et al. Body mass index can determine the healing of reflux esophagitis with Los Angeles Grades C and D by\\nesomeprazole. Am J Gastroenterol 2008;103:2209-14.\\n7\\n. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J\\nGastroenterol Hepatol 2007;22:815-20.\\n8\\n. \\nSchwab M et al. Esomeprazole-induced healing of gastroesophageal reflux disease is unrelated to he genotype of CYP2C19:\\nevidence from clinical and pharmacokinetic data. Clin Pharmacol Ther 2005;78:627-34.\\n9\\n. \\nKuo CH et al. Efficacy of levofloxacin-based rescue therapy for Helicobacter pylori infection after standard triple therapy: a\\nrandomized controlled trial. J Antimicrob Chemother 2009;63:1017-24.\\n10\\n. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple\\ntherapy with regular proton pump inhibitor dosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n11\\n. \\nMiehlke S et al. One-week once-daily triple therapy with esomeprazole, moxifloxacin, and rifabutin for eradication of persistent\\nHelicobacter pylori resistant to both metronidazole and clarithromycin. Helicobacter 2008;13:69-74.\\n12\\n. \\nMiehlke S et al. Randomized trial of rifabutin-based triple therapy and high-dose dual therapy for rescue treatment of Helicobacter\\npylori resistant to both metronidazole and clarithromycin. Aliment Pharmacol Ther 2006;24:395-403.\\n13\\n. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to\\nCYP2C19 metabolism. Aliment Pharmacol Ther 2005;21:283-8.\\n14\\n. \\nSPC Nexium (Nederlands en Amerikaans).\\nDate 05-03-2018\\nCYP2C19 UM: esomeprazol\\n1826\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation may lead to faster inactivation of esomeprazole, there is insufficient evidence to support an effect on the\\ntherapeutic effectiveness and side effects.\\nLiterature:\\n1\\n. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with\\nproton pump inhibitors. Meta Gene 2016;9:159-64.\\n2\\n. \\nDury S et al. Agranulocytosis induced by proton pump inhibitors. J Clin Gastroenterol 2012;46:859.\\n3\\n. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics\\nand CYP2C19 polymorphism. Aliment Pharmacol Ther 2010;31:150-9.\\nDate 05-03-2018\\nCYP2C9 IM: fenprocoumon\\n1875\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 46}),\n",
       " Document(page_content='this causes problems with normal initiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n4\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n5\\n. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon\\nmetabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n7\\n. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-\\ngenetic parameters. Eur J Clin Pharmacol 2011;67:371-81.\\n8\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n10\\n. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol\\n2009;65:783-8.\\n11\\n. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb\\nThrombolysis 2009;28:211-4.\\n12\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n13\\n. \\nBohrer T et al. Left ventricular non-compaction associated with a genetic variant of the CYP2C9 gene. Heart Lung Circ\\n2006;15:269-71.\\n14\\n. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9\\ngenotypes are not. Eur J Clin Pharmacol 2006;62:713-20.\\n15\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n16\\n. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol\\nTher 2004;76:409-17.\\n17\\n. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and\\nin vivo. Xenobiotica 2004;34:847-59.\\n18\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n19\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;14:27-33.\\n20\\n. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy\\nvolunteers. Pharmacogenetics 2004;14:19-26.\\n21\\n. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin\\nPharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9 PM: fenprocoumon\\n1876\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that\\nthis causes problems with normal initiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 47}),\n",
       " Document(page_content='Haemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n4\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n5\\n. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon\\nmetabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n7\\n. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-\\ngenetic parameters. Eur J Clin Pharmacol 2011;67:371-81.\\n8\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n10\\n. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol\\n2009;65:783-8.\\n11\\n. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb\\nThrombolysis 2009;28:211-4.\\n12\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n13\\n. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9\\ngenotypes are not. Eur J Clin Pharmacol 2006;62:713-20.\\n14\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n15\\n. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol\\nTher 2004;76:409-17.\\n16\\n. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and\\nin vivo. Xenobiotica 2004;34:847-59.\\n17\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n18\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;14:27-33.\\n19\\n. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy\\nvolunteers. Pharmacogenetics 2004;14:19-26.\\n20\\n. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin\\nPharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9*1/*2: fenprocoumon\\n1870\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that\\nthis causes problems with normal initiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n4\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n5\\n. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 48}),\n",
       " Document(page_content='metabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n7\\n. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-\\ngenetic parameters. Eur J Clin Pharmacol 2011;67:371-81.\\n8\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n10\\n. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol\\n2009;65:783-8.\\n11\\n. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb\\nThrombolysis 2009;28:211-4.\\n12\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n13\\n. \\nBohrer T et al. Left ventricular non-compaction associated with a genetic variant of the CYP2C9 gene. Heart Lung Circ\\n2006;15:269-71.\\n14\\n. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9\\ngenotypes are not. Eur J Clin Pharmacol 2006;62:713-20.\\n15\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n16\\n. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol\\nTher 2004;76:409-17.\\n17\\n. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and\\nin vivo. Xenobiotica 2004;34:847-59.\\n18\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n19\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;14:27-33.\\n20\\n. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy\\nvolunteers. Pharmacogenetics 2004;14:19-26.\\n21\\n. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin\\nPharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9*1/*3: fenprocoumon\\n1871\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that\\nthis causes problems with normal initiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n4\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n5\\n. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon\\nmetabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n7\\n. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-\\ngenetic parameters. Eur J Clin Pharmacol 2011;67:371-81.\\n8\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n10\\n. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 49}),\n",
       " Document(page_content='2009;65:783-8.\\n11\\n. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb\\nThrombolysis 2009;28:211-4.\\n12\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n13\\n. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9\\ngenotypes are not. Eur J Clin Pharmacol 2006;62:713-20.\\n14\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n15\\n. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol\\nTher 2004;76:409-17.\\n16\\n. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and\\nin vivo. Xenobiotica 2004;34:847-59.\\n17\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n18\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;14:27-33.\\n19\\n. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy\\nvolunteers. Pharmacogenetics 2004;14:19-26.\\n20\\n. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin\\nPharmacol 2003;59:213-9.\\nDate 14-05-2018\\nCYP2C9*2/*2: fenprocoumon\\n1872\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose and possibly in an extension of the time required to\\nachieve a stable INR. However, there is not enough evidence to confirm that this causes problems with normal initiation of the therapy\\n(i.e. frequent INR monitoring).\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n4\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n5\\n. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon\\nmetabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n7\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n8\\n. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol\\n2009;65:783-8.\\n9\\n. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb\\nThrombolysis 2009;28:211-4.\\n10\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n11\\n. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9\\ngenotypes are not. Eur J Clin Pharmacol 2006;62:713-20.\\n12\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n13\\n. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol\\nTher 2004;76:409-17.\\n14\\n. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and\\nin vivo. Xenobiotica 2004;34:847-59.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 50}),\n",
       " Document(page_content='15\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n16\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;14:27-33.\\n17\\n. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy\\nvolunteers. Pharmacogenetics 2004;14:19-26.\\nDate 14-05-2018\\nCYP2C9*2/*3: fenprocoumon\\n1873\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that\\nthis causes problems with normal initiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n4\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n5\\n. \\nAbduljalil K. et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n6\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon\\nmetabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n7\\n. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-\\ngenetic parameters. Eur J Clin Pharmacol 2011;67:371-81.\\n8\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n9\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n10\\n. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol\\n2009;65:783-8.\\n11\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n12\\n. \\nzu Schwabedissen CM et al. Obesity is associated with a slower response to initial phenprocoumon therapy whereas CYP2C9\\ngenotypes are not. Eur J Clin Pharmacol 2006;62:713-20.\\n13\\n. \\nVisser LE et al. Allelic variants of cytochrome P450 2C9 modify the interaction between nonsteroidal anti-inflammatory drugs and\\ncoumarin anticoagulants. Clin Pharmacol Ther 2005;77:479-85.\\n14\\n. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol\\nTher 2004;76:409-17.\\n15\\n. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and\\nin vivo. Xenobiotica 2004;34:847-59.\\n16\\n. \\nVisser LE et al. The risk of bleeding complications in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Thromb Haemost 2004;92:61-6.\\n17\\n. \\nVisser LE et al. The risk of overanticoagulation in patients with cytochrome P450 CYP2C9*2 or CYP2C9*3 alleles on\\nacenocoumarol or phenprocoumon. Pharmacogenetics 2004;14:27-33.\\n18\\n. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy\\nvolunteers. Pharmacogenetics 2004;14:19-26.\\n19\\n. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin\\nPharmacol 2003;59:213-9.\\nDate 14-05-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 51}),\n",
       " Document(page_content='CYP2C9*3/*3: fenprocoumon\\n1874\\nNO action is required for this gene-drug interaction.\\nThe genetic variation can result in a reduction in the required maintenance dose. However, there is not enough evidence to confirm that\\nthis causes problems with normal initiation of the therapy (i.e. frequent INR monitoring).\\nLiterature:\\n1\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n2\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n3\\n. \\nWerner D et al. Pharmacogenetic characteristics of patients with complicated phenprocoumon dosing. Eur J Clin Pharmacol\\n2009;65:783-8.\\n4\\n. \\nSchalekamp T et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n5\\n. \\nSchalekamp T et al. Effects of cytochrome P450 2C9 polymorphisms on phenprocoumon anticoagulation status. Clin Pharmacol\\nTher 2004;76:409-17.\\n6\\n. \\nUfer M et al. Genetic polymorphisms of cytochrome P450 2C9 causing reduced phenprocoumon (S)-7-hydroxylation in vitro and\\nin vivo. Xenobiotica 2004;34:847-59.\\n7\\n. \\nKirchheiner J et al. Effects of CYP2C9 polymorphisms on the pharmacokinetics of R- and S-phenprocoumon in healthy\\nvolunteers. Pharmacogenetics 2004;14:19-26.\\n8\\n. \\nHummers-Pradier E et al. Determination of bleeding risk using genetic markers in patients taking phenprocoumon. Eur J Clin\\nPharmacol 2003;59:213-9.\\nDate 14-05-2018\\nVKORC1 -1639 AA: fenprocoumon\\n1912\\nAn INR ≥ 6, resulting in an increased risk of bleeding, occurs in 17% of these patients with standard regulation by the Anticoagulation\\nClinic. The genetic variation increases the sensitivity to phenprocoumon.\\nMonitoring by a ANTICOAGULATION CLINIC:\\nrecommend to use 50% of the standard initial dose\\nNO monitoring by a anticoagulation clinic:\\nrecommend to use 50% of the standard initial dose\\nrecommend more frequent monitoring of the INR\\nFor patients younger than 75 years, the initial dose and the maintenance dose can be calculated using an algorithm as found in EU-\\nPACT: see https://www.knmp.nl/patientenzorg/medicatiebewaking/farmacogenetica for a calculation tool in the form of an Excel file.\\nHowever, for patients aged 75 years and older, this algorithm increases the risk of an INR above the therapeutic range compared to an\\nalgorithm without gene variations. Therefore, use of this algorithm is not recommended for these patients.\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nBotton et al. A new algorithm for weekly phenprocoumon dose variation in a southern Brazilian population: role for CYP2C9,\\nCYP3A4/5 and VKORC1 genes polymorphisms. Basic Clin Pharmacol Toxicol 2014;114:323-9\\n4\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n5\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n6\\n. \\nAbduljalil K et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/ pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 52}),\n",
       " Document(page_content='7\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon\\nmetabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n8\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of CYP2C9 and VKORC1 genotypes in phenprocoumon users. J Thromb\\nHaemost. 2012;10:2610-2.\\n9\\n. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-\\ngenetic parameters. Eur J Clin Pharmacol 2011;67:371-81.\\n10\\n. \\nTeichert M et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1, CYP2C9, and CYP4F2 genes.\\nPharmacogenet Genomics 2011;21:26-34.\\n11\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n12\\n. \\nNowak-Göttl U et al. In pediatric patients, age has more impact on dosing of vitamin K antagonists than VKORC1 or CYP2C9\\ngenotypes. Blood 2010;116:6101-5.\\n13\\n. \\nPuehringer H et al. VKORC1 -1639G>A and CYP2C9*3 are the major genetic predictors of phenprocoumon dose requirement.\\nEur J Clin Pharmacol 2010;66:591-8.\\n14\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n15\\n. \\nArnold ML et al. Pharmacogenetic testing for guiding de novo phenprocoumon therapy in stroke patients. Cerebrovasc Dis\\n2009;28:468-71.\\n16\\n. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb\\nThrombolysis 2009;28:211-4.\\n17\\n. \\nSchalekamp et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n18\\n. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med\\n2005;2:e312.\\nDate 10-09-2018\\nVKORC1 -1639 GA: fenprocoumon\\n1911\\nNO action is needed for this gene-drug interaction.\\nThe gene variation leads to a lower dose requirement, but regular monitoring of patients ensures that this does not lead to a distinct\\nincrease in the risk of bleeding.\\nLiterature:\\n1\\n. \\nZhang Y et al. Age-stratified outcome of a genotype-guided dosing algorithm for acenocoumarol and phenprocoumon. J Thromb\\nHaemost 2017;15:454-464.\\n2\\n. \\nBrehm K et al. Genetic variations of phenprocoumon metabolism in patients with ventricular assist devices. Eur J Cardiothorac\\nSurg 2016;50:275-80.\\n3\\n. \\nBoton et al. A new algorithm for weekly phenprocoumon dose variation in a southern Brazilian population: role for CYP2C9,\\nCYP3A4/5 and VKORC1 genes polymorphisms. Basic Clin Pharmacol Toxicol 2014;114:323-9.\\n4\\n. \\nVerhoef TI et al. A randomized trial of genotype-guided dosing of acenocoumarol and phenprocoumon. N Engl J Med\\n2013;369:2304-12.\\n5\\n. \\nBaranova EV et al. Dosing algorithms for vitamin K antagonists across VKORC1 and CYP2C9 genotypes. J Thromb Haemost\\n2017;15:465-472.\\n6\\n. \\nAbduljalil K et al. Quantifying the effect of covariates on concentrations and effects of steady-state phenprocoumon using a\\npopulation pharmacokinetic/ pharmacodynamic model. Clin Pharmacokinet 2013;52:359-71.\\n7\\n. \\nBrehm K et al. Mechanical heart valve recipients: anticoagulation in patients with genetic variations of phenprocoumon\\nmetabolism. Eur J Cardiothorac Surg 2013;44:309-14.\\n8\\n. \\nVerhoef TI et al. Long-term anticoagulant effects of CYP2C9 and VKORC1 genotypes in phenprocoumon users. J Thromb\\nHaemost. 2012;10:2610-2.\\n9\\n. \\nGeisen C et al. Prediction of phenprocoumon maintenance dose and phenprocoumon plasma concentration by genetic and non-\\ngenetic parameters. Eur J Clin Pharmacol 2011;67:371-81.\\n10\\n. \\nTeichert M et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1, CYP2C9, and CYP4F2 genes.\\nPharmacogenet Genomics 2011;21:26-34.\\n11\\n. \\nLuxembourg B et al. Impact of pharmacokinetic (CYP2C9) and pharmacodynamic (VKORC1, F7, GGCX, CALU, EPHX1) gene\\nvariants on the initiation and maintenance phases of phenprocoumon therapy. Thromb Haemost 2011;105:169-80.\\n12\\n. \\nNowak-Göttl U et al. In pediatric patients, age has more impact on dosing of vitamin K antagonists than VKORC1 or CYP2C9\\ngenotypes. Blood 2010;116:6101-5.\\n13\\n. \\nPuehringer H et al. VKORC1 -1639G>A and CYP2C9*3 are the major genetic predictors of phenprocoumon dose requirement.\\nEur J Clin Pharmacol 2010;66:591-8.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 53}),\n",
       " Document(page_content='14\\n. \\nCadamuro J et al. Genetic determinants of acenocoumarol and phenprocoumon maintenance dose requirements. Eur J Clin\\nPharmacol 2010;66:253-60.\\n15\\n. \\nArnold ML et al. Pharmacogenetic testing for guiding de novo phenprocoumon therapy in stroke patients. Cerebrovasc Dis\\n2009;28:468-71.\\n16\\n. \\nQazim B et al. Dependency of phenprocoumon dosage on polymorphisms in the VKORC1 and CYP2C9 genes. J Thromb\\nThrombolysis 2009;28:211-4.\\n17\\n. \\nSchmeits PC et al. Investigating unexpected INRs: in search of the culprit--adherence, interactions, genetics, and superwarfarin.\\nNeth J Med 2009;67:76-8.\\n18\\n. \\nSchalekamp et al. VKORC1 and CYP2C9 genotypes and phenprocoumon anticoagulation status: interaction between both\\ngenotypes affects dose requirement. Clin Pharmacol Ther 2007;81:185-93.\\n19\\n. \\nReitsma PH et al. A C1173T dimorphism in the VKORC1 gene determines coumarin sensitivity and bleeding risk. PLoS Med\\n2005;2:e312.\\nDate 10-09-2018\\nCYP2C9 IM: fenytoine\\n1676\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1\\n. \\nThe loading dose does not need to be adjusted.\\n2\\n. \\nFor the other doses, use 75% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3\\n. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1\\n. \\nTassaneeyakul W et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-\\nrelated severe cutaneous adverse reactions in a Thai population. Pharmacogenet Genomics 2016;26:225-34.\\n2\\n. \\nOrtega-Vázquez A et al. CYP2C9, CYP2C19, ABCB1 genetic polymorphisms and phenytoin plasma concentrations in Mexican-\\nMestizo patients with epilepsy. Pharmacogenomics J 2016;16:286-92.\\n3\\n. \\nYamamoto Y et al. Individualized phenytoin therapy for Japanese pediatric patients with epilepsy based on CYP2C9 and\\nCYP2C19 genotypes. Ther Drug Monit 2015;37:229-35.\\n4\\n. \\nChung WH et al. Genetic variants associated with phenytoin-related severe cutaneous adverse reactions. JAMA 2014;312:525-34.\\n5\\n. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9\\nvariants with phenytoin toxicity. Eur J Neurol 2011;18:1159-64.\\n6\\n. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in\\nIndian epileptic patients. Eur J Clin Pharmacol 2010;66:689-96.\\n7\\n. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet\\nGenomics 2010;20:58-63.\\n8\\n. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol\\n2009;49:1483-7.\\n9\\n. \\nMcCluggage LK et al. Phenytoin toxicity due to genetic polymorphism. Neurocrit Care 2009;10:222-4.\\n10\\n. \\nLee SY et al. Contributions of CYP2C9/CYP2C19 genotypes and drug interaction to the phenytoin treatment in the Korean\\nepileptic patients in the clinical setting. J Biochem Mol Biol 2007;40:448-52.\\n11\\n. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs\\ncarbamazepine and phenytoin. Proc Natl Acad Sci U S A 2005;102:5507-12.\\n12\\n. \\nHung CC et al. Dosage recommendation of phenytoin for patients with epilepsy with different CYP2C9/CYP2C19\\npolymorphisms. Ther Drug Monit 2004;26:534-40.\\n13\\n. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement.\\nPharmacogenetics 2001;11:287-91.\\n14\\n. \\nNinomiya H et al. Genetic polymorphism of the CYP2C subfamily and excessive serum phenytoin concentration with central\\nnervous system intoxication. Ther Drug Monit 2000;22:230-2.\\n15\\n. \\nMamiya K et al. The effects of genetic polymorphisms of CYP2C9 and CYP2C19 on phenytoin metabolism in Japanese adult\\npatients with epilepsy: studies in stereoselective hydroxylation and population pharmacokinetics. Epilepsia 1998;39:1317-23.\\n16\\n. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 31-10-2016', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 54}),\n",
       " Document(page_content='CYP2C9 PM: fenytoine\\n1677\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1\\n. \\nThe loading dose does not need to be adjusted.\\n2\\n. \\nFor the other doses, use 40-50% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3\\n. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1\\n. \\nOrtega-Vázquez A et al. CYP2C9, CYP2C19, ABCB1 genetic polymorphisms and phenytoin plasma concentrations in Mexican-\\nMestizo patients with epilepsy. Pharmacogenomics J 2016;16:286-92.\\n2\\n. \\nKidd RS et al. Identification of a null allele of CYP2C9 in an African-American exhibiting toxicity to phenytoin.\\nPharmacogenetics 2001;11:803-8.\\nDate 31-10-2016\\nCYP2C9*1/*2: fenytoine\\n1678\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1\\n. \\nThe loading dose does not need to be adjusted.\\n2\\n. \\nFor the other doses, use 75% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3\\n. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1\\n. \\nOrtega-Vázquez A et al. CYP2C9, CYP2C19, ABCB1 genetic polymorphisms and phenytoin plasma concentrations in Mexican-\\nMestizo patients with epilepsy. Pharmacogenomics J 2016;16:286-92.\\n2\\n. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9\\nvariants with phenytoin toxicity. Eur J Neurol 2011;18:1159-64.\\n3\\n. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in\\nIndian epileptic patients. Eur J Clin Pharmacol 2010;66:689-96.\\n4\\n. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet\\nGenomics 2010;20:58-63.\\n5\\n. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol\\n2009;49:1483-7.\\n6\\n. \\nRosemary J et al. Influence of the CYP2C9 AND CYP2C19 polymorphisms on phenytoin hydroxylation in healthy individuals\\nfrom south India. Indian J Med Res 2006;123:665-70.\\n7\\n. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs\\ncarbamazepine and phenytoin. Proc Natl Acad Sci U S A 2005;102:5507-12.\\n8\\n. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n9\\n. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement.\\nPharmacogenetics 2001;11:287-91.\\n10\\n. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels.\\nPharmacogenomics J 2001;1:204-10.\\n11\\n. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for\\nphenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n12\\n. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 31-10-2016', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 55}),\n",
       " Document(page_content='CYP2C9*1/*3: fenytoine\\n1679\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects. The life-threatening\\ncutaneous side effects Stevens-Johnson Syndrome and toxic epidermal necrolysis may occur, especially in Asian patients.\\nRecommendation:\\n1\\n. \\nThe loading dose does not need to be adjusted.\\n2\\n. \\nFor the other doses, use 75% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3\\n. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or, especially in Asian\\npatients, rash) occur.\\nLiterature:\\n1\\n. \\nTassaneeyakul W et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-\\nrelated severe cutaneous adverse reactions in a Thai population. Pharmacogenet Genomics 2016;26:225-34.\\n2\\n. \\nYamamoto Y et al. Individualized phenytoin therapy for Japanese pediatric patients with epilepsy based on CYP2C9 and\\nCYP2C19 genotypes. Ther Drug Monit 2015;37:229-35.\\n3\\n. \\nChung WH et al. Genetic variants associated with phenytoin-related severe cutaneous adverse reactions. JAMA 2014;312:525-34.\\n4\\n. \\nHung CC et al. Effects of polymorphisms in six candidate genes on phenytoin maintenance therapy in Han Chinese patients.\\nPharmacogenomics 2012;13:1339-49.\\n5\\n. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9\\nvariants with phenytoin toxicity. Eur J Neurol 2011;18:1159-64.\\n6\\n. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in\\nIndian epileptic patients. Eur J Clin Pharmacol 2010;66:689-96.\\n7\\n. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet\\nGenomics 2010;20:58-63.\\n8\\n. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol\\n2009;49:1483-7.\\n9\\n. \\nMcCluggage LK et al. Phenytoin toxicity due to genetic polymorphism. Neurocrit Care 2009;10:222-4.\\n10\\n. \\nLee SY et al. Contributions of CYP2C9/CYP2C19 genotypes and drug interaction to the phenytoin treatment in the Korean\\nepileptic patients in the clinical setting. J Biochem Mol Biol 2007;40:448-52.\\n11\\n. \\nRosemary J et al. Influence of the CYP2C9 AND CYP2C19 polymorphisms on phenytoin hydroxylation in healthy individuals\\nfrom south India. Indian J Med Res 2006;123:665-70.\\n12\\n. \\nHung CC et al. Dosage recommendation of phenytoin for patients with epilepsy with different CYP2C9/CYP2C19\\npolymorphisms. Ther Drug Monit 2004;26:534-40.\\n13\\n. \\nSoga Y et al. CYP2C polymorphisms, phenytoin metabolism and gingival overgrowth in epileptic subjects. Life Sci 2004;74:827-\\n34.\\n14\\n. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n15\\n. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels.\\nPharmacogenomics J 2001;1:204-10.\\n16\\n. \\nNinomiya H et al. Genetic polymorphism of the CYP2C subfamily and excessive serum phenytoin concentration with central\\nnervous system intoxication. Ther Drug Monit 2000;22:230-2.\\n17\\n. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for\\nphenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n18\\n. \\nMamiya K et al. The effects of genetic polymorphisms of CYP2C9 and CYP2C19 on phenytoin metabolism in Japanese adult\\npatients with epilepsy: studies in stereoselective hydroxylation and population pharmacokinetics. Epilepsia 1998;39:1317-23.\\n19\\n. \\nOdani A et al. Genetic polymorphism of the CYP2C subfamily and its effect on the pharmacokinetics of phenytoin in Japanese\\npatients with epilepsy. Clin Pharmacol Ther 1997;62:287-92.\\n20\\n. \\nHashimoto Y et al. Effect of CYP2C polymorphisms on the pharmacokinetics of phenytoin in Japanese patients with epilepsy.\\nBiol Pharm Bull 1996;19:1103-5.\\n21\\n. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 31-10-2016\\nCYP2C9*2/*2: fenytoine\\n1680\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1\\n. \\nThe loading dose does not need to be adjusted.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 56}),\n",
       " Document(page_content='2\\n. \\nFor the other doses, use 50% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3\\n. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1\\n. \\nDepondt C et al. A candidate gene study of antiepileptic drug tolerability and -efficacy identifies an association of CYP2C9\\nvariants with phenytoin toxicity. Eur J Neurol 2011;18:1159-64.\\n2\\n. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet\\nGenomics 2010;20:58-63.\\n3\\n. \\nHennessy S et al. CYP2C9, CYP2C19, and ABCB1 genotype and hospitalization for phenytoin toxicity. J Clin Pharmacol\\n2009;49:1483-7.\\n4\\n. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs\\ncarbamazepine and phenytoin. Proc Natl Acad Sci U S A 2005;102:5507-12.\\n5\\n. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n6\\n. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement.\\nPharmacogenetics 2001;11:287-91.\\n7\\n. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels.\\nPharmacogenomics J 2001;1:204-10.\\n8\\n. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for\\nphenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n9\\n. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 31-10-2016\\nCYP2C9*2/*3: fenytoine\\n1681\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1\\n. \\nThe loading dose does not need to be adjusted.\\n2\\n. \\nFor the other doses, use 50% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.\\n3\\n. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or rash) occur.\\nLiterature:\\n1\\n. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in\\nIndian epileptic patients. Eur J Clin Pharmacol 2010;66:689-96.\\n2\\n. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet\\nGenomics 2010;20:58-63.\\n3\\n. \\nCaraco Y et al. Phenytoin metabolic ratio: a putative marker of CYP2C9 activity in vivo. Pharmacogenetics 2001;11:587-96.\\n4\\n. \\nVan der Weide J et al. The effect of genetic polymorphism of cytochrome P450 CYP2C9 on phenytoin dose requirement.\\nPharmacogenetics 2001;11:287-91.\\n5\\n. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 31-10-2016\\nCYP2C9*3/*3: fenytoine\\n1682\\nGenetic variation reduces conversion of phenytoin to inactive metabolites. This increases the risk of side effects. The life-threatening\\ncutaneous side effects Stevens-Johnson Syndrome and toxic epidermal necrolysis may occur, especially in Asian patients.\\nRecommendation:\\n1\\n. \\nThe loading dose does not need to be adjusted.\\n2\\n. \\nFor the other doses, use 40% of the standard dose and assess the dose based on effect and serum concentration after 7-10 days.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 57}),\n",
       " Document(page_content='3\\n. \\nAdvise the patient to get in touch if side effects (such as ataxia, nystagmus, slurred speech, sedation or, especially in Asian\\npatients, rash) occur.\\nLiterature:\\n1\\n. \\nTassaneeyakul W et al. Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-\\nrelated severe cutaneous adverse reactions in a Thai population. Pharmacogenet Genomics 2016;26:225-34.\\n2\\n. \\nChung WH et al. Genetic variants associated with phenytoin-related severe cutaneous adverse reactions. JAMA 2014;312:525-34.\\n3\\n. \\nHung CC et al. Effects of polymorphisms in six candidate genes on phenytoin maintenance therapy in Han Chinese patients.\\nPharmacogenomics 2012;13:1339-49.\\n4\\n. \\nKesavan R et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on phenytoin-induced neurological toxicity in\\nIndian epileptic patients. Eur J Clin Pharmacol 2010;66:689-96.\\n5\\n. \\nAzzato EM et al. Maternal EPHX1 polymorphisms and risk of phenytoin-induced congenital malformations. Pharmacogenet\\nGenomics 2010;20:58-63.\\n6\\n. \\nJose L et al. Acenocoumarol and phenytoin toxicity in the presence of CYP2C9 mutation. J Assoc Physicians India 2008;56:250-2.\\n7\\n. \\nRamasamy K et al. Severe phenytoin toxicity in a CYP2C9\\n3\\n3 homozygous mutant from India. Neurol India 2007;55:408-9.\\n8\\n. \\nLee SY et al. Contributions of CYP2C9/CYP2C19 genotypes and drug interaction to the phenytoin treatment in the Korean\\nepileptic patients in the clinical setting. J Biochem Mol Biol 2007;40:448-52.\\n9\\n. \\nRosemary J et al. Influence of the CYP2C9 AND CYP2C19 polymorphisms on phenytoin hydroxylation in healthy individuals\\nfrom south India. Indian J Med Res 2006;123:665-70.\\n10\\n. \\nTate SK et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs\\ncarbamazepine and phenytoin. Proc Natl Acad Sci U S A 2005;102:5507-12.\\n11\\n. \\nBrandolese R et al. Severe phenytoin intoxication in a subject homozygous for CYP2C9*3. Clin Pharmacol Ther 2001;70:391-4.\\n12\\n. \\nKerb R et al. The predictive value of MDR1, CYP2C9, and CYP2C19 polymorphisms for phenytoin plasma levels.\\nPharmacogenomics J 2001;1:204-10.\\n13\\n. \\nAynacioglu AS et al. Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for\\nphenytoin. Br J Clin Pharmacol 1999;48:409-15.\\n14\\n. \\nKidd RS et al. Pharmacokinetics of chlorpheniramine, phenytoin, glipizide and nifedipine in an individual homozygous for the\\nCYP2C9*3 allele. Pharmacogenetics 1999;9:71-80.\\n15\\n. \\nwww.nvza.nl, TDM monografie voor fenytoïne.\\nDate 31-10-2016\\nCYP2D6 IM: flecainide\\n1593\\nThe genetic variation reduces conversion of flecainide to inactive metabolites. This may increase the risk of side effects.\\nRecommendation:\\nIndications other than diagnosis of Brugada syndrome:\\n1\\n. \\nreduce the dose to 75% of the standard dose and record an ECG and monitor the plasma concentration\\nProvocation test for diagnosis of Brugada syndrome:\\nNo action required.\\nAt a dose of 2.0 mg/kg body weight to a maximum of 150 mg, the response is better for patients with alleles that result in reduced\\nactivity.\\nAll 5 patients with these alleles and 20% of the patients with two fully active alleles exhibited a response within 30 minutes.\\nLiterature:\\n1\\n. \\nCalvo D et al. Time-dependent responses to provocative testing with flecainide in the diagnosis of Brugada syndrome. Heart\\nRhythm 2015;12:350-7.\\n2\\n. \\nHu M et al. Effects of CYP2D6\\n10, CYP3A5\\n3, CYP1A2*1F, and ABCB1 C3435T polymorphisms on the pharmacokinetics of\\nflecainide in healthy Chinese subjects. Drug Metabol Drug Interact 2012;27:33-9.\\n3\\n. \\nLim KS et al. Changes in the QTc interval after administration of flecainide acetate, with and without coadministered paroxetine,\\nin relation to cytochrome P450 2D6 genotype: data from an open-label, two-period, single-sequence crossover study in healthy\\nKorean male subjects. Clin Ther 2010;32:659-66.\\n4\\n. \\nLim KS et al. Pharmacokinetic interaction of flecainide and paroxetine in relation to the CYP2D6*10 allele in healthy Korean\\nsubjects. Br J Clin Pharmacol 2008;66:660-6.\\n5\\n. \\nDoki K et al. Effect of CYP2D6 genotype on flecainide pharmacokinetics in Japanese patients with supraventricular', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 58}),\n",
       " Document(page_content='tachyarrhythmia. Eur J Clin Pharmacol 2006;62:919-26.\\nDate 24-08-2016\\nCYP2D6 PM: flecainide\\n1592\\nThe genetic variation reduces conversion of flecainide to inactive metabolites. This increases the risk of side effects.\\nRecommendation:\\n1\\n. \\nreduce the dose to 50% of the standard dose and record an ECG and monitor the plasma concentration\\nLiterature:\\n1\\n. \\nPalmiere C et al. Usefulness of post-mortem biochemistry in forensic pathology: illustrative case reports. Leg Med (Tokyo)\\n2012;14:27-35.\\n2\\n. \\nTenneze L et al. Pharmacokinetics and electrocardiographic effects of a new controlled-release form of flecainide acetate:\\ncomparison with the standard form and influence of the CYP2D6 polymorphism. Clin Pharmacol Ther 2002;72:112-22.\\n3\\n. \\nFunck-Brentano C et al. Variable disposition kinetics and electrocardiographic effects of flecainide during repeated dosing in\\nhumans: contribution of genetic factors, dose-dependent clearance, and interaction with amiodarone. Clin Pharmacol Ther\\n1994;55:256-69.\\n4\\n. \\nGross AS et al. Polymorphic flecainide disposition under conditions of uncontrolled urine flow and pH. Eur J Clin Pharmacol\\n1991;40:155-62.\\n5\\n. \\nGross AS et al. Stereoselective disposition of flecainide in relation to the sparteine/ debrisoquine metaboliser phenotype. Br J Clin\\nPharmacol 1989;28:555-66.\\n6\\n. \\nMikus G et al. The influence of the sparteine/debrisoquin phenotype on the disposition of flecainide. Clin Pharmacol Ther\\n1989;45:562-7.\\nDate 24-08-2016\\nCYP2D6 UM: flecainide\\n1594\\nThe genetic variation increases conversion of flecainide to inactive metabolites. A higher dose is possibly required as a result.\\nRecommendation:\\nThere are no data about the pharmacokinetics and/or the effects of flecainide in UM.\\n1\\n. \\nmonitor the plasma concentration as a precaution and record an ECG or select an alternative\\nExamples of anti-arrhythmic drugs that are not metabolised via CYP2D6 (or to a lesser extent) include sotalol, disopyramide,\\nquinidine and amiodarone.\\nLiterature:\\n-\\nDate 24-08-2016\\nHLA-B*5701: flucloxacilline\\n4652\\nHLA-B*5701-positive patients have an 80-fold elevated risk of flucloxacillin-induced liver injury. However, the incidence is low (1-2\\nper 1000 individuals).\\nRecommendation:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 59}),\n",
       " Document(page_content='1\\n. \\nRegularly monitor the patient’s liver function\\n2\\n. \\nChoose an alternative if liver enzymes and/or bilirubin levels are elevated\\nLiterature:\\n1\\n. \\nVera JH et al. The safety of flucloxacillin in HIV-infected patients with positive HLA-B*5701 genotype. Aids 2013;27:484-5.\\n2\\n. \\nPhilps EJ and Mallal SA. HLA-B*5701 and flucloxacillin associated drug-induced liver disease. Aids 2013;27:491-2.\\n3\\n. \\nDaly AK et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nature Genetics\\n2009;41:816-9.\\n4\\n. \\nSmPC Floxapen.\\nDate 20-11-2017\\nCYP2D6 IM: flufenazine\\n2451\\nThis is NOT a gene-drug interaction.\\nDespite the fact that the SmPC for fluphenazine lists CYP2D6 as the metabolising enzyme, this cannot be substantiated by the available\\nliterature.\\nLiterature:\\n1\\n. \\nSPC Anatensol decanoaat.\\nDate 26-05-2009\\nCYP2D6 PM: flufenazine\\n2450\\nThis is NOT a gene-drug interaction.\\nDespite the fact that the SmPC for fluphenazine lists CYP2D6 as the metabolising enzyme, this cannot be substantiated by the available\\nliterature.\\nLiterature:\\n1\\n. \\nSPC Anatensol decanoaat.\\nDate 26-05-2009\\nCYP2D6 UM: flufenazine\\n2452\\nThis is NOT a gene-drug interaction.\\nDespite the fact that the SmPC for fluphenazine lists CYP2D6 as the metabolising enzyme, this cannot be substantiated by the available\\nliterature.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 60}),\n",
       " Document(page_content=\"Literature:\\n1\\n. \\nSPC Anatensol decanoaat.\\nDate 26-05-2009\\nDPD genact 0: fluorouracil cutaan\\n6192\\nGenetic variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive\\nmetabolites means that the normal dose is an overdose.\\nRecommendation:\\nChoose an alternative\\nNOTE: If a patient has two different genetic variations that lead to a non-functional DPD enzyme (e.g. *2A and* 13), this\\nrecommendation only applies if the variations are on a different allele.\\nIf both variations are on the same allele, this patient has gene activity score 1, for which no increased risk of severe, potentially\\nfatal toxicity has been found with cutaneous use. These two situations can only be distinguished by determining the enzyme\\nactivity (phenotyping).\\nLiterature:\\n1\\n. \\nHenricks LM et al. Treatment algorithm for homozygous or compound heterozygous DPYD variant allele carriers with low-dose\\ncapecitabine. JCO Precis Oncol 2017 Oct 8 [Epub ahead of print].\\n2\\n. \\nHenricks LM et al. Capecitabine-based treatment of a patient with a novel DPYD genotype and complete dihydropyrimidine\\ndehydrogenase deficiency. Int J Cancer 2017 Sep 20 [Epub ahead of print].\\n3\\n. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-\\nfluorouracil. J Gastrointest Cancer 2017;48:66-69.\\n4\\n. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-\\ntreated patients. Pharmacogenomics 2016;17:721-9.\\n5\\n. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin\\nOncol 2016;34:227-34.\\n6\\n. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the\\nQUASAR2 study, systematic review, and meta-analysis. J Clin Oncol 2014; 32:1031-9.\\n7\\n. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of\\ncapecitabine in advanced colorectal cancer. Clin Cancer Res 2011; 17:3455-68.\\n8\\n. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-\\nrelated toxicity in cancer patients. PLoS ONE 2008;3:e4003.\\n9\\n. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection\\nof dihydropyrimidine dehydrogenase deficiency. Cancer Lett 2007;249:271-82.\\n10\\n. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-\\nfluorouracil tolerance. Mol Cancer Ther 2006;5:2895-904.\\n11\\n. \\nVan Kuilenburg AB et al. High prevalence of the IVS14 + 1G>A mutation in the dihydropyrimidine dehydrogenase gene of\\npatients with severe 5-fluorouracil-associated toxicity. Pharmacogenetics 2002;12:555-8.\\n12\\n. \\nRaida M et al. Prevalence of a common point mutation in the dihydropyrimidine dehydrogenase (DPD) gene within the 5'-splice\\ndonor site of intron 14 in patients with severe 5-fluorouracil (5-FU)-related toxicity compared with controls. Clin Cancer Res\\n2001;7:2832-9.\\n13\\n. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-\\nfluorouracil-associated toxicity: identification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n14\\n. \\nJohnson MR et al. Life-threatening toxicity in a dihydropyrimidine dehydrogenase-deficient patient after treatment with topical 5-\\nfluorouracil. Clin Cancer Res 1999;5:2006-11.\\n15\\n. \\nSPC Efudix crème en Carac cream (VS).\\nDate 20-11-2017\\nDPD genact 0,5: fluorouracil/capecitabine\\n4893\\nGenetic variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive\\nmetabolites means that the normal dose is an overdose.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 61}),\n",
       " Document(page_content='Recommendation:\\n\\u200b\\nStart with 25% of the standard dose or choose an alternative.\\nAdjustment of the initial dose should be guided by toxicity and effectiveness.\\nTegafur is not an alternative, as this is also metabolised by DPD.\\nNOTE: This recommendation only applies if the two genetic variations are on a different allele.\\nIf both variations are on the same allele, this patient has gene activity score 1 and the recommendation for that gene activity score\\nshould be followed. These two situations can only be distinguished by determining the enzyme activity (phenotyping).\\nLiterature:\\n1\\n. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-\\nfluorouracil. J Gastrointest Cancer 2017;48:66-69.\\n2\\n. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-\\ntreated patients. Pharmacogenomics 2016;17:721-9.\\n3\\n. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin\\nOncol 2016;34:227-34.\\n4\\n. \\nLee AM et al. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147). J Natl\\nCancer Inst 2014;106:dju298.\\n5\\n. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of\\ncapecitabine in advanced colorectal cancer. Clin Cancer Res 2011; 17:3455-68.\\n6\\n. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection\\nof dihydropyrimidine dehydrogenase deficiency. Cancer Lett 2007;249:271-82.\\n7\\n. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-\\nfluorouracil tolerance. Mol Cancer Ther 2006;5:2895-904.\\n8\\n. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS), en Xeloda (VS).\\nDate 20-11-2017\\nDPD genact 1,5: fluorouracil/capecitabine\\n4894\\nGenetic variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive\\nmetabolites means that the normal dose is an overdose.\\nRecommendation:\\nStart with 75% of the standard dose or choose an alternative.\\nAdjustment of the initial dose should be guided by toxicity and effectiveness.\\nTegafur is not an alternative, as this is also metabolised by DPD.\\nLiterature:\\n1\\n. \\nMeulendijks D et al. Pretreatment serum uracil concentration as a predictor of severe and fatal fluoropyrimidine-associated\\ntoxicity. Br J Cancer 2017;116:1415-24.\\n2\\n. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-\\ntreated patients. Pharmacogenomics 2016;17:721-9.\\n3\\n. \\nLee AM et al. Association between DPYD c.1129-5923 C>G/hapB3 and severe toxicity to 5-fluorouracil-based chemotherapy in\\nstage III colon cancer patients: NCCTG N0147 (Alliance). Pharmacogenet Genomics 2016;26:133-7.\\n4\\n. \\nMeulendijks D et al. Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe\\nfluoropyrimidine-associated toxicity: a systematic review and meta-analysis of individual patient data. Lancet Oncol\\n2015;16:1639-50.\\n5\\n. \\nLee AM et al. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147). J Natl\\nCancer Inst 2014;106:dju298.\\n6\\n. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the\\nQUASAR2 study, systematic review, and meta-analysis. J Clin Oncol 2014; 32:1031-9.\\n7\\n. \\nTerrazzino S et al. DPYD IVS14+1 G>A and 2846A>T genotyping for the prediction of severe fluoropyrimidine-related toxicity:\\na meta-analysis. Pharmacogenomics2013; 14:1255-72.\\n8\\n. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of\\ncapecitabine in advanced colorectal cancer. Clin Cancer Res 2011; 17:3455-68.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 62}),\n",
       " Document(page_content='9\\n. \\nKristensen MH et al. Variants in the dihydropyrimidine dehydrogenase, methylenetetrahydrofolate reductase and thymidylate\\nsynthase genes predict early toxicity of 5-fluorouracil in colorectal cancer patients. J Int Med Res 2010; 38:870-83.\\n10\\n. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-\\nrelated toxicity in cancer patients. PLoS ONE 2008;3:e4003.\\n11\\n. \\nCapitain O et al. The influence of fluorouracil outcome parameters on tolerance and efficacy in patients with advanced colorectal\\ncancer. Pharmacogenomics J 2008;8:256-67.\\n12\\n. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection\\nof dihydropyrimidine dehydrogenase deficiency. Cancer Lett 2007;249:271-82.\\n13\\n. \\nCho HJ et al. Thymidylate synthase (TYMS) and dihydropyrimidine dehydrogenase (DPYD) polymorphisms in the Korean\\npopulation for prediction of 5-fluorouracil-associated toxicity. Ther Drug Monit 2007;29:190-6.\\n14\\n. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-\\nfluorouracil tolerance. Mol Cancer Ther 2006;5:2895-904.\\n15\\n. \\nYamaguchi K et al. Germline mutation of dihydropyrimidine dehydrogenese gene among a Japanese population in relation to\\ntoxicity to 5-fluorouracil. Jpn J Cancer Res 2001;92:337-42.\\n16\\n. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-\\nfluorouracil-associated toxicity: identification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n17\\n. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS) en Xeloda (VS).\\nDate 20-11-2017\\nDPD genact 1: fluorouracil/capecitabine\\n2552\\nGenetic variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive\\nmetabolites means that the normal dose is an overdose.\\nRecommendation:\\nStart with 50% of the standard dose or choose an alternative.\\nAdjustment of the initial dose should be guided by toxicity and effectiveness.\\nTegafur is not an alternative, as this is also metabolised by DPD.\\nNB1: The dose reduction described here is well substantiated for *1/*2A and 1236A/1236A.\\nThe dose reduction for patients with 2846T (2846T/2846T or 1236A/2846T) is based on, among other factors, the dose reductions\\nidentified for *1/2846T.\\nNB2: If a patient has two different genetic variations that result in a partially functional DPD enzyme (e.g. 2846T and 1236A), this\\nrecommendation applies if the variations are on a different allele. If both variations are on the same allele, the gene activity score\\nis between 1 and 1.5, depending on whether and how the two gene variations influence each other and on other factors that\\ninfluence the DPD activity. Whether a gene activity score of 1 or 1.5 needs to be assigned in the case of two different genetic\\nvariations can only be determined by measuring the enzyme activity (phenotyping).\\nLiterature:\\n1\\n. \\nHenricks LM et al. Treatment algorithm for homozygous or compound heterozygous DPYD variant allele carriers with low-dose\\ncapecitabine. JCO Precis Oncol 2017 Oct 8 [Epub ahead of print].\\n2\\n. \\nMeulendijks D et al. Pretreatment serum uracil concentration as a predictor of severe and fatal fluoropyrimidine-associated\\ntoxicity. Br J Cancer 2017;116:1415-24.\\n3\\n. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-\\nfluorouracil. J Gastrointest Cancer 2017;48:66-69.\\n4\\n. \\nMeulendijks D et al. Patients homozygous for DPYD c.1129-5923C>G/ haplotype B3 have partial DPD deficiency and require a\\ndose reduction when treated with fluoropyrimidines. Cancer Chemother Pharmacol 2016;78:875-80.\\n5\\n. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-\\ntreated patients. Pharmacogenomics 2016;17:721-9.\\n6\\n. \\nLee AM et al. Association between DPYD c.1129-5923 C>G/hapB3 and severe toxicity to 5-fluorouracil-based chemotherapy in\\nstage III colon cancer patients: NCCTG N0147 (Alliance). Pharmacogenet Genomics 2016;26:133-7.\\n7\\n. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin\\nOncol 2016;34:227-34.\\n8\\n. \\nMeulendijks D et al. Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe\\nfluoropyrimidine-associated toxicity: a systematic review and meta-analysis of individual patient data. Lancet Oncol\\n2015;16:1639-50.\\n9\\n. \\nLee AM et al. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147). J Natl\\nCancer Inst 2014;106:dju298.\\n10\\n. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the\\nQUASAR2 study, systematic review, and meta-analysis. J Clin Oncol 2014; 32:1031-9.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 63}),\n",
       " Document(page_content=\"11\\n. \\nTerrazzino S et al. DPYD IVS14+1 G>A and 2846A>T genotyping for the prediction of severe fluoropyrimidine-related toxicity:\\na meta-analysis. Pharmacogenomics2013; 14:1255-72.\\n12\\n. \\nMagnani E et al. Fluoropyrimidine toxicity in patients with dihydropyrimidine dehydrogenase splice site variant: the need for\\nfurther revision of dose and schedule. Intern Emerg Med 2013; 8:417-23.\\n13\\n. \\nVulsteke C et al. Genetic variability in the multidrug resistance associated protein-1 (ABCC1/MRP1) predicts hematological\\ntoxicity in breast cancer patients receiving (neo-)adjuvant chemotherapy with 5-fluorouracil, epirubicin and cyclophosphamide\\n(FEC). Ann Oncol 2013; 24:1513-25.\\n14\\n. \\nvan Kuilenburg AB et al. Evaluation of 5-fluorouracil pharmacokinetics in cancer patients with a c.1905+1 G>A mutation in\\nDPYD by means of a Bayesian limited sampling strategy. Clin Pharmacokinet 2012;51:163-74.\\n15\\n. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of\\ncapecitabine in advanced colorectal cancer. Clin Cancer Res 2011; 17:3455-68.\\n16\\n. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-\\nrelated toxicity in cancer patients. PLoS ONE 2008;3:e4003.\\n17\\n. \\nCapitain O et al. The influence of fluorouracil outcome parameters on tolerance and efficacy in patients with advanced colorectal\\ncancer. Pharmacogenomics J 2008;8:256-67.\\n18\\n. \\nSulzyc-Bielicka V et al. 5-Fluorouracil toxicity-attributable IVS14 + 1G > A mutation of the dihydropyrimidine dehydrogenase\\ngene in Polish colorectal cancer patients. Pharmacol Rep 2008;60:238-42.\\n19\\n. \\nSchwab M et al. Role of genetic and nongenetic factors for fluorouracil treatment-related severe toxicity: a prospective clinical\\ntrial by the German 5-FU Toxicity Study Group. J Clin Oncol 2008;26:2131-8.\\n20\\n. \\nMercier C et al. Prospective phenotypic screening for DPD deficiency prior to 5-FU administration: Decrease in toxicity, not in\\nefficacy. J Clin Oncol 2008;26(May 20 suppl):abstr 14556. (meeting abstract)\\n21\\n. \\nJatoi A et al. Paclitaxel, carboplatin, 5-fluorouracil, and radiation for locally advanced esophageal cancer: phase II results of\\npreliminary pharmacologic and molecular efforts to mitigate toxicity and predict outcomes: North Central Cancer Treatment\\nGroup (N0044). Am J Clin Oncol 2007;30:507-13.\\n22\\n. \\nMagné N et al. Dihydropyrimidine dehydrogenase activity and the IVS14+1G>A mutation in patients developing 5FU-related\\ntoxicity. Br J Clin Pharmacol 2007;64:237-40.\\n23\\n. \\nSaif MW et al. Dihydropyrimidine dehydrogenase deficiency (GPD) in GI malignancies: experience of 4-years. Pak J Med Sci Q\\n2007;23:832-9.\\n24\\n. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection\\nof dihydropyrimidine dehydrogenase deficiency. Cancer Lett 2007;249:271-82.\\n25\\n. \\nCho HJ et al. Thymidylate synthase (TYMS) and dihydropyrimidine dehydrogenase (DPYD) polymorphisms in the Korean\\npopulation for prediction of 5-fluorouracil-associated toxicity. Ther Drug Monit 2007;29:190-6.\\n26\\n. \\nSalgado J et al. Polymorphisms in the thymidylate synthase and dihydropyrimidine dehydrogenase genes predict response and\\ntoxicity to capecitabine-raltitrexed in colorectal cancer. Oncol Rep 2007;17:325-8.\\n27\\n. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-\\nfluorouracil tolerance. Mol Cancer Ther 2006;5:2895-904.\\n28\\n. \\nLargillier R et al. Pharmacogenetics of capecitabine in advanced breast cancer patients. Clin Cancer Res 2006;12:5496-502.\\n29\\n. \\nSalgueiro N et al. Mutations in exon 14 of dihydropyrimidine dehydrogenase and 5-Fluorouracil toxicity in Portuguese colorectal\\ncancer patients. Genet Med 2004;6:102-7.\\n30\\n. \\nVan Kuilenburg AB et al. High prevalence of the IVS14 + 1G>A mutation in the dihydropyrimidine dehydrogenase gene of\\npatients with severe 5-fluorouracil-associated toxicity. Pharmacogenetics 2002;12:555-8.\\n31\\n. \\nRaida M et al. Prevalence of a common point mutation in the dihydropyrimidine dehydrogenase (DPD) gene within the 5'-splice\\ndonor site of intron 14 in patients with severe 5-fluorouracil (5-FU)- related toxicity compared with controls. Clin Cancer Res\\n2001;7:2832-9.\\n32\\n. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-\\nfluorouracil-associated toxicity: identification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n33\\n. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS) en Xeloda (VS).\\nDate 20-11-2017\\nDPD genact 0: fluorouracil/capecitabine,systemisch\\n2551\\nGenetic variation increases the risk of severe, potentially fatal toxicity. A reduced conversion of fluorouracil/capecitabine to inactive\\nmetabolites means that the standard dose is a more than 100-fold overdose.\\nRecommendation:\\n1\\n. \\nChoose an alternative\\nTegafur is not an alternative, as this is also metabolised by DPD.\\n2\\n. \\nIf an alternative is not possible:\\ndetermine the residual DPD activity in mononuclear cells from peripheral blood and adjust the initial dose accordingly.\\nA patient with 0.5% of the normal DPD activity tolerated 0.8% of the standard dose (150 mg capecitabine every 5 days). A\\npatient with undetectable DPD activity tolerated 0.43% of the standard dose (150 mg capecitabine every 5 days with every\\nthird dose skipped)\\nThe average Caucasian DPD activity is 9.9 nmol/hour per mg protein.\\nadjust the initial dose based on toxicity and efficacy.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 64}),\n",
       " Document(page_content=\"NOTE: If a patient has two different genetic variations that lead to a non-functional DPD enzyme (e.g. *2A and *13), this\\nrecommendation only applies if the variations are on a different allele. If both variations are on the same allele, this patient has gene\\nactivity score 1 and the recommendation for that gene activity score should be followed. These two situations can only be distinguished\\nby determining the enzyme activity (phenotyping).\\nLiterature:\\n1\\n. \\nHenricks LM et al. Treatment algorithm for homozygous or compound heterozygous DPYD variant allele carriers with low-dose\\ncapecitabine. JCO Precis Oncol 2017 Oct 8 [Epub ahead of print].\\n2\\n. \\nHenricks LM et al. Capecitabine-based treatment of a patient with a novel DPYD genotype and complete dihydropyrimidine\\ndehydrogenase deficiency. Int J Cancer 2017 Sep 20 [Epub ahead of print].\\n3\\n. \\nKodali S et al. Capecitabine-induced severe toxicity secondary to DPD deficiency and successful treatment with low dose 5-\\nfluorouracil. J Gastrointest Cancer 2017;48:66-69.\\n4\\n. \\nMeulendijks D et al. Patients homozygous for DPYD c.1129-5923C>G/ haplotype B3 have partial DPD deficiency and require a\\ndose reduction when treated with fluoropyrimidines. Cancer Chemother Pharmacol 2016;78:875-80.\\n5\\n. \\nLunenburg CA et al. Evaluation of clinical implementation of prospective DPYD genotyping in 5-fluorouracil- or capecitabine-\\ntreated patients. Pharmacogenomics 2016;17:721-9.\\n6\\n. \\nDeenen MJ et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin\\nOncol 2016;34:227-34.\\n7\\n. \\nRosmarin D et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the\\nQUASAR2 study, systematic review, and meta-analysis. J Clin Oncol 2014; 32:1031-9.\\n8\\n. \\nDeenen MJ et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of\\ncapecitabine in advanced colorectal cancer. Clin Cancer Res 2011; 17:3455-68.\\n9\\n. \\nGross E et al. Strong association of a common dihydropyrimidine dehydrogenase gene polymorphism with fluoropyrimidine-\\nrelated toxicity in cancer patients. PLoS ONE 2008;3:e4003.\\n10\\n. \\nBoisdron-Celle M et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection\\nof dihydropyrimidine dehydrogenase deficiency. Cancer Lett 2007;249:271-82.\\n11\\n. \\nMorel A et al. Clinical relevance of different dihydropyrimidine dehydrogenase gene single nucleotide polymorphisms on 5-\\nfluorouracil tolerance. Mol Cancer Ther 2006;5:2895-904.\\n12\\n. \\nVan Kuilenburg AB et al. High prevalence of the IVS14 + 1G>A mutation in the dihydropyrimidine dehydrogenase gene of\\npatients with severe 5-fluorouracil-associated toxicity. Pharmacogenetics 2002;12:555-8.\\n13\\n. \\nRaida M et al. Prevalence of a common point mutation in the dihydropyrimidine dehydrogenase (DPD) gene within the 5'-splice\\ndonor site of intron 14 in patients with severe 5-fluorouracil (5-FU)- related toxicity compared with controls. Clin Cancer Res\\n2001;7:2832-9.\\n14\\n. \\nvan Kuilenburg AB et al. Clinical implications of dihydropyrimidine dehydrogenase (DPD) deficiency in patients with severe 5-\\nfluorouracil-associated toxicity: identification of new mutations in the DPD gene. Clin Cancer Res 2000;6:4705-12.\\n15\\n. \\nJohnson MR et al. Life-threatening toxicity in a dihydropyrimidine dehydrogenase-deficient patient after treatment with topical 5-\\nfluorouracil. Clin Cancer Res 1999;5:2006-11.\\n16\\n. \\nSPC’s Fluorouracil PCH, Xeloda, Efudix crème, Fluorouracil (VS), Xeloda (VS) en Carac cream (VS).\\nDate 20-11-2017\\nCYP2D6 IM: fluoxetine\\n5997\\nNO action is needed for this gene-drug interaction.\\nThe ratio of fluoxetine/norfluoxetine increases as a result of the reduced activity of CYP2D6. However, this does not affect the sum of\\nthe plasma concentrations of the active substances (fluoxetine and norfluoxetine). There is no effect on adverse events or response.\\nLiterature:\\n1\\n. \\nGassó P et al. Effect of CYP2D6, CYP2C9 and ABCB1 genotypes on fluoxetine plasma concentrations and clinical improvement\\nin children and adolescent patients. Pharmacogenomics J 2014;14:457-62.\\n2\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\n3\\n. \\nScordo MG et al. Influence of CYP2C9, 2C19 and 2D6 genetic polymorphisms on the steady-state plasma concentrations of the\\nenantiomers of fluoxetine and norfluoxetine. Basic Clin Pharmacol Toxicol 2005;97:296-301.\\n4\\n. \\nLLerena A et al. Effect of CYP2D6 and CYP2C9 genotypes on fluoxetine and norfluoxetine plasma concentrations during steady-\\nstate conditions. Eur J Clin Pharmacol 2004;59:869-73.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 65}),\n",
       " Document(page_content='Date 14-05-2018\\nCYP2D6 PM: fluoxetine\\n5996\\nNO action is needed for this gene-drug interaction.\\nThe ratio of fluoxetine/norfluoxetine increases as a result of the reduced activity of CYP2D6. However, this does not affect the sum of\\nthe plasma concentrations of the active substances (fluoxetine and norfluoxetine). There is insufficient evidence to support an effect on\\nadverse events or response.\\nLiterature:\\n1\\n. \\nGassó P et al. Effect of CYP2D6, CYP2C9 and ABCB1 genotypes on fluoxetine plasma concentrations and clinical improvement\\nin children and adolescent patients. Pharmacogenomics J 2014;14:457-62.\\n2\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\n3\\n. \\nScordo MG et al. Influence of CYP2C9, 2C19 and 2D6 genetic polymorphisms on the steady-state plasma concentrations of the\\nenantiomers of fluoxetine and norfluoxetine. Basic Clin Pharmacol Toxicol 2005;97:296-301.\\n4\\n. \\nLLerena A et al. Effect of CYP2D6 and CYP2C9 genotypes on fluoxetine and norfluoxetine plasma concentrations during steady-\\nstate conditions. Eur J Clin Pharmacol 2004;59:869-73.\\n5\\n. \\nRoberts RL et al. No evidence of increased adverse drug reactions in cytochrome P450 CYP2D6 poor metabolizers treated with\\nfluoxetine or nortriptyline. Hum Psychopharmacol 2004;19:17-23.\\n6\\n. \\nPerucca E et al. Fluoxetine-induced movement disorders and deficient CYP2D6 enzyme activity. Mov Disord 1997;12:624-5.\\n7\\n. \\nSPC Prozac, USA, 30-01-09.\\nDate 14-05-2018\\nCYP2D6 UM: fluoxetine\\n5998\\nNO action is needed for this gene-drug interaction.\\nThe ratio of fluoxetine/norfluoxetine decreases as a result of the increased activity of CYP2D6. However, this does not affect the sum of\\nthe plasma concentrations of the active substances (fluoxetine and norfluoxetine). There is no effect on adverse events or response.\\nLiterature:\\n1\\n. \\nGassó P et al. Effect of CYP2D6, CYP2C9 and ABCB1 genotypes on fluoxetine plasma concentrations and clinical improvement\\nin children and adolescent patients. Pharmacogenomics J 2014;14:457-62.\\n2\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\n3\\n. \\nScordo MG et al. Influence of CYP2C9, 2C19 and 2D6 genetic polymorphisms on the steady-state plasma concentrations of the\\nenantiomers of fluoxetine and norfluoxetine. Basic Clin Pharmacol Toxicol 2005;97:296-301.\\n4\\n. \\nLLerena A et al. Effect of CYP2D6 and CYP2C9 genotypes on fluoxetine and norfluoxetine plasma concentrations during steady-\\nstate conditions. Eur J Clin Pharmacol 2004;59:869-73.\\nDate 14-05-2018\\nCYP2D6 IM: flupentixol\\n1532\\nThis is NOT a gene-drug interaction.\\nNo studies have been published in which the kinetics and the effects of flupentixol were studied for this phenotype.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 66}),\n",
       " Document(page_content='Literature:\\n-\\nDate 14-12-2005\\nCYP2D6 PM: flupentixol\\n1534\\nThis is NOT a gene-drug interaction.\\nNo studies have been published in which the kinetics and the effects of flupentixol were studied for this phenotype.\\nLiterature:\\n -\\nDate 14-12-2005\\nCYP2D6 UM: flupentixol\\n1533\\nThis is NOT a gene-drug interaction.\\nNo studies have been published in which the kinetics and the effects of flupentixol were studied for this phenotype.\\nLiterature:\\n-\\nDate 14-12-2005\\nSLCO1B1 521CC: fluvastatine\\n4060\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with\\nincreased cholesterol synthesis rate. Pharmacogenet Genomics 2008;18:921-6.\\n2\\n. \\nCouvert P et al. Association between a frequent allele of the gene encoding OATP1B1 and enhanced LDL-lowering response to\\nfluvastatin therapy. Pharmacogenomics 2008;9:1217-27.\\n3\\n. \\nSinger JB et al. Genetic analysis of fluvastatin response and dyslipidemia in renal transplant recipients. J Lipid Res 2007;48:2072-\\n8.\\n4\\n. \\nNiemi M et al. SLCO1B1 polymorphism and sex affect the pharmacokinetics of pravastatin but not fluvastatin. Clin Pharmacol\\nTher 2006;80:356-66.\\n5\\n. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J\\n2005;5:352-8.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 67}),\n",
       " Document(page_content='Date 27-11-2012\\nSLCO1B1 521TC: fluvastatine\\n4059\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with\\nincreased cholesterol synthesis rate. Pharmacogenet Genomics 2008;18:921-6.\\n2\\n. \\nCouvert P et al. Association between a frequent allele of the gene encoding OATP1B1 and enhanced LDL-lowering response to\\nfluvastatin therapy. Pharmacogenomics 2008;9:1217-27.\\n3\\n. \\nSinger JB et al. Genetic analysis of fluvastatin response and dyslipidemia in renal transplant recipients. J Lipid Res 2007;48:2072-\\n8.\\n4\\n. \\nNiemi M et al. SLCO1B1 polymorphism and sex affect the pharmacokinetics of pravastatin but not fluvastatin. Clin Pharmacol\\nTher 2006;80:356-66.\\n5\\n. \\nThompson JF et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharmacogenomics J\\n2005;5:352-8.\\nDate 27-11-2012\\nCYP2C19 IM: fluvoxamine\\n3510\\nThis is NOT a gene-drug interaction\\nLiterature:\\n1\\n. \\nJan MW et al. Pharmacokinetics of fluvoxamine in relation to CYP2C19 phenotype and genotype. Drug Metabol Drug Interact\\n2002;19:1-11.\\n2\\n. \\nSpigset O et al. The major fluvoxamine metabolite in urine is formed by CYP2D6. Eur J Clin Pharmacol 2001;57:653-8.\\n3\\n. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J\\nClin Pharmacol 1997;52:129-33.\\nDate 14-05-2018\\nCYP2C19 PM: fluvoxamine\\n3509\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nJan MW et al. Pharmacokinetics of fluvoxamine in relation to CYP2C19 phenotype and genotype. Drug Metabol Drug Interact\\n2002;19:1-11.\\n2\\n. \\nSpigset O et al. The major fluvoxamine metabolite in urine is formed by CYP2D6. Eur J Clin Pharmacol 2001;57:653-8.\\n3\\n. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J\\nClin Pharmacol 1997;52:129-33.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 68}),\n",
       " Document(page_content='Date 14-05-2018\\nCYP2C19 UM: fluvoxamine\\n3511\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nJan MW et al. Pharmacokinetics of fluvoxamine in relation to CYP2C19 phenotype and genotype. Drug Metabol Drug Interact\\n2002;19:1-11.\\n2\\n. \\nSpigset O et al. The major fluvoxamine metabolite in urine is formed by CYP2D6. Eur J Clin Pharmacol 2001;57:653-8.\\n3\\n. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J\\nClin Pharmacol 1997;52:129-33.\\nDate 14-05-2018\\nCYP2D6 IM: fluvoxamine\\n5994\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of fluvoxamine can increase as a result of the reduced activity of CYP2D6.\\nHowever, there is insufficient scientific substantiation of an increase in the risk of side effects.\\nLiterature:\\n1\\n. \\nSuzuki Y et al. CYP2D6 genotype and smoking influence fluvoxamine steady-state concentration in Japanese psychiatric patients:\\nlessons for genotype-phenotype association study design in translational pharmacogenetics. J Psychopharmacol 2011;25:908-14.\\n2\\n. \\nSugahara H et al. Effect of smoking and CYP2D6 polymorphisms on the extent of fluvoxamine-alprazolam interaction in patients\\nwith psychosomatic disease. Eur J Clin Pharmacol 2009;65:699-704.\\n3\\n. \\nSuzuki Y et al. Polymorphisms in the 5-hydroxytryptamine 2A receptor and cytochromeP4502D6 genes synergistically predict\\nfluvoxamine-induced side effects in Japanese depressed patients. Neuropsychopharmacology 2006;31:825-31.\\n4\\n. \\nGerstenberg G et al. Effects of the CYP2D6 genotype and cigarette smoking on the steady-state plasma concentrations of\\nfluvoxamine and its major metabolite fluvoxamino acid in Japanese depressed patients. Ther Drug Monit 2003;25:463-8.\\n5\\n. \\nGerstenberg G et al. Relationship between clinical effects of fluvoxamine and the steady-state plasma concentrations of\\nfluvoxamine and its major metabolite fluvoxamino acid in Japanese depressed patients. Psychopharmacology (Berl) 2003;167:443-\\n8.\\n6\\n. \\nOhara K et al. CYP2D6*10 alleles do not determine plasma fluvoxamine concentration/dose ratio in Japanese subjects. Eur J Clin\\nPharmacol 2003;58:659-61.\\nDate 14-05-2018\\nCYP2D6 PM: fluvoxamine\\n5993\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of fluvoxamine can increase as a result of the reduced activity of CYP2D6.\\nHowever, there is no evidence to substantiate an increase in the risk of adverse events.\\nLiterature:\\n1\\n. \\nChristensen M et al. Low daily 10-mg and 20-mg doses of fluvoxamine inhibit the metabolism of both caffeine (cytochrome', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 69}),\n",
       " Document(page_content='P4501A2) and omeprazole (cytochrome P4502C19). Clin Pharmacol Ther 2002;71:141-52.\\n2\\n. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J\\nClin Pharmacol 1997;52:129-33.\\n3\\n. \\nCarrillo JA et al. Disposition of fluvoxamine in humans is determined by the polymorphic CYP2D6 and also by the CYP1A2\\nactivity. Clin Pharmacol Ther 1996;60:183-90.\\n4\\n. \\nSPC’s Fevarin en Luvox (VS).\\nDate 14-05-2018\\nCYP2D6 UM: fluvoxamine\\n5995\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of fluvoxamine can decrease as a result of the increased activity of CYP2D6.\\nHowever, there is no scientific substantation of a reduced effectiveness.\\nLiterature:\\n1\\n. \\nSuzuki Y et al. CYP2D6 genotype and smoking influence fluvoxamine steady-state concentration in Japanese psychiatric patients:\\nlessons for genotype-phenotype association study design in translational pharmacogenetics. J Psychopharmacol 2011;25:908-14.\\n2\\n. \\nSpigset O et al. Relationship between fluvoxamine pharmacokinetics and CYP2D6/CYP2C19 phenotype polymorphisms. Eur J\\nClin Pharmacol 1997;52:129-33.\\n3\\n. \\nCarrillo JA et al. Disposition of fluvoxamine in humans is determined by the polymorphic CYP2D6 and also by the CYP1A2\\nactivity. Clin Pharmacol Ther 1996;60:183-90.\\nDate 14-05-2018\\nCYP2D6 IM: gefitinib\\n4871\\nNO action is needed for this gene-drug interaction.\\nSide effects can occur more frequently, as the gene variation increases the gefitinib plasma concentration. However, the side effects are\\nreversible and manageable, to an extent that adjustment of the therapy in advance is not necessary.\\nLiterature:\\n1\\n. \\nHirose T et al. Association of pharmacokinetics and pharmacogenomics with safety and efficacy of gefitinib in patients with EGFR\\nmutation positive advanced non-small cell lung cancer. Lung Cancer 2016;93:69-76.\\n2\\n. \\nSugiyama E et al. Impact of single nucleotide polymorphisms on severe hepatotoxicity induced by EGFR tyrosine kinase\\ninhibitors in patients with non-small cell lung cancer harboring EGFR mutations. Lung Cancer 2015;90:307-13.\\n3\\n. \\nKobayashi H et al. Relationship among gefitinib exposure, polymorphisms of its metabolizing enzymes and transporters, and side\\neffects in Japanese patients with non-small-cell lung cancer. Clin Lung Cancer 2015;16:274-81.\\n4\\n. \\nTakimoto T et al. Polymorphisms of CYP2D6 gene and gefitinib-induced hepatotoxicity. Clinical Lung Cancer 2013;14:502-7.\\n5\\n. \\nSuzumura T et al. Reduced CYP2D6 function is associated with gefitinib-induced rash in patients with non-small cell lung cancer.\\nBMC Cancer 2012;12:568.\\n6\\n. \\nChhun S et al. Gefitinib-phenytoin interaction is not correlated with the C-erythromycin breath test in healthy male volunteers. Br\\nJ Clin Pharmacol 2009;68:226-37.\\nDate 19-11-2018\\nCYP2D6 PM: gefitinib\\n4872\\nNO action is needed for this gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 70}),\n",
       " Document(page_content='The gefitinib plasma concentration may increase due to reduced CYP2D6 activity. However, there is no evidence to suggest that side\\neffects increase to an extent that adjustment of therapy is needed.\\nLiterature:\\n1\\n. \\nChhun S et al. Gefitinib-phenytoin interaction is not correlated with the C-erythromycin breath test in healthy male volunteers. Br\\nJ Clin Pharmacol 2009;68:226-37.\\n2\\n. \\nSwaisland HC et al. Exploring the relationship between expression of cytochrome P450 enzymes and gefitinib pharmacokinetics.\\nClin Pharmacokinet 2006;45:633-44.\\n3\\n. \\nSPC Iressa.\\nDate 19-11-2018\\nCYP2D6 UM: gefitinib\\n4873\\nNO action is needed for this gene-drug interaction.\\nThe gene variation may lead to a decrease in the gefitinib plasma concentration. In practice, an alternative is only chosen if non-response\\nto gefitinib has been proved. Moreover, dose adjustments guided by the gefitinib plasma concentration are rarely performed in clinical\\npractice as the analytical method is not available in most hospitals.\\nLiterature:\\n1\\n. \\nChhun S et al. Gefitinib-phenytoin interaction is not correlated with the C-erythromycin breath test in healthy male volunteers. Br\\nJ Clin Pharmacol 2009;68:226-37.\\n2\\n. \\nSwaisland HC et al. Exploring the relationship between expression of cytochrome P450 enzymes and gefitinib pharmacokinetics.\\nClin Pharmacokinet 2006;45:633-44.\\n3\\n. \\nSPC Iressa.\\nDate 19-11-2018\\nCYP2C9 IM: glibenclamide\\n1882\\nNO action is required for this gene-drug interaction.\\nThe only relevant clinical consequence that was found is an increased effectiveness of glibenclamide without an increase in the frequency\\nand severity of hypoglycaemia.\\nLiterature:\\n1\\n. \\nSurendiran A et al. Influence of CYP2C9 gene polymorphisms on response to glibenclamide in type 2 diabetes mellitus patients.\\nEur J Clin Pharmacol 2011;67:797-801.\\n2\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3\\n. \\nYin OQ et al. CYP2C9, but not CYP2C19, polymorphisms affect the pharmacokinetics and pharmacodynamics of glyburide in\\nChinese subjects. Clin Pharmacol Ther 2005;78:370-7.\\n4\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\n5\\n. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response\\nin healthy volunteers. Clin Pharmacol Ther 2002;71:286-96.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 71}),\n",
       " Document(page_content='Date 20-11-2017\\nCYP2C9 PM: glibenclamide\\n1883\\nNO action is required for this gene-drug interaction.\\nNo relevant clinical consequences have been found for the genetic variation.\\nLiterature:\\n1\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n2\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n3\\n. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response\\nin healthy volunteers. Clin Pharmacol Ther 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*1/*2: glibenclamide\\n1877\\nNO action is required for this gene-drug interaction.\\nThe only relevant clinical consequence that was found is an increased effectiveness of glibenclamide without an increase in frequency\\nand severity of hypoglycaemia for a group of 1 *1/*2 and 15 *1/*3.\\nLiterature:\\n1\\n. \\nSurendiran A et al. Influence of CYP2C9 gene polymorphisms on response to glibenclamide in type 2 diabetes mellitus patients.\\nEur J Clin Pharmacol 2011;67:797-801.\\n2\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\n4\\n. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response\\nin healthy volunteers. Clin Pharmacol Ther 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*1/*3: glibenclamide\\n1878\\nNO action is required for this gene-drug interaction.\\nThe only relevant clinical consequence is an increased effectiveness of glibenclamide without an increase in frequency and severity of\\nhypoglycaemia.\\nLiterature:\\n1\\n. \\nSurendiran A et al. Influence of CYP2C9 gene polymorphisms on response to glibenclamide in type 2 diabetes mellitus patients.\\nEur J Clin Pharmacol 2011;67:797-801.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 72}),\n",
       " Document(page_content='2\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3\\n. \\nYin OQ et al. CYP2C9, but not CYP2C19, polymorphisms affect the pharmacokinetics and pharmacodynamics of glyburide in\\nChinese subjects. Clin Pharmacol Ther 2005;78:370-7.\\n4\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\n5\\n. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response\\nin healthy volunteers. Clin Pharmacol Ther 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*2/*2: glibenclamide\\n1879\\nNO action is required for this gene-drug interaction.\\nNo significant clinical consequences have been found for the genetic variation.\\nLiterature:\\n1\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n2\\n. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response\\nin healthy volunteers. Clin Pharmacol Ther 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*2/*3: glibenclamide\\n1880\\nNO action is required for this gene-drug interaction.\\nNo significant kinetic or clinical consequences have been found for this genetic variation.\\nLiterature:\\n1\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n2\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n3\\n. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response\\nin healthy volunteers. Clin Pharmacol Ther 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9*3/*3: glibenclamide\\n1881\\nNO action is required for this gene-drug interaction.\\nNo relevant clinical consequences have been found for this genetic variation.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 73}),\n",
       " Document(page_content=\"Literature:\\n1\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n2\\n. \\nKirchheiner J et al. Impact of CYP2C9 amino acid polymorphisms on glyburide kinetics and on the insulin and glucose response\\nin healthy volunteers. Clin Pharmacol Ther 2002;71:286-96.\\nDate 20-11-2017\\nCYP2C9 IM: gliclazide\\n1889\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1\\n. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J\\nDiabetes Investig 2016;7:764-8.\\n2\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n3\\n. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS\\nstudy. Clin Pharmacol Ther 2010;87:52-6.\\n4\\n. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas.\\nPharmacogenomics 2009;10:1781-7.\\n5\\n. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J\\nPharmacol 2008;153:1579-86.\\n6\\n. \\nZhang Y et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on pharmacokinetics of gliclazide MR in Chinese\\nsubjects. Br J Clin Pharmacol 2007;64:67-74.\\nDate 20-11-2017\\nCYP2C9 PM: gliclazide\\n1890\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1\\n. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J\\nDiabetes Investig 2016;7:764-8.\\n2\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n3\\n. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS\\nstudy. Clin Pharmacol Ther 2010;87:52-6.\\n4\\n. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J\\nPharmacol 2008;153:1579-86.\\nDate 20-11-2017\\nCYP2C9*1/*2: gliclazide\\n1884\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 74}),\n",
       " Document(page_content=\"NO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide\\nLiterature:\\n1\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n2\\n. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS\\nstudy. Clin Pharmacol Ther 2010;87:52-6.\\n3\\n. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas.\\nPharmacogenomics 2009;10:1781-7.\\n4\\n. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J\\nPharmacol 2008;153:1579-86.\\nDate 20-11-2017\\nCYP2C9*1/*3: gliclazide\\n1885\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1\\n. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J\\nDiabetes Investig 2016;7:764-8.\\n2\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n3\\n. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS\\nstudy. Clin Pharmacol Ther 2010;87:52-6.\\n4\\n. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas.\\nPharmacogenomics 2009;10:1781-7.\\n5\\n. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J\\nPharmacol 2008;153:1579-86.\\n6\\n. \\nZhang Y et al. Influence of CYP2C9 and CYP2C19 genetic polymorphisms on pharmacokinetics of gliclazide MR in Chinese\\nsubjects. Br J Clin Pharmacol 2007;64:67-74.\\nDate 20-11-2017\\nCYP2C9*2/*2: gliclazide\\n1886\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide.\\nLiterature:\\n1\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n2\\n. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS\\nstudy. Clin Pharmacol Ther 2010;87:52-6.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 75}),\n",
       " Document(page_content=\"3\\n. \\nXu H et al. Effects of St John's wort and CYP2C9 genotype on the pharmacokinetics and pharmacodynamics of gliclazide. Br J\\nPharmacol 2008;153:1579-86.\\nDate 20-11-2017\\nCYP2C9*2/*3: gliclazide\\n1887\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide.\\nLiterature:\\n1\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n2\\n. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS\\nstudy. Clin Pharmacol Ther 2010;87:52-6.\\nDate 20-11-2017\\nCYP2C9*3/*3: gliclazide\\n1888\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of gliclazide without significantly increasing the risk of hypoglycaemia.\\nLiterature:\\n1\\n. \\nZeng W et al. CYP2C93 variant is associated with antidiabetes efficacy of gliclazide in Chinese type 2 diabetes patients. J\\nDiabetes Investig 2016;7:764-8.\\n2\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n3\\n. \\nZhou K et al. Loss-of-function CYP2C9 variants improve therapeutic response to sulfonylureas in type 2 diabetes: a Go-DARTS\\nstudy. Clin Pharmacol Ther 2010;87:52-6.\\nDate 20-11-2017\\nCYP2C9 IM: glimepiride\\n1896\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of glimepiride.\\nLiterature:\\n1\\n. \\nBhatt D et al. Investigating the role of plasma glucose concentration as a phenotypic marker for CYP2C9 genetic variants, in the\\ndiabetic population of Gujarat. Indian J Pharm Sci 2014;76:72-7.\\n2\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 76}),\n",
       " Document(page_content='3\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n4\\n. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas.\\nPharmacogenomics 2009;10:1781-7.\\n5\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n6\\n. \\nSuzuki K et al. Effect of CYP2C9 genetic polymorphisms on the efficacy and pharmacokinetics of glimepiride in subjects with\\ntype 2 diabetes. Diabetes Res Clin Pract 2006;72:148-54.\\n7\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n8\\n. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther\\n2005;78:90-2.\\n9\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9 PM: glimepiride\\n1897\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation increases the risk of hypoglycaemia, this risk remains low. This also means that the genetic variation\\nincreases the effectiveness. A lack of effectiveness is a much more common problem with sulphonyl urea derivatives than the occurrence\\nof hypoglycaemia.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n4\\n. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther\\n2005;78:90-2.\\n5\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*1/*2: glimepiride\\n1891\\nNO action is required for this gene-drug interaction.\\nNo significant kinetic or clinical consequences have been found for the genetic variation.\\nLiterature:\\n1\\n. \\nBhatt D et al. Investigating the role of plasma glucose concentration as a phenotypic marker for CYP2C9 genetic variants, in the\\ndiabetic population of Gujarat. Indian J Pharm Sci 2014;76:72-7.\\n2\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n3\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n4\\n. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas.\\nPharmacogenomics 2009;10:1781-7.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 77}),\n",
       " Document(page_content='5\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n6\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n7\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*1/*3: glimepiride\\n1892\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases the effectiveness of glimepiride.\\nLiterature:\\n1\\n. \\nBhatt D et al. Investigating the role of plasma glucose concentration as a phenotypic marker for CYP2C9 genetic variants, in the\\ndiabetic population of Gujarat. Indian J Pharm Sci 2014;76:72-7.\\n2\\n. \\nGökalp O et al. Mild hypoglycaemic attacks induced by sulphonylureas related to CYP2C9, CYP2C19 and CYP2C8\\npolymorphisms in routine clinical setting. Eur J Clin Pharmacol 2011;67:1223-9.\\n3\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n4\\n. \\nRagia G et al. Presence of CYP2C9*3 allele increases risk for hypoglycemia in Type 2 diabetic patients treated with sulfonylureas.\\nPharmacogenomics 2009;10:1781-7.\\n5\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n6\\n. \\nSuzuki K et al. Effect of CYP2C9 genetic polymorphisms on the efficacy and pharmacokinetics of glimepiride in subjects with\\ntype 2 diabetes. Diabetes Res Clin Pract 2006;72:148-54.\\n7\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n8\\n. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther\\n2005;78:90-2.\\n9\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*2/*2: glimepiride\\n1893\\nNO action is required for this gene-drug interaction.\\nNo significant clinical effects were observed for this genetic variation.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\nDate 20-11-2017', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 78}),\n",
       " Document(page_content='CYP2C9*2/*3: glimepiride\\n1894\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation increases the risk of hypoglycaemia, this risk remains low. This also means that the genetic variation\\nincreases the effectiveness. A lack of effectiveness is a much more common problem with sulphonyl urea derivatives than the occurrence\\nof hypoglycaemia.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nBecker ML et al. Cytochrome P450 2C9 *2 and *3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n4\\n. \\nNiemi M et al. Glyburide and glimepiride pharmacokinetics in subjects with different CYP2C9 genotypes. Clin Pharmacol Ther\\n2002;72:326-32.\\nDate 20-11-2017\\nCYP2C9*3/*3: glimepiride\\n1895\\nNO action is required for this gene-drug interaction.\\nAlthough the genetic variation increases the risk of hypoglycaemia, this risk remains low. This also means that the genetic variation\\nincreases the effectiveness. A lack of effectiveness is a much more common problem with sulphonyl urea derivatives than the occurrence\\nof hypoglycaemia.\\nLiterature:\\n1\\n. \\nHolstein A et al. Association between CYP2C9 slow metabolizer genotypes and severe hypoglycaemia on medication with\\nsulphonylurea hypoglycaemic agents. Br J Clin Pharmacol 2005;60:103-6.\\n2\\n. \\nWang R et al. Pharmacokinetics of glimepiride and cytochrome P450 2C9 genetic polymorphisms. Clin Pharmacol Ther\\n2005;78:90-2.\\nDate 20-11-2017\\nCYP2D6 IM: haloperidol\\n1551\\nNO action is required for this gene-drug interaction.\\nLiterature:\\n1\\n. \\nPanagiotidis G et al. Depot haloperidol treatment in outpatients with schizophrenia on monotherapy: impact of CYP2D6\\npolymorphism on pharmacokinetics and treatment outcome. Ther Drug Monit 2007;29:417-22.\\n2\\n. \\nPark JY et al. Combined effects of itraconazole and CYP2D6*10 genetic polymorphism on the pharmacokinetics and\\npharmacodynamics of haloperidol in healthy subjects. J Clin Psychopharmacol 2006;26:135-42.\\n3\\n. \\nLLerena A et al. Relationship between haloperidol plasma concentration, debrisoquine metabolic ratio, CYP2D6 and CYP2C9\\ngenotypes in psychiatric patients. Pharmacopsychiatry 2004;37:69-73.\\n4\\n. \\nDesai M et al. Pharmacokinetics and QT interval pharmacodynamics of oral haloperidol in poor and extensive metabolizers of\\nCYP2D6. Pharmacogenomics J 2003;3:105-13.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 79}),\n",
       " Document(page_content='5\\n. \\nBrockmoller J et al. The impact of the CYP2D6 polymorphism on haloperidol pharmacokinetics and on the outcome of\\nhaloperidol treatment. Clin Pharmacol Ther 2002;72:438-52.\\n6\\n. \\nYasui-Furukori N et al. Effect of the CYP2D6 genotype on prolactin concentration in schizophrenic patients treated with\\nhaloperidol. Schizophr Res 2001 1;52:139-42.\\n7\\n. \\nPan L et al. Effects of smoking, CYP2D6 genotype, and concomitant drug intake on the steady state plasma concentrations of\\nhaloperidol and reduced haloperidol in schizophrenic inpatients. Ther Drug Monit 1999;21:489-97.\\n8\\n. \\nLlerena A et al. Haloperidol disposition is dependent on the debrisoquine hydroxylation phenotype: increased plasma levels of the\\nreduced metabolite in poor metabolizers. Ther Drug Monit 1992;14:261-4.\\n9\\n. \\nLlerena A. Haloperidol disposition is dependent on debrisoquine hydroxylation phenotype. Ther Drug Monit 1992;14:92-7.\\nDate 22-03-2006\\nCYP2D6 PM: haloperidol\\n1552\\nThe genetic polymorphism leads to decreased metabolic capacity of CYP2D6, which may cause increased plasma concentrations of\\nhaloperidol and the active metabolite.\\nRecommendation:\\n1\\n. \\nAdvise the prescriber to:\\n1\\n. \\ndecrease the initial dose to 50% of the standard initial dose and adjust the dose according to the effect,\\n2\\n. \\nor prescribe an alternative.\\nAnti-psychotics that are not metabolised via CYP2D6 - or to a much lesser extent - include, for example, flupentixol,\\nfluphenazine, quetiapine, olanzapine or clozapine.\\nLiterature:\\n1\\n. \\nPanagiotidis G et al. Depot haloperidol treatment in outpatients with schizophrenia on monotherapy: impact of CYP2D6\\npolymorphism on pharmacokinetics and treatment outcome. Ther Drug Monit 2007;29:417-22.\\n2\\n. \\nLLerena A et al. Relationship between haloperidol plasma concentration, debrisoquine metabolic ratio, CYP2D6 and CYP2C9\\ngenotypes in psychiatric patients. Pharmacopsychiatry 2004;37:69-73.\\n3\\n. \\nDesai M et al. Pharmacokinetics and QT interval pharmacodynamics of oral haloperidol in poor and extensive metabolizers of\\nCYP2D6. Pharmacogenomics J 2003;3:105-13.\\n4\\n. \\nBrockmoller J et al. The impact of the CYP2D6 polymorphism on haloperidol pharmacokinetics and on the outcome of\\nhaloperidol treatment. Clin Pharmacol Ther 2002;72:438-52.\\n5\\n. \\nYasui-Furukori N et al. Effect of the CYP2D6 genotype on prolactin concentration in schizophrenic patients treated with\\nhaloperidol. Schizophr Res 2001 1;52:139-42.\\n6\\n. \\nPan L et al. Effects of smoking, CYP2D6 genotype, and concomitant drug intake on the steady state plasma concentrations of\\nhaloperidol and reduced haloperidol in schizophrenic inpatients. Ther Drug Monit 1999;21:489-97.\\n7\\n. \\nLlerena A et al. Haloperidol disposition is dependent on the debrisoquine hydroxylation phenotype: increased plasma levels of the\\nreduced metabolite in poor metabolizers. Ther Drug Monit 1992;14:261-4.\\n8\\n. \\nLlerena A. Haloperidol disposition is dependent on debrisoquine hydroxylation phenotype. Ther Drug Monit 1992;14:92-7.\\nDate 22-03-2006\\nCYP2D6 UM: haloperidol\\n1553\\nThe genetic polymorphism leads to increased metabolic capacity of CYP2D6, which may cause decreased plasma concentrations of\\nhaloperidol and the active metabolite reduced haloperidol.\\nRecommendation:\\nIt is not possible to offer substantiated advice for dose adjustment due to the limited amount of available literature.\\n1\\n. \\nAdvise the prescriber to:\\n1\\n. \\nbe alert to possible reduced plasma concentrations of haloperidol and reduced haloperidol and increase the dose based on\\nresults of therapeutic drug monitoring,\\n2\\n. \\nor prescribe an alternative according to the current guidelines.\\nAnti-psychotics that are not metabolised via CYP2D6 - or to a much lesser extent - include, for example, flupentixol,\\nfluphenazine, quetiapine, olanzapine or clozapine.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 80}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nPanagiotidis G et al. Depot haloperidol treatment in outpatients with schizophrenia on monotherapy: impact of CYP2D6\\npolymorphism on pharmacokinetics and treatment outcome. Ther Drug Monit 2007;29:417-22.\\n2\\n. \\nBrockmoller J ea. The impact of the CYP2D6 polymorphism on haloperidol pharmacokinetics and on the outcome of haloperidol\\ntreatment. Clin Pharmacol Ther 2002;72:438-52.\\nDate 22-03-2006\\nCYP2C19 IM: imipramine\\n1913\\nNO action is required for this gene-drug interaction.\\nThe genetic variation increases imipramine plasma concentrations, but not imipramine+desipramine plasma concentrations, which\\ngovern effectiveness and side effects.\\nLiterature:\\n1\\n. \\nSchenk PW et al. The CYP2C19*17 genotype is associated with lower imipramine plasma concentrations in a large group of\\ndepressed patients. Pharmacogenomics J 2010;10:219-25.\\n2\\n. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large\\ngroup of depressed patients. Mol Psychiatry 2008;13:597-605.\\n3\\n. \\nMadsen H et al. Imipramine demethylation in vivo: impact of CYP1A2, CYP2C19, and CYP3A4. Clin Pharmacol Ther\\n1997;61:319-24.\\n4\\n. \\nKoyama E et al. Steady-state plasma concentrations of imipramine and desipramine in relation to S-mephenytoin 4-hydroxylation\\nstatus in Japanese depressive patients. J Clin Psychopharmacol 1996;16:286-93.\\n5\\n. \\nMadsen H et al. Imipramine metabolism in relation to the sparteine and mephenytoin oxidation polymorphisms--a population\\nstudy. Br J Clin Pharmacol 1995;39:433-9.\\n6\\n. \\nKoyama E et al. Metabolic disposition of imipramine in oriental subjects: relation to metoprolol alpha-hydroxylation and S-\\nmephenytoin 4-hydroxylation phenotypes. J Pharmacol Exp Ther 1994;271:860-7.\\n7\\n. \\nSkjelbo E et al. The N-demethylation of imipramine correlates with the oxidation of S-mephenytoin (S/R-ratio). A population\\nstudy. Br J Clin Pharmacol 1993;35:331-4.\\nDate 10-09-2018\\nCYP2C19 PM: imipramine\\n1914\\nThe risk of side effects is increased. The gene variation results in an increase in the plasma concentration of imipramine+desipramine.\\nuse 70% of the standard dose and monitor the effect and side effects, or the imipramine and desipramine plasma concentrations to\\ndetermine the maintenance dose.\\nor avoid imipramine\\nAntidepressants that are not or to a lesser extent metabolised by CYP2C19 include, for example, nortriptyline, fluvoxamine and\\nmirtazapine.\\nLiterature:\\n1\\n. \\nSchenk PW et al. The CYP2C19*17 genotype is associated with lower imipramine plasma concentrations in a large group of\\ndepressed patients. Pharmacogenomics J 2010;10:219-25.\\n2\\n. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large\\ngroup of depressed patients. Mol Psychiatry 2008;13:597-605.\\n3\\n. \\nMorinobu S et al. Effects of genetic defects in the CYP2C19 gene on the N-demethylation of imipramine, and clinical outcome of', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 81}),\n",
       " Document(page_content='imipramine therapy. Psychiatry Clin Neurosci 997;51:253-7.\\n4\\n. \\nMadsen H et al. Imipramine demethylation in vivo: impact of CYP1A2, CYP2C19, and CYP3A4. Clin Pharmacol Ther\\n1997;61:319-24.\\n5\\n. \\nKoyama E et al. Steady-state plasma concentrations of imipramine and desipramine in relation to S-mephenytoin 4-hydroxylation\\nstatus in Japanese depressive patients. J Clin Psychopharmacol 1996;16:286-93.\\n6\\n. \\nMadsen H et al. Imipramine metabolism in relation to the sparteine and mephenytoin oxidation polymorphisms--a population\\nstudy. Br J Clin Pharmacol 1995;39:433-9.\\n7\\n. \\nKoyama E et al. Metabolic disposition of imipramine in oriental subjects: relation to metoprolol alpha-hydroxylation and S-\\nmephenytoin 4-hydroxylation phenotypes. J Pharmacol Exp Ther 1994;271:860-7.\\n8\\n. \\nSkjelbo E et al. The N-demethylation of imipramine correlates with the oxidation of S-mephenytoin (S/R-ratio). A population\\nstudy. Br J Clin Pharmacol 1993;35:331-4.\\n9\\n. \\nSkjelbo E et al. The mephenytoin oxidation polymorphism is partially responsible for the N-demethylation of imipramine. Clin\\nPharmacol Ther 1991;49:18-23.\\nDate 10-09-2018\\nCYP2C19 UM: imipramine\\n1915\\nNO action is required for this gene-drug interaction.\\nThe genetic variation decreases imipramine plasma concentrations, but not imipramine+desipramine plasma concentrations, which\\ngovern effectiveness and side effects.\\nLiterature:\\n1\\n. \\nSchenk PW et al. The CYP2C19*17 genotype is associated with lower imipramine plasma concentrations in a large group of\\ndepressed patients. Pharmacogenomics J 2010;10:219-25.\\nDate 10-09-2018\\nCYP2D6 IM: imipramine\\n1545\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of imipramine and\\ndesipramine.\\nuse 70% of the standard dose and monitor the effect and side effects or the plasma concentrations of imipramine and desipramine\\nin order to set the maintenance dose\\nThe therapeutic range is 150-300 ng/mL for the sum of the imipramine and desipramine plasma concentrations. Values exceeding\\n500 ng/mL are considered toxic.\\nLiterature:\\n1\\n. \\nSchliessbach J et al. Effect of single-dose imipramine on chronic low-back and experimental pain. A randomized controlled trial.\\nPLoS One 2018;13:e0195776.\\n2\\n. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large\\ngroup of depressed patients. Mol Psychiatry 2008;13:597-605.\\n3\\n. \\nSindrup SH et al. Nonlinear kinetics of imipramine in low and medium plasma level ranges. Ther Drug Monit 1990;12:445-9.\\n4\\n. \\nBrosen K et al. Imipramine demethylation and hydroxylation: impact of the sparteine oxidation phenotype. Clin Pharmacol Ther\\n1986;40:543-9.\\nDate 19-11-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 82}),\n",
       " Document(page_content='CYP2D6 PM: imipramine\\n1544\\nThe risk of side effects may be increased, because the gene variation leads to increased plasma concentrations of imipramine and the\\nactive metabolite desipramine.\\nuse 30% of the standard dose and monitor the effect and side effects or the plasma concentrations of imipramine and desipramine\\nin order to set the maintenance dose\\nThe therapeutic range is 150-300 ng/mL for the sum of the imipramine and desipramine plasma concentrations. Values exceeding\\n500 ng/mL are considered toxic.\\nLiterature:\\n1\\n. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large\\ngroup of depressed patients. Mol Psychiatry 2008;13:597-605.\\n2\\n. \\nKoyama E et al. Metabolic disposition of imipramine in oriental subjects: relation to metoprolol alpha-hydroxylation and S-\\nmephenytoin 4\"-hydroxylation phenotypes. J Pharmacol Exp Ther 1994;271:860-7.\\n3\\n. \\nSindrup SH et al. Nonlinear kinetics of imipramine in low and medium plasma level ranges. Ther Drug Monit 1990;12:445-9.\\n4\\n. \\nBalant-Gorgia AE et al. High blood concentrations of imipramine or clomipramine and therapeutic failure: a case report study\\nusing drug monitoring data. Ther Drug Monit 1989;11:415-20.\\n5\\n. \\nBrosen K et al. Imipramine demethylation and hydroxylation: impact of the sparteine oxidation phenotype. Clin Pharmacol Ther\\n1986;40:543-9.\\n6\\n. \\nBrosen K et al. Steady-state concentrations of imipramine and its metabolites in relation to the sparteine/debrisoquine\\npolymorphism. Eur J Clin Pharmacol 1986;30:679-84.\\n7\\n. \\nSmPC Tofranil-PM (VS).\\nDate 19-11-2018\\nCYP2D6 UM: imipramine\\n1546\\nThe risk of ineffectiveness and cardiotoxic side effects may be increased. The gene variation leads to reduced plasma concentrations of\\nimipramine and the active metabolite desipramine and to increased plasma concentrations of the potentially cardiotoxic hydroxy\\nmetabolites.\\nuse 1.7 times the standard dose and monitor the effect and side effects or the plasma concentrations of imipramine and desipramine\\nin order to set the maintenance dose\\nif a dose increase is not wanted due to the potentially cardiotoxic hydroxy metabolites: avoid imipramine.\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\nLiterature:\\n1\\n. \\nSchenk PW et al. Association of graded allele-specific changes in CYP2D6 function with imipramine dose requirement in a large\\ngroup of depressed patients. Mol Psychiatry 2008;13:597-605.\\n2\\n. \\nSindrup SH et al. Nonlinear kinetics of imipramine in low and medium plasma level ranges. Ther Drug Monit 1990;12:445-9.\\nDate 19-11-2018\\nUGT1A1 *1/*28: irinotecan\\n1693\\nNO action is needed for this gene-drug interaction.\\nThis genetic variation (*1/*28) is more common in Western populations than the wild-type (*1/*1). This means that treatment is largely\\ngeared to patients with this genetic variation. Adjustment of the treatment is therefore not useful.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 83}),\n",
       " Document(page_content=\"Literature:\\n1\\n. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a\\nmeta-analysis. Cancer Chemother Pharmacol 2017;79:1109-1117.\\n2\\n. \\nLiu XH et al. Predictive value of UGT1A1*28 polymorphism in irinotecan-based chemotherapy. J Cancer 2017;8:691-703.\\n3\\n. \\nDias MM et al. The effect of the UGT1A1*28 allele on survival after irinotecan-based chemotherapy: a collaborative meta-\\nanalysis. Pharmacogenomics J 2014;14:424-31.\\n4\\n. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis.\\nBiomarkers. 2014;19:56-62.\\n5\\n. \\nLiu X et al. Association of UGT1A1*28 polymorphisms with irinotecan-induced toxicities in colorectal cancer: a meta-analysis in\\nCaucasians. Pharmacogenomics J 2014;14:120-9.\\n6\\n. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs\\n2013;31:1559-67.\\n7\\n. \\nLiu X et al. Association between UGT1A1*28 polymorphisms and clinical outcomes of irinotecan-based chemotherapies in\\ncolorectal cancer: a meta-analysis in Caucasians. PLoS One 2013;8:e58489.\\n8\\n. \\nDias MM et al. Impact of the UGT1A1*28 allele on response to irinotecan: a systematic review and meta-analysis.\\nPharmacogenomics 2012;13:889-99.\\n9\\n. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 genotype and irinotecan-induced neutropenia: low doses also\\nincrease risk. Clin Cancer Res 2010;16:3832-42.\\n10\\n. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 polymorphism and irinotecan-induced diarrhoea: a meta-analysis.\\nEur J Cancer 2010;46:1856-65.\\n11\\n. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer\\nChemother Pharmacol 2009;65:97-105.Kweekel DM et al. UGT1A1*28 genotype and irinotecan dosage in patients with\\nmetastatic colorectal cancer: a Dutch Colorectal Cancer Group study. Br J Cancer 2008;99:275-82.\\n12\\n. \\nLiu CY et al. UGT1A1*28 polymorphism predicts irinotecan-induced severe toxicities without affecting treatment outcome and\\nsurvival in patients with metastatic colorectal carcinoma. Cancer 2008;112:1932-40.\\n13\\n. \\nLankisch TO et al. Gilbert's Syndrome and irinotecan toxicity: combination with UDP-glucuronosyl-transferase 1A7 variants\\nincreases risk. Cancer Epidemiol Biomarkers Prev 2008;17:695-701.\\n14\\n. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of\\nUGT1A1\\n6 and \\n28. Pharmacogenet Genomics 2007;17:497-504.\\n15\\n. \\nStewart CF et al. UGT1A1 promoter genotype correlates with SN-38 pharmacokinetics, but not severe toxicity in patients\\nreceiving low-dose irinotecan. J Clin Oncol 2007;25:2594-600.\\n16\\n. \\nCôté JF et al. UGT1A1 polymorphism can predict hematologic toxicity in patients treated with irinotecan. Clin Cancer Res\\n2007;13:3269-75.\\n17\\n. \\nRamchandani RP et al. The role of SN-38 exposure, UGT1A1*28 polymorphism, and baseline bilirubin level in predicting severe\\nirinotecan toxicity. J Clin Pharmacol 2007;47:78-86.\\n18\\n. \\nZárate Romero R et al. Potential application of GSTT1-null genotype in predicting toxicity associated to 5-fluouracil irinotecan\\nand leucovorin regimen in advanced stage colorectal cancer patients. Oncol Rep 2006;16:497-503.\\n19\\n. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype\\nscreening: a double-blind, randomized, placebo-controlled study. Oncologist 2006;11:944-54.\\n20\\n. \\nToffoli G et al. The role of UGT1A1*28 polymorphism in the pharmacodynamics and pharmacokinetics of irinotecan in patients\\nwith metastatic colorectal cancer. J Clin Oncol 2006;24:3061-8.\\n21\\n. \\nHan JY et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in\\npatients with non-small-cell lung cancer treated with irinotecan and cisplatin. J Clin Oncol 2006;24:2237-44.\\n22\\n. \\nMcLeod HL et al. UGT1A1*28, toxicity and outcome in advanced colorectal cancer: results from Trial N9741. J Clin Oncol\\n2006;24 (suppl. abstr. 3520).\\n23\\n. \\nMassacesi C et al. Uridine diphosphate glucuronosyl transferase 1A1 promoter polymorphism predicts the risk of gastrointestinal\\ntoxicity and fatigue induced by irinotecan-based chemotherapy. Cancer 2006;106:1007-16.\\n24\\n. \\nWright MA et al. A phase I pharmacologic and pharmacogenetic trial of sequential 24-hour infusion of irinotecan followed by\\nleucovorin and a 48-hour infusion of fluorouracil in adult patients with solid tumors. Clin Cancer Res 2005;11:4144-50.\\n25\\n. \\nKweekel DM et al. Ondersteuning van de chemotherapiekeuze. Pharm Weekblad 2005;20:685-7.\\n26\\n. \\nSteiner M et al. 5-fluorouracil/irinotecan induced lethal toxicity as a result of a combined pharmacogenetic syndrome: report of a\\ncase. J Clin Pathol 2005;58:553-5.\\n27\\n. \\nSoepenberg O et al. Phase I pharmacokinetic, food effect, and pharmacogenetic study of oral irinotecan given as semisolid matrix\\ncapsules in patients with solid tumors. Clin Cancer Res 2005;11:1504-11.\\n28\\n. \\nZhou Q et al. Pharmacogenetic profiling across the irinotecan pathway in Asian patients with cancer. Br J Clin Pharmacol\\n2005;59:415-24.\\n29\\n. \\nMarcuello E et al. UGT1A1 gene variations and irinotecan treatment in patients with metastatic colorectal cancer. Br J Cancer\\n2004;91:678-82.\\n30\\n. \\nRouits E et al. Relevance of different UGT1A1 polymorphisms in irinotecan-induced toxicity: a molecular and clinical study of 75\\npatients. Clin Cancer Res 2004;10:5151-9.\\n31\\n. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol\\n2004;44:854-60.\\n32\\n. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered\\nJapanese patients with cancer. Clin Pharmacol Ther 2004;75:501-15.\\n33\\n. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of\\nirinotecan. J Clin Oncol 2004;22:1382-8.\\n34\\n. \\nFont A et al. Weekly regimen of irinotecan/docetaxel in previously treated non-small cell lung cancer patients and correlation with\\nuridine diphosphate glucuronosyl-transferase 1A1 (UGT1A1) polymorphism. Invest New Drugs 2003;21:435-43.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 84}),\n",
       " Document(page_content=\"35\\n. \\nMathijssen RH et al. Irinotecan pathway genotype analysis to predict pharmacokinetics. Clin Cancer Res 2003;9:3246-53.\\n36\\n. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n37\\n. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer\\nRes 2000;60:6921-6.\\n38\\n. \\nWasserman E et al. Severe CPT-11 toxicity in patients with Gilbert’s syndrome: two case reports. Ann Oncol 1997;8:1049-51.\\nDate 05-03-2018\\nUGT1A1 *28/*28: irinotecan\\n1694\\nSerious, life-threatening adverse events occur more often in patients with this genetic variation. The genetic variation reduces conversion\\nof irinotecan to inactive metabolites.\\nStart with 70% of the standard dose\\nIf the patient tolerates this initial dose, the dose can be increased, guided by the neutrophil count.\\nLiterature:\\n1\\n. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a\\nmeta-analysis. Cancer Chemother Pharmacol 2017;79:1109-1117.\\n2\\n. \\nLiu XH et al. Predictive value of UGT1A1*28 polymorphism in irinotecan-based chemotherapy. J Cancer 2017;8:691-703.\\n3\\n. \\nLu CY et al. Clinical implication of UGT1A1 promoter polymorphism for irinotecan dose escalation in metastatic colorectal\\ncancer patients treated with bevacizumab combined with FOLFIRI in the first-line setting. Transl Oncol 2015;8:474-9.\\n4\\n. \\nDias MM et al. The effect of the UGT1A1*28 allele on survival after irinotecan-based chemotherapy: a collaborative meta-\\nanalysis. Pharmacogenomics J 2014;14:424-31.\\n5\\n. \\nHan FF et al. Associations between UGT1A1\\n6 or UGT1A1\\n6/*28 polymorphisms and irinotecan-induced neutropenia in Asian\\ncancer patients. Cancer Chemother Pharmacol 2014;73:779-88.\\n6\\n. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis.\\nBiomarkers. 2014;19:56-62.\\n7\\n. \\nLiu X et al. Association of UGT1A1*28 polymorphisms with irinotecan-induced toxicities in colorectal cancer: a meta-analysis in\\nCaucasians. Pharmacogenomics J 2014;14:120-9.\\n8\\n. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs\\n2013;31:1559-67.\\n9\\n. \\nLiu X et al. Association between UGT1A1*28 polymorphisms and clinical outcomes of irinotecan-based chemotherapies in\\ncolorectal cancer: a meta-analysis in Caucasians. PLoS One 2013;8:e58489.\\n10\\n. \\nDias MM et al. Impact of the UGT1A1*28 allele on response to irinotecan: a systematic review and meta-analysis.\\nPharmacogenomics 2012;13:889-99.\\n11\\n. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 genotype and irinotecan-induced neutropenia: low doses also\\nincrease risk. Clin Cancer Res 2010;16:3832-42.\\n12\\n. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 polymorphism and irinotecan-induced diarrhoea: a meta-analysis.\\nEur J Cancer 2010;46:1856-65.\\n13\\n. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer\\nChemother Pharmacol 2009;65:97-105.\\n14\\n. \\nKweekel DM et al. UGT1A1*28 genotype and irinotecan dosage in patients with metastatic colorectal cancer: a Dutch Colorectal\\nCancer Group study. Br J Cancer 2008;99:275-82.\\n15\\n. \\nLiu CY et al. UGT1A1*28 polymorphism predicts irinotecan-induced severe toxicities without affecting treatment outcome and\\nsurvival in patients with metastatic colorectal carcinoma. Cancer 2008;112:1932-40.\\n16\\n. \\nLankisch TO et al. Gilbert's Syndrome and irinotecan toxicity: combination with UDP-glucuronosyl-transferase 1A7 variants\\nincreases risk. Cancer Epidemiol Biomarkers Prev 2008;17:695-701.\\n17\\n. \\nHoskins JM et al. UGT1A1*28 genotype and irinotecan-induced neutropenia: dose matters. J Natl Cancer Inst 2007;99:1290-5.\\n18\\n. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of\\nUGT1A1\\n6 and \\n28. Pharmacogenet Genomics 2007;17:497-504.\\n19\\n. \\nStewart CF et al. UGT1A1 promoter genotype correlates with SN-38 pharmacokinetics, but not severe toxicity in patients\\nreceiving low-dose irinotecan. J Clin Oncol 2007;25:2594-600.\\n20\\n. \\nCôté JF et al. UGT1A1 polymorphism can predict hematologic toxicity in patients treated with irinotecan. Clin Cancer Res\\n2007;13:3269-75.\\n21\\n. \\nRamchandani RP et al. The role of SN-38 exposure, UGT1A1*28 polymorphism, and baseline bilirubin level in predicting severe\\nirinotecan toxicity. J Clin Pharmacol 2007;47:78-86.\\n22\\n. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype\\nscreening: a double-blind, randomized, placebo-controlled study. Oncologist 2006;11:944-54.\\n23\\n. \\nToffoli G et al. The role of UGT1A1*28 polymorphism in the pharmacodynamics and pharmacokinetics of irinotecan in patients\\nwith metastatic colorectal cancer. J Clin Oncol 2006;24:3061-8.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 85}),\n",
       " Document(page_content='24\\n. \\nMcLeod HL et al. UGT1A1*28, toxicity and outcome in advanced colorectal cancer: results from Trial N9741. J Clin Oncol\\n2006;24 (suppl. abstr. 3520).\\n25\\n. \\nMassacesi C et al. Uridine diphosphate glucuronosyl transferase 1A1 promoter polymorphism predicts the risk of gastrointestinal\\ntoxicity and fatigue induced by irinotecan-based chemotherapy. Cancer 2006;106:1007-16.\\n26\\n. \\nKweekel DM et al. Ondersteuning van de chemotherapiekeuze. Pharm Weekblad 2005;20:685-7.\\n27\\n. \\nSoepenberg O et al. Phase I pharmacokinetic, food effect, and pharmacogenetic study of oral irinotecan given as semisolid matrix\\ncapsules in patients with solid tumors. Clin Cancer Res 2005;11:1504-11.\\n28\\n. \\nZhou Q et al. Pharmacogenetic profiling across the irinotecan pathway in Asian patients with cancer. Br J Clin Pharmacol\\n2005;59:415-24.\\n29\\n. \\nCarlini LE et al. UGT1A7 and UGT1A9 polymorphisms predict response and toxicity in colorectal cancer patients treated with\\ncapecitabine/ irinotecan. Clin Cancer Res 2005;11:1226-36.\\n30\\n. \\nKitagawa C et al. Genetic polymorphism in the phenobarbital-responsive enhancer module of the UDP-glucuronosyltransferase\\n1A1 gene and irinotecan toxicity. Pharmacogenet Genomics 2005;15:35-41.\\n31\\n. \\nMarcuello E et al. UGT1A1 gene variations and irinotecan treatment in patients with metastatic colorectal cancer. Br J Cancer\\n2004;91:678-82.\\n32\\n. \\nRouits E et al. Relevance of different UGT1A1 polymorphisms in irinotecan-induced toxicity: a molecular and clinical study of 75\\npatients. Clin Cancer Res 2004;10:5151-9.\\n33\\n. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol\\n2004;44:854-60.\\n34\\n. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered\\nJapanese patients with cancer. Clin Pharmacol Ther 2004;75:501-15.\\n35\\n. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of\\nirinotecan. J Clin Oncol 2004;22:1382-8.\\n36\\n. \\nFont A et al. Weekly regimen of irinotecan/docetaxel in previously treated non-small cell lung cancer patients and correlation with\\nuridine diphosphate glucuronosyl-transferase 1A1 (UGT1A1) polymorphism. Invest New Drugs 2003;21:435-43.\\n37\\n. \\nMathijssen RH et al. Irinotecan pathway genotype analysis to predict pharmacokinetics. Clin Cancer Res 2003;9:3246-53.\\n38\\n. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n39\\n. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer\\nRes 2000;60:6921-6.\\n40\\n. \\nWasserman E et al. Severe CPT-11 toxicity in patients with Gilbert’s syndrome: two case reports. Ann Oncol 1997;8:1049-51.\\n41\\n. \\nSPC’s Campto en Camptosar (VS).\\nDate 05-03-2018\\nUGT1A1 IM: irinotecan\\n1691\\nNO action is needed for this gene-drug interaction.\\nThis genetic variation (IM) is more common in Western populations than the wild-type (\\n*1/\\n*1). This means that treatment is largely\\ngeared to patients with this genetic variation. Adjustment of the treatment is therefore not useful.\\nLiterature:\\n1\\n. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a\\nmeta-analysis. Cancer Chemother Pharmacol 2017;79:1109-1117.\\n2\\n. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis.\\nBiomarkers. 2014;19:56-62.\\n3\\n. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs\\n2013;31:1559-67.\\n4\\n. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer\\nChemother Pharmacol 2009;65:97-105.\\n5\\n. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of\\nUGT1A1\\n6 and \\n28. Pharmacogenet Genomics 2007;17:497-504.\\n6\\n. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype\\nscreening: a double-blind, randomized, placebo-controlled study. Oncologist 2006;11:944-54.\\n7\\n. \\nHan JY et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in\\npatients with non-small-cell lung cancer treated with irinotecan and cisplatin. J Clin Oncol 2006;24:2237-44.\\n8\\n. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol\\n2004;44:854-60.\\n9\\n. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered\\nJapanese patients with cancer. Clin Pharmacol Ther 2004;75:501-15.\\n10\\n. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 86}),\n",
       " Document(page_content='irinotecan. J Clin Oncol 2004;22:1382-8.\\n11\\n. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n12\\n. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer\\nRes 2000;60:6921-6.\\nDate 05-03-2018\\nUGT1A1 PM: irinotecan\\n1692\\nSerious, life-threatening adverse events occur more often in patients with this genetic variation. The genetic variation reduces conversion\\nof irinotecan to inactive metabolites.\\nStart with 70% of the standard dose\\nIf the patient tolerates this initial dose, the dose can be increased, guided by the neutrophil count.\\nLiterature:\\n1\\n. \\nChen X et al. UGT1A1 polymorphisms with irinotecan-induced toxicities and treatment outcome in Asians with lung cancer: a\\nmeta-analysis. Cancer Chemother Pharmacol 2017;79:1109-1117.\\n2\\n. \\nHan FF et al. Associations between UGT1A1\\n6 or UGT1A1\\n6/*28 polymorphisms and irinotecan-induced neutropenia in Asian\\ncancer patients. Cancer Chemother Pharmacol 2014;73:779-88.\\n3\\n. \\nChen YJ et al. The association of UGT1A1\\n6 and UGT1A1\\n28 with irinotecan-induced neutropenia in Asians: a meta-analysis.\\nBiomarkers. 2014;19:56-62.\\n4\\n. \\nGoetz MP et al. UGT1A1 genotype-guided phase I study of irinotecan, oxaliplatin, and capecitabine. Invest New Drugs\\n2013;31:1559-67.\\n5\\n. \\nHu ZY et al. Dose-dependent association between UGT1A1*28 polymorphism and irinotecan-induced diarrhoea: a meta-analysis.\\nEur J Cancer 2010;46:1856-65.\\n6\\n. \\nDenlinger CS et al. Pharmacokinetic analysis of irinotecan plus bevacizumab in patients with advanced solid tumors. Cancer\\nChemother Pharmacol 2009;65:97-105.\\n7\\n. \\nMinami H et al. Irinotecan pharmacokinetics/pharmacodynamics and UGT1A genetic polymorphisms in Japanese: roles of\\nUGT1A1\\n6 and \\n28. Pharmacogenet Genomics 2007;17:497-504.\\n8\\n. \\nde Jong FA et al. Prophylaxis of irinotecan-induced diarrhea with neomycin and potential role for UGT1A1*28 genotype\\nscreening: a double-blind, randomized, placebo-controlled study. Oncologist 2006;11:944-54.\\n9\\n. \\nHan JY et al. Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in\\npatients with non-small-cell lung cancer treated with irinotecan and cisplatin. J Clin Oncol 2006;24:2237-44.\\n10\\n. \\nPaoluzzi L et al. Influence of genetic variants in UGT1A1 and UGT1A9 on the in vivo glucuronidation of SN-38. J Clin Pharmacol\\n2004;44:854-60.\\n11\\n. \\nSai K et al. UGT1A1 haplotypes associated with reduced glucuronidation and increased serum bilirubin in irinotecan-administered\\nJapanese patients with cancer. Clin Pharmacol Ther 2004;75:501-15.\\n12\\n. \\nInnocenti F et al. Genetic variants in the UDP-glucuronosyl-transferase 1A1 gene predict the risk of severe neutropenia of\\nirinotecan. J Clin Oncol 2004;22:1382-8.\\n13\\n. \\nIyer L et al. UGT1A1*28 polymorphism as a determinant of irinotecan disposition and toxicity. Pharmacogenomics J 2002;2:43-7.\\n14\\n. \\nAndo Y et al. Polymorphisms of UDP-glucuronosyl-transferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer\\nRes 2000;60:6921-6.\\nDate 05-03-2018\\nCYP2D6 IM: kinidine\\n2534\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 87}),\n",
       " Document(page_content='Date 24-08-2016\\nCYP2D6 PM: kinidine\\n2533\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nNielsen F et al. Lack of relationship between quinidine pharmacokinetics and the sparteine oxidation polymorphism. Eur J Clin\\nPharmacol 1995;48:501-4.\\n2\\n. \\nBrøsen K et al. Quinidine kinetics after a single oral dose in relation to the sparteine oxidation polymorphism in man. Br J Clin\\nPharmacol 1990;29:248-53.\\n3\\n. \\nMikus G et al. Pharmacokinetics and metabolism of quinidine in extensive and poor metabolisers of sparteine. Eur J Clin\\nPharmacol 1986;31:69-72.\\nDate 24-08-2016\\nCYP2D6 UM: kinidine\\n2535\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2C19 IM: lansoprazol\\n1831\\nNO action is needed for this gene-drug interaction.\\nThe higher plasma concentration of lansoprazole results in an increase in the therapeutic effectiveness, without an increase in the\\nincidence of side effects.\\nLiterature:\\n1\\n. \\nLang JE et al. Lansoprazole is associated with worsening asthma control in children with the CYP2C19 poor metabolizer\\nphenotype. Ann Am Thorac Soc 2015 Jun;12:878-85.\\n2\\n. \\nLi CY et al. A correlative study of polymorphisms of CYP2C19 and MDR1 C3435T with the pharmacokinetic profiles of\\nlansoprazole and its main metabolites following single oral administration in healthy adult Chinese subjects. Eur J Drug Metab\\nPharmacokinet 2014;39:121-8.\\n3\\n. \\nLima JJ et al. Association of CYP2C19 polymorphisms and lansoprazole-associated respiratory adverse effects in children. J\\nPediatr 2013;163:686-91.\\n4\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n5\\n. \\nZhang D et al. Effects of CYP2C19 polymorphism on the pharmacokinetics of lansoprazole and its main metabolites in healthy\\nChinese subjects. Xenobiotica 2011;41:511-7.\\n6\\n. \\nTamura T et al. Improvements in Helicobacter pylori eradication rates through clinical CYP2C19 genotyping. Nagoya J Med Sci\\n2011;73:25-31.\\n7\\n. \\nXu HR et al. The effect of CYP2C19 activity on pharmacokinetics of lansoprazole and its active metabolites in healthy subjects.\\nPharm Biol 2010;48:947-52.\\n8\\n. \\nLee JH et al. The influence of CYP2C19 polymorphism on eradication of Helicobacter pylori: a prospective randomized study of', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 88}),\n",
       " Document(page_content='lansoprazole and rabeprazole. Gut Liver 2010;4:201-6.\\n9\\n. \\nFuruta T et al. CYP2C19 genotype is associated with symptomatic recurrence of GERD during maintenance therapy with low-\\ndose lansoprazole. Eur J Clin Pharmacol 2009;65:693-8.\\n10\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n11\\n. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in\\nCaucasians. Br J Clin Pharmacol 2008;65:752-60.\\n12\\n. \\nSakurai Y et al. Population pharmacokinetics and proton pump inhibitory effects of intravenous lansoprazole in healthy Japanese\\nmales. Biol Pharm Bull 2007;30:2238-43.\\n13\\n. \\nKawamura M et al. Cytochrome P450 2C19 polymorphism influences the preventive effect of lansoprazole on the recurrence of\\nerosive reflux esophagitis. J Gastroenterol Hepatol 2007;22:222-6.\\n14\\n. \\nFuruta T et al. Effect of concomitant dosing of famotidine with lansoprazole on gastric acid secretion in relation to CYP2C19\\ngenotype status. Aliment Pharmacol Ther 2005;22:67-74.\\n15\\n. \\nKawamura M et al. Study Group of GERD. The effects of lansoprazole on erosive reflux oesophagitis are influenced by CYP2C19\\npolymorphism. Aliment Pharmacol Ther 2003;17:965-73.\\n16\\n. \\nFuruta T et al. Effect of cytochrome P4502C19 genotypic differences on cure rates for gastroesophageal reflux disease by\\nlansoprazole. Clin Pharmacol Ther 2002;72:453-60.\\n17\\n. \\nShirai N et al. Comparison of lansoprazole and famotidine for gastric acid inhibition during the daytime and night-time in different\\nCYP2C19 genotype groups. Aliment Pharmacol Ther 2002;16:837-46.\\n18\\n. \\nFuruta T et al. Effect of high-dose lansoprazole on intragastric pH in subjects who are homozygous extensive metabolizers of\\ncytochrome P4502C19. Clin Pharmacol Ther 2001;70:484-92.\\n19\\n. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol\\nTher 2000;14:1259-66.\\n20\\n. \\nHowden CW et al. Dose-response evaluation of the antisecretory effect of continuous infusion intravenous lansoprazole regimens\\nover 48 h. Aliment Pharmacol Ther 2006;23:975-84.\\n21\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n22\\n. \\nFuruta T et al. Effect of MDR1 C3435T polymorphism on cure rates of Helicobacter pylori infection by triple therapy with\\nlansoprazole, amoxicillin and clarithromycin in relation to CYP 2C19 genotypes and 23S rRNA genotypes of H. pylori. Aliment\\nPharmacol Ther 2007;26:693-703.\\n23\\n. \\nSugimoto M et al. Evidence that the degree and duration of acid suppression are related to Helicobacter pylori eradication by triple\\ntherapy. Helicobacter 2007;12:317-23.\\n24\\n. \\nFuruta T et al. Pharmacogenomics-based tailored versus standard therapeutic regimen for eradication of H. pylori. Clin Pharmacol\\nTher 2007;81:521-8.\\n25\\n. \\nPadol S et al. The effect of CYP2C19 polymorphisms on H. pylori eradication rate in dual and triple first-line PPI therapies: a\\nmeta-analysis. Am J Gastroenterol 2006;101:1467-75.\\n26\\n. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of\\nclarithromycin-sensitive strains of Helicobacter pylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n27\\n. \\nFuruta T et al. Influence of CYP2C19 polymorphism and Helicobacter pylori genotype determined from gastric tissue samples on\\nresponse to triple therapy for H pylori infection. Clin Gastroenterol Hepatol 2005;3:564-73.\\n28\\n. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin\\nGastroenterol Hepatol 2004;2:22-30.\\n29\\n. \\nKawabata H et al. Effect of different proton pump inhibitors, differences in CYP2C19 genotype and antibiotic resistance on the\\neradication rate of Helicobacter pylori infection by a 1-week regimen of proton pump inhibitor, amoxicillin and clarithromycin.\\nAliment Pharmacol Ther 2003;17:259-64.\\n30\\n. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori\\ninfection with lansoprazole- or rabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n31\\n. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a\\nproton pump inhibitor, amoxicillin, and clarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n32\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n33\\n. \\nQiao HL et al. Pharmacokinetics of three proton pump inhibitors in Chinese subjects in relation to the CYP2C19 genotype. Eur J\\nClin Pharmacol 2006;62:107-12.\\n34\\n. \\nMiura M et al. Pharmacokinetic differences between the enantiomers of lansoprazole and its metabolite, 5-hydroxylansoprazole, in\\nrelation to CYP2C19 genotypes. Eur J Clin Pharmacol 2004;60:623-8.\\n35\\n. \\nIeiri I et al. Comparison of the kinetic disposition of and serum gastrin change by lansoprazole versus rabeprazole during an 8-day\\ndosing scheme in relation to CYP2C19 polymorphism. Eur J Clin Pharmacol 2001;57:485-92.\\n36\\n. \\nSakai T et al. CYP2C19 genotype and pharmacokinetics of three proton pump inhibitors in healthy subjects. Pharm Res\\n2001;18:721-7.\\n37\\n. \\nKatsuki H et al. Genetic polymorphism of CYP2C19 and lansoprazole pharmacokinetics in Japanese subjects. Eur J Clin\\nPharmacol 1997;52:391-6.\\nDate 05-03-2018\\nCYP2C19 PM: lansoprazol\\n1832', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 89}),\n",
       " Document(page_content='NO action is needed for this gene-drug interaction.\\nThe higher plasma concentration of lansoprazole results in an increase in the therapeutic effectiveness, without an increase in the\\nincidence of side effects.\\nLiterature:\\n1\\n. \\nLiou JM et al. Sequential therapy for 10 days versus triple therapy for 14 days in the eradication of Helicobacter pylori in the\\ncommunity and hospital populations: a randomised trial. Gut 2016;65:1784-1792.\\n2\\n. \\nLiou JM et al. Levofloxacin sequential therapy vs levofloxacin triple therapy in the second-line treatment of Helicobacter pylori: a\\nrandomized trial. Am J Gastroenterol 2016;111:381-7.\\n3\\n. \\nLang JE et al. Lansoprazole is associated with worsening asthma control in children with the CYP2C19 poor metabolizer\\nphenotype. Ann Am Thorac Soc 2015 Jun;12:878-85.\\n4\\n. \\nLi CY et al. A correlative study of polymorphisms of CYP2C19 and MDR1 C3435T with the pharmacokinetic profiles of\\nlansoprazole and its main metabolites following single oral administration in healthy adult Chinese subjects. Eur J Drug Metab\\nPharmacokinet 2014;39:121-8.\\n5\\n. \\nLima JJ et al. Association of CYP2C19 polymorphisms and lansoprazole-associated respiratory adverse effects in children. J\\nPediatr 2013;163:686-91.\\n6\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n7\\n. \\nZhang D et al. Effects of CYP2C19 polymorphism on the pharmacokinetics of lansoprazole and its main metabolites in healthy\\nChinese subjects. Xenobiotica 2011;41:511-7.\\n8\\n. \\nTamura T et al. Improvements in Helicobacter pylori eradication rates through clinical CYP2C19 genotyping. Nagoya J Med Sci\\n2011;73:25-31.\\n9\\n. \\nXu HR et al. The effect of CYP2C19 activity on pharmacokinetics of lansoprazole and its active metabolites in healthy subjects.\\nPharm Biol 2010;48:947-52.\\n10\\n. \\nLee JH et al. The influence of CYP2C19 polymorphism on eradication of Helicobacter pylori: a prospective randomized study of\\nlansoprazole and rabeprazole. Gut Liver 2010;4:201-6.\\n11\\n. \\nFuruta T et al. CYP2C19 genotype is associated with symptomatic recurrence of GERD during maintenance therapy with low-\\ndose lansoprazole. Eur J Clin Pharmacol 2009;65:693-8.\\n12\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n13\\n. \\nSakurai Y et al. Population pharmacokinetics and proton pump inhibitory effects of intravenous lansoprazole in healthy Japanese\\nmales. Biol Pharm Bull 2007;30:2238-43.\\n14\\n. \\nKawamura M et al. Cytochrome P450 2C19 polymorphism influences the preventive effect of lansoprazole on the recurrence of\\nerosive reflux esophagitis. J Gastroenterol Hepatol 2007;22:222-6.\\n15\\n. \\nFuruta T et al. Effect of concomitant dosing of famotidine with lansoprazole on gastric acid secretion in relation to CYP2C19\\ngenotype status. Aliment Pharmacol Ther 2005;22:67-74.\\n16\\n. \\nKawamura M et al. Study Group of GERD. The effects of lansoprazole on erosive reflux oesophagitis are influenced by CYP2C19\\npolymorphism. Aliment Pharmacol Ther 2003;17:965-73.\\n17\\n. \\nFuruta T et al. Effect of cytochrome P4502C19 genotypic differences on cure rates for gastroesophageal reflux disease by\\nlansoprazole. Clin Pharmacol Ther 2002;72:453-60.\\n18\\n. \\nShirai N et al. Comparison of lansoprazole and famotidine for gastric acid inhibition during the daytime and night-time in different\\nCYP2C19 genotype groups. Aliment Pharmacol Ther 2002;16:837-46.\\n19\\n. \\nFuruta T et al. Effect of high-dose lansoprazole on intragastric pH in subjects who are homozygous extensive metabolizers of\\ncytochrome P4502C19. Clin Pharmacol Ther 2001;70:484-92.\\n20\\n. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol\\nTher 2000;14:1259-66.\\n21\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n22\\n. \\nFuruta T et al. Effect of MDR1 C3435T polymorphism on cure rates of Helicobacter pylori infection by triple therapy with\\nlansoprazole, amoxicillin and clarithromycin in relation to CYP 2C19 genotypes and 23S rRNA genotypes of H. pylori. Aliment\\nPharmacol Ther 2007;26:693-703.\\n23\\n. \\nSugimoto M et al. Evidence that the degree and duration of acid suppression are related to Helicobacter pylori eradication by triple\\ntherapy. Helicobacter 2007;12:317-23.\\n24\\n. \\nFuruta T et al. Pharmacogenomics-based tailored versus standard therapeutic regimen for eradication of H. pylori. Clin Pharmacol\\nTher 2007;81:521-8.\\n25\\n. \\nPadol S et al. The effect of CYP2C19 polymorphisms on H. pylori eradication rate in dual and triple first-line PPI therapies: a\\nmeta-analysis. Am J Gastroenterol 2006;101:1467-75.\\n26\\n. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of\\nclarithromycin-sensitive strains of Helicobacter pylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n27\\n. \\nFuruta T et al. Influence of CYP2C19 polymorphism and Helicobacter pylori genotype determined from gastric tissue samples on\\nresponse to triple therapy for H pylori infection. Clin Gastroenterol Hepatol 2005;3:564-73.\\n28\\n. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin\\nGastroenterol Hepatol 2004;2:22-30.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 90}),\n",
       " Document(page_content='29\\n. \\nKawabata H et al. Effect of different proton pump inhibitors, differences in CYP2C19 genotype and antibiotic resistance on the\\neradication rate of Helicobacter pylori infection by a 1-week regimen of proton pump inhibitor, amoxicillin and clarithromycin.\\nAliment Pharmacol Ther 2003;17:259-64.\\n30\\n. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori\\ninfection with lansoprazole- or rabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n31\\n. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a\\nproton pump inhibitor, amoxicillin, and clarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n32\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n33\\n. \\nQiao HL et al. Pharmacokinetics of three proton pump inhibitors in Chinese subjects in relation to the CYP2C19 genotype. Eur J\\nClin Pharmacol 2006;62:107-12.\\n34\\n. \\nMiura M et al. Pharmacokinetic differences between the enantiomers of lansoprazole and its metabolite, 5-hydroxylansoprazole, in\\nrelation to CYP2C19 genotypes. Eur J Clin Pharmacol 2004;60:623-8.\\n35\\n. \\nHu YR et al. Pharmacokinetics of lansoprazole in Chinese healthy subjects in relation to CYP2C19 genotypes. Acta Pharmacol Sin\\n2004;25:986-90.\\n36\\n. \\nIeiri I et al. Comparison of the kinetic disposition of and serum gastrin change by lansoprazole versus rabeprazole during an 8-day\\ndosing scheme in relation to CYP2C19 polymorphism. Eur J Clin Pharmacol 2001;57:485-92.\\n37\\n. \\nSakai T et al. CYP2C19 genotype and pharmacokinetics of three proton pump inhibitors in healthy subjects. Pharm Res\\n2001;18:721-7.\\n38\\n. \\nKatsuki H et al. Genetic polymorphism of CYP2C19 and lansoprazole pharmacokinetics in Japanese subjects. Eur J Clin\\nPharmacol 1997;52:391-6.\\n39\\n. \\nSPC Prezal.\\nDate 05-03-2018\\nCYP2C19 UM: lansoprazol\\n1833\\nThe genetic variation may reduce lansoprazole plasma concentrations and therefore lansoprazole effectiveness.\\nRecommendation:\\nFor Helicobacter pylori ERADICATION THERAPY:\\n1\\n. \\nUse a 4-fold higher dose\\n2\\n. \\nAdvise the patient to contact their doctor if symptoms of dyspepsia persist\\nOTHER INDICATIONS:\\n1\\n. \\nBe alert to reduced effectiveness\\n2\\n. \\nIf necessary, use a 4-fold higher dose\\n3\\n. \\nAdvise the patient to report persisting symptoms of dyspepsia\\nLiterature:\\n1\\n. \\nLiou JM et al. Sequential therapy for 10 days versus triple therapy for 14 days in the eradication of Helicobacter pylori in the\\ncommunity and hospital populations: a randomised trial. Gut 2016;65:1784-1792.\\n2\\n. \\nLiou JM et al. Levofloxacin sequential therapy vs levofloxacin triple therapy in the second-line treatment of Helicobacter pylori: a\\nrandomized trial. Am J Gastroenterol 2016;111:381-7.\\n3\\n. \\nLang JE et al. Lansoprazole is associated with worsening asthma control in children with the CYP2C19 poor metabolizer\\nphenotype. Ann Am Thorac Soc 2015 Jun;12:878-85.\\n4\\n. \\nLi CY et al. A correlative study of polymorphisms of CYP2C19 and MDR1 C3435T with the pharmacokinetic profiles of\\nlansoprazole and its main metabolites following single oral administration in healthy adult Chinese subjects. Eur J Drug Metab\\nPharmacokinet 2014;39:121-8.\\n5\\n. \\nLima JJ et al. Association of CYP2C19 polymorphisms and lansoprazole-associated respiratory adverse effects in children. J\\nPediatr 2013;163:686-91.\\n6\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n7\\n. \\nZhang D et al. Effects of CYP2C19 polymorphism on the pharmacokinetics of lansoprazole and its main metabolites in healthy\\nChinese subjects. Xenobiotica 2011;41:511-7.\\n8\\n. \\nTamura T et al. Improvements in Helicobacter pylori eradication rates through clinical CYP2C19 genotyping. Nagoya J Med Sci\\n2011;73:25-31.\\n9\\n. \\nXu HR et al. The effect of CYP2C19 activity on pharmacokinetics of lansoprazole and its active metabolites in healthy subjects.\\nPharm Biol 2010;48:947-52.\\n10\\n. \\nLee JH et al. The influence of CYP2C19 polymorphism on eradication of Helicobacter pylori: a prospective randomized study of\\nlansoprazole and rabeprazole. Gut Liver 2010;4:201-6.\\n11\\n. \\nFuruta T et al. CYP2C19 genotype is associated with symptomatic recurrence of GERD during maintenance therapy with low-', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 91}),\n",
       " Document(page_content='dose lansoprazole. Eur J Clin Pharmacol 2009;65:693-8.\\n12\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n13\\n. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in\\nCaucasians. Br J Clin Pharmacol 2008;65:752-60.\\n14\\n. \\nSakurai Y et al. Population pharmacokinetics and proton pump inhibitory effects of intravenous lansoprazole in healthy Japanese\\nmales. Biol Pharm Bull 2007;30:2238-43.\\n15\\n. \\nKawamura M et al. Cytochrome P450 2C19 polymorphism influences the preventive effect of lansoprazole on the recurrence of\\nerosive reflux esophagitis. J Gastroenterol Hepatol 2007;22:222-6.\\n16\\n. \\nFuruta T et al. Effect of concomitant dosing of famotidine with lansoprazole on gastric acid secretion in relation to CYP2C19\\ngenotype status. Aliment Pharmacol Ther 2005;22:67-74.\\n17\\n. \\nKawamura M et al. Study Group of GERD. The effects of lansoprazole on erosive reflux oesophagitis are influenced by CYP2C19\\npolymorphism. Aliment Pharmacol Ther 2003;17:965-73.\\n18\\n. \\nFuruta T et al. Effect of cytochrome P4502C19 genotypic differences on cure rates for gastroesophageal reflux disease by\\nlansoprazole. Clin Pharmacol Ther 2002;72:453-60.\\n19\\n. \\nShirai N et al. Comparison of lansoprazole and famotidine for gastric acid inhibition during the daytime and night-time in different\\nCYP2C19 genotype groups. Aliment Pharmacol Ther 2002;16:837-46.\\n20\\n. \\nFuruta T et al. Effect of high-dose lansoprazole on intragastric pH in subjects who are homozygous extensive metabolizers of\\ncytochrome P4502C19. Clin Pharmacol Ther 2001;70:484-92.\\n21\\n. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol\\nTher 2000;14:1259-66.\\n22\\n. \\nHowden CW et al. Dose-response evaluation of the antisecretory effect of continuous infusion intravenous lansoprazole regimens\\nover 48 h. Aliment Pharmacol Ther 2006;23:975-84.\\n23\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n24\\n. \\nFuruta T et al. Effect of MDR1 C3435T polymorphism on cure rates of Helicobacter pylori infection by triple therapy with\\nlansoprazole, amoxicillin and clarithromycin in relation to CYP 2C19 genotypes and 23S rRNA genotypes of H. pylori. Aliment\\nPharmacol Ther 2007;26:693-703.\\n25\\n. \\nSugimoto M et al. Evidence that the degree and duration of acid suppression are related to Helicobacter pylori eradication by triple\\ntherapy. Helicobacter 2007;12:317-23.\\n26\\n. \\nFuruta T et al. Pharmacogenomics-based tailored versus standard therapeutic regimen for eradication of H. pylori. Clin Pharmacol\\nTher 2007;81:521-8.\\n27\\n. \\nPadol S et al. The effect of CYP2C19 polymorphisms on H. pylori eradication rate in dual and triple first-line PPI therapies: a\\nmeta-analysis. Am J Gastroenterol 2006;101:1467-75.\\n28\\n. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of\\nclarithromycin-sensitive strains of Helicobacter pylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n29\\n. \\nFuruta T et al. Influence of CYP2C19 polymorphism and Helicobacter pylori genotype determined from gastric tissue samples on\\nresponse to triple therapy for H pylori infection. Clin Gastroenterol Hepatol 2005;3:564-73.\\n30\\n. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin\\nGastroenterol Hepatol 2004;2:22-30.\\n31\\n. \\nKawabata H et al. Effect of different proton pump inhibitors, differences in CYP2C19 genotype and antibiotic resistance on the\\neradication rate of Helicobacter pylori infection by a 1-week regimen of proton pump inhibitor, amoxicillin and clarithromycin.\\nAliment Pharmacol Ther 2003;17:259-64.\\n32\\n. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori\\ninfection with lansoprazole- or rabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n33\\n. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a\\nproton pump inhibitor, amoxicillin, and clarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n34\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n35\\n. \\nQiao HL et al. Pharmacokinetics of three proton pump inhibitors in Chinese subjects in relation to the CYP2C19 genotype. Eur J\\nClin Pharmacol 2006;62:107-12.\\n36\\n. \\nMiura M et al. Pharmacokinetic differences between the enantiomers of lansoprazole and its metabolite, 5-hydroxylansoprazole, in\\nrelation to CYP2C19 genotypes. Eur J Clin Pharmacol 2004;60:623-8.\\n37\\n. \\nHu YR et al. Pharmacokinetics of lansoprazole in Chinese healthy subjects in relation to CYP2C19 genotypes. Acta Pharmacol Sin\\n2004;25:986-90.\\n38\\n. \\nIeiri I et al. Comparison of the kinetic disposition of and serum gastrin change by lansoprazole versus rabeprazole during an 8-day\\ndosing scheme in relation to CYP2C19 polymorphism. Eur J Clin Pharmacol 2001;57:485-92.\\n39\\n. \\nSakai T et al. CYP2C19 genotype and pharmacokinetics of three proton pump inhibitors in healthy subjects. Pharm Res\\n2001;18:721-7.\\n40\\n. \\nKatsuki H et al. Genetic polymorphism of CYP2C19 and lansoprazole pharmacokinetics in Japanese subjects. Eur J Clin\\nPharmacol 1997;52:391-6.\\n41\\n. \\nSPC Prezal.\\nDate 05-03-2018\\nCYP2D6 IM: methylfenidaat\\n2528', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 92}),\n",
       " Document(page_content='This is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 PM: methylfenidaat\\n2527\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nDeVane CL et al. Single-dose pharmacokinetics of methylphenidate in CYP2D6 extensive and poor metabolizers. J Clin\\nPsychopharmacol 2000;20:347-9.\\nDate 24-08-2016\\nCYP2D6 UM: methylfenidaat\\n2529\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 IM: metoprolol\\n1554\\nThe gene variation reduces the conversion of metoprolol to inactive metabolites. However, the clinical consequences are limited mainly\\nto the occurrence of asymptomatic bradycardia.\\nRecommendation:\\nIf a GRADUAL REDUCTION in HEART RATE is desired, or in the event of SYMPTOMATIC BRADYCARDIA:\\n1\\n. \\nincrease the dose in smaller steps and/or prescribe no more than 50% of the standard dose\\nOTHER CASES:\\n1\\n. \\nno action required\\nLiterature:\\n1\\n. \\nHamadeh IS et al. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol\\nTher 2014;96:175-81.\\n2\\n. \\nBatty JA et al. An investigation of CYP2D6 genotype and response to metoprolol CR/XL during dose titration in patients with\\nheart failure: a MERIT-HF substudy. Clin Pharmacol Ther 2014;95:321-30.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 93}),\n",
       " Document(page_content='3\\n. \\nRau T et al. Impact of the CYP2D6 genotype on the clinical effects of metoprolol: a prospective longitudinal study. Clin\\nPharmacol Ther 2009;85:269-72.\\n4\\n. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users.\\nClin Pharmacol Ther 2009;85:45-50.\\n5\\n. \\nGoryachkina K et al. CYP2D6 is a major determinant of metoprolol disposition and effects in hospitalized Russian patients treated\\nfor acute myocardial infarction. Eur J Clin Pharmacol 2008;64:1163-73.\\n6\\n. \\nYuan H et al. Effects of polymorphism of the beta(1) adrenoreceptor and CYP2D6 on the therapeutic effects of metoprolol. J Int\\nMed Res 2008;36:1354-62.\\n7\\n. \\nJin SK et al. Influence of CYP2D6*10 on the pharmacokinetics of metoprolol in healthy Korean volunteers. J Clin Pharm Ther\\n200;33:567-73.\\n8\\n. \\nIsmail R et al. The relevance of CYP2D6 genetic polymorphism on chronic metoprolol therapy in cardiovascular patients. J Clin\\nPharm Ther 2006;31:99-109.\\n9\\n. \\nTerra SG et al. Beta1-adrenergic receptor polymorphisms and left ventricular remodeling changes in response to beta-blocker\\ntherapy. Clin Pharmacol Ther 2005;77:127-137.\\n10\\n. \\nNozawa T et al. Influence of CYP2D6 genotype on metoprolol plasma concentration and beta-adrenergic inhibition during long-\\nterm treatment: a comparison with bisoprolol. J Cardiovasc Pharmacol 2005;46:713-20.\\n11\\n. \\nFux R et al. Impact of CYP2D6 genotype on adverse effects during treatment with metoprolol: a prospective clinical study. Clin\\nPharmacol Ther 2005;78:378-87.\\n12\\n. \\nZineh I et al. Pharmacokinetics and CYP2D6 genotypes do not predict metoprolol adverse events or efficacy in hypertension. Clin\\nPharmacol Ther 2004;76:536-44.\\n13\\n. \\nTaguchi M et al. Effect of CYP2D6*10 on pharmacokinetic variability of routinely administered metoprolol in middle-aged and\\nelderly Japanese patients. Eur J Clin Pharmacol 2003;59:385-8.\\n14\\n. \\nRau T et al. Effect of the CYP2D6 genotype on metoprolol metabolism persists during long-term treatment. Pharmacogenetics\\n2002;12:465-72.\\n15\\n. \\nHuang JD et al. Pharmacokinetics of metoprolol enantiomers in Chinese subjects of major CYP2D6 genotypes. Clin Pharmacol\\nTher 1999;65:402-7.\\n16\\n. \\nKoytchev R et al. Influence of the cytochrome P4502D6*4 allele on the pharmacokinetics of controlled-release metoprolol. Eur J\\nClin Pharmacol 1998;54:469-74.\\nDate 25-05-2016\\nCYP2D6 PM: metoprolol\\n1555\\nThe gene variation reduces the conversion of metoprolol to inactive metabolites. However, the clinical consequences are limited mainly\\nto the occurrence of asymptomatic bradycardia.\\nRecommendation:\\nIf a GRADUAL REDUCTION in HEART RATE is desired, or in the event of SYMPTOMATIC BRADYCARDIA:\\n1\\n. \\nincrease the dose in smaller steps and/or prescribe no more than 25% of the standard dose\\nOTHER CASES:\\n1\\n. \\nno action required\\nLiterature:\\n1\\n. \\nHamadeh IS et al. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol\\nTher 2014;96:175-81.\\n2\\n. \\nBatty JA et al. An investigation of CYP2D6 genotype and response to metoprolol CR/XL during dose titration in patients with\\nheart failure: a MERIT-HF substudy. Clin Pharmacol Ther 2014;95:321-30.\\n3\\n. \\nRau T et al. Impact of the CYP2D6 genotype on the clinical effects of metoprolol: a prospective longitudinal study. Clin\\nPharmacol Ther 2009;85:269-72.\\n4\\n. \\nBijl MJ et al. Genetic variation in the CYP2D6 gene is associated with a lower heart rate and blood pressure in beta-blocker users.\\nClin Pharmacol Ther 2009;85:45-50.\\n5\\n. \\nGoryachkina K et al. CYP2D6 is a major determinant of metoprolol disposition and effects in hospitalized Russian patients treated\\nfor acute myocardial infarction. Eur J Clin Pharmacol 2008;64:1163-73.\\n6\\n. \\nYuan H et al. Effects of polymorphism of the beta(1) adrenoreceptor and CYP2D6 on the therapeutic effects of metoprolol. J Int\\nMed Res 2008;36:1354-62.\\n7\\n. \\nSeeringer A et al. Enantiospecific pharmacokinetics of metoprolol in CYP2D6 ultra-rapid metabolizers and correlation with\\nexercise-induced heart rate. Eur J Clin Pharmacol 2008;64:883-8.\\n8\\n. \\nIsmail R et al. The relevance of CYP2D6 genetic polymorphism on chronic metoprolol therapy in cardiovascular patients. J Clin\\nPharm Ther 2006;31:99-109.\\n9\\n. \\nTerra SG et al. Beta1-adrenergic receptor polymorphisms and left ventricular remodeling changes in response to beta-blocker', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 94}),\n",
       " Document(page_content='therapy. Clin Pharmacol Ther 2005;77:127-137.\\n10\\n. \\nFux R et al. Impact of CYP2D6 genotype on adverse effects during treatment with metoprolol: a prospective clinical study. Clin\\nPharmacol Ther 2005;78:378-87.\\n11\\n. \\nZineh I et al. Pharmacokinetics and CYP2D6 genotypes do not predict metoprolol adverse events or efficacy in hypertension. Clin\\nPharmacol Ther 2004;76:536-44.\\n12\\n. \\nKirchheiner J et al. Impact of the ultrarapid metabolizer genotype of cytochrome P450 2D6 on metoprolol pharmacokinetics and\\npharmacodynamics. Clin Pharmacol Ther 2004;76:302-12.\\n13\\n. \\nWuttke H et al. Increased frequency of cytochrome P450 2D6 poor metabolizers among patients with metoprolol-associated\\nadverse effects. Clin Pharmacol Ther 2002;72:429-37.\\n14\\n. \\nRau T et al. Effect of the CYP2D6 genotype on metoprolol metabolism persists during long-term treatment. Pharmacogenetics\\n2002;12:465-72.\\n15\\n. \\nLaurent-Kenesi MA et al. Influence of CYP2D6-dependent metabolism on the steady-state pharmacokinetics and\\npharmacodynamics of metoprolol and nicardipine, alone and in combination. Br J Clin Pharmacol 1993;36:531-8.\\n16\\n. \\nLewis RV et al. Influence of debrisoquine oxidation phenotype on exercise tolerance and subjective fatigue after metoprolol and\\natenolol in healthy subjects. Br J Clin Pharmacol 1991;31:391-8.\\n17\\n. \\nClark DWJ et al. Adverse effects from metoprolol are not generally associated with oxidation status. Br J Clin Pharmacol\\n1984;18:965-966.\\n18\\n. \\nLennard MS et al. Differential stereoselective metabolism of metoprolol in extensive and poor debrisoquin metabolizers. Clin\\nPharmacol Ther 1983;34:732-7.\\nDate 25-05-2016\\nCYP2D6 UM: metoprolol\\n1556\\nThe gene variation increases the conversion of metoprolol to inactive metabolites. This can increase the dose requirement. However, with\\na target dose of 200 mg/day, there was no effect on the blood pressure and hardly any effect on the reduction of the heart rate.\\nRecommendation:\\n1\\n. \\nuse the maximum dose for the relevant indication as a target dose\\n2\\n. \\nif the effectiveness is still insufficient: increase the dose based on effectiveness and side effects to 2.5 times the standard dose or\\nselect an alternative\\nPossible alternatives include:\\nHEART FAILURE: bisoprolol or carvedilol. Bisoprolol: advantage: not metabolised by CYP2D6; disadvantage: elimination\\ndepends on the kidney function. Carvedilol: advantage: elimination does not depend on the kidney function; disadvantage: is\\nmetabolised (to a lesser extent than metoprolol) by CYP2D6.\\nOTHER INDICATIONS: atenolol or bisoprolol. Neither is metabolised by CYP2D6.\\nLiterature:\\n1\\n. \\nHamadeh IS et al. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol\\nTher 2014;96:175-81.\\n2\\n. \\nGoryachkina K et al. CYP2D6 is a major determinant of metoprolol disposition and effects in hospitalized Russian patients treated\\nfor acute myocardial infarction. Eur J Clin Pharmacol 2008;64:1163-73.\\n3\\n. \\nSeeringer A et al. Enantiospecific pharmacokinetics of metoprolol in CYP2D6 ultra-rapid metabolizers and correlation with\\nexercise-induced heart rate. Eur J Clin Pharmacol 2008;64:883-8.\\n4\\n. \\nIsmail R et al. The relevance of CYP2D6 genetic polymorphism on chronic metoprolol therapy in cardiovascular patients. J Clin\\nPharm Ther 2006;31:99-109.\\n5\\n. \\nFux R et al. Impact of CYP2D6 genotype on adverse effects during treatment with metoprolol: a prospective clinical study. Clin\\nPharmacol Ther 2005;78:378-87.\\n6\\n. \\nKirchheiner J et al. Impact of the ultrarapid metabolizer genotype of cytochrome P450 2D6 on metoprolol pharmacokinetics and\\npharmacodynamics. Clin Pharmacol Ther 2004;76:302-12.\\nDate 25-05-2016\\nCYP2C19 IM: mirtazapine\\n3507\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 95}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nGrasmäder K et al. Population pharmacokinetic analysis of mirtazapine. Eur J Clin Pharmacol 2004;60:473-80.\\nDate 10-09-2018\\nCYP2C19 PM: mirtazapine\\n3506\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nJohnson M et al. A poor metabolizer for cytochromes P450 2D6 and 2C19: a case report on antidepressant treatment. CNS Spectr\\n2006;11:757-60.\\nDate 10-09-2018\\nCYP2C19 UM: mirtazapine\\n3508\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nGrasmäder K et al. Population pharmacokinetic analysis of mirtazapine. Eur J Clin Pharmacol 2004;60:473-80.\\nDate 10-09-2018\\nCYP2D6 IM: mirtazapine\\n2002\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of mirtazapine does not result in an increase in the side effects.\\nLiterature:\\n1\\n. \\nBorobia AM et al. Influence of sex and CYP2D6 genotype on mirtazapine disposition, evaluated in Spanish healthy volunteers.\\nPharmacol Res 2009;59:393-8.\\n2\\n. \\nLind AB et al. Steady-state concentrations of mirtazapine, N-desmethylmirtazapine, 8-hydroxymirtazapine and their enantiomers\\nin relation to cytochrome P450 2D6 genotype, age and smoking behaviour. Clin Pharmacokinet 2009;48:63-70.\\n3\\n. \\nGrasmader K et al. Population pharmacokinetic analysis of mirtazapine. Eur J Clin Pharmacol 2004;60:473-80.\\n4\\n. \\nMurphy GM Jr et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\nDate 27-11-2012', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 96}),\n",
       " Document(page_content='CYP2D6 PM: mirtazapine\\n2001\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of mirtazapine does not result - or hardly results - in an increase in the side effects.\\nLiterature:\\n1\\n. \\nRamaekers JG et al. Residual effects of esmirtazapine on actual driving performance: overall findings and an exploratory analysis\\ninto the role of CYP2D6 phenotype. Psychopharmacology 2011;215:321-32.\\n2\\n. \\nBorobia AM et al. Influence of sex and CYP2D6 genotype on mirtazapine disposition, evaluated in Spanish healthy volunteers.\\nPharmacol Res 2009;59:393-8.\\n3\\n. \\nLind AB et al. Steady-state concentrations of mirtazapine, N-desmethylmirtazapine, 8-hydroxymirtazapine and their enantiomers\\nin relation to cytochrome P450 2D6 genotype, age and smoking behaviour. Clin Pharmacokinet 2009;48:63-70.\\n4\\n. \\nBrockmoller J et al. Pharmacokinetics of mirtazapine: enantioselective effects of the CYP2D6 ultra rapid metabolizer genotype\\nand correlation with adverse effects. Clin Pharmacol Ther 2007;81:699-707.\\n5\\n. \\nJohnson M et al. A poor metabolizer for cytochromes P450 2D6 and 2C19: a case report on antidepressant treatment. CNS Spectr\\n2006;11:757-60.\\n6\\n. \\nStephan PL et al. Adverse drug reactions following nonresponse in a depressed patient with CYP2D6 deficiency and low CYP\\n3A4/5 activity. Pharmacopsychiatry 2006;39:150-2.\\n7\\n. \\nKirchheiner J et al. Impact of the CYP2D6 ultrarapid metabolizer genotype on mirtazapine pharmacokinetics and adverse events in\\nhealthy volunteers. J Clin Psychopharmacol 2004;24:647-52.\\n8\\n. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and\\nclinical effects of antidepressants in a naturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n9\\n. \\nMurphy GM Jr et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\nDate 27-11-2012\\nCYP2D6 UM: mirtazapine\\n2003\\nNO action is required for this gene-drug interaction.\\nThe effect on the plasma concentration of mirtazapine is small. No effect has been demonstrated with regard to effectiveness or side\\neffects.\\nLiterature:\\n1\\n. \\nLind AB et al. Steady-state concentrations of mirtazapine, N-desmethylmirtazapine, 8-hydroxymirtazapine and their enantiomers\\nin relation to cytochrome P450 2D6 genotype, age and smoking behaviour. Clin Pharmacokinet 2009;48:63-70.\\n2\\n. \\nBrockmoller J et al. Pharmacokinetics of mirtazapine: enantioselective effects of the CYP2D6 ultra rapid metabolizer genotype\\nand correlation with adverse effects. Clin Pharmacol Ther 2007;81:699-707.\\n3\\n. \\nKirchheiner J et al. Impact of the CYP2D6 ultrarapid metabolizer genotype on mirtazapine pharmacokinetics and adverse events in\\nhealthy volunteers. J Clin Psychopharmacol 2004;24:647-52.\\n4\\n. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and\\nclinical effects of antidepressants in a naturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\nDate 27-11-2012\\nCYP2C19 IM: moclobemide\\n1991\\nNO action is needed for this gene-drug interaction.\\nAlthough the moclobemide plasma concentration may increase as a result of the decreased CYP2C19 metabolic capacity, this does not\\nlead to an increased incidence of side effects, in as far as is known.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 97}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nGram LF et al. Moclobemide, a substrate of CYP2C19 and an inhibitor of CYP2C19, CYP2D6, and CYP1A2: a panel study. Clin\\nPharmacol Ther 1995;57:670-7.\\n2\\n. \\nSPC Aurorix.\\nDate 23-05-2012\\nCYP2C19 PM: moclobemide\\n1992\\nNO action is needed for this gene-drug interaction.\\nAlthough the moclobemide plasma concentration may increase as a result of the decreased CYP2C19 metabolic capacity, this does not\\nlead to an increased incidence of side effects, in as far as is known.\\nLiterature:\\n1\\n. \\nYu KS et al. Effect of omeprazole on the pharmacokinetics of moclobemide according to the genetic polymorphism of CYP2C19.\\nClin Pharmacol Ther 2001;69:266-73.\\n2\\n. \\nGram LF et al. Moclobemide, a substrate of CYP2C19 and an inhibitor of CYP2C19, CYP2D6, and CYP1A2: a panel study. Clin\\nPharmacol Ther 1995;57:670-7.\\n3\\n. \\nSPC Aurorix.\\nDate 23-05-2012\\nCYP2C19 UM: moclobemide\\n1993\\nNO action is needed for this gene-drug interaction.\\nAlthough the moclobemide plasma concentration may decrease as a result of increased CYP2C19 metabolic capacity, this does not lead\\nto increased effectiveness, in as far as is known.\\nLiterature:\\n1\\n. \\nGram LF et al. Moclobemide, a substrate of CYP2C19 and an inhibitor of CYP2C19, CYP2D6, and CYP1A2: a panel study. Clin\\nPharmacol Ther 1995;57:670-7.\\n2\\n. \\nSPC Aurorix.\\nDate 23-05-2012\\nCYP2D6 IM: nortriptyline\\n1557\\nThe risk of side effects may be increased, because the gene variation leads to an increased plasma concentration of nortriptyline.\\nuse 60% of the standard dose and monitor the effect and side effects or the plasma concentration of nortriptyline in order to set the\\nmaintenance dose\\nThe therapeutic range of nortriptyline is 50-150 ng/mL. Values exceeding 250 ng/mL are considered toxic.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 98}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nBerm E et al. Relation between CYP2D6 genotype, phenotype and therapeutic drug concentrations among nortriptyline and\\nvenlafaxine users in old age psychiatry. Pharmacopsychiatry 2016;49:186-190.\\n2\\n. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl)\\n2015;232:2609-17.\\n3\\n. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol\\n2014;28:133-41.\\n4\\n. \\nLee SY et al. Sequence-based CYP2D6 genotyping in the Korean population. Ther Drug Monit 2006;28:382-7.\\n5\\n. \\nLee S et al. A case report of a poor metabolizer of CYP2D6 presented with unusual responses to nortriptyline medication. J Korean\\nMed Sci 2004;19:750-2.\\n6\\n. \\nDalen P et al. Disposition of debrisoquine and nortriptyline in Korean subjects in relation to CYP2D6 genotypes, and comparison\\nwith Caucasians. Br J Clin Pharmacol 2003;55:630-4.\\n7\\n. \\nKvist EE et al. Quantitative pharmacogenetics of nortriptyline: a novel approach. Clin Pharmacokinet 2001;40:869-77.\\n8\\n. \\nMorita S et al. Steady-state plasma levels of nortriptyline and its hydroxylated metabolites in Japanese patients: impact of\\nCYP2D6 genotype on the hydroxylation of nortriptyline. J Clin Psychopharmacol 2000;20:141-9.\\n9\\n. \\nYue QJ et al. Pharmacokinetics of nortriptyline and its 10-hydroxymetabolite in Chinese subjects of different CYP2D6 genotypes.\\nClin Pharmacol Ther 1998;64:384-90.\\n10\\n. \\nChen S et al. The cytochrome P450 2D6 (CYP2D6) enzyme polymorphism: screening costs and influence on clinical outcomes in\\npsychiatry. Clin Pharmacol Ther 1996;60:522-34.\\nDate 19-11-2018\\nCYP2D6 PM: nortriptyline\\n1558\\nThe risk of side effects may be increased, because the gene variation leads to an increased plasma concentration of nortriptyline.\\nuse 40% of the standard dose and monitor the effect and side effects or the plasma concentration of nortriptyline in order to set the\\nmaintenance dose The therapeutic range of nortriptyline is 50-150 ng/mL. Values exceeding 250 ng/mL are considered toxic.\\nLiterature:\\n1\\n. \\nBerm E et al. Relation between CYP2D6 genotype, phenotype and therapeutic drug concentrations among nortriptyline and\\nvenlafaxine users in old age psychiatry. Pharmacopsychiatry 2016;49:186-190.\\n2\\n. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl)\\n2015;232:2609-17.\\n3\\n. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol\\n2014;28:133-41.\\n4\\n. \\nRoberts et al. No evidence of increased adverse drug reactions in cytochrome P450 CYP2D6 poor metabolizers treated with\\nfluoxetine or nortriptyline. Hum Psychopharmacol 2004 Jan;19:17-23.\\n5\\n. \\nKvist EE et al. Quantitative pharmacogenetics of nortriptyline: a novel approach. Clin Pharmacokinet 2001;40:869-77.\\n6\\n. \\nDalen P et al. 10-Hydroxylation of nortriptyline in white persons with 0, 1, 2, 3, and 13 functional CYP2D6 genes. Clin Pharmacol\\nTher 1998;63:444-52.\\n7\\n. \\nDahl M et al. Steady-state plasma levels of nortriptyline and its 10-hydroxymetabolite: relationship to the CYP2D6 genotype.\\nPsychopharmacol 1996;123:315-9.\\n8\\n. \\nChen S et al. The cytochrome P450 2D6 (CYP2D6) enzyme polymorphism: screening costs and influence on clinical outcomes in\\npsychiatry. Clin Pharmacol Ther 1996;60:522-34.\\n9\\n. \\nBertilsson L et al. Slow hydroxylation of nortriptyline and concomitant poor debrisoquine hydroxylation: clinical implications.\\nLancet 1981;1:560-1.\\n10\\n. \\nSmPC Pamelor (VS).\\nDate 19-11-2018\\nCYP2D6 UM: nortriptyline\\n1559\\nThe risk of ineffectiveness and cardiotoxic effects may be increased. The gene variation leads to a decrease in the plasma concentration', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 99}),\n",
       " Document(page_content='of nortriptyline and an increase in the plasma concentration of the cardiotoxic metabolite Z-10-hydroxynortriptyline.\\nuse 1.7 times the standard dose and monitor the effect and side effects or the plasma concentration of nortriptyline and be alert to\\nan increase in the plasma concentration of the cardiotoxic metabolite Z-10-hydroxynortriptyline\\nPlasma concentrations of Z-hydroxynortriptyline exceeding 40 ng/mL are considered toxic.\\nif a dose increase is not wanted due to the cardiotoxic hydroxy metabolite: avoid nortriptyline\\nAntidepressants that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\nLiterature:\\n1\\n. \\nHodgson K et al. Exploring the role of drug-metabolising enzymes in antidepressant side effects. Psychopharmacology (Berl)\\n2015;232:2609-17.\\n2\\n. \\nHodgson K et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol\\n2014;28:133-41.\\n3\\n. \\nLee SY et al. Sequence-based CYP2D6 genotyping in the Korean population. Ther Drug Monit 2006;28:382-7.\\n4\\n. \\nKvist EE et al. Quantitative pharmacogenetics of nortriptyline: a novel approach. Clin Pharmacokinet 2001;40:869-77.\\n5\\n. \\nDalen P et al. 10-Hydroxylation of nortriptyline in white persons with 0, 1, 2, 3, and 13 functional CYP2D6 genes. Clin Pharmacol\\nTher 1998;63:444-52.\\n6\\n. \\nBertilsson L et al. Molecular basis for rational megaprescribing in ultrarapid hydroxylators of debrisoquine. Lancet 1993;341:63.\\nDate 19-11-2018\\nCYP2D6 IM: olanzapine\\n1560\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\n2\\n. \\nNozawa M et al. The relationship between the response of clinical symptoms and plasma olanzapine concentration, based on\\npharmacogenetics: Juntendo University Schizophrenia Projects (JUSP). Ther Drug Monit 2008;30:35-40.\\n3\\n. \\nIwahashi K. Olanzapine metabolism by CYP1A2/CYP2D6 and hyperglycaemia. Acta Neuropsychiatrica 2004;16:229-230.\\nDate 22-03-2006\\nCYP2D6 PM: olanzapine\\n1561\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nThomas P et al. Correlates of response to olanzapine in a North Indian schizophrenia sample. Psychiatry Res 2008;161:275-83.\\n2\\n. \\nCarrillo JA et al. Role of the smoking-induced cytochrome P450 (CYP)1A2 and polymorphic CYP2D6 in steady-state\\nconcentration of olanzapine. J Clin Psychopharmacol 2003;23:119-27.\\n3\\n. \\nHagg S et al. Olanzapine disposition in humans is unrelated to CYP1A2 and CYP2D6 phenotypes. Eur J Clin Pharmacol\\n2001;57:493-7.\\nDate 22-03-2006', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 100}),\n",
       " Document(page_content=\"CYP2D6 UM: olanzapine\\n1562\\nThis is NOT a gene-drug interaction.\\nNo studies have been published in which the pharmacokinetics and effects of the use of olanzapine on this phenotype were studied.\\nStudies with PM and IM found no significant association between the genotype and clinical effects (clinical improvements, non-response\\nand extrapyramidal side effects and changes in insulin levels).\\nLiterature:\\n-\\nDate 22-03-2006\\nCYP2C19 IM: omeprazol\\n1839\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of omeprazole results in an increase in the therapeutic effectiveness, without an increase in the side\\neffects.\\nLiterature:\\n1\\n. \\nPark S et al. Effects of CYP2C19 genetic polymorphisms on PK/PD responses of omeprazole in Korean healthy volunteers. J\\nKorean Med Sci 2017;32:729-736.\\n2\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3\\n. \\nWang L et al. Ilaprazole for the treatment of duodenal ulcer: a randomized, double-blind and controlled phase III trial. Curr Med\\nRes Opin 2012;28:101-9.\\n4\\n. \\nZendehdel N et al. Role of cytochrome P450 2C19 genetic polymorphisms in the therapeutic efficacy of omeprazole in Iranian\\npatients with erosive reflux esophagitis. Arch Iran Med 2010;13:406-12.\\n5\\n. \\nHelsby NA et al. Omeprazole-induced acute interstitial nephritis is not related to CYP2C19 genotype or CYP2C19 phenotype. Br J\\nClin Pharmacol 2010;69:516-9.\\n6\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n7\\n. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in\\nCaucasians. Br J Clin Pharmacol 2008;65:752-60.\\n8\\n. \\nOhkusa T et al. Effect of CYP2C19 polymorphism on the safety and efficacy of omeprazole in Japanese patients with recurrent\\nreflux esophagitis. Aliment Pharmacol Ther 2005;21:1331-9.\\n9\\n. \\nRoh HK et al. Omeprazole treatment of Korean patients: effects on gastric pH and gastrin release in relation to CYP2C19 geno-\\nand phenotypes. Basic Clin Pharmacol Toxicol 2004;95:112-9.\\n10\\n. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH.\\nAliment Pharmacol Ther 2001;15:1929-37.\\n11\\n. \\nSagar M et al. Effects of omeprazole on intragastric pH and plasma gastrin are dependent on the CYP2C19 polymorphism.\\nGastroenterology 2000;119:670-6.\\n12\\n. \\nFuruta T et al. CYP2C19 genotype status and effect of omeprazole on intragastric pH in humans. Clin Pharmacol Ther\\n1999;65:552-61.\\n13\\n. \\nShimatani T et al. Effect of omeprazole 10 mg on intragastric pH in three different CYP2C19 genotypes, compared with\\nomeprazole 20 mg and lafutidine 20 mg, a new H2-receptor antagonist. Aliment Pharmacol Ther 2003;18:1149–1157.\\n14\\n. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to\\nCYP2C19 genotype. Dig Dis Sci 2008;53:933-7.\\n15\\n. \\nSugimoto M et al. Initial 48-hour acid inhibition by intravenous infusion of omeprazole, famotidine, or both in relation to\\ncytochrome P450 2C19 genotype status. Clin Pharmacol Ther 2006;80:539-48.\\n16\\n. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19\\ngenotypic differences. Dig Dis Sci 2005;50:1625-31.\\n17\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n18\\n. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of\\nclarithromycin-sensitive strains of Helicobacter pylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 101}),\n",
       " Document(page_content=\"19\\n. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders\\nwith Helicobacter pylori infection. Eur J Clin Pharmacol 2005;61:375-9.\\n20\\n. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to\\nCYP2C19 metabolism. Aliment Pharmacol Ther 2005;21:283-8.\\n21\\n. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin\\nGastroenterol Hepatol 2004;2:22-30.\\n22\\n. \\nSapone A et al. The clinical role of cytochrome p450 genotypes in Helicobacter pylori management. Am J Gastroenterol\\n2003;98:1010-5.\\n23\\n. \\nMiwa H et al. Clarithromycin resistance, but not CYP2C-19 polymorphism, has a major impact on treatment success in 7-day\\ntreatment regimen for cure of H. pylori infection: a multiple logistic regression analysis. Dig Dis Sci 2001;46:2445-50.\\n24\\n. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton\\npump inhibitor (omeprazole or rabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n25\\n. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a\\nproton pump inhibitor, amoxicillin, and clarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n26\\n. \\nAoyama N et al. Sufficient effect of 1-week omeprazole and amoxicillin dual treatment for Helicobacter pylori eradication in\\ncytochrome P450 2C19 poor metabolizers. J Gastroenterol 1999;34 Suppl 11:80-3.\\n27\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n28\\n. \\nHu XP et al. Effects of CYP2C19 genetic polymorphism on the pharmacokinetics and pharmacodynamics of omeprazole in\\nChinese people. J Clin Pharm Ther 2007;32:517-24.\\n29\\n. \\nSmPC Prilosec (VS).\\nDate 05-03-2018\\nCYP2C19 PM: omeprazol\\n1840\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of omeprazole results in an increase in the therapeutic effectiveness, without an increase in the side\\neffects.\\nLiterature:\\n1\\n. \\nPark S et al. Effects of CYP2C19 genetic polymorphisms on PK/PD responses of omeprazole in Korean healthy volunteers. J\\nKorean Med Sci 2017;32:729-736.\\n2\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3\\n. \\nWang L et al. Ilaprazole for the treatment of duodenal ulcer: a randomized, double-blind and controlled phase III trial. Curr Med\\nRes Opin 2012;28:101-9.\\n4\\n. \\nHelsby NA et al. Omeprazole-induced acute interstitial nephritis is not related to CYP2C19 genotype or CYP2C19 phenotype. Br J\\nClin Pharmacol 2010;69:516-9.\\n5\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n6\\n. \\nOhkusa T et al. Effect of CYP2C19 polymorphism on the safety and efficacy of omeprazole in Japanese patients with recurrent\\nreflux esophagitis. Aliment Pharmacol Ther 2005;21:1331-9.\\n7\\n. \\nRoh HK et al. Omeprazole treatment of Korean patients: effects on gastric pH and gastrin release in relation to CYP2C19 geno-\\nand phenotypes. Basic Clin Pharmacol Toxicol 2004;95:112-9.\\n8\\n. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH.\\nAliment Pharmacol Ther 2001;15:1929-37.\\n9\\n. \\nSagar M et al. Effects of omeprazole on intragastric pH and plasma gastrin are dependent on the CYP2C19 polymorphism.\\nGastroenterology 2000;119:670-6.\\n10\\n. \\nFuruta T et al. CYP2C19 genotype status and effect of omeprazole on intragastric pH in humans. Clin Pharmacol Ther\\n1999;65:552-61.\\n11\\n. \\nShimatani T et al. Effect of omeprazole 10 mg on intragastric pH in three different CYP2C19 genotypes, compared with\\nomeprazole 20 mg and lafutidine 20 mg, a new H2-receptor antagonist. Aliment Pharmacol Ther 2003;18:1149–1157.\\n12\\n. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to\\nCYP2C19 genotype. Dig Dis Sci 2008;53:933-7.\\n13\\n. \\nSugimoto M et al. Initial 48-hour acid inhibition by intravenous infusion of omeprazole, famotidine, or both in relation to\\ncytochrome P450 2C19 genotype status. Clin Pharmacol Ther 2006;80:539-48.\\n14\\n. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases.\\nJ Gastroenterol Hepatol 2006;21:1381-7.\\n15\\n. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19\\ngenotypic differences. Dig Dis Sci 2005;50:1625-31.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 102}),\n",
       " Document(page_content='16\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n17\\n. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of\\nclarithromycin-sensitive strains of Helicobacter pylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n18\\n. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to\\nCYP2C19 metabolism. Aliment Pharmacol Ther 2005;21:283-8.\\n19\\n. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin\\nGastroenterol Hepatol 2004;2:22-30.\\n20\\n. \\nSapone A et al. The clinical role of cytochrome p450 genotypes in Helicobacter pylori management. Am J Gastroenterol\\n2003;98:1010-5.\\n21\\n. \\nMiwa H et al. Clarithromycin resistance, but not CYP2C-19 polymorphism, has a major impact on treatment success in 7-day\\ntreatment regimen for cure of H. pylori infection: a multiple logistic regression analysis. Dig Dis Sci 2001;46:2445-50.\\n22\\n. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton\\npump inhibitor (omeprazole or rabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n23\\n. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a\\nproton pump inhibitor, amoxicillin, and clarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n24\\n. \\nTanigawara Y et al. CYP2C19 genotype-related efficacy of omeprazole for the treatment of infection caused by Helicobacter\\npylori. Clin Pharmacol Ther 1999;66:528-34.\\n25\\n. \\nAoyama N et al. Sufficient effect of 1-week omeprazole and amoxicillin dual treatment for Helicobacter pylori eradication in\\ncytochrome P450 2C19 poor metabolizers. J Gastroenterol 1999;34 Suppl 11:80-3.\\n26\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n27\\n. \\nHu XP et al. Effects of CYP2C19 genetic polymorphism on the pharmacokinetics and pharmacodynamics of omeprazole in\\nChinese people. J Clin Pharm Ther 2007;32:517-24.\\n28\\n. \\nSmPC’s Losec en Prilosec (VS).\\nDate 05-03-2018\\nCYP2C19 UM: omeprazol\\n1841\\nThe genetic variation may lead to a reduced omeprazole plasma concentration and therefore reduced effectiveness.\\nRecommendation:\\nFor Helicobacter pylori ERADICATION THERAPY:\\n1\\n. \\nuse a 3-fold higher dose\\n2\\n. \\nadvise the patient to contact their doctor if symptoms of dyspepsia persist\\nOTHER INDICATIONS:\\n1\\n. \\nbe alert to reduced effectiveness\\n2\\n. \\nif necessary, use a 3-fold higher dose\\n3\\n. \\nadvise the patient to report persisting symptoms of dyspepsia\\nLiterature:\\n1\\n. \\nPark S et al. Effects of CYP2C19 genetic polymorphisms on PK/PD responses of omeprazole in Korean healthy volunteers. J\\nKorean Med Sci 2017;32:729-736.\\n2\\n. \\nChwiesko A et al. Effects of different omeprazole dosing on gastric pH in non-variceal upper gastrointestinal bleeding: a\\nrandomized prospective study. J Dig Dis 2016;17:588-599.\\n3\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n4\\n. \\nDury S et al. Agranulocytosis induced by proton pump inhibitors. J Clin Gastroenterol 2012;46:859.\\n5\\n. \\nWang L et al. Ilaprazole for the treatment of duodenal ulcer: a randomized, double-blind and controlled phase III trial. Curr Med\\nRes Opin 2012;28:101-9.\\n6\\n. \\nZendehdel N et al. Role of cytochrome P450 2C19 genetic polymorphisms in the therapeutic efficacy of omeprazole in Iranian\\npatients with erosive reflux esophagitis. Arch Iran Med 2010;13:406-12.\\n7\\n. \\nHelsby NA et al. Omeprazole-induced acute interstitial nephritis is not related to CYP2C19 genotype or CYP2C19 phenotype. Br J\\nClin Pharmacol 2010;69:516-9.\\n8\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n9\\n. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in\\nCaucasians. Br J Clin Pharmacol 2008;65:752-60.\\n10\\n. \\nOhkusa T et al. Effect of CYP2C19 polymorphism on the safety and efficacy of omeprazole in Japanese patients with recurrent\\nreflux esophagitis. Aliment Pharmacol Ther 2005;21:1331-9.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 103}),\n",
       " Document(page_content=\"11\\n. \\nRoh HK et al. Omeprazole treatment of Korean patients: effects on gastric pH and gastrin release in relation to CYP2C19 geno-\\nand phenotypes. Basic Clin Pharmacol Toxicol 2004;95:112-9.\\n12\\n. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH.\\nAliment Pharmacol Ther 2001;15:1929-37.\\n13\\n. \\nSagar M et al. Effects of omeprazole on intragastric pH and plasma gastrin are dependent on the CYP2C19 polymorphism.\\nGastroenterology 2000;119:670-6.\\n14\\n. \\nFuruta T et al. CYP2C19 genotype status and effect of omeprazole on intragastric pH in humans. Clin Pharmacol Ther\\n1999;65:552-61.\\n15\\n. \\nShimatani T et al. Effect of omeprazole 10 mg on intragastric pH in three different CYP2C19 genotypes, compared with\\nomeprazole 20 mg and lafutidine 20 mg, a new H2-receptor antagonist. Aliment Pharmacol Ther 2003;18:1149–1157.\\n16\\n. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to\\nCYP2C19 genotype. Dig Dis Sci 2008;53:933-7.\\n17\\n. \\nSugimoto M et al. Initial 48-hour acid inhibition by intravenous infusion of omeprazole, famotidine, or both in relation to\\ncytochrome P450 2C19 genotype status. Clin Pharmacol Ther 2006;80:539-48.\\n18\\n. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases.\\nJ Gastroenterol Hepatol 2006;21:1381-7.\\n19\\n. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19\\ngenotypic differences. Dig Dis Sci 2005;50:1625-31.\\n20\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n21\\n. \\nSugimoto M et al. Influences of proinflammatory and anti-inflammatory cytokine polymorphisms on eradication rates of\\nclarithromycin-sensitive strains of Helicobacter pylori by triple therapy. Clin Pharmacol Ther 2006;80:41-50.\\n22\\n. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders\\nwith Helicobacter pylori infection. Eur J Clin Pharmacol 2005;61:375-9.\\n23\\n. \\nSheu BS et al. Esomeprazole 40 mg twice daily in triple therapy and the efficacy of Helicobacter pylori eradication related to\\nCYP2C19 metabolism. Aliment Pharmacol Ther 2005;21:283-8.\\n24\\n. \\nFuruta T et al. Polymorphism of interleukin-1beta affects the eradication rates of Helicobacter pylori by triple therapy. Clin\\nGastroenterol Hepatol 2004;2:22-30.\\n25\\n. \\nSapone A et al. The clinical role of cytochrome p450 genotypes in Helicobacter pylori management. Am J Gastroenterol\\n2003;98:1010-5.\\n26\\n. \\nMiwa H et al. Clarithromycin resistance, but not CYP2C-19 polymorphism, has a major impact on treatment success in 7-day\\ntreatment regimen for cure of H. pylori infection: a multiple logistic regression analysis. Dig Dis Sci 2001;46:2445-50.\\n27\\n. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton\\npump inhibitor (omeprazole or rabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n28\\n. \\nFuruta T et al. Effect of genotypic differences in CYP2C19 on cure rates for Helicobacter pylori infection by triple therapy with a\\nproton pump inhibitor, amoxicillin, and clarithromycin. Clin Pharmacol Ther 2001;69:158-68.\\n29\\n. \\nTanigawara Y et al. CYP2C19 genotype-related efficacy of omeprazole for the treatment of infection caused by Helicobacter\\npylori. Clin Pharmacol Ther 1999;66:528-34.\\n30\\n. \\nAoyama N et al. Sufficient effect of 1-week omeprazole and amoxicillin dual treatment for Helicobacter pylori eradication in\\ncytochrome P450 2C19 poor metabolizers. J Gastroenterol 1999;34 Suppl 11:80-3.\\n31\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n32\\n. \\nRocha A et al. Investigation of the in vivo activity of CYP3A in Brazilian volunteers: comparison of midazolam and omeprazole\\nas drug markers. Eur J Clin Pharmacol 2008;64:901-6.\\n33\\n. \\nBaldwin RM et al. Increased omeprazole metabolism in carriers of the CYP2C19*17 allele; a pharmacokinetic study in healthy\\nvolunteers. Br J Clin Pharmacol 2008;65:767-74.\\n34\\n. \\nHu XP et al. Effects of CYP2C19 genetic polymorphism on the pharmacokinetics and pharmacodynamics of omeprazole in\\nChinese people. J Clin Pharm Ther 2007;32:517-24.\\n35\\n. \\nSim SC et al. A common novel CYP2C19 gene variant causes ultrarapid drug metabolism relevant for the drug response to proton\\npump inhibitors and antidepressants. Clin Pharmacol Ther 2006;79:103-13.\\n36\\n. \\nSmPC’s Losec en Prilosec (VS).\\nDate 05-03-2018\\nCYP2D6 IM: oxycodon\\n1587\\nNO action is required for this gene-drug interaction.\\nThe reduced conversion of oxycodone to the more active metabolite oxymorphone does not result in reduced analgesia for patients.\\nLiterature:\\n1\\n. \\nCajanus K et al. Analgesic plasma concentrations of oxycodone after surgery for breast cancer-which factors matter? Clin\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 104}),\n",
       " Document(page_content='Pharmacol Ther 2017 Jun 23 [Epub ahead of print].\\n2\\n. \\nStamer UM et al. CYP2D6 genotype dependent oxycodone metabolism in postoperative patients. PloS One 2013;8:e60239.\\n3\\n. \\nNaito T et al. CYP3A5*3 affects plasma disposition of noroxycodone and dose escalation in cancer patients receiving oxycodone.\\nJ Clin Pharmacol 2011;51:1529-38.\\n4\\n. \\nLemberg KK et al. Does co-administration of paroxetine change oxycodone analgesia: an interaction study in chronic pain\\npatients. Scan Jour Pain 2010:1;24-33.\\n5\\n. \\nComer SD et al. Abuse liability of oxycodone as a function of pain and drug use history. Drug Alcohol Depend 2010;109:130-8.\\n6\\n. \\nJannetto PJ et al. Utilization of pharmacogenomics and therapeutic drug monitoring for opioid pain management.\\nPharmacogenomics 2009;10:1157-67.\\nDate 20-11-2017\\nCYP2D6 PM: oxycodon\\n1586\\nNO action is required for this gene-drug interaction.\\nThe reduced conversion of oxycodone to the more active metabolite oxymorphone does not result in reduced analgesia in patients.\\nLiterature:\\n1\\n. \\nCajanus K et al. Analgesic plasma concentrations of oxycodone after surgery for breast cancer-which factors matter? Clin\\nPharmacol Ther 2017 Jun 23 [Epub ahead of print].\\n2\\n. \\nLam J et al. Putative association of ABCB1 2677G>T/A with oxycodone-induced central nervous system depression in\\nbreastfeeding mothers. Ther Drug Monit 2013;35:466-72.\\n3\\n. \\nStamer UM et al. CYP2D6 genotype dependent oxycodone metabolism in postoperative patients. PloS One 2013;8:e60239.\\n4\\n. \\nAndreassen TN et al. Do CYP2D6 genotypes reflect oxycodone requirements for cancer patients treated for cancer pain? A cross-\\nsectional multicentre study. Eur J Clin Pharmacol 2012;68:55-64.\\n5\\n. \\nZwisler ST et al. Impact of CYP2D6 genotype on post-operative intravenous oxycodone analgesia. Acta Anaesthesiol Scan\\n2010;54:232-40.\\n6\\n. \\nSamer CF et al. Genetic polymorphisms and drug interactions modulating CYP2D6 and CYP3A activities have a major effect on\\noxycodone analgesic efficacy and safety. Br J Pharmacol 2010;16:919-30.\\n7\\n. \\nComer SD et al. Abuse liability of oxycodone as a function of pain and drug use history. Drug Alcohol Depend 2010;109:130-8.\\n8\\n. \\nJannetto PJ et al. Utilization of pharmacogenomics and therapeutic drug monitoring for opioid pain management.\\nPharmacogenomics 2009;10:1157-67.\\n9\\n. \\nZwisler ST et al. The hypoalgesic effect of oxycodone in human experimental pain models in relation to the CYP2D6 oxidation\\npolymorphism. Basic Clin Pharmacol Toxicol 2009;104:335-44.\\n10\\n. \\nFoster A et al. Complicated pain management in a CYP450 2D6 poor metabolizer. Pain Pract 2007;7:352-6.\\n11\\n. \\nSusce MT et al. Response to hydrocodone, codeine and oxycodone in a CYP2D6 poor metabolizer. Prog Neuropsychopharmacol\\nBiol Psychiatry 2006;30:1356-8.\\n12\\n. \\nMaddocks I et al. Attenuation of morphine-induced delirium in palliative care by substitution with infusion of oxycodone. J Pain\\nSymptom Manage 1996;12:182-9.\\nDate 20-11-2017\\nCYP2D6 UM: oxycodon\\n1588\\nNO action is required for this gene-drug interaction.\\nThe increased conversion of oxycodone to the more active metabolite oxymorphone does not result in an increase in side effects in\\npatients.\\nLiterature:\\n1\\n. \\nCajanus K et al. Analgesic plasma concentrations of oxycodone after surgery for breast cancer-which factors matter? Clin\\nPharmacol Ther 2017 Jun 23 [Epub ahead of print].\\n2\\n. \\nLam J et al. Putative association of ABCB1 2677G>T/A with oxycodone-induced central nervous system depression in', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 105}),\n",
       " Document(page_content='breastfeeding mothers. Ther Drug Monit 2013;35:466-72.\\n3\\n. \\nStamer UM et al. CYP2D6 genotype dependent oxycodone metabolism in postoperative patients. PloS One 2013;8:e60239.\\n4\\n. \\nAndreassen TN et al. Do CYP2D6 genotypes reflect oxycodone requirements for cancer patients treated for cancer pain? A cross-\\nsectional multicentre study. Eur J Clin Pharmacol 2012;68:55-64.\\n5\\n. \\nLemberg KK et al. Does co-administration of paroxetine change oxycodone analgesia: an interaction study in chronic pain\\npatients. Scan Jour Pain 2010:1;24-33.\\n6\\n. \\nSamer CF et al. Genetic polymorphisms and drug interactions modulating CYP2D6 and CYP3A activities have a major effect on\\noxycodone analgesic efficacy and safety. Br J Pharmacol 2010;16:919-30.\\n7\\n. \\nde Leon J et al. Adverse drug reactions to oxycodone and hydrocodone in CYP2D6 ultrarapid metabolizers. J Clin\\nPsychopharmacol 2003;23:420-1.\\nDate 20-11-2017\\nCYP2C19 IM: pantoprazol\\n1847\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of pantoprazole results in an increase in the therapeutic effectiveness, without an increase in the side\\neffects.\\nLiterature:\\n1\\n. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with\\nproton pump inhibitors. Meta Gene 2016;9:159-64.\\n2\\n. \\nOrmeci A et al. Effect of cytochrome P450 2C19 polymorphisms on the Helicobacter pylori eradication rate following two-week\\ntriple therapy with pantoprazole or rabeprazole. Eur Rev Med Pharmacol Sci 2016;20:879-85.\\n3\\n. \\nHsu PI et al. A randomized controlled study comparing reverse hybrid therapy and standard triple therapy for Helicobacter pylori\\ninfection. Medicine (Baltimore) 2015;94:e2104.\\n4\\n. \\nRomán M et al. Evaluation of the relationship between polymorphisms in CYP2C19 and the pharmacokinetics of omeprazole,\\npantoprazole and rabeprazole. Pharmacogenomics 2014;15:1893-901.\\n5\\n. \\nGawrońska-Szklarz B et al. CYP2C19 polymorphism affects single-dose pharmacokinetics of oral pantoprazole in healthy\\nvolunteers. Eur J Clin Pharmacol 2012;68:1267-74.\\n6\\n. \\nSheu BS et al. CYP2C19 genotypes determine the efficacy of on-demand therapy of pantoprazole for reflux esophagitis as Los-\\nAngeles grades C and D. J Gastroenterol Hepatol 2012;27:104-9.\\n7\\n. \\nThacker DL et al. Stereoselective pharmacokinetics of stable isotope (+/-)-[13C]-pantoprazole: Implications for a rapid screening\\nphenotype test of CYP2C19 activity. Chirality 2011;23:904-9.\\n8\\n. \\nChen WY et al. Double-dosed pantoprazole accelerates the sustained symptomatic response in overweight and obese patients with\\nreflux esophagitis in Los Angeles grades A and B. Am J Gastroenterol 2010;105:1046-52.\\n9\\n. \\nGawrońska-Szklarz B et al. Effects of CYP2C19, MDR1, and interleukin 1-B gene variants on the eradication rate of Helicobacter\\npylori infection by triple therapy with pantoprazole, amoxicillin, and metronidazole. Eur J Clin Pharmacol 2010;66:681-7.\\n10\\n. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics\\nand CYP2C19 polymorphism. Aliment Pharmacol Ther 2010;31:150-9.\\n11\\n. \\nHunfeld NG et al. Effect of CYP2C19 \\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in\\nCaucasians. Br J Clin Pharmacol 2008;65:752-60.\\n12\\n. \\nChoi KD et al. Optimal dose of intravenous pantoprazole in patients with peptic ulcer bleeding requiring endoscopic hemostasis in\\nKorea. J Gastroenterol Hepatol 2009;24:1617-24.\\n13\\n. \\nOh JH et al. Low-dose intravenous pantoprazole for optimal inhibition of gastric acid in Korean patients. J Gastroenterol Hepatol\\n2007;22:1429-34.\\n14\\n. \\nOh JH et al. Effects of CYP2C19 and MDR1 genotype on the eradication rate of Helicobacter pylori infection by triple therapy\\nwith pantoprazole, amoxycillin and clarithromycin. J Gastroenterol Hepatol 2009;24:294-8.\\n15\\n. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple\\ntherapy with regular proton pump inhibitor dosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n16\\n. \\nKurzawski M et al. Effect of CYP2C19*17 gene variant on Helicobacter pylori eradication in peptic ulcer patients. Eur J Clin\\nPharmacol 2006;62:877-80.\\n17\\n. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders\\nwith Helicobacter pylori infection. Eur J Clin Pharmacol 2005;61:375-9.\\n18\\n. \\nKearns GL et al. Single-dose pharmacokinetics of oral and intravenous pantoprazole in children and adolescents. J Clin Pharmacol\\n2008;48:1356-65.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 106}),\n",
       " Document(page_content='Date 05-03-2018\\nCYP2C19 PM: pantoprazol\\n1848\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of pantoprazole results in an increase in the therapeutic effectiveness, without an increase in the side\\neffects.\\nLiterature:\\n1\\n. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with\\nproton pump inhibitors. Meta Gene 2016;9:159-64.\\n2\\n. \\nOrmeci A et al. Effect of cytochrome P450 2C19 polymorphisms on the Helicobacter pylori eradication rate following two-week\\ntriple therapy with pantoprazole or rabeprazole. Eur Rev Med Pharmacol Sci 2016;20:879-85.\\n3\\n. \\nHsu PI et al. A randomized controlled study comparing reverse hybrid therapy and standard triple therapy for Helicobacter pylori\\ninfection. Medicine (Baltimore) 2015;94:e2104.\\n4\\n. \\nGawrońska-Szklarz B et al. CYP2C19 polymorphism affects single-dose pharmacokinetics of oral pantoprazole in healthy\\nvolunteers. Eur J Clin Pharmacol 2012;68:1267-74.\\n5\\n. \\nSheu BS et al. CYP2C19 genotypes determine the efficacy of on-demand therapy of pantoprazole for reflux esophagitis as Los-\\nAngeles grades C and D. J Gastroenterol Hepatol 2012;27:104-9.\\n6\\n. \\nThacker DL et al. Stereoselective pharmacokinetics of stable isotope (+/-)-[13C]-pantoprazole: Implications for a rapid screening\\nphenotype test of CYP2C19 activity. Chirality 2011;23:904-9.\\n7\\n. \\nChen WY et al. Double-dosed pantoprazole accelerates the sustained symptomatic response in overweight and obese patients with\\nreflux esophagitis in Los Angeles grades A and B. Am J Gastroenterol 2010;105:1046-52.\\n8\\n. \\nGawrońska-Szklarz B et al. Effects of CYP2C19, MDR1, and interleukin 1-B gene variants on the eradication rate of Helicobacter\\npylori infection by triple therapy with pantoprazole, amoxicillin, and metronidazole. Eur J Clin Pharmacol 2010;66:681-7.\\n9\\n. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19\\ngenotypes. J Clin Gastroenterol 2009;43:920-5.\\n10\\n. \\nHunfeld NG et al. Effect of CYP2C19 \\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in\\nCaucasians. Br J Clin Pharmacol 2008;65:752-60.\\n11\\n. \\nChoi KD et al. Optimal dose of intravenous pantoprazole in patients with peptic ulcer bleeding requiring endoscopic hemostasis in\\nKorea. J Gastroenterol Hepatol 2009;24:1617-24.\\n12\\n. \\nOh JH et al. Low-dose intravenous pantoprazole for optimal inhibition of gastric acid in Korean patients. J Gastroenterol Hepatol\\n2007;22:1429-34.\\n13\\n. \\nOh JH et al. Effects of CYP2C19 and MDR1 genotype on the eradication rate of Helicobacter pylori infection by triple therapy\\nwith pantoprazole, amoxycillin and clarithromycin. J Gastroenterol Hepatol 2009;24:294-8.\\n14\\n. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple\\ntherapy with regular proton pump inhibitor dosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n15\\n. \\nKurzawski M et al. Effect of CYP2C19*17 gene variant on Helicobacter pylori eradication in peptic ulcer patients. Eur J Clin\\nPharmacol 2006;62:877-80.\\n16\\n. \\nKearns GL et al. Single-dose pharmacokinetics of oral and intravenous pantoprazole in children and adolescents. J Clin Pharmacol\\n2008;48:1356-65.\\n17\\n. \\nSPC’s Pantozol en Protonix I.V. (VS).\\nDate 05-03-2018\\nCYP2C19 UM: pantoprazol\\n1849\\nThe genetic variation may lead to reduced pantoprazole plasma concentrations and therefore reduced pantoprazole effectiveness.\\nRecommendation:\\nFor Helicobacter pylori ERADICATION THERAPY:\\n1\\n. \\nuse a 5-fold higher dose\\n2\\n. \\nadvise the patient to contact their doctor if symptoms of dyspepsia persist\\nOTHER INDICATIONS:\\n1\\n. \\nbe alert to reduced effectiveness\\n2\\n. \\nif necessary, use a 5-fold higher dose\\n3\\n. \\nadvise the patient to report persisting symptoms of dyspepsia', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 107}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nDeshpande N et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with\\nproton pump inhibitors. Meta Gene 2016;9:159-64.\\n2\\n. \\nOrmeci A et al. Effect of cytochrome P450 2C19 polymorphisms on the Helicobacter pylori eradication rate following two-week\\ntriple therapy with pantoprazole or rabeprazole. Eur Rev Med Pharmacol Sci 2016;20:879-85.\\n3\\n. \\nHsu PI et al. A randomized controlled study comparing reverse hybrid therapy and standard triple therapy for Helicobacter pylori\\ninfection. Medicine (Baltimore) 2015;94:e2104.\\n4\\n. \\nRomán M et al. Evaluation of the relationship between polymorphisms in CYP2C19 and the pharmacokinetics of omeprazole,\\npantoprazole and rabeprazole. Pharmacogenomics 2014;15:1893-901.\\n5\\n. \\nGawrońska-Szklarz B et al. CYP2C19 polymorphism affects single-dose pharmacokinetics of oral pantoprazole in healthy\\nvolunteers. Eur J Clin Pharmacol 2012;68:1267-74.\\n6\\n. \\nSheu BS et al. CYP2C19 genotypes determine the efficacy of on-demand therapy of pantoprazole for reflux esophagitis as Los-\\nAngeles grades C and D. J Gastroenterol Hepatol 2012;27:104-9.\\n7\\n. \\nThacker DL et al. Stereoselective pharmacokinetics of stable isotope (+/-)-[13C]-pantoprazole: Implications for a rapid screening\\nphenotype test of CYP2C19 activity. Chirality 2011;23:904-9.\\n8\\n. \\nChen WY et al. Double-dosed pantoprazole accelerates the sustained symptomatic response in overweight and obese patients with\\nreflux esophagitis in Los Angeles grades A and B. Am J Gastroenterol 2010;105:1046-52.\\n9\\n. \\nGawrońska-Szklarz B et al. Effects of CYP2C19, MDR1, and interleukin 1-B gene variants on the eradication rate of Helicobacter\\npylori infection by triple therapy with pantoprazole, amoxicillin, and metronidazole. Eur J Clin Pharmacol 2010;66:681-7.\\n10\\n. \\nHunfeld NG et al. A comparison of the acid-inhibitory effects of esomeprazole and pantoprazole in relation to pharmacokinetics\\nand CYP2C19 polymorphism. Aliment Pharmacol Ther 2010;31:150-9.\\n11\\n. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19\\ngenotypes. J Clin Gastroenterol 2009;43:920-5.\\n12\\n. \\nHunfeld NG et al. Effect of CYP2C19\\n2 and \\n17 mutations on pharmacodynamics and kinetics of proton pump inhibitors in\\nCaucasians. Br J Clin Pharmacol 2008;65:752-60.\\n13\\n. \\nChoi KD et al. Optimal dose of intravenous pantoprazole in patients with peptic ulcer bleeding requiring endoscopic hemostasis in\\nKorea. J Gastroenterol Hepatol 2009;24:1617-24.\\n14\\n. \\nOh JH et al. Low-dose intravenous pantoprazole for optimal inhibition of gastric acid in Korean patients. J Gastroenterol Hepatol\\n2007;22:1429-34.\\n15\\n. \\nOh JH et al. Effects of CYP2C19 and MDR1 genotype on the eradication rate of Helicobacter pylori infection by triple therapy\\nwith pantoprazole, amoxycillin and clarithromycin. J Gastroenterol Hepatol 2009;24:294-8.\\n16\\n. \\nKang JM et al. Effect of the CYP2C19 polymorphism on the eradication rate of Helicobacter pylori infection by 7-day triple\\ntherapy with regular proton pump inhibitor dosage. J Gastroenterol Hepatol 2008;23:1287-91.\\n17\\n. \\nKurzawski M et al. Effect of CYP2C19*17 gene variant on Helicobacter pylori eradication in peptic ulcer patients. Eur J Clin\\nPharmacol 2006;62:877-80.\\n18\\n. \\nGawronska-Szklarz B et al. Effect of CYP2C19 and MDR1 polymorphisms on cure rate in patients with acid-related disorders\\nwith Helicobacter pylori infection. Eur J Clin Pharmacol 2005;61:375-9.\\n19\\n. \\nKearns GL et al. Single-dose pharmacokinetics of oral and intravenous pantoprazole in children and adolescents. J Clin Pharmacol\\n2008;48:1356-65.\\n20\\n. \\nSPC’s Pantozol en Protonix I.V. (VS).\\nDate 05-03-2018\\nCYP2D6 IM: paroxetine\\n1563\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of paroxetine can increase as a result of the reduced activity of CYP2D6. However, studies did not find any\\nclinical effects.\\nLiterature:\\n1\\n. \\nJanssen PK et al. Nonresponders to daily paroxetine and another SSRI in men with lifelong premature ejaculation: a\\npharmacokinetic dose-escalation study for a rare phenomenon. Korean J Urol 2014;55:599-607.\\n2\\n. \\nSaruwatari J et al. Possible impact of the CYP2D6*10 polymorphism on the nonlinear pharmacokinetic parameter estimates of\\nparoxetine in Japanese patients with major depressive disorders. Pharmgenomics Pers Med 2014;7:121-7.\\n3\\n. \\nMurata Y et al. Severe sleepiness and excess sleep duration induced by paroxetine treatment is a beneficial pharmacological effect,\\nnot an adverse reaction. J Affect Disord 2013;150:1209-12.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 108}),\n",
       " Document(page_content='4\\n. \\nMurata Y et al. Effects of the serotonin 1A, 2A, 2C, 3A, and 3B and serotonin transporter gene polymorphisms on the occurrence\\nof paroxetine discontinuation syndrome. J Clin Psychopharmacol 2010;30:11-7.\\n5\\n. \\nVervers FF et al. Effect of cytochrome P450 2D6 genotype on maternal paroxetine plasma concentrations during pregnancy. Clin\\nPharmacokinet 2009;48:677-83.\\n6\\n. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6\\npolymorphisms. Int J Psychiatry Clin Pract 2009;13:345-348.\\n7\\n. \\nGex-Fabry M et al. CYP2D6 and ABCB1 genetic variability: influence on paroxetine plasma level and therapeutic response. Ther\\nDrug Monit 2008;30:474-82.\\n8\\n. \\nKuhn UD et al. Reboxetine and cytochrome P450--comparison with paroxetine treatment in humans. Int J Clin Pharmacol Ther\\n2007;45:36-46.\\n9\\n. \\nSugai T et al. The effect of 5-hydroxytryptamine 3A and 3B receptor genes on nausea induced by paroxetine. Pharmacogenomics J\\n2006;6:351-6.\\n10\\n. \\nFindling RL et al. Multiple dose pharmacokinetics of paroxetine in children and adolescents with major depressive disorder or\\nobsessive-compulsive disorder. Neuropsychopharmacology 2006;31:1274-85.\\n11\\n. \\nFeng Y et al. Paroxetine: population pharmacokinetic analysis in late-life depression using sparse concentration sampling. Br J\\nClin Pharmacol 2006;61:558-69.\\n12\\n. \\nUeda M et al. The impact of CYP2D6 genotypes on the plasma concentration of paroxetine in Japanese psychiatric patients. Prog\\nNeuro-psychopharmacol Biol Psychiatry 2006;30:486-91.\\n13\\n. \\nSawamura K ea. Effects of dosage and CYP2D6-mutated allele on plasma concentration of paroxetine. Eur J Clin Pharmacol\\n2004;60:553-7.\\n14\\n. \\nMurphy G ea. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\n15\\n. \\nOzdemir V ea. Paroxetine steady-state plasma concentration in relation to CYP2D6 genotype in extensive metabolizers. J Clin\\nPsychopharmacol 1999;19:472-5.\\nDate 14-05-2018\\nCYP2D6 PM: paroxetine\\n1564\\nNO action is needed for this gene-drug interaction.\\nThe plasma concentration of paroxetine can increase as a result of the reduced activity of CYP2D6. However, studies did not find any\\nclinical effects.\\nLiterature:\\n1\\n. \\nVervers FF et al. Effect of cytochrome P450 2D6 genotype on maternal paroxetine plasma concentrations during pregnancy. Clin\\nPharmacokinet 2009;48:677-83.\\n2\\n. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6\\npolymorphisms. Int J Psychiatry Clin Pract 2009;13:345-348.\\n3\\n. \\nGex-Fabry M et al. CYP2D6 and ABCB1 genetic variability: influence on paroxetine plasma level and therapeutic response. Ther\\nDrug Monit 2008;30:474-82.\\n4\\n. \\nKuhn UD et al. Reboxetine and cytochrome P450--comparison with paroxetine treatment in humans. Int J Clin Pharmacol Ther\\n2007;45:36-46.\\n5\\n. \\nFindling RL et al. Multiple dose pharmacokinetics of paroxetine in children and adolescents with major depressive disorder or\\nobsessive-compulsive disorder. Neuropsychopharmacology 2006;31:1274-85.\\n6\\n. \\nFeng Y et al. Paroxetine: population pharmacokinetic analysis in late-life depression using sparse concentration sampling. Br J\\nClin Pharmacol 2006;61:558-69.\\n7\\n. \\nSawamura K et al. Effects of dosage and CYP2D6-mutated allele on plasma concentration of paroxetine. Eur J Clin Pharmacol\\n2004;60:553-7.\\n8\\n. \\nCharlier C et al. Polymorphisms in the CYP 2D6 gene: association with plasma concentrations of fluoxetine and paroxetine. Ther\\nDrug Monit 2003;25:738-42.\\n9\\n. \\nMurphy G et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\n10\\n. \\nSindrup SH et al. Pharmacokinetics of the selective serotonin reuptake inhibitor paroxetine: nonlinearity and relation to the\\nsparteine oxidation polymorphism. Clin Pharmacol Ther 1992;51:288-95.\\n11\\n. \\nSindrup SH et al. The relationship between paroxetine and the sparteine oxidation polymorphism. Clin Pharmacol Ther\\n1992;51:278-87.\\nDate 14-05-2018', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 109}),\n",
       " Document(page_content='CYP2D6 UM: paroxetine\\n1565\\nEfficacy will probably be lacking. The genetic variation increases the conversion of paroxetine.\\nIt is not possible to offer substantiated advice for dose adjustment based on the literature.\\navoid paroxetine\\nAntidepressants that are not metabolised by CYP2D6, or to a lesser extent, include for example citalopram or sertraline.\\nLiterature:\\n1\\n. \\nGex-Fabry M et al. CYP2D6 and ABCB1 genetic variability: influence on paroxetine plasma level and therapeutic response. Ther\\nDrug Monit 2008;30:474-82.\\n2\\n. \\nFindling RL et al. Multiple dose pharmacokinetics of paroxetine in children and adolescents with major depressive disorder or\\nobsessive-compulsive disorder. Neuropsychopharmacology 2006;31:1274-85.\\n3\\n. \\nFeng Y et al. Paroxetine: population pharmacokinetic analysis in late-life depression using sparse concentration sampling. Br J\\nClin Pharmacol 2006;61:558-69.\\n4\\n. \\nGüzey C et al. Low serum concentrations of paroxetine in CYP2D6 ultrarapid metabolizers. J Clin Psychopharmacol 2006;26:211-\\n2.\\n5\\n. \\nCharlier C et al. Polymorphisms in the CYP 2D6 gene: association with plasma concentrations of fluoxetine and paroxetine. Ther\\nDrug Monit 2003;25:738-42.\\n6\\n. \\nMurphy G et al. Pharmacogenetics of antidepressant medication intolerance. Am J Psychiatry 2003;160:1830-5.\\nDate 14-05-2018\\nCYP2D6 IM: pimozide\\n2448\\nThe risk of QT-prolongation – and thereby also the risk of torsade de points – is theoretically increased, because the genetic variation\\nresults in an increase in the plasma concentration of pimozide. The risk of an excessively high plasma concentration can be negated by\\nfollowing the dose recommendations provided below.\\nRecommendation:\\nuse no more than the following doses (80% of the standard maximum dose):\\nadults 16 mg/day\\nchildren 0.08 mg/kg per day to a maximum of 3 mg/day\\nLiterature:\\n1\\n. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of\\naripiprazole, haloperidol, pimozide, and risperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 en\\npersoonlijke communicatie (gemiddelde dosisgecorrigeerde pimozidedalconcentraties).\\n2\\n. \\nNucci G et al. Population pharmacokinetic modelling of pimozide and its relation to CYP2D6 genotype. Poster presented at the\\nannual meeting of population approach group in Europe 2007.\\nDate 19-11-2018\\nCYP2D6 PM: pimozide\\n2447\\nThe risk of QT-prolongation – and thereby also the risk of torsade de points – is theoretically increased, because the genetic variation\\nresults in an increase in the plasma concentration of pimozide. The risk of an excessively high plasma concentration can be negated by\\nfollowing the dose recommendations provided below.\\nuse no more than the following doses (50% of the standard maximum dose):\\nadults 10 mg/day\\nchildren 0.05 mg/kg per day to a maximum of 2 mg/day', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 110}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of\\naripiprazole, haloperidol, pimozide, and risperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 en\\npersoonlijke communicatie (gemiddelde dosisgecorrigeerde pimozidedalconcentraties).\\n2\\n. \\nNucci G et al. Population pharmacokinetic modelling of pimozide and its relation to CYP2D6 genotype. Poster presented at the\\nannual meeting of population approach group in Europe 2007.\\n3\\n. \\nDesta Z et al. Effect of clarithromycin on the pharmacokinetics and pharmacodynamics of pimozide in healthy poor and extensive\\nmetabolizers of cytochrome P450 2D6 (CYP2D6). Clin Pharmacol Ther 1999;65:10-20.\\n4\\n. \\nPharmacogenetic changes to the FDA-approved Orap (pimozide) label include adult and pediatric dosing recommendations for\\nCYP2D6 poor metabolizers. FDA-nieuwsbericht 27-09-11.\\n5\\n. \\nSPC Orap (NL en VS).\\nDate 19-11-2018\\nCYP2D6 UM: pimozide\\n2449\\nNO action is required for this gene-drug interaction.\\nThis gene variation can result in lower pimozide concentrations. However, there is no evidence of reduced effectiveness.\\nLiterature:\\n1\\n. \\nvan der Weide K et al. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of\\naripiprazole, haloperidol, pimozide, and risperidone in psychiatric patients. J Clin Psychopharmacol 2015;35:228-36 en\\npersoonlijke communicatie (gemiddelde dosisgecorrigeerde pimozidedalconcentraties).\\nDate 19-11-2018\\nCYP2C19 IM: prasugrel\\n2546\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous\\ncoronary intervention. Circ Genom Precis Med 2018;11:e002069.\\n2\\n. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events\\nin Japanese patients with acute coronary syndrome: The PRASFIT-ACS study. J Cardiol 2016;68:29-36.\\n3\\n. \\nDoll JA et al. Impact of CYP2C19 metabolizer status on patients with ACS treated with prasugrel versus clopidogrel. J Am Coll\\nCardiol 2016;67:936-47.\\n4\\n. \\nVarenhorst C et al. Genetic variation of CYP2C19 affects both pharmacokinetic and pharmacodynamic responses to clopidogrel\\nbut not prasugrel in aspirin-treated patients with coronary artery disease. Eur Heart J 2009;30:1744-52.\\n5\\n. \\nMega JL et al. Cytochrome P450 genetic polymorphisms and the response to prasugrel: relationship to pharmacokinetic,\\npharmacodynamic, and clinical outcomes. Circulation 2009;119:2553-60.\\n6\\n. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response\\nto clopidogrel but not prasugrel. J Thromb Haemost 2007;5:2429-36.\\n7\\n. \\nSPC Efient (NL en VS).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 111}),\n",
       " Document(page_content='Date 19-11-2018\\nCYP2C19 PM: prasugrel\\n2545\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nLee CR et al. Clinical outcomes and sustainability of using CYP2C19 genotype-guided antiplatelet therapy after percutaneous\\ncoronary intervention. Circ Genom Precis Med 2018;11:e002069.\\n2\\n. \\nDeiman BA et al. Reduced number of cardiovascular events and increased cost-effectiveness by genotype-guided antiplatelet\\ntherapy in patients undergoing percutaneous coronary interventions in the Netherlands. Neth Heart J 2016;24:589-99.\\n3\\n. \\nOgawa H et al. Effects of CYP2C19 allelic variants on inhibition of platelet aggregation and major adverse cardiovascular events\\nin Japanese patients with acute coronary syndrome: The PRASFIT-ACS study. J Cardiol 2016;68:29-36.\\n4\\n. \\nDoll JA et al. Impact of CYP2C19 metabolizer status on patients with ACS treated with prasugrel versus clopidogrel. J Am Coll\\nCardiol 2016;67:936-47.\\n5\\n. \\nVarenhorst C et al. Genetic variation of CYP2C19 affects both pharmacokinetic and pharmacodynamic responses to clopidogrel\\nbut not prasugrel in aspirin-treated patients with coronary artery disease. Eur Heart J 2009;30:1744-52.\\n6\\n. \\nMega JL et al. Cytochrome P450 genetic polymorphisms and the response to prasugrel: relationship to pharmacokinetic,\\npharmacodynamic, and clinical outcomes. Circulation 2009;119:2553-60.\\n7\\n. \\nBrandt JT et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response\\nto clopidogrel but not prasugrel. J Thromb Haemost 2007;5:2429-36.\\n8\\n. \\nSPC Efient (NL en VS).\\nDate 19-11-2018\\nCYP2C19 UM: prasugrel\\n2547\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nMega JL et al. Cytochrome P450 genetic polymorphisms and the response to prasugrel: relationship to pharmacokinetic,\\npharmacodynamic, and clinical outcomes. Circulation 2009;119:2553-60.\\n2\\n. \\nSPC Efient (NL en VS).\\nDate 19-11-2018\\nCYP2D6 IM: propafenon\\n1596\\nGenetic variation increases the sum of the plasma concentrations of propafenone and the active metabolite 5-hydroxypropafenone. This\\nmay increase the risk of side effects.\\nRecommendation:\\nIt is not possible to offer adequately substantiated recommendations for dose adjustment based on the literature.\\n1\\n. \\nEither guide the dose by therapeutic drug monitoring, perform an ECG and be alert to side effects\\n2\\n. \\nOr choose an alternative\\nAntiarrhythmic drugs that are hardly if at all metabolised by CYP2D6 include, for example, sotalol, disopyramide, quinidine and\\namiodarone.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 112}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nMörike K et al. Propafenone for the prevention of atrial tachyarrhythmias after cardiac surgery: a randomized, double-blind\\nplacebo-controlled trial. Clin Pharmacol Ther 2008;84:104-10.\\n2\\n. \\nChen B et al. Influence of CYP2D6*10B genotype on pharmacokinetics of propafenone enantiomers in Chinese subjects. Acta\\nPharmacol Sin 2003;24:1277-80.\\n3\\n. \\nCai WM et al. Effect of CYP2D6*10 genotype on propafenone pharmacodynamics in Chinese patients with ventricular\\narrhythmia. Acta Pharmacol Sin 2002;23:1040-4.\\n4\\n. \\nCai WM et al. Simultaneous modeling of pharmacokinetics and pharmacodynamics of propafenone in healthy subjects. Acta\\nPharmacol Sin 2001;22:956-60.\\nDate 24-08-2016\\nCYP2D6 PM: propafenon\\n1595\\nGenetic variation increases the sum of the plasma concentrations of propafenone and the active metabolite 5-hydroxypropafenone. This\\nincreases the risk of side effects.\\nRecommendation:\\n1\\n. \\nReduce the dose to 30% of the standard dose, perform an ECG and monitor plasma concentrations\\nLiterature:\\n1\\n. \\nMörike K et al. Propafenone for the prevention of atrial tachyarrhythmias after cardiac surgery: a randomized, double-blind\\nplacebo-controlled trial. Clin Pharmacol Ther 2008;84:104-10.\\n2\\n. \\nJazwinska-Tarnawska E et al. The influence of CYP2D6 polymorphism on the antiarrhythmic efficacy of propafenone in patients\\nwith paroxysmal atrial fibrillation during 3 months propafenone prophylactic treatment. Int J Clin Pharmacol Ther 2001;39:288-\\n92.\\n3\\n. \\nChow MS et al. Evaluation of CYP2D6 oxidation of dextromethorphan and propafenone in a Chinese population with atrial\\nfibrillation. J Clin Pharmacol 2001;41:92-6.\\n4\\n. \\nLabbe L et al. Pharmacokinetic and pharmacodynamic interaction between mexiletine and propafenone in human beings. Clin\\nPharmacol Ther 2000;68:44-57.\\n5\\n. \\nDilger K et al. Consequences of rifampicin treatment on propafenone disposition in extensive and poor metabolizers of CYP2D6.\\nPharmacogenetics 1999;9:551-9.\\n6\\n. \\nCai WM et al. The influence of CYP2D6 activity on the kinetics of propafenone enantiomers in Chinese subjects. Br J Clin\\nPharmacol 1999;47:553-6.\\n7\\n. \\nMorike K et al. Propafenone in a usual dose produces severe side-effects: the impact of genetically determined metabolic status on\\ndrug therapy. J Intern Med 1995;238:469-72.\\n8\\n. \\nMorike KE et al. Quinidine-enhanced beta-blockade during treatment with propafenone in extensive metabolizer human subjects.\\nClin Pharmacol Ther 1994;55:28-34.\\n9\\n. \\nLee JT et al. The role of genetically determined polymorphic drug metabolism in the beta-blockade produced by propafenone. N\\nEngl J Med 1990;21:1764-8.\\n10\\n. \\nSiddoway LA et al. Polymorphism of propafenone metabolism and disposition in man: clinical and pharmacokinetic\\nconsequences. Circulation 1987;75:785-91.\\n11\\n. \\nSPC Rytmonorm.\\n12\\n. \\nSPC Rythmol SR (VS).\\nDate 24-08-2016\\nCYP2D6 UM: propafenon\\n1597\\nGenetic variation decreases the sum of the plasma concentrations of propafenone and the active metabolite 5-hydroxypropafenone. This\\nincreases the risk of reduced or no efficacy.\\nRecommendation:\\nIt is not possible to offer adequately substantiated recommendations for dose adjustment based on the literature.\\n1\\n. \\nEither monitor plasma concentrations, perform an ECG and be alert to reduced efficacy of the therapy.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 113}),\n",
       " Document(page_content='2\\n. \\nOr choose an alternative\\nAntiarrhythmic drugs that are hardly if at all metabolised by CYP2D6 include, for example, sotalol, disopyramide, quinidine and\\namiodarone.\\nLiterature:\\n1\\n. \\nMörike K et al. Propafenone for the prevention of atrial tachyarrhythmias after cardiac surgery: a randomized, double-blind\\nplacebo-controlled trial. Clin Pharmacol Ther 2008;84:104-10.\\n2\\n. \\nJazwinska-Tarnawska E et al. The influence of CYP2D6 polymorphism on the antiarrhythmic efficacy of propafenone in patients\\nwith paroxysmal atrial fibrillation during 3 months propafenone prophylactic treatment. Int J Clin Pharmacol Ther 2001;39:288-\\n92.\\nDate 24-08-2016\\nCYP2D6 IM: quetiapine\\n2394\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nKato D et al. Delirium resolving upon switching from risperidone to quetiapine: implication of CYP2D6 genotype.\\nPsychosomatics 2005;46:374-5.\\nDate 26-05-2009\\nCYP2D6 PM: quetiapine\\n2393\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 26-05-2009\\nCYP2D6 UM: quetiapine\\n2395\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 26-05-2009', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 114}),\n",
       " Document(page_content=\"CYP2C19 IM: rabeprazol\\n1856\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of rabeprazole does not result in an increase in side effects.\\nLiterature:\\n1\\n. \\nNakamura K et al. Limited effect of rebamipide in addition to proton pump inhibitor (PPI) in the treatment of post-endoscopic\\nsubmucosal dissection gastric ulcers: a randomized controlled trial comparing PPI plus rebamipide combination therapy with PPI\\nmonotherapy. Gut Liver 2016;10:917-924.\\n2\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3\\n. \\nKinoshita Y et al. Randomised clinical trial: a multicentre, double-blind, placebo- controlled study on the efficacy and safety of\\nrabeprazole 5 mg or 10 mg once daily in patients with non-erosive reflux disease. Aliment Pharmacol Ther 2011;33:213-24.\\n4\\n. \\nLay CS et al. Correlation of CYP2C19 genetic polymorphisms with Helicobacter pylori eradication in patients with cirrhosis and\\npeptic ulcer. J Chin Med Assoc 2010;73:188-93.\\n5\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n6\\n. \\nYamano HO et al. Plasma concentration of rabeprazole after 8-week administration in gastroesophageal reflux disease patients and\\nintragastric pH elevation. J Gastroenterol Hepatol 2008;23:534-40.\\n7\\n. \\nLee YC et al. Influence of cytochrome P450 2C19 genetic polymorphism and dosage of rabeprazole on accuracy of proton-pump\\ninhibitor testing in Chinese patients with gastroesophageal reflux disease. J Gastroenterol Hepatol 2007;22:1286-92.\\n8\\n. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J\\nGastroenterol Hepatol 2007;22:815-20.\\n9\\n. \\nAriizumi K et al. Therapeutic effects of 10 mg/day rabeprazole administration on reflux esophagitis was not influenced by the\\nCYP2C19 polymorphism. J Gastroenterol Hepatol 2006;21:1428-34.\\n10\\n. \\nHu YM et al. Pharmacodynamic and kinetic effect of rabeprazole on serum gastrin level in relation to CYP2C19 polymorphism in\\nChinese Hans. World J Gastroenterol 2006;12:4750-3.\\n11\\n. \\nSugimoto M et al. Comparison of an increased dosage regimen of rabeprazole versus a concomitant dosage regimen of famotidine\\nwith rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450 2C19 genotypes. Clin Pharmacol Ther\\n2005;77:302-11.\\n12\\n. \\nSugimoto M et al. Different dosage regimens of rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450\\n2C19 genotype status. Clin Pharmacol Ther 2004;76:290-301.\\n13\\n. \\nShimatani T et al. Rabeprazole 10 mg twice daily is superior to 20 mg once daily for night-time gastric acid suppression. Aliment\\nPharmacol Ther 2004;19:113-22.\\n14\\n. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH.\\nAliment Pharmacol Ther 2001;15:1929-37.\\n15\\n. \\nHorai Y et al. Pharmacodynamic effects and kinetic disposition of rabeprazole in relation to CYP2C19 genotypes. Aliment\\nPharmacol Ther 2001;15:793-803.\\n16\\n. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol\\nTher 2000;14:1259-66.\\n17\\n. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to\\nCYP2C19 genotype. Dig Dis Sci 2008;53:933-7.\\n18\\n. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19\\ngenotypic differences. Dig Dis Sci 2005;50:1625-31.\\n19\\n. \\nYang JC et al. Pharmacokinetic- pharmacodynamic analysis of the role of CYP2C19 genotypes in short-term rabeprazole-based\\ntriple therapy against Helicobacter pylori. Br J Clin Pharmacol 2009;67:503-10.\\n20\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n21\\n. \\nKuwayama H et al. Rabeprazole-based eradication therapy for Helicobacter pylori: a large-scale study in Japan. Aliment\\nPharmacol Ther 2007;25:1105-13.\\n22\\n. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori\\ninfection with lansoprazole- or rabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n23\\n. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton\\npump inhibitor (omeprazole or rabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n24\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n25\\n. \\nSPC Pariet.\\nDate 05-03-2018\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 115}),\n",
       " Document(page_content=\"CYP2C19 PM: rabeprazol\\n1857\\nNO action is required for this gene-drug interaction.\\nThe higher plasma concentration of rabeprazole does not result in an increase in side effects.\\nLiterature:\\n1\\n. \\nNakamura K et al. Limited effect of rebamipide in addition to proton pump inhibitor (PPI) in the treatment of post-endoscopic\\nsubmucosal dissection gastric ulcers: a randomized controlled trial comparing PPI plus rebamipide combination therapy with PPI\\nmonotherapy. Gut Liver 2016;10:917-924.\\n2\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3\\n. \\nKinoshita Y et al. Randomised clinical trial: a multicentre, double-blind, placebo- controlled study on the efficacy and safety of\\nrabeprazole 5 mg or 10 mg once daily in patients with non-erosive reflux disease. Aliment Pharmacol Ther 2011;33:213-24.\\n4\\n. \\nLay CS et al. Correlation of CYP2C19 genetic polymorphisms with Helicobacter pylori eradication in patients with cirrhosis and\\npeptic ulcer. J Chin Med Assoc 2010;73:188-93.\\n5\\n. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19\\ngenotypes. J Clin Gastroenterol 2009;43:920-5.\\n6\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n7\\n. \\nYamano HO et al. Plasma concentration of rabeprazole after 8-week administration in gastroesophageal reflux disease patients and\\nintragastric pH elevation. J Gastroenterol Hepatol 2008;23:534-40.\\n8\\n. \\nLee YC et al. Influence of cytochrome P450 2C19 genetic polymorphism and dosage of rabeprazole on accuracy of proton-pump\\ninhibitor testing in Chinese patients with gastroesophageal reflux disease. J Gastroenterol Hepatol 2007;22:1286-92.\\n9\\n. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J\\nGastroenterol Hepatol 2007;22:815-20.\\n10\\n. \\nAriizumi K et al. Therapeutic effects of 10 mg/day rabeprazole administration on reflux esophagitis was not influenced by the\\nCYP2C19 polymorphism. J Gastroenterol Hepatol 2006;21:1428-34.\\n11\\n. \\nHu YM et al. Pharmacodynamic and kinetic effect of rabeprazole on serum gastrin level in relation to CYP2C19 polymorphism in\\nChinese Hans. World J Gastroenterol 2006;12:4750-3.\\n12\\n. \\nSugimoto M et al. Comparison of an increased dosage regimen of rabeprazole versus a concomitant dosage regimen of famotidine\\nwith rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450 2C19 genotypes. Clin Pharmacol Ther\\n2005;77:302-11.\\n13\\n. \\nSugimoto M et al. Different dosage regimens of rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450\\n2C19 genotype status. Clin Pharmacol Ther 2004;76:290-301.\\n14\\n. \\nShimatani T et al. Rabeprazole 10 mg twice daily is superior to 20 mg once daily for night-time gastric acid suppression. Aliment\\nPharmacol Ther 2004;19:113-22.\\n15\\n. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH.\\nAliment Pharmacol Ther 2001;15:1929-37.\\n16\\n. \\nHorai Y et al. Pharmacodynamic effects and kinetic disposition of rabeprazole in relation to CYP2C19 genotypes. Aliment\\nPharmacol Ther 2001;15:793-803.\\n17\\n. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol\\nTher 2000;14:1259-66.\\n18\\n. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to\\nCYP2C19 genotype. Dig Dis Sci 2008;53:933-7.\\n19\\n. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases.\\nJ Gastroenterol Hepatol 2006;21: 1381-7.\\n20\\n. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19\\ngenotypic differences. Dig Dis Sci 2005;50:1625-31.\\n21\\n. \\nYang JC et al. Pharmacokinetic- pharmacodynamic analysis of the role of CYP2C19 genotypes in short-term rabeprazole-based\\ntriple therapy against Helicobacter pylori. Br J Clin Pharmacol 2009;67:503-10.\\n22\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n23\\n. \\nKuwayama H et al. Rabeprazole-based eradication therapy for Helicobacter pylori: a large-scale study in Japan. Aliment\\nPharmacol Ther 2007;25:1105-13.\\n24\\n. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori\\ninfection with lansoprazole- or rabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\\n25\\n. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton\\npump inhibitor (omeprazole or rabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n26\\n. \\nHokari K et al. Efficacy of triple therapy with rabeprazole for Helicobacter pylori infection and CYP2C19 genetic polymorphism.\\nAliment Pharmacol Ther 2001;15:1479-84.\\n27\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n28\\n. \\nSPC’s Pariet en Aciphex (VS).\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 116}),\n",
       " Document(page_content=\"Date 05-03-2018\\nCYP2C19 UM: rabeprazol\\n1858\\nNO action is required for this gene-drug interaction.\\nThere is currently insufficient information about this gene variation to recommend any action. Moreover, the fact that there are no\\ndifferences in effectiveness between PM and EM patients also makes differences in effectiveness between UM and EM patients less\\nlikely.\\nLiterature:\\n1\\n. \\nNakamura K et al. Limited effect of rebamipide in addition to proton pump inhibitor (PPI) in the treatment of post-endoscopic\\nsubmucosal dissection gastric ulcers: a randomized controlled trial comparing PPI plus rebamipide combination therapy with PPI\\nmonotherapy. Gut Liver 2016;10:917-924.\\n2\\n. \\nTang HL et al. Effects of CYP2C19 loss-of-function variants on the eradication of H. pylori infection in patients treated with\\nproton pump inhibitor-based triple therapy regimens: a meta-analysis of randomized clinical trials. PLoS One 2013;8:e62162.\\n3\\n. \\nKinoshita Y et al. Randomised clinical trial: a multicentre, double-blind, placebo- controlled study on the efficacy and safety of\\nrabeprazole 5 mg or 10 mg once daily in patients with non-erosive reflux disease. Aliment Pharmacol Ther 2011;33:213-24.\\n4\\n. \\nLay CS et al. Correlation of CYP2C19 genetic polymorphisms with Helicobacter pylori eradication in patients with cirrhosis and\\npeptic ulcer. J Chin Med Assoc 2010;73:188-93.\\n5\\n. \\nTseng PH et al. A comparative study of proton-pump inhibitor tests for Chinese reflux patients in relation to the CYP2C19\\ngenotypes. J Clin Gastroenterol 2009;43:920-5.\\n6\\n. \\nSaitoh T et al. Influences of CYP2C19 polymorphism on recurrence of reflux esophagitis during proton pump inhibitor\\nmaintenance therapy. Hepatogastroenterology 2009;56:703-6.\\n7\\n. \\nYamano HO et al. Plasma concentration of rabeprazole after 8-week administration in gastroesophageal reflux disease patients and\\nintragastric pH elevation. J Gastroenterol Hepatol 2008;23:534-40.\\n8\\n. \\nLee YC et al. Influence of cytochrome P450 2C19 genetic polymorphism and dosage of rabeprazole on accuracy of proton-pump\\ninhibitor testing in Chinese patients with gastroesophageal reflux disease. J Gastroenterol Hepatol 2007;22:1286-92.\\n9\\n. \\nLi ZS et al. Effect of esomeprazole and rabeprazole on intragastric pH in healthy Chinese: an open, randomized crossover trial. J\\nGastroenterol Hepatol 2007;22:815-20.\\n10\\n. \\nAriizumi K et al. Therapeutic effects of 10 mg/day rabeprazole administration on reflux esophagitis was not influenced by the\\nCYP2C19 polymorphism. J Gastroenterol Hepatol 2006;21:1428-34.\\n11\\n. \\nHu YM et al. Pharmacodynamic and kinetic effect of rabeprazole on serum gastrin level in relation to CYP2C19 polymorphism in\\nChinese Hans. World J Gastroenterol 2006;12:4750-3.\\n12\\n. \\nSugimoto M et al. Comparison of an increased dosage regimen of rabeprazole versus a concomitant dosage regimen of famotidine\\nwith rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450 2C19 genotypes. Clin Pharmacol Ther\\n2005;77:302-11.\\n13\\n. \\nSugimoto M et al. Different dosage regimens of rabeprazole for nocturnal gastric acid inhibition in relation to cytochrome P450\\n2C19 genotype status. Clin Pharmacol Ther 2004;76:290-301.\\n14\\n. \\nShimatani T et al. Rabeprazole 10 mg twice daily is superior to 20 mg once daily for night-time gastric acid suppression. Aliment\\nPharmacol Ther 2004;19:113-22.\\n15\\n. \\nShirai N et al. Effects of CYP2C19 genotypic differences in the metabolism of omeprazole and rabeprazole on intragastric pH.\\nAliment Pharmacol Ther 2001;15:1929-37.\\n16\\n. \\nHorai Y et al. Pharmacodynamic effects and kinetic disposition of rabeprazole in relation to CYP2C19 genotypes. Aliment\\nPharmacol Ther 2001;15:793-803.\\n17\\n. \\nAdachi K et al. CYP2C19 genotype status and intragastric pH during dosing with lansoprazole or rabeprazole. Aliment Pharmacol\\nTher 2000;14:1259-66.\\n18\\n. \\nAndo T et al. Endoscopic analysis of gastric ulcer after one week's treatment with omeprazole and rabeprazole in relation to\\nCYP2C19 genotype. Dig Dis Sci 2008;53:933-7.\\n19\\n. \\nJi S et al. Comparison of the efficacy of rabeprazole 10 mg and omeprazole 20 mg for the healing rapidity of peptic ulcer diseases.\\nJ Gastroenterol Hepatol 2006;21: 1381-7.\\n20\\n. \\nAndo T et al. A comparative study on endoscopic ulcer healing of omeprazole versus rabeprazole with respect to CYP2C19\\ngenotypic differences. Dig Dis Sci 2005;50:1625-31.\\n21\\n. \\nYang JC et al. Pharmacokinetic- pharmacodynamic analysis of the role of CYP2C19 genotypes in short-term rabeprazole-based\\ntriple therapy against Helicobacter pylori. Br J Clin Pharmacol 2009;67:503-10.\\n22\\n. \\nZhao F et al. Effect of CYP2C19 genetic polymorphisms on the efficacy of proton pump inhibitor-based triple therapy for\\nHelicobacter pylori eradication: a meta-analysis. Helicobacter 2008;13:532-41.\\n23\\n. \\nKuwayama H et al. Rabeprazole-based eradication therapy for Helicobacter pylori: a large-scale study in Japan. Aliment\\nPharmacol Ther 2007;25:1105-13.\\n24\\n. \\nMiki I et al. Impact of clarithromycin resistance and CYP2C19 genetic polymorphism on treatment efficacy of Helicobacter pylori\\ninfection with lansoprazole- or rabeprazole-based triple therapy in Japan. Eur J Gastroenterol Hepatol 2003;15:27-33.\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 117}),\n",
       " Document(page_content='25\\n. \\nDojo M et al. Effects of CYP2C19 gene polymorphism on cure rates for Helicobacter pylori infection by triple therapy with proton\\npump inhibitor (omeprazole or rabeprazole), amoxycillin and clarithromycin in Japan. Dig Liver Dis 2001;33:671-5.\\n26\\n. \\nHokari K et al. Efficacy of triple therapy with rabeprazole for Helicobacter pylori infection and CYP2C19 genetic polymorphism.\\nAliment Pharmacol Ther 2001;15:1479-84.\\n27\\n. \\nInaba T et al. Helicobacter pylori infection: CYP2C19 genotype and serum ferritin. J Gastroenterol Hepatol 2002;17:748-53.\\n28\\n. \\nSPC’s Pariet en Aciphex (VS).\\nDate 05-03-2018\\nCYP2D6 IM: risperidon\\n1536\\nNO action is needed for this gene-drug interaction.\\nThere is little evidence to support an increase in side effects caused by the genetic variation. The genetic variation may lead to a decrease\\nin the required maintenance dose. However, as the effect on the dose is smaller than that of the normal biological variation, action is not\\nuseful.\\nLiterature:\\n1\\n. \\nSuzuki Y et al. Effect of risperidone metabolism and P-glycoprotein gene polymorphism on QT interval in patients with\\nschizophrenia. Pharmacogenomics J 2014 Mar 4 [Epub ahead of print]\\n2\\n. \\nAlmoguera B et al. CYP2D6 poor metabolizer status might be associated with better response to risperidone treatment.\\nPharmacogenet Genomics 2013 ;23:627-30.\\n3\\n. \\nAlmoguera B et al. Association of common genetic variants with risperidone adverse events in a Spanish schizophrenic\\npopulation. Pharmacogenomics J 2013; 13:197-204. PubMed PMID: 22212732.\\n4\\n. \\nMas S et al. Intuitive pharmacogenetics: spontaneous risperidone dosage is related to CYP2D6, CYP3A5 and ABCB1 genotypes.\\nPharmacogenomics J 2012; 12:255-9.\\n5\\n. \\nJovanović N et al. The role of CYP2D6 and ABCB1 pharmacogenetics in drug-naïve patients with first-episode schizophrenia\\ntreated with risperidone. Eur J Clin Pharmacol 2010; 66:1109-17.\\n6\\n. \\nWang L et al. Serum prolactin levels, plasma risperidone levels, polymorphism of cytochrome P450 2D6 and clinical response in\\npatients with schizophrenia. J Psychopharmacol 2007;21:837-42.\\n7\\n. \\nde Leon J et al. A study of genetic (CYP2D6 and ABCB1) and environmental (drug inhibitors and inducers) variables that may\\ninfluence plasma risperidone levels. Pharmacopsychiatry 2007;40:93-102.\\n8\\n. \\nCho HY et al. Pharmacokinetics and bioequivalence evaluation of risperidone in healthy male subjects with different CYP2D6\\ngenotypes. Arch Pharm Res 2006;29:525-33.\\n9\\n. \\nLane HY et al. Risperidone-related weight gain: genetic and nongenetic predictors. J Clin Psychopharmacol 2006;26:128-34.\\n10\\n. \\nRiedel M et al. Risperidone plasma levels, clinical response and side-effects. Eur Arch Psychiatry Clin Neurosci 2005;255:261-8.\\n11\\n. \\nKato D et al. Delirium resolving upon switching from risperidone to quetiapine: implication of CYP2D6 genotype.\\nPsychosomatics 2005;46:374-5.\\n12\\n. \\nKakihara S et al. Prediction of response to risperidone treatment with respect to plasma concencentrations of risperidone,\\ncatecholamine metabolites, and polymorphism of cytochrome P450 2D6. Int Clin Psychopharmacol 2005;20:71-8.\\n13\\n. \\nde Leon J et al. The CYP2D6 poor metabolizer phenotype may be associated with risperidone adverse drug reactions and\\ndiscontinuation. J Clin Psychiatry 2005;66:15-27.\\n14\\n. \\nLlerena A et al. QTc interval, CYP2D6 and CYP2C9 genotypes and risperidone plasma concentrations. J Psychopharmacol\\n2004;18:189-93.\\n15\\n. \\nYasui-Furukori N et al. Effects of various factors on steady-state plasma concentrations of risperidone and 9-hydroxyrisperidone:\\nlack of impact of MDR-1 genotypes. Br J Clin Pharmacol 2004;57:569-75.\\n16\\n. \\nRoh HK et al. Risperidone metabolism in relation to CYP2D6*10 allele in Korean schizophrenic patients. Eur J Clin Pharmacol\\n2001;57:671-5.\\n17\\n. \\nScordo MG et al. Cytochrome P450 2D6 genotype and steady state plasma levels of risperidone and 9-hydroxyrisperidone.\\nPsychopharmacology 1999;147:300-5.\\n18\\n. \\nBork JA et al. A pilot study on risperidone metabolism: the role of cytochromes P450 2D6 and 3A. J Clin Psychiatry\\n1999;60:469-76.\\nDate 27-05-2015\\nCYP2D6 PM: risperidon\\n1537\\nNO action is needed for this gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 118}),\n",
       " Document(page_content='The genetic variation can result in both an increase in side effects and a stronger decrease in schizophrenia symptoms. In addition to this,\\nthe genetic variation may lead to a decrease in the required maintenance dose. However, as the effect on the dose is smaller than that of\\nthe normal biological variation, action is not useful.\\nLiterature:\\n1\\n. \\nAlmoguera B et al. CYP2D6 poor metabolizer status might be associated with better response to risperidone treatment.\\nPharmacogenet Genomics 2013 ;23:627-30.\\n2\\n. \\nAlmoguera B et al. Association of common genetic variants with risperidone adverse events in a Spanish schizophrenic\\npopulation. Pharmacogenomics J 2013; 13:197-204. PubMed PMID: 22212732.\\n3\\n. \\nMas S et al. Intuitive pharmacogenetics: spontaneous risperidone dosage is related to CYP2D6, CYP3A5 and ABCB1 genotypes.\\nPharmacogenomics J 2012; 12:255-9.\\n4\\n. \\nJovanović N et al. The role of CYP2D6 and ABCB1 pharmacogenetics in drug-naïve patients with first-episode schizophrenia\\ntreated with risperidone. Eur J Clin Pharmacol 2010; 66:1109-17.\\n5\\n. \\nde Leon J et al. A study of genetic (CYP2D6 and ABCB1) and environmental (drug inhibitors and inducers) variables that may\\ninfluence plasma risperidone levels. Pharmacopsychiatry 2007;40:93-102.\\n6\\n. \\nde Leon J et al. Polymorphic variations in GSTM1, GSTT1, PgP, CYP2D6, CYP3A5, and dopamine D2 and D3 receptors and\\ntheir association with tardive dyskinesia in severe mental illness. J Clin Psychopharmacol 2005;25:448-56.\\n7\\n. \\nde Leon J et al. The CYP2D6 poor metabolizer phenotype may be associated with risperidone adverse drug reactions and\\ndiscontinuation. J Clin Psychiatry 2005;66:15-27.\\n8\\n. \\nLlerena A et al. QTc interval, CYP2D6 and CYP2C9 genotypes and risperidone plasma concentrations. J Psychopharmacol\\n2004;18:189-93.\\n9\\n. \\nKohnke MD et al. Cytochrome P450 2D6 deficiency and its clinical relevance in a patient treated with risperidone.\\nPharmacopsychiatry 2002;35(3):116-8.\\n10\\n. \\nScordo MG et al. Cytochrome P450 2D6 genotype and steady state plasma levels of risperidone and 9-hydroxyrisperidone.\\nPsychopharmacology 1999;147:300-5.\\n11\\n. \\nBork JA et al. A pilot study on risperidone metabolism: the role of cytochromes P450 2D6 and 3A. J Clin Psychiatry\\n1999;60:469-76.\\n12\\n. \\nSPC Risperdal.\\nDate 27-05-2015\\nCYP2D6 UM: risperidon\\n1535\\nNO action is needed for this gene-drug interaction.\\nGenetic variation may lead to an increase in the required maintenance dose. However, as the effect is smaller than that of the normal\\nbiological variation, action is not useful.\\nLiterature:\\n1\\n. \\nAlmoguera B et al. CYP2D6 poor metabolizer status might be associated with better response to risperidone treatment.\\nPharmacogenet Genomics 2013 ;23:627-30.\\n2\\n. \\nAlmoguera B et al. Association of common genetic variants with risperidone adverse events in a Spanish schizophrenic\\npopulation. Pharmacogenomics J 2013; 13:197-204. PubMed PMID: 22212732.\\n3\\n. \\nMas S et al. Intuitive pharmacogenetics: spontaneous risperidone dosage is related to CYP2D6, CYP3A5 and ABCB1 genotypes.\\nPharmacogenomics J 2012; 12:255-9.\\n4\\n. \\nde Leon J et al. A study of genetic (CYP2D6 and ABCB1) and environmental (drug inhibitors and inducers) variables that may\\ninfluence plasma risperidone levels. Pharmacopsychiatry 2007;40:93-102.\\n5\\n. \\nde Leon J et al. The CYP2D6 poor metabolizer phenotype may be associated with risperidone adverse drug reactions and\\ndiscontinuation. J Clin Psychiatry 2005;66:15-27.\\n6\\n. \\nLlerena A et al. QTc interval, CYP2D6 and CYP2C9 genotypes and risperidone plasma concentrations. J Psychopharmacol\\n2004;18:189-93.\\n7\\n. \\nGuzey C et al. Risperidone metabolism and the impact of being a cytochrome P450 2D6 ultrarapid metabolizer. J Clin Psychiatry\\n2000;61:600-1.\\n8\\n. \\nScordo MG et al. Cytochrome P450 2D6 genotype and steady state plasma levels of risperidone and 9-hydroxyrisperidone.\\nPsychopharmacology 1999;147:300-5.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 119}),\n",
       " Document(page_content='Date 27-05-2015\\nCYP2C19 IM: sertraline\\n2008\\nNO action is needed for this gene-drug interaction.\\nThe gene variation has a minor effect on the sertraline plasma concentration. No effect on side effects was found.\\nLiterature:\\n1\\n. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy\\nvolunteers. Basic Clin Pharmacol Toxicol 2018;122:501-511.\\n2\\n. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome.\\nBrain Dev 2017;39:483-492.\\n3\\n. \\nYuce-Artun N et al. Influence of CYP2B6 and CYP2C19 polymorphisms on sertraline metabolism in major depression patients.\\nInt J Clin Pharm 2016;38:388-94.\\n4\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\n5\\n. \\nRudberg I et al. Serum concentrations of sertraline and N-desmethyl sertraline in relation to CYP2C19 genotype in psychiatric\\npatients. Eur J Clin Pharmacol 2008;64:1181-8.\\n6\\n. \\nWang JH et al. Pharmacokinetics of sertraline in relation to genetic polymorphism of CYP2C19. Clin Pharmacol Ther 2001;70:42-\\n7.\\nDate 14-05-2018\\nCYP2C19 PM: sertraline\\n2009\\nThe risk of side effects is increased. The gene variation leads to increased plasma concentrations of sertraline\\nDo not give doses exceeding 75 mg/day\\nGuide the dose by response and side effects and/or sertraline plasma concentration.\\nLiterature:\\n1\\n. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome.\\nBrain Dev 2017;39:483-492.\\n2\\n. \\nYuce-Artun N et al. Influence of CYP2B6 and CYP2C19 polymorphisms on sertraline metabolism in major depression patients.\\nInt J Clin Pharm 2016;38:388-94.\\n3\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\n4\\n. \\nRudberg I et al. Serum concentrations of sertraline and N-desmethyl sertraline in relation to CYP2C19 genotype in psychiatric\\npatients. Eur J Clin Pharmacol 2008;64:1181-8.\\n5\\n. \\nGrasmader K et al. Impact of polymorphisms of cytochrome-P450 isoenzymes 2C9, 2C19 and 2D6 on plasma concentrations and\\nclinical effects of antidepressants in a naturalistic clinical setting. Eur J Clin Pharmacol 2004;60:329-36.\\n6\\n. \\nWang JH et al. Pharmacokinetics of sertraline in relation to genetic polymorphism of CYP2C19. Clin Pharmacol Ther 2001;70:42-\\n7.\\n7\\n. \\nSPC Zoloft.\\nDate 14-05-2018\\nCYP2C19 UM: sertraline\\n2010\\nNO action is needed for this gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 120}),\n",
       " Document(page_content='The gene variation has a negligible effect on the plasma concentration of sertraline. Moreover, no significant effect on response and side\\neffects has been found.\\nLiterature:\\n1\\n. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome.\\nBrain Dev 2017;39:483-492.\\n2\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\n3\\n. \\nRudberg I et al. Serum concentrations of sertraline and N-desmethyl sertraline in relation to CYP2C19 genotype in psychiatric\\npatients. Eur J Clin Pharmacol 2008;64:1181-8.\\nDate 14-05-2018\\nCYP2D6 IM: sertraline\\n3513\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy\\nvolunteers. Basic Clin Pharmacol Toxicol 2018;122:501-511.\\n2\\n. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome.\\nBrain Dev 2017;39:483-492.\\n3\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\nDate 14-05-2018\\nCYP2D6 PM: sertraline\\n3512\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy\\nvolunteers. Basic Clin Pharmacol Toxicol 2018;122:501-511.\\n2\\n. \\nAlOlaby RR et al. Molecular biomarkers predictive of sertraline treatment response in young children with fragile X syndrome.\\nBrain Dev 2017;39:483-492.\\n3\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\n4\\n. \\nHamelin BA et al. The disposition of fluoxetine but not sertraline is altered in poor metabolizers of debrisoquin. Clin Pharmacol\\nTher 1996;60:512-21.\\nDate 14-05-2018\\nCYP2D6 UM: sertraline\\n3514\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 121}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nSaiz-Rodríguez M et al. Effect of polymorphisms on the pharmacokinetics, pharmacodynamics and safety of sertraline in healthy\\nvolunteers. Basic Clin Pharmacol Toxicol 2018;122:501-511.\\n2\\n. \\nBrandl EJ et al. Influence of CYP2D6 and CYP2C19 gene variants on antidepressant response in obsessive-compulsive disorder.\\nPharmacogenomics J 2014;14:176-81.\\nDate 14-05-2018\\nSLCO1B1 521CC: simvastatine\\n4056\\nThe genetic polymorphism leads to reduced simvastatin transport to the liver. This increases simvastatin plasma concentrations and\\ntherefore the risk of myopathy.\\nRecommendation:\\n1\\n. \\nChoose an alternative\\nConsider any additional risk factors for statin-induced myopathy.\\nRosuvastatin and pravastatin are influenced to a lesser extent by SLCO1B1 polymorphisms. They are also not influenced by\\nCYP3A4 inhibitors such as amiodarone, verapamil and diltiazem.\\nFluvastatin is not influenced by SLCO1B1 polymorphisms or CYP3A4 inhibitors.\\nLiterature:\\n1\\n. \\nHu M et al. Intronic variants in SLCO1B1 related to statin-induced myopathy are associated with the low-density lipoprotein\\ncholesterol response to statins in Chinese patients with hyperlipidaemia. Pharmacogenet Genomics 2012 Jun 1 [Epub ahead of\\nprint].\\n2\\n. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and\\natorvastatin. Pharmacogenomics J 2012;12:233-7.\\n3\\n. \\nSortica VA et al. SLCO1B1 gene variability influences lipid-lowering efficacy on simvastatin therapy in Southern Brazilians. Clin\\nChem Lab Med 2012;50:441-8.\\n4\\n. \\nBailey KM et al. Hepatic metabolism and transporter gene variants enhance response to rosuvastatin in patients with acute\\nmyocardial infarction: the GEOSTAT-1 Study. Circ Cardiovasc Genet 2010;3:276-85.\\n5\\n. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n6\\n. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with\\nincreased cholesterol synthesis rate. Pharmacogenet Genomics 2008;18:921-6.\\n7\\n. \\nSEARCH Collaborative Group. SLCO1B1 variants and statin-induced myopathy -- a genomewide study. N Engl J Med\\n2008;359:789-99.\\n8\\n. \\nPasanen MK et al. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genomics\\n2006;16:873-9.\\n9\\n. \\nRamsey LB et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin-induced\\nmyopathy: 2014 update. Clin Pharmacol Ther 2014;96:423-8.\\nDate 25-05-2016\\nSLCO1B1 521TC: simvastatine\\n4055\\nThe genetic polymorphism may lead to reduced simvastatin transport to the liver. This may increase simvastatin plasma concentrations\\nand therefore the risk of myopathy.\\nRecommendation:\\n1\\n. \\nChoose an alternative\\nConsider any additional risk factors for statin-induced myopathy.\\nRosuvastatin and pravastatin are influenced to a lesser extent by SLCO1B1 polymorphisms. They are also not influenced by\\nCYP3A4 inhibitors such as amiodarone, verapamil and diltiazem.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 122}),\n",
       " Document(page_content='Fluvastatin is not influenced by SLCO1B1 polymorphisms or CYP3A4 inhibitors.\\n2\\n. \\nIf an alternative is not an option:\\n1\\n. \\nAvoid simvastatin doses exceeding 40 mg/day\\n2\\n. \\nAdvise the patient to contact their doctor in the event of muscle symptoms.\\nLiterature:\\n1\\n. \\nHu M et al. Intronic variants in SLCO1B1 related to statin-induced myopathy are associated with the low-density lipoprotein\\ncholesterol response to statins in Chinese patients with hyperlipidaemia. Pharmacogenet Genomics 2012 Jun 1 [Epub ahead of\\nprint].\\n2\\n. \\nBrunham LR et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and\\natorvastatin. Pharmacogenomics J 2012;12:233-7.\\n3\\n. \\nSortica VA et al. SLCO1B1 gene variability influences lipid-lowering efficacy on simvastatin therapy in Southern Brazilians. Clin\\nChem Lab Med 2012;50:441-8.\\n4\\n. \\nBailey KM et al. Hepatic metabolism and transporter gene variants enhance response to rosuvastatin in patients with acute\\nmyocardial infarction: the GEOSTAT-1 Study. Circ Cardiovasc Genet 2010;3:276-85.\\n5\\n. \\nVoora D et al. The SLCO1B1*5 genetic variant is associated with statin-induced side effects. J Am Coll Cardiol 2009;54:1609-16.\\n6\\n. \\nPasanen MK et al. Polymorphism of the hepatic influx transporter organic anion transporting polypeptide 1B1 is associated with\\nincreased cholesterol synthesis rate. Pharmacogenet Genomics 2008;18:921-6.\\n7\\n. \\nSEARCH Collaborative Group. SLCO1B1 variants and statin-induced myopathy -- a genomewide study. N Engl J Med\\n2008;359:789-99.\\n8\\n. \\nPasanen MK et al. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genomics\\n2006;16:873-9.\\n9\\n. \\nRamsey LB et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin-induced\\nmyopathy: 2014 update. Clin Pharmacol Ther 2014;96:423-8.\\nDate 25-05-2016\\nCYP2D6 IM: sotalol\\n2540\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 PM: sotalol\\n2539\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n-\\nDate 24-08-2016\\nCYP2D6 UM: sotalol\\n2541\\nThis is NOT a gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 123}),\n",
       " Document(page_content='Literature:\\n-\\nDate 24-08-2016\\nCYP3A5 heterozygote expressor: tacrolimus\\n2358\\nGenetic variation results in an increased conversion of tacrolimus to inactive metabolites and as a result a higher dose is required.\\nAdjustment of the initial dose results in an increased chance of reaching a tacrolimus concentration within the target range before the\\nstart of therapeutic drug monitoring on day three. However, there is no direct evidence that this results in improved clinical results.\\nRecommendation:\\nIndications OTHER than liver transplantation:\\n1\\n. \\nStart with 1.75 times of the standard initial dose that would yield the desired result in non-expressors\\nAdjustment of the dose should then be based on therapeutic drug monitoring.\\nNOTE: The initial dose that yields the desired result in non-expressors can be lower than the normal initial dose. In the\\nexample provided below, this dose was 75 % of the standard initial dose. A 1.75 time dose increase corresponds in this case\\nto a 1.3 time dose increase of the standard initial dose.\\nFor example: After three days, Thervet et al. found a median trough concentration for tacrolimus of 12.3 ng/mL at an initial\\ndose of 0.15 mg/kg twice daily for heterozygous kidney transplant patients. Their target value was 10 - 15 ng/mL. This is\\nlower than the target value that is used in the Netherlands in the first two - four weeks after kidney transplantation (15 - 20\\nng/mL).\\nFor the reference group of non-expressors, a median trough concentration of 16.6 ng/mL and 12.0 ng/mL was achieved at a\\ndosage of 0.1 mg/kg twice daily and 0.075 mg/kg twice daily respectively. In this hospital, the first dose is administered\\nbefore the CYP3A5 genotype is known. The second dose is reduced according to the genotype.\\nLIVER transplantation:\\nIn addition to the patient’s genotype, the metabolism of tacrolimus is also determined by the genotype of the transplanted liver.\\nLIVER is also of the genotype HETEROZYGOUS EXPRESSOR:\\n1\\n. \\nStart with 1.75 times the standard initial dose\\nAdjustment of the dose should then be based on therapeutic drug monitoring.\\nLIVER has a DIFFERENT genotype:\\nThere is insufficient evidence in the literature to support a dose recommendation.\\nLiterature:\\n1\\n. \\nRojas L et al. Effect of CYP3A5*3 on kidney transplant recipients treated with tacrolimus: a systematic review and meta-analysis\\nof observational studies. Pharmacogenomics J 2015;15:38-48.\\n2\\n. \\nBuendia JA et al. Effects of combinational CYP3A5 6986A>G polymorphism in graft liver and native intestine on the\\npharmacokinetics of tacrolimus in liver transplant patients: a meta-analysis. Ther Drug Monit 2014;36:442-7.\\n3\\n. \\nUesugi M et al. Impact of cytochrome P450 3A5 polymorphism in graft livers on the frequency of acute cellular rejection in\\nliving-donor liver transplantation. Pharmacogenet Genomics 2014;24:356-66.\\n4\\n. \\nTerrazzino S et al. The effect of CYP3A5 6986A>G and ABCB1 3435C>T on tacrolimus dose-adjusted trough levels and acute\\nrejection rates in renal transplant patients: a systematic review and meta-analysis. Pharmacogenet Genomics 2012;22:642-5.\\n5\\n. \\nTang HL et al. Lower tacrolimus daily dose requirements and acute rejection rates in the CYP3A5 non-expressers than expressers.\\nPharmacogenet Genomics 2011;21:713-20.\\n6\\n. \\nJacobson PA et al. Novel polymorphisms associated with tacrolimus trough concentrations: results from a multicenter kidney\\ntransplant consortium. Transplantation 2011;91:300-8.\\n7\\n. \\nThervet E et al. Optimization of initial tacrolimus dose using pharmacogenetic testing. Clin Pharmacol Ther 2010; 87: 721-726.\\n8\\n. \\nSatoh S et al. Lack of tacrolimus circadian pharmacokinetics and CYP3A5 pharmacogenetics in the early and maintenance stages\\nin Japanese renal transplant recipients. Br J Clin Pharmacol 2008;66:207-14.\\n9\\n. \\nKlauke B et al. No association between single nucleotide polymorphisms and the development of nephrotoxicity after orthotopic\\nheart transplantation. J Heart Lung Transplant 2008;27:741-5.\\n10\\n. \\nFukudo M et al. Impact of MDR1 and CYP3A5 on the oral clearance of tacrolimus and tacrolimus-related renal dysfunction in\\nadult living-donor liver transplant patients. Pharmacogenet Genomics 2008;18:413-23.\\n11\\n. \\nHesselink DA et al. CYP3A5 genotype is not associated with a higher risk of acute rejection in tacrolimus-treated renal transplant', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 124}),\n",
       " Document(page_content='recipients. Pharmacogenet Genomics 2008;18:339-48.\\n12\\n. \\nKuypers DR et al. CYP3A5 and CYP3A4 but not MDR1 single-nucleotide polymorphisms determine long-term tacrolimus\\ndisposition and drug-related nephrotoxicity in renal recipients. Clin Pharmacol Ther 2007;82:711-25.\\n13\\n. \\nRenders L et al. CYP3A5 genotype markedly influences the pharmacokinetics of tacrolimus and sirolimus in kidney transplant\\nrecipients. Clin Pharmacol Ther 2007;81:228-34.\\n14\\n. \\nMourad M et al. The influence of genetic polymorphisms of cytochrome P450 3A5 and ABCB1 on starting dose- and weight-\\nstandardized tacrolimus trough concentrations after kidney transplantation in relation to renal function. Clin Chem Lab Med\\n2006;44:1192-8.\\n15\\n. \\nRoy JN et al. Cyp3A4, Cyp3A5, and MDR-1 genetic influences on tacrolimus pharmacokinetics in renal transplant recipients.\\nPharmacogenet Genomics 2006;16:659-65.\\n16\\n. \\nCheung CY et al. Influence of different allelic variants of the CYP3A and ABCB1 genes on the tacrolimus pharmacokinetic\\nprofile of Chinese renal transplant recipients. Pharmacogenomics 2006;7:563-74.\\n17\\n. \\nUesugi M et al. Effect of intestinal CYP3A5 on postoperative tacrolimus trough levels in living-donor liver transplant recipients.\\nPharmacogenet Genomics 2006;16:119-27.\\n18\\n. \\nZhang X et al. Influence of CYP3A5 and MDR1 polymorphisms on tacrolimus concentration in the early stage after renal\\ntransplantation. Clin Transplant 2005;19:638-43.\\n19\\n. \\nTada H et al. Impact of CYP3A5 and MDR1(ABCB1) C3435T polymorphisms on the pharmacokinetics of tacrolimus in renal\\ntransplant recipients. Transplant Proc 2005;37:1730-2.\\n20\\n. \\nMacphee IA et al. Tacrolimus pharmacogenetics: the CYP3A5*1 allele predicts low dose-normalized tacrolimus blood\\nconcentrations in whites and South Asians. Transplantation 2005;79:499-502.\\n21\\n. \\nHesselink DA et al. Genetic polymorphisms of the CYP3A4, CYP3A5, and MDR-1 genes and pharmacokinetics of the calcineurin\\ninhibitors cyclosporine and tacrolimus. Clin Pharmacol Ther 2003;74:245-54.\\nDate 09-11-2015\\nCYP3A5 homozygote expressor: tacrolimus\\n2357\\nGenetic variation results in an increased conversion of tacrolimus to inactive metabolites and as a result a higher dose is required.\\nAdjustment of the initial dose results in an increased chance of reaching a tacrolimus concentration within the target range before the\\nstart of therapeutic drug monitoring on day three. However, there is no direct evidence that this results in improved clinical results.\\nRecommendation:\\nIndications OTHER than liver transplantation:\\n1\\n. \\nStart with 2.5 times the standard initial dose that would yield the desired result in non-expressors\\nAdjustment of the dose should then be based on therapeutic drug monitoring.\\nNOTE: The initial dose that yields the desired result in non-expressors can be lower than the standard initial dose. In the\\nexample provided below, this dose was 75 % of the standard initial dose. A 2.5 time dose increase corresponds in this case to\\na 2 time dose increase of the standard initial dose.\\nFor example: After three days, Thervet et al. found a median trough concentration for tacrolimus of 14.0 ng/mL at an initial\\ndose of 0.15 mg/kg twice daily for four kidney transplant patients, who were homozygous expressors. This was 5.6 ng/mL\\n(n = 6) for an initial dose of 0.1 mg/kg twice daily. Their target value was 10 - 15 ng/mL. This is lower than the target value\\nthat is used in the Netherlands in the first two - four weeks after kidney transplantation (15 - 20 ng/mL).\\nFor the reference group of non-expressors, a median trough concentration of 16.6 ng/mL and 12.0 ng/mL was achieved at a\\ndosage of 0.1 mg/kg twice daily and 0.075 mg/kg twice daily respectively. In this hospital, the first dose is administered\\nbefore the CYP3A5 genotype is known. The second dose is reduced according to the genotype.\\nLIVER transplantation:\\nIn addition to the patient’s genotype, the metabolism of tacrolimus is also determined by the genotype of the transplanted liver.\\nLIVER is also of the genotype HOMOZYGOUS EXPRESSOR:\\n1\\n. \\nStart with 2.5 times the standard initial dose Adjustment of the dose should then be based on therapeutic drug\\nmonitoring.\\nLIVER has a DIFFERENT genotype:\\nThere is insufficient evidence in the literature to support a dose recommendation.\\nLiterature:\\n1\\n. \\nRojas L et al. Effect of CYP3A5*3 on kidney transplant recipients treated with tacrolimus: a systematic review and meta-analysis\\nof observational studies. Pharmacogenomics J 2015;15:38-48.\\n2\\n. \\nBuendia JA et al. Effects of combinational CYP3A5 6986A>G polymorphism in graft liver and native intestine on the\\npharmacokinetics of tacrolimus in liver transplant patients: a meta-analysis. Ther Drug Monit 2014;36:442-7.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 125}),\n",
       " Document(page_content='3\\n. \\nUesugi M et al. Impact of cytochrome P450 3A5 polymorphism in graft livers on the frequency of acute cellular rejection in\\nliving-donor liver transplantation. Pharmacogenet Genomics 2014;24:356-66.\\n4\\n. \\nTerrazzino S et al. The effect of CYP3A5 6986A>G and ABCB1 3435C>T on tacrolimus dose-adjusted trough levels and acute\\nrejection rates in renal transplant patients: a systematic review and meta-analysis. Pharmacogenet Genomics 2012;22:642-5.\\n5\\n. \\nTang HL et al. Lower tacrolimus daily dose requirements and acute rejection rates in the CYP3A5 non-expressers than expressers.\\nPharmacogenet Genomics 2011;21:713-20.\\n6\\n. \\nJacobson PA et al. Novel polymorphisms associated with tacrolimus trough concentrations: results from a multicenter kidney\\ntransplant consortium. Transplantation 2011;91:300-8.\\n7\\n. \\nThervet E et al. Optimization of initial tacrolimus dose using pharmacogenetic testing. Clin Pharmacol Ther 2010; 87: 721-726.\\n8\\n. \\nSatoh S et al. Lack of tacrolimus circadian pharmacokinetics and CYP3A5 pharmacogenetics in the early and maintenance stages\\nin Japanese renal transplant recipients. Br J Clin Pharmacol 2008;66:207-14.\\n9\\n. \\nKlauke B et al. No association between single nucleotide polymorphisms and the development of nephrotoxicity after orthotopic\\nheart transplantation. J Heart Lung Transplant 2008;27:741-5.\\n10\\n. \\nFukudo M et al. Impact of MDR1 and CYP3A5 on the oral clearance of tacrolimus and tacrolimus-related renal dysfunction in\\nadult living-donor liver transplant patients. Pharmacogenet Genomics 2008;18:413-23.\\n11\\n. \\nHesselink DA et al. CYP3A5 genotype is not associated with a higher risk of acute rejection in tacrolimus-treated renal transplant\\nrecipients. Pharmacogenet Genomics 2008;18:339-48.\\n12\\n. \\nKuypers DR et al. CYP3A5 and CYP3A4 but not MDR1 single-nucleotide polymorphisms determine long-term tacrolimus\\ndisposition and drug-related nephrotoxicity in renal recipients. Clin Pharmacol Ther 2007;82:711-25.\\n13\\n. \\nRenders L et al. CYP3A5 genotype markedly influences the pharmacokinetics of tacrolimus and sirolimus in kidney transplant\\nrecipients. Clin Pharmacol Ther 2007;81:228-34.\\n14\\n. \\nMourad M et al. The influence of genetic polymorphisms of cytochrome P450 3A5 and ABCB1 on starting dose- and weight-\\nstandardized tacrolimus trough concentrations after kidney transplantation in relation to renal function. Clin Chem Lab Med\\n2006;44:1192-8.\\n15\\n. \\nRoy JN et al. Cyp3A4, Cyp3A5, and MDR-1 genetic influences on tacrolimus pharmacokinetics in renal transplant recipients.\\nPharmacogenet Genomics 2006;16:659-65.\\n16\\n. \\nCheung CY et al. Influence of different allelic variants of the CYP3A and ABCB1 genes on the tacrolimus pharmacokinetic\\nprofile of Chinese renal transplant recipients. Pharmacogenomics 2006;7:563-74.\\n17\\n. \\nUesugi M et al. Effect of intestinal CYP3A5 on postoperative tacrolimus trough levels in living-donor liver transplant recipients.\\nPharmacogenet Genomics 2006;16:119-27.\\n18\\n. \\nZhang X et al. Influence of CYP3A5 and MDR1 polymorphisms on tacrolimus concentration in the early stage after renal\\ntransplantation. Clin Transplant 2005;19:638-43.\\n19\\n. \\nTada H et al. Impact of CYP3A5 and MDR1(ABCB1) C3435T polymorphisms on the pharmacokinetics of tacrolimus in renal\\ntransplant recipients. Transplant Proc 2005;37:1730-2.\\n20\\n. \\nMacphee IA et al. Tacrolimus pharmacogenetics: the CYP3A5*1 allele predicts low dose-normalized tacrolimus blood\\nconcentrations in whites and South Asians. Transplantation 2005;79:499-502.\\n21\\n. \\nHesselink DA et al. Genetic polymorphisms of the CYP3A4, CYP3A5, and MDR-1 genes and pharmacokinetics of the calcineurin\\ninhibitors cyclosporine and tacrolimus. Clin Pharmacol Ther 2003;74:245-54.\\nDate 09-11-2015\\nCYP2D6 IM: tamoxifen\\n1602\\nThis gene variation reduces the conversion of tamoxifen to the active metabolite endoxifen. This can result in reduced effectiveness.\\nRecommendation:\\n1\\n. \\nselect an alternative or measure the endoxifen concentration and increase the dose if necessary by a factor of 1.5-2\\nAromatase inhibitors are a possible alternative for post-menopausal women.\\n2\\n. \\nif TAMOXIFEN is selected: avoid co-medication with CYP2D6 inhibitors such as paroxetine and fluoxetine\\nLiterature:\\n1\\n. \\nWelzen ME et al. The effect of tamoxifen dose increment in patients with impaired CYP2D6 activity. Ther Drug Monit\\n2015;37:501-7.\\n2\\n. \\nMartinez de Dueñas E et al. Adjusting the dose of tamoxifen in patients with early breast cancer and CYP2D6 poor metabolizer\\nphenotype. Breast 2014;23:400-6.\\n3\\n. \\nProvince MA et al. CYP2D6 genotype and adjuvant tamoxifen: meta-analysis of heterogeneous study populations. Clin Pharmacol\\nTher 2014;95:216-27.\\n4\\n. \\nJung JA et al. Association between CYP2D6 genotypes and the clinical outcomes of adjuvant tamoxifen for breast cancer: a meta-\\nanalysis. Pharmacogenomics2014;15:49-60.\\n5\\n. \\nCronin-Fenton DP et al. Metabolism and transport of tamoxifen in relation to its effectiveness: new perspectives on an ongoing', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 126}),\n",
       " Document(page_content='controversy. Future Oncol 2014;10:107-22.\\n6\\n. \\nLum DW et al. CYP2D6 genotype and tamoxifen response for breast cancer: a systematic review and meta-analysis. PLoS One\\n2013;8:e76648.\\n7\\n. \\nZeng Z et al. CYP2D6 polymorphisms influence tamoxifen treatment outcomes in breast cancer patients: a meta-analysis. Cancer\\nChemother Pharmacol 2013;72:287-303.\\n8\\n. \\nRegan MM et al. CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer:\\nthe breast international group 1-98 trial. J Natl Cancer Inst 2012;104:441-51.\\n9\\n. \\nRae JM et al. CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen-treated breast cancer patients. J Natl Cancer\\nInst 2012;104:452-60.\\n10\\n. \\nPark IH et al. Lack of any association between functionally significant CYP2D6 polymorphisms and clinical outcomes in early\\nbreast cancer patients receiving adjuvant tamoxifen treatment. Breast Cancer Res Treat 2012;131:455-61.\\n11\\n. \\nKiyotani K et al. Dose-adjustment study of tamoxifen based on CYP2D6 genotypes in Japanese breast cancer patients. Breast\\nCancer Res Treat 2012;131:137-45.\\n12\\n. \\nBarginear MF et al. Increasing tamoxifen dose in breast cancer patients based on CYP2D6 genotypes and endoxifen levels: effect\\non active metabolite isomers and the antiestrogenic activity score. Clin Pharmacol Ther 2011;90:605-11.\\n13\\n. \\nIrvin WJ Jr et al. Genotype-guided tamoxifen dosing increases active metabolite exposure in women with reduced CYP2D6\\nmetabolism: a multicenter study. J Clin Oncol 2011;29:3232-9.\\n14\\n. \\nLash TL et al. CYP2D6 inhibition and breast cancer recurrence in a population-based study in Denmark. J Natl Cancer Inst\\n2011;103:489-500.\\n15\\n. \\nThompson AM et al. Comprehensive CYP2D6 genotype and adherence affect outcome in breast cancer patients treated with\\ntamoxifen monotherapy. Breast Cancer Res Treat 2011;125:279-87.\\n16\\n. \\nStingl JC et al. Impact of CYP2D6 *4 genotype on progression free survival in tamoxifen breast cancer treatment. Curr Med Res\\nOpin 2010;26:2535-42.\\n17\\n. \\nLammers LA et al. The impact of CYP2D6-predicted phenotype on tamoxifen treatment outcome in patients with metastatic breast\\ncancer. Br J Cancer 2010;103:765-71.\\n18\\n. \\nSchroth W et al. CYP2D6 polymorphisms as predictors of outcome in breast cancer patients treated with tamoxifen: expanded\\npolymorphism coverage improves risk stratification. Clin Cancer Res 2010;16:4468-77.\\n19\\n. \\nSeruga B et al. Cytochrome P450 2D6 and outcomes of adjuvant tamoxifen therapy: results of a meta-analysis. Breast Cancer Res\\nTreat 2010;122:609-17.\\n20\\n. \\nKiyotani K et al. Lessons for pharmacogenomics studies: association study between CYP2D6 genotype and tamoxifen response.\\nPharmacogenet Genomics 2010;20:565-8.\\n21\\n. \\nAbraham JE et al. CYP2D6 gene variants: association with breast cancer specific survival in a cohort of breast cancer patients\\nfrom the United Kingdom treated with adjuvant tamoxifen. Breast Cancer Res 2010;12:R64.\\n22\\n. \\nSchroth W et al. Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated\\nwith tamoxifen. JAMA 2009;302:1429-36.\\n23\\n. \\nGjerde J et al. Effects of CYP2D6 and SULT1A1 genotypes including SULT1A1 gene copy number on tamoxifen metabolism.\\nAnn Oncol 2008;19:56-61.\\n24\\n. \\nSchroth W et al. Breast cancer treatment outcome with adjuvant tamoxifen relative to patient CYP2D6 and CYP2C19 genotypes. J\\nClin Oncol 2007;25:5187-93.\\n25\\n. \\nLim HS et al. Clinical implications of CYP2D6 genotypes predictive of tamoxifen pharmacokinetics in metastatic breast cancer. J\\nClin Oncol 2007;25:3837-45.\\n26\\n. \\nGonzalez-Santiago S et al. CYP2D6*4 polymorphism as blood predictive biomarker of breast cancer relapse in patients receiving\\nadjuvant tamoxifen. J Clin Oncol 2007;25(18S):590.\\n27\\n. \\nWegman P et al. Genetic variants of CYP3A5, CYP2D6, SULT1A1, UGT2B15 and tamoxifen response in postmenopausal\\npatients with breast cancer. Breast Cancer Res 2007;9:R7.\\n28\\n. \\nGoetz MP et al. The impact of cytochrome P450 2D6 metabolism in women receiving adjuvant tamoxifen. Breast Cancer Res\\nTreat 2007;101:113-21.\\n29\\n. \\nBorges S et al. Quantitative effect of CYP2D6 genotype and inhibitors on tamoxifen metabolism: implication for optimization of\\nbreast cancer treatment. Clin Pharmacol Ther 2006;80:61-74.\\n30\\n. \\nNowell SA et al. Association of genetic variation in tamoxifen-metabolizing enzymes with overall survival and recurrence of\\ndisease in breast cancer patients. Breast Cancer Res Treat 2005;91:249-58.\\n31\\n. \\nJin Y et al. CYP2D6 genotype, antidepressant use, and tamoxifen metabolism during adjuvant breast cancer treatment. J Natl\\nCancer Inst 2005;97:30-9.\\n32\\n. \\nWegman P et al. Genotype of metabolic enzymes and the benefit of tamoxifen in postmenopausal breast cancer patients. Breast\\nCancer Res 2005;7:R284-90.\\nDate 09-11-2015\\nCYP2D6 PM: tamoxifen\\n1601\\nThis gene variation reduces the conversion of tamoxifen to the active metabolite endoxifen. This can result in reduced effectiveness.\\nRecommendation:\\n1\\n. \\nselect an alternative or increase the dose to 40 mg/day and monitor the endoxifen concentration\\nStudies have demonstrated that PM can achieve an adequate endoxifen concentration when the dose is increased to 40-60 mg/day.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 127}),\n",
       " Document(page_content='Aromatase inhibitors are a possible alternative for post-menopausal women.\\nLiterature:\\n1\\n. \\nWelzen ME et al. The effect of tamoxifen dose increment in patients with impaired CYP2D6 activity. Ther Drug Monit\\n2015;37:501-7.\\n2\\n. \\nMartinez de Dueñas E et al. Adjusting the dose of tamoxifen in patients with early breast cancer and CYP2D6 poor metabolizer\\nphenotype. Breast 2014;23:400-6.\\n3\\n. \\nProvince MA et al. CYP2D6 genotype and adjuvant tamoxifen: meta-analysis of heterogeneous study populations. Clin Pharmacol\\nTher 2014;95:216-27.\\n4\\n. \\nJung JA et al. Association between CYP2D6 genotypes and the clinical outcomes of adjuvant tamoxifen for breast cancer: a meta-\\nanalysis. Pharmacogenomics2014;15:49-60.\\n5\\n. \\nCronin-Fenton DP et al. Metabolism and transport of tamoxifen in relation to its effectiveness: new perspectives on an ongoing\\ncontroversy. Future Oncol 2014;10:107-22.\\n6\\n. \\nLum DW et al. CYP2D6 genotype and tamoxifen response for breast cancer: a systematic review and meta-analysis. PLoS One\\n2013;8:e76648.\\n7\\n. \\nZeng Z et al. CYP2D6 polymorphisms influence tamoxifen treatment outcomes in breast cancer patients: a meta-analysis. Cancer\\nChemother Pharmacol 2013;72:287-303.\\n8\\n. \\nRegan MM et al. CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer:\\nthe breast international group 1-98 trial. J Natl Cancer Inst 2012;104:441-51.\\n9\\n. \\nRae JM et al. CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen-treated breast cancer patients. J Natl Cancer\\nInst 2012;104:452-60.\\n10\\n. \\nBarginear MF et al. Increasing tamoxifen dose in breast cancer patients based on CYP2D6 genotypes and endoxifen levels: effect\\non active metabolite isomers and the antiestrogenic activity score. Clin Pharmacol Ther 2011;90:605-11.\\n11\\n. \\nIrvin WJ Jr et al. Genotype-guided tamoxifen dosing increases active metabolite exposure in women with reduced CYP2D6\\nmetabolism: a multicenter study. J Clin Oncol 2011;29:3232-9.\\n12\\n. \\nLash TL et al. CYP2D6 inhibition and breast cancer recurrence in a population-based study in Denmark. J Natl Cancer Inst\\n2011;103:489-500.\\n13\\n. \\nThompson AM et al. Comprehensive CYP2D6 genotype and adherence affect outcome in breast cancer patients treated with\\ntamoxifen monotherapy. Breast Cancer Res Treat 2011;125:279-87.\\n14\\n. \\nStingl JC et al. Impact of CYP2D6 *4 genotype on progression free survival in tamoxifen breast cancer treatment. Curr Med Res\\nOpin 2010;26:2535-42.\\n15\\n. \\nLammers LA et al. The impact of CYP2D6-predicted phenotype on tamoxifen treatment outcome in patients with metastatic breast\\ncancer. Br J Cancer 2010;103:765-71.\\n16\\n. \\nSchroth W et al. CYP2D6 polymorphisms as predictors of outcome in breast cancer patients treated with tamoxifen: expanded\\npolymorphism coverage improves risk stratification. Clin Cancer Res 2010;16:4468-77.\\n17\\n. \\nSeruga B et al. Cytochrome P450 2D6 and outcomes of adjuvant tamoxifen therapy: results of a meta-analysis. Breast Cancer Res\\nTreat 2010;122:609-17.\\n18\\n. \\nKiyotani K et al. Lessons for pharmacogenomics studies: association study between CYP2D6 genotype and tamoxifen response.\\nPharmacogenet Genomics 2010;20:565-8.\\n19\\n. \\nAbraham JE et al. CYP2D6 gene variants: association with breast cancer specific survival in a cohort of breast cancer patients\\nfrom the United Kingdom treated with adjuvant tamoxifen. Breast Cancer Res 2010;12:R64.\\n20\\n. \\nSchroth W et al. Association between CYP2D6 polymorphisms and outcomes among women with early stage breast cancer treated\\nwith tamoxifen. JAMA 2009;302:1429-36.\\n21\\n. \\nGjerde J et al. Effects of CYP2D6 and SULT1A1 genotypes including SULT1A1 gene copy number on tamoxifen metabolism.\\nAnn Oncol 2008;19:56-61.\\n22\\n. \\nSchroth W et al. Breast cancer treatment outcome with adjuvant tamoxifen relative to patient CYP2D6 and CYP2C19 genotypes. J\\nClin Oncol 2007;25:5187-93.\\n23\\n. \\nGonzalez-Santiago S et al. CYP2D6*4 polymorphism as blood predictive biomarker of breast cancer relapse in patients receiving\\nadjuvant tamoxifen. J Clin Oncol 2007;25(18S):590.\\n24\\n. \\nWegman P et al. Genetic variants of CYP3A5, CYP2D6, SULT1A1, UGT2B15 and tamoxifen response in postmenopausal\\npatients with breast cancer. Breast Cancer Res 2007;9:R7.\\n25\\n. \\nGoetz MP et al. The impact of cytochrome P450 2D6 metabolism in women receiving adjuvant tamoxifen. Breast Cancer Res\\nTreat 2007;101:113-21.\\n26\\n. \\nBonanni B ea. Polymorphism in the CYP2D6 tamoxifen-metabolizing gene influences clinical effect but not hot flashes: data from\\nthe Italian Tamoxifen Trial. J Clin Oncol 2006;24:3708-9.\\n27\\n. \\nBorges S et al. Quantitative effect of CYP2D6 genotype and inhibitors on tamoxifen metabolism: implication for optimization of\\nbreast cancer treatment. Clin Pharmacol Ther 2006;80:61-74.\\n28\\n. \\nGoetz MP et al. Pharmacogenetics of tamoxifen biotransformation is associated with clinical outcomes of efficacy and hot flashes.\\nJ Clin Oncol 2005;23:9312-8.\\n29\\n. \\nNowell SA et al. Association of genetic variation in tamoxifen-metabolizing enzymes with overall survival and recurrence of\\ndisease in breast cancer patients. Breast Cancer Res Treat 2005;91:249-58.\\n30\\n. \\nJin Y et al. CYP2D6 genotype, antidepressant use, and tamoxifen metabolism during adjuvant breast cancer treatment. J Natl\\nCancer Inst 2005;97:30-9.\\n31\\n. \\nWegman P et al. Genotype of metabolic enzymes and the benefit of tamoxifen in postmenopausal breast cancer patients. Breast', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 128}),\n",
       " Document(page_content='Cancer Res 2005;7:R284-90.\\n32\\n. \\nSPC Tamoxifen PCH.\\nDate 09-11-2015\\nCYP2D6 UM: tamoxifen\\n1603\\nNO action is needed for this gene-drug interaction.\\nAs a result of the genetic variation, the plasma concentration of the active metabolites 4-hydroxytamoxifen and endoxifen can increase.\\nHowever, there is no evidence that this results in an increase in the side effects.\\nLiterature:\\n1\\n. \\nMartinez de Dueñas E et al. Adjusting the dose of tamoxifen in patients with early breast cancer and CYP2D6 poor metabolizer\\nphenotype. Breast 2014;23:400-6.\\n2\\n. \\nSchroth W et al. CYP2D6 polymorphisms as predictors of outcome in breast cancer patients treated with tamoxifen: expanded\\npolymorphism coverage improves risk stratification. Clin Cancer Res 2010;16:4468-77.\\n3\\n. \\nGjerde J et al. Effects of CYP2D6 and SULT1A1 genotypes including SULT1A1 gene copy number on tamoxifen metabolism.\\nAnn Oncol 2008;19:56-61.\\n4\\n. \\nLim HS et al. Clinical implications of CYP2D6 genotypes predictive of tamoxifen pharmacokinetics in metastatic breast cancer. J\\nClin Oncol 2007;25:3837-45.\\nDate 09-11-2015\\nDPD genact 0,5: tegafur\\n4891\\nGenetic variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur to inactive metabolites means that\\nthe normal dose is an overdose.\\nRecommendation:\\nChoose an alternative or start with a low dose and adjust the initial dose based on toxicity and efficacy\\nFluorouracil and capecitabine are not alternatives, as these are also metabolised by DPD.\\nIt is not possible to offer substantiated advice for dose reduction based on the literature. For fluorouracil and capecitabine, starting\\nwith 25% of the standard dose is recommended.\\nNOTE: This recommendation only applies if the two gene variations are on a different allele. If both variations are on the same\\nallele, this patient has gene activity score 1 and the recommendation for that gene activity score should be followed. These two\\nsituations can only be distinguished by determining the enzyme activity (phenotyping).\\nLiterature:\\n1\\n. \\nCubero DI et al. Tegafur-uracil is a safe alternative for the treatment of colorectal cancer in patients with partial dihydropyrimidine\\ndehydrogenase deficiency: a proof of principle. Ther Adv Med Oncol 2012;4:167-72.\\n2\\n. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or\\ncapecitabine. Ann Intern Med 2010;153:767-8.\\n3\\n. \\nSPC Teysuno.\\nDate 20-11-2017\\nDPD genact 0: tegafur\\n2553\\nGenetic variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur to inactive metabolites means that', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 129}),\n",
       " Document(page_content='the normal dose is an overdose.\\nRecommendation:\\nChoose an alternative\\nDo not choose fluorouracil or capecitabine, as these are also metabolised by DPD.\\nIf an alternative is not possible: start with a very low dose and adjust the initial dose based on toxicity and efficacy.\\nA substantiated recommendation for dose reduction cannot be made based on the literature.\\nThe recommendation for fluorouracil and capecitabine is to determine the residual DPD activity in mononuclear cells from\\nperipheral blood and to adjust the initial dose accordingly. A patient with 0.5% of the normal DPD activity tolerated 0.8% of the\\nstandard capecitabine dose (150 mg every 5 days). A patient with undetectable DPD activity tolerated 0.43% of the standard\\ncapecitabine dose (150 mg every 5 days with every third dose skipped)\\nThe average Caucasian DPD activity is 9.9 nmol/hour per mg protein.\\nNOTE: If a patient has two different gene variations that lead to a non-functional DPD enzyme (e.g. *2A and *13), this recommendation\\nonly applies if the variations are on a different allele. If both variations are on the same allele, this patient has gene activity score 1 and\\nthe recommendation for that gene activity score should be followed. These two situations can only be distinguished by determining the\\nenzyme activity (phenotyping).\\nLiterature:\\n1\\n. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or\\ncapecitabine. Ann Intern Med 2010;153:767-8.\\n2\\n. \\nSPC Teysuno.\\nDate 20-11-2017\\nDPD genact 1,5: tegafur\\n4892\\nGenetic variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur into inactive metabolites means\\nthat the normal dose is an overdose.\\nRecommendation:\\n1\\n. \\nChoose an alternative or start with a low dose and adjust the initial dose based on toxicity and efficacy.\\nFluorouracil and capecitabine are not alternatives, as these are also metabolised by DPD.\\nIt is not possible to offer substantiated advice for dose reduction based on the literature. For fluorouracil and capecitabine, starting\\nwith 75 % of the normal dose is recommended.\\nLiterature:\\n1\\n. \\nCubero DI et al. Tegafur-uracil is a safe alternative for the treatment of colorectal cancer in patients with partial dihydropyrimidine\\ndehydrogenase deficiency: a proof of principle. Ther Adv Med Oncol 2012;4:167-72.\\n2\\n. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or\\ncapecitabine. Ann Intern Med 2010;153:767-8.\\n3\\n. \\nSPC Teysuno.\\nDate 20-11-2017\\nDPD genact 1: tegafur\\n2554\\nGenetic variation increases the risk of severe, possibly fatal toxicity. A reduced conversion of tegafur into inactive metabolites means\\nthat the normal dose is an overdose.\\nRecommendation:\\n1\\n. \\nChoose an alternative or start with a low dose and adjust the initial dose based on toxicity and efficacy', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 130}),\n",
       " Document(page_content='Fluorouracil and capecitabine are not alternatives, as these are also metabolised by DPD.\\nIt is not possible to offer substantiated advice for dose reduction based on the literature. For fluorouracil and capecitabine, starting\\nwith 50 % of the standard dose is recommended.\\nNOTE: If a patient has two different gene variations that result in a partially functional DPD enzyme (e.g. 2846T and 1236A), this\\nrecommendation only applies if the variations are on a different allele. If both variations are on the same allele, the gene activity\\nscore is between 1 and 1.5, depending on whether and how the two gene variations influence each other and on other factors that\\ninfluence the DPD activity. Whether a gene activity score of 1 or 1.5 needs to be assigned in the case of two different genetic\\nvariations can only be determined by measuring the enzyme activity (phenotyping).\\nLiterature:\\n1\\n. \\nCubero DI et al. Tegafur-uracil is a safe alternative for the treatment of colorectal cancer in patients with partial dihydropyrimidine\\ndehydrogenase deficiency: a proof of principle. Ther Adv Med Oncol 2012;4:167-72.\\n2\\n. \\nDeenen MJ et al. Standard-dose tegafur combined with uracil is not safe treatment after severe toxicity from 5-fluorouracil or\\ncapecitabine. Ann Intern Med 2010;153:767-8.\\n3\\n. \\nSPC Teysuno.\\nDate 20-11-2017\\nCYP2C19 IM: ticagrelor\\n3516\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nRath PC et al. A study on the impact of CYP2C19 genotype and platelet reactivity assay on patients undergoing PCI. Indian Heart\\nJ 2015;67:114-21.\\n2\\n. \\nSteg PG et al. Stent thrombosis with ticagrelor versus clopidogrel in patients with acute coronary syndromes: an analysis from the\\nprospective, randomized PLATO trial. Circulation 2013;128:1055-65.\\n3\\n. \\nTantry US et al. First analysis of the relation between CYP2C19 genotype and pharmacodynamics in patients treated with\\nticagrelor versus clopidogrel: the ONSET/OFFSET and RESPOND genotype studies. Circ Cardiovasc Genet 2010;3:556-66.\\n4\\n. \\nWallentin L et al. Effect of CYP2C19 and ABCB1 single nucleotide polymorphisms on outcomes of treatment with ticagrelor\\nversus clopidogrel for acute coronary syndromes: a genetic substudy of the PLATO trial. Lancet 2010;376:1320-8.\\n5\\n. \\nSPC’s Brilique (NL) en Brilinta (VS).\\nDate 19-11-2018\\nCYP2C19 PM: ticagrelor\\n3515\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nZhong Z et al. Effect of cytochrome P450 2C19 polymorphism on adverse cardiovascular events after drug-eluting stent\\nimplantation in a large Hakka population with acute coronary syndrome receiving clopidogrel in southern China. Eur J Clin\\nPharmacol 2018;74:423-31.\\n2\\n. \\nShen DL et al. Clinical value of CYP2C19 genetic testing for guiding the antiplatelet therapy in a Chinese population. J\\nCardiovasc Pharmacol 2016;67:232-6.\\n3\\n. \\nXiong R et al. A randomized controlled trial to assess the efficacy and safety of doubling dose clopidogrel versus ticagrelor for the\\ntreatment of acute coronary syndrome in patients with CYP2C19*2 homozygotes. Int J Clin Exp Med 2015;8:13310-6.\\n4\\n. \\nRath PC et al. A study on the impact of CYP2C19 genotype and platelet reactivity assay on patients undergoing PCI. Indian Heart\\nJ 2015;67:114-21.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 131}),\n",
       " Document(page_content=\"5\\n. \\nSteg PG et al. Stent thrombosis with ticagrelor versus clopidogrel in patients with acute coronary syndromes: an analysis from the\\nprospective, randomized PLATO trial. Circulation 2013;128:1055-65.\\n6\\n. \\nTantry US et al. First analysis of the relation between CYP2C19 genotype and pharmacodynamics in patients treated with\\nticagrelor versus clopidogrel: the ONSET/OFFSET and RESPOND genotype studies. Circ Cardiovasc Genet 2010;3:556-66.\\n7\\n. \\nWallentin L et al. Effect of CYP2C19 and ABCB1 single nucleotide polymorphisms on outcomes of treatment with ticagrelor\\nversus clopidogrel for acute coronary syndromes: a genetic substudy of the PLATO trial. Lancet 2010;376:1320-8.\\n8\\n. \\nSPC’s Brilique (NL) en Brilinta (VS).\\nDate 19-11-2018\\nCYP2C19 UM: ticagrelor\\n3517\\nThis is NOT a gene-drug interaction.\\nLiterature:\\n1\\n. \\nRath PC et al. A study on the impact of CYP2C19 genotype and platelet reactivity assay on patients undergoing PCI. Indian Heart\\nJ 2015;67:114-21.\\n2\\n. \\nSteg PG et al. Stent thrombosis with ticagrelor versus clopidogrel in patients with acute coronary syndromes: an analysis from the\\nprospective, randomized PLATO trial. Circulation 2013;128:1055-65.\\n3\\n. \\nTantry US et al. First analysis of the relation between CYP2C19 genotype and pharmacodynamics in patients treated with\\nticagrelor versus clopidogrel: the ONSET/OFFSET and RESPOND genotype studies. Circ Cardiovasc Genet 2010;3:556-66.\\nDate 19-11-2018\\nTPMT IM: tioguanine\\n1907\\nGenetic variation reduces conversion of thioguanine to inactive metabolites. This increases the risk of serious adverse events such as\\nmyelosuppression.\\nRecommendation:\\n1\\n. \\nStart with 75% of the standard dose\\nAdjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and efficacy.\\nThe frequency of monitoring should be increased.\\nLiterature:\\n1\\n. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of\\nthiopurine methyltransferase pharmacogenetics. Br J Haematol 2014 Nov 29 [Epub ahead of print].\\n2\\n. \\nWray L et al. TPMT and MTHFR genotype is not associated with altered risk of thioguanine-related sinusoidal obstruction\\nsyndrome in pediatric acute lymphoblastic leukemia: a report from the Children's Oncology Group. Pediatr Blood Cancer\\n2014;61:2086-8.\\n3\\n. \\nLennard L et al. The thiopurine methyltransferase genetic polymorphism is associated with thioguanine-related veno-occlusive\\ndisease of the liver in children with acute lymphoblastic leukemia. Clin Pharmacol Ther 2006;80:375-83.\\n4\\n. \\nStanden GR et al. Heterozygosity for the thiopurine methyltransferase *3A allele in an acute non-lymphoblastic leukaemia patient\\nwith delayed marrow regeneration following H-DAT chemotherapy. Br J Haematol 2001;112:1089.\\n5\\n. \\nTeml A et al. A prospective, open-label trial of 6-thioguanine in patients with ulcerative or indeterminate colitis. Scand J\\nGastroenterol 2005;40:1205-13.\\n6\\n. \\nHerrlinger KR et al. Thioguanine-nucleotides do not predict efficacy of tioguanine in Crohn's disease. Aliment Pharmacol Ther\\n2004;19:1269-76.\\nDate 27-05-2015\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 132}),\n",
       " Document(page_content=\"TPMT PM: tioguanine\\n1908\\nGenetic variation reduces conversion of thioguanine to inactive metabolites. This increases the risk of serious, life-threatening adverse\\nevents such as myelosuppression.\\nRecommendation:\\n1\\n. \\nChoose an alternative or start with 6-7% of the standard dose\\nAny adjustment of the initial dose should be guided by toxicity (monitoring of blood counts) and effectiveness.\\nThe frequency of monitoring should be increased.\\n2\\n. \\nIf the dose is decreased: advise patients to seek medical attention when symptoms of myelosuppression (such as severe sore  throat\\nin combination with fever, regular nosebleeds and tendency to bruising) develop\\nLiterature:\\n1\\n. \\nLennard L et al. Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of\\nthiopurine methyltransferase pharmacogenetics. Br J Haematol 2014 Nov 29 [Epub ahead of print].\\n2\\n. \\nWray L et al. TPMT and MTHFR genotype is not associated with altered risk of thioguanine-related sinusoidal obstruction\\nsyndrome in pediatric acute lymphoblastic leukemia: a report from the Children's Oncology Group. Pediatr Blood Cancer\\n2014;61:2086-8.\\n3\\n. \\nMares WG et al. Safe 6-thioguanine therapy of a TPMT deficient Crohn's disease patient by using therapeutic drug monitoring. J\\nCrohns Colitis 2009;3:128-30.\\n4\\n. \\nMcBride KL et al. Severe 6-thioguanine-induced marrow aplasia in a child with acute lymphoblastic leukemia and inherited\\nthiopurine methyltransferase deficiency. J Pediatr Hematol Oncol 2000;22:441-5.\\n5\\n. \\nSPC Lanvis.\\nDate 27-05-2015\\nCYP2C9 IM: tolbutamide\\n1903\\nNO action is required for this gene-drug interaction.\\nThere is insufficient evidence to state that the increased tolbutamide plasma concentration has any clinical consequences.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy\\nvolunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4\\n. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm\\nTher 2005;30:241-9.\\n5\\n. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n6\\n. \\nLee CR et al. Tolbutamide, flurbiprofen, and losartan as probes of CYP2C9 activity in humans. J Clin Pharmacol 2003;43:84-91.\\n7\\n. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol\\nTher 2002;72:562-71.\\n8\\n. \\nShon JH et al. Effects of CYP2C19 and CYP2C9 genetic polymorphisms on the disposition of and blood glucose lowering\\nresponse to tolbutamide in humans. Pharmacogenetics 2002;12:111-9.\\n9\\n. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose\\nresponse in healthy volunteers. Pharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\", metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 133}),\n",
       " Document(page_content='CYP2C9 PM: tolbutamide\\n1904\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy\\nvolunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4\\n. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm\\nTher 2005;30:241-9.\\n5\\n. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n6\\n. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol\\nTher 2002;72:562-71.\\n7\\n. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose\\nresponse in healthy volunteers. Pharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*1/*2: tolbutamide\\n1898\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy\\nvolunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4\\n. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n5\\n. \\nLee CR et al. Tolbutamide, flurbiprofen, and losartan as probes of CYP2C9 activity in humans. J Clin Pharmacol 2003;43:84-91.\\n6\\n. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol\\nTher 2002;72:562-71.\\n7\\n. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose\\nresponse in healthy volunteers. Pharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*1/*3: tolbutamide\\n1899\\nNO action is required for this gene-drug interaction.\\nThere is insufficient evidence to state that the increased tolbutamide plasma concentration has any clinical consequences.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 134}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy\\nvolunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4\\n. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm\\nTher 2005;30:241-9.\\n5\\n. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n6\\n. \\nLee CR et al. Tolbutamide, flurbiprofen, and losartan as probes of CYP2C9 activity in humans. J Clin Pharmacol 2003;43:84-91.\\n7\\n. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol\\nTher 2002;72:562-71.\\n8\\n. \\nShon JH et al. Effects of CYP2C19 and CYP2C9 genetic polymorphisms on the disposition of and blood glucose lowering\\nresponse to tolbutamide in humans. Pharmacogenetics 2002;12:111-9.\\n9\\n. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose\\nresponse in healthy volunteers. Pharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*2/*2: tolbutamide\\n1900\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n3\\n. \\nJetter A et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol 2004;60:165-71.\\n4\\n. \\nLee CR et al Evaluation of cytochrome P4502C9 metabolic activity with tolbutamide in CYP2C91 heterozygotes. Clin Pharmacol\\nTher 2002;72:562-71.\\n5\\n. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose\\nresponse in healthy volunteers. Pharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*2/*3: tolbutamide\\n1901\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy\\nvolunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 135}),\n",
       " Document(page_content='mellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4\\n. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose\\nresponse in healthy volunteers. Pharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2C9*3/*3: tolbutamide\\n1902\\nNO action is required for this gene-drug interaction.\\nNo clinical consequences of the increased tolbutamide plasma concentration were observed.\\nLiterature:\\n1\\n. \\nSwen JJ et al. Effect of CYP2C9 polymorphisms on prescribed dose and time-to-stable dose of sulfonylureas in primary care\\npatients with Type 2 diabetes mellitus. Pharmacogenomics 2010;11:1517-23.\\n2\\n. \\nVormfelde SV et al. Relative impact of genotype and enzyme induction on the metabolic capacity of CYP2C9 in healthy\\nvolunteers. Clin Pharmacol Ther 2009;86:54-61.\\n3\\n. \\nBecker ML et al. Cytochrome P450 2C9 \\n2 and \\n3 polymorphisms and the dose and effect of sulfonylurea in type II diabetes\\nmellitus. Clin Pharmacol Ther 2008;83:288-92.\\n4\\n. \\nChen K et al. Relationship of P450 2C9 genetic polymorphisms in Chinese and the pharmacokinetics of tolbutamide. J Clin Pharm\\nTher 2005;30:241-9.\\n5\\n. \\nKirchheiner J et al. Impact of CYP2C9 and CYP2C19 polymorphisms on tolbutamide kinetics and the insulin and glucose\\nresponse in healthy volunteers. Pharmacogenetics 2002;12:101-9.\\nDate 20-11-2017\\nCYP2D6 IM: tramadol\\n1590\\nThe genetic variation reduces the conversion of tramadol to a metabolite with a higher activity. This can result in reduced analgesia.\\nRecommendation:\\nIt is not possible to provide a recommendation for dose adjustment, because the total analgesic effect changes when the ratio between the\\nmother compound and the active metabolite changes.\\n1\\n. \\nbe alert to a reduced effectiveness\\n2\\n. \\nin the case of inadequate effectiveness:\\n1\\n. \\ntry a dose increase\\n2\\n. \\nif this does not work: choose an alternative\\nDo not select codeine, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n3\\n. \\nif no alternative is selected: advise the patient to report inadequate analgesia\\nLiterature:\\n1\\n. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2\\n. \\nDong H et al. Effect of the CYP2D6 gene polymorphism on postoperative analgesia of tramadol in Han nationality nephrectomy\\npatients. Eur J Clin Pharmacol 2015;71:681-6.\\n3\\n. \\nZhao Q et al. A logistic equation to determine the validity of tramadol from related gene polymorphisms and psychological\\nfactors. Pharmacogenomics 2014;15:487-95.\\n4\\n. \\nMatouskova O et al. Pupillometry in healthy volunteers as a biomarker of tramadol efficacy. J Clin Pharm Ther 2011;36:513-7.\\n5\\n. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain\\n2010;11:1274-81.\\n6\\n. \\nKim E et al. Adverse events in analgesic treatment with tramadol associated with CYP2D6 extensive-metaboliser and OPRM1\\nhigh-expression variants. Ann Rheum Dis 2010;69:1889-90.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 136}),\n",
       " Document(page_content='7\\n. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates\\nand infants. Pediatr Res 2008;63:674-9.\\n8\\n. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin\\nPharmacol Ther 2007;82:41-7.\\n9\\n. \\nSlanar O et al. Miotic action of tramadol is determined by CYP2D6 genotype. Physiol Res 2007;56:129-36.\\n10\\n. \\nWang G et al. Effect of the CYP2D6*10 C188T polymorphism on postoperative tramadol analgesia in a Chinese population. Eur J\\nClin Pharmacol 2006;62:927-31.\\n11\\n. \\nBorlak J et al. A rapid and simple CYP2D6 genotyping assay: case study with the analgetic tramadol. Metabolism 2003;52:1439-\\n43.\\n12\\n. \\nGan SH et al. Correlation of tramadol pharmacokinetics and CYP2D6*10 genotype in Malaysian subjects. J Pharm Biomed Anal\\n2002;30:189-195.\\n13\\n. \\nAbdel-Rahman SM et al. Concordance between tramadol and dextromethorphan parent/metabolite ratios: the influence of\\nCYP2D6 and non-CYP2D6 pathways on biotransformation. J Clin Pharmacol 2002;42:24-9.\\nDate 20-11-2017\\nCYP2D6 IM: tramadol\\n1590\\nThe genetic variation reduces the conversion of tramadol to a metabolite with a higher activity. This can result in reduced analgesia.\\nRecommendation:\\nIt is not possible to provide a recommendation for dose adjustment, because the total analgesic effect changes when the ratio between the\\nmother compound and the active metabolite changes.\\n1\\n. \\nbe alert to a reduced effectiveness\\n2\\n. \\nin the case of inadequate effectiveness:\\n1\\n. \\ntry a dose increase\\n2\\n. \\nif this does not work: choose an alternative\\nDo not select codeine, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n3\\n. \\nif no alternative is selected: advise the patient to report inadequate analgesia\\nLiterature:\\n1\\n. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2\\n. \\nDong H et al. Effect of the CYP2D6 gene polymorphism on postoperative analgesia of tramadol in Han nationality nephrectomy\\npatients. Eur J Clin Pharmacol 2015;71:681-6.\\n3\\n. \\nZhao Q et al. A logistic equation to determine the validity of tramadol from related gene polymorphisms and psychological\\nfactors. Pharmacogenomics 2014;15:487-95.\\n4\\n. \\nMatouskova O et al. Pupillometry in healthy volunteers as a biomarker of tramadol efficacy. J Clin Pharm Ther 2011;36:513-7.\\n5\\n. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain\\n2010;11:1274-81.\\n6\\n. \\nKim E et al. Adverse events in analgesic treatment with tramadol associated with CYP2D6 extensive-metaboliser and OPRM1\\nhigh-expression variants. Ann Rheum Dis 2010;69:1889-90.\\n7\\n. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates\\nand infants. Pediatr Res 2008;63:674-9.\\n8\\n. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin\\nPharmacol Ther 2007;82:41-7.\\n9\\n. \\nSlanar O et al. Miotic action of tramadol is determined by CYP2D6 genotype. Physiol Res 2007;56:129-36.\\n10\\n. \\nWang G et al. Effect of the CYP2D6*10 C188T polymorphism on postoperative tramadol analgesia in a Chinese population. Eur J\\nClin Pharmacol 2006;62:927-31.\\n11\\n. \\nBorlak J et al. A rapid and simple CYP2D6 genotyping assay: case study with the analgetic tramadol. Metabolism 2003;52:1439-\\n43.\\n12\\n. \\nGan SH et al. Correlation of tramadol pharmacokinetics and CYP2D6*10 genotype in Malaysian subjects. J Pharm Biomed Anal\\n2002;30:189-195.\\n13\\n. \\nAbdel-Rahman SM et al. Concordance between tramadol and dextromethorphan parent/metabolite ratios: the influence of\\nCYP2D6 and non-CYP2D6 pathways on biotransformation. J Clin Pharmacol 2002;42:24-9.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 137}),\n",
       " Document(page_content='Date 20-11-2017\\nCYP2D6 PM: tramadol\\n1589\\nThe genetic variation reduces the conversion of tramadol to a metabolite with a higher activity. This can result in reduced analgesia.\\nRecommendation:\\nIt is not possible to provide a recommendation for dose adjustment, because the total analgesic effect changes when the ratio between the\\nmother compound and the active metabolite changes.\\n1\\n. \\nbe alert to a reduced effectiveness\\n2\\n. \\nin the case of inadequate effectiveness:\\n1\\n. \\ntry a dose increase.\\n2\\n. \\nif this does not work: choose an alternative\\nDo not select codeine, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n3\\n. \\nif no alternative is selected: advise the patient to report inadequate analgesia\\nLiterature:\\n1\\n. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2\\n. \\nMatouskova O et al. Pupillometry in healthy volunteers as a biomarker of tramadol efficacy. J Clin Pharm Ther 2011;36:513-7.\\n3\\n. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain\\n2010;11:1274-81.\\n4\\n. \\nHalling J et al. CYP2D6 polymorphism in relation to tramadol metabolism: a study of faroese patients. Ther Drug Monit\\n2008;30:271-5.\\n5\\n. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates\\nand infants. Pediatr Res 2008;63:674-9.\\n6\\n. \\nKirchheiner J et al. Effects of the CYP2D6 gene duplication on the pharmacokinetics and pharmacodynamics of tramadol. J Clin\\nPsychopharmacol 2008;28:78-83.\\n7\\n. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin\\nPharmacol Ther 2007;82:41-7.\\n8\\n. \\nGarcía-Quetglas E et al. Pharmacokinetics of tramadol enantiomers and their respective phase I metabolites in relation to CYP2D6\\nphenotype. Pharmacol Res 2007;55:122-30.\\n9\\n. \\nSlanar O et al. Miotic action of tramadol is determined by CYP2D6 genotype. Physiol Res 2007;56:129-36.\\n10\\n. \\nPedersen RS et al. Enantioselective pharmacokinetics of tramadol in CYP2D6 extensive and poor metabolizers. Eur J Clin\\nPharmacol 2006;62:513-21.\\n11\\n. \\nEnggaard TP et al. The analgesic effect of tramadol after intravenous injection in healthy volunteers in relation to CYP2D6.\\nAnesth Analg 2006;102:146-50.\\n12\\n. \\nFliegert F et al. The effects of tramadol on static and dynamic pupillometry in healthy subjects--the relationship between\\npharmacodynamics, pharmacokinetics and CYP2D6 metaboliser status. Eur J Clin Pharmacol 2005;61:257-66.\\n13\\n. \\nBorlak J et al. A rapid and simple CYP2D6 genotyping assay: case study with the analgetic tramadol. Metabolism 2003;52:1439-\\n43.\\n14\\n. \\nStamer UM et al. Impact of CYP2D6 genotype on postoperative tramadol analgesia. Pain 2003;105:231-8.\\n15\\n. \\nAbdel-Rahman SM et al. Concordance between tramadol and dextromethorphan parent/metabolite ratios: the influence of\\nCYP2D6 and non-CYP2D6 pathways on biotransformation. J Clin Pharmacol 2002;42:24-9.\\n16\\n. \\nPaar WD et al. Polymorphic CYP2D6 mediates O-demethylation of the opioid analgesic tramadol. Eur J Clin Pharmacol\\n1997;53:235-9.\\n17\\n. \\nPoulsen L et al. The hypoalgesic effect of tramadol in relation to CYP2D6. Clin Pharmacol Ther 1996;60:636-44.\\n18\\n. \\nSPC Ultram (VS).\\nDate 20-11-2017\\nCYP2D6 PM: tramadol\\n1589\\nThe genetic variation reduces the conversion of tramadol to a metabolite with a higher activity. This can result in reduced analgesia.\\nRecommendation:\\nIt is not possible to provide a recommendation for dose adjustment, because the total analgesic effect changes when the ratio between the\\nmother compound and the active metabolite changes.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 138}),\n",
       " Document(page_content='1\\n. \\nbe alert to a reduced effectiveness\\n2\\n. \\nin the case of inadequate effectiveness:\\n1\\n. \\ntry a dose increase.\\n2\\n. \\nif this does not work: choose an alternative\\nDo not select codeine, as this is also metabolised by CYP2D6.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in analgesia in patients.\\n3\\n. \\nif no alternative is selected: advise the patient to report inadequate analgesia\\nLiterature:\\n1\\n. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2\\n. \\nMatouskova O et al. Pupillometry in healthy volunteers as a biomarker of tramadol efficacy. J Clin Pharm Ther 2011;36:513-7.\\n3\\n. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain\\n2010;11:1274-81.\\n4\\n. \\nHalling J et al. CYP2D6 polymorphism in relation to tramadol metabolism: a study of faroese patients. Ther Drug Monit\\n2008;30:271-5.\\n5\\n. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates\\nand infants. Pediatr Res 2008;63:674-9.\\n6\\n. \\nKirchheiner J et al. Effects of the CYP2D6 gene duplication on the pharmacokinetics and pharmacodynamics of tramadol. J Clin\\nPsychopharmacol 2008;28:78-83.\\n7\\n. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin\\nPharmacol Ther 2007;82:41-7.\\n8\\n. \\nGarcía-Quetglas E et al. Pharmacokinetics of tramadol enantiomers and their respective phase I metabolites in relation to CYP2D6\\nphenotype. Pharmacol Res 2007;55:122-30.\\n9\\n. \\nSlanar O et al. Miotic action of tramadol is determined by CYP2D6 genotype. Physiol Res 2007;56:129-36.\\n10\\n. \\nPedersen RS et al. Enantioselective pharmacokinetics of tramadol in CYP2D6 extensive and poor metabolizers. Eur J Clin\\nPharmacol 2006;62:513-21.\\n11\\n. \\nEnggaard TP et al. The analgesic effect of tramadol after intravenous injection in healthy volunteers in relation to CYP2D6.\\nAnesth Analg 2006;102:146-50.\\n12\\n. \\nFliegert F et al. The effects of tramadol on static and dynamic pupillometry in healthy subjects--the relationship between\\npharmacodynamics, pharmacokinetics and CYP2D6 metaboliser status. Eur J Clin Pharmacol 2005;61:257-66.\\n13\\n. \\nBorlak J et al. A rapid and simple CYP2D6 genotyping assay: case study with the analgetic tramadol. Metabolism 2003;52:1439-\\n43.\\n14\\n. \\nStamer UM et al. Impact of CYP2D6 genotype on postoperative tramadol analgesia. Pain 2003;105:231-8.\\n15\\n. \\nAbdel-Rahman SM et al. Concordance between tramadol and dextromethorphan parent/metabolite ratios: the influence of\\nCYP2D6 and non-CYP2D6 pathways on biotransformation. J Clin Pharmacol 2002;42:24-9.\\n16\\n. \\nPaar WD et al. Polymorphic CYP2D6 mediates O-demethylation of the opioid analgesic tramadol. Eur J Clin Pharmacol\\n1997;53:235-9.\\n17\\n. \\nPoulsen L et al. The hypoalgesic effect of tramadol in relation to CYP2D6. Clin Pharmacol Ther 1996;60:636-44.\\n18\\n. \\nSPC Ultram (VS).\\nDate 20-11-2017\\nCYP2D6 UM: tramadol\\n1591\\nThe genetic variation increases the conversion of tramadol to a metabolite with a stronger opioid effect. This can result in an increase in\\npotentially life-threatening side effects.\\nRecommendation:\\nAs the total analgesic effect changes when the ratio between the mother compound and the active metabolite changes, the effect of a dose\\nreduction cannot be predicted with certainty.\\nselect an alternative\\nDo not choose codeine, as it is contra-indicated for CYP2D6 UM.\\nMorphine is not metabolised by CYP2D6.\\nOxycodone is metabolised by CYP2D6 to a limited extent, but this does not result in differences in side effects in patients.\\nif an alternative is not possible:\\nuse 40% of the standard dose\\nadvise the patient to report side effects (such as drowsiness, confusion, constipation, nausea and vomiting, respiratory\\ndepression or urine retention).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 139}),\n",
       " Document(page_content='Literature:\\n1\\n. \\nSeripa D et al. Role of CYP2D6 polymorphisms in the outcome of postoperative pain treatment. Pain Med 2015;16:2012-23.\\n2\\n. \\nOrliaguet G et al. A case of respiratory depression in a child with ultrarapid CYP2D6 metabolism after tramadol. Pediatrics\\n2015;135:e753-5.\\n3\\n. \\nSaarikoski T et al. Effects of terbinafine and itraconazole on the pharmacokinetics of orally administered tramadol. Eur J Clin\\nPharmacol 2015;71:321-7.\\n4\\n. \\nRauers NI et al. Antagonistic effects of ondansetron and tramadol? A randomized placebo and active drug controlled study. J Pain\\n2010;11:1274-81.\\n5\\n. \\nStamer UM et al. Respiratory depression with tramadol in a patient with renal impairment and CYP2D6 gene duplication. Anesth\\nAnalg 2008;107:926-9.\\n6\\n. \\nAllegaert K et al. Postmenstrual age and CYP2D6 polymorphisms determine tramadol O-demethylation in critically ill neonates\\nand infants. Pediatr Res 2008;63:674-9.\\n7\\n. \\nKirchheiner J et al. Effects of the CYP2D6 gene duplication on the pharmacokinetics and pharmacodynamics of tramadol. J Clin\\nPsychopharmacol 2008;28:78-83.\\n8\\n. \\nStamer UM et al. Concentrations of tramadol and O-desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin\\nPharmacol Ther 2007;82:41-7.\\n9\\n. \\nGleason PP et al. Debilitating reaction following the initial dose of tramadol. Ann Pharmacother 1997;31:1150-2.\\n10\\n. \\nSPC Ultram (VS).\\nDate 20-11-2017\\nCYP2D6 IM: venlafaxine\\n1539\\nThe genetic polymorphism leads to decreased metabolic capacity of CYP2D6. This can cause an increase in the plasma concentration of\\nvenlafaxine and a decrease in the plasma concentration of the active metabolite O-desmethylvenlafaxine.\\nRecommendation:\\nIt is not possible to offer adequately substantiated advice for dose reduction based on the literature.\\n1\\n. \\nChoose an alternative\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\n2\\n. \\nIf an alternative is not an option and side effects occur:\\n1\\n. \\nreduce the dose\\n2\\n. \\ncheck the plasma concentrations of venlafaxine and O-desmethylvenlafaxine\\nIt is not known whether it is possible to reduce the dose to such an extent that the side effects disappear, while the\\neffectiveness is maintained. In general, it is assumed that the effectiveness is determined by the sum of the plasma\\nconcentrations of venlafaxine and O-desmethylvenlafaxine. However, the side effects do not appear to be related to this sum.\\nLiterature:\\n1\\n. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6\\npolymorphisms. Int J Psychiatry Clin Pract 2009;13:345-8.\\n2\\n. \\nHermann M et al. Serum concentrations of venlafaxine and its metabolites O-desmethylvenlafaxine and N-desmethylvenlafaxine\\nin heterozygous carriers of the CYP2D6\\n3, \\n4 or *5 allele. Eur J Clin Pharmacol 2008;64:483-7.\\n3\\n. \\nMcAlpine DE et al. Cytochrome P450 2D6 genotype variation and venlafaxine dosage. Mayo Clin Proc 2007;82:1065-8.\\n4\\n. \\nShams ME et al. CYP2D6 polymorphism and clinical effect of the antidepressant venlafaxine. J Clin Pharm Ther 2006;31:493-\\n502.\\n5\\n. \\nWhyte EM et al. CYP2D6 genotype and venlafaxine-XR concentrations in depressed elderly. Int J Geriatr Psychiatry\\n2006;21:542-9.\\n6\\n. \\nFukuda T et al. The impact of the CYP2D6 and CYP2C19 genotypes on venlafaxine pharmacokinetics in a Japanese population.\\nEur J Clin Pharmacol 2000;56:175-80.\\n7\\n. \\nVeefkind AH et al. Venlafaxine serum levels and CYP2D6 genotype. Ther Drug Monit 2000;22:202-8.\\n8\\n. \\nFukuda T et al. Effect of the CYP2D6*10 genotype on venlafaxine pharmacokinetics in healthy adult volunteers. Br J Clin\\nPharmacol 1999;47:450-3.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 140}),\n",
       " Document(page_content='Date 23-05-2012\\nCYP2D6 PM: venlafaxine\\n1538\\nThe genetic polymorphism leads to decreased metabolic capacity of CYP2D6. This can cause an increase in the plasma concentration of\\nvenlafaxine and a decrease in the plasma concentration of the active metabolite O-desmethylvenlafaxine. There are indications that the\\neffectiveness of venlafaxine is reduced in depression patients with this genetic polymorphism.\\nRecommendation:\\nIt is not possible to offer adequately substantiated advice for dose reduction based on the literature.\\n1\\n. \\nChoose an alternative\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\n2\\n. \\nIf an alternative is not an option and side effects occur:\\n1\\n. \\nreduce the dose\\n2\\n. \\ncheck the plasma concentrations of venlafaxine and O-desmethylvenlafaxine\\nIt is not known whether it is possible to reduce the dose to such an extent that the side effects disappear, while the\\neffectiveness is maintained. In general, it is assumed that the effectiveness is determined by the sum of the plasma\\nconcentrations of venlafaxine and O-desmethylvenlafaxine. However, the side effects do not appear to be related to this sum.\\nFurthermore, a reduced effectiveness of venlafaxine has been observed in depression patients with this genetic\\npolymorphism.\\nLiterature:\\n1\\n. \\nLobello KW et al. Cytochrome P450 2D6 phenotype predicts antidepressant efficacy of venlafaxine: a secondary analysis of 4\\nstudies in major depressive disorder. J Clin Psychiatry 2010;71:1482-7.\\n2\\n. \\nVan Nieuwerburgh FC et al. Response to serotonin reuptake inhibitors in OCD is not influenced by common CYP2D6\\npolymorphisms. Int J Psychiatry Clin Pract 2009;13:345-8.\\n3\\n. \\nPreskorn S et al. Comparison of the pharmacokinetics of venlafaxine extended release and desvenlafaxine in extensive and poor\\ncytochrome P450 2D6 metabolizers. J Clin Psychopharmacol 2009;29:39-43.\\n4\\n. \\nHermann M et al. Serum concentrations of venlafaxine and its metabolites O-desmethylvenlafaxine and N-desmethylvenlafaxine\\nin heterozygous carriers of the CYP2D6\\n3, \\n4 or *5 allele. Eur J Clin Pharmacol 2008;64:483-7.\\n5\\n. \\nShams ME et al. CYP2D6 polymorphism and clinical effect of the antidepressant venlafaxine. J Clin Pharm Ther 2006;31:493-\\n502.\\n6\\n. \\nWhyte EM et al. CYP2D6 genotype and venlafaxine-XR concentrations in depressed elderly. Int J Geriatr Psychiatry\\n2006;21:542-9.\\n7\\n. \\nEap CB et al. Role of CYP2D6 in the stereoselective disposition of venlafaxine in humans. Pharmacogenetics 2003;13:39-47.\\n8\\n. \\nLessard E et al. Influence of CYP2D6 activity on the disposition and cardiovascular toxicity of the antidepressant agent\\nvenlafaxine in humans. Pharmacogenetics 1999;9:435-43.\\n9\\n. \\nVeefkind AH et al. Venlafaxine serum levels and CYP2D6 genotype. Ther Drug Monit 2000;22:202-8.\\nDate 23-05-2012\\nCYP2D6 UM: venlafaxine\\n1540\\nThe genetic polymorphism leads to increased metabolic capacity of CYP2D6. This can cause a decrease in the plasma concentration of\\nvenlafaxine and an increase in the plasma concentration of the active metabolite O-desmethylvenlafaxine.\\nRecommendation:\\n1\\n. \\nbe alert to a possible decrease in the sum of the plasma concentrations of venlafaxine and the active metabolite O-\\ndesmethylvenlafaxine\\n2\\n. \\nif necessary, increase the dose to 150% of the standard dose\\n3\\n. \\nif dose adjustment based on therapeutic drug monitoring is not possible, an alternative should be selected\\nAntidepressants that are not metabolised by CYP2D6 - or to a lesser extent - include, for example, citalopram and sertraline.\\nLiterature:', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 141}),\n",
       " Document(page_content='1\\n. \\nShams ME et al. CYP2D6 polymorphism and clinical effect of the antidepressant venlafaxine. J Clin Pharm Ther 2006;31:493-\\n502.\\n2\\n. \\nVeefkind AH et al. Venlafaxine serum levels and CYP2D6 genotype. Ther Drug Monit 2000;22:202-8.\\nDate 23-05-2012\\nCYP2C19 IM: voriconazol\\n1683\\nThe gene variation can reduce the conversion of voriconazole and consequently increase the plasma concentration. This could result in\\nimproved efficacy or an increase in the risk of side effects.\\nRecommendation:\\nMonitor the plasma concentration\\nLiterature:\\n1\\n. \\nLi X et al. Effect of cytochrome P450 2C19 polymorphisms on the clinical outcomes of voriconazole: a systematic review and\\nmeta-analysis. Eur J Clin Pharmacol 2016;72:1185-93.\\n2\\n. \\nWang Y et al. Risk factors for voriconazole-associated hepatotoxicity in patients in the intensive care unit. Pharmacotherapy\\n2016;36:757-65.\\n3\\n. \\nChuwongwattana S et al. A prospective observational study of CYP2C19 polymorphisms and voriconazole plasma level in adult\\nThai patients with invasive aspergillosis. Drug Metab Pharmacokinet 2016;31:117-22.\\n4\\n. \\nTeusink A et al. Genotype-directed dosing leads to optimized voriconazole levels in pediatric patients receiving hematopoietic\\nstem cell transplantation. Biol Blood Marrow Transplant 2016;22:482-6.\\n5\\n. \\nLamoureux F et al. Impact of CYP2C19 genetic polymorphisms on voriconazole dosing and exposure in adult patients with\\ninvasive fungal infections. Int J Antimicrob Agents 2016;47:124-31.\\n6\\n. \\nChawla PK et al. Correlation of CYP2C19 genotype with plasma voriconazole levels: a preliminary retrospective study in Indians.\\nInt J Clin Pharm 2015;37:925-30.\\n7\\n. \\nYamada T et al. Saturated metabolism of voriconazole N-oxidation resulting in nonlinearity of pharmacokinetics of voriconazole\\nat clinical doses. Biol Pharm Bull 2015;38:1496-503.\\n8\\n. \\nMori M et al. Pharmacokinetics and safety of voriconazole intravenous-to-oral switch regimens in immunocompromised Japanese\\npediatric patients. Antimicrob Agents Chemother 2015;59:1004-13.\\n9\\n. \\nWang T et al. Efficacy and safety of voriconazole and CYP2C19 polymorphism for optimised dosage regimens in patients with\\ninvasive fungal infections. Int J Antimicrob Agents 2014;44:436-42.\\n10\\n. \\nLiu P et al. Population pharmacokinetic-pharmacodynamic analysis of voriconazole and anidulafungin in adult patients with\\ninvasive aspergillosis. Antimicrob Agents Chemother 2014;58:4727-36.\\n11\\n. \\nZonios D et al. Voriconazole metabolism, toxicity, and the effect of cytochrome P450 2C19 genotype. J Infect Dis 2014;209:1941-\\n8.\\n12\\n. \\nHicks JK et al. Voriconazole plasma concentrations in immunocompromised pediatric patients vary by CYP2C19 diplotypes.\\nPharmacogenomics 2014;15:1065-78.\\n13\\n. \\nKim SH et al. Clinical impact of cytochrome P450 2C19 genotype on the treatment of invasive aspergillosis under routine\\ntherapeutic drug monitoring of voriconazole in a Korean population. Infect Chemother 2013;45:406-14.\\n14\\n. \\nRacil Z et al. Monitoring trough voriconazole plasma concentrations in haematological patients: real life multicentre experience.\\nMycoses 2012;55:483-92.\\n15\\n. \\nKim SH et al. Voriconazole-related severe adverse events: clinical application of therapeutic drug monitoring in Korean patients.\\nInt J Infect Dis 2011;15:e753-8.\\n16\\n. \\nBerge M et al. Effect of cytochrome P450 2C19 genotype on voriconazole exposure in cystic fibrosis lung transplant patients. Eur\\nJ Clin Pharmacol 2011;67:253-60.\\n17\\n. \\nBrüggemann RJ et al. Pharmacokinetics and safety of 14 days intravenous voriconazole in allogeneic haematopoietic stem cell\\ntransplant recipients. J Antimicrob Chemother 2010;65:107-13.\\n18\\n. \\nMatsumoto K et al. Correlation between voriconazole trough plasma concentration and hepatotoxicity in patients with different\\nCYP2C19 genotypes. Int J Antimicrob Agents 2009;34:91-4.\\n19\\n. \\nKarlsson MO et al. Population pharmacokinetic analysis of voriconazole plasma concentration data from pediatric studies.\\nAntimicrob Agents Chemother 2009;53:935-44.\\n20\\n. \\nWeiss J et al. CYP2C19 genotype is a major factor contributing to the highly variable pharmacokinetics of voriconazole. J Clin\\nPharmacol 2009;49:196-204.\\n21\\n. \\nLevin MD et al. Hepatotoxicity of oral and intravenous voriconazole in relation to cytochrome P450 polymorphisms. J Antimicrob\\nChemother 2007;60:1104-7.\\n22\\n. \\nMikus G et al. Potent cytochrome P450 2C19 genotype-related interaction between voriconazole and the cytochrome P450 3A4\\ninhibitor ritonavir. Clin Pharmacol Ther 2006;80:126-35.\\n23\\n. \\nRengelshausen J et al. Opposite effects of short-term and long-term St John’s wort intake on voriconazole pharmacokinetics. Clin', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 142}),\n",
       " Document(page_content='Pharmacol Ther 2005;78:25-33.\\n24\\n. \\nIkeda Y et al. Pharmacokinetics of voriconazole and cytochrome P450 2C19 genetic status. Clin Pharmacol Ther 2004;75:587-8.\\n25\\n. \\nSPC Vfend.\\nDate 01-05-2017\\nCYP2C19 PM: voriconazol\\n1684\\nThe gene variation can reduce the conversion of voriconazole and consequently increase the plasma concentration. This could result in\\nimproved efficacy or an increase in the risk of side effects.\\nInitially, the risk of side effects is of particular interest.\\nRecommendation:\\nUse 50% of the standard dose and monitor the plasma concentration\\nLiterature:\\n1\\n. \\nLi X et al. Effect of cytochrome P450 2C19 polymorphisms on the clinical outcomes of voriconazole: a systematic review and\\nmeta-analysis. Eur J Clin Pharmacol 2016;72:1185-93.\\n2\\n. \\nWang Y et al. Risk factors for voriconazole-associated hepatotoxicity in patients in the intensive care unit. Pharmacotherapy\\n2016;36:757-65.\\n3\\n. \\nChuwongwattana S et al. A prospective observational study of CYP2C19 polymorphisms and voriconazole plasma level in adult\\nThai patients with invasive aspergillosis. Drug Metab Pharmacokinet 2016;31:117-22.\\n4\\n. \\nTeusink A et al. Genotype-directed dosing leads to optimized voriconazole levels in pediatric patients receiving hematopoietic\\nstem cell transplantation. Biol Blood Marrow Transplant 2016;22:482-6.\\n5\\n. \\nLamoureux F et al. Impact of CYP2C19 genetic polymorphisms on voriconazole dosing and exposure in adult patients with\\ninvasive fungal infections. Int J Antimicrob Agents 2016;47:124-31.\\n6\\n. \\nChawla PK et al. Correlation of CYP2C19 genotype with plasma voriconazole levels: a preliminary retrospective study in Indians.\\nInt J Clin Pharm 2015;37:925-30.\\n7\\n. \\nYamada T et al. Saturated metabolism of voriconazole N-oxidation resulting in nonlinearity of pharmacokinetics of voriconazole\\nat clinical doses. Biol Pharm Bull 2015;38:1496-503.\\n8\\n. \\nMori M et al. Pharmacokinetics and safety of voriconazole intravenous-to-oral switch regimens in immunocompromised Japanese\\npediatric patients. Antimicrob Agents Chemother 2015;59:1004-13.\\n9\\n. \\nWang T et al. Efficacy and safety of voriconazole and CYP2C19 polymorphism for optimised dosage regimens in patients with\\ninvasive fungal infections. Int J Antimicrob Agents 2014;44:436-42.\\n10\\n. \\nLiu P et al. Population pharmacokinetic-pharmacodynamic analysis of voriconazole and anidulafungin in adult patients with\\ninvasive aspergillosis. Antimicrob Agents Chemother 2014;58:4727-36.\\n11\\n. \\nZonios D et al. Voriconazole metabolism, toxicity, and the effect of cytochrome P450 2C19 genotype. J Infect Dis 2014;209:1941-\\n8.\\n12\\n. \\nHicks JK et al. Voriconazole plasma concentrations in immunocompromised pediatric patients vary by CYP2C19 diplotypes.\\nPharmacogenomics 2014;15:1065-78.\\n13\\n. \\nKim SH et al. Clinical impact of cytochrome P450 2C19 genotype on the treatment of invasive aspergillosis under routine\\ntherapeutic drug monitoring of voriconazole in a Korean population. Infect Chemother 2013;45:406-14.\\n14\\n. \\nDriscoll TA et al. Comparison of pharmacokinetics and safety of voriconazole intravenous-to-oral switch in immunocompromised\\nadolescents and healthy adults. Antimicrob Agents Chemother 2011;55:5780-9.\\n15\\n. \\nKim SH et al. Voriconazole-related severe adverse events: clinical application of therapeutic drug monitoring in Korean patients.\\nInt J Infect Dis 2011;15:e753-8.\\n16\\n. \\nLei HP et al. Lack of effect of Ginkgo biloba on voriconazole pharmacokinetics in Chinese volunteers identified as CYP2C19 poor\\nand extensive metabolizers. Ann Pharmacother 2009;43:726-31.\\n17\\n. \\nWang G et al. The CYP2C19 ultra-rapid metabolizer genotype influences the pharmacokinetics of voriconazole in healthy male\\nvolunteers. Eur J Clin Pharmacol 2009;65:281-5.\\n18\\n. \\nWeiss J et al. CYP2C19 genotype is a major factor contributing to the highly variable pharmacokinetics of voriconazole. J Clin\\nPharmacol 2009;49:196-204.\\n19\\n. \\nMikus G et al. Potent cytochrome P450 2C19 genotype-related interaction between voriconazole and the cytochrome P450 3A4\\ninhibitor ritonavir. Clin Pharmacol Ther 2006;80:126-35.\\n20\\n. \\nRengelshausen J et al. Opposite effects of short-term and long-term St John\"s wort intake on voriconazole pharmacokinetics. Clin\\nPharmacol Ther 2005;78:25-33.\\n21\\n. \\nIkeda Y et al. Pharmacokinetics of voriconazole and cytochrome P450 2C19 genetic status. Clin Pharmacol Ther 2004;75:587-8.\\n22\\n. \\nSPC Vfend.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 143}),\n",
       " Document(page_content='Date 01-05-2017\\nCYP2C19 UM: voriconazol\\n1685\\nThe gene variation increases the conversion of voriconazole, which increases the risk of ineffectiveness.\\nRecommendation:\\nUse an initial dose that is 1.5x higher and monitor the plasma concentration\\nLiterature:\\n1\\n. \\nWilliams K et al. Association of CYP2C19 \\n17/\\n17 genotype with the risk of voriconazole-associated squamous cell carcinoma.\\nJAMA Dermatol 2016;152:719-20.\\n2\\n. \\nLamoureux F et al. Impact of CYP2C19 genetic polymorphisms on voriconazole dosing and exposure in adult patients with\\ninvasive fungal infections. Int J Antimicrob Agents 2016;47:124-31.\\n3\\n. \\nChawla PK et al. Correlation of CYP2C19 genotype with plasma voriconazole levels: a preliminary retrospective study in Indians.\\nInt J Clin Pharm 2015;37:925-30.\\n4\\n. \\nZonios D et al. Voriconazole metabolism, toxicity, and the effect of cytochrome P450 2C19 genotype. J Infect Dis 2014;209:1941-\\n8.\\n5\\n. \\nHicks JK et al. Voriconazole plasma concentrations in immunocompromised pediatric patients vary by CYP2C19 diplotypes.\\nPharmacogenomics 2014;15:1065-78.\\n6\\n. \\nDriscoll TA et al. Comparison of pharmacokinetics and safety of voriconazole intravenous-to-oral switch in immunocompromised\\nchildren and healthy adults. Antimicrob Agents Chemother 2011;55:5770-9.\\n7\\n. \\nDriscoll TA et al. Comparison of pharmacokinetics and safety of voriconazole intravenous-to-oral switch in immunocompromised\\nadolescents and healthy adults. Antimicrob Agents Chemother 2011;55:5780-9.\\n8\\n. \\nBerge M et al. Effect of cytochrome P450 2C19 genotype on voriconazole exposure in cystic fibrosis lung transplant patients. Eur\\nJ Clin Pharmacol 2011;67:253-60.\\n9\\n. \\nWang G et al. The CYP2C19 ultra-rapid metabolizer genotype influences the pharmacokinetics of voriconazole in healthy male\\nvolunteers. Eur J Clin Pharmacol 2009;65:281-5.\\n10\\n. \\nWeiss J et al. CYP2C19 genotype is a major factor contributing to the highly variable pharmacokinetics of voriconazole. J Clin\\nPharmacol 2009;49:196-204.\\n11\\n. \\nLi X et al. Effect of cytochrome P450 2C19 polymorphisms on the clinical outcomes of voriconazole: a systematic review and\\nmeta-analysis. Eur J Clin Pharmacol 2016;72:1185-93.\\nDate 01-05-2017\\nCYP2C9 IM: warfarine\\n6233\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1\\n. \\nuse 65% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm. Algorithms for Caucasian patients usually\\ncontain only the *2 and *3 allele. If the activity of the reduced-activity alleles is comparable to the activity of *2 or *3, then the\\nalgorithm can be completed as if *1/*2 or *1/*3 is present. See https://www.knmp.nl/producten-en-diensten/gebruiksrecht-g-\\nstandaard/medicatiebewaking-g-standaard/background-information-pharmacogenetics for Excel files containing calculation modules for\\noral and equivalent intravenous doses. From day 6 on the standard algorithm without genotype information can be used to calculate the\\ndose.\\nModified dose algorithms have been developed for patients of African or (East) Asian heritage.\\nLiterature:\\n1\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015; 39:228-34.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 144}),\n",
       " Document(page_content='2\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;\\n177:654-7.\\n3\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014; 12:1480-7.\\n4\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014; 174:1330-8.\\n5\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013; 168:4234-43.\\n6\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012; 7:e44064.\\n7\\n. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin\\nPharmacol 2009; 65:365-75.\\n8\\n. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review\\nand meta-analysis. Genet Med 2005; 7:97-104.\\n9\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9 PM: warfarine\\n6234\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1\\n. \\nuse 20% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm.\\nAlgorithms for Caucasian patients usually contain only the *2 and *3 allele. If the activity of the reduced-activity alleles is comparable\\nto the activity of *2 or *3, then the algorithm can be completed as if *2 or *3 is present. See https://www.knmp.nl/producten-en-\\ndiensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-pharmacogenetics for Excel files\\ncontaining calculation modules for oral and equivalent intravenous doses. From day 6 on the standard algorithm without genotype\\ninformation can be used to calculate the dose.\\nModified dose algorithms have been developed for patients of African or (East) Asian heritage.\\nLiterature:\\n1\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015; 39:228-34.\\n2\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;\\n177:654-7.\\n3\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014; 12:1480-7.\\n4\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014; 174:1330-8.\\n5\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013; 168:4234-43.\\n6\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012; 7:e44064.\\n7\\n. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin\\nPharmacol 2009; 65:365-75.\\n8\\n. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review\\nand meta-analysis. Genet Med 2005; 7:97-104.\\n9\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*1/*2: warfarine\\n6228\\nNO action is required for this gene-drug interaction.', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 145}),\n",
       " Document(page_content='Genetic variation may lead to a decrease in the required maintenance dose. However, there is insufficient evidence that this causes\\nproblems when therapy is initiated as usual.\\nLiterature:\\n1\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015; 39:228-34.\\n2\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;\\n177:654-7.\\n3\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014; 12:1480-7.\\n4\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014; 174:1330-8.\\n5\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013; 168:4234-43.\\n6\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012; 7:e44064.\\n7\\n. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin\\nPharmacol 2009; 65:365-75.\\n8\\n. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review\\nand meta-analysis. Genet Med 2005; 7:97-104.\\n9\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*1/*3: warfarine\\n6229\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1\\n. \\nuse 65% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see\\nhttps://www.knmp.nl/producten-en-diensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-\\npharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015; 39:228-34.\\n2\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;\\n177:654-7.\\n3\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014; 12:1480-7.\\n4\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014; 174:1330-8.\\n5\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013; 168:4234-43.\\n6\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012; 7:e44064.\\n7\\n. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin\\nPharmacol 2009; 65:365-75.\\n8\\n. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review\\nand meta-analysis. Genet Med 2005; 7:97-104.\\n9\\n. \\nSPC Coumadin (VS).', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 146}),\n",
       " Document(page_content='Date 24-08-2016\\nCYP2C9*2/*2: warfarine\\n6230\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1\\n. \\nuse 65% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see\\nhttps://www.knmp.nl/producten-en-diensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-\\npharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015; 39:228-34.\\n2\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;\\n177:654-7.\\n3\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014; 12:1480-7.\\n4\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014; 174:1330-8.\\n5\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013; 168:4234-43.\\n6\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012; 7:e44064.\\n7\\n. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin\\nPharmacol 2009; 65:365-75.\\n8\\n. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review\\nand meta-analysis. Genet Med 2005; 7:97-104.\\n9\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*2/*3: warfarine\\n6231\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1\\n. \\nuse 45% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see\\nhttps://www.knmp.nl/producten-en-diensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-\\npharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015; 39:228-34.\\n2\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;\\n177:654-7.\\n3\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 147}),\n",
       " Document(page_content='systematic review and meta-analysis. J Thromb Haemost 2014; 12:1480-7.\\n4\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014; 174:1330-8.\\n5\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013; 168:4234-43.\\n6\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012; 7:e44064.\\n7\\n. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin\\nPharmacol 2009; 65:365-75.\\n8\\n. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review\\nand meta-analysis. Genet Med 2005; 7:97-104.\\n9\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nCYP2C9*3/*3: warfarine\\n6232\\nThis gene variation reduces the conversion of warfarin to inactive metabolites. This can increase the risk of bleeding.\\nRecommendation:\\n1\\n. \\nuse 20% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see\\nhttps://www.knmp.nl/producten-en-diensten/gebruiksrecht-g-standaard/medicatiebewaking-g-standaard/background-information-\\npharmacogenetics.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015; 39:228-34.\\n2\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol 2014;\\n177:654-7.\\n3\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014; 12:1480-7.\\n4\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014; 174:1330-8.\\n5\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013; 168:4234-43.\\n6\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012; 7:e44064.\\n7\\n. \\nLindh JD et al. Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin\\nPharmacol 2009; 65:365-75.\\n8\\n. \\nSanderson S et al. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin-treated patients: a HuGEnet systematic review\\nand meta-analysis. Genet Med 2005; 7:97-104.\\n9\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nVKORC1 -1639 AA: warfarine\\n6236\\nThe genetic variation results in increased sensitivity to warfarin. This results in an increase in the risk of excessively severe inhibition of\\nblood clotting (INR > 4) during the first month of the treatment.\\nRecommendation:\\n1\\n. \\nuse 60% of the standard initial dose\\nThe genotype-specific initial dose and maintenance dose can be calculated using an algorithm, as used in EU-PACT: see', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 148}),\n",
       " Document(page_content='https://www.knmp.nl/patientenzorg/medicatiebewaking/farmacogenetica.\\nFrom day 6 on the standard algorithm without genotype information can be used to calculate the dose.\\nLiterature:\\n1\\n. \\nZhang J et al. The influence of VKORC1 gene polymorphism on warfarin maintenance dosage in pediatric patients: A systematic\\nreview and meta-analysis. Thromb Res 2015;136:955-61.\\n2\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015;39:228-34.\\n3\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmaco-genetic dosing of warfarin. Int J Cardiol\\n2014;177:654-7.\\n4\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014;12:1480-7.\\n5\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014;174:1330-8.\\n6\\n. \\nJin B et al. The impact of VKORC1-1639G\\n\\u2009\\n>\\n\\u2009\\nA genetic polymorphism upon warfarin dose requirement in different ethnic\\npopulations. Curr Med Res Opin 2014;30:1505-11.\\n7\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013;168:4234-43.\\n8\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012;7:e44064.\\n9\\n. \\nYang L et al. Impact of VKORC1 gene polymorphism on interindividual and interethnic warfarin dosage requirement--a\\nsystematic review and meta analysis. Thromb Res 2010;125:e159-66.\\n10\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016\\nVKORC1 -1639 GA: warfarine\\n6235\\nNO action is required for this gene-drug interaction.\\nThe genetic variation results in a reduction in the required dose and an increase in the risk of excessively severe inhibition of blood\\nclotting during the first month of the treatment. However, the effect is small and GA is also the most common genotype, meaning that\\nthe standard treatment will primarily be based on patients with this genotype.\\nLiterature:\\n1\\n. \\nZhang J et al. The influence of VKORC1 gene polymorphism on warfarin maintenance dosage in pediatric patients: A systematic\\nreview and meta-analysis. Thromb Res 2015;136:955-61.\\n2\\n. \\nLiao Z et al. Meta-analysis of randomized controlled trials reveals an improved clinical outcome of using genotype plus clinical\\nalgorithm for warfarin dosing. J Thromb Thrombolysis 2015;39:228-34.\\n3\\n. \\nXu H et al. Meta-analysis of efficacy and safety of genotype-guided pharmacogenetic dosing of warfarin. Int J Cardiol\\n2014;177:654-7.\\n4\\n. \\nFranchini M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a\\nsystematic review and meta-analysis. J Thromb Haemost 2014;12:1480-7.\\n5\\n. \\nStergiopoulos K et al. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical\\ntrials. JAMA Intern Med 2014;174:1330-8.\\n6\\n. \\nJin B et al. The impact of VKORC1-1639G\\n\\u2009\\n>\\n\\u2009\\nA genetic polymorphism upon warfarin dose requirement in different ethnic\\npopulations. Curr Med Res Opin 2014;30:1505-11.\\n7\\n. \\nYang J et al. Influence of CYP2C9 and VKORC1 genotypes on the risk of hemorrhagic complications in warfarin-treated patients:\\na systematic review and meta-analysis. Int J Cardiol 2013;168:4234-43.\\n8\\n. \\nJorgensen AL et al. Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis.\\nPLoS One 2012;7:e44064.\\n9\\n. \\nSPC Coumadin (VS).\\nDate 24-08-2016', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 149}),\n",
       " Document(page_content='CYP2D6 IM: zuclopentixol\\n1548\\nThe genetic polymorphism leads to decreased metabolic capacity of CYP2D6, which may cause increased zuclopentixol plasma\\nconcentrations.\\nRecommendation:\\n1\\n. \\nAdvise the prescriber to start with 75% of the standard dose or to choose an alternative according to the current guidelines.\\nAntipsychotics that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, flupentixol, quetiapine,\\nolanzapine and clozapine.\\nLiterature:\\n1\\n. \\nVan Berlo-van de Laar et al. Dosering aanpassen aan eliminatiesnelheid. Pharm Weekblad 2004; 139: 740-43.\\n2\\n. \\nJaanson P et al. Maintenance therapy with zuclopentixol decanoate: associations between plasma concentrations, neurological side\\neffects and CYP2D6 genotype. Psychopharmacology 2002;162:67-73.\\n3\\n. \\nJerling M et al. The CYP2D6 genotype predicts the oral clearance of the neuroleptic agents perphenazine and zuclopentixol. Clin\\nPharmacol Ther 1996;59:423-8.\\nDate 14-12-2005\\nCYP2D6 PM: zuclopentixol\\n1547\\nThe genetic polymorphism leads to decreased metabolic capacity of CYP2D6, which may cause increased zuclopentixol plasma\\nconcentrations.\\nRecommendation:\\n1\\n. \\nAdvise the prescriber to start with 50% of the standard dose or to choose an alternative according to the current guidelines.\\nAntipsychotics that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, flupentixol, quetiapine,\\nolanzapine and clozapine.\\nLiterature:\\n1\\n. \\nVan Berlo-van de Laar et al. Dosering aanpassen aan eliminatiesnelheid. Pharm Weekblad 2004; 139: 740-43.\\n2\\n. \\nJaanson P et al. Maintenance therapy with zuclopentixol decanoate: associations between plasma concentrations, neurological side\\neffects and CYP2D6 genotype. Psychopharmacology 2002;162:67-73.\\n3\\n. \\nLinnet K et al. Influence of Cyp2D6 genetic polymorphism on ratios of steady- tate serum concentration to dose of the neuroleptic\\nzuclopentixol. Ther Drug Monit 1996;18:629-34.\\n4\\n. \\nJerling M et al. The CYP2D6 genotype predicts the oral clearance of the neuroleptic agents perphenazine and zuclopentixol. Clin\\nPharmacol Ther 1996;59:423-8.\\n5\\n. \\nDahl ML et al. Disposition of the neuroleptic zuclopentixol cosegregates with the polymorphic hydroxylation of debrisoquine in\\nhumans. Acta Psychiatr Scand 1991;84:99-102.\\nDate 14-12-2005\\nCYP2D6 UM: zuclopentixol\\n1549\\nThe genetic polymorphism leads to increased metabolic capacity of CYP2D6, which may cause decreased zuclopentixol plasma\\nconcentrations.\\nRecommendation:\\nNo data have been published from studies into the pharmacokinetics and effects of zuclopentixol for this phenotype.\\n1\\n. \\nAs a precaution, the prescriber should advised to be alert to a decreased zuclopentixol plasma concentration and - if necessary - the', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 150}),\n",
       " Document(page_content='dose should be increased on the basis of the clinical effect, or an alternative should be prescribed according to the current\\nguidelines.\\nAntipsychotics that are not metabolised via CYP2D6 - or to a lesser extent - include, for example, flupentixol, quetiapine,\\nolanzapine and clozapine.\\nLiterature:\\n-\\nDate 14-12-2005', metadata={'source': '/home/dhanushb/Wellytics/RAG_data/all_files/DPWG_November_2018.pdf', 'page': 151})]"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "for doc in pdfdocs:\n",
    "    doc.page_content = doc.page_content.replace('\\t', ' ')\n",
    "pdfdocs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "ename": "UnicodeDecodeError",
     "evalue": "'utf-8' codec can't decode byte 0xfa in position 31: invalid start byte",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mUnicodeDecodeError\u001b[0m                        Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[44], line 4\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mpandas\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m \u001b[38;5;21;01mpd\u001b[39;00m\n\u001b[1;32m      2\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mnumpy\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m \u001b[38;5;21;01mnp\u001b[39;00m\n\u001b[0;32m----> 4\u001b[0m links \u001b[38;5;241m=\u001b[39m \u001b[43mpd\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mread_csv\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43m/home/dhanushb/Wellytics/RAG_data/Sources.csv\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m      6\u001b[0m websites \u001b[38;5;241m=\u001b[39m links[links[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mhtml\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m.\u001b[39mnotna()][\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mhtml\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m.\u001b[39mto_list()\n\u001b[1;32m      8\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mlangchain_community\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mdocument_loaders\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m WebBaseLoader\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pandas/io/parsers/readers.py:1026\u001b[0m, in \u001b[0;36mread_csv\u001b[0;34m(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, date_format, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, encoding_errors, dialect, on_bad_lines, delim_whitespace, low_memory, memory_map, float_precision, storage_options, dtype_backend)\u001b[0m\n\u001b[1;32m   1013\u001b[0m kwds_defaults \u001b[38;5;241m=\u001b[39m _refine_defaults_read(\n\u001b[1;32m   1014\u001b[0m     dialect,\n\u001b[1;32m   1015\u001b[0m     delimiter,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   1022\u001b[0m     dtype_backend\u001b[38;5;241m=\u001b[39mdtype_backend,\n\u001b[1;32m   1023\u001b[0m )\n\u001b[1;32m   1024\u001b[0m kwds\u001b[38;5;241m.\u001b[39mupdate(kwds_defaults)\n\u001b[0;32m-> 1026\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_read\u001b[49m\u001b[43m(\u001b[49m\u001b[43mfilepath_or_buffer\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mkwds\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pandas/io/parsers/readers.py:620\u001b[0m, in \u001b[0;36m_read\u001b[0;34m(filepath_or_buffer, kwds)\u001b[0m\n\u001b[1;32m    617\u001b[0m _validate_names(kwds\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mnames\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m))\n\u001b[1;32m    619\u001b[0m \u001b[38;5;66;03m# Create the parser.\u001b[39;00m\n\u001b[0;32m--> 620\u001b[0m parser \u001b[38;5;241m=\u001b[39m \u001b[43mTextFileReader\u001b[49m\u001b[43m(\u001b[49m\u001b[43mfilepath_or_buffer\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwds\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    622\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m chunksize \u001b[38;5;129;01mor\u001b[39;00m iterator:\n\u001b[1;32m    623\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m parser\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pandas/io/parsers/readers.py:1620\u001b[0m, in \u001b[0;36mTextFileReader.__init__\u001b[0;34m(self, f, engine, **kwds)\u001b[0m\n\u001b[1;32m   1617\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39moptions[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mhas_index_names\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m=\u001b[39m kwds[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mhas_index_names\u001b[39m\u001b[38;5;124m\"\u001b[39m]\n\u001b[1;32m   1619\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhandles: IOHandles \u001b[38;5;241m|\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m\n\u001b[0;32m-> 1620\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_engine \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_engine\u001b[49m\u001b[43m(\u001b[49m\u001b[43mf\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mengine\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pandas/io/parsers/readers.py:1898\u001b[0m, in \u001b[0;36mTextFileReader._make_engine\u001b[0;34m(self, f, engine)\u001b[0m\n\u001b[1;32m   1895\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mValueError\u001b[39;00m(msg)\n\u001b[1;32m   1897\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1898\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mmapping\u001b[49m\u001b[43m[\u001b[49m\u001b[43mengine\u001b[49m\u001b[43m]\u001b[49m\u001b[43m(\u001b[49m\u001b[43mf\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43moptions\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1899\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m:\n\u001b[1;32m   1900\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhandles \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py:93\u001b[0m, in \u001b[0;36mCParserWrapper.__init__\u001b[0;34m(self, src, **kwds)\u001b[0m\n\u001b[1;32m     90\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m kwds[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mdtype_backend\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mpyarrow\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[1;32m     91\u001b[0m     \u001b[38;5;66;03m# Fail here loudly instead of in cython after reading\u001b[39;00m\n\u001b[1;32m     92\u001b[0m     import_optional_dependency(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mpyarrow\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[0;32m---> 93\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_reader \u001b[38;5;241m=\u001b[39m \u001b[43mparsers\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mTextReader\u001b[49m\u001b[43m(\u001b[49m\u001b[43msrc\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwds\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     95\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39munnamed_cols \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_reader\u001b[38;5;241m.\u001b[39munnamed_cols\n\u001b[1;32m     97\u001b[0m \u001b[38;5;66;03m# error: Cannot determine type of 'names'\u001b[39;00m\n",
      "File \u001b[0;32mparsers.pyx:574\u001b[0m, in \u001b[0;36mpandas._libs.parsers.TextReader.__cinit__\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mparsers.pyx:663\u001b[0m, in \u001b[0;36mpandas._libs.parsers.TextReader._get_header\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mparsers.pyx:874\u001b[0m, in \u001b[0;36mpandas._libs.parsers.TextReader._tokenize_rows\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mparsers.pyx:891\u001b[0m, in \u001b[0;36mpandas._libs.parsers.TextReader._check_tokenize_status\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32mparsers.pyx:2053\u001b[0m, in \u001b[0;36mpandas._libs.parsers.raise_parser_error\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;31mUnicodeDecodeError\u001b[0m: 'utf-8' codec can't decode byte 0xfa in position 31: invalid start byte"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "links = pd.read_csv(\"/home/dhanushb/Wellytics/RAG_data/Sources.csv\")\n",
    "\n",
    "websites = links[links[\"html\"].notna()][\"html\"].to_list()\n",
    "\n",
    "from langchain_community.document_loaders import WebBaseLoader\n",
    "from langchain_community.document_loaders.recursive_url_loader import RecursiveUrlLoader\n",
    "\n",
    "webdocs = []\n",
    "\n",
    "for website in websites:\n",
    "    loader = WebBaseLoader(website)\n",
    "    doc = loader.load()\n",
    "    webdocs.extend(doc)\n",
    "\n",
    "webdocs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
    "\n",
    "text_splitter = RecursiveCharacterTextSplitter(chunk_size=5000, chunk_overlap=1000)\n",
    "documents = text_splitter.split_documents(pdfdocs)\n",
    "jsondocs = text_splitter.split_documents(jsondata)\n",
    "#webdocuments = text_splitter.split_documents(webdocs[:8])\n",
    "\n",
    "documents += jsondocs + csvdata \n",
    "#documents2 = documents + webdocuments"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For our specific application, let's use 1000 characters instead:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.vectorstores import DocArrayInMemorySearch\n",
    "\n",
    "#vectorstore = DocArrayInMemorySearch.from_documents(documents, embedding=embeddings)\n",
    "\n",
    "from langchain_community.vectorstores import Qdrant\n",
    "\n",
    "#vectorstore = Qdrant.from_documents(documents, embedding=embeddings)\n",
    "\n",
    "from langchain_community.vectorstores import Chroma\n",
    "\n",
    "#vectorstore = Chroma.from_documents(documents, embedding=embeddings)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 63,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import os\n",
    "from dotenv import load_dotenv\n",
    "\n",
    "load_dotenv()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.vectorstores import Pinecone\n",
    "\n",
    "vectorstore = Pinecone.from_documents(documents, embedding=embeddings, index_name=\"rag-data\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_pinecone import PineconeVectorStore\n",
    "\n",
    "vectorstore = PineconeVectorStore(embedding=embeddings, index_name=\"rag-data\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "retriever = vectorstore.as_retriever()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can get a retriever directly from the vector store we created before: "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can create a map with the two inputs by using the [`RunnableParallel`](https://python.langchain.com/docs/expression_language/how_to/map) and [`RunnablePassthrough`](https://python.langchain.com/docs/expression_language/how_to/passthrough) classes. This will allow us to pass the context and question to the prompt as a map with the keys \"context\" and \"question.\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_core.runnables import RunnableParallel, RunnablePassthrough\n",
    "\n",
    "setup = RunnableParallel(context=retriever, question=RunnablePassthrough())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Questions</th>\n",
       "      <th>Expected_response</th>\n",
       "      <th>Expected_source</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>As part of my liver transplant, I take tacroli...</td>\n",
       "      <td>For a CYP3A5 extensive metabolizer. Increase s...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>I'm Indian, and my family has a history of epi...</td>\n",
       "      <td>As per CPIC HLA-B*15:02 might be observed in 2...</td>\n",
       "      <td>https://www.pharmgkb.org/haplotype/PA165954769...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Allopurinol is what I take for gout. For my se...</td>\n",
       "      <td>There can be moderate interaction while consid...</td>\n",
       "      <td>https://insilico-cyp.charite.de/SuperCYPsPred/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Is it true that while taking clopidogrel, I sh...</td>\n",
       "      <td>Indeed, you should refrain from drinking grape...</td>\n",
       "      <td>Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Dizziness is one of the adverse reactions that...</td>\n",
       "      <td>Dizziness drawiness,extreme thirst,fast, shall...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>What are the pharmacogenomic panel advised to ...</td>\n",
       "      <td>G6PD, CYP2D6, UGT1A1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>I have a 7 year old patient with HLA-B1502 pos...</td>\n",
       "      <td>Sodium Valproate</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>How does CYP2C9 affect anticoagulants mode of ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/disease/PA166123298/l...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>Which are the antidiabetic medications influen...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA164712711/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>What are the adverse outcomes associated with ...</td>\n",
       "      <td>Patients with the rs1801133 AA genotype and Ar...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>172 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             Questions  \\\n",
       "0    As part of my liver transplant, I take tacroli...   \n",
       "1    I'm Indian, and my family has a history of epi...   \n",
       "2    Allopurinol is what I take for gout. For my se...   \n",
       "3    Is it true that while taking clopidogrel, I sh...   \n",
       "4    Dizziness is one of the adverse reactions that...   \n",
       "..                                                 ...   \n",
       "167  What are the pharmacogenomic panel advised to ...   \n",
       "168  I have a 7 year old patient with HLA-B1502 pos...   \n",
       "169  How does CYP2C9 affect anticoagulants mode of ...   \n",
       "170  Which are the antidiabetic medications influen...   \n",
       "171  What are the adverse outcomes associated with ...   \n",
       "\n",
       "                                     Expected_response  \\\n",
       "0    For a CYP3A5 extensive metabolizer. Increase s...   \n",
       "1    As per CPIC HLA-B*15:02 might be observed in 2...   \n",
       "2    There can be moderate interaction while consid...   \n",
       "3    Indeed, you should refrain from drinking grape...   \n",
       "4    Dizziness drawiness,extreme thirst,fast, shall...   \n",
       "..                                                 ...   \n",
       "167                               G6PD, CYP2D6, UGT1A1   \n",
       "168                                   Sodium Valproate   \n",
       "169                                                NaN   \n",
       "170                                                NaN   \n",
       "171  Patients with the rs1801133 AA genotype and Ar...   \n",
       "\n",
       "                                       Expected_source  \n",
       "0    https://www.fda.gov/drugs/science-and-research...  \n",
       "1    https://www.pharmgkb.org/haplotype/PA165954769...  \n",
       "2    https://insilico-cyp.charite.de/SuperCYPsPred/...  \n",
       "3    Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...  \n",
       "4    https://www.fda.gov/drugs/science-and-research...  \n",
       "..                                                 ...  \n",
       "167                                                NaN  \n",
       "168                                                NaN  \n",
       "169  https://www.pharmgkb.org/disease/PA166123298/l...  \n",
       "170  https://www.pharmgkb.org/chemical/PA164712711/...  \n",
       "171                                                NaN  \n",
       "\n",
       "[172 rows x 3 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "Questions = pd.read_csv(\"/home/dhanushb/Wellytics/RAG_data/Questions.csv\")\n",
    "Questions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Questions</th>\n",
       "      <th>Expected_response</th>\n",
       "      <th>Expected_source</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>As part of my liver transplant, I take tacroli...</td>\n",
       "      <td>For a CYP3A5 extensive metabolizer. Increase s...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>I'm Indian, and my family has a history of epi...</td>\n",
       "      <td>As per CPIC HLA-B*15:02 might be observed in 2...</td>\n",
       "      <td>https://www.pharmgkb.org/haplotype/PA165954769...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Allopurinol is what I take for gout. For my se...</td>\n",
       "      <td>There can be moderate interaction while consid...</td>\n",
       "      <td>https://insilico-cyp.charite.de/SuperCYPsPred/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Is it true that while taking clopidogrel, I sh...</td>\n",
       "      <td>Indeed, you should refrain from drinking grape...</td>\n",
       "      <td>Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Dizziness is one of the adverse reactions that...</td>\n",
       "      <td>Dizziness drawiness,extreme thirst,fast, shall...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>My patient has CYP2D6 poor metabolism and does...</td>\n",
       "      <td>CPIC guideline for ondansetron and CYP2D6 has ...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>What are side effects associated wtih DPYD poo...</td>\n",
       "      <td>5-FU has many adverse drug effects when used s...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>What are the pharmacogenomic panel advised to ...</td>\n",
       "      <td>G6PD, CYP2D6, UGT1A1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>I have a 7 year old patient with HLA-B1502 pos...</td>\n",
       "      <td>Sodium Valproate</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>What are the adverse outcomes associated with ...</td>\n",
       "      <td>Patients with the rs1801133 AA genotype and Ar...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>134 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             Questions  \\\n",
       "0    As part of my liver transplant, I take tacroli...   \n",
       "1    I'm Indian, and my family has a history of epi...   \n",
       "2    Allopurinol is what I take for gout. For my se...   \n",
       "3    Is it true that while taking clopidogrel, I sh...   \n",
       "4    Dizziness is one of the adverse reactions that...   \n",
       "..                                                 ...   \n",
       "149  My patient has CYP2D6 poor metabolism and does...   \n",
       "166  What are side effects associated wtih DPYD poo...   \n",
       "167  What are the pharmacogenomic panel advised to ...   \n",
       "168  I have a 7 year old patient with HLA-B1502 pos...   \n",
       "171  What are the adverse outcomes associated with ...   \n",
       "\n",
       "                                     Expected_response  \\\n",
       "0    For a CYP3A5 extensive metabolizer. Increase s...   \n",
       "1    As per CPIC HLA-B*15:02 might be observed in 2...   \n",
       "2    There can be moderate interaction while consid...   \n",
       "3    Indeed, you should refrain from drinking grape...   \n",
       "4    Dizziness drawiness,extreme thirst,fast, shall...   \n",
       "..                                                 ...   \n",
       "149  CPIC guideline for ondansetron and CYP2D6 has ...   \n",
       "166  5-FU has many adverse drug effects when used s...   \n",
       "167                               G6PD, CYP2D6, UGT1A1   \n",
       "168                                   Sodium Valproate   \n",
       "171  Patients with the rs1801133 AA genotype and Ar...   \n",
       "\n",
       "                                       Expected_source  \n",
       "0    https://www.fda.gov/drugs/science-and-research...  \n",
       "1    https://www.pharmgkb.org/haplotype/PA165954769...  \n",
       "2    https://insilico-cyp.charite.de/SuperCYPsPred/...  \n",
       "3    Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...  \n",
       "4    https://www.fda.gov/drugs/science-and-research...  \n",
       "..                                                 ...  \n",
       "149                                                NaN  \n",
       "166                                                NaN  \n",
       "167                                                NaN  \n",
       "168                                                NaN  \n",
       "171                                                NaN  \n",
       "\n",
       "[134 rows x 3 columns]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"\n",
    "quests = Questions[Questions[\"Expected_response\"].notna()]\n",
    "quests.to_csv(\"/home/dhanushb/Wellytics/RAG_data/results/quests.csv\")\n",
    "quests\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.llms import Ollama\n",
    "\n",
    "MODELS = [\"mistral\", \"gemma\", \"llama2\", \"llama3\"] # also mixtral\n",
    "\n",
    "questions = quests[\"Questions\"].to_list()[96:]\n",
    "\n",
    "resps = pd.read_csv(\"/home/dhanushb/Wellytics/RAG_data/results/resps.csv\")\n",
    "\n",
    "for question in questions:\n",
    "    resp_dict = {}\n",
    "    for MODEL in MODELS:\n",
    "        model = Ollama(model=MODEL)\n",
    "        chain = setup | prompt | model | parser\n",
    "        response = chain.invoke(question)\n",
    "        context = [docs.page_content for docs in retriever.invoke(question)]\n",
    "        # Add response and context to the dataframe immediately after they are generated\n",
    "        resp_dict[MODEL+\"_resp\"] = [response]\n",
    "        resp_dict[MODEL+\"_cont\"] = [context]\n",
    "    resps_row = pd.DataFrame(resp_dict)\n",
    "    resps = pd.concat([resps, resps_row], ignore_index=True)\n",
    "    resps.to_csv(\"/home/dhanushb/Wellytics/RAG_data/results/resps.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "ename": "ArrowTypeError",
     "evalue": "Expected bytes, got a 'float' object",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mArrowTypeError\u001b[0m                            Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[36], line 23\u001b[0m\n\u001b[1;32m     15\u001b[0m data \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m     16\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mquestion\u001b[39m\u001b[38;5;124m\"\u001b[39m: quests[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mQuestions\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m.\u001b[39mto_list(),\n\u001b[1;32m     17\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124manswer\u001b[39m\u001b[38;5;124m\"\u001b[39m: quests[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mmistral_resp\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m.\u001b[39mto_list(),\n\u001b[1;32m     18\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mcontexts\u001b[39m\u001b[38;5;124m\"\u001b[39m: quests[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mcontext\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m.\u001b[39mto_list(),\n\u001b[1;32m     19\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mground_truth\u001b[39m\u001b[38;5;124m\"\u001b[39m: quests[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mExpected_response\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m.\u001b[39mto_list()\n\u001b[1;32m     20\u001b[0m }\n\u001b[1;32m     22\u001b[0m \u001b[38;5;66;03m# Convert dict to dataset\u001b[39;00m\n\u001b[0;32m---> 23\u001b[0m dataset \u001b[38;5;241m=\u001b[39m \u001b[43mDataset\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfrom_dict\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdata\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     25\u001b[0m result \u001b[38;5;241m=\u001b[39m evaluate(\n\u001b[1;32m     26\u001b[0m     dataset \u001b[38;5;241m=\u001b[39m dataset, \n\u001b[1;32m     27\u001b[0m     metrics\u001b[38;5;241m=\u001b[39m[\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     36\u001b[0m     ],\n\u001b[1;32m     37\u001b[0m )\n\u001b[1;32m     38\u001b[0m df \u001b[38;5;241m=\u001b[39m result\u001b[38;5;241m.\u001b[39mto_pandas()\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/datasets/arrow_dataset.py:963\u001b[0m, in \u001b[0;36mDataset.from_dict\u001b[0;34m(cls, mapping, features, info, split)\u001b[0m\n\u001b[1;32m    961\u001b[0m     arrow_typed_mapping[col] \u001b[38;5;241m=\u001b[39m data\n\u001b[1;32m    962\u001b[0m mapping \u001b[38;5;241m=\u001b[39m arrow_typed_mapping\n\u001b[0;32m--> 963\u001b[0m pa_table \u001b[38;5;241m=\u001b[39m \u001b[43mInMemoryTable\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfrom_pydict\u001b[49m\u001b[43m(\u001b[49m\u001b[43mmapping\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mmapping\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    964\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m info \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m    965\u001b[0m     info \u001b[38;5;241m=\u001b[39m DatasetInfo()\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/datasets/table.py:758\u001b[0m, in \u001b[0;36mInMemoryTable.from_pydict\u001b[0;34m(cls, *args, **kwargs)\u001b[0m\n\u001b[1;32m    742\u001b[0m \u001b[38;5;129m@classmethod\u001b[39m\n\u001b[1;32m    743\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mfrom_pydict\u001b[39m(\u001b[38;5;28mcls\u001b[39m, \u001b[38;5;241m*\u001b[39margs, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[1;32m    744\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m    745\u001b[0m \u001b[38;5;124;03m    Construct a Table from Arrow arrays or columns.\u001b[39;00m\n\u001b[1;32m    746\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    756\u001b[0m \u001b[38;5;124;03m        `datasets.table.Table`\u001b[39;00m\n\u001b[1;32m    757\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 758\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mcls\u001b[39m(\u001b[43mpa\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mTable\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfrom_pydict\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m)\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/table.pxi:1920\u001b[0m, in \u001b[0;36mpyarrow.lib._Tabular.from_pydict\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/table.pxi:5992\u001b[0m, in \u001b[0;36mpyarrow.lib._from_pydict\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/array.pxi:385\u001b[0m, in \u001b[0;36mpyarrow.lib.asarray\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/array.pxi:247\u001b[0m, in \u001b[0;36mpyarrow.lib.array\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/array.pxi:112\u001b[0m, in \u001b[0;36mpyarrow.lib._handle_arrow_array_protocol\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/datasets/arrow_writer.py:193\u001b[0m, in \u001b[0;36mTypedSequence.__arrow_array__\u001b[0;34m(self, type)\u001b[0m\n\u001b[1;32m    191\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    192\u001b[0m     trying_cast_to_python_objects \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mTrue\u001b[39;00m\n\u001b[0;32m--> 193\u001b[0m     out \u001b[38;5;241m=\u001b[39m \u001b[43mpa\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43marray\u001b[49m\u001b[43m(\u001b[49m\u001b[43mcast_to_python_objects\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdata\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43monly_1d_for_numpy\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mTrue\u001b[39;49;00m\u001b[43m)\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    194\u001b[0m \u001b[38;5;66;03m# use smaller integer precisions if possible\u001b[39;00m\n\u001b[1;32m    195\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtrying_int_optimization:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/array.pxi:355\u001b[0m, in \u001b[0;36mpyarrow.lib.array\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/array.pxi:42\u001b[0m, in \u001b[0;36mpyarrow.lib._sequence_to_array\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/error.pxi:154\u001b[0m, in \u001b[0;36mpyarrow.lib.pyarrow_internal_check_status\u001b[0;34m()\u001b[0m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pyarrow/error.pxi:91\u001b[0m, in \u001b[0;36mpyarrow.lib.check_status\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;31mArrowTypeError\u001b[0m: Expected bytes, got a 'float' object"
     ]
    }
   ],
   "source": [
    "from datasets import Dataset\n",
    "from ragas import evaluate\n",
    "from ragas.metrics.critique import harmfulness\n",
    "from ragas.metrics import (\n",
    "    faithfulness,\n",
    "    answer_relevancy,\n",
    "    context_recall,\n",
    "    context_precision,\n",
    "    context_entity_recall,\n",
    "    answer_similarity,\n",
    "    answer_correctness,\n",
    ")\n",
    "\n",
    "# To dict\n",
    "data = {\n",
    "    \"question\": quests[\"Questions\"].to_list(),\n",
    "    \"answer\": quests[\"mistral_resp\"].to_list(),\n",
    "    \"contexts\": quests[\"context\"].to_list(),\n",
    "    \"ground_truth\": quests[\"Expected_response\"].to_list()\n",
    "}\n",
    "\n",
    "# Convert dict to dataset\n",
    "dataset = Dataset.from_dict(data)\n",
    "\n",
    "result = evaluate(\n",
    "    dataset = dataset, \n",
    "    metrics=[\n",
    "        context_entity_recall,\n",
    "        context_precision,\n",
    "        context_recall,\n",
    "        faithfulness,\n",
    "        answer_relevancy,\n",
    "        answer_similarity,\n",
    "        answer_correctness,\n",
    "        harmfulness\n",
    "    ],\n",
    ")\n",
    "df = result.to_pandas()\n",
    "\n",
    "df\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Questions</th>\n",
       "      <th>Expected_response</th>\n",
       "      <th>Expected_source</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>I have a family history of depression and rece...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/genotypeResults?q=%7B...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>I have read that metformin leads to severe kid...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA450395/ove...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>I drink socially. I recently underwent pharmac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>I have seizures from the age of 5 years. I am ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>I have allergic rhinitis and take Reswas. Phar...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>I am a 62 year old female with history of 6 ye...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>Why is tramadol banned in sports?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>What are the antiseizure medications permissib...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>Can pharmacogenomics predict response to homeo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>Is blackcurrant extract natural antidiabetic a...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>Can MTHFR gene predict Vitamin B12 deficiency?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>What are the drugs that can affect my kidney f...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>Can atenolol prevent cardiac arrest?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>I am CYP2D6 ultrametaboliser, what are the dru...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>How does UGT1A1 impact irinotecan metabolism?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/literature/15139362</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>What is Steven Johnson syndrome and how is HLA...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>Why is G6PD important for diabetic patients?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA164712711/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>Can you predict warfarin response with only VK...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA451906/gui...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>What Pgx guidelines apply to my patient who is...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>What are the minimum set of genes I should tes...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/ampAllelesToTest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>My patient is identified with MYBPC3 variant a...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>Can I start PARP inhibitors for my patient wit...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>Can exondys be advised to limb girdle muscular...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>Will sitagliptin and metformin combination the...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>Can codiene be prescribed to a breast feeding ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>My patient has MTHFR rs1801133 AA genotype, sh...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>Can Criggler najar syndrome patient be prescri...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>Is there a role of CYP2C19 poor metabolism in ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>Can solid tumor test sample be subjected to ph...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>Can progesterone therapy help in iVF planning ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>Is there a pharmacogenomics guideline for meth...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>What are the safer alteratives for omeprazole ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA450704/gui...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>What is the tacrolimus dosage recommended for ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA451578/gui...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>What is indeterminate status in genotype?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.frontiersin.org/journals/pharmacol...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>Do ultrametabolisers require increased dosage ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>Are there any risk of oral contraceptive medic...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA452637/cli...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>How does CYP2C9 affect anticoagulants mode of ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/disease/PA166123298/l...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>Which are the antidiabetic medications influen...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA164712711/...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             Questions Expected_response  \\\n",
       "39   I have a family history of depression and rece...               NaN   \n",
       "47   I have read that metformin leads to severe kid...               NaN   \n",
       "48   I drink socially. I recently underwent pharmac...               NaN   \n",
       "52   I have seizures from the age of 5 years. I am ...               NaN   \n",
       "54   I have allergic rhinitis and take Reswas. Phar...               NaN   \n",
       "61   I am a 62 year old female with history of 6 ye...               NaN   \n",
       "62                   Why is tramadol banned in sports?               NaN   \n",
       "63   What are the antiseizure medications permissib...               NaN   \n",
       "64   Can pharmacogenomics predict response to homeo...               NaN   \n",
       "65   Is blackcurrant extract natural antidiabetic a...               NaN   \n",
       "66      Can MTHFR gene predict Vitamin B12 deficiency?               NaN   \n",
       "67   What are the drugs that can affect my kidney f...               NaN   \n",
       "68                Can atenolol prevent cardiac arrest?               NaN   \n",
       "70   I am CYP2D6 ultrametaboliser, what are the dru...               NaN   \n",
       "71       How does UGT1A1 impact irinotecan metabolism?               NaN   \n",
       "72   What is Steven Johnson syndrome and how is HLA...               NaN   \n",
       "73        Why is G6PD important for diabetic patients?               NaN   \n",
       "74   Can you predict warfarin response with only VK...               NaN   \n",
       "94   What Pgx guidelines apply to my patient who is...               NaN   \n",
       "113  What are the minimum set of genes I should tes...               NaN   \n",
       "150  My patient is identified with MYBPC3 variant a...               NaN   \n",
       "151  Can I start PARP inhibitors for my patient wit...               NaN   \n",
       "152  Can exondys be advised to limb girdle muscular...               NaN   \n",
       "153  Will sitagliptin and metformin combination the...               NaN   \n",
       "154  Can codiene be prescribed to a breast feeding ...               NaN   \n",
       "155  My patient has MTHFR rs1801133 AA genotype, sh...               NaN   \n",
       "156  Can Criggler najar syndrome patient be prescri...               NaN   \n",
       "157  Is there a role of CYP2C19 poor metabolism in ...               NaN   \n",
       "158  Can solid tumor test sample be subjected to ph...               NaN   \n",
       "159  Can progesterone therapy help in iVF planning ...               NaN   \n",
       "160  Is there a pharmacogenomics guideline for meth...               NaN   \n",
       "161  What are the safer alteratives for omeprazole ...               NaN   \n",
       "162  What is the tacrolimus dosage recommended for ...               NaN   \n",
       "163          What is indeterminate status in genotype?               NaN   \n",
       "164  Do ultrametabolisers require increased dosage ...               NaN   \n",
       "165  Are there any risk of oral contraceptive medic...               NaN   \n",
       "169  How does CYP2C9 affect anticoagulants mode of ...               NaN   \n",
       "170  Which are the antidiabetic medications influen...               NaN   \n",
       "\n",
       "                                       Expected_source  \n",
       "39   https://www.pharmgkb.org/genotypeResults?q=%7B...  \n",
       "47   https://www.pharmgkb.org/chemical/PA450395/ove...  \n",
       "48                                                 NaN  \n",
       "52                                                 NaN  \n",
       "54                                                 NaN  \n",
       "61                                                 NaN  \n",
       "62                                                 NaN  \n",
       "63                                                 NaN  \n",
       "64                                                 NaN  \n",
       "65                                                 NaN  \n",
       "66                                                 NaN  \n",
       "67                                                 NaN  \n",
       "68                                                 NaN  \n",
       "70                                                 NaN  \n",
       "71        https://www.pharmgkb.org/literature/15139362  \n",
       "72                                                 NaN  \n",
       "73   https://www.pharmgkb.org/chemical/PA164712711/...  \n",
       "74   https://www.pharmgkb.org/chemical/PA451906/gui...  \n",
       "94                                                 NaN  \n",
       "113          https://www.pharmgkb.org/ampAllelesToTest  \n",
       "150                                                NaN  \n",
       "151                                                NaN  \n",
       "152                                                NaN  \n",
       "153                                                NaN  \n",
       "154                                                NaN  \n",
       "155                                                NaN  \n",
       "156                                                NaN  \n",
       "157                                                NaN  \n",
       "158                                                NaN  \n",
       "159                                                NaN  \n",
       "160                                                NaN  \n",
       "161  https://www.pharmgkb.org/chemical/PA450704/gui...  \n",
       "162  https://www.pharmgkb.org/chemical/PA451578/gui...  \n",
       "163  https://www.frontiersin.org/journals/pharmacol...  \n",
       "164  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...  \n",
       "165  https://www.pharmgkb.org/chemical/PA452637/cli...  \n",
       "169  https://www.pharmgkb.org/disease/PA166123298/l...  \n",
       "170  https://www.pharmgkb.org/chemical/PA164712711/...  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"\n",
    "na_quests = Questions[Questions[\"Expected_response\"].isna()]\n",
    "na_quests.to_csv(\"/home/dhanushb/Wellytics/RAG_data/results/na_quests.csv\")\n",
    "na_quests\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_4757/1267869142.py:6: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  na_quests[\"Context\"] = cont\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Questions</th>\n",
       "      <th>Expected_response</th>\n",
       "      <th>Expected_source</th>\n",
       "      <th>Context</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>I have a family history of depression and rece...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/genotypeResults?q=%7B...</td>\n",
       "      <td>[CYP2D6 PM: imipramine\\n1544\\nThe risk of side...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>I have read that metformin leads to severe kid...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA450395/ove...</td>\n",
       "      <td>[4. \\nBatty JA et al. An investigation of CYP2...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>I drink socially. I recently underwent pharmac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[It is not known whether dose reduction in adv...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>I have seizures from the age of 5 years. I am ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Drug: escitalopram\\nGene1: CYP2C19\\nPhenotype...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>I have allergic rhinitis and take Reswas. Phar...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1:...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>I am a 62 year old female with history of 6 ye...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[{\"guideline\": {\"id\": \"PA166262261\", \"name\": \"...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>Why is tramadol banned in sports?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[2007;55:122-30.\\n9. \\nSlanar O et al. Miotic ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>What are the antiseizure medications permissib...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Drug: lansoprazole\\nGene1: CYP2C19\\nPhenotype...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>Can pharmacogenomics predict response to homeo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Literature:\\n1. \\nAmmar H et al. Clinical and...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>Is blackcurrant extract natural antidiabetic a...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Genetic variation may lead to a decrease in t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>Can MTHFR gene predict Vitamin B12 deficiency?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[1. \\nHuang J et al. Are gene polymorphisms re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>What are the drugs that can affect my kidney f...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[5. \\nWright DFB et al. The impact of diuretic...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>Can atenolol prevent cardiac arrest?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[31. \\nThijssen HH et al. The possession of th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>I am CYP2D6 ultrametaboliser, what are the dru...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[{\"citations\": [{\"id\": 15143242, \"title\": \"Cli...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>How does UGT1A1 impact irinotecan metabolism?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/literature/15139362</td>\n",
       "      <td>[CYP2D6 PM: imipramine\\n1544\\nThe risk of side...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>What is Steven Johnson syndrome and how is HLA...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[{\"guideline\": {\"id\": \"PA166265341\", \"name\": \"...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>Why is G6PD important for diabetic patients?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA164712711/...</td>\n",
       "      <td>[{\"citations\": [{\"id\": 15135762, \"title\": \"Exp...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>Can you predict warfarin response with only VK...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA451906/gui...</td>\n",
       "      <td>[{\"citations\": [{\"id\": 15096965, \"title\": \"Cli...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>What Pgx guidelines apply to my patient who is...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>What are the minimum set of genes I should tes...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/ampAllelesToTest</td>\n",
       "      <td>[{\"citations\": [{\"id\": 7283333, \"title\": \"Phar...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>My patient is identified with MYBPC3 variant a...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Eur J Clin Pharmacol 2007;63:1147-51.\\n5\\n. \\...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>Can I start PARP inhibitors for my patient wit...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[3\\n. \\nUesugi M et al. Impact of cytochrome P...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>Can exondys be advised to limb girdle muscular...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[{\"citations\": [], \"guideline\": {\"objCls\": \"Gu...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>Will sitagliptin and metformin combination the...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Pharmacol 2005;60:103-6.\\n7. \\nNiemi M et al....</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>Can codiene be prescribed to a breast feeding ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[controversy. Future Oncol 2014;10:107-22.\\n6\\...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>My patient has MTHFR rs1801133 AA genotype, sh...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[1. \\nHuang J et al. Are gene polymorphisms re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>Can Criggler najar syndrome patient be prescri...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[{\"citations\": [{\"id\": 15071835, \"title\": \"Rec...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>Is there a role of CYP2C19 poor metabolism in ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Drug: escitalopram\\nGene1: CYP2C19\\nPhenotype...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>Can solid tumor test sample be subjected to ph...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Drug: escitalopram\\nGene1: CYP2C19\\nPhenotype...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>Can progesterone therapy help in iVF planning ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1:...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>Is there a pharmacogenomics guideline for meth...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[{\"citations\": [], \"guideline\": {\"objCls\": \"Gu...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>What are the safer alteratives for omeprazole ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA450704/gui...</td>\n",
       "      <td>[Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1:...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>What is the tacrolimus dosage recommended for ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA451578/gui...</td>\n",
       "      <td>[{\"citations\": [{\"id\": 15109243, \"title\": \"Pha...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>What is indeterminate status in genotype?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.frontiersin.org/journals/pharmacol...</td>\n",
       "      <td>[{\"citations\": [], \"guideline\": {\"objCls\": \"Gu...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>Do ultrametabolisers require increased dosage ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "      <td>[significant increased risk of side effects.\\n...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>Are there any risk of oral contraceptive medic...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA452637/cli...</td>\n",
       "      <td>[Drug: rosuvastatin\\nGene1: ABCG2\\nPhenotype1:...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>How does CYP2C9 affect anticoagulants mode of ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/disease/PA166123298/l...</td>\n",
       "      <td>[acenocoumarol or phenprocoumon. Pharmacogenet...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>Which are the antidiabetic medications influen...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA164712711/...</td>\n",
       "      <td>[5. \\nWright DFB et al. The impact of diuretic...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             Questions Expected_response  \\\n",
       "39   I have a family history of depression and rece...               NaN   \n",
       "47   I have read that metformin leads to severe kid...               NaN   \n",
       "48   I drink socially. I recently underwent pharmac...               NaN   \n",
       "52   I have seizures from the age of 5 years. I am ...               NaN   \n",
       "54   I have allergic rhinitis and take Reswas. Phar...               NaN   \n",
       "61   I am a 62 year old female with history of 6 ye...               NaN   \n",
       "62                   Why is tramadol banned in sports?               NaN   \n",
       "63   What are the antiseizure medications permissib...               NaN   \n",
       "64   Can pharmacogenomics predict response to homeo...               NaN   \n",
       "65   Is blackcurrant extract natural antidiabetic a...               NaN   \n",
       "66      Can MTHFR gene predict Vitamin B12 deficiency?               NaN   \n",
       "67   What are the drugs that can affect my kidney f...               NaN   \n",
       "68                Can atenolol prevent cardiac arrest?               NaN   \n",
       "70   I am CYP2D6 ultrametaboliser, what are the dru...               NaN   \n",
       "71       How does UGT1A1 impact irinotecan metabolism?               NaN   \n",
       "72   What is Steven Johnson syndrome and how is HLA...               NaN   \n",
       "73        Why is G6PD important for diabetic patients?               NaN   \n",
       "74   Can you predict warfarin response with only VK...               NaN   \n",
       "94   What Pgx guidelines apply to my patient who is...               NaN   \n",
       "113  What are the minimum set of genes I should tes...               NaN   \n",
       "150  My patient is identified with MYBPC3 variant a...               NaN   \n",
       "151  Can I start PARP inhibitors for my patient wit...               NaN   \n",
       "152  Can exondys be advised to limb girdle muscular...               NaN   \n",
       "153  Will sitagliptin and metformin combination the...               NaN   \n",
       "154  Can codiene be prescribed to a breast feeding ...               NaN   \n",
       "155  My patient has MTHFR rs1801133 AA genotype, sh...               NaN   \n",
       "156  Can Criggler najar syndrome patient be prescri...               NaN   \n",
       "157  Is there a role of CYP2C19 poor metabolism in ...               NaN   \n",
       "158  Can solid tumor test sample be subjected to ph...               NaN   \n",
       "159  Can progesterone therapy help in iVF planning ...               NaN   \n",
       "160  Is there a pharmacogenomics guideline for meth...               NaN   \n",
       "161  What are the safer alteratives for omeprazole ...               NaN   \n",
       "162  What is the tacrolimus dosage recommended for ...               NaN   \n",
       "163          What is indeterminate status in genotype?               NaN   \n",
       "164  Do ultrametabolisers require increased dosage ...               NaN   \n",
       "165  Are there any risk of oral contraceptive medic...               NaN   \n",
       "169  How does CYP2C9 affect anticoagulants mode of ...               NaN   \n",
       "170  Which are the antidiabetic medications influen...               NaN   \n",
       "\n",
       "                                       Expected_source  \\\n",
       "39   https://www.pharmgkb.org/genotypeResults?q=%7B...   \n",
       "47   https://www.pharmgkb.org/chemical/PA450395/ove...   \n",
       "48                                                 NaN   \n",
       "52                                                 NaN   \n",
       "54                                                 NaN   \n",
       "61                                                 NaN   \n",
       "62                                                 NaN   \n",
       "63                                                 NaN   \n",
       "64                                                 NaN   \n",
       "65                                                 NaN   \n",
       "66                                                 NaN   \n",
       "67                                                 NaN   \n",
       "68                                                 NaN   \n",
       "70                                                 NaN   \n",
       "71        https://www.pharmgkb.org/literature/15139362   \n",
       "72                                                 NaN   \n",
       "73   https://www.pharmgkb.org/chemical/PA164712711/...   \n",
       "74   https://www.pharmgkb.org/chemical/PA451906/gui...   \n",
       "94                                                 NaN   \n",
       "113          https://www.pharmgkb.org/ampAllelesToTest   \n",
       "150                                                NaN   \n",
       "151                                                NaN   \n",
       "152                                                NaN   \n",
       "153                                                NaN   \n",
       "154                                                NaN   \n",
       "155                                                NaN   \n",
       "156                                                NaN   \n",
       "157                                                NaN   \n",
       "158                                                NaN   \n",
       "159                                                NaN   \n",
       "160                                                NaN   \n",
       "161  https://www.pharmgkb.org/chemical/PA450704/gui...   \n",
       "162  https://www.pharmgkb.org/chemical/PA451578/gui...   \n",
       "163  https://www.frontiersin.org/journals/pharmacol...   \n",
       "164  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...   \n",
       "165  https://www.pharmgkb.org/chemical/PA452637/cli...   \n",
       "169  https://www.pharmgkb.org/disease/PA166123298/l...   \n",
       "170  https://www.pharmgkb.org/chemical/PA164712711/...   \n",
       "\n",
       "                                               Context  \n",
       "39   [CYP2D6 PM: imipramine\\n1544\\nThe risk of side...  \n",
       "47   [4. \\nBatty JA et al. An investigation of CYP2...  \n",
       "48   [It is not known whether dose reduction in adv...  \n",
       "52   [Drug: escitalopram\\nGene1: CYP2C19\\nPhenotype...  \n",
       "54   [Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1:...  \n",
       "61   [{\"guideline\": {\"id\": \"PA166262261\", \"name\": \"...  \n",
       "62   [2007;55:122-30.\\n9. \\nSlanar O et al. Miotic ...  \n",
       "63   [Drug: lansoprazole\\nGene1: CYP2C19\\nPhenotype...  \n",
       "64   [Literature:\\n1. \\nAmmar H et al. Clinical and...  \n",
       "65   [Genetic variation may lead to a decrease in t...  \n",
       "66   [1. \\nHuang J et al. Are gene polymorphisms re...  \n",
       "67   [5. \\nWright DFB et al. The impact of diuretic...  \n",
       "68   [31. \\nThijssen HH et al. The possession of th...  \n",
       "70   [{\"citations\": [{\"id\": 15143242, \"title\": \"Cli...  \n",
       "71   [CYP2D6 PM: imipramine\\n1544\\nThe risk of side...  \n",
       "72   [{\"guideline\": {\"id\": \"PA166265341\", \"name\": \"...  \n",
       "73   [{\"citations\": [{\"id\": 15135762, \"title\": \"Exp...  \n",
       "74   [{\"citations\": [{\"id\": 15096965, \"title\": \"Cli...  \n",
       "94   [Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype1...  \n",
       "113  [{\"citations\": [{\"id\": 7283333, \"title\": \"Phar...  \n",
       "150  [Eur J Clin Pharmacol 2007;63:1147-51.\\n5\\n. \\...  \n",
       "151  [3\\n. \\nUesugi M et al. Impact of cytochrome P...  \n",
       "152  [{\"citations\": [], \"guideline\": {\"objCls\": \"Gu...  \n",
       "153  [Pharmacol 2005;60:103-6.\\n7. \\nNiemi M et al....  \n",
       "154  [controversy. Future Oncol 2014;10:107-22.\\n6\\...  \n",
       "155  [1. \\nHuang J et al. Are gene polymorphisms re...  \n",
       "156  [{\"citations\": [{\"id\": 15071835, \"title\": \"Rec...  \n",
       "157  [Drug: escitalopram\\nGene1: CYP2C19\\nPhenotype...  \n",
       "158  [Drug: escitalopram\\nGene1: CYP2C19\\nPhenotype...  \n",
       "159  [Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1:...  \n",
       "160  [{\"citations\": [], \"guideline\": {\"objCls\": \"Gu...  \n",
       "161  [Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1:...  \n",
       "162  [{\"citations\": [{\"id\": 15109243, \"title\": \"Pha...  \n",
       "163  [{\"citations\": [], \"guideline\": {\"objCls\": \"Gu...  \n",
       "164  [significant increased risk of side effects.\\n...  \n",
       "165  [Drug: rosuvastatin\\nGene1: ABCG2\\nPhenotype1:...  \n",
       "169  [acenocoumarol or phenprocoumon. Pharmacogenet...  \n",
       "170  [5. \\nWright DFB et al. The impact of diuretic...  "
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "questions = na_quests[\"Questions\"].to_list()\n",
    "\n",
    "cont = []\n",
    "for question in questions:\n",
    "    cont.append([docs.page_content for docs in retriever.invoke(question)])\n",
    "na_quests[\"Context\"] = cont\n",
    "\n",
    "na_quests.to_csv(\"/home/dhanushb/Wellytics/RAG_data/results/na_quests.csv\")\n",
    "\n",
    "na_quests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "ename": "ProtocolError",
     "evalue": "Failed to connect; did you specify the correct index name?",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mRemoteDisconnected\u001b[0m                        Traceback (most recent call last)",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/connectionpool.py:793\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, preload_content, decode_content, **response_kw)\u001b[0m\n\u001b[1;32m    792\u001b[0m \u001b[38;5;66;03m# Make the request on the HTTPConnection object\u001b[39;00m\n\u001b[0;32m--> 793\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    794\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    795\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    796\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    797\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    798\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    799\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    800\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    801\u001b[0m \u001b[43m    \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mretries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    802\u001b[0m \u001b[43m    \u001b[49m\u001b[43mresponse_conn\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mresponse_conn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    803\u001b[0m \u001b[43m    \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mpreload_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    804\u001b[0m \u001b[43m    \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mdecode_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    805\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mresponse_kw\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    806\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    808\u001b[0m \u001b[38;5;66;03m# Everything went great!\u001b[39;00m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/connectionpool.py:537\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, body, headers, retries, timeout, chunked, response_conn, preload_content, decode_content, enforce_content_length)\u001b[0m\n\u001b[1;32m    536\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 537\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgetresponse\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    538\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (BaseSSLError, \u001b[38;5;167;01mOSError\u001b[39;00m) \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/connection.py:466\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    465\u001b[0m \u001b[38;5;66;03m# Get the response from http.client.HTTPConnection\u001b[39;00m\n\u001b[0;32m--> 466\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43msuper\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgetresponse\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    468\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:1375\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1374\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1375\u001b[0m     \u001b[43mresponse\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbegin\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1376\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:318\u001b[0m, in \u001b[0;36mHTTPResponse.begin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    317\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 318\u001b[0m     version, status, reason \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_read_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    319\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m status \u001b[38;5;241m!=\u001b[39m CONTINUE:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:287\u001b[0m, in \u001b[0;36mHTTPResponse._read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    284\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m line:\n\u001b[1;32m    285\u001b[0m     \u001b[38;5;66;03m# Presumably, the server closed the connection before\u001b[39;00m\n\u001b[1;32m    286\u001b[0m     \u001b[38;5;66;03m# sending a valid response.\u001b[39;00m\n\u001b[0;32m--> 287\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m RemoteDisconnected(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mRemote end closed connection without\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    288\u001b[0m                              \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m response\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    289\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n",
      "\u001b[0;31mRemoteDisconnected\u001b[0m: Remote end closed connection without response",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mProtocolError\u001b[0m                             Traceback (most recent call last)",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/utils/error_handling.py:10\u001b[0m, in \u001b[0;36mvalidate_and_convert_errors.<locals>.inner_func\u001b[0;34m(*args, **kwargs)\u001b[0m\n\u001b[1;32m      9\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m---> 10\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mfunc\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     11\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m MaxRetryError \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/data/index.py:399\u001b[0m, in \u001b[0;36mIndex.query\u001b[0;34m(self, top_k, vector, id, namespace, filter, include_values, include_metadata, sparse_vector, *args, **kwargs)\u001b[0m\n\u001b[1;32m    386\u001b[0m args_dict \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_parse_non_empty_args(\n\u001b[1;32m    387\u001b[0m     [\n\u001b[1;32m    388\u001b[0m         (\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mvector\u001b[39m\u001b[38;5;124m\"\u001b[39m, vector),\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    397\u001b[0m     ]\n\u001b[1;32m    398\u001b[0m )\n\u001b[0;32m--> 399\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_vector_api\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mquery\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    400\u001b[0m \u001b[43m    \u001b[49m\u001b[43mQueryRequest\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    401\u001b[0m \u001b[43m        \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs_dict\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    402\u001b[0m \u001b[43m        \u001b[49m\u001b[43m_check_type\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_check_type\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    403\u001b[0m \u001b[43m        \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43m{\u001b[49m\u001b[43mk\u001b[49m\u001b[43m:\u001b[49m\u001b[43m \u001b[49m\u001b[43mv\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mfor\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mk\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mv\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;129;43;01min\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mkwargs\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mitems\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mif\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mk\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;129;43;01mnot\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;129;43;01min\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43m_OPENAPI_ENDPOINT_PARAMS\u001b[49m\u001b[43m}\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    404\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    405\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43m{\u001b[49m\u001b[43mk\u001b[49m\u001b[43m:\u001b[49m\u001b[43m \u001b[49m\u001b[43mv\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mfor\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mk\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mv\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;129;43;01min\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mkwargs\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mitems\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mif\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mk\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;129;43;01min\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43m_OPENAPI_ENDPOINT_PARAMS\u001b[49m\u001b[43m}\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    406\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    407\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m parse_query_response(response)\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/api_client.py:772\u001b[0m, in \u001b[0;36mEndpoint.__call__\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m    762\u001b[0m \u001b[38;5;250m\u001b[39m\u001b[38;5;124;03m\"\"\" This method is invoked when endpoints are called\u001b[39;00m\n\u001b[1;32m    763\u001b[0m \u001b[38;5;124;03mExample:\u001b[39;00m\n\u001b[1;32m    764\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    770\u001b[0m \n\u001b[1;32m    771\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[0;32m--> 772\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcallable\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/api/data_plane_api.py:844\u001b[0m, in \u001b[0;36mDataPlaneApi.__init__.<locals>.__query\u001b[0;34m(self, query_request, **kwargs)\u001b[0m\n\u001b[1;32m    842\u001b[0m kwargs[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mquery_request\u001b[39m\u001b[38;5;124m'\u001b[39m] \u001b[38;5;241m=\u001b[39m \\\n\u001b[1;32m    843\u001b[0m     query_request\n\u001b[0;32m--> 844\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcall_with_http_info\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/api_client.py:834\u001b[0m, in \u001b[0;36mEndpoint.call_with_http_info\u001b[0;34m(self, **kwargs)\u001b[0m\n\u001b[1;32m    832\u001b[0m     params[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mheader\u001b[39m\u001b[38;5;124m'\u001b[39m][\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mContent-Type\u001b[39m\u001b[38;5;124m'\u001b[39m] \u001b[38;5;241m=\u001b[39m header_list\n\u001b[0;32m--> 834\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mapi_client\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcall_api\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    835\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msettings\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mendpoint_path\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msettings\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mhttp_method\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    836\u001b[0m \u001b[43m    \u001b[49m\u001b[43mparams\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mpath\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    837\u001b[0m \u001b[43m    \u001b[49m\u001b[43mparams\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mquery\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    838\u001b[0m \u001b[43m    \u001b[49m\u001b[43mparams\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mheader\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    839\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparams\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mbody\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    840\u001b[0m \u001b[43m    \u001b[49m\u001b[43mpost_params\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparams\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mform\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    841\u001b[0m \u001b[43m    \u001b[49m\u001b[43mfiles\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparams\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mfile\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    842\u001b[0m \u001b[43m    \u001b[49m\u001b[43mresponse_type\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msettings\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mresponse_type\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    843\u001b[0m \u001b[43m    \u001b[49m\u001b[43mauth_settings\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msettings\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mauth\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    844\u001b[0m \u001b[43m    \u001b[49m\u001b[43masync_req\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43masync_req\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    845\u001b[0m \u001b[43m    \u001b[49m\u001b[43m_check_type\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43m_check_return_type\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    846\u001b[0m \u001b[43m    \u001b[49m\u001b[43m_return_http_data_only\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43m_return_http_data_only\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    847\u001b[0m \u001b[43m    \u001b[49m\u001b[43m_preload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43m_preload_content\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    848\u001b[0m \u001b[43m    \u001b[49m\u001b[43m_request_timeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43m_request_timeout\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    849\u001b[0m \u001b[43m    \u001b[49m\u001b[43m_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_host\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    850\u001b[0m \u001b[43m    \u001b[49m\u001b[43mcollection_formats\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparams\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mcollection_format\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/api_client.py:409\u001b[0m, in \u001b[0;36mApiClient.call_api\u001b[0;34m(self, resource_path, method, path_params, query_params, header_params, body, post_params, files, response_type, auth_settings, async_req, _return_http_data_only, collection_formats, _preload_content, _request_timeout, _host, _check_type)\u001b[0m\n\u001b[1;32m    408\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m async_req:\n\u001b[0;32m--> 409\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m__call_api\u001b[49m\u001b[43m(\u001b[49m\u001b[43mresource_path\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    410\u001b[0m \u001b[43m                           \u001b[49m\u001b[43mpath_params\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mquery_params\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheader_params\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    411\u001b[0m \u001b[43m                           \u001b[49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mpost_params\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mfiles\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    412\u001b[0m \u001b[43m                           \u001b[49m\u001b[43mresponse_type\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mauth_settings\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    413\u001b[0m \u001b[43m                           \u001b[49m\u001b[43m_return_http_data_only\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mcollection_formats\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    414\u001b[0m \u001b[43m                           \u001b[49m\u001b[43m_preload_content\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_request_timeout\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_host\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    415\u001b[0m \u001b[43m                           \u001b[49m\u001b[43m_check_type\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    417\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mpool\u001b[38;5;241m.\u001b[39mapply_async(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m__call_api, (resource_path,\n\u001b[1;32m    418\u001b[0m                                                method, path_params,\n\u001b[1;32m    419\u001b[0m                                                query_params,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    427\u001b[0m                                                _request_timeout,\n\u001b[1;32m    428\u001b[0m                                                _host, _check_type))\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/api_client.py:196\u001b[0m, in \u001b[0;36mApiClient.__call_api\u001b[0;34m(self, resource_path, method, path_params, query_params, header_params, body, post_params, files, response_type, auth_settings, _return_http_data_only, collection_formats, _preload_content, _request_timeout, _host, _check_type)\u001b[0m\n\u001b[1;32m    194\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    195\u001b[0m     \u001b[38;5;66;03m# perform request and return response\u001b[39;00m\n\u001b[0;32m--> 196\u001b[0m     response_data \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    197\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mquery_params\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mquery_params\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheader_params\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    198\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpost_params\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mpost_params\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    199\u001b[0m \u001b[43m        \u001b[49m\u001b[43m_preload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_preload_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    200\u001b[0m \u001b[43m        \u001b[49m\u001b[43m_request_timeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_request_timeout\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    201\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m PineconeApiException \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/api_client.py:455\u001b[0m, in \u001b[0;36mApiClient.request\u001b[0;34m(self, method, url, query_params, headers, post_params, body, _preload_content, _request_timeout)\u001b[0m\n\u001b[1;32m    454\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m method \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mPOST\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[0;32m--> 455\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrest_client\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mPOST\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    456\u001b[0m \u001b[43m                                 \u001b[49m\u001b[43mquery_params\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mquery_params\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    457\u001b[0m \u001b[43m                                 \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    458\u001b[0m \u001b[43m                                 \u001b[49m\u001b[43mpost_params\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mpost_params\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    459\u001b[0m \u001b[43m                                 \u001b[49m\u001b[43m_preload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_preload_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    460\u001b[0m \u001b[43m                                 \u001b[49m\u001b[43m_request_timeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_request_timeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    461\u001b[0m \u001b[43m                                 \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    462\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m method \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mPUT\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/rest.py:302\u001b[0m, in \u001b[0;36mRESTClientObject.POST\u001b[0;34m(self, url, headers, query_params, post_params, body, _preload_content, _request_timeout)\u001b[0m\n\u001b[1;32m    300\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mPOST\u001b[39m(\u001b[38;5;28mself\u001b[39m, url, headers\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, query_params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, post_params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m,\n\u001b[1;32m    301\u001b[0m          body\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, _preload_content\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m, _request_timeout\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m):\n\u001b[0;32m--> 302\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mPOST\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    303\u001b[0m \u001b[43m                        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    304\u001b[0m \u001b[43m                        \u001b[49m\u001b[43mquery_params\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mquery_params\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    305\u001b[0m \u001b[43m                        \u001b[49m\u001b[43mpost_params\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mpost_params\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    306\u001b[0m \u001b[43m                        \u001b[49m\u001b[43m_preload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_preload_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    307\u001b[0m \u001b[43m                        \u001b[49m\u001b[43m_request_timeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_request_timeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    308\u001b[0m \u001b[43m                        \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/core/client/rest.py:179\u001b[0m, in \u001b[0;36mRESTClientObject.request\u001b[0;34m(self, method, url, query_params, headers, body, post_params, _preload_content, _request_timeout)\u001b[0m\n\u001b[1;32m    178\u001b[0m         request_body \u001b[38;5;241m=\u001b[39m json\u001b[38;5;241m.\u001b[39mdumps(body)\n\u001b[0;32m--> 179\u001b[0m     r \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mpool_manager\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    180\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    181\u001b[0m \u001b[43m        \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest_body\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    182\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m_preload_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    183\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    184\u001b[0m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    185\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m headers[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mContent-Type\u001b[39m\u001b[38;5;124m'\u001b[39m] \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mapplication/x-www-form-urlencoded\u001b[39m\u001b[38;5;124m'\u001b[39m:  \u001b[38;5;66;03m# noqa: E501\u001b[39;00m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/_request_methods.py:144\u001b[0m, in \u001b[0;36mRequestMethods.request\u001b[0;34m(self, method, url, body, fields, headers, json, **urlopen_kw)\u001b[0m\n\u001b[1;32m    143\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 144\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest_encode_body\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    145\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mfields\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mfields\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43murlopen_kw\u001b[49m\n\u001b[1;32m    146\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/_request_methods.py:279\u001b[0m, in \u001b[0;36mRequestMethods.request_encode_body\u001b[0;34m(self, method, url, fields, headers, encode_multipart, multipart_boundary, **urlopen_kw)\u001b[0m\n\u001b[1;32m    277\u001b[0m extra_kw\u001b[38;5;241m.\u001b[39mupdate(urlopen_kw)\n\u001b[0;32m--> 279\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mextra_kw\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/poolmanager.py:444\u001b[0m, in \u001b[0;36mPoolManager.urlopen\u001b[0;34m(self, method, url, redirect, **kw)\u001b[0m\n\u001b[1;32m    443\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 444\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mu\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest_uri\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkw\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    446\u001b[0m redirect_location \u001b[38;5;241m=\u001b[39m redirect \u001b[38;5;129;01mand\u001b[39;00m response\u001b[38;5;241m.\u001b[39mget_redirect_location()\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/connectionpool.py:847\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, preload_content, decode_content, **response_kw)\u001b[0m\n\u001b[1;32m    845\u001b[0m     new_e \u001b[38;5;241m=\u001b[39m ProtocolError(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection aborted.\u001b[39m\u001b[38;5;124m\"\u001b[39m, new_e)\n\u001b[0;32m--> 847\u001b[0m retries \u001b[38;5;241m=\u001b[39m \u001b[43mretries\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mincrement\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    848\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mnew_e\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_pool\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43msys\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mexc_info\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m2\u001b[39;49m\u001b[43m]\u001b[49m\n\u001b[1;32m    849\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    850\u001b[0m retries\u001b[38;5;241m.\u001b[39msleep()\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/util/retry.py:470\u001b[0m, in \u001b[0;36mRetry.increment\u001b[0;34m(self, method, url, response, error, _pool, _stacktrace)\u001b[0m\n\u001b[1;32m    469\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m read \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mFalse\u001b[39;00m \u001b[38;5;129;01mor\u001b[39;00m method \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mor\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_is_method_retryable(method):\n\u001b[0;32m--> 470\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[43mreraise\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mtype\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43merror\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    471\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m read \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/util/util.py:38\u001b[0m, in \u001b[0;36mreraise\u001b[0;34m(tp, value, tb)\u001b[0m\n\u001b[1;32m     37\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m value\u001b[38;5;241m.\u001b[39m__traceback__ \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m tb:\n\u001b[0;32m---> 38\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m value\u001b[38;5;241m.\u001b[39mwith_traceback(tb)\n\u001b[1;32m     39\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m value\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/connectionpool.py:793\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, preload_content, decode_content, **response_kw)\u001b[0m\n\u001b[1;32m    792\u001b[0m \u001b[38;5;66;03m# Make the request on the HTTPConnection object\u001b[39;00m\n\u001b[0;32m--> 793\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    794\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    795\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    796\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    797\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    798\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    799\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    800\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    801\u001b[0m \u001b[43m    \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mretries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    802\u001b[0m \u001b[43m    \u001b[49m\u001b[43mresponse_conn\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mresponse_conn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    803\u001b[0m \u001b[43m    \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mpreload_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    804\u001b[0m \u001b[43m    \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mdecode_content\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    805\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mresponse_kw\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    806\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    808\u001b[0m \u001b[38;5;66;03m# Everything went great!\u001b[39;00m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/connectionpool.py:537\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, body, headers, retries, timeout, chunked, response_conn, preload_content, decode_content, enforce_content_length)\u001b[0m\n\u001b[1;32m    536\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 537\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgetresponse\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    538\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (BaseSSLError, \u001b[38;5;167;01mOSError\u001b[39;00m) \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/urllib3/connection.py:466\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    465\u001b[0m \u001b[38;5;66;03m# Get the response from http.client.HTTPConnection\u001b[39;00m\n\u001b[0;32m--> 466\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43msuper\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgetresponse\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    468\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:1375\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1374\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1375\u001b[0m     \u001b[43mresponse\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbegin\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1376\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:318\u001b[0m, in \u001b[0;36mHTTPResponse.begin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    317\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 318\u001b[0m     version, status, reason \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_read_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    319\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m status \u001b[38;5;241m!=\u001b[39m CONTINUE:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:287\u001b[0m, in \u001b[0;36mHTTPResponse._read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    284\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m line:\n\u001b[1;32m    285\u001b[0m     \u001b[38;5;66;03m# Presumably, the server closed the connection before\u001b[39;00m\n\u001b[1;32m    286\u001b[0m     \u001b[38;5;66;03m# sending a valid response.\u001b[39;00m\n\u001b[0;32m--> 287\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m RemoteDisconnected(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mRemote end closed connection without\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    288\u001b[0m                              \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m response\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    289\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n",
      "\u001b[0;31mProtocolError\u001b[0m: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[0;31mProtocolError\u001b[0m                             Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[12], line 12\u001b[0m\n\u001b[1;32m     10\u001b[0m i \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m1\u001b[39m\n\u001b[1;32m     11\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m question \u001b[38;5;129;01min\u001b[39;00m questions:\n\u001b[0;32m---> 12\u001b[0m     resp\u001b[38;5;241m.\u001b[39mappend(\u001b[43mchain\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43minvoke\u001b[49m\u001b[43m(\u001b[49m\u001b[43mquestion\u001b[49m\u001b[43m)\u001b[49m)\n\u001b[1;32m     13\u001b[0m     na_quests[MODEL \u001b[38;5;241m+\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m_resp\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m=\u001b[39m resp \u001b[38;5;241m+\u001b[39m [np\u001b[38;5;241m.\u001b[39mnan] \u001b[38;5;241m*\u001b[39m (n \u001b[38;5;241m-\u001b[39m i)\n\u001b[1;32m     14\u001b[0m     na_quests\u001b[38;5;241m.\u001b[39mto_csv(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m/home/dhanushb/Wellytics/RAG_data/results/na_quests.csv\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/runnables/base.py:2393\u001b[0m, in \u001b[0;36mRunnableSequence.invoke\u001b[0;34m(self, input, config)\u001b[0m\n\u001b[1;32m   2391\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m   2392\u001b[0m     \u001b[38;5;28;01mfor\u001b[39;00m i, step \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28menumerate\u001b[39m(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msteps):\n\u001b[0;32m-> 2393\u001b[0m         \u001b[38;5;28minput\u001b[39m \u001b[38;5;241m=\u001b[39m \u001b[43mstep\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43minvoke\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m   2394\u001b[0m \u001b[43m            \u001b[49m\u001b[38;5;28;43minput\u001b[39;49m\u001b[43m,\u001b[49m\n\u001b[1;32m   2395\u001b[0m \u001b[43m            \u001b[49m\u001b[38;5;66;43;03m# mark each step as a child run\u001b[39;49;00m\n\u001b[1;32m   2396\u001b[0m \u001b[43m            \u001b[49m\u001b[43mpatch_config\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m   2397\u001b[0m \u001b[43m                \u001b[49m\u001b[43mconfig\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mcallbacks\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrun_manager\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_child\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43mf\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mseq:step:\u001b[39;49m\u001b[38;5;132;43;01m{\u001b[39;49;00m\u001b[43mi\u001b[49m\u001b[38;5;241;43m+\u001b[39;49m\u001b[38;5;241;43m1\u001b[39;49m\u001b[38;5;132;43;01m}\u001b[39;49;00m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m   2398\u001b[0m \u001b[43m            \u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m   2399\u001b[0m \u001b[43m        \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   2400\u001b[0m \u001b[38;5;66;03m# finish the root run\u001b[39;00m\n\u001b[1;32m   2401\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/runnables/base.py:3036\u001b[0m, in \u001b[0;36mRunnableParallel.invoke\u001b[0;34m(self, input, config)\u001b[0m\n\u001b[1;32m   3023\u001b[0m     \u001b[38;5;28;01mwith\u001b[39;00m get_executor_for_config(config) \u001b[38;5;28;01mas\u001b[39;00m executor:\n\u001b[1;32m   3024\u001b[0m         futures \u001b[38;5;241m=\u001b[39m [\n\u001b[1;32m   3025\u001b[0m             executor\u001b[38;5;241m.\u001b[39msubmit(\n\u001b[1;32m   3026\u001b[0m                 step\u001b[38;5;241m.\u001b[39minvoke,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   3034\u001b[0m             \u001b[38;5;28;01mfor\u001b[39;00m key, step \u001b[38;5;129;01min\u001b[39;00m steps\u001b[38;5;241m.\u001b[39mitems()\n\u001b[1;32m   3035\u001b[0m         ]\n\u001b[0;32m-> 3036\u001b[0m         output \u001b[38;5;241m=\u001b[39m {key: future\u001b[38;5;241m.\u001b[39mresult() \u001b[38;5;28;01mfor\u001b[39;00m key, future \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mzip\u001b[39m(steps, futures)}\n\u001b[1;32m   3037\u001b[0m \u001b[38;5;66;03m# finish the root run\u001b[39;00m\n\u001b[1;32m   3038\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/runnables/base.py:3036\u001b[0m, in \u001b[0;36m<dictcomp>\u001b[0;34m(.0)\u001b[0m\n\u001b[1;32m   3023\u001b[0m     \u001b[38;5;28;01mwith\u001b[39;00m get_executor_for_config(config) \u001b[38;5;28;01mas\u001b[39;00m executor:\n\u001b[1;32m   3024\u001b[0m         futures \u001b[38;5;241m=\u001b[39m [\n\u001b[1;32m   3025\u001b[0m             executor\u001b[38;5;241m.\u001b[39msubmit(\n\u001b[1;32m   3026\u001b[0m                 step\u001b[38;5;241m.\u001b[39minvoke,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m   3034\u001b[0m             \u001b[38;5;28;01mfor\u001b[39;00m key, step \u001b[38;5;129;01min\u001b[39;00m steps\u001b[38;5;241m.\u001b[39mitems()\n\u001b[1;32m   3035\u001b[0m         ]\n\u001b[0;32m-> 3036\u001b[0m         output \u001b[38;5;241m=\u001b[39m {key: \u001b[43mfuture\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mresult\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m \u001b[38;5;28;01mfor\u001b[39;00m key, future \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mzip\u001b[39m(steps, futures)}\n\u001b[1;32m   3037\u001b[0m \u001b[38;5;66;03m# finish the root run\u001b[39;00m\n\u001b[1;32m   3038\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m/usr/lib/python3.10/concurrent/futures/_base.py:458\u001b[0m, in \u001b[0;36mFuture.result\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m    456\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m CancelledError()\n\u001b[1;32m    457\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_state \u001b[38;5;241m==\u001b[39m FINISHED:\n\u001b[0;32m--> 458\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m__get_result\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    459\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    460\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mTimeoutError\u001b[39;00m()\n",
      "File \u001b[0;32m/usr/lib/python3.10/concurrent/futures/_base.py:403\u001b[0m, in \u001b[0;36mFuture.__get_result\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    401\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_exception:\n\u001b[1;32m    402\u001b[0m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 403\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_exception\n\u001b[1;32m    404\u001b[0m     \u001b[38;5;28;01mfinally\u001b[39;00m:\n\u001b[1;32m    405\u001b[0m         \u001b[38;5;66;03m# Break a reference cycle with the exception in self._exception\u001b[39;00m\n\u001b[1;32m    406\u001b[0m         \u001b[38;5;28mself\u001b[39m \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m\n",
      "File \u001b[0;32m/usr/lib/python3.10/concurrent/futures/thread.py:58\u001b[0m, in \u001b[0;36m_WorkItem.run\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     55\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m\n\u001b[1;32m     57\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m---> 58\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfn\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     59\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m exc:\n\u001b[1;32m     60\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mfuture\u001b[38;5;241m.\u001b[39mset_exception(exc)\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/retrievers.py:194\u001b[0m, in \u001b[0;36mBaseRetriever.invoke\u001b[0;34m(self, input, config, **kwargs)\u001b[0m\n\u001b[1;32m    175\u001b[0m \u001b[38;5;250m\u001b[39m\u001b[38;5;124;03m\"\"\"Invoke the retriever to get relevant documents.\u001b[39;00m\n\u001b[1;32m    176\u001b[0m \n\u001b[1;32m    177\u001b[0m \u001b[38;5;124;03mMain entry point for synchronous retriever invocations.\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    191\u001b[0m \u001b[38;5;124;03m    retriever.invoke(\"query\")\u001b[39;00m\n\u001b[1;32m    192\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m    193\u001b[0m config \u001b[38;5;241m=\u001b[39m ensure_config(config)\n\u001b[0;32m--> 194\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_relevant_documents\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    195\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;28;43minput\u001b[39;49m\u001b[43m,\u001b[49m\n\u001b[1;32m    196\u001b[0m \u001b[43m    \u001b[49m\u001b[43mcallbacks\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mconfig\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mcallbacks\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    197\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtags\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mconfig\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mtags\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    198\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmetadata\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mconfig\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mmetadata\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    199\u001b[0m \u001b[43m    \u001b[49m\u001b[43mrun_name\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mconfig\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mrun_name\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    200\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    201\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/_api/deprecation.py:148\u001b[0m, in \u001b[0;36mdeprecated.<locals>.deprecate.<locals>.warning_emitting_wrapper\u001b[0;34m(*args, **kwargs)\u001b[0m\n\u001b[1;32m    146\u001b[0m     warned \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mTrue\u001b[39;00m\n\u001b[1;32m    147\u001b[0m     emit_warning()\n\u001b[0;32m--> 148\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mwrapped\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/retrievers.py:323\u001b[0m, in \u001b[0;36mBaseRetriever.get_relevant_documents\u001b[0;34m(self, query, callbacks, tags, metadata, run_name, **kwargs)\u001b[0m\n\u001b[1;32m    321\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    322\u001b[0m     run_manager\u001b[38;5;241m.\u001b[39mon_retriever_error(e)\n\u001b[0;32m--> 323\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m e\n\u001b[1;32m    324\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    325\u001b[0m     run_manager\u001b[38;5;241m.\u001b[39mon_retriever_end(\n\u001b[1;32m    326\u001b[0m         result,\n\u001b[1;32m    327\u001b[0m     )\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/retrievers.py:316\u001b[0m, in \u001b[0;36mBaseRetriever.get_relevant_documents\u001b[0;34m(self, query, callbacks, tags, metadata, run_name, **kwargs)\u001b[0m\n\u001b[1;32m    314\u001b[0m _kwargs \u001b[38;5;241m=\u001b[39m kwargs \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_expects_other_args \u001b[38;5;28;01melse\u001b[39;00m {}\n\u001b[1;32m    315\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_new_arg_supported:\n\u001b[0;32m--> 316\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_get_relevant_documents\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    317\u001b[0m \u001b[43m        \u001b[49m\u001b[43mquery\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mrun_manager\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrun_manager\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43m_kwargs\u001b[49m\n\u001b[1;32m    318\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    319\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    320\u001b[0m     result \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_get_relevant_documents(query, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39m_kwargs)\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/vectorstores.py:696\u001b[0m, in \u001b[0;36mVectorStoreRetriever._get_relevant_documents\u001b[0;34m(self, query, run_manager)\u001b[0m\n\u001b[1;32m    692\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m_get_relevant_documents\u001b[39m(\n\u001b[1;32m    693\u001b[0m     \u001b[38;5;28mself\u001b[39m, query: \u001b[38;5;28mstr\u001b[39m, \u001b[38;5;241m*\u001b[39m, run_manager: CallbackManagerForRetrieverRun\n\u001b[1;32m    694\u001b[0m ) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m List[Document]:\n\u001b[1;32m    695\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msearch_type \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msimilarity\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[0;32m--> 696\u001b[0m         docs \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mvectorstore\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msimilarity_search\u001b[49m\u001b[43m(\u001b[49m\u001b[43mquery\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msearch_kwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    697\u001b[0m     \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msearch_type \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msimilarity_score_threshold\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[1;32m    698\u001b[0m         docs_and_similarities \u001b[38;5;241m=\u001b[39m (\n\u001b[1;32m    699\u001b[0m             \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mvectorstore\u001b[38;5;241m.\u001b[39msimilarity_search_with_relevance_scores(\n\u001b[1;32m    700\u001b[0m                 query, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msearch_kwargs\n\u001b[1;32m    701\u001b[0m             )\n\u001b[1;32m    702\u001b[0m         )\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_pinecone/vectorstores.py:247\u001b[0m, in \u001b[0;36mPineconeVectorStore.similarity_search\u001b[0;34m(self, query, k, filter, namespace, **kwargs)\u001b[0m\n\u001b[1;32m    228\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21msimilarity_search\u001b[39m(\n\u001b[1;32m    229\u001b[0m     \u001b[38;5;28mself\u001b[39m,\n\u001b[1;32m    230\u001b[0m     query: \u001b[38;5;28mstr\u001b[39m,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    234\u001b[0m     \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs: Any,\n\u001b[1;32m    235\u001b[0m ) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m List[Document]:\n\u001b[1;32m    236\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"Return pinecone documents most similar to query.\u001b[39;00m\n\u001b[1;32m    237\u001b[0m \n\u001b[1;32m    238\u001b[0m \u001b[38;5;124;03m    Args:\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    245\u001b[0m \u001b[38;5;124;03m        List of Documents most similar to the query and score for each\u001b[39;00m\n\u001b[1;32m    246\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 247\u001b[0m     docs_and_scores \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msimilarity_search_with_score\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    248\u001b[0m \u001b[43m        \u001b[49m\u001b[43mquery\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mk\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mk\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mfilter\u001b[39;49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mfilter\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnamespace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mnamespace\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\n\u001b[1;32m    249\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    250\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m [doc \u001b[38;5;28;01mfor\u001b[39;00m doc, _ \u001b[38;5;129;01min\u001b[39;00m docs_and_scores]\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_pinecone/vectorstores.py:192\u001b[0m, in \u001b[0;36mPineconeVectorStore.similarity_search_with_score\u001b[0;34m(self, query, k, filter, namespace)\u001b[0m\n\u001b[1;32m    174\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21msimilarity_search_with_score\u001b[39m(\n\u001b[1;32m    175\u001b[0m     \u001b[38;5;28mself\u001b[39m,\n\u001b[1;32m    176\u001b[0m     query: \u001b[38;5;28mstr\u001b[39m,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    179\u001b[0m     namespace: Optional[\u001b[38;5;28mstr\u001b[39m] \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mNone\u001b[39;00m,\n\u001b[1;32m    180\u001b[0m ) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m List[Tuple[Document, \u001b[38;5;28mfloat\u001b[39m]]:\n\u001b[1;32m    181\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"Return pinecone documents most similar to query, along with scores.\u001b[39;00m\n\u001b[1;32m    182\u001b[0m \n\u001b[1;32m    183\u001b[0m \u001b[38;5;124;03m    Args:\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    190\u001b[0m \u001b[38;5;124;03m        List of Documents most similar to the query and score for each\u001b[39;00m\n\u001b[1;32m    191\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 192\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msimilarity_search_by_vector_with_score\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    193\u001b[0m \u001b[43m        \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_embedding\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43membed_query\u001b[49m\u001b[43m(\u001b[49m\u001b[43mquery\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mk\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mk\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mfilter\u001b[39;49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mfilter\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnamespace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mnamespace\u001b[49m\n\u001b[1;32m    194\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_pinecone/vectorstores.py:209\u001b[0m, in \u001b[0;36mPineconeVectorStore.similarity_search_by_vector_with_score\u001b[0;34m(self, embedding, k, filter, namespace)\u001b[0m\n\u001b[1;32m    207\u001b[0m     namespace \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_namespace\n\u001b[1;32m    208\u001b[0m docs \u001b[38;5;241m=\u001b[39m []\n\u001b[0;32m--> 209\u001b[0m results \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_index\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mquery\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    210\u001b[0m \u001b[43m    \u001b[49m\u001b[43mvector\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43membedding\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    211\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtop_k\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mk\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    212\u001b[0m \u001b[43m    \u001b[49m\u001b[43minclude_metadata\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mTrue\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    213\u001b[0m \u001b[43m    \u001b[49m\u001b[43mnamespace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mnamespace\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    214\u001b[0m \u001b[43m    \u001b[49m\u001b[38;5;28;43mfilter\u001b[39;49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mfilter\u001b[39;49m\u001b[43m,\u001b[49m\n\u001b[1;32m    215\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    216\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m res \u001b[38;5;129;01min\u001b[39;00m results[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mmatches\u001b[39m\u001b[38;5;124m\"\u001b[39m]:\n\u001b[1;32m    217\u001b[0m     metadata \u001b[38;5;241m=\u001b[39m res[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mmetadata\u001b[39m\u001b[38;5;124m\"\u001b[39m]\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/pinecone/utils/error_handling.py:17\u001b[0m, in \u001b[0;36mvalidate_and_convert_errors.<locals>.inner_func\u001b[0;34m(*args, **kwargs)\u001b[0m\n\u001b[1;32m     15\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m\n\u001b[1;32m     16\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m ProtocolError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m---> 17\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m ProtocolError(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to connect; did you specify the correct index name?\u001b[39m\u001b[38;5;124m\"\u001b[39m) \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01me\u001b[39;00m\n",
      "\u001b[0;31mProtocolError\u001b[0m: Failed to connect; did you specify the correct index name?"
     ]
    }
   ],
   "source": [
    "from langchain_community.llms import Ollama\n",
    "\n",
    "MODELS = [\"mistral\", \"gemma\", \"llama2\", \"llama3\"]\n",
    "\n",
    "n = len(questions)\n",
    "for MODEL in MODELS:\n",
    "    model = Ollama(model=MODEL)\n",
    "    chain = setup | prompt | model | parser\n",
    "    resp = []\n",
    "    i = 1\n",
    "    for question in questions:\n",
    "        resp.append(chain.invoke(question))\n",
    "        na_quests[MODEL + \"_resp\"] = resp + [np.nan] * (n - i)\n",
    "        na_quests.to_csv(\"/home/dhanushb/Wellytics/RAG_data/results/na_quests.csv\")\n",
    "        i += 1\n",
    "na_quests"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Reference for Evaluation \n",
    "https://docs.ragas.io/en/stable/concepts/metrics/faithfulness.html\n",
    "https://towardsdatascience.com/evaluating-rag-applications-with-ragas-81d67b0ee31a"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Evaluation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['As part of my liver transplant, I take tacrolimus. My doctor recently informed me that I had a high chance of graft rejection and performed a pharmacogenetic test to determine whether my dosage needs to be adjusted. What does it indicate that I have CYP3A5 extensive metabolizer, according to my test results that I received today?',\n",
       " \"I'm Indian, and my family has a history of epilepsy. My doctor is worried about the seizure event I experienced last week. My results from the genetic test she recommended show that I have an HLA-B*15:02 variation. Will this affect how I use anticonvulsants? What other options are there? Does my ethnicity play a role? Tell me where you found the information.\",\n",
       " \"Allopurinol is what I take for gout. For my seizures, my doctor has recommended phenytoin. I'm concerned that there might be a connection between the drugs I take for seizures and my gout. What can I do?\",\n",
       " 'Is it true that while taking clopidogrel, I should avoid grapefruit or grapefruit juice? I adore grapefruit; can obtaining a genetic test assist me in locating an alternative antiplatelet medication that does not conflict with grapefruit? Could you also let me know where you found this information?']"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "questions = quests[\"Questions\"].to_list()[:4]\n",
    "questions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['For a CYP3A5 extensive metabolizer. Increase starting dose 1.5 to 2 times recommended starting dose,as lower probability of achieving target concentrations and may result in higher rejection risk. Measure drug concentrations and adjust dosage based on trough whole blood tacrolimus concentrations. ',\n",
       " 'As per CPIC HLA-B*15:02 might be observed in 2.6% of central and south asian population.Here ethnicity might not play a major role wrt to the gene concerned,but HLA-B*15:02 variation,will affect the use of anticonvulsants Avoid usage of phenytoin,fosphenytoin,carbamazepine and oxcarbazepine incase of HLA-B*15:02 Positive/Heterozygous',\n",
       " 'There can be moderate interaction while considering CYP2C9 with respect to allopurinol-phenytoin interaction which can be substrate-inhibitor /induced, whereas while considering HLA-B*15:02 both are contraindicated for HLA-B*15:02-Positive as it increases risk of drug-induced Stevens-Johnson syndrome/toxic epidermal necrolysis(SJS/TEN)',\n",
       " \"Indeed, you should refrain from drinking grapefruit or grapefruit juice when taking clopidogrel. Compounds in grapefruit have the ability to disrupt the body's clopidogrel metabolism, potentially reducing the medication's efficacy.\"]"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ground_truths = quests[\"Expected_response\"].to_list()[:4]\n",
    "ground_truths"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/_api/deprecation.py:119: LangChainDeprecationWarning: The method `BaseRetriever.get_relevant_documents` was deprecated in langchain-core 0.1.46 and will be removed in 0.3.0. Use invoke instead.\n",
      "  warn_deprecated(\n"
     ]
    }
   ],
   "source": [
    "from datasets import Dataset\n",
    "\n",
    "answers = []\n",
    "contexts = []\n",
    "\n",
    "model = Ollama(model=\"mistral\")\n",
    "chain = setup | prompt | model | parser\n",
    "\n",
    "# Inference\n",
    "for query in questions:\n",
    "  answers.append(chain.invoke(query))\n",
    "  contexts.append([docs.page_content for docs in retriever.invoke(query)])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\nfrom ragas.metrics import ContextRelevancy\\ncontext_relevancy = ContextRelevancy()\\n\\nresults = context_relevancy.score(dataset)\\nresults\\n'"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# To dict\n",
    "data = {\n",
    "    \"question\": questions,\n",
    "    \"answer\": answers,\n",
    "    \"contexts\": contexts,\n",
    "    \"ground_truth\": ground_truths\n",
    "}\n",
    "\n",
    "# Convert dict to dataset\n",
    "dataset = Dataset.from_dict(data)\n",
    "\n",
    "\"\"\"\n",
    "from ragas.metrics import ContextRelevancy\n",
    "context_relevancy = ContextRelevancy()\n",
    "\n",
    "results = context_relevancy.score(dataset)\n",
    "results\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "8675c675a8b64ae0acf50c9d9fc24af2",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Evaluating:   0%|          | 0/32 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>question</th>\n",
       "      <th>answer</th>\n",
       "      <th>contexts</th>\n",
       "      <th>ground_truth</th>\n",
       "      <th>context_precision</th>\n",
       "      <th>context_recall</th>\n",
       "      <th>faithfulness</th>\n",
       "      <th>answer_relevancy</th>\n",
       "      <th>context_entity_recall</th>\n",
       "      <th>answer_similarity</th>\n",
       "      <th>answer_correctness</th>\n",
       "      <th>harmfulness</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>As part of my liver transplant, I take tacroli...</td>\n",
       "      <td>Based on your test results, you are an extens...</td>\n",
       "      <td>[recipients. Pharmacogenet Genomics 2008;18:33...</td>\n",
       "      <td>For a CYP3A5 extensive metabolizer. Increase s...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.428571</td>\n",
       "      <td>0.872951</td>\n",
       "      <td>0.250000</td>\n",
       "      <td>0.909565</td>\n",
       "      <td>0.727391</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>I'm Indian, and my family has a history of epi...</td>\n",
       "      <td>The information provided suggests that geneti...</td>\n",
       "      <td>[{\"citations\": [{\"id\": 15071835, \"title\": \"Rec...</td>\n",
       "      <td>As per CPIC HLA-B*15:02 might be observed in 2...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>0.923077</td>\n",
       "      <td>0.858193</td>\n",
       "      <td>0.400000</td>\n",
       "      <td>0.880687</td>\n",
       "      <td>0.897601</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Allopurinol is what I take for gout. For my se...</td>\n",
       "      <td>The information provided discusses the impact...</td>\n",
       "      <td>[that a stronger inhibition of the uric acid p...</td>\n",
       "      <td>There can be moderate interaction while consid...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.571429</td>\n",
       "      <td>0.892649</td>\n",
       "      <td>0.223169</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Is it true that while taking clopidogrel, I sh...</td>\n",
       "      <td>Answer: While taking clopidogrel, it is gener...</td>\n",
       "      <td>[Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype1...</td>\n",
       "      <td>Indeed, you should refrain from drinking grape...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.500000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.891083</td>\n",
       "      <td>0.166667</td>\n",
       "      <td>0.912578</td>\n",
       "      <td>0.790659</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            question  \\\n",
       "0  As part of my liver transplant, I take tacroli...   \n",
       "1  I'm Indian, and my family has a history of epi...   \n",
       "2  Allopurinol is what I take for gout. For my se...   \n",
       "3  Is it true that while taking clopidogrel, I sh...   \n",
       "\n",
       "                                              answer  \\\n",
       "0   Based on your test results, you are an extens...   \n",
       "1   The information provided suggests that geneti...   \n",
       "2   The information provided discusses the impact...   \n",
       "3   Answer: While taking clopidogrel, it is gener...   \n",
       "\n",
       "                                            contexts  \\\n",
       "0  [recipients. Pharmacogenet Genomics 2008;18:33...   \n",
       "1  [{\"citations\": [{\"id\": 15071835, \"title\": \"Rec...   \n",
       "2  [that a stronger inhibition of the uric acid p...   \n",
       "3  [Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype1...   \n",
       "\n",
       "                                        ground_truth  context_precision  \\\n",
       "0  For a CYP3A5 extensive metabolizer. Increase s...                1.0   \n",
       "1  As per CPIC HLA-B*15:02 might be observed in 2...                1.0   \n",
       "2  There can be moderate interaction while consid...                1.0   \n",
       "3  Indeed, you should refrain from drinking grape...                1.0   \n",
       "\n",
       "   context_recall  faithfulness  answer_relevancy  context_entity_recall  \\\n",
       "0        1.000000      0.428571          0.872951               0.250000   \n",
       "1        0.666667      0.923077          0.858193               0.400000   \n",
       "2        0.000000      1.000000          0.000000               0.571429   \n",
       "3        0.500000      0.000000          0.891083               0.166667   \n",
       "\n",
       "   answer_similarity  answer_correctness  harmfulness  \n",
       "0           0.909565            0.727391            0  \n",
       "1           0.880687            0.897601            0  \n",
       "2           0.892649            0.223169            0  \n",
       "3           0.912578            0.790659            0  "
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from ragas import evaluate\n",
    "from ragas.metrics.critique import harmfulness\n",
    "from ragas.metrics import (\n",
    "    faithfulness,\n",
    "    answer_relevancy,\n",
    "    context_recall,\n",
    "    context_precision,\n",
    "    context_entity_recall,\n",
    "    answer_similarity,\n",
    "    answer_correctness,\n",
    ")\n",
    "\n",
    "result = evaluate(\n",
    "    dataset = dataset, \n",
    "    metrics=[\n",
    "        context_precision,\n",
    "        context_recall,\n",
    "        faithfulness,\n",
    "        answer_relevancy,\n",
    "        context_entity_recall,\n",
    "        answer_similarity,\n",
    "        answer_correctness,\n",
    "        harmfulness\n",
    "    ],\n",
    ")\n",
    "\n",
    "df = result.to_pandas()\n",
    "df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Evaluation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0.1</th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Questions</th>\n",
       "      <th>Expected_response</th>\n",
       "      <th>Expected_source</th>\n",
       "      <th>mistral_resp</th>\n",
       "      <th>gemma_resp</th>\n",
       "      <th>llama2_resp</th>\n",
       "      <th>llama3_resp</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>39</td>\n",
       "      <td>I have a family history of depression and rece...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/genotypeResults?q=%7B...</td>\n",
       "      <td>Based on your CYP2D6 genotype result (*1x≥3/*...</td>\n",
       "      <td>**Implications of having CYP2D6 *1x≥3/*2x2 Ult...</td>\n",
       "      <td>Based on the information provided in the table...</td>\n",
       "      <td>Based on the provided information about pharma...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>47</td>\n",
       "      <td>I have read that metformin leads to severe kid...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA450395/ove...</td>\n",
       "      <td>Pharmacogenomics can provide insights into ho...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Pharmacogenomics is the study of how genetic v...</td>\n",
       "      <td>A very interesting question!\\n\\nMetformin is a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>48</td>\n",
       "      <td>I drink socially. I recently underwent pharmac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>It seems that the information provided is abo...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the DPWG ...</td>\n",
       "      <td>A great question!\\n\\nAs a friendly AI assistan...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>52</td>\n",
       "      <td>I have seizures from the age of 5 years. I am ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes, pharmacogenomic testing could potentiall...</td>\n",
       "      <td>The provided text indicates that individuals w...</td>\n",
       "      <td>Response: Yes, pharmacogenomic testing can hel...</td>\n",
       "      <td>A great question!\\n\\nIn this case, we're deali...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>54</td>\n",
       "      <td>I have allergic rhinitis and take Reswas. Phar...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on your pharmacogenomic test results (C...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the pharm...</td>\n",
       "      <td>Thank you for reaching out with your concern! ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>5</td>\n",
       "      <td>61</td>\n",
       "      <td>I am a 62 year old female with history of 6 ye...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The information provided discusses genetic va...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the DPWG ...</td>\n",
       "      <td>Based on the information provided, we can anal...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>6</td>\n",
       "      <td>62</td>\n",
       "      <td>Why is tramadol banned in sports?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Tramadol is not necessarily banned in sports,...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Tramadol is a synthetic opioid medication that...</td>\n",
       "      <td>Based on the provided information, tramadol is...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>7</td>\n",
       "      <td>63</td>\n",
       "      <td>What are the antiseizure medications permissib...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The provided data does not contain specific i...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The list of antiseizure medications that are p...</td>\n",
       "      <td>I can provide information on pharmacogenomics ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>8</td>\n",
       "      <td>64</td>\n",
       "      <td>Can pharmacogenomics predict response to homeo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Pharmacogenomics primarily studies the role o...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Answer: There is no scientific evidence to sup...</td>\n",
       "      <td>Based on the provided information, I'd like to...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>9</td>\n",
       "      <td>65</td>\n",
       "      <td>Is blackcurrant extract natural antidiabetic a...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The text provided does not contain informatio...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the liter...</td>\n",
       "      <td>Hello!\\n\\nAs a pharmacogenomics AI assistant, ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10</td>\n",
       "      <td>66</td>\n",
       "      <td>Can MTHFR gene predict Vitamin B12 deficiency?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on the information provided, there is n...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The DPWG guideline for the gene-drug interacti...</td>\n",
       "      <td>A great question!\\n\\nAccording to the provided...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>11</td>\n",
       "      <td>67</td>\n",
       "      <td>What are the drugs that can affect my kidney f...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on the provided text, the following dru...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the SPC, ...</td>\n",
       "      <td>Based on the provided information, there are n...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>12</td>\n",
       "      <td>68</td>\n",
       "      <td>Can atenolol prevent cardiac arrest?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>While atenolol is a beta-blocker commonly use...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the SPC, ...</td>\n",
       "      <td>According to the provided documents, there is ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>13</td>\n",
       "      <td>70</td>\n",
       "      <td>I am CYP2D6 ultrametaboliser, what are the dru...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The data provided indicates that CYP2D6 metab...</td>\n",
       "      <td>The provided text suggests that drugs impacted...</td>\n",
       "      <td>Based on the information provided in the respo...</td>\n",
       "      <td>According to the provided pharmacogenomics inf...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>14</td>\n",
       "      <td>71</td>\n",
       "      <td>How does UGT1A1 impact irinotecan metabolism?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/literature/15139362</td>\n",
       "      <td>The UGT1A1 gene is associated with the metabo...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The UGT1A1 gene encodes for the enzyme uridine...</td>\n",
       "      <td>According to the provided literature, the UGT1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>15</td>\n",
       "      <td>72</td>\n",
       "      <td>What is Steven Johnson syndrome and how is HLA...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Steven-Johnson Syndrome (SJS) and Toxic Epide...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Steven Johnson Syndrome (SJS) and Toxic Epider...</td>\n",
       "      <td>Steven-Johnson syndrome (SJS) and Toxic Epider...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>16</td>\n",
       "      <td>73</td>\n",
       "      <td>Why is G6PD important for diabetic patients?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA164712711/...</td>\n",
       "      <td>The information provided does not discuss G6P...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>G6PD (Glucose-6-phosphate dehydrogenase) is an...</td>\n",
       "      <td>G6PD (Glucose-6-phosphate dehydrogenase) is no...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>17</td>\n",
       "      <td>74</td>\n",
       "      <td>Can you predict warfarin response with only VK...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA451906/gui...</td>\n",
       "      <td>While it is well established that variations ...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>No, based on the information provided in the d...</td>\n",
       "      <td>Based on the provided literature, it appears t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>18</td>\n",
       "      <td>94</td>\n",
       "      <td>What Pgx guidelines apply to my patient who is...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on the literature provided, there are s...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the PGX findings, your patient is an ...</td>\n",
       "      <td>Based on the pharmacogenomics research, for a ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>19</td>\n",
       "      <td>113</td>\n",
       "      <td>What are the minimum set of genes I should tes...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/ampAllelesToTest</td>\n",
       "      <td>To establish a comprehensive genetic testing ...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>To identify genetic variants associated with a...</td>\n",
       "      <td>Based on the provided information, it seems th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>20</td>\n",
       "      <td>150</td>\n",
       "      <td>My patient is identified with MYBPC3 variant a...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>To answer your question, it seems there is no...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The recommendation for patients with hypertrop...</td>\n",
       "      <td>Based on the literature provided earlier:\\n\\n1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>21</td>\n",
       "      <td>151</td>\n",
       "      <td>Can I start PARP inhibitors for my patient wit...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The provided data does not directly answer yo...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the docum...</td>\n",
       "      <td>Based on the provided information, I'd be happ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>22</td>\n",
       "      <td>152</td>\n",
       "      <td>Can exondys be advised to limb girdle muscular...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on the provided data, there is no infor...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The American College of Rheumatology (ACR) gui...</td>\n",
       "      <td>A very specific question!\\n\\nBased on the prov...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>23</td>\n",
       "      <td>153</td>\n",
       "      <td>Will sitagliptin and metformin combination the...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The information provided does not directly an...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The answer to whether sitagliptin and metformi...</td>\n",
       "      <td>The question you are asking involves the genet...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>24</td>\n",
       "      <td>154</td>\n",
       "      <td>Can codiene be prescribed to a breast feeding ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The information provided does not directly ad...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the RNPG...</td>\n",
       "      <td>Based on the provided information, here's what...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>25</td>\n",
       "      <td>155</td>\n",
       "      <td>My patient has MTHFR rs1801133 AA genotype, sh...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on the provided information, it does no...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Arnica is a homeopathic remedy that is commonl...</td>\n",
       "      <td>I'm happy to help you with that!\\n\\nAccording ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>26</td>\n",
       "      <td>156</td>\n",
       "      <td>Can Criggler najar syndrome patient be prescri...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on the information provided, there is n...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the docum...</td>\n",
       "      <td>A pharmacogenomics question!\\n\\nCriggler Najja...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>27</td>\n",
       "      <td>157</td>\n",
       "      <td>Is there a role of CYP2C19 poor metabolism in ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Based on the provided literature, there is no...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the given information, there is some ...</td>\n",
       "      <td>Based on the provided document, it seems that ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>28</td>\n",
       "      <td>158</td>\n",
       "      <td>Can solid tumor test sample be subjected to ph...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes, solid tumor tissue can be used for pharm...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Answer: Based on the information provided in t...</td>\n",
       "      <td>No, a solid tumor test sample cannot be used f...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>29</td>\n",
       "      <td>159</td>\n",
       "      <td>Can progesterone therapy help in iVF planning ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Answer: Progesterone therapy is not typically...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The answer to whether progesterone therapy can...</td>\n",
       "      <td>I can help you find the answer.\\n\\nAs an AI as...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>30</td>\n",
       "      <td>160</td>\n",
       "      <td>Is there a pharmacogenomics guideline for meth...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The information provided does not specificall...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Pharmacogenomics is the study of how genetic v...</td>\n",
       "      <td>I'm an AI assistant trained to provide underst...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>31</td>\n",
       "      <td>161</td>\n",
       "      <td>What are the safer alteratives for omeprazole ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA450704/gui...</td>\n",
       "      <td>Safer alternatives for omeprazole for a patie...</td>\n",
       "      <td>The safer alternatives for omeprazole for a pa...</td>\n",
       "      <td>Based on the information provided in the FDA r...</td>\n",
       "      <td>According to the provided pharmacogenomic reco...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>32</td>\n",
       "      <td>162</td>\n",
       "      <td>What is the tacrolimus dosage recommended for ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA451578/gui...</td>\n",
       "      <td>The provided information does not specify the...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The table does not provide information on tacr...</td>\n",
       "      <td>The question you're asking doesn't match the i...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>33</td>\n",
       "      <td>163</td>\n",
       "      <td>What is indeterminate status in genotype?</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.frontiersin.org/journals/pharmacol...</td>\n",
       "      <td>Indeterminate status in genotyping refers to ...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>\"Indeterminate status\" in a genotype refers t...</td>\n",
       "      <td>I've got the answer for you!\\n\\nAccording to t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>34</td>\n",
       "      <td>164</td>\n",
       "      <td>Do ultrametabolisers require increased dosage ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "      <td>Answer: Ultrametabolizers (UM) for the CYP2D6...</td>\n",
       "      <td>Ultrametabolizers do not require increased dos...</td>\n",
       "      <td>The DPWG guideline provides recommendations fo...</td>\n",
       "      <td>According to the provided information, for pat...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>35</td>\n",
       "      <td>165</td>\n",
       "      <td>Are there any risk of oral contraceptive medic...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA452637/cli...</td>\n",
       "      <td>Yes, the risk and efficacy of certain oral co...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the docum...</td>\n",
       "      <td>Based on the provided text, here are some pote...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>36</td>\n",
       "      <td>169</td>\n",
       "      <td>How does CYP2C9 affect anticoagulants mode of ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/disease/PA166123298/l...</td>\n",
       "      <td>The Cytochrome P450 2C9 (CYP2C9) is an enzyme...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The cytochrome P450 2C9 (CYP2C9) enzyme plays ...</td>\n",
       "      <td>According to the provided information, CYP2C9 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>37</td>\n",
       "      <td>170</td>\n",
       "      <td>Which are the antidiabetic medications influen...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.pharmgkb.org/chemical/PA164712711/...</td>\n",
       "      <td>The document provided does not mention any in...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The following are the antidiabetic medications...</td>\n",
       "      <td>According to the provided document, there is n...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    Unnamed: 0.1  Unnamed: 0  \\\n",
       "0              0          39   \n",
       "1              1          47   \n",
       "2              2          48   \n",
       "3              3          52   \n",
       "4              4          54   \n",
       "5              5          61   \n",
       "6              6          62   \n",
       "7              7          63   \n",
       "8              8          64   \n",
       "9              9          65   \n",
       "10            10          66   \n",
       "11            11          67   \n",
       "12            12          68   \n",
       "13            13          70   \n",
       "14            14          71   \n",
       "15            15          72   \n",
       "16            16          73   \n",
       "17            17          74   \n",
       "18            18          94   \n",
       "19            19         113   \n",
       "20            20         150   \n",
       "21            21         151   \n",
       "22            22         152   \n",
       "23            23         153   \n",
       "24            24         154   \n",
       "25            25         155   \n",
       "26            26         156   \n",
       "27            27         157   \n",
       "28            28         158   \n",
       "29            29         159   \n",
       "30            30         160   \n",
       "31            31         161   \n",
       "32            32         162   \n",
       "33            33         163   \n",
       "34            34         164   \n",
       "35            35         165   \n",
       "36            36         169   \n",
       "37            37         170   \n",
       "\n",
       "                                            Questions  Expected_response  \\\n",
       "0   I have a family history of depression and rece...                NaN   \n",
       "1   I have read that metformin leads to severe kid...                NaN   \n",
       "2   I drink socially. I recently underwent pharmac...                NaN   \n",
       "3   I have seizures from the age of 5 years. I am ...                NaN   \n",
       "4   I have allergic rhinitis and take Reswas. Phar...                NaN   \n",
       "5   I am a 62 year old female with history of 6 ye...                NaN   \n",
       "6                   Why is tramadol banned in sports?                NaN   \n",
       "7   What are the antiseizure medications permissib...                NaN   \n",
       "8   Can pharmacogenomics predict response to homeo...                NaN   \n",
       "9   Is blackcurrant extract natural antidiabetic a...                NaN   \n",
       "10     Can MTHFR gene predict Vitamin B12 deficiency?                NaN   \n",
       "11  What are the drugs that can affect my kidney f...                NaN   \n",
       "12               Can atenolol prevent cardiac arrest?                NaN   \n",
       "13  I am CYP2D6 ultrametaboliser, what are the dru...                NaN   \n",
       "14      How does UGT1A1 impact irinotecan metabolism?                NaN   \n",
       "15  What is Steven Johnson syndrome and how is HLA...                NaN   \n",
       "16       Why is G6PD important for diabetic patients?                NaN   \n",
       "17  Can you predict warfarin response with only VK...                NaN   \n",
       "18  What Pgx guidelines apply to my patient who is...                NaN   \n",
       "19  What are the minimum set of genes I should tes...                NaN   \n",
       "20  My patient is identified with MYBPC3 variant a...                NaN   \n",
       "21  Can I start PARP inhibitors for my patient wit...                NaN   \n",
       "22  Can exondys be advised to limb girdle muscular...                NaN   \n",
       "23  Will sitagliptin and metformin combination the...                NaN   \n",
       "24  Can codiene be prescribed to a breast feeding ...                NaN   \n",
       "25  My patient has MTHFR rs1801133 AA genotype, sh...                NaN   \n",
       "26  Can Criggler najar syndrome patient be prescri...                NaN   \n",
       "27  Is there a role of CYP2C19 poor metabolism in ...                NaN   \n",
       "28  Can solid tumor test sample be subjected to ph...                NaN   \n",
       "29  Can progesterone therapy help in iVF planning ...                NaN   \n",
       "30  Is there a pharmacogenomics guideline for meth...                NaN   \n",
       "31  What are the safer alteratives for omeprazole ...                NaN   \n",
       "32  What is the tacrolimus dosage recommended for ...                NaN   \n",
       "33          What is indeterminate status in genotype?                NaN   \n",
       "34  Do ultrametabolisers require increased dosage ...                NaN   \n",
       "35  Are there any risk of oral contraceptive medic...                NaN   \n",
       "36  How does CYP2C9 affect anticoagulants mode of ...                NaN   \n",
       "37  Which are the antidiabetic medications influen...                NaN   \n",
       "\n",
       "                                      Expected_source  \\\n",
       "0   https://www.pharmgkb.org/genotypeResults?q=%7B...   \n",
       "1   https://www.pharmgkb.org/chemical/PA450395/ove...   \n",
       "2                                                 NaN   \n",
       "3                                                 NaN   \n",
       "4                                                 NaN   \n",
       "5                                                 NaN   \n",
       "6                                                 NaN   \n",
       "7                                                 NaN   \n",
       "8                                                 NaN   \n",
       "9                                                 NaN   \n",
       "10                                                NaN   \n",
       "11                                                NaN   \n",
       "12                                                NaN   \n",
       "13                                                NaN   \n",
       "14       https://www.pharmgkb.org/literature/15139362   \n",
       "15                                                NaN   \n",
       "16  https://www.pharmgkb.org/chemical/PA164712711/...   \n",
       "17  https://www.pharmgkb.org/chemical/PA451906/gui...   \n",
       "18                                                NaN   \n",
       "19          https://www.pharmgkb.org/ampAllelesToTest   \n",
       "20                                                NaN   \n",
       "21                                                NaN   \n",
       "22                                                NaN   \n",
       "23                                                NaN   \n",
       "24                                                NaN   \n",
       "25                                                NaN   \n",
       "26                                                NaN   \n",
       "27                                                NaN   \n",
       "28                                                NaN   \n",
       "29                                                NaN   \n",
       "30                                                NaN   \n",
       "31  https://www.pharmgkb.org/chemical/PA450704/gui...   \n",
       "32  https://www.pharmgkb.org/chemical/PA451578/gui...   \n",
       "33  https://www.frontiersin.org/journals/pharmacol...   \n",
       "34  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...   \n",
       "35  https://www.pharmgkb.org/chemical/PA452637/cli...   \n",
       "36  https://www.pharmgkb.org/disease/PA166123298/l...   \n",
       "37  https://www.pharmgkb.org/chemical/PA164712711/...   \n",
       "\n",
       "                                         mistral_resp  \\\n",
       "0    Based on your CYP2D6 genotype result (*1x≥3/*...   \n",
       "1    Pharmacogenomics can provide insights into ho...   \n",
       "2    It seems that the information provided is abo...   \n",
       "3    Yes, pharmacogenomic testing could potentiall...   \n",
       "4    Based on your pharmacogenomic test results (C...   \n",
       "5    The information provided discusses genetic va...   \n",
       "6    Tramadol is not necessarily banned in sports,...   \n",
       "7    The provided data does not contain specific i...   \n",
       "8    Pharmacogenomics primarily studies the role o...   \n",
       "9    The text provided does not contain informatio...   \n",
       "10   Based on the information provided, there is n...   \n",
       "11   Based on the provided text, the following dru...   \n",
       "12   While atenolol is a beta-blocker commonly use...   \n",
       "13   The data provided indicates that CYP2D6 metab...   \n",
       "14   The UGT1A1 gene is associated with the metabo...   \n",
       "15   Steven-Johnson Syndrome (SJS) and Toxic Epide...   \n",
       "16   The information provided does not discuss G6P...   \n",
       "17   While it is well established that variations ...   \n",
       "18   Based on the literature provided, there are s...   \n",
       "19   To establish a comprehensive genetic testing ...   \n",
       "20   To answer your question, it seems there is no...   \n",
       "21   The provided data does not directly answer yo...   \n",
       "22   Based on the provided data, there is no infor...   \n",
       "23   The information provided does not directly an...   \n",
       "24   The information provided does not directly ad...   \n",
       "25   Based on the provided information, it does no...   \n",
       "26   Based on the information provided, there is n...   \n",
       "27   Based on the provided literature, there is no...   \n",
       "28   Yes, solid tumor tissue can be used for pharm...   \n",
       "29   Answer: Progesterone therapy is not typically...   \n",
       "30   The information provided does not specificall...   \n",
       "31   Safer alternatives for omeprazole for a patie...   \n",
       "32   The provided information does not specify the...   \n",
       "33   Indeterminate status in genotyping refers to ...   \n",
       "34   Answer: Ultrametabolizers (UM) for the CYP2D6...   \n",
       "35   Yes, the risk and efficacy of certain oral co...   \n",
       "36   The Cytochrome P450 2C9 (CYP2C9) is an enzyme...   \n",
       "37   The document provided does not mention any in...   \n",
       "\n",
       "                                           gemma_resp  \\\n",
       "0   **Implications of having CYP2D6 *1x≥3/*2x2 Ult...   \n",
       "1   The provided text does not contain any informa...   \n",
       "2   The provided text does not contain any informa...   \n",
       "3   The provided text indicates that individuals w...   \n",
       "4   The provided text does not contain any informa...   \n",
       "5   The provided text does not contain any informa...   \n",
       "6   The provided text does not contain any informa...   \n",
       "7   The provided text does not contain any informa...   \n",
       "8   The provided text does not contain any informa...   \n",
       "9   The provided text does not contain any informa...   \n",
       "10  The provided text does not contain any informa...   \n",
       "11  The provided text does not contain any informa...   \n",
       "12  The provided text does not contain any informa...   \n",
       "13  The provided text suggests that drugs impacted...   \n",
       "14  The provided text does not contain any informa...   \n",
       "15  The provided text does not contain any informa...   \n",
       "16  The provided text does not contain any informa...   \n",
       "17  The provided text does not contain any informa...   \n",
       "18  The provided text does not contain any informa...   \n",
       "19  The provided text does not contain any informa...   \n",
       "20  The provided text does not contain any informa...   \n",
       "21  The provided text does not contain any informa...   \n",
       "22  The provided text does not contain any informa...   \n",
       "23  The provided text does not contain any informa...   \n",
       "24  The provided text does not contain any informa...   \n",
       "25  The provided text does not contain any informa...   \n",
       "26  The provided text does not contain any informa...   \n",
       "27  The provided text does not contain any informa...   \n",
       "28  The provided text does not contain any informa...   \n",
       "29  The provided text does not contain any informa...   \n",
       "30  The provided text does not contain any informa...   \n",
       "31  The safer alternatives for omeprazole for a pa...   \n",
       "32  The provided text does not contain any informa...   \n",
       "33  The provided text does not contain any informa...   \n",
       "34  Ultrametabolizers do not require increased dos...   \n",
       "35  The provided text does not contain any informa...   \n",
       "36  The provided text does not contain any informa...   \n",
       "37  The provided text does not contain any informa...   \n",
       "\n",
       "                                          llama2_resp  \\\n",
       "0   Based on the information provided in the table...   \n",
       "1   Pharmacogenomics is the study of how genetic v...   \n",
       "2   Based on the information provided in the DPWG ...   \n",
       "3   Response: Yes, pharmacogenomic testing can hel...   \n",
       "4   Based on the information provided in the pharm...   \n",
       "5   Based on the information provided in the DPWG ...   \n",
       "6   Tramadol is a synthetic opioid medication that...   \n",
       "7   The list of antiseizure medications that are p...   \n",
       "8   Answer: There is no scientific evidence to sup...   \n",
       "9   Based on the information provided in the liter...   \n",
       "10  The DPWG guideline for the gene-drug interacti...   \n",
       "11  Based on the information provided in the SPC, ...   \n",
       "12  Based on the information provided in the SPC, ...   \n",
       "13  Based on the information provided in the respo...   \n",
       "14  The UGT1A1 gene encodes for the enzyme uridine...   \n",
       "15  Steven Johnson Syndrome (SJS) and Toxic Epider...   \n",
       "16  G6PD (Glucose-6-phosphate dehydrogenase) is an...   \n",
       "17  No, based on the information provided in the d...   \n",
       "18  Based on the PGX findings, your patient is an ...   \n",
       "19  To identify genetic variants associated with a...   \n",
       "20  The recommendation for patients with hypertrop...   \n",
       "21  Based on the information provided in the docum...   \n",
       "22  The American College of Rheumatology (ACR) gui...   \n",
       "23  The answer to whether sitagliptin and metformi...   \n",
       "24   Based on the information provided in the RNPG...   \n",
       "25  Arnica is a homeopathic remedy that is commonl...   \n",
       "26  Based on the information provided in the docum...   \n",
       "27  Based on the given information, there is some ...   \n",
       "28  Answer: Based on the information provided in t...   \n",
       "29  The answer to whether progesterone therapy can...   \n",
       "30  Pharmacogenomics is the study of how genetic v...   \n",
       "31  Based on the information provided in the FDA r...   \n",
       "32  The table does not provide information on tacr...   \n",
       "33   \"Indeterminate status\" in a genotype refers t...   \n",
       "34  The DPWG guideline provides recommendations fo...   \n",
       "35  Based on the information provided in the docum...   \n",
       "36  The cytochrome P450 2C9 (CYP2C9) enzyme plays ...   \n",
       "37  The following are the antidiabetic medications...   \n",
       "\n",
       "                                          llama3_resp  \n",
       "0   Based on the provided information about pharma...  \n",
       "1   A very interesting question!\\n\\nMetformin is a...  \n",
       "2   A great question!\\n\\nAs a friendly AI assistan...  \n",
       "3   A great question!\\n\\nIn this case, we're deali...  \n",
       "4   Thank you for reaching out with your concern! ...  \n",
       "5   Based on the information provided, we can anal...  \n",
       "6   Based on the provided information, tramadol is...  \n",
       "7   I can provide information on pharmacogenomics ...  \n",
       "8   Based on the provided information, I'd like to...  \n",
       "9   Hello!\\n\\nAs a pharmacogenomics AI assistant, ...  \n",
       "10  A great question!\\n\\nAccording to the provided...  \n",
       "11  Based on the provided information, there are n...  \n",
       "12  According to the provided documents, there is ...  \n",
       "13  According to the provided pharmacogenomics inf...  \n",
       "14  According to the provided literature, the UGT1...  \n",
       "15  Steven-Johnson syndrome (SJS) and Toxic Epider...  \n",
       "16  G6PD (Glucose-6-phosphate dehydrogenase) is no...  \n",
       "17  Based on the provided literature, it appears t...  \n",
       "18  Based on the pharmacogenomics research, for a ...  \n",
       "19  Based on the provided information, it seems th...  \n",
       "20  Based on the literature provided earlier:\\n\\n1...  \n",
       "21  Based on the provided information, I'd be happ...  \n",
       "22  A very specific question!\\n\\nBased on the prov...  \n",
       "23  The question you are asking involves the genet...  \n",
       "24  Based on the provided information, here's what...  \n",
       "25  I'm happy to help you with that!\\n\\nAccording ...  \n",
       "26  A pharmacogenomics question!\\n\\nCriggler Najja...  \n",
       "27  Based on the provided document, it seems that ...  \n",
       "28  No, a solid tumor test sample cannot be used f...  \n",
       "29  I can help you find the answer.\\n\\nAs an AI as...  \n",
       "30  I'm an AI assistant trained to provide underst...  \n",
       "31  According to the provided pharmacogenomic reco...  \n",
       "32  The question you're asking doesn't match the i...  \n",
       "33  I've got the answer for you!\\n\\nAccording to t...  \n",
       "34  According to the provided information, for pat...  \n",
       "35  Based on the provided text, here are some pote...  \n",
       "36  According to the provided information, CYP2C9 ...  \n",
       "37  According to the provided document, there is n...  "
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "na_quests = pd.read_csv(\"/home/dhanushb/Wellytics/RAG_data/results/na_quests.csv\")\n",
    "#na_quests = na_quests.drop([\"Unnamed: 0.1\"],axis=1)\n",
    "na_quests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [],
   "source": [
    "na_quests.to_csv(\"/home/dhanushb/Wellytics/RAG_data/results/na_quests.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0.1</th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Questions</th>\n",
       "      <th>Expected_response</th>\n",
       "      <th>Expected_source</th>\n",
       "      <th>context</th>\n",
       "      <th>mistral_resp</th>\n",
       "      <th>gemma_resp</th>\n",
       "      <th>llama2_resp</th>\n",
       "      <th>llama3_resp</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>As part of my liver transplant, I take tacroli...</td>\n",
       "      <td>For a CYP3A5 extensive metabolizer. Increase s...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "      <td>['recipients. Pharmacogenet Genomics 2008;18:3...</td>\n",
       "      <td>Based on your test results, if you are an ext...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the docum...</td>\n",
       "      <td>According to your pharmacogenetic test results...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>I'm Indian, and my family has a history of epi...</td>\n",
       "      <td>As per CPIC HLA-B*15:02 might be observed in 2...</td>\n",
       "      <td>https://www.pharmgkb.org/haplotype/PA165954769...</td>\n",
       "      <td>['{\"citations\": [{\"id\": 15071835, \"title\": \"Re...</td>\n",
       "      <td>The genetic variant HLA-B*15:02 that you have...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The information you provided is from a scienti...</td>\n",
       "      <td>I'm an AI assistant trained to provide underst...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>Allopurinol is what I take for gout. For my se...</td>\n",
       "      <td>There can be moderate interaction while consid...</td>\n",
       "      <td>https://insilico-cyp.charite.de/SuperCYPsPred/...</td>\n",
       "      <td>['that a stronger inhibition of the uric acid ...</td>\n",
       "      <td>Answer: Phenytoin and allopurinol are two dis...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>It is understandable to have concerns about po...</td>\n",
       "      <td>A great question!\\n\\nAs an AI assistant traine...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>3</td>\n",
       "      <td>Is it true that while taking clopidogrel, I sh...</td>\n",
       "      <td>Indeed, you should refrain from drinking grape...</td>\n",
       "      <td>Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...</td>\n",
       "      <td>['Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype...</td>\n",
       "      <td>While taking clopidogrel, it is generally rec...</td>\n",
       "      <td>The provided text does not contain information...</td>\n",
       "      <td>Yes, it is true that clopidogrel can interact ...</td>\n",
       "      <td>A pharmacogenomics enthusiast!\\n\\nAccording to...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>4</td>\n",
       "      <td>Dizziness is one of the adverse reactions that...</td>\n",
       "      <td>Dizziness drawiness,extreme thirst,fast, shall...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "      <td>['1. \\nGanoci L et al. ABCB1, ABCG2 and CYP2D6...</td>\n",
       "      <td>Based on the literature provided, there is ev...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The information provided in the response is ba...</td>\n",
       "      <td>I can help you understand the relationship bet...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>129</td>\n",
       "      <td>149</td>\n",
       "      <td>My patient has CYP2D6 poor metabolism and does...</td>\n",
       "      <td>CPIC guideline for ondansetron and CYP2D6 has ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[\"Drug: sertraline\\nGene1: CYP2C19\\nPhenotype1...</td>\n",
       "      <td>Based on the provided data, there are no spec...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The guidelines for prescribing sertraline for ...</td>\n",
       "      <td>The guidelines provided do not specifically ad...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>130</td>\n",
       "      <td>166</td>\n",
       "      <td>What are side effects associated wtih DPYD poo...</td>\n",
       "      <td>5-FU has many adverse drug effects when used s...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['Drug: lornoxicam\\nGene1: CYP2C9\\nPhenotype1:...</td>\n",
       "      <td>The HLAB system is not directly involved in p...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>HLA-B and CYP2C9 genetic polymorphisms are imp...</td>\n",
       "      <td>According to the provided pharmacogenomics inf...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>131</td>\n",
       "      <td>167</td>\n",
       "      <td>What are the pharmacogenomic panel advised to ...</td>\n",
       "      <td>G6PD, CYP2D6, UGT1A1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['{\"citations\": [{\"id\": 15143242, \"title\": \"Cl...</td>\n",
       "      <td>The given text suggests that for a patient wi...</td>\n",
       "      <td>The provided text suggests that due to the pat...</td>\n",
       "      <td>Based on the information provided in the liter...</td>\n",
       "      <td>Based on the provided information, for a patie...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>132</td>\n",
       "      <td>168</td>\n",
       "      <td>I have a 7 year old patient with HLA-B1502 pos...</td>\n",
       "      <td>Sodium Valproate</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['9\\n. \\nKristensen MH et al. Variants in the ...</td>\n",
       "      <td>The side effects associated with DPD (dihydro...</td>\n",
       "      <td>Patients with DPYD poor metabolism of fluorour...</td>\n",
       "      <td>Based on the provided document, the side effec...</td>\n",
       "      <td>According to the provided document, individual...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>133</td>\n",
       "      <td>171</td>\n",
       "      <td>What are the adverse outcomes associated with ...</td>\n",
       "      <td>Patients with the rs1801133 AA genotype and Ar...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[\"Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1...</td>\n",
       "      <td>To provide an appropriate pharmacogenomic (PG...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The answer is not explicitly stated in the giv...</td>\n",
       "      <td>The input document doesn't contain information...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>134 rows × 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Unnamed: 0.1  Unnamed: 0  \\\n",
       "0               0           0   \n",
       "1               1           1   \n",
       "2               2           2   \n",
       "3               3           3   \n",
       "4               4           4   \n",
       "..            ...         ...   \n",
       "129           129         149   \n",
       "130           130         166   \n",
       "131           131         167   \n",
       "132           132         168   \n",
       "133           133         171   \n",
       "\n",
       "                                             Questions  \\\n",
       "0    As part of my liver transplant, I take tacroli...   \n",
       "1    I'm Indian, and my family has a history of epi...   \n",
       "2    Allopurinol is what I take for gout. For my se...   \n",
       "3    Is it true that while taking clopidogrel, I sh...   \n",
       "4    Dizziness is one of the adverse reactions that...   \n",
       "..                                                 ...   \n",
       "129  My patient has CYP2D6 poor metabolism and does...   \n",
       "130  What are side effects associated wtih DPYD poo...   \n",
       "131  What are the pharmacogenomic panel advised to ...   \n",
       "132  I have a 7 year old patient with HLA-B1502 pos...   \n",
       "133  What are the adverse outcomes associated with ...   \n",
       "\n",
       "                                     Expected_response  \\\n",
       "0    For a CYP3A5 extensive metabolizer. Increase s...   \n",
       "1    As per CPIC HLA-B*15:02 might be observed in 2...   \n",
       "2    There can be moderate interaction while consid...   \n",
       "3    Indeed, you should refrain from drinking grape...   \n",
       "4    Dizziness drawiness,extreme thirst,fast, shall...   \n",
       "..                                                 ...   \n",
       "129  CPIC guideline for ondansetron and CYP2D6 has ...   \n",
       "130  5-FU has many adverse drug effects when used s...   \n",
       "131                               G6PD, CYP2D6, UGT1A1   \n",
       "132                                   Sodium Valproate   \n",
       "133  Patients with the rs1801133 AA genotype and Ar...   \n",
       "\n",
       "                                       Expected_source  \\\n",
       "0    https://www.fda.gov/drugs/science-and-research...   \n",
       "1    https://www.pharmgkb.org/haplotype/PA165954769...   \n",
       "2    https://insilico-cyp.charite.de/SuperCYPsPred/...   \n",
       "3    Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...   \n",
       "4    https://www.fda.gov/drugs/science-and-research...   \n",
       "..                                                 ...   \n",
       "129                                                NaN   \n",
       "130                                                NaN   \n",
       "131                                                NaN   \n",
       "132                                                NaN   \n",
       "133                                                NaN   \n",
       "\n",
       "                                               context  \\\n",
       "0    ['recipients. Pharmacogenet Genomics 2008;18:3...   \n",
       "1    ['{\"citations\": [{\"id\": 15071835, \"title\": \"Re...   \n",
       "2    ['that a stronger inhibition of the uric acid ...   \n",
       "3    ['Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype...   \n",
       "4    ['1. \\nGanoci L et al. ABCB1, ABCG2 and CYP2D6...   \n",
       "..                                                 ...   \n",
       "129  [\"Drug: sertraline\\nGene1: CYP2C19\\nPhenotype1...   \n",
       "130  ['Drug: lornoxicam\\nGene1: CYP2C9\\nPhenotype1:...   \n",
       "131  ['{\"citations\": [{\"id\": 15143242, \"title\": \"Cl...   \n",
       "132  ['9\\n. \\nKristensen MH et al. Variants in the ...   \n",
       "133  [\"Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1...   \n",
       "\n",
       "                                          mistral_resp  \\\n",
       "0     Based on your test results, if you are an ext...   \n",
       "1     The genetic variant HLA-B*15:02 that you have...   \n",
       "2     Answer: Phenytoin and allopurinol are two dis...   \n",
       "3     While taking clopidogrel, it is generally rec...   \n",
       "4     Based on the literature provided, there is ev...   \n",
       "..                                                 ...   \n",
       "129   Based on the provided data, there are no spec...   \n",
       "130   The HLAB system is not directly involved in p...   \n",
       "131   The given text suggests that for a patient wi...   \n",
       "132   The side effects associated with DPD (dihydro...   \n",
       "133   To provide an appropriate pharmacogenomic (PG...   \n",
       "\n",
       "                                            gemma_resp  \\\n",
       "0    The provided text does not contain any informa...   \n",
       "1    The provided text does not contain any informa...   \n",
       "2    The provided text does not contain any informa...   \n",
       "3    The provided text does not contain information...   \n",
       "4    The provided text does not contain any informa...   \n",
       "..                                                 ...   \n",
       "129  The provided text does not contain any informa...   \n",
       "130  The provided text does not contain any informa...   \n",
       "131  The provided text suggests that due to the pat...   \n",
       "132  Patients with DPYD poor metabolism of fluorour...   \n",
       "133  The provided text does not contain any informa...   \n",
       "\n",
       "                                           llama2_resp  \\\n",
       "0    Based on the information provided in the docum...   \n",
       "1    The information you provided is from a scienti...   \n",
       "2    It is understandable to have concerns about po...   \n",
       "3    Yes, it is true that clopidogrel can interact ...   \n",
       "4    The information provided in the response is ba...   \n",
       "..                                                 ...   \n",
       "129  The guidelines for prescribing sertraline for ...   \n",
       "130  HLA-B and CYP2C9 genetic polymorphisms are imp...   \n",
       "131  Based on the information provided in the liter...   \n",
       "132  Based on the provided document, the side effec...   \n",
       "133  The answer is not explicitly stated in the giv...   \n",
       "\n",
       "                                           llama3_resp  \n",
       "0    According to your pharmacogenetic test results...  \n",
       "1    I'm an AI assistant trained to provide underst...  \n",
       "2    A great question!\\n\\nAs an AI assistant traine...  \n",
       "3    A pharmacogenomics enthusiast!\\n\\nAccording to...  \n",
       "4    I can help you understand the relationship bet...  \n",
       "..                                                 ...  \n",
       "129  The guidelines provided do not specifically ad...  \n",
       "130  According to the provided pharmacogenomics inf...  \n",
       "131  Based on the provided information, for a patie...  \n",
       "132  According to the provided document, individual...  \n",
       "133  The input document doesn't contain information...  \n",
       "\n",
       "[134 rows x 10 columns]"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "quests = pd.read_csv(\"/home/dhanushb/Wellytics/RAG_data/results/quests_with_resps.csv\")\n",
    "quests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Questions</th>\n",
       "      <th>Expected_response</th>\n",
       "      <th>Expected_source</th>\n",
       "      <th>context</th>\n",
       "      <th>mistral_resp</th>\n",
       "      <th>gemma_resp</th>\n",
       "      <th>llama2_resp</th>\n",
       "      <th>llama3_resp</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>As part of my liver transplant, I take tacroli...</td>\n",
       "      <td>For a CYP3A5 extensive metabolizer. Increase s...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "      <td>['recipients. Pharmacogenet Genomics 2008;18:3...</td>\n",
       "      <td>Based on your test results, if you are an ext...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>Based on the information provided in the docum...</td>\n",
       "      <td>According to your pharmacogenetic test results...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>I'm Indian, and my family has a history of epi...</td>\n",
       "      <td>As per CPIC HLA-B*15:02 might be observed in 2...</td>\n",
       "      <td>https://www.pharmgkb.org/haplotype/PA165954769...</td>\n",
       "      <td>['{\"citations\": [{\"id\": 15071835, \"title\": \"Re...</td>\n",
       "      <td>The genetic variant HLA-B*15:02 that you have...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The information you provided is from a scienti...</td>\n",
       "      <td>I'm an AI assistant trained to provide underst...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>Allopurinol is what I take for gout. For my se...</td>\n",
       "      <td>There can be moderate interaction while consid...</td>\n",
       "      <td>https://insilico-cyp.charite.de/SuperCYPsPred/...</td>\n",
       "      <td>['that a stronger inhibition of the uric acid ...</td>\n",
       "      <td>Answer: Phenytoin and allopurinol are two dis...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>It is understandable to have concerns about po...</td>\n",
       "      <td>A great question!\\n\\nAs an AI assistant traine...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>Is it true that while taking clopidogrel, I sh...</td>\n",
       "      <td>Indeed, you should refrain from drinking grape...</td>\n",
       "      <td>Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...</td>\n",
       "      <td>['Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype...</td>\n",
       "      <td>While taking clopidogrel, it is generally rec...</td>\n",
       "      <td>The provided text does not contain information...</td>\n",
       "      <td>Yes, it is true that clopidogrel can interact ...</td>\n",
       "      <td>A pharmacogenomics enthusiast!\\n\\nAccording to...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>Dizziness is one of the adverse reactions that...</td>\n",
       "      <td>Dizziness drawiness,extreme thirst,fast, shall...</td>\n",
       "      <td>https://www.fda.gov/drugs/science-and-research...</td>\n",
       "      <td>['1. \\nGanoci L et al. ABCB1, ABCG2 and CYP2D6...</td>\n",
       "      <td>Based on the literature provided, there is ev...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The information provided in the response is ba...</td>\n",
       "      <td>I can help you understand the relationship bet...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>149</td>\n",
       "      <td>My patient has CYP2D6 poor metabolism and does...</td>\n",
       "      <td>CPIC guideline for ondansetron and CYP2D6 has ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[\"Drug: sertraline\\nGene1: CYP2C19\\nPhenotype1...</td>\n",
       "      <td>Based on the provided data, there are no spec...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The guidelines for prescribing sertraline for ...</td>\n",
       "      <td>The guidelines provided do not specifically ad...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>166</td>\n",
       "      <td>What are side effects associated wtih DPYD poo...</td>\n",
       "      <td>5-FU has many adverse drug effects when used s...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['Drug: lornoxicam\\nGene1: CYP2C9\\nPhenotype1:...</td>\n",
       "      <td>The HLAB system is not directly involved in p...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>HLA-B and CYP2C9 genetic polymorphisms are imp...</td>\n",
       "      <td>According to the provided pharmacogenomics inf...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>167</td>\n",
       "      <td>What are the pharmacogenomic panel advised to ...</td>\n",
       "      <td>G6PD, CYP2D6, UGT1A1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['{\"citations\": [{\"id\": 15143242, \"title\": \"Cl...</td>\n",
       "      <td>The given text suggests that for a patient wi...</td>\n",
       "      <td>The provided text suggests that due to the pat...</td>\n",
       "      <td>Based on the information provided in the liter...</td>\n",
       "      <td>Based on the provided information, for a patie...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>168</td>\n",
       "      <td>I have a 7 year old patient with HLA-B1502 pos...</td>\n",
       "      <td>Sodium Valproate</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['9\\n. \\nKristensen MH et al. Variants in the ...</td>\n",
       "      <td>The side effects associated with DPD (dihydro...</td>\n",
       "      <td>Patients with DPYD poor metabolism of fluorour...</td>\n",
       "      <td>Based on the provided document, the side effec...</td>\n",
       "      <td>According to the provided document, individual...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>171</td>\n",
       "      <td>What are the adverse outcomes associated with ...</td>\n",
       "      <td>Patients with the rs1801133 AA genotype and Ar...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[\"Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1...</td>\n",
       "      <td>To provide an appropriate pharmacogenomic (PG...</td>\n",
       "      <td>The provided text does not contain any informa...</td>\n",
       "      <td>The answer is not explicitly stated in the giv...</td>\n",
       "      <td>The input document doesn't contain information...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>134 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Unnamed: 0                                          Questions  \\\n",
       "0             0  As part of my liver transplant, I take tacroli...   \n",
       "1             1  I'm Indian, and my family has a history of epi...   \n",
       "2             2  Allopurinol is what I take for gout. For my se...   \n",
       "3             3  Is it true that while taking clopidogrel, I sh...   \n",
       "4             4  Dizziness is one of the adverse reactions that...   \n",
       "..          ...                                                ...   \n",
       "129         149  My patient has CYP2D6 poor metabolism and does...   \n",
       "130         166  What are side effects associated wtih DPYD poo...   \n",
       "131         167  What are the pharmacogenomic panel advised to ...   \n",
       "132         168  I have a 7 year old patient with HLA-B1502 pos...   \n",
       "133         171  What are the adverse outcomes associated with ...   \n",
       "\n",
       "                                     Expected_response  \\\n",
       "0    For a CYP3A5 extensive metabolizer. Increase s...   \n",
       "1    As per CPIC HLA-B*15:02 might be observed in 2...   \n",
       "2    There can be moderate interaction while consid...   \n",
       "3    Indeed, you should refrain from drinking grape...   \n",
       "4    Dizziness drawiness,extreme thirst,fast, shall...   \n",
       "..                                                 ...   \n",
       "129  CPIC guideline for ondansetron and CYP2D6 has ...   \n",
       "130  5-FU has many adverse drug effects when used s...   \n",
       "131                               G6PD, CYP2D6, UGT1A1   \n",
       "132                                   Sodium Valproate   \n",
       "133  Patients with the rs1801133 AA genotype and Ar...   \n",
       "\n",
       "                                       Expected_source  \\\n",
       "0    https://www.fda.gov/drugs/science-and-research...   \n",
       "1    https://www.pharmgkb.org/haplotype/PA165954769...   \n",
       "2    https://insilico-cyp.charite.de/SuperCYPsPred/...   \n",
       "3    Bailey DG, Malcolm J, Arnold O, Spence JD. Gra...   \n",
       "4    https://www.fda.gov/drugs/science-and-research...   \n",
       "..                                                 ...   \n",
       "129                                                NaN   \n",
       "130                                                NaN   \n",
       "131                                                NaN   \n",
       "132                                                NaN   \n",
       "133                                                NaN   \n",
       "\n",
       "                                               context  \\\n",
       "0    ['recipients. Pharmacogenet Genomics 2008;18:3...   \n",
       "1    ['{\"citations\": [{\"id\": 15071835, \"title\": \"Re...   \n",
       "2    ['that a stronger inhibition of the uric acid ...   \n",
       "3    ['Drug: clopidogrel\\nGene1: CYP2C19\\nPhenotype...   \n",
       "4    ['1. \\nGanoci L et al. ABCB1, ABCG2 and CYP2D6...   \n",
       "..                                                 ...   \n",
       "129  [\"Drug: sertraline\\nGene1: CYP2C19\\nPhenotype1...   \n",
       "130  ['Drug: lornoxicam\\nGene1: CYP2C9\\nPhenotype1:...   \n",
       "131  ['{\"citations\": [{\"id\": 15143242, \"title\": \"Cl...   \n",
       "132  ['9\\n. \\nKristensen MH et al. Variants in the ...   \n",
       "133  [\"Drug: omeprazole\\nGene1: CYP2C19\\nPhenotype1...   \n",
       "\n",
       "                                          mistral_resp  \\\n",
       "0     Based on your test results, if you are an ext...   \n",
       "1     The genetic variant HLA-B*15:02 that you have...   \n",
       "2     Answer: Phenytoin and allopurinol are two dis...   \n",
       "3     While taking clopidogrel, it is generally rec...   \n",
       "4     Based on the literature provided, there is ev...   \n",
       "..                                                 ...   \n",
       "129   Based on the provided data, there are no spec...   \n",
       "130   The HLAB system is not directly involved in p...   \n",
       "131   The given text suggests that for a patient wi...   \n",
       "132   The side effects associated with DPD (dihydro...   \n",
       "133   To provide an appropriate pharmacogenomic (PG...   \n",
       "\n",
       "                                            gemma_resp  \\\n",
       "0    The provided text does not contain any informa...   \n",
       "1    The provided text does not contain any informa...   \n",
       "2    The provided text does not contain any informa...   \n",
       "3    The provided text does not contain information...   \n",
       "4    The provided text does not contain any informa...   \n",
       "..                                                 ...   \n",
       "129  The provided text does not contain any informa...   \n",
       "130  The provided text does not contain any informa...   \n",
       "131  The provided text suggests that due to the pat...   \n",
       "132  Patients with DPYD poor metabolism of fluorour...   \n",
       "133  The provided text does not contain any informa...   \n",
       "\n",
       "                                           llama2_resp  \\\n",
       "0    Based on the information provided in the docum...   \n",
       "1    The information you provided is from a scienti...   \n",
       "2    It is understandable to have concerns about po...   \n",
       "3    Yes, it is true that clopidogrel can interact ...   \n",
       "4    The information provided in the response is ba...   \n",
       "..                                                 ...   \n",
       "129  The guidelines for prescribing sertraline for ...   \n",
       "130  HLA-B and CYP2C9 genetic polymorphisms are imp...   \n",
       "131  Based on the information provided in the liter...   \n",
       "132  Based on the provided document, the side effec...   \n",
       "133  The answer is not explicitly stated in the giv...   \n",
       "\n",
       "                                           llama3_resp  \n",
       "0    According to your pharmacogenetic test results...  \n",
       "1    I'm an AI assistant trained to provide underst...  \n",
       "2    A great question!\\n\\nAs an AI assistant traine...  \n",
       "3    A pharmacogenomics enthusiast!\\n\\nAccording to...  \n",
       "4    I can help you understand the relationship bet...  \n",
       "..                                                 ...  \n",
       "129  The guidelines provided do not specifically ad...  \n",
       "130  According to the provided pharmacogenomics inf...  \n",
       "131  Based on the provided information, for a patie...  \n",
       "132  According to the provided document, individual...  \n",
       "133  The input document doesn't contain information...  \n",
       "\n",
       "[134 rows x 9 columns]"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "quests = quests.drop([\"Unnamed: 0.1\"], axis=1)\n",
    "quests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "159d63fc120c4372946e348cec13b924",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Evaluating:   0%|          | 0/1072 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread Thread-90:\n",
      "Traceback (most recent call last):\n",
      "  File \"/usr/lib/python3.10/threading.py\", line 1016, in _bootstrap_inner\n",
      "    self.run()\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/executor.py\", line 96, in run\n",
      "    results = self.loop.run_until_complete(self._aresults())\n",
      "  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n",
      "    return future.result()\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/executor.py\", line 84, in _aresults\n",
      "    raise e\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/executor.py\", line 79, in _aresults\n",
      "    r = await future\n",
      "  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 571, in _wait_for_one\n",
      "    return f.result()  # May raise f.exception().\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/executor.py\", line 38, in sema_coro\n",
      "    return await coro\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/executor.py\", line 112, in wrapped_callable_async\n",
      "    return counter, await callable(*args, **kwargs)\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/metrics/base.py\", line 123, in ascore\n",
      "    raise e\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/metrics/base.py\", line 119, in ascore\n",
      "    score = await self._ascore(row=row, callbacks=group_cm, is_async=is_async)\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/metrics/_context_entities_recall.py\", line 183, in _ascore\n",
      "    contexts = await self.get_entities(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/metrics/_context_entities_recall.py\", line 158, in get_entities\n",
      "    result = await self.llm.generate(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/llms/base.py\", line 92, in generate\n",
      "    return await agenerate_text_with_retry(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/tenacity/_asyncio.py\", line 88, in async_wrapped\n",
      "    return await fn(*args, **kwargs)\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/tenacity/_asyncio.py\", line 47, in __call__\n",
      "    do = self.iter(retry_state=retry_state)\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/tenacity/__init__.py\", line 325, in iter\n",
      "    raise retry_exc.reraise()\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/tenacity/__init__.py\", line 158, in reraise\n",
      "    raise self.last_attempt.result()\n",
      "  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n",
      "    return self.__get_result()\n",
      "  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n",
      "    raise self._exception\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/tenacity/_asyncio.py\", line 50, in __call__\n",
      "    result = await fn(*args, **kwargs)\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/llms/base.py\", line 169, in agenerate_text\n",
      "    return await self.langchain_llm.agenerate_prompt(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/language_models/chat_models.py\", line 609, in agenerate_prompt\n",
      "    return await self.agenerate(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/language_models/chat_models.py\", line 569, in agenerate\n",
      "    raise exceptions[0]\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_core/language_models/chat_models.py\", line 754, in _agenerate_with_cache\n",
      "    result = await self._agenerate(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/langchain_openai/chat_models/base.py\", line 623, in _agenerate\n",
      "    response = await self.async_client.create(messages=message_dicts, **params)\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/resources/chat/completions.py\", line 1159, in create\n",
      "    return await self._post(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/_base_client.py\", line 1790, in post\n",
      "    return await self.request(cast_to, opts, stream=stream, stream_cls=stream_cls)\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/_base_client.py\", line 1493, in request\n",
      "    return await self._request(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/_base_client.py\", line 1569, in _request\n",
      "    return await self._retry_request(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/_base_client.py\", line 1615, in _retry_request\n",
      "    return await self._request(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/_base_client.py\", line 1569, in _request\n",
      "    return await self._retry_request(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/_base_client.py\", line 1615, in _retry_request\n",
      "    return await self._request(\n",
      "  File \"/home/dhanushb/Wellytics/AIdemo/lib/python3.10/site-packages/openai/_base_client.py\", line 1584, in _request\n",
      "    raise self._make_status_error_from_response(err.response) from None\n",
      "openai.RateLimitError: Error code: 429 - {'error': {'message': 'Rate limit reached for gpt-3.5-turbo in organization org-RZekQoHoeej4RpRvyhheWDit on tokens per min (TPM): Limit 160000, Used 153599, Requested 11346. Please try again in 1.854s. Visit https://platform.openai.com/account/rate-limits to learn more.', 'type': 'tokens', 'param': None, 'code': 'rate_limit_exceeded'}}\n"
     ]
    },
    {
     "ename": "ExceptionInRunner",
     "evalue": "The runner thread which was running the jobs raised an exeception. Read the traceback above to debug it. You can also pass `raise_exceptions=False` incase you want to show only a warning message instead.",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mExceptionInRunner\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[64], line 25\u001b[0m\n\u001b[1;32m     22\u001b[0m \u001b[38;5;66;03m# Convert dict to dataset\u001b[39;00m\n\u001b[1;32m     23\u001b[0m dataset \u001b[38;5;241m=\u001b[39m Dataset\u001b[38;5;241m.\u001b[39mfrom_dict(data)\n\u001b[0;32m---> 25\u001b[0m result \u001b[38;5;241m=\u001b[39m \u001b[43mevaluate\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m     26\u001b[0m \u001b[43m    \u001b[49m\u001b[43mdataset\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m \u001b[49m\u001b[43mdataset\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\n\u001b[1;32m     27\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmetrics\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m[\u001b[49m\n\u001b[1;32m     28\u001b[0m \u001b[43m        \u001b[49m\u001b[43mcontext_entity_recall\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     29\u001b[0m \u001b[43m        \u001b[49m\u001b[43mcontext_precision\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     30\u001b[0m \u001b[43m        \u001b[49m\u001b[43mcontext_recall\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     31\u001b[0m \u001b[43m        \u001b[49m\u001b[43mfaithfulness\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     32\u001b[0m \u001b[43m        \u001b[49m\u001b[43manswer_relevancy\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     33\u001b[0m \u001b[43m        \u001b[49m\u001b[43manswer_similarity\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     34\u001b[0m \u001b[43m        \u001b[49m\u001b[43manswer_correctness\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     35\u001b[0m \u001b[43m        \u001b[49m\u001b[43mharmfulness\u001b[49m\n\u001b[1;32m     36\u001b[0m \u001b[43m    \u001b[49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m     37\u001b[0m \u001b[43m)\u001b[49m\n\u001b[1;32m     38\u001b[0m df \u001b[38;5;241m=\u001b[39m result\u001b[38;5;241m.\u001b[39mto_pandas()\n\u001b[1;32m     39\u001b[0m df\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/evaluation.py:250\u001b[0m, in \u001b[0;36mevaluate\u001b[0;34m(dataset, metrics, llm, embeddings, callbacks, in_ci, is_async, run_config, raise_exceptions, column_map)\u001b[0m\n\u001b[1;32m    247\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m evaluation_group_cm\u001b[38;5;241m.\u001b[39mended:\n\u001b[1;32m    248\u001b[0m         evaluation_rm\u001b[38;5;241m.\u001b[39mon_chain_error(e)\n\u001b[0;32m--> 250\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m e\n\u001b[1;32m    251\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    252\u001b[0m     result \u001b[38;5;241m=\u001b[39m Result(\n\u001b[1;32m    253\u001b[0m         scores\u001b[38;5;241m=\u001b[39mDataset\u001b[38;5;241m.\u001b[39mfrom_list(scores),\n\u001b[1;32m    254\u001b[0m         dataset\u001b[38;5;241m=\u001b[39mdataset,\n\u001b[1;32m    255\u001b[0m         binary_columns\u001b[38;5;241m=\u001b[39mbinary_metrics,\n\u001b[1;32m    256\u001b[0m     )\n",
      "File \u001b[0;32m~/Wellytics/AIdemo/lib/python3.10/site-packages/ragas/evaluation.py:232\u001b[0m, in \u001b[0;36mevaluate\u001b[0;34m(dataset, metrics, llm, embeddings, callbacks, in_ci, is_async, run_config, raise_exceptions, column_map)\u001b[0m\n\u001b[1;32m    230\u001b[0m results \u001b[38;5;241m=\u001b[39m executor\u001b[38;5;241m.\u001b[39mresults()\n\u001b[1;32m    231\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m results \u001b[38;5;241m==\u001b[39m []:\n\u001b[0;32m--> 232\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m ExceptionInRunner()\n\u001b[1;32m    234\u001b[0m \u001b[38;5;66;03m# convert results to dataset_like\u001b[39;00m\n\u001b[1;32m    235\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m i, _ \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28menumerate\u001b[39m(dataset):\n",
      "\u001b[0;31mExceptionInRunner\u001b[0m: The runner thread which was running the jobs raised an exeception. Read the traceback above to debug it. You can also pass `raise_exceptions=False` incase you want to show only a warning message instead."
     ]
    }
   ],
   "source": [
    "from datasets import Dataset\n",
    "from ragas import evaluate\n",
    "from ragas.metrics.critique import harmfulness\n",
    "from ragas.metrics import (\n",
    "    faithfulness,\n",
    "    answer_relevancy,\n",
    "    context_recall,\n",
    "    context_precision,\n",
    "    context_entity_recall,\n",
    "    answer_similarity,\n",
    "    answer_correctness,\n",
    ")\n",
    "\n",
    "# To dict\n",
    "data = {\n",
    "    \"question\": list(map(str, quests[\"Questions\"].to_list())),\n",
    "    \"answer\": list(map(str, quests[\"mistral_resp\"].to_list())),\n",
    "    \"contexts\": [[str(context)] for context in quests[\"context\"].to_list()],\n",
    "    \"ground_truth\": list(map(str, quests[\"Expected_response\"].to_list()))\n",
    "}\n",
    "\n",
    "# Convert dict to dataset\n",
    "dataset = Dataset.from_dict(data)\n",
    "\n",
    "result = evaluate(\n",
    "    dataset = dataset, \n",
    "    metrics=[\n",
    "        context_entity_recall,\n",
    "        context_precision,\n",
    "        context_recall,\n",
    "        faithfulness,\n",
    "        answer_relevancy,\n",
    "        answer_similarity,\n",
    "        answer_correctness,\n",
    "        harmfulness\n",
    "    ],\n",
    ")\n",
    "df = result.to_pandas()\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
